{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "53deaf67",
   "metadata": {},
   "source": [
    "# TRAINING: DeBERTa, early_stopping_patience = 4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "d77c04f6",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "2024-04-04 14:03:06.269680: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA\n",
      "To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.\n",
      "2024-04-04 14:03:07.275285: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer.so.7'; dlerror: libnvinfer.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/nvidia/lib:/usr/local/nvidia/lib64\n",
      "2024-04-04 14:03:07.275367: W tensorflow/compiler/xla/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libnvinfer_plugin.so.7'; dlerror: libnvinfer_plugin.so.7: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /usr/local/nvidia/lib:/usr/local/nvidia/lib64\n",
      "2024-04-04 14:03:07.275374: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Cannot dlopen some TensorRT libraries. If you would like to use Nvidia GPU with TensorRT, please make sure the missing libraries mentioned above are installed properly.\n"
     ]
    }
   ],
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "import time\n",
    "\n",
    "from sklearn.metrics import classification_report, confusion_matrix, accuracy_score, recall_score, precision_score, f1_score\n",
    "\n",
    "import datasets\n",
    "from datasets import Dataset\n",
    "from transformers import AutoTokenizer\n",
    "from transformers import DataCollatorWithPadding\n",
    "from transformers import AutoModelForSequenceClassification, TrainingArguments, Trainer\n",
    "from transformers import EarlyStoppingCallback, IntervalStrategy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "9923c890",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/transformers/convert_slow_tokenizer.py:455: UserWarning: The sentencepiece tokenizer that you are converting to a fast tokenizer uses the byte fallback option which is not implemented in the fast tokenizers. In practice this means that the fast version of the tokenizer can produce unknown tokens whereas the sentencepiece version would have converted these unknown tokens into a sequence of byte tokens matching the original piece of text.\n",
      "  \"The sentencepiece tokenizer that you are converting to a fast tokenizer uses the byte fallback option\"\n",
      "Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.\n"
     ]
    }
   ],
   "source": [
    "model_name = \"microsoft/deberta-v3-large\"\n",
    "\n",
    "tokenizer = AutoTokenizer.from_pretrained(model_name)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "0da146f6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>claim</th>\n",
       "      <th>abstract</th>\n",
       "      <th>label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Angiotensin converting enzyme inhibitors are a...</td>\n",
       "      <td>Renal considerations in angiotensin converting...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Reducing H3k4me3 methylation induces mouse epi...</td>\n",
       "      <td>MLL1 Inhibition Reprograms Epiblast Stem Cells...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Expression of oncolytic virus antigens as pept...</td>\n",
       "      <td>Detecting and targeting tumor relapse by its r...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Varenicline monotherapy is more effective afte...</td>\n",
       "      <td>Combination varenicline and bupropion SR for t...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Acute ablation of Snail in the embryonic corte...</td>\n",
       "      <td>Control of Apoptosis by Asymmetric Cell Divisi...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>966</th>\n",
       "      <td>Higher plasma levels of I-carnitine, when asso...</td>\n",
       "      <td>Intestinal microbiota metabolism of L-carnitin...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>967</th>\n",
       "      <td>Women with a higher birth weight are more like...</td>\n",
       "      <td>Birth Size and Breast Cancer Risk: Re-analysis...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>968</th>\n",
       "      <td>In domesticated populations of Saccharomyces c...</td>\n",
       "      <td>Domestication and Divergence of Saccharomyces ...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>Active Ly49Q induces neutrophil polarization.</td>\n",
       "      <td>The Ly49Q receptor plays a crucial role in neu...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>970</th>\n",
       "      <td>ADAR1 binds to Dicer to cleave pre-miRNA.</td>\n",
       "      <td>ADAR1 Forms a Complex with Dicer to Promote Mi...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>971 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 claim  \\\n",
       "0    Angiotensin converting enzyme inhibitors are a...   \n",
       "1    Reducing H3k4me3 methylation induces mouse epi...   \n",
       "2    Expression of oncolytic virus antigens as pept...   \n",
       "3    Varenicline monotherapy is more effective afte...   \n",
       "4    Acute ablation of Snail in the embryonic corte...   \n",
       "..                                                 ...   \n",
       "966  Higher plasma levels of I-carnitine, when asso...   \n",
       "967  Women with a higher birth weight are more like...   \n",
       "968  In domesticated populations of Saccharomyces c...   \n",
       "969      Active Ly49Q induces neutrophil polarization.   \n",
       "970          ADAR1 binds to Dicer to cleave pre-miRNA.   \n",
       "\n",
       "                                              abstract  label  \n",
       "0    Renal considerations in angiotensin converting...      1  \n",
       "1    MLL1 Inhibition Reprograms Epiblast Stem Cells...      1  \n",
       "2    Detecting and targeting tumor relapse by its r...      0  \n",
       "3    Combination varenicline and bupropion SR for t...      2  \n",
       "4    Control of Apoptosis by Asymmetric Cell Divisi...      0  \n",
       "..                                                 ...    ...  \n",
       "966  Intestinal microbiota metabolism of L-carnitin...      1  \n",
       "967  Birth Size and Breast Cancer Risk: Re-analysis...      1  \n",
       "968  Domestication and Divergence of Saccharomyces ...      0  \n",
       "969  The Ly49Q receptor plays a crucial role in neu...      1  \n",
       "970  ADAR1 Forms a Complex with Dicer to Promote Mi...      1  \n",
       "\n",
       "[971 rows x 3 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "with open('./SciFact_train.csv', 'rb') as fh:\n",
    "    df_train = pd.read_csv(fh)\n",
    "df_train.drop('Unnamed: 0', axis=1, inplace=True)\n",
    "df_train"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "b5aada8a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>claim</th>\n",
       "      <th>abstract</th>\n",
       "      <th>label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MICAL redox enzymes regulate actin dynamics in...</td>\n",
       "      <td>Drosophila MICAL regulates myofilament organiz...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>The cytokine receptor which regulates chemotax...</td>\n",
       "      <td>Myeloperoxidase acts as a profibrotic mediator...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>RUNX1 is downregulated or mutated in TLX1 T-ALL.</td>\n",
       "      <td>Reverse engineering of TLX oncogenic transcrip...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>There is no relation between Erythromycin use ...</td>\n",
       "      <td>Use of macrolides in mother and child and risk...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Risk of cardiovascular events can be cut by a ...</td>\n",
       "      <td>Cardiovascular protection with antihypertensiv...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>Genes involved in pre-mRNA splicing have a sig...</td>\n",
       "      <td>A genome-wide siRNA screen reveals diverse cel...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>Metastatic colorectal cancer treated with a si...</td>\n",
       "      <td>Chemotherapy options in elderly and frail pati...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>Lack of clpC does not affect sporulation effic...</td>\n",
       "      <td>High-Throughput Genetic Screens Identify a Lar...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>GATA3 regulates cell cycle progression in bone...</td>\n",
       "      <td>GATA-3 regulates hematopoietic stem cell maint...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>Chenodeoxycholic acid treatment increases brow...</td>\n",
       "      <td>The Bile Acid Chenodeoxycholic Acid Increases ...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>120 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 claim  \\\n",
       "0    MICAL redox enzymes regulate actin dynamics in...   \n",
       "1    The cytokine receptor which regulates chemotax...   \n",
       "2     RUNX1 is downregulated or mutated in TLX1 T-ALL.   \n",
       "3    There is no relation between Erythromycin use ...   \n",
       "4    Risk of cardiovascular events can be cut by a ...   \n",
       "..                                                 ...   \n",
       "115  Genes involved in pre-mRNA splicing have a sig...   \n",
       "116  Metastatic colorectal cancer treated with a si...   \n",
       "117  Lack of clpC does not affect sporulation effic...   \n",
       "118  GATA3 regulates cell cycle progression in bone...   \n",
       "119  Chenodeoxycholic acid treatment increases brow...   \n",
       "\n",
       "                                              abstract  label  \n",
       "0    Drosophila MICAL regulates myofilament organiz...      1  \n",
       "1    Myeloperoxidase acts as a profibrotic mediator...      0  \n",
       "2    Reverse engineering of TLX oncogenic transcrip...      1  \n",
       "3    Use of macrolides in mother and child and risk...      2  \n",
       "4    Cardiovascular protection with antihypertensiv...      1  \n",
       "..                                                 ...    ...  \n",
       "115  A genome-wide siRNA screen reveals diverse cel...      1  \n",
       "116  Chemotherapy options in elderly and frail pati...      2  \n",
       "117  High-Throughput Genetic Screens Identify a Lar...      0  \n",
       "118  GATA-3 regulates hematopoietic stem cell maint...      1  \n",
       "119  The Bile Acid Chenodeoxycholic Acid Increases ...      1  \n",
       "\n",
       "[120 rows x 3 columns]"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "with open('./SciFact_valid.csv', 'rb') as fh:\n",
    "    df_valid = pd.read_csv(fh)\n",
    "df_valid.drop('Unnamed: 0', axis=1, inplace=True)\n",
    "df_valid"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "26b9974c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>claim</th>\n",
       "      <th>abstract</th>\n",
       "      <th>label</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Women are more susceptible to death due to pne...</td>\n",
       "      <td>Gender-dependent differences in outcome after ...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Cellular clocks are not predictive of mitosis ...</td>\n",
       "      <td>Robust synchronization of coupled circadian an...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Cells undergoing methionine restriction may ac...</td>\n",
       "      <td>microRNAs: A Safeguard against Turmoil? Emergi...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Replacement of OCT4 and SOX2 genes with GATA3 ...</td>\n",
       "      <td>Reprogramming of human fibroblasts to pluripot...</td>\n",
       "      <td>0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>IgA plasma cells that are specific for transgl...</td>\n",
       "      <td>High abundance of plasma cells secreting trans...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>Charcoal shows no benefit for acute paraquat p...</td>\n",
       "      <td>Effect of activated charcoal hemoperfusion on ...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>The risk of breast cancer among parous women d...</td>\n",
       "      <td>Pregnancy characteristics and maternal risk of...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>Inositol lipid 3-phosphatase PTEN converts Ptd...</td>\n",
       "      <td>PTEN Regulates PI(3,4)P2 Signaling Downstream ...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>Combination nicotine replacement therapies wit...</td>\n",
       "      <td>Combination varenicline and bupropion SR for t...</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>Women with a lower birth weight are more likel...</td>\n",
       "      <td>Intrauterine factors and risk of breast cancer...</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>122 rows × 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 claim  \\\n",
       "0    Women are more susceptible to death due to pne...   \n",
       "1    Cellular clocks are not predictive of mitosis ...   \n",
       "2    Cells undergoing methionine restriction may ac...   \n",
       "3    Replacement of OCT4 and SOX2 genes with GATA3 ...   \n",
       "4    IgA plasma cells that are specific for transgl...   \n",
       "..                                                 ...   \n",
       "117  Charcoal shows no benefit for acute paraquat p...   \n",
       "118  The risk of breast cancer among parous women d...   \n",
       "119  Inositol lipid 3-phosphatase PTEN converts Ptd...   \n",
       "120  Combination nicotine replacement therapies wit...   \n",
       "121  Women with a lower birth weight are more likel...   \n",
       "\n",
       "                                              abstract  label  \n",
       "0    Gender-dependent differences in outcome after ...      1  \n",
       "1    Robust synchronization of coupled circadian an...      2  \n",
       "2    microRNAs: A Safeguard against Turmoil? Emergi...      0  \n",
       "3    Reprogramming of human fibroblasts to pluripot...      0  \n",
       "4    High abundance of plasma cells secreting trans...      2  \n",
       "..                                                 ...    ...  \n",
       "117  Effect of activated charcoal hemoperfusion on ...      1  \n",
       "118  Pregnancy characteristics and maternal risk of...      2  \n",
       "119  PTEN Regulates PI(3,4)P2 Signaling Downstream ...      1  \n",
       "120  Combination varenicline and bupropion SR for t...      1  \n",
       "121  Intrauterine factors and risk of breast cancer...      2  \n",
       "\n",
       "[122 rows x 3 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "with open('./SciFact_test.csv', 'rb') as fh:\n",
    "    df_test = pd.read_csv(fh)\n",
    "df_test.drop('Unnamed: 0', axis=1, inplace=True)\n",
    "df_test"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "a80595ae",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>label</th>\n",
       "      <th>text</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1</td>\n",
       "      <td>[CLS]Angiotensin converting enzyme inhibitors ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1</td>\n",
       "      <td>[CLS]Reducing H3k4me3 methylation induces mous...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>0</td>\n",
       "      <td>[CLS]Expression of oncolytic virus antigens as...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2</td>\n",
       "      <td>[CLS]Varenicline monotherapy is more effective...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>0</td>\n",
       "      <td>[CLS]Acute ablation of Snail in the embryonic ...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>966</th>\n",
       "      <td>1</td>\n",
       "      <td>[CLS]Higher plasma levels of I-carnitine, when...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>967</th>\n",
       "      <td>1</td>\n",
       "      <td>[CLS]Women with a higher birth weight are more...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>968</th>\n",
       "      <td>0</td>\n",
       "      <td>[CLS]In domesticated populations of Saccharomy...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>969</th>\n",
       "      <td>1</td>\n",
       "      <td>[CLS]Active Ly49Q induces neutrophil polarizat...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>970</th>\n",
       "      <td>1</td>\n",
       "      <td>[CLS]ADAR1 binds to Dicer to cleave pre-miRNA....</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>971 rows × 2 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     label                                               text\n",
       "0        1  [CLS]Angiotensin converting enzyme inhibitors ...\n",
       "1        1  [CLS]Reducing H3k4me3 methylation induces mous...\n",
       "2        0  [CLS]Expression of oncolytic virus antigens as...\n",
       "3        2  [CLS]Varenicline monotherapy is more effective...\n",
       "4        0  [CLS]Acute ablation of Snail in the embryonic ...\n",
       "..     ...                                                ...\n",
       "966      1  [CLS]Higher plasma levels of I-carnitine, when...\n",
       "967      1  [CLS]Women with a higher birth weight are more...\n",
       "968      0  [CLS]In domesticated populations of Saccharomy...\n",
       "969      1  [CLS]Active Ly49Q induces neutrophil polarizat...\n",
       "970      1  [CLS]ADAR1 binds to Dicer to cleave pre-miRNA....\n",
       "\n",
       "[971 rows x 2 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_train['text'] = tokenizer.cls_token + df_train['claim'] + tokenizer.sep_token + df_train['abstract'] + tokenizer.sep_token\n",
    "df_train.drop(columns=['claim', 'abstract'], inplace=True)\n",
    "df_train_robertaL"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "7dbf3967",
   "metadata": {},
   "outputs": [],
   "source": [
    "df_valid['text'] = tokenizer.cls_token + df_valid['claim'] + tokenizer.sep_token + df_valid['abstract'] + tokenizer.sep_token\n",
    "df_valid.drop(columns=['claim', 'abstract'], inplace=True)\n",
    "\n",
    "df_test['text'] = tokenizer.cls_token + df_test['claim'] + tokenizer.sep_token + df_test['abstract'] + tokenizer.sep_token\n",
    "df_test.drop(columns=['claim', 'abstract'], inplace=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "8405de1a",
   "metadata": {},
   "outputs": [],
   "source": [
    "konacno_train = Dataset.from_pandas(df_train)\n",
    "konacno_valid = Dataset.from_pandas(df_valid)\n",
    "konacno_test = Dataset.from_pandas(df_test)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "0dbefa04",
   "metadata": {},
   "outputs": [],
   "source": [
    "my_dataset_dict = datasets.DatasetDict({\n",
    "    'train': konacno_train,\n",
    "    'valid': konacno_valid,\n",
    "    'test': konacno_test\n",
    "})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "c891ad3f",
   "metadata": {},
   "outputs": [],
   "source": [
    "def preprocess_function(examples):\n",
    "    max_input_length = 512\n",
    "    return tokenizer(examples[\"text\"], truncation=True, padding=True, max_length=max_input_length)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "60fa4150",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "0ab4bd8139234019a4dd322da361aa1f",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "  0%|          | 0/1 [00:00<?, ?ba/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "tokenized = my_dataset_dict.map(preprocess_function, batched=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "414d2aea",
   "metadata": {},
   "outputs": [],
   "source": [
    "data_collator = DataCollatorWithPadding(tokenizer=tokenizer)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "3daadac0",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Some weights of the model checkpoint at microsoft/deberta-v3-large were not used when initializing DebertaV2ForSequenceClassification: ['mask_predictions.dense.weight', 'lm_predictions.lm_head.dense.bias', 'lm_predictions.lm_head.bias', 'lm_predictions.lm_head.dense.weight', 'lm_predictions.lm_head.LayerNorm.weight', 'mask_predictions.dense.bias', 'lm_predictions.lm_head.LayerNorm.bias', 'mask_predictions.classifier.weight', 'mask_predictions.LayerNorm.weight', 'mask_predictions.LayerNorm.bias', 'mask_predictions.classifier.bias']\n",
      "- This IS expected if you are initializing DebertaV2ForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
      "- This IS NOT expected if you are initializing DebertaV2ForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n",
      "Some weights of DebertaV2ForSequenceClassification were not initialized from the model checkpoint at microsoft/deberta-v3-large and are newly initialized: ['classifier.bias', 'classifier.weight', 'pooler.dense.bias', 'pooler.dense.weight']\n",
      "You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n"
     ]
    }
   ],
   "source": [
    "model = AutoModelForSequenceClassification.from_pretrained(model_name, num_labels=3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "f7202461",
   "metadata": {},
   "outputs": [],
   "source": [
    "def compute_metrics(p):    \n",
    "    pred, labels = p\n",
    "    pred = np.argmax(pred, axis=1)\n",
    "    \n",
    "    accuracy = accuracy_score(y_true=labels, y_pred=pred)\n",
    "    recall = recall_score(y_true=labels, y_pred=pred, average='weighted')\n",
    "    precision = precision_score(y_true=labels, y_pred=pred, average='weighted')\n",
    "    f1 = f1_score(y_true=labels, y_pred=pred, average='weighted')    \n",
    "    return {\"accuracy\": accuracy, \"precision\": precision, \"recall\": recall, \"f1\": f1}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "a08b7b2b",
   "metadata": {},
   "outputs": [],
   "source": [
    "training_args = TrainingArguments(\n",
    "    output_dir=\"./SCIFACT_deberta_early4\",\n",
    "    learning_rate=1e-5,\n",
    "    per_device_train_batch_size=4,\n",
    "    per_device_eval_batch_size=4,\n",
    "    num_train_epochs=15,\n",
    "    save_total_limit = 5,\n",
    "    weight_decay=0.01,\n",
    "    evaluation_strategy=IntervalStrategy.STEPS,\n",
    "    eval_steps = 50,\n",
    "    #save_strategy=\"epoch\",\n",
    "    metric_for_best_model = 'f1',\n",
    "    load_best_model_at_end=True,\n",
    "    push_to_hub=False,\n",
    ")\n",
    "\n",
    "trainer = Trainer(\n",
    "    model=model,\n",
    "    args=training_args,\n",
    "    train_dataset=tokenized['train'],\n",
    "    eval_dataset=tokenized['valid'],\n",
    "    tokenizer=tokenizer,\n",
    "    data_collator=data_collator,\n",
    "    compute_metrics=compute_metrics,\n",
    "    callbacks=[EarlyStoppingCallback(early_stopping_patience=4)]\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "5e07dc90",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/transformers/optimization.py:415: FutureWarning: This implementation of AdamW is deprecated and will be removed in a future version. Use the PyTorch implementation torch.optim.AdamW instead, or set `no_deprecation_warning=True` to disable this warning\n",
      "  FutureWarning,\n",
      "You're using a DebertaV2TokenizerFast tokenizer. Please note that with a fast tokenizer, using the `__call__` method is faster than using a method to encode the text followed by a call to the `pad` method to get a padded encoding.\n"
     ]
    },
    {
     "data": {
      "text/html": [
       "\n",
       "    <div>\n",
       "      \n",
       "      <progress value='1750' max='3645' style='width:300px; height:20px; vertical-align: middle;'></progress>\n",
       "      [1750/3645 17:21 < 18:49, 1.68 it/s, Epoch 7/15]\n",
       "    </div>\n",
       "    <table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       " <tr style=\"text-align: left;\">\n",
       "      <th>Step</th>\n",
       "      <th>Training Loss</th>\n",
       "      <th>Validation Loss</th>\n",
       "      <th>Accuracy</th>\n",
       "      <th>Precision</th>\n",
       "      <th>Recall</th>\n",
       "      <th>F1</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <td>50</td>\n",
       "      <td>No log</td>\n",
       "      <td>1.064412</td>\n",
       "      <td>0.500000</td>\n",
       "      <td>0.394088</td>\n",
       "      <td>0.500000</td>\n",
       "      <td>0.439178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>100</td>\n",
       "      <td>No log</td>\n",
       "      <td>0.978910</td>\n",
       "      <td>0.533333</td>\n",
       "      <td>0.468738</td>\n",
       "      <td>0.533333</td>\n",
       "      <td>0.462983</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>150</td>\n",
       "      <td>No log</td>\n",
       "      <td>0.808694</td>\n",
       "      <td>0.700000</td>\n",
       "      <td>0.561105</td>\n",
       "      <td>0.700000</td>\n",
       "      <td>0.619164</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>200</td>\n",
       "      <td>No log</td>\n",
       "      <td>0.823899</td>\n",
       "      <td>0.708333</td>\n",
       "      <td>0.553501</td>\n",
       "      <td>0.708333</td>\n",
       "      <td>0.621076</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>250</td>\n",
       "      <td>No log</td>\n",
       "      <td>0.811878</td>\n",
       "      <td>0.691667</td>\n",
       "      <td>0.541890</td>\n",
       "      <td>0.691667</td>\n",
       "      <td>0.607684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>300</td>\n",
       "      <td>No log</td>\n",
       "      <td>0.843940</td>\n",
       "      <td>0.683333</td>\n",
       "      <td>0.567405</td>\n",
       "      <td>0.683333</td>\n",
       "      <td>0.604465</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>350</td>\n",
       "      <td>No log</td>\n",
       "      <td>1.189721</td>\n",
       "      <td>0.583333</td>\n",
       "      <td>0.464774</td>\n",
       "      <td>0.583333</td>\n",
       "      <td>0.499988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>400</td>\n",
       "      <td>No log</td>\n",
       "      <td>0.610991</td>\n",
       "      <td>0.725000</td>\n",
       "      <td>0.579398</td>\n",
       "      <td>0.725000</td>\n",
       "      <td>0.641019</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>450</td>\n",
       "      <td>No log</td>\n",
       "      <td>0.634677</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>0.828800</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>0.793913</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>500</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.507479</td>\n",
       "      <td>0.850000</td>\n",
       "      <td>0.848339</td>\n",
       "      <td>0.850000</td>\n",
       "      <td>0.848261</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>550</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.848513</td>\n",
       "      <td>0.775000</td>\n",
       "      <td>0.798059</td>\n",
       "      <td>0.775000</td>\n",
       "      <td>0.780978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>600</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.902545</td>\n",
       "      <td>0.825000</td>\n",
       "      <td>0.836549</td>\n",
       "      <td>0.825000</td>\n",
       "      <td>0.822188</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>650</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.608826</td>\n",
       "      <td>0.858333</td>\n",
       "      <td>0.861422</td>\n",
       "      <td>0.858333</td>\n",
       "      <td>0.856563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>700</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.857717</td>\n",
       "      <td>0.791667</td>\n",
       "      <td>0.797414</td>\n",
       "      <td>0.791667</td>\n",
       "      <td>0.789749</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>750</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.703258</td>\n",
       "      <td>0.858333</td>\n",
       "      <td>0.861432</td>\n",
       "      <td>0.858333</td>\n",
       "      <td>0.857947</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>800</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.881203</td>\n",
       "      <td>0.850000</td>\n",
       "      <td>0.850000</td>\n",
       "      <td>0.850000</td>\n",
       "      <td>0.850000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>850</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.921887</td>\n",
       "      <td>0.833333</td>\n",
       "      <td>0.835264</td>\n",
       "      <td>0.833333</td>\n",
       "      <td>0.833241</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>900</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>1.153631</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>0.822109</td>\n",
       "      <td>0.800000</td>\n",
       "      <td>0.800136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>950</td>\n",
       "      <td>0.840100</td>\n",
       "      <td>0.697188</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.881111</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.876389</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1000</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.640499</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.885461</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.883698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1050</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.641993</td>\n",
       "      <td>0.891667</td>\n",
       "      <td>0.892184</td>\n",
       "      <td>0.891667</td>\n",
       "      <td>0.891542</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1100</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.801292</td>\n",
       "      <td>0.866667</td>\n",
       "      <td>0.875768</td>\n",
       "      <td>0.866667</td>\n",
       "      <td>0.868092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1150</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.834576</td>\n",
       "      <td>0.866667</td>\n",
       "      <td>0.875768</td>\n",
       "      <td>0.866667</td>\n",
       "      <td>0.868092</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1200</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.709056</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.880261</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.875097</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1250</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.737562</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.887161</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.884106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1300</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.753582</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.887061</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.883384</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1350</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.807305</td>\n",
       "      <td>0.866667</td>\n",
       "      <td>0.871121</td>\n",
       "      <td>0.866667</td>\n",
       "      <td>0.867653</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1400</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.766246</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.888532</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.884684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1450</td>\n",
       "      <td>0.362300</td>\n",
       "      <td>0.717823</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.877438</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.874939</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1500</td>\n",
       "      <td>0.106700</td>\n",
       "      <td>0.725981</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.888532</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.884684</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1550</td>\n",
       "      <td>0.106700</td>\n",
       "      <td>0.702229</td>\n",
       "      <td>0.891667</td>\n",
       "      <td>0.894968</td>\n",
       "      <td>0.891667</td>\n",
       "      <td>0.892558</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1600</td>\n",
       "      <td>0.106700</td>\n",
       "      <td>0.708824</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.884101</td>\n",
       "      <td>0.883333</td>\n",
       "      <td>0.883610</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1650</td>\n",
       "      <td>0.106700</td>\n",
       "      <td>0.785390</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.877381</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.875768</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1700</td>\n",
       "      <td>0.106700</td>\n",
       "      <td>0.807137</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.876946</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.875649</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <td>1750</td>\n",
       "      <td>0.106700</td>\n",
       "      <td>0.866123</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.878994</td>\n",
       "      <td>0.875000</td>\n",
       "      <td>0.876262</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table><p>"
      ],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n",
      "/home/milos.kosprdic.ivi/.local/lib/python3.7/site-packages/sklearn/metrics/_classification.py:1318: UndefinedMetricWarning: Precision is ill-defined and being set to 0.0 in labels with no predicted samples. Use `zero_division` parameter to control this behavior.\n",
      "  _warn_prf(average, modifier, msg_start, len(result))\n"
     ]
    }
   ],
   "source": [
    "start_time = time.time()\n",
    "trainer.train()\n",
    "total_time = time.time()-start_time"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "9ff5d832",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [],
      "text/plain": [
       "<IPython.core.display.HTML object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "predictions = trainer.predict(tokenized['test'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "9a772078",
   "metadata": {},
   "outputs": [],
   "source": [
    "classifications = []\n",
    "for pred in predictions[0]:\n",
    "    classifications.append(np.argmax(pred))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "65542927",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "              precision    recall  f1-score   support\n",
      "\n",
      " NO_EVIDENCE       0.88      0.95      0.91        44\n",
      "     SUPPORT       0.90      0.88      0.89        51\n",
      "  CONTRADICT       0.88      0.78      0.82        27\n",
      "\n",
      "    accuracy                           0.89       122\n",
      "   macro avg       0.88      0.87      0.88       122\n",
      "weighted avg       0.89      0.89      0.88       122\n",
      "\n"
     ]
    }
   ],
   "source": [
    "target_names = ['NO_EVIDENCE', 'SUPPORT', 'CONTRADICT']\n",
    "print(classification_report(tokenized[\"test\"]['label'], classifications, target_names=target_names))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "4e393065",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array([[42,  1,  1],\n",
       "       [ 4, 45,  2],\n",
       "       [ 2,  4, 21]])"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "confusion_matrix(tokenized[\"test\"]['label'], classifications)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "bd727f4d",
   "metadata": {},
   "outputs": [],
   "source": [
    "trainer.save_model(\"DeBERTa_early4\")"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
