{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Combine BioASQ with Our IR"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import os\n",
    "import json\n",
    "import torch\n",
    "import time\n",
    "from opensearchpy import OpenSearch\n",
    "from sentence_transformers import SentenceTransformer\n",
    "\n",
    "import json\n",
    "import time\n",
    "from tqdm import tqdm  # Import tqdm for the progress bar"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Setting for queries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Connection opened...\n"
     ]
    }
   ],
   "source": [
    "host = '3.23.103.76' #host = 'localhost' \n",
    "port = 9200\n",
    "auth =('admin','IVIngi2024!') #auth = ('admin','admin') \n",
    "client_lexical = OpenSearch(\n",
    "    hosts = [{'host': host, 'port': port}],\n",
    "    http_auth = auth,\n",
    "    use_ssl = True,\n",
    "    verify_certs = False,\n",
    "    ssl_assert_hostname = False,\n",
    "    ssl_show_warn = False,\n",
    "    timeout=500, \n",
    "    max_retries=1\n",
    "    #connection_class=RequestsHttpConnection \n",
    "#    http_compress = True, # enables gzip compression for request bodies\n",
    "#    use_ssl = False,\n",
    "#   verify_certs = False,\n",
    "#    ssl_assert_hostname = False,\n",
    "#    ssl_show_warn = False\n",
    ")\n",
    "print(\"Connection opened...\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'took': 1, 'timed_out': False, '_shards': {'total': 4, 'successful': 4, 'skipped': 0, 'failed': 0}, 'hits': {'total': {'value': 1, 'relation': 'eq'}, 'max_score': 15.262341, 'hits': [{'_index': 'medline-faiss-hnsw-lexical-pmid', '_id': '15858239', '_score': 15.262341, '_source': {}}]}}\n"
     ]
    }
   ],
   "source": [
    "\n",
    "query_text={\n",
    "  \"query\": {\n",
    "    \"match\": {\n",
    "      \"pmid\": \"15858239\"\n",
    "    }\n",
    "  },\n",
    "  \"_source\": [\"text\"]  \n",
    "}\n",
    "\n",
    "# Execute the query\n",
    "response = client_lexical.search(\n",
    "    index=\"medline-faiss-hnsw-lexical-pmid\",\n",
    "    body=query_text\n",
    ")\n",
    "print(response)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Prepare data for generarive component"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {},
   "outputs": [],
   "source": [
    "#Get text from our IR for given pmid\n",
    "\n",
    "    \n",
    "def get_document_text(pmid, client):\n",
    "    # Format the query text with the correct PubMed ID\n",
    "    query_text = {\n",
    "        \"query\": {\n",
    "            \"match\": {\n",
    "                \"pmid\": pmid\n",
    "            }\n",
    "        },\n",
    "        \"_source\": [\"full_text\"]\n",
    "    }\n",
    "\n",
    "    # Use the provided OpenSearch client to execute the query\n",
    "    response = client_lexical.search(\n",
    "        index=\"medline-faiss-hnsw-lexical-pmid\",\n",
    "        body=query_text\n",
    "    )\n",
    "\n",
    "    # Check if the response has hits and return the cleaned text\n",
    "    if response['hits']['hits']:\n",
    "        # Extract the full text from the first hit\n",
    "        full_text = response['hits']['hits'][0]['_source']['full_text']\n",
    "        # Clean the text by removing new lines and excess white spaces\n",
    "        cleaned_text = ' '.join(full_text.replace('\\n', ' ').split())\n",
    "        return cleaned_text\n",
    "    else:\n",
    "        # Return a default message or handle as needed\n",
    "        return f\"No document found for PMID: {pmid}\"\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Phase 3 Studies Comparing Brodalumab with Ustekinumab in Psoriasis. BACKGROUND Early clinical studies suggested that the anti-interleukin-17 receptor A monoclonal antibody brodalumab has efficacy in the treatment of psoriasis. METHODS In two phase 3 studies (AMAGINE-2 and AMAGINE-3), patients with moderate-to-severe psoriasis were randomly assigned to receive brodalumab (210 mg or 140 mg every 2 weeks), ustekinumab (45 mg for patients with a body weight ≤100 kg and 90 mg for patients >100 kg), or placebo. At week 12, patients receiving brodalumab were randomly assigned again to receive a brodalumab maintenance dose of 210 mg every 2 weeks or 140 mg every 2 weeks, every 4 weeks, or every 8 weeks; patients receiving ustekinumab continued to receive ustekinumab every 12 weeks, and patients receiving placebo received 210 mg of brodalumab every 2 weeks. The primary aims were to evaluate the superiority of brodalumab over placebo at week 12 with respect to at least a 75% reduction in the psoriasis area-and-severity index score (PASI 75) and a static physician's global assessment (sPGA) score of 0 or 1 (clear or almost clear skin), as well as the superiority of brodalumab over ustekinumab at week 12 with respect to a 100% reduction in PASI score (PASI 100). RESULTS At week 12, the PASI 75 response rates were higher with brodalumab at the 210-mg and 140-mg doses than with placebo (86% and 67%, respectively, vs. 8% [AMAGINE-2] and 85% and 69%, respectively, vs. 6% [AMAGINE-3]; P<0.001); the rates of sPGA scores of 0 or 1 were also higher with brodalumab (P<0.001). The week 12 PASI 100 response rates were significantly higher with 210 mg of brodalumab than with ustekinumab (44% vs. 22% [AMAGINE-2] and 37% vs. 19% [AMAGINE-3], P<0.001). The PASI 100 response rates with 140 mg of brodalumab were 26% in AMAGINE-2 (P=0.08 for the comparison with ustekinumab) and 27% in AMAGINE-3 (P=0.007). Rates of neutropenia were higher with brodalumab and with ustekinumab than with placebo. Mild or moderate candida infections were more frequent with brodalumab than with ustekinumab or placebo. Through week 52, the rates of serious infectious episodes were 1.0 (AMAGINE-2) and 1.3 (AMAGINE-3) per 100 patient-years of exposure to brodalumab. CONCLUSIONS Brodalumab treatment resulted in significant clinical improvements in patients with moderate-to-severe psoriasis. (Funded by Amgen; AMAGINE-2 and AMAGINE-3 ClinicalTrials.gov numbers, NCT01708603 and NCT01708629.).\n"
     ]
    }
   ],
   "source": [
    "#checking one example\n",
    "pmid = \"26422722\"  # Example PubMed ID\n",
    "document_text = get_document_text(pmid, client_lexical)\n",
    "print(document_text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "metadata": {},
   "outputs": [],
   "source": [
    "json_structure = {\n",
    "    \"questions\": [\n",
    "        {\n",
    "            \"question\": \"Question here\",\n",
    "            \"type\": \"summary\",\n",
    "            \"exact_answer\": [\"Exact answer here\"],\n",
    "            \"ideal_answer\": \"Ideal answer here\",\n",
    "            \"abstracts\": []\n",
    "        }\n",
    "    ]\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {},
   "outputs": [],
   "source": [
    "\"\"\"def populate_abstracts(question, pmids, client):\n",
    "   \n",
    "    #Populate the 'abstracts' field for a given question with texts fetched using PubMed IDs. \n",
    "    #Args:\n",
    "    #question (dict): The question dictionary to populate.\n",
    "    #pmids (list of str): List of PubMed IDs.\n",
    "    #client (OpenSearchClient): The OpenSearch client instance.\n",
    "   \n",
    "    # Clear existing abstracts (if this function is to be reused)\n",
    "    question['abstracts'] = []\n",
    "\n",
    "    for pmid in tqdm(pmids, desc=\"Fetching texts\", leave=False):\n",
    "        text = get_document_text(pmid, client)\n",
    "        question['abstracts'].append({\n",
    "            \"id\": pmid,\n",
    "            \"text\": text\n",
    "        })\n",
    "\"\"\"\n",
    "def populate_abstracts(question, pmids, client):\n",
    "    # Initialize the list with None to reserve order\n",
    "    abstracts = [None] * len(pmids)\n",
    "\n",
    "    # Using tqdm to show progress for fetching document texts\n",
    "    for index, pmid in enumerate(tqdm(pmids, desc=\"Fetching texts\", leave=False)):\n",
    "        text = get_document_text(pmid, client)\n",
    "        abstracts[index] = {\n",
    "            \"id\": pmid,\n",
    "            \"text\": text\n",
    "        }\n",
    "\n",
    "    # Filter out None values in case any text could not be fetched\n",
    "    question['abstracts'] = [abstract for abstract in abstracts if abstract]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "                                                             "
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'question': 'What are the latest treatments for diabetes?', 'abstracts': [{'id': '26422722', 'text': \"Phase 3 Studies Comparing Brodalumab with Ustekinumab in Psoriasis. BACKGROUND Early clinical studies suggested that the anti-interleukin-17 receptor A monoclonal antibody brodalumab has efficacy in the treatment of psoriasis. METHODS In two phase 3 studies (AMAGINE-2 and AMAGINE-3), patients with moderate-to-severe psoriasis were randomly assigned to receive brodalumab (210 mg or 140 mg every 2 weeks), ustekinumab (45 mg for patients with a body weight ≤100 kg and 90 mg for patients >100 kg), or placebo. At week 12, patients receiving brodalumab were randomly assigned again to receive a brodalumab maintenance dose of 210 mg every 2 weeks or 140 mg every 2 weeks, every 4 weeks, or every 8 weeks; patients receiving ustekinumab continued to receive ustekinumab every 12 weeks, and patients receiving placebo received 210 mg of brodalumab every 2 weeks. The primary aims were to evaluate the superiority of brodalumab over placebo at week 12 with respect to at least a 75% reduction in the psoriasis area-and-severity index score (PASI 75) and a static physician's global assessment (sPGA) score of 0 or 1 (clear or almost clear skin), as well as the superiority of brodalumab over ustekinumab at week 12 with respect to a 100% reduction in PASI score (PASI 100). RESULTS At week 12, the PASI 75 response rates were higher with brodalumab at the 210-mg and 140-mg doses than with placebo (86% and 67%, respectively, vs. 8% [AMAGINE-2] and 85% and 69%, respectively, vs. 6% [AMAGINE-3]; P<0.001); the rates of sPGA scores of 0 or 1 were also higher with brodalumab (P<0.001). The week 12 PASI 100 response rates were significantly higher with 210 mg of brodalumab than with ustekinumab (44% vs. 22% [AMAGINE-2] and 37% vs. 19% [AMAGINE-3], P<0.001). The PASI 100 response rates with 140 mg of brodalumab were 26% in AMAGINE-2 (P=0.08 for the comparison with ustekinumab) and 27% in AMAGINE-3 (P=0.007). Rates of neutropenia were higher with brodalumab and with ustekinumab than with placebo. Mild or moderate candida infections were more frequent with brodalumab than with ustekinumab or placebo. Through week 52, the rates of serious infectious episodes were 1.0 (AMAGINE-2) and 1.3 (AMAGINE-3) per 100 patient-years of exposure to brodalumab. CONCLUSIONS Brodalumab treatment resulted in significant clinical improvements in patients with moderate-to-severe psoriasis. (Funded by Amgen; AMAGINE-2 and AMAGINE-3 ClinicalTrials.gov numbers, NCT01708603 and NCT01708629.).\"}]}\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r"
     ]
    }
   ],
   "source": [
    "#example for populating abstract\n",
    "question = {\"question\": \"What are the latest treatments for diabetes?\"}\n",
    "pmids = ['26422722']  # Example PMIDs\n",
    "\n",
    "# Populate abstracts for the question\n",
    "populate_abstracts(question, pmids, client_lexical)\n",
    "\n",
    "# Printing out the populated question to see the result\n",
    "print(question)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "import json\n",
    "#evaluation_file = 'training12b_new.json'\n",
    "#with open(evaluation_file, 'r') as f:\n",
    " #   data = json.load(f)\n",
    "\n",
    "#print(len(data['questions']))\n",
    "#BioASQ_filepath = 'training12b_new.json'\n",
    "#BioASQ_filepath = 'BioASQ_small.json'\n",
    "BioASQ_filepath = 'BioASQ_10_abstracts.json'\n",
    "def read_bioasq_file(filepath):\n",
    "    \"\"\"\n",
    "    Read the BioASQ JSON file.\n",
    "    \n",
    "    Args:\n",
    "    filepath (str): The path to the BioASQ file.\n",
    "    \n",
    "    Returns:\n",
    "    list: A list of questions from the BioASQ file.\n",
    "    \"\"\"\n",
    "    with open(filepath, 'r') as file:\n",
    "        bioasq_data = json.load(file)\n",
    "    return bioasq_data['questions']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 53,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "def create_custom_json_structure(bioasq_questions, client):\n",
    "    \"\"\"\n",
    "    Create a custom JSON structure based on BioASQ data.\n",
    "    \n",
    "    Args:\n",
    "    bioasq_questions (list): List of questions from BioASQ data.\n",
    "    client (OpenSearchClient): The OpenSearch client instance.\n",
    "    \n",
    "    Returns:\n",
    "    dict: The custom JSON structure populated with abstracts.\n",
    "    \"\"\"\n",
    "    json_structure = {\"questions\": []}\n",
    "\n",
    "    for question in tqdm(bioasq_questions, desc=\"Processing questions\"):\n",
    "        # Create a new question entry\n",
    "        new_question = {\n",
    "            \"id\": question.get(\"id\", \"No ID\"), \n",
    "            \"question\": question.get(\"body\", \"No question body\"),\n",
    "            \"type\": question.get(\"type\", \"\"),\n",
    "            \"exact_answer\": question.get(\"exact_answer\", []),\n",
    "            \"ideal_answer\": question.get(\"ideal_answer\", \"No ideal answer\"),\n",
    "            \"abstracts\": []\n",
    "        }\n",
    "\n",
    "        # Extract PubMed IDs from the 'documents' URLs\n",
    "        pmids = [doc.split('/')[-1] for doc in question.get('documents', [])]\n",
    "        \n",
    "        # Populate abstracts\n",
    "        populate_abstracts(new_question, pmids, client)\n",
    "        \n",
    "        # Add the populated question to the json structure\n",
    "        json_structure['questions'].append(new_question)\n",
    "\n",
    "    return json_structure"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Create custom BioASQ-ALL - with abstract's text from our IR for pmid from bioASQ"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Processing questions: 100%|██████████| 182/182 [04:49<00:00,  1.59s/it]\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Time taken to create the custom JSON: 290.20 seconds\n"
     ]
    }
   ],
   "source": [
    "# Example usage\n",
    "start_time = time.time()\n",
    "\n",
    "bioasq_questions = read_bioasq_file(BioASQ_filepath)\n",
    "custom_json = create_custom_json_structure(bioasq_questions, client_lexical)\n",
    "\n",
    "end_time = time.time()\n",
    "elapsed_time = end_time - start_time\n",
    "# Save the new JSON structure to a file\n",
    "with open('Generative/custom_bioasq_output_10_abstracts_without_newline.json', 'w') as outfile:\n",
    "    json.dump(custom_json, outfile, indent=4)\n",
    "\n",
    "print(f\"Time taken to create the custom JSON: {elapsed_time:.2f} seconds\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Create subset of BioASQ - question with 10 relevant abstract"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Filtered data has been saved to BioASQ_10_abstracts.json\n",
      "Number od questions with 10 or more abstracts: 182\n"
     ]
    }
   ],
   "source": [
    "\n",
    "def filter_questions_with_ten_or_more_documents(input_filepath, output_filepath):\n",
    "    # Load the data from the BioASQ JSON file\n",
    "    with open(input_filepath, 'r') as file:\n",
    "        data = json.load(file)\n",
    "    \n",
    "    filtered_questions = []\n",
    "\n",
    "    # Filter questions based on the number of documents\n",
    "    for question in data['questions']:\n",
    "        # Check if there are 10 or more documents\n",
    "        if len(question['documents']) == 10:\n",
    "            filtered_questions.append(question)\n",
    "\n",
    "    # Prepare the final structure with filtered questions\n",
    "    final_data = {'questions': filtered_questions}\n",
    "    # Save the filtered data to a new JSON file\n",
    "    with open(output_filepath, 'w') as outfile:\n",
    "        json.dump(final_data, outfile, indent=4)\n",
    "\n",
    "    print(f\"Filtered data has been saved to {output_filepath}\")\n",
    "    print(\"Number od questions with 10 or more abstracts:\", len(filtered_questions))\n",
    "# Define file paths\n",
    "input_filepath = 'training12b_new.json'  # Set the correct path to your BioASQ file\n",
    "output_filepath = 'BioASQ_10_abstracts.json'\n",
    "\n",
    "# Process the file\n",
    "filter_questions_with_ten_or_more_documents(input_filepath, output_filepath)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <font color='red'>Kraj pripreme podataka za generativnu komponentu</font>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 111,
   "metadata": {},
   "outputs": [],
   "source": [
    "from qdrant_client import QdrantClient\n",
    "from qdrant_client.http.models import Distance, VectorParams\n",
    "from qdrant_client.http.models import PointStruct\n",
    "from qdrant_client.http import models\n",
    "\n",
    "\n",
    "# 3.145.52.195\n",
    "#client_semantic = QdrantClient(host, port=6333, timeout = 60)\n",
    "\n",
    "verifai_ip='3.23.103.76'\n",
    "qdrant_port=6333\n",
    "TIMEOUT=60\n",
    "url = f\"https://{verifai_ip}:{qdrant_port}\"\n",
    "qdrant_api=\"8da7725d78141e19a9bf3d878f4cb333fedb56eed9727904b46ce4b32e1ce085\"\n",
    "client_semantic = QdrantClient(url=url, api_key=qdrant_api, timeout=TIMEOUT, https=True,**{'verify': False})\n",
    "#client_semantic = QdrantClient(host, port=6333, timeout = 60)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Select the type of lexical indexing: "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 112,
   "metadata": {},
   "outputs": [],
   "source": [
    "lexical_pmid = False\n",
    "\n",
    "if lexical_pmid:\n",
    "    index_name_lexical = 'medline-faiss-hnsw-lexical-pmid'\n",
    "else:\n",
    "    index_name_lexical ='medline-faiss-hnsw-lexical'\n",
    "\n",
    "coll_name_semantic = \"medline-faiss-hnsw\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 113,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Device cpu\n"
     ]
    }
   ],
   "source": [
    "model_card = 'sentence-transformers/msmarco-distilbert-base-tas-b'\n",
    "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
    "print(f\"Device {device}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 114,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "c:\\Users\\Adela\\anaconda3\\Lib\\site-packages\\transformers\\tokenization_utils_base.py:1601: FutureWarning: `clean_up_tokenization_spaces` was not set. It will be set to `True` by default. This behavior will be depracted in transformers v4.45, and will be then set to `False` by default. For more details check this issue: https://github.com/huggingface/transformers/issues/31884\n",
      "  warnings.warn(\n"
     ]
    }
   ],
   "source": [
    "model = SentenceTransformer(model_card)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Query"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 115,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package punkt to\n",
      "[nltk_data]     C:\\Users\\Adela\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package punkt is already up-to-date!\n",
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\Adela\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n"
     ]
    }
   ],
   "source": [
    "import nltk\n",
    "\n",
    "# Ensure that the necessary NLTK data is downloaded\n",
    "nltk.download('punkt')\n",
    "nltk.download('stopwords')\n",
    "\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.tokenize import word_tokenize\n",
    "import pandas as pd\n",
    "\n",
    "\n",
    "\n",
    "\n",
    "class QueryProcessor:\n",
    "    def __init__(self, index_lexical:str = \"medline-faiss-hnsw-lexical\",lexical_pmid = False, index_name_semantic =\"medline-faiss-hnsw\", rescore = False, model=None, lexical_client=None, semantic_client=None, output_file_path=\"queries/queries.tsv\", stopwords=set([])):\n",
    "        self.index_lexical_name = index_lexical\n",
    "        self.index_name_semantic = index_name_semantic\n",
    "        # 2 index name (?)\n",
    "        self.model = model\n",
    "        #self.lexical_pmid = lexical_pmid\n",
    "        self.lexical_client = lexical_client\n",
    "        self.semantic_client = semantic_client\n",
    "        self.output_file_path = output_file_path\n",
    "        self.stop_words = stopwords\n",
    "        self.query_result = []\n",
    "        self.rescore = rescore\n",
    "        self.lexical_query = self.lexical_query_pmid if lexical_pmid else self.lexical_query\n",
    "    \n",
    "    def set_rescore(self, rescore):\n",
    "        self.rescore = rescore\n",
    "\n",
    "    def preprocess_query(self, query_str):\n",
    "        return ' '.join([word for word in word_tokenize(query_str) if word.lower() not in self.stop_words])\n",
    "\n",
    "    def save_results(self):\n",
    "        with open(self.output_file_path, \"w\") as file:\n",
    "            json.dump(self.query_result, file, indent=4)\n",
    "      \n",
    "    \n",
    "    def reorder_pmid(self, retrived_documents):\n",
    "        pmid_scores = {}\n",
    "        \n",
    "        # Iterate through the set data\n",
    "        for _, value in retrived_documents.items():\n",
    "            pmid = value['pmid']\n",
    "            score = value['score']\n",
    "            \n",
    "            # Check if pmid already exists in the dictionary\n",
    "            if pmid in pmid_scores:\n",
    "                pmid_scores[pmid] += score\n",
    "            else:\n",
    "                pmid_scores[pmid] = score\n",
    "           \n",
    "        return pmid_scores\n",
    "    \n",
    "    def lexical_query(self, query_str, limit=10):\n",
    "        if self.lexical_client == None:\n",
    "            raise ValueError(\"No Lexical client defined\")\n",
    "        \n",
    "        query = {\n",
    "                \"size\": limit,\n",
    "                \"query\": {\n",
    "                    \"multi_match\": {\n",
    "                        \"query\": query_str,\n",
    "                        \"fields\": [\"text\"]\n",
    "                    }\n",
    "                }\n",
    "            }\n",
    "       \n",
    "        results = self.lexical_client.search(index=self.index_lexical_name, body=query) \n",
    "        retrived_documents = {}\n",
    "        max_score = results['hits']['max_score']\n",
    "     \n",
    "        for hit in results[\"hits\"][\"hits\"]:\n",
    "            \n",
    "            pmid = hit[\"_source\"][\"pmid\"]\n",
    "            score = hit[\"_score\"]\n",
    "            auto_id = hit[\"_id\"]\n",
    "            \n",
    "            \n",
    "            \n",
    "            retrived_documents[auto_id] = {\n",
    "                \"score\": round(score/max_score, 5),\n",
    "                \"pmid\": pmid\n",
    "                }\n",
    "        \n",
    "        retrived_documents = self.reorder_pmid(retrived_documents)\n",
    "        return retrived_documents #adjust the return \n",
    "    \n",
    "    def lexical_query_pmid(self, query_str, limit=10):\n",
    "        #print(\"Lexical = \",query_str)\n",
    "        if self.lexical_client == None:\n",
    "            raise ValueError(\"No Lexical client defined\")\n",
    "        \n",
    "        query = {\n",
    "                \"size\": limit,\n",
    "                \"query\": {\n",
    "                    \"multi_match\": {\n",
    "                        \"query\": query_str,\n",
    "                        \"fields\": [\"full_text\"]\n",
    "                    }\n",
    "                }\n",
    "            }\n",
    "        \n",
    "        results = self.lexical_client.search(index=self.index_lexical_name, body=query) \n",
    "        \n",
    "        retrieved_documents = {}\n",
    "        max_score = results['hits']['max_score']\n",
    "        for hit in results[\"hits\"][\"hits\"]:\n",
    "            \n",
    "            pmid = hit[\"_source\"][\"pmid\"]\n",
    "            score = hit[\"_score\"] / max_score\n",
    "            \n",
    "            retrieved_documents[pmid] = score\n",
    "            \n",
    "        return retrieved_documents #adjust the return \n",
    "\n",
    "    def semantic_query(self, query, limit=10):\n",
    "        #print(\"semantic = \",query)\n",
    "        if self.semantic_client == None:\n",
    "            raise ValueError(\"No Semantic client defined\")\n",
    "        if self.model == None:\n",
    "            raise ValueError(\"No model defined\")\n",
    "        \n",
    "        query_vector = self.model.encode(query).tolist()\n",
    "    \n",
    "        search_params=models.SearchParams(\n",
    "            quantization=models.QuantizationSearchParams(rescore=self.rescore)\n",
    "            )\n",
    "        results = self.semantic_client.search(collection_name=self.index_name_semantic,query_vector=query_vector,search_params=search_params, limit=limit)\n",
    "    \n",
    "        #results = self.semantic_client.search(collection_name=self.index_name_semantic,query_vector=query_vector, limit=limit)\n",
    "        \n",
    "        retrived_documents = {}\n",
    "        max_score = None\n",
    "        for i,document in enumerate(results):\n",
    "            \n",
    "            pmid = document.payload['pmid']\n",
    "            score = document.score\n",
    "            if i == 0:\n",
    "                # first score is the max\n",
    "                max_score = score\n",
    "            retrived_documents[document.id] = { 'pmid': pmid, 'score': round(score / max_score, 5) } \n",
    "\n",
    "        retrived_documents = self.reorder_pmid(retrived_documents)\n",
    "        \n",
    "        return retrived_documents\n",
    "    \n",
    "\n",
    "    def hybrid_query(self, query_lexical, query_semantic, lex_parameter = 0.5, semantic_parameter = 0.5, limit=10):\n",
    "        if (lex_parameter + semantic_parameter) > 1:\n",
    "            raise ValueError(\"Uncorrect parameters for Hybrid Queries\")\n",
    "        lexical_results = self.lexical_query(query_lexical, limit = limit) \n",
    "        semantic_results = self.semantic_query(query_semantic, limit)\n",
    "        max_score = 0\n",
    "        retrived_documents = {}\n",
    "        \n",
    "        for lex_pmid in lexical_results:\n",
    "            score = lexical_results[lex_pmid] * lex_parameter\n",
    "            if lex_pmid in semantic_results:\n",
    "                score += semantic_results[lex_pmid] * semantic_parameter\n",
    "\n",
    "            retrived_documents[lex_pmid] = score\n",
    "            max_score = max(max_score, score)\n",
    "            \n",
    "\n",
    "        for semantic_pmid in semantic_results:\n",
    "            if semantic_pmid not in lexical_results:\n",
    "                score = semantic_results[semantic_pmid] * semantic_parameter\n",
    "                retrived_documents[semantic_pmid] = score\n",
    "                max_score = max(max_score, score)\n",
    "                \n",
    "        return retrived_documents # just to have a starting point\n",
    "\n",
    "    def execute_query(self, query_str, query_type='lexical', lex_parameter = 0.5, semantic_parameter = 0.5,limit = 10,save = True, stopwords_preprocessing=True):\n",
    "        #print(\"Before = \",query_str)\n",
    "        text_query = self.preprocess_query(query_str) if stopwords_preprocessing else query_str\n",
    "        \n",
    "        if query_type == 'lexical':\n",
    "            results = self.lexical_query(text_query, limit=limit) \n",
    "        \n",
    "        elif query_type == 'semantic':\n",
    "            results = self.semantic_query(query_str, limit=limit)\n",
    "            \n",
    "    #The final list of documents is sorted by the combined score, allowing the system to return a balanced mix of results that consider both lexical and semantic relevance.\n",
    "        elif query_type == 'hybrid':\n",
    "            results = self.hybrid_query(text_query, query_str, lex_parameter, semantic_parameter, limit=limit)\n",
    "        else:\n",
    "            raise ValueError(\"Invalid query type specified. Choose 'lexical', 'semantic', or 'hybrid'.\")\n",
    "        \n",
    "        \n",
    "        document_retrived = sorted(results.items(), key=lambda x: x[1], reverse=True)\n",
    "        document_retrived = document_retrived[:limit+1] # in the hybrid search we can return more documents\n",
    "        #print(\"Results \", document_retrived)\n",
    "        if save:\n",
    "            self.process_results(document_retrived, query_str, query_type)\n",
    "\n",
    "        return document_retrived\n",
    "    \n",
    "    # needs to be rewrited\n",
    "    def process_results(self, results, query_str,query_type):\n",
    "        \n",
    "        retrieved_documents = []\n",
    "        for element in results:\n",
    "            \n",
    "            pmid,_ = element\n",
    "            query = {\n",
    "                    \"query\": {\n",
    "                        \"term\": {\n",
    "                        \"pmid\": int(pmid)\n",
    "                        }\n",
    "                    }\n",
    "                }\n",
    "\n",
    "            results = self.lexical_client.search(index=self.index_lexical_name, body=query) \n",
    "            full_text = results['hits']['hits'][0][\"_source\"]['full_text']\n",
    "            pmid = results['hits']['hits'][0][\"_source\"]['pmid']\n",
    "\n",
    "            retrieved_documents.append({\n",
    "                \"pmid\": pmid,\n",
    "                \"text\": full_text\n",
    "            })\n",
    "\n",
    "        dict_to_save = {'query': query_str, 'query_type': query_type, 'abstracts' : retrieved_documents}\n",
    "        self.query_result.append(dict_to_save)  \n",
    "\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[('31431276', 0.7), ('1570146', 0.6332199999999999), ('26692249', 0.630973), ('30509429', 0.6260939999999999), ('37751184', 0.620501), ('37777849', 0.620501), ('33132093', 0.607124), ('31272626', 0.590828), ('38090918', 0.589488), ('36621448', 0.588007), ('1622374', 0.5859139999999999)]\n"
     ]
    }
   ],
   "source": [
    "#known_pmid = '15795476'\n",
    "qp = QueryProcessor(\n",
    "        model=model,  # You need to define or load your model\n",
    "        lexical_client=client_lexical,  # Opensearch client \n",
    "        semantic_client=client_semantic  # Qdrant client\n",
    "    )\n",
    "print(qp.execute_query(\"What is the cause of hip pain\",query_type='hybrid',lex_parameter=0.7, semantic_parameter=0.3,limit=10))\n",
    "#print(qp.hybrid_query(client_lexical,client_semantic,lex_parameter=0.7,semantic_parameter=0.3,limit=10))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Query parser\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Doing some queries"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 117,
   "metadata": {},
   "outputs": [],
   "source": [
    "model = SentenceTransformer(model_card)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 118,
   "metadata": {},
   "outputs": [],
   "source": [
    "query_parser = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('10797929', 0.5),\n",
       " ('15877281', 0.5),\n",
       " ('37560515', 0.49687),\n",
       " ('29597095', 0.494725),\n",
       " ('20870045', 0.49326),\n",
       " ('29922639', 0.49277),\n",
       " ('9462748', 0.490925),\n",
       " ('22303795', 0.49066),\n",
       " ('24914010', 0.490565),\n",
       " ('19332160', 0.49053),\n",
       " ('22106036', 0.49053)]"
      ]
     },
     "execution_count": 119,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query_parser.execute_query(query_str=\"Which gene is responsible for disfunction in speech for children?\", query_type='hybrid')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluation"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Loading the evaluation file - BioASQ_10_or_more.json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "182\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "evaluation_file = 'BioASQ_10_abstracts.json'\n",
    "\n",
    "with open(evaluation_file, 'r') as f:\n",
    "    data = json.load(f)\n",
    "\n",
    "print(len(data['questions']))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluation Function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 131,
   "metadata": {},
   "outputs": [],
   "source": [
    "import numpy as np\n",
    "\n",
    "\n",
    "def clean_documents(documents):\n",
    "    output_documents = []  ## Changed from output_documents = set()\n",
    "    for doc in documents:\n",
    "        output_documents.append((doc.replace(\"http://www.ncbi.nlm.nih.gov/pubmed/\",\"\")))\n",
    "    return output_documents\n",
    "\n",
    "def average_precision(retrived_doc, true_doc):\n",
    "    # Initialize variables\n",
    "    precision_sum = 0\n",
    "    num_retrieved_docs = 0\n",
    "    \n",
    "    # Calculate precision at each relevant document position\n",
    "    for i, retrived in enumerate(retrived_doc, start=1):\n",
    "        pmid,_ = retrived\n",
    "        if pmid in true_doc:  # Check if the document is relevant\n",
    "            num_retrieved_docs += 1\n",
    "            precision_sum += num_retrieved_docs / i  # Calculate precision at cutoff i\n",
    "\n",
    "    # Calculate average precision\n",
    "    if num_retrieved_docs == 0:\n",
    "        return 0\n",
    "    else:\n",
    "        return precision_sum / num_retrieved_docs\n",
    "\n",
    "\n",
    "def evaluation(query_parser, data, query_type,alpha=0.5, beta=0.5, stopwords_preprocessing = False, path = \"query_result.json\"):\n",
    "    avg_precisions_sum = [] # sum all average precision and divide with number of queries \n",
    "    precisions_sum = []\n",
    "    queries_time = []\n",
    "    for i,question in enumerate(data['questions']):\n",
    "        dict_to_save = {}\n",
    "        query = question['body']\n",
    "        type_question = question['type']\n",
    "        dict_to_save['query'] = query\n",
    "        dict_to_save['query_type'] = query_type\n",
    "        dict_to_save['question_type']=type_question # ADDED LATER FOR THE PURPOSE OF STATISTIC\n",
    "        #relevant_documents = clean_documents(question['documents'])\n",
    "        #(NEW_EVAL)Return only the first 10 abstracts\n",
    "        relevant_documents = clean_documents(question['documents'])[:10]\n",
    "        start_time = time.time()\n",
    "        \n",
    "        #results = query_parser.execute_query(query,query_type = query_type, lex_parameter = alpha, semantic_parameter = beta,limit = len(relevant_documents), save=False, stopwords_preprocessing = stopwords_preprocessing)\n",
    "       \n",
    "        #(NEW_EVAL)In the next line, limit is set to 10 for retrieving from our IR\n",
    "        results = query_parser.execute_query(query,query_type = query_type, lex_parameter = alpha, semantic_parameter = beta,limit = 10, save=False, stopwords_preprocessing = stopwords_preprocessing)\n",
    "\n",
    "        queries_time.append(time.time() - start_time)\n",
    "        \n",
    "        #results = [('20598273',1), ('4',1), ('6650562',1), ('2',1),('21995290',1),('15617541',1),('23001136',1),('8896569',1), ('12239580',1)]\n",
    "        dict_to_save['true_documents'] = relevant_documents\n",
    "        dict_to_save['retrieved_documents'] = results\n",
    "       \n",
    "\n",
    "    \n",
    "        number_retrieved_documents = 0\n",
    "        for pmid,_ in results:\n",
    "            if pmid in relevant_documents:\n",
    "                number_retrieved_documents +=1\n",
    "\n",
    "        precision = number_retrieved_documents / len(relevant_documents)\n",
    "        recall = number_retrieved_documents / len(relevant_documents)\n",
    "        avg_precision = average_precision(results, relevant_documents)\n",
    "        \n",
    "        precisions_sum.append(precision)\n",
    "        #recalls.append(recall)\n",
    "        \n",
    "        avg_precisions_sum.append(avg_precision)\n",
    "        \n",
    "        dict_to_save['precision'] = precision\n",
    "        #dict_to_save['recall'] = recall\n",
    "        dict_to_save['avg_precision'] = avg_precision\n",
    "        with open(path, 'a') as output_file:\n",
    "            output_file.write(json.dumps(dict_to_save) + '\\n')\n",
    "        if (i+1) % 500 == 0:\n",
    "            print(f\"Analyzed {i+1} queries\")\n",
    "            print(\"Actual Results...\")\n",
    "            print(f\"Mean precision = {np.mean(precisions_sum):.3f}\")\n",
    "            #print(f\"Mean recall = {np.mean(recalls):.3f}\")\n",
    "            print(f\"Mean Average Precision = {np.mean(avg_precisions_sum):.3f}\")\n",
    "            print(f\"Mean Time needed to execute a query = {np.mean(queries_time):.3f}\")\n",
    "    print(\"FINAL RESULTS \")\n",
    "    print(f\"Mean precision = {np.mean(precisions_sum):.3f}\")\n",
    "    #print(f\"Mean recall = {np.mean(recalls):.3f}\")\n",
    "    print(f\"Mean Average Precision = {np.mean(avg_precisions_sum):.3f}\")\n",
    "    print(f\"Mean Time needed to execute a query = {np.mean(queries_time):.3f}\")\n",
    "    print('Relevantni',relevant_documents)\n",
    "    print('Vraćeni',results)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 130,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'questions': [{'body': 'Is the protein Papilin secreted?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/3320045',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7515725',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20805556',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19297413',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19724244',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15094122',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12666201',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21784067',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11076767',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15094110'],\n",
       "   'ideal_answer': ['Yes,  papilin is a secreted protein'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '54e25eaaae9738404b000017',\n",
       "   'snippets': [{'offsetInBeginSection': 1085,\n",
       "     'offsetInEndSection': 1307,\n",
       "     'text': 'Using expression analysis, we identify three genes that are transcriptionally regulated by HLH-2: the protocadherin cdh-3, and two genes encoding secreted extracellular matrix proteins, mig-6/papilin and him-4/hemicentin. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21784067',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 226,\n",
       "     'offsetInEndSection': 458,\n",
       "     'text': 'We found that mig-6 encodes long (MIG-6L) and short (MIG-6S) isoforms of the extracellular matrix protein papilin, each required for distinct aspects of DTC migration. Both MIG-6 isoforms have a predicted N-terminal papilin cassette',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19297413',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 111,\n",
       "     'text': 'apilins are homologous, secreted extracellular matrix proteins which share a common order of protein domains. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15094122',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 404,\n",
       "     'text': 'The TSR superfamily is a diverse family of extracellular matrix and transmembrane proteins, many of which have functions related to regulating matrix organization, cell-cell interactions and cell guidance. This review samples some of the contemporary literature regarding TSR superfamily members (e.g. F-spondin, UNC-5, ADAMTS, papilin, and TRAP) where specific functions are assigned to the TSR domains.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15094110',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 43,\n",
       "     'text': 'Papilins are extracellular matrix proteins ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12666201',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 48,\n",
       "     'text': 'Papilin is an extracellular matrix glycoprotein ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11076767',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 731,\n",
       "     'offsetInEndSection': 882,\n",
       "     'text': ' Collagen IV, laminin, glutactin, papilin, and other extracellular matrix proteins were made primarily by hemocytes and were secreted into the medium. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7515725',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'A sulfated glycoprotein was isolated from the culture media of Drosophila Kc cells and named papilin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3320045',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is RANKL secreted from the cells?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22948539',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23698708',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23827649',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22901753',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21618594',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23835909',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23632157',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24265865',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24267510',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22867712'],\n",
       "   'ideal_answer': ['Receptor activator of nuclear factor κB ligand (RANKL) is a cytokine predominantly secreted by osteoblasts.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/TNF11_RAT',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015870',\n",
       "    'http://www.uniprot.org/uniprot/TNF11_HUMAN'],\n",
       "   'type': 'yesno',\n",
       "   'id': '55262a9787ecba3764000009',\n",
       "   'snippets': [{'offsetInBeginSection': 114,\n",
       "     'offsetInEndSection': 242,\n",
       "     'text': 'Osteoprotegerin (OPG) is a soluble secreted factor that acts as a decoy receptor for receptor activator of NF-κB ligand (RANKL) ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24267510',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 266,\n",
       "     'text': 'Osteoprotegerin (OPG) is a secreted glycoprotein and a member of the tumor necrosis factor receptor superfamily. It usually functions in bone remodeling, by inhibiting osteoclastogenesis through interaction with a receptor activator of the nuclear factor κB (RANKL).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24265865',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1110,\n",
       "     'offsetInEndSection': 1247,\n",
       "     'text': 'e RANKL/OPG ratio secreted by osteoblasts increased and RANK expression by osteoclasts increased, leading to increased osteoclastogenesis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23835909',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 228,\n",
       "     'text': 'Osteoprotegerin (OPG) is an essential secreted protein in bone turnover due to its role as a decoy receptor for the Receptor Activator of Nuclear Factor-kB ligand (RANKL) in the osteoclasts, thus inhibiting their differentiation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23827649',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 324,\n",
       "     'offsetInEndSection': 439,\n",
       "     'text': 'We identify a TNFSF11 transcript variant that extends the originally identified transcript encoding secreted RANKL.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23698708',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 95,\n",
       "     'text': 'Activated human T cells express alternative mRNA transcripts encoding a secreted form of RANKL.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23698708',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 471,\n",
       "     'offsetInEndSection': 621,\n",
       "     'text': 'OPG, on the other hand, is secreted by osteoblast as a decoy receptor for RANKL, prevents RANKL from binding to RANK and thus prevents bone resorption',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23632157',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 270,\n",
       "     'offsetInEndSection': 486,\n",
       "     'text': 'Receptor activator of nuclear factor κB ligand (RANKL) and osteoprotegerin (OPG) are cytokines predominantly secreted by osteoblasts and play a central role in differentiation and functional activation of osteoclasts',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22948539',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 690,\n",
       "     'offsetInEndSection': 1005,\n",
       "     'text': 'Although B. abortus-activated T cells actively secreted the pro-osteoclastogenic cytokines RANKL and IL-17, osteoclastogenesis depended on IL-17, because osteoclast generation induced by Brucella-activated T cells was completely abrogated when these cells were cultured with BMMs from IL-17 receptor knockout mice. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22901753',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 468,\n",
       "     'offsetInEndSection': 758,\n",
       "     'text': ' osteoclastogenesis and bone destruction in autoimmune arthritis. We isolated human fibroblasts from RA, pyrophosphate arthropathy (PPA) and osteoarthritis (OA) patients and analyzed their RANKL/OPG expression profile and the capacity of their secreted factors to induce osteoclastogenesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22867712',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 269,\n",
       "     'offsetInEndSection': 478,\n",
       "     'text': 'Osteoprotegerin (OPG) and receptor activator of nuclear factor κB ligand (RANKL) are cytokines predominantly secreted by osteoblasts and play critical roles in the differentiation and function of osteoclasts. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21618594',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Where is the protein Pannexin1 located?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24642372',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25056878',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24694658',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25698922',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17064878',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24782784',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26100513',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25007779',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19409451',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24194896'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17397470',\n",
       "     'o': 'C544702'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A17400569'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#altLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A17397470'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17400568',\n",
       "     'o': 'Panx1 protein, zebrafish'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17399545',\n",
       "     'o': 'zfPanx1 protein, zebrafish'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17400569',\n",
       "     'o': 'pannexin 1 protein, zebrafish'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A17399545'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17397470',\n",
       "     'o': 'pannexin1 protein, zebrafish'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A17400568'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A17397470'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17399545',\n",
       "     'o': 'C544702'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17400568',\n",
       "     'o': 'C544702'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17400569',\n",
       "     'o': 'C544702'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17397470',\n",
       "     'o': 'MeSH'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17399545',\n",
       "     'o': 'MeSH'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17400568',\n",
       "     'o': 'MeSH'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17400569',\n",
       "     'o': 'MeSH'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#broader',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C0969748'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17909603',\n",
       "     'o': 'Brachydanio rerio Proteins'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17885251',\n",
       "     'o': 'Zebra Danio Proteins'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A2030874',\n",
       "     'o': 'Danio rerio Proteins'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A18005491',\n",
       "     'o': 'Zebrafish Proteins'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17885252',\n",
       "     'o': 'Zebrafish Proteins [Chemical/Ingredient]'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17909604',\n",
       "     'o': 'Zebra Fish Proteins'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#narrower',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0969748',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C2743999'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#broader',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C0110611'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A18025970',\n",
       "     'o': 'Connexins [Chemical/Ingredient]'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0361672',\n",
       "     'o': 'Gap Junction Proteins'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15367648',\n",
       "     'o': 'Connexin'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17882031',\n",
       "     'o': 'Protein, Gap Junction'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1305072',\n",
       "     'o': 'connexin'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0188058',\n",
       "     'o': 'Gap Junction Protein'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A18025971',\n",
       "     'o': 'Proteins, Gap Junction'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0361674',\n",
       "     'o': 'Junction Protein, Gap'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0361675',\n",
       "     'o': 'Junction Proteins, Gap'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A12992664',\n",
       "     'o': 'Gap Junction Channel Proteins'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A4365650',\n",
       "     'o': 'Connexins'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#narrower',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0110611',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C2743999'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#inScheme',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q29ZM8',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_5132395A4D38008'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_5132395A4D38008',\n",
       "     'o': 'Pannexin1'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/relatedConcept',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/relation/R120084487',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C2743999'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/relation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2743999',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/relation/R119818176'}],\n",
       "   'ideal_answer': ['The protein Pannexin1 is localized to the plasma membranes.'],\n",
       "   'exact_answer': ['plasma membrane'],\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0051179'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56af9f130a360a5e45000015',\n",
       "   'snippets': [{'offsetInBeginSection': 394,\n",
       "     'offsetInEndSection': 620,\n",
       "     'text': 'zfPanx1 was identified on the surface of horizontal cell dendrites invaginating deeply into the cone pedicle near the glutamate release sites of the cones, providing in vivo evidence for hemichannel formation at that location.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19409451',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 845,\n",
       "     'offsetInEndSection': 915,\n",
       "     'text': 'pannexin1, a vertebrate homolog of invertebrate gap junction proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17064878',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 175,\n",
       "     'offsetInEndSection': 422,\n",
       "     'text': 'The specific profile of gap junction proteins, the connexins, expressed in these different cell types forms compartments of intercellular communication that can be further shaped by the release of extracellular nucleotides via pannexin1 channels. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26100513',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 310,\n",
       "     'offsetInEndSection': 430,\n",
       "     'text': 'Recent studies demonstrated that ATP can be released from cells in a controlled manner through pannexin (Panx) channels.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24642372',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 59,\n",
       "     'text': 'The ATP release channel Pannexin1 (Panx1) is self-regulated',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24694658',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 104,\n",
       "     'text': 'The membrane protein Pannexin1 forms two open-channel conformations depending on the mode of activation.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25056878',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 92,\n",
       "     'offsetInEndSection': 143,\n",
       "     'text': ' Pannexin1 channels traffic to the plasma membrane.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25698922',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 204,\n",
       "     'offsetInEndSection': 340,\n",
       "     'text': 'We previously showed that pannexins form oligomeric channels but unlike connexins and innexins, they form only single membrane channels.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25698922',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 30,\n",
       "     'text': 'ATP release channel Pannexin1 ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25007779',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 209,\n",
       "     'offsetInEndSection': 359,\n",
       "     'text': 'Pannexin1 (Panx1) is a newly discovered extracellular ATP release channel with a wide tissue distribution and diverse biological functions in mammals.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25007779',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'In mammals, a single pannexin1 gene (Panx1) is widely expressed in the CNS including the inner and outer retinae, forming large-pore voltage-gated membrane channels, which are involved in calcium and ATP signaling. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24194896',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 100,\n",
       "     'offsetInEndSection': 366,\n",
       "     'text': 'Six of them form a \"gap junction hemichannel-like\" structure that connects the cytoplasm with the extracellular space; here they will be called Panx1 channels. These are poorly selective channels permeable to ions, small metabolic substrate, and signaling molecules.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24782784',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Are ultraconserved elements often transcribed?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24037088',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22094949',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20202189',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22617881',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24384562',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22328099',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21187392',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19660540',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16705037',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23393190'],\n",
       "   'ideal_answer': ['Yes.  Especially, a large fraction of non-exonic UCEs is transcribed across all developmental stages examined from only one DNA strand.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '553a8d78f321868558000003',\n",
       "   'snippets': [{'offsetInBeginSection': 222,\n",
       "     'offsetInEndSection': 479,\n",
       "     'text': 'Starting from a genome-wide expression profiling, we demonstrate for the first time a functional link between oxygen deprivation and the modulation of long noncoding transcripts from ultraconserved regions, termed transcribed-ultraconserved regions (T-UCRs)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24037088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1317,\n",
       "     'offsetInEndSection': 1476,\n",
       "     'text': 'Our data gives a first glimpse of a novel functional hypoxic network comprising protein-coding transcripts and noncoding RNAs (ncRNAs) from the T-UCRs category',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24037088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 157,\n",
       "     'text': 'Highly conserved elements discovered in vertebrates are present in non-syntenic loci of tunicates, act as enhancers and can be transcribed during development',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23393190',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 969,\n",
       "     'offsetInEndSection': 1303,\n",
       "     'text': \"The majority of these regions map onto ultraconserved elements and we demonstrate that they can act as functional enhancers within the organism of origin, as well as in cross-transgenesis experiments, and that they are transcribed in extant species of Olfactores. We refer to the elements as 'Olfactores conserved non-coding elements'\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23393190',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 504,\n",
       "     'offsetInEndSection': 1083,\n",
       "     'text': 'We used a custom microarray to assess the levels of UCE transcription during mouse development and integrated these data with published microarray and next-generation sequencing datasets as well as with newly produced PCR validation experiments. We show that a large fraction of non-exonic UCEs is transcribed across all developmental stages examined from only one DNA strand. Although the nature of these transcripts remains a mistery, our meta-analysis of RNA-Seq datasets indicates that they are unlikely to be short RNAs and that some of them might encode nuclear transcripts',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20202189',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1506,\n",
       "     'offsetInEndSection': 1876,\n",
       "     'text': 'Our data shows that the concurrent presence of enhancer and transcript function in non-exonic UCE elements is more widespread than previously shown. Moreover through our own experiments as well as the use of next-generation sequencing datasets, we were able to show that the RNAs encoded by non-exonic UCEs are likely to be long RNAs transcribed from only one DNA strand',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20202189',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 117,\n",
       "     'text': 'Short ultraconserved promoter regions delineate a class of preferentially expressed alternatively spliced transcripts',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19660540',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 360,\n",
       "     'offsetInEndSection': 564,\n",
       "     'text': 'The importance of other classes of non-coding RNAs, such as long intergenic ncRNAs (lincRNAs) and transcribed ultraconserved regions (T-UCRs) as altered elements in neoplasia, is also gaining recognition.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22617881',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 324,\n",
       "     'offsetInEndSection': 555,\n",
       "     'text': 'Other ncRNAs, such as PIWI-interacting RNAs (piRNAs), small nucleolar RNAs (snoRNAs), transcribed ultraconserved regions (T-UCRs) and large intergenic non-coding RNAs (lincRNAs) are emerging as key elements of cellular homeostasis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22094949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 969,\n",
       "     'offsetInEndSection': 1232,\n",
       "     'text': 'The majority of these regions map onto ultraconserved elements and we demonstrate that they can act as functional enhancers within the organism of origin, as well as in cross-transgenesis experiments, and that they are transcribed in extant species of Olfactores.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23393190',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 105,\n",
       "     'offsetInEndSection': 320,\n",
       "     'text': 'Transcribed ultraconserved regions (T-UCRs) are a subset of 481 sequences longer than 200 bp, which are absolutely conserved between orthologous regions of human, rat and mouse genomes, and are actively transcribed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22328099',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'Highly conserved elements discovered in vertebrates are present in non-syntenic loci of tunicates, act as enhancers and can be transcribed during development.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23393190',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'The Evf-2 noncoding RNA is transcribed from the Dlx-5/6 ultraconserved region and functions as a Dlx-2 transcriptional coactivator.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16705037',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 484,\n",
       "     'offsetInEndSection': 630,\n",
       "     'text': 'In this report, we show that the Dlx-5/6 ultraconserved region is transcribed to generate an alternatively spliced form of Evf-1, the ncRNA Evf-2.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16705037',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1544,\n",
       "     'offsetInEndSection': 1738,\n",
       "     'text': 'These studies identify a critical role for TUC338 in regulation of transformed cell growth and of transcribed ultraconserved ncRNA as a unique class of genes involved in the pathobiology of HCC.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21187392',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 297,\n",
       "     'offsetInEndSection': 426,\n",
       "     'text': 'Transcribed ultraconserved region (T-UCR) transcripts are a novel class of lncRNAs transcribed from ultraconserved regions (UCRs)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24384562',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 969,\n",
       "     'offsetInEndSection': 1231,\n",
       "     'text': 'The majority of these regions map onto ultraconserved elements and we demonstrate that they can act as functional enhancers within the organism of origin, as well as in cross-transgenesis experiments, and that they are transcribed in extant species of Olfactores',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23393190',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 93,\n",
       "     'offsetInEndSection': 307,\n",
       "     'text': 'Transcribed ultraconserved regions (T-UCRs) are a subset of 481 sequences longer than 200 bp, which are absolutely conserved between orthologous regions of human, rat and mouse genomes, and are actively transcribed',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22328099',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 324,\n",
       "     'offsetInEndSection': 554,\n",
       "     'text': 'Other ncRNAs, such as PIWI-interacting RNAs (piRNAs), small nucleolar RNAs (snoRNAs), transcribed ultraconserved regions (T-UCRs) and large intergenic non-coding RNAs (lincRNAs) are emerging as key elements of cellular homeostasis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22094949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 51,\n",
       "     'text': 'Transcribed ultraconserved region in human cancers.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24384562',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 730,\n",
       "     'offsetInEndSection': 859,\n",
       "     'text': 'We show that a large fraction of non-exonic UCEs is transcribed across all developmental stages examined from only one DNA strand',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20202189',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1052,\n",
       "     'offsetInEndSection': 1265,\n",
       "     'text': 'Although uc.338 is partially located within the poly(rC) binding protein 2 (PCBP2) gene, the transcribed ncRNA encoding uc.338 is expressed independently of PCBP2 and was cloned as a 590-bp RNA gene, termed TUC338',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21187392',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1624,\n",
       "     'offsetInEndSection': 1846,\n",
       "     'text': 'Moreover through our own experiments as well as the use of next-generation sequencing datasets, we were able to show that the RNAs encoded by non-exonic UCEs are likely to be long RNAs transcribed from only one DNA strand.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20202189',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Mutation of which gene is implicated in the Brain-lung-thyroid syndrome?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22166853',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20020530',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25759798',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26196025',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24129101',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22488412',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24171694',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23430038',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21867529'],\n",
       "   'ideal_answer': ['Brain-lung-thyroid syndrome (BLTS) characterized by congenital hypothyroidism, respiratory distress syndrome, and benign hereditary chorea is caused by thyroid transcription factor 1 (NKX2-1/TTF1) mutations.'],\n",
       "   'exact_answer': ['thyroid transcription factor 1'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D009154',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D005796'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56c1f03bef6e394741000053',\n",
       "   'snippets': [{'offsetInBeginSection': 151,\n",
       "     'offsetInEndSection': 362,\n",
       "     'text': ' The disorder is caused by mutations to the NKX2.1 (TITF1) gene and also forms part of the \"brain-lung-thyroid syndrome\", in which additional developmental abnormalities of lung and thyroid tissue are observed. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26196025',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 106,\n",
       "     'text': 'Novel NKX2-1 Frameshift Mutations in Patients with Atypical Phenotypes of the Brain-Lung-Thyroid Syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25759798',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 235,\n",
       "     'text': 'OBJECTIVES: To verify the involvement of NKX2-1 gene in infants with brain-lung-thyroid (BLT) syndrome and hypothyroid phenotypes variable among congenital hypothyroidism (CH) or idiopathic mild hypothyroidism (IMH) of postnatal onset.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25759798',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 879,\n",
       "     'offsetInEndSection': 988,\n",
       "     'text': 'They were carriers of new de novo heterozygous frameshift mutations of NKX2-1 (c.177delG and c.153_166del14).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25759798',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1273,\n",
       "     'offsetInEndSection': 1444,\n",
       "     'text': 'CONCLUSIONS: Two novel heterozygous frameshift mutations of NKX2-1 were identified in 2 cases selected on the basis of a BLT-like phenotype among 183 hypothyroid infants. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25759798',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 80,\n",
       "     'text': 'NKX2-1 mutations in brain-lung-thyroid syndrome: a case series of four patients.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24129101',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 207,\n",
       "     'text': 'Brain-lung-thyroid syndrome (BLTS) characterized by congenital hypothyroidism, respiratory distress syndrome, and benign hereditary chorea is caused by thyroid transcription factor 1 (NKX2-1/TTF1) mutations.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24129101',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 364,\n",
       "     'offsetInEndSection': 1250,\n",
       "     'text': 'Two of the four patients presenting with the triad of BLTS had NKX2-1 mutations, and one of these NKX2-1 [c.890_896del (p.Ala327Glyfs*52)] is a novel variant. The third patient without any identified NKX2-1 mutations was a carrier of mitochondrial mutation; this raises the possibility of mitochondrial mutations contributing to thyroid dysgenesis. Although rare, the triad of congenital hypothyroidism, neurological, and respiratory signs is highly suggestive of NKX2-1 anomalies. Screening for NKX2-1 mutations in patients with thyroid, lung, and neurological abnormalities will enable a unifying diagnosis and genetic counseling for the affected families. In addition, identification of an NKX2-1 defect would be helpful in allaying the concerns about inadequate thyroxine supplementation as the cause of neurological defects observed in some children with congenital hypothyroidism.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24129101',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 205,\n",
       "     'offsetInEndSection': 377,\n",
       "     'text': 'Loss-of-function mutations in NKX2.1, a gene vital to the normal development and function of the brain, lungs, and thyroid, have been identified in a number of individuals.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24171694',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 207,\n",
       "     'text': 'BACKGROUND: NKX2.1 mutations have been identified in patients displaying complete or partial brain-lung-thyroid syndrome, which can include benign hereditary chorea (BHC), hypothyroidism and/or lung disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22488412',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 873,\n",
       "     'offsetInEndSection': 1057,\n",
       "     'text': 'CONCLUSION: MLPA should be considered as a complementary tool in patients with partial or total brain-lung-thyroid syndrome when direct sequencing failed to identify NKX2.1 mutations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22488412',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 144,\n",
       "     'text': 'Mutations in NKX2-1 cause neurological, pulmonary, and thyroid hormone impairment. Recently, the disease was named brain-lung-thyroid syndrome. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22166853',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 384,\n",
       "     'offsetInEndSection': 808,\n",
       "     'text': 'Genetic analysis of NKX2-1 revealed a novel missense mutation (p.Val205Phe) in two patients who were cousins and their maternal families, and a novel 2.6-Mb deletion including NKX2-1 on chromosome 14 in the other patient. Congenital hypothyroidism was not detected on neonatal screening in the patient with the missense mutation, and frequent respiratory infections were observed in the patient with the deletion in NKX2-1. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22166853',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 152,\n",
       "     'offsetInEndSection': 420,\n",
       "     'text': 'Haploinsufficiency of NKX2.1, the gene encoding the thyroid transcription factor-1 (TTF-1)--critical for lung, thyroid and central nervous system morphogenesis and function--causes a rare form of progressive respiratory failure designated brain-lung-thyroid syndrome. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21867529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 506,\n",
       "     'offsetInEndSection': 607,\n",
       "     'text': 'We report a novel TTF-1 molecular defect causing recurrent respiratory failure episodes in an infant.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21867529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1257,\n",
       "     'offsetInEndSection': 1522,\n",
       "     'text': 'In conclusion, ILD in patients with NKX2-1 mutations was associated with altered surfactant protein metabolism, and both gain and loss of function of the mutated NKX2-1 genes on surfactant protein promoters were associated with ILD in \"Brain-Lung-Thyroid syndrome\".',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20020530',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 219,\n",
       "     'text': 'Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 485,\n",
       "     'offsetInEndSection': 751,\n",
       "     'text': 'The objectives of the present study were (i) detection of NKX2-1 mutations in patients with CH associated with pneumopathy and/or BHC, (ii) functional analysis of new mutations in vitro and (iii) description of the phenotypic spectrum of brain-lung-thyroid syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': 'Five new TTF1/NKX2.1 mutations in brain-lung-thyroid syndrome: rescue by PAX8 synergism in one case.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': 'NKX2-1 mutations leading to surfactant protein promoter dysregulation cause interstitial lung disease in \"Brain-Lung-Thyroid Syndrome\".',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20020530',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 149,\n",
       "     'text': 'Multiplex Ligation-dependent Probe Amplification improves the detection rate of NKX2.1 mutations in patients affected by brain-lung-thyroid syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22488412',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 195,\n",
       "     'text': 'NKX2.1 mutations have been identified in patients displaying complete or partial brain-lung-thyroid syndrome, which can include benign hereditary chorea (BHC), hypothyroidism and/or lung disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22488412',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1173,\n",
       "     'offsetInEndSection': 1376,\n",
       "     'text': 'The clinical spectrum of 6 own and 40 published patients with NKX2-1 mutations ranged from the complete triad of brain-lung-thyroid syndrome (50%), brain and thyroid disease (30%), to isolated BHC (13%).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 406,\n",
       "     'offsetInEndSection': 673,\n",
       "     'text': 'Haploinsufficiency of NKX2.1, the gene encoding the thyroid transcription factor-1 (TTF-1)--critical for lung, thyroid and central nervous system morphogenesis and function--causes a rare form of progressive respiratory failure designated brain-lung-thyroid syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21867529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 220,\n",
       "     'text': 'Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 975,\n",
       "     'offsetInEndSection': 1179,\n",
       "     'text': 'The clinical spectrum of 6 own and 40 published patients with NKX2-1 mutations ranged from the complete triad of brain-lung-thyroid syndrome (50%), brain and thyroid disease (30%), to isolated BHC (13%). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 997,\n",
       "     'offsetInEndSection': 1262,\n",
       "     'text': 'In conclusion, ILD in patients with NKX2-1 mutations was associated with altered surfactant protein metabolism, and both gain and loss of function of the mutated NKX2-1 genes on surfactant protein promoters were associated with ILD in \"Brain-Lung-Thyroid syndrome\".',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20020530',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 208,\n",
       "     'text': 'BACKGROUND: NKX2.1 mutations have been identified in patients displaying complete or partial brain-lung-thyroid syndrome, which can include benign hereditary chorea (BHC), hypothyroidism and/or lung disease. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22488412',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 876,\n",
       "     'offsetInEndSection': 1060,\n",
       "     'text': 'CONCLUSION: MLPA should be considered as a complementary tool in patients with partial or total brain-lung-thyroid syndrome when direct sequencing failed to identify NKX2.1 mutations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22488412',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 208,\n",
       "     'text': 'Brain-lung-thyroid syndrome (BLTS) characterized by congenital hypothyroidism, respiratory distress syndrome, and benign hereditary chorea is caused by thyroid transcription factor 1 (NKX2-1/TTF1) mutations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24129101',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 487,\n",
       "     'text': 'Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC). The objectives of the present study were (i) detection of NKX2-1 mutations in patients with CH associated with pneumopathy and/or BHC, (ii) functional analysis of new mutations in vitro and (iii) description of the phenotypic spectrum of brain-lung-thyroid syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 408,\n",
       "     'text': ' Mutations of genes affecting surfactant homeostasis, such as SFTPB, SFTPC and ABCA3, lead to diffuse lung disease in neonates and children. Haploinsufficiency of NKX2.1, the gene encoding the thyroid transcription factor-1 (TTF-1)--critical for lung, thyroid and central nervous system morphogenesis and function--causes a rare form of progressive respiratory failure designated brain-lung-thyroid syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21867529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 153,\n",
       "     'offsetInEndSection': 362,\n",
       "     'text': 'The disorder is caused by mutations to the NKX2.1 (TITF1) gene and also forms part of the \"brain-lung-thyroid syndrome\", in which additional developmental abnormalities of lung and thyroid tissue are observed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26196025',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 141,\n",
       "     'offsetInEndSection': 494,\n",
       "     'text': 'Haploinsufficiency of NKX2.1, the gene encoding the thyroid transcription factor-1 (TTF-1)--critical for lung, thyroid and central nervous system morphogenesis and function--causes a rare form of progressive respiratory failure designated brain-lung-thyroid syndrome. Molecular mechanisms involved in this syndrome are heterogeneous and poorly explored.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21867529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 208,\n",
       "     'text': 'Brain-lung-thyroid syndrome (BLTS) characterized by congenital hypothyroidism, respiratory distress syndrome, and benign hereditary chorea is caused by thyroid transcription factor 1 (NKX2-1/TTF1) mutations.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24129101',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 141,\n",
       "     'offsetInEndSection': 408,\n",
       "     'text': 'Haploinsufficiency of NKX2.1, the gene encoding the thyroid transcription factor-1 (TTF-1)--critical for lung, thyroid and central nervous system morphogenesis and function--causes a rare form of progressive respiratory failure designated brain-lung-thyroid syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21867529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 136,\n",
       "     'text': 'NKX2-1 mutations leading to surfactant protein promoter dysregulation cause interstitial lung disease in \"Brain-Lung-Thyroid Syndrome\".',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20020530',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 220,\n",
       "     'text': 'Thyroid transcription factor 1 (NKX2-1/TITF1) mutations cause brain-lung-thyroid syndrome, characterized by congenital hypothyroidism (CH), infant respiratory distress syndrome (IRDS) and benign hereditary chorea (BHC).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 81,\n",
       "     'text': 'NKX2-1 mutations in brain-lung-thyroid syndrome: a case series of four patients.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24129101',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 107,\n",
       "     'text': 'Novel NKX2-1 Frameshift Mutations in Patients with Atypical Phenotypes of the Brain-Lung-Thyroid Syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25759798',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 976,\n",
       "     'offsetInEndSection': 1179,\n",
       "     'text': 'The clinical spectrum of 6 own and 40 published patients with NKX2-1 mutations ranged from the complete triad of brain-lung-thyroid syndrome (50%), brain and thyroid disease (30%), to isolated BHC (13%).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19336474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 230,\n",
       "     'text': 'Mutations in the gene encoding thyroid transcription factor, NKX2-1, result in neurologic abnormalities, hypothyroidism, and neonatal respiratory distress syndrome (RDS) that together are known as the brain-thyroid-lung syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23430038',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List side effects of SGLT2 inhibitors?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24341330',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22977310',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24400675',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24825435',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23807940',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24631482',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25488697',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24455799',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23042029',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25962253'],\n",
       "   'ideal_answer': ['SGLT2 inhibitors can be associated with urogenital infections related to the enhanced glycosuria, and low blood pressure.'],\n",
       "   'exact_answer': [['urinary tract infections'],\n",
       "    ['genital infections'],\n",
       "    ['low blood pressure']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D051297',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D064420',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D065606'],\n",
       "   'type': 'list',\n",
       "   'id': '571e1e11bb137a4b0c000004',\n",
       "   'snippets': [{'offsetInBeginSection': 515,\n",
       "     'offsetInEndSection': 697,\n",
       "     'text': 'Due to side effects such as urinary tract and genital infections and decrease in blood pressure, proper patient selection for drug initiation and close monitoring will be important. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23807940',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1618,\n",
       "     'offsetInEndSection': 1732,\n",
       "     'text': 'Urogenital side-effects related to the enhanced glycosuria can be troublesome, yet seldom lead to discontinuation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24825435',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 800,\n",
       "     'offsetInEndSection': 1019,\n",
       "     'text': 'Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects, both of which are evaluated in this review. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24825435',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 800,\n",
       "     'offsetInEndSection': 1017,\n",
       "     'text': 'Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects, both of which are evaluated in this review',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24825435',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 94,\n",
       "     'text': 'Effect of SGLT2 inhibitors in a murine model of urinary tract infection with Candida albicans.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24400675',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 801,\n",
       "     'offsetInEndSection': 975,\n",
       "     'text': 'Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24825435',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1619,\n",
       "     'offsetInEndSection': 1733,\n",
       "     'text': 'Urogenital side-effects related to the enhanced glycosuria can be troublesome, yet seldom lead to discontinuation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24825435',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 801,\n",
       "     'offsetInEndSection': 1019,\n",
       "     'text': 'Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects, both of which are evaluated in this review.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24825435',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1152,\n",
       "     'offsetInEndSection': 1348,\n",
       "     'text': 'There are some side effects that warrant further investigation and establishing whether SGLT2 inhibition provides a renal benefit relies on future long-term studies with specific renal end-points.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23042029',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 920,\n",
       "     'offsetInEndSection': 1074,\n",
       "     'text': 'It should be noted that the currently approved agents have side effects that include an increased incidence of genital infections, predominantly in women.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24631482',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the mode of action of everolimus?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/17890266',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24183081',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19620795',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20384580',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19299048',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16625599',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16699448',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21279702',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18025810',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22139982'],\n",
       "   'ideal_answer': ['Everolimus is a drug that binds to mTORC1 and inhibits activation of the mTOR signalling pathway. It is used in targeted cancer therapy protocols or after transplantation for maintenance immunosuppression, against allograft rejection.'],\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4267185'],\n",
       "   'type': 'summary',\n",
       "   'id': '52f125332059c6d71c000007',\n",
       "   'snippets': [{'offsetInBeginSection': 520,\n",
       "     'offsetInEndSection': 575,\n",
       "     'text': 'Target-of-Rapamycin Inhibitors (Sirolimus, Everolimus),',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22139982',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 161,\n",
       "     'offsetInEndSection': 372,\n",
       "     'text': 'Although the mTOR inhibitor everolimus is effective for the treatment of patients who have failed TKI therapy, it is important to consider all available treatment options before switching therapy mode of action.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21279702',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 323,\n",
       "     'offsetInEndSection': 771,\n",
       "     'text': 'The mTOR pathway, and its upstream regulators in the PI3K/PTEN/AKT cascade, are altered in a variety of experimental and human malignancies.This has led to the prediction that mTOR inhibitors may be used as anticancer agents. With the recent approval of two mTOR-targeted drugs (temsirolimus and everolimus) for the treatment of renal cell carcinoma and mantle cell lymphoma, this paradigm has been effectively translated into the clinical setting.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20384580',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 406,\n",
       "     'offsetInEndSection': 811,\n",
       "     'text': 'mTOR inhibitors such as Temsirolimus (CCI779) and Everolimus (RAD001) are effective for suppressing cell growth with inhibiting mTOR kinase activity. Rapamycin and its related analogs such as Temsirolimus and Everolimus are less toxic for humans compared with other anti-VEGFR inhibitors and has been used as an immunosuppressive agent. These agents have an inhibitory activity against the mTORC1 complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19620795',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 763,\n",
       "     'offsetInEndSection': 887,\n",
       "     'text': 'mTOR inhibitors act on the signaling pathway PI3K/AKT/mTOR, and key molecules are temsirolimus, everolimus, and deforolimus.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19299048',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 494,\n",
       "     'offsetInEndSection': 690,\n",
       "     'text': 'The introduction of the inhibitors of the mammalian target of rapamycin sirolimus and everolimus in renal transplantation has increased the repertoire of immunosuppressive protocols substantially.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17890266',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': 'Octreotide and the mTOR inhibitor RAD001 (everolimus) block proliferation and interact with the Akt-mTOR-p70S6K pathway',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18025810',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 368,\n",
       "     'offsetInEndSection': 508,\n",
       "     'text': 'RAD001 (everolimus) is a novel agent that is being trialled in the treatment of neuro-endocrine tumours, and is known to interact with mTOR.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18025810',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1159,\n",
       "     'offsetInEndSection': 1305,\n",
       "     'text': 'In this cell model, octreotide and RAD001 appear to act through a similar pathway and inhibit the Akt-mTOR-p70S6 kinase pathway downstream of Akt.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18025810',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 57,\n",
       "     'text': 'Target of rapamycin inhibitors (sirolimus and everolimus)',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16699448',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 64,\n",
       "     'text': 'Target of rapamycin inhibitors (TOR-I; sirolimus and everolimus)',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16625599',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'Target of rapamycin inhibitors (TOR-I) (sirolimus, everolimus) are immunosuppressive agents with a novel mode of action',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16625599',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 367,\n",
       "     'offsetInEndSection': 625,\n",
       "     'text': 'These three classes are tyrosine kinase inhibitors with sunitinib and sorafenib, the anti-VEGF antibodies (bevacizumab which is associated with alpha interferon in the treatment of advanced kidney cancer) and mTOR inhibitors with temsirolimus and everolimus.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24183081',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which technique is used for detection of EWS/FLI1 fusion transcripts?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23475435',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20231617',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9552022',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16157025',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10379685',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24293381',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9135495',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17154184',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15363317',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15565546'],\n",
       "   'ideal_answer': ['Molecular detection of EWS-FLI1 chimeric transcripts in Ewing family tumors is carried out by reverse transcription-polymerase chain reaction (RT-PCR).'],\n",
       "   'exact_answer': ['Reverse transcription - polymerase chain reaction (RT-PCR)'],\n",
       "   'type': 'factoid',\n",
       "   'id': '553653a5bc4f83e828000007',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 520,\n",
       "     'text': 'We evaluated the feasibility and usefulness of reverse transcriptase-polymerase chain reaction (RT-PCR) on fine-needle aspirates for categorization of small blue round cell tumors (SBRCTs). A total of 51 cases, including 25 Ewing sarcoma/peripheral primitive neuroectodermal tumors (PNETs), 11 rhabdomyosarcomas, 13 neuroblastomas, and 2 desmoplastic small round cell tumors (DSRCTs) were analyzed. The detection of the EWS-FLI1 (20/25) and EWS-ERG (4/25) fusion transcripts resolved 24 of 25 cases of Ewing sarcoma/PNET',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20231617',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 133,\n",
       "     'text': 'Molecular detection of EWS-FLI1 chimeric transcripts in Ewing family tumors by nested reverse transcription-polymerase chain reaction',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10379685',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 374,\n",
       "     'offsetInEndSection': 1042,\n",
       "     'text': 'To assess the feasibility and reliability of the molecular detection of the transcript originating from the chimeric gene in paraffin-embedded tumor specimens, we performed a nested reverse transcription-polymerase chain reaction (RT-PCR)-based assay to detect the EWS-FLI1 chimeric message in a series of Ewing family tumors. Of 24 paraffin-embedded tumor specimens from 23 cases analyzed, the chimeric message was detectable in 20 (83%) specimens from 20 cases (87%) by this nested RT-PCR assay, whereas none of 7 small round cell tumors not from this family (3 alveolar rhabdomyosarcomas, 2 neuroblastomas, 2 malignant lymphomas) showed detectable chimeric messages',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10379685',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 3113,\n",
       "     'offsetInEndSection': 3359,\n",
       "     'text': \"Using our different sets of exon specific primer pairs, it was possible to detect 4 different breakpoints of ews/fli1 fusion transcripts and the ews/erg fusion by RT-PCR in RNA isolates from formalin-fixed, paraffin-embedded Ewing's tumor tissue.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15565546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1814,\n",
       "     'offsetInEndSection': 2011,\n",
       "     'text': 'Molecular detection of EWS-Fli1 fusion transcripts in formalin-fixed paraffin-embedded material by nested RT-PCR is feasible and is useful for the diagnosis and differential diagnosis of ES/pPNETs.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16157025',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1729,\n",
       "     'offsetInEndSection': 2045,\n",
       "     'text': 'We established a novel RT-PCR method, using 3 different exon specific sets of PCR primer pairs, selected according to the coding ews and fli1 nucleotide sequences (NCBI database), suitable for RT-PCR identification of variant ews/fli1 fusion transcripts in RNA isolated from formalin-fixed, paraffin-embedded tissue.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15565546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 236,\n",
       "     'offsetInEndSection': 393,\n",
       "     'text': 'In this study, we evaluated reverse transcriptase-polymerase chain (RT-PCR) for EWS-FLI1 fusion transcripts in 18 neurally derived small round cell tumours.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9135495',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 684,\n",
       "     'offsetInEndSection': 869,\n",
       "     'text': 'We performed a clinical and pathologic analysis of 112 patients with ES in which EWS-FLI1 fusion transcripts were identified by reverse-transcriptase polymerase chain reaction (RT-PCR).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9552022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 197,\n",
       "     'text': 'Molecular detection of EWS-Fli1 fusion transcripts in formalin-fixed paraffin-embedded material by nested RT-PCR is feasible and is useful for the diagnosis and differential diagnosis of ES/pPNETs.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16157025',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 467,\n",
       "     'offsetInEndSection': 677,\n",
       "     'text': 'RT-PCR confirmed that SK-NEP-1 expresses EWS-FLI1 gene fusion transcripts characteristic of Ewing sarcoma, and DNA sequencing demonstrated the joining of exon 7 of EWS with exon 5 of FLI1 for these transcripts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17154184',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 834,\n",
       "     'offsetInEndSection': 937,\n",
       "     'text': 'Reverse transcriptase-polymerase chain reaction confirmed the expression of EWS-FLI1 fusion transcripts',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23475435',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 199,\n",
       "     'offsetInEndSection': 626,\n",
       "     'text': 'The aims of this study were (1) to present the diverse clinicopathological and molecular profile of EFTs in our settings, (2) to identify a pragmatic approach for diagnosing EFTs, especially for application of ancillary techniques, namely RT-PCR for specific transcripts (EWS-FLI1, EWS-ERG) and FISH for EWSR1 gene rearrangement, in certain cases and (3) to show the utility of tissue microarray in establishing a new FISH test',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24293381',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 156,\n",
       "     'text': \"[Detection of EWS-FLI1 fusion transcript in Ewing's sarcoma/peripheral primitive neuroectodermal tumors by one-step RT-PCR using paraffin-embedded tissues].\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15363317',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1835,\n",
       "     'offsetInEndSection': 2032,\n",
       "     'text': 'Molecular detection of EWS-Fli1 fusion transcripts in formalin-fixed paraffin-embedded material by nested RT-PCR is feasible and is useful for the diagnosis and differential diagnosis of ES/pPNETs.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16157025',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1751,\n",
       "     'offsetInEndSection': 2066,\n",
       "     'text': 'We established a novel RT-PCR method, using 3 different exon specific sets of PCR primer pairs, selected according to the coding ews and fli1 nucleotide sequences (NCBI database), suitable for RT-PCR identification of variant ews/fli1 fusion transcripts in RNA isolated from formalin-fixed, paraffin-embedded tissue',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15565546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 191,\n",
       "     'text': 'Molecular detection of EWS-FLI1 chimeric transcripts in Ewing family tumors by nested reverse transcription-polymerase chain reaction: application to archival paraffin-embedded tumor tissues.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10379685',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'What is the application of the Bimolecular Fluorescence Complementation (BiFC) assay in Drosophila embryos?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/16454041',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19771334',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18573091',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17406412',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25151172',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23317900',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21091444',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18155474',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17534848'],\n",
       "   'ideal_answer': ['Bimolecular fluorescence complementation (BiFC) is a powerful method for studying protein-protein interactions in different cell types and organisms. This method was recently developed in the fruit fly Drosophila melanogaster, allowing analyzing protein interaction properties in a physiologically relevant developing context.'],\n",
       "   'exact_answer': ['The study of protein-protein interactions in a physiologically relevant developing context.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D004330',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D004331',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D013050',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D005453'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56c868a95795f9a73e000017',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 598,\n",
       "     'text': 'Bimolecular fluorescence complementation (BiFC) is a powerful method for studying protein-protein interactions in different cell types and organisms. This method was recently developed in the fruit fly Drosophila melanogaster, allowing analyzing protein interaction properties in a physiologically relevant developing context. Here we present a detailed protocol for performing BiFC with the Venus fluorescent protein in live Drosophila embryos, taking the Hox-PBC partnership as an illustrative test case. This protocol applies to any transcription factor and split fluorescent protein in general.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25151172',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 101,\n",
       "     'offsetInEndSection': 416,\n",
       "     'text': 'The understanding of developmental complexity will, therefore, require the characterization of protein interactions within their proper environment. The bimolecular fluorescence complementation (BiFC) technology offers this possibility as it enables the direct visualization of protein interactions in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 575,\n",
       "     'offsetInEndSection': 1119,\n",
       "     'text': 'Using a Hox protein partnership as a test case, we investigated the suitability of BiFC for the study of protein interactions in the living Drosophila embryo. Importantly, all BiFC parameters were established with constructs that were stably expressed under the control of endogenous promoters. Under these physiological conditions, we showed that BiFC is specific and sensitive enough to analyse dynamic protein interactions. We next used BiFC in a candidate interaction screen, which led to the identification of several Hox protein partners.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 463,\n",
       "     'offsetInEndSection': 657,\n",
       "     'text': 'Using fluorescent proteins, we previously developed a bimolecular fluorescence complementation (BiFC) assay and a multicolor BiFC assay to visualize protein-protein interactions in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16454041',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 417,\n",
       "     'offsetInEndSection': 562,\n",
       "     'text': 'The bimolecular fluorescence complementation (BiFC) assay represents one of these imaging tools for direct visualization of PPIs in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21091444',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 296,\n",
       "     'offsetInEndSection': 455,\n",
       "     'text': 'The bimolecular fluorescence complementation (BiFC) assay provides an approach for the visualization of protein interactions and modifications in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19771334',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 417,\n",
       "     'offsetInEndSection': 580,\n",
       "     'text': 'The bimolecular fluorescence complementation (BiFC) assay provides a direct approach for the visualization of molecular interactions in living cells and organisms.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18155474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 718,\n",
       "     'offsetInEndSection': 949,\n",
       "     'text': 'The purpose of this protocol is to calculate signal-to-noise (S/N) ratio in the bimolecular fluorescence complementation (BiFC) assay and to provide a semi-quantitative analysis of protein-protein interaction (PPI) in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23317900',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 479,\n",
       "     'offsetInEndSection': 605,\n",
       "     'text': 'Bimolecular fluorescence complementation (BiFC) analysis enables direct visualization of protein interactions in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18573091',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 167,\n",
       "     'text': 'The bimolecular fluorescence complementation (BiFC) assay has been widely accepted for studying in vivo detection of protein-protein interactions in several organisms.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17534848',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 121,\n",
       "     'text': 'Visualization of protein interactions in living Drosophila embryos by the bimolecular fluorescence complementation assay.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 567,\n",
       "     'offsetInEndSection': 735,\n",
       "     'text': 'RESULTS: Using a Hox protein partnership as a test case, we investigated the suitability of BiFC for the study of protein interactions in the living Drosophila embryo. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 146,\n",
       "     'text': 'Design and implementation of bimolecular fluorescence complementation (BiFC) assays for the visualization of protein interactions in living cells.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17406412',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 417,\n",
       "     'offsetInEndSection': 735,\n",
       "     'text': 'However, its potential has rarely been applied in embryos of animal model organisms and was only performed under transient protein expression levels. RESULTS: Using a Hox protein partnership as a test case, we investigated the suitability of BiFC for the study of protein interactions in the living Drosophila embryo. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 406,\n",
       "     'offsetInEndSection': 715,\n",
       "     'text': 'However, its potential has rarely been applied in embryos of animal model organisms and was only performed under transient protein expression levels.  Using a Hox protein partnership as a test case, we investigated the suitability of BiFC for the study of protein interactions in the living Drosophila embryo.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 239,\n",
       "     'offsetInEndSection': 555,\n",
       "     'text': 'The bimolecular fluorescence complementation (BiFC) technology offers this possibility as it enables the direct visualization of protein interactions in living cells. However, its potential has rarely been applied in embryos of animal model organisms and was only performed under transient protein expression levels.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 300,\n",
       "     'offsetInEndSection': 466,\n",
       "     'text': 'we previously developed a bimolecular fluorescence complementation (BiFC) assay and a multicolor BiFC assay to visualize protein-protein interactions in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16454041',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 556,\n",
       "     'offsetInEndSection': 715,\n",
       "     'text': 'Using a Hox protein partnership as a test case, we investigated the suitability of BiFC for the study of protein interactions in the living Drosophila embryo.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21276241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 272,\n",
       "     'offsetInEndSection': 466,\n",
       "     'text': 'Using fluorescent proteins, we previously developed a bimolecular fluorescence complementation (BiFC) assay and a multicolor BiFC assay to visualize protein-protein interactions in living cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16454041',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which pathological condition of the heart is known as hypertrophic cardiomyopathy (HCM)?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25044876',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23782526',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25309450',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25191275',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25209314',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25328416',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22665960',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21507890',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25228955',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25081404'],\n",
       "   'ideal_answer': ['Hypertrophic cardiomyopathy (HCM) has been recently recognized as the most common inherited cardiovascular disorder, affecting 1 in 500 adults worldwide. HCM is characterized by myocyte hypertrophy resulting in thickening of the ventricular wall, myocyte disarray, interstitial and/or replacement fibrosis, decreased ventricular cavity volume and diastolic dysfunction. HCM is also the most common cause of sudden death in the young particularly among athletes. A large proportion of patients diagnosed with HCM have mutations in sarcomeric proteins. HCM is the most prevalent genetic disorder affecting the heart and is typically inherited in an autosomal dominant pattern. Adults with cardiomyopathy suffer SCD or adverse events such as stroke and heart failure from HCM.'],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:11984'],\n",
       "   'type': 'summary',\n",
       "   'id': '54dcb29dc0bb8dce23000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 410,\n",
       "     'text': 'Hypertrophic cardiomyopathy (HCM) is a primary disease of the cardiac muscle that occurs mainly due to mutations (>1,400 variants) in genes encoding for the cardiac sarcomere. HCM, the most common familial form of cardiomyopathy, affecting one in every 500 people in the general population, is typically inherited in an autosomal dominant pattern, and presents variable expressivity and age-related penetrance.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25328416',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': 'Familial hypertrophic cardiomyopathy (HCM), due to point mutations in genes for sarcomere proteins such as myosin, occurs in 1/500 people and is the most common cause of sudden death in young individuals.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25309450',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 443,\n",
       "     'offsetInEndSection': 689,\n",
       "     'text': 'n HCM, the modified protein function leads, over years to decades, to secondary remodeling with substantial morphological changes, such as hypertrophy, myofibrillar disarray, and extensive fibrosis associated with severe functional deterioration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25309450',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'Hypertrophic cardiomyopathy (HCM) is the most common cause of sudden cardiac death (SCD) in the young, particularly among athletes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25228955',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 6,\n",
       "     'offsetInEndSection': 134,\n",
       "     'text': 'Familial hypertrophic cardiomyopathy (HCM) is one the most common heart disorders, with gene mutations in the cardiac sarcomere.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25209314',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 522,\n",
       "     'text': 'Hypertrophic cardiomyopathy (HCM) has been recently recognized as the most common inherited cardiovascular disorder, affecting 1 in 500 adults worldwide. HCM is characterized by myocyte hypertrophy resulting in thickening of the ventricular wall, myocyte disarray, interstitial and/or replacement fibrosis, decreased ventricular cavity volume and diastolic dysfunction. HCM is also the most common cause of sudden death in the young. A large proportion of patients diagnosed with HCM have mutations in sarcomeric proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25191275',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 240,\n",
       "     'text': 'Hypertrophic cardiomyopathy (HCM) is a genetic cardiomyopathy. The prevalence of phenotypic expression, in the absence of another systemic or cardiac disease causing increased left ventricular (LV) wall thickness, is estimated to be 1:500. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25081404',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 562,\n",
       "     'offsetInEndSection': 819,\n",
       "     'text': 'HCM is the most prevalent genetic disorder affecting the heart, it often goes undiagnosed until midlife after patients show symptoms of myocardial remodeling. Adults with cardiomyopathy suffer SCD or adverse events such as stroke and heart failure from HCM.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25044876',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 254,\n",
       "     'text': 'Hypertrophic cardiomyopathy (HCM) is a clinically heterogeneous autosomal dominant heart disease characterised by left ventricular hypertrophy in the absence of another cardiac or systemic disease that is capable of producing significant wall thickening.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21507890',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 189,\n",
       "     'text': 'Hypertrophic cardiomyopathy (HCM) is an autosomal dominant inherited genetic disease characterized by compensatory pathological left ventricle (LV) hypertrophy due to sarcomere dysfunction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22665960',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 217,\n",
       "     'text': 'Hypertrophic cardiomyopathy (HCM) is a clinically heterogeneous genetic heart disease characterized by left ventricular hypertrophy in the absence of another disease that could explain the wall thickening.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23782526',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which is the most important prognosis sub-classification in Chronic Lymphocytic Leukemia?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22560084',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16083281',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23468975',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20353875',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16014569',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20090781',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17786276',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19500131',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16825496',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19127482'],\n",
       "   'ideal_answer': ['The mutational status of the immunoglobulin heavy variable (IGHV) genes, defines two subsets: mutated and unmutated CLL. Unmutated CLL patients show a shorter progression-free and overall survival than mutated CLL patients.'],\n",
       "   'exact_answer': ['The mutational status of the IGHV genes.'],\n",
       "   'type': 'factoid',\n",
       "   'id': '51739df58ed59a060a00001c',\n",
       "   'snippets': [{'offsetInBeginSection': 621,\n",
       "     'offsetInEndSection': 879,\n",
       "     'text': 'One of the most reliable prognostic markers in chronic lymphocytic leukemia (CLL) is the mutational status of immunoglobulin heavy variable (IGHV) genes, which defines 2 subsets, mutated CLL (M-CLL) and unmutated CLL (U-CLL), with different clinical courses.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22560084',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 119,\n",
       "     'offsetInEndSection': 268,\n",
       "     'text': 'Mutational status of the immunoglobulin heavy chain variable regions (IGHVs) of CLL cells offers useful prognostic information for high-risk patients',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20090781',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 270,\n",
       "     'offsetInEndSection': 496,\n",
       "     'text': 'but time and economical costs originally prevented it from being routinely used in a clinical setting. Instead, alternative markers of IGHV status, such as zeta-associated protein (ZAP70) or messenger RNA levels are often used',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20090781',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 109,\n",
       "     'offsetInEndSection': 282,\n",
       "     'text': 'Patients with unmutated IgV(H) gene show a shorter progression-free and overall survival than patients with immunoglobulin heavy chain variable regions (IgV(H)) gene mutated',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19127482',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'What is the molecular function of the Chd1 protein?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/16263726',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16468993',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17098252',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21177652',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23533627',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21623345',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15647753',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23319591',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16606615',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19948887'],\n",
       "   'ideal_answer': ['The ATP-dependent chromatin-remodelling enzyme Chd1 is a 168-kDa protein consisting of a double chromodomain, Snf2-related ATPase domain, and a C-terminal DNA-binding domain. One of the two chromodomains of Chd1 specifically interacts with the methylated lysine 4 mark on histone H3 that is associated with transcriptional activity. Human CHD1 is an ATP-dependent chromatin remodeling protein, as a factor that directly and selectively recognizes histone H3 methylated on lysine 4.'],\n",
       "   'exact_answer': ['Chd1 is an ATP-dependent DNA helicase'],\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/CHD1_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/CHD1_DROME'],\n",
       "   'type': 'factoid',\n",
       "   'id': '57092332cf1c325851000018',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 95,\n",
       "     'text': 'The DNA-binding domain of the Chd1 chromatin-remodelling enzyme contains SANT and SLIDE domains',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21623345',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'The ATP-dependent chromatin-remodelling enzyme Chd1 is a 168-kDa protein consisting of a double chromodomain, Snf2-related ATPase domain, and a C-terminal DNA-binding domain.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21623345',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 449,\n",
       "     'offsetInEndSection': 728,\n",
       "     'text': 'The presence of these domains in ISWI and Chd1 chromatin-remodelling enzymes may provide a means of efficiently harnessing the action of the Snf2-related ATPase domain for the purpose of nucleosome spacing and provide an explanation for partial redundancy between these proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21623345',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 694,\n",
       "     'offsetInEndSection': 827,\n",
       "     'text': 'Here we identify the chromatin remodelling protein Chd1 (chromo-ATPase/helicase-DNA binding domain 1) as a component of SAGA and SLIK',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15647753',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 856,\n",
       "     'offsetInEndSection': 1012,\n",
       "     'text': 'one of the two chromodomains of Chd1 specifically interacts with the methylated lysine 4 mark on histone H3 that is associated with transcriptional activity',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15647753',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 121,\n",
       "     'text': 'Human but not yeast CHD1 binds directly and selectively to histone H3 methylated at lysine 4 via its tandem chromodomains',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16263726',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 196,\n",
       "     'offsetInEndSection': 375,\n",
       "     'text': 'In the current study, we identify human CHD1, an ATP-dependent chromatin remodeling protein, as a factor that directly and selectively recognizes histone H3 methylated on lysine 4',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16263726',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 597,\n",
       "     'offsetInEndSection': 692,\n",
       "     'text': 'human CHD1 binds to methylated H3K4 in a manner that requires both of its tandem chromodomains.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16263726',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 789,\n",
       "     'offsetInEndSection': 1034,\n",
       "     'text': 'Our findings indicate that yeast and human CHD1 have diverged in their ability to discriminate covalently modified histones and link histone modification-recognition and non-covalent chromatin remodeling activities within a single human protein.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16263726',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 235,\n",
       "     'offsetInEndSection': 397,\n",
       "     'text': 'We find that the yeast Isw1a, Isw2, and Chd1 enzymes preferentially move nucleosomes toward more central locations on short DNA fragments whereas Isw1b does not. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16606615',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 91,\n",
       "     'text': 'Analysis of nucleosome repositioning by yeast ISWI and Chd1 chromatin remodeling complexes.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16606615',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 555,\n",
       "     'offsetInEndSection': 719,\n",
       "     'text': 'However, a key difference is that the Isw1a, Isw2, and Chd1 enzymes are unable to move nucleosomes to positions closer than 15 bp from a DNA end, whereas Isw1b can.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16606615',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 103,\n",
       "     'text': 'The ISWI and CHD1 chromatin remodelling activities influence ADH2 expression and chromatin organization',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16468993',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 794,\n",
       "     'offsetInEndSection': 1002,\n",
       "     'text': 'Here we show that the absence of the Isw1 and Chd1 ATP-dependent chromatin remodelling activities delays the maximal expression of ADH2 without impairing the chromatin remodelling that occurs upon activation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16468993',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1494,\n",
       "     'offsetInEndSection': 1701,\n",
       "     'text': 'Thus, the ISWI and Chd1 remodelling factors are not only involved in transcription-related chromatin remodelling, but also are required to maintain a specific chromatin configuration across the yeast genome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16468993',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1352,\n",
       "     'offsetInEndSection': 1492,\n",
       "     'text': 'the lack of Chd1 combined with the absence of Isw1 and Isw2 impairs nucleosome spacing along the ADH2 gene, and genome-wide in S. cerevisiae',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16468993',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 884,\n",
       "     'offsetInEndSection': 991,\n",
       "     'text': 'Through inclusion of Chd1 sequences in homology searches SLIDE domains were identified in CHD6-9 proteins. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21623345',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 210,\n",
       "     'text': 'The molecular motor protein CHD1 has been implicated in the regulation of transcription and in the transcription-independent genome-wide incorporation of H3.3 into paternal chromatin in Drosophila melanogaster.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21177652',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1178,\n",
       "     'offsetInEndSection': 1333,\n",
       "     'text': 'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23533627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 442,\n",
       "     'offsetInEndSection': 574,\n",
       "     'text': 'CHD1 is known to bind H3K4me3 in mammalian cells, and Neurospora CHD1 is required for proper regulation of the frequency (frq) gene.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23319591',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 211,\n",
       "     'text': 'The molecular motor protein CHD1 has been implicated in the regulation of transcription and in the transcription-independent genome-wide incorporation of H3.3 into paternal chromatin in Drosophila melanogaster. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21177652',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1185,\n",
       "     'offsetInEndSection': 1358,\n",
       "     'text': 'Thus, while CHD1 is localized to transcriptionally active regions of the genome, it can function to alter the levels of HP1a, perhaps through changes in methylation of H3K9.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23533627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 125,\n",
       "     'text': 'Histone H3K4 and K36 methylation, Chd1 and Rpd3S oppose the functions of Saccharomyces cerevisiae Spt4-Spt5 in transcription.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19948887',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1029,\n",
       "     'offsetInEndSection': 1184,\n",
       "     'text': 'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23533627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 211,\n",
       "     'text': 'The molecular motor protein CHD1 has been implicated in the regulation of transcription and in the transcription-independent genome-wide incorporation of H3.3 into paternal chromatin in Drosophila melanogaster.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21177652',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1030,\n",
       "     'offsetInEndSection': 1359,\n",
       "     'text': 'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II. Thus, while CHD1 is localized to transcriptionally active regions of the genome, it can function to alter the levels of HP1a, perhaps through changes in methylation of H3K9.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23533627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 270,\n",
       "     'text': ' Chromodomain from heterochromatin protein 1 and polycomb protein is known to be a lysine-methylated histone H3 tail-binding module. Chromo-helicase/ATPase DNA-binding protein 1 (CHD1) is an ATP-dependent chromatin remodeling factor, containing two tandem chromodomains.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17098252',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 857,\n",
       "     'offsetInEndSection': 1185,\n",
       "     'text': 'Over-expression of an ATPase inactive form of CHD1 did not result in severe chromosomal defects, suggesting that the ATPase activity is required for this in vivo phenotype. Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23533627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1030,\n",
       "     'offsetInEndSection': 1185,\n",
       "     'text': 'Interestingly, changes in CHD1 protein levels did not correlate with changes in the levels of the euchromatin mark H3K4me3 or elongating RNA Polymerase II.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23533627',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is protein M3/6 a dual specificity phosphatase?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/15888437',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11566103',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23679081',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22100391',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8910287',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11948422',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12524447',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12598532',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10915787',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23159405'],\n",
       "   'ideal_answer': ['M3/6 (DUSP8) is a dual-specificity phosphatase implicated in the dephosphorylation and inactivation of JNK and, to a lesser extent, p38 MAPKs.',\n",
       "    'Yes. Phosphatases play a particularly important role in this respect, by tightly controlling MAPK phosphorylation and activation. M3/6 (DUSP8) is a dual-specificity phosphatase implicated in the dephosphorylation and inactivation of JNK and, to a lesser extent, p38 MAPKs and is found in a complex with these kinases, along with other pathway components, held together by scaffold proteins.',\n",
       "    'The protein M3/6 (DUSP8) is a dual-specificity phosphatase implicated in the dephosphorylation and inactivation of JNK'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016791',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D054638',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D054637',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0004712',\n",
       "    'http://www.uniprot.org/uniprot/DUS8_MOUSE',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D054641',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008138',\n",
       "    'http://www.uniprot.org/uniprot/DUS2_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/DUS1_RAT',\n",
       "    'http://www.uniprot.org/uniprot/DUS3_HUMAN',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0017017',\n",
       "    'http://www.uniprot.org/uniprot/DUS8_HUMAN'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5512cce26a8cde6b7200000c',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 144,\n",
       "     'text': 'Involvement of the dual-specificity phosphatase M3/6 in c-Jun N-terminal kinase inactivation following cerebral ischemia in the rat hippocampus.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23679081',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 324,\n",
       "     'offsetInEndSection': 453,\n",
       "     'text': 'The results revealed upregulation of dual-specificity phosphatase M3/6 (DUSP8) activity at 4\\xa0h of reperfusion in rat hippocampi. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23679081',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 163,\n",
       "     'offsetInEndSection': 323,\n",
       "     'text': 'This study examines the molecular mechanism underlying JNK dephosphorylation and inactivation evoked by dual-specificity phosphates following cerebral ischemia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23679081',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 291,\n",
       "     'offsetInEndSection': 677,\n",
       "     'text': 'Phosphatases play a particularly important role in this respect, by tightly controlling MAPK phosphorylation and activation. M3/6 (DUSP8) is a dual-specificity phosphatase implicated in the dephosphorylation and inactivation of JNK and, to a lesser extent, p38 MAPKs and is found in a complex with these kinases, along with other pathway components, held together by scaffold proteins. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23159405',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 151,\n",
       "     'offsetInEndSection': 448,\n",
       "     'text': 'Dual-specificity phosphatases (DUSPs) play a very important role in these events by modulating the extent of JNK phosphorylation and activation and thus regulating cellular responses to stress. M3/6 (DUSP8) is one of the dual-specificity protein phosphatases with distinct specificity towards JNK.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22100391',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 637,\n",
       "     'offsetInEndSection': 702,\n",
       "     'text': 'M3/6 is a dual-specificity phosphatase selective for JNK [7, 8]. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11566103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 253,\n",
       "     'offsetInEndSection': 555,\n",
       "     'text': 'Here we describe two new dual specificity phosphatases of the CL100/MKP-1 family that are selective for inactivating ERK or JNK/SAPK and p38 MAP kinases when expressed in COS-7 cells. M3/6 is the first phosphatase of this family to display highly specific inactivation of JNK/SAPK and p38 MAP kinases. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8910287',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 137,\n",
       "     'offsetInEndSection': 319,\n",
       "     'text': 'We previously demonstrated that the dual specificity phosphatases (DSPs) MKP7 and M3/6 bind the scaffold JNK-interacting protein-1 (JIP-1) and inactivate the bound subset of JNK (1).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15888437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 15,\n",
       "     'offsetInEndSection': 52,\n",
       "     'text': 'the dual-specificity phosphatase M3/6',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23679081',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 360,\n",
       "     'offsetInEndSection': 402,\n",
       "     'text': ' dual-specificity phosphatase M3/6 (DUSP8)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23679081',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 416,\n",
       "     'offsetInEndSection': 523,\n",
       "     'text': 'M3/6 (DUSP8) is a dual-specificity phosphatase implicated in the dephosphorylation and inactivation of JNK ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23159405',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 19,\n",
       "     'offsetInEndSection': 56,\n",
       "     'text': 'the M3/6 dual-specificity phosphatase',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22100391',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 345,\n",
       "     'offsetInEndSection': 447,\n",
       "     'text': 'M3/6 (DUSP8) is one of the dual-specificity protein phosphatases with distinct specificity towards JNK',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22100391',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 637,\n",
       "     'offsetInEndSection': 693,\n",
       "     'text': 'M3/6 is a dual-specificity phosphatase selective for JNK',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11566103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 133,\n",
       "     'text': 'The dual specificity phosphatases M3/6 and MKP-3 are highly selective for inactivation of distinct mitogen-activated protein kinases.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8910287',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': 'Phosphorylation of the M3/6 dual-specificity phosphatase enhances the activation of JNK by arsenite.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22100391',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 581,\n",
       "     'offsetInEndSection': 681,\n",
       "     'text': 'Indeed, expanded polyglutamine impaired the solubility of the dual-specificity JNK phosphatase M3/6.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12598532',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 76,\n",
       "     'text': 'Regulation of dual-specificity phosphatases M3/6 and hVH5 by phorbol esters.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10915787',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 345,\n",
       "     'offsetInEndSection': 448,\n",
       "     'text': 'M3/6 (DUSP8) is one of the dual-specificity protein phosphatases with distinct specificity towards JNK.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22100391',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 636,\n",
       "     'offsetInEndSection': 699,\n",
       "     'text': 'M3/6 is a dual-specificity phosphatase selective for JNK [7, 8]',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11566103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 344,\n",
       "     'offsetInEndSection': 446,\n",
       "     'text': 'M3/6 (DUSP8) is one of the dual-specificity protein phosphatases with distinct specificity towards JNK',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22100391',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 262,\n",
       "     'offsetInEndSection': 429,\n",
       "     'text': 'Here we describe how diverse cellular stresses affect differently the stability and activity of a JNK-inactivating dual-specificity threonine-tyrosine phosphatase M3/6',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11948422',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 637,\n",
       "     'offsetInEndSection': 700,\n",
       "     'text': 'M3/6 is a dual-specificity phosphatase selective for JNK [7, 8]',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11566103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 109,\n",
       "     'text': 'Regulation of dual-specificity phosphatases M3/6 and hVH5 by phorbol esters. Analysis of a delta-like domain.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10915787',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 324,\n",
       "     'offsetInEndSection': 451,\n",
       "     'text': 'The results revealed upregulation of dual-specificity phosphatase M3/6 (DUSP8) activity at 4\\xa0h of reperfusion in rat hippocampi',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23679081',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 581,\n",
       "     'offsetInEndSection': 680,\n",
       "     'text': 'Indeed, expanded polyglutamine impaired the solubility of the dual-specificity JNK phosphatase M3/6',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12598532',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 553,\n",
       "     'offsetInEndSection': 689,\n",
       "     'text': 'Here we report that JIP-1 also binds the dual-specificity phosphatases MKP7 and M3/6 via a region independent of its JNK binding domain.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12524447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 49,\n",
       "     'text': 'Differential regulation of M3/6 (DUSP8) signaling',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23159405',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Is the tricarboxylic acid (TCA) cycle affected in inflammation?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23776651',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24727425',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21523508',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24361092',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21287666',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25461442',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24086109',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25686106',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25798621',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18383346'],\n",
       "   'ideal_answer': ['Metabolic reprogramming is implicated in macrophage activation. In many cases, intermediates of the TCA cycle are involved in the response to hypoxic conditions brought about by inflammation.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007249',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002952',\n",
       "    'http://amigo.geneontology.org/amigo/term/GO:0006099'],\n",
       "   'type': 'yesno',\n",
       "   'id': '56c4a217b04e159d0e000001',\n",
       "   'snippets': [{'offsetInBeginSection': 210,\n",
       "     'offsetInEndSection': 555,\n",
       "     'text': 'In this study, the levels of amino acids and trichloroacetic acid (TCA) cycle-related molecules in the colonic tissues and sera of patients with ulcerative colitis (UC) were profiled by gas chromatography/mass spectrometry (GC/MS), with the aim of evaluating whether the clinical state induced by UC leads to variations in the amino acid profile',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21523508',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1142,\n",
       "     'offsetInEndSection': 1290,\n",
       "     'text': 'Our study raises the possibility that GC/MS-based profiling of amino acids and TCA cycle-related molecules is a useful early diagnostic tool for UC.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21523508',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 152,\n",
       "     'text': 'Succinate is an intermediate of the tricarboxylic acid (TCA) cycle, and plays a crucial role in adenosine triphosphate (ATP) generation in mitochondria.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24361092',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 876,\n",
       "     'offsetInEndSection': 1025,\n",
       "     'text': 'In plasma, most metabolites in the central metabolic pathway (glycolysis and TCA cycle) were significantly downregulated after zymosan administration',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23776651',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 568,\n",
       "     'offsetInEndSection': 804,\n",
       "     'text': 'Thus, IL-1beta+TNFalpha treated astrocytes show a marked decrease in glycogen levels, a slight but not significant decrease in lactate release as well as a massive increase in both the pentose phosphate pathway and TCA cycle activities.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18383346',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 663,\n",
       "     'offsetInEndSection': 879,\n",
       "     'text': 'A total of 77 and 92 metabolites were detected in serum and colon tissue, respectively, and among the metabolites the compositions of TCA cycle intermediates and amino acids changed depending on the degree of colitis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21287666',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 857,\n",
       "     'offsetInEndSection': 1119,\n",
       "     'text': 'Extension of these findings identified a functional role for stretch-induced inhibition of succinate dehydrogenase (SDH) in mediating normoxic HIF1A stabilization, concomitant increases in glycolytic capacity, and improved tricarboxylic acid (TCA) cycle function',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24086109',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1756,\n",
       "     'offsetInEndSection': 2006,\n",
       "     'text': 'These studies reveal a surprising role for HIF1A in lung protection during ALI, where normoxic HIF1A stabilization and HIF-dependent control of alveolar-epithelial glucose metabolism function as an endogenous feedback loop to dampen lung inflammation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24086109',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1347,\n",
       "     'offsetInEndSection': 1570,\n",
       "     'text': 'These results suggest a cataplerosis of the TCA cycle induced by phenobarbital, caused by the massive withdrawal of succinyl-CoA by ALAS induction, such that the TCA cycle is unable to supply the reduced cofactors to the RC',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24727425',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 639,\n",
       "     'offsetInEndSection': 910,\n",
       "     'text': 'The mitochondrial respiratory chain (RC) and the tricarboxylic acid (TCA) cycle were explored in the Hmbs(-/-) mouse model. RC and TCA cycle were significantly affected in comparison to controls in mice treated with phenobarbital with decreased activities of RC complexes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24727425',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1460,\n",
       "     'offsetInEndSection': 1711,\n",
       "     'text': 'Several changes in substrate utilization for energy homeostasis were identified in severe AAH, including increased glucose consumption by the pentose phosphate pathway, altered tricarboxylic acid (TCA) cycle activity, and enhanced peptide catabolism. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25461442',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 456,\n",
       "     'offsetInEndSection': 814,\n",
       "     'text': 'Enhanced mitochondrial glucose oxidation was achieved by increased recruitment of the NOTCH1 intracellular domain (NICD1) to nuclear and mitochondrial genes that encode respiratory chain components and by NOTCH-dependent induction of pyruvate dehydrogenase phosphatase 1 (Pdp1) expression, pyruvate dehydrogenase activity, and glucose flux to the TCA cycle. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25798621',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 63,\n",
       "     'text': 'Metabolic reprogramming is implicated in macrophage activation,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25798621',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1294,\n",
       "     'offsetInEndSection': 1526,\n",
       "     'text': 'BHB blocks the NLRP3 inflammasome without undergoing oxidation in the TCA cycle, and independently of uncoupling protein-2 (UCP2), sirtuin-2 (SIRT2), the G protein-coupled receptor GPR109A or hydrocaboxylic acid receptor 2 (HCAR2). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25686106',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1865,\n",
       "     'offsetInEndSection': 2030,\n",
       "     'text': 'Our findings suggest that the anti-inflammatory effects of caloric restriction or ketogenic diets may be linked to BHB-mediated inhibition of the NLRP3 inflammasome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25686106',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 46,\n",
       "     'text': 'Succinate: a metabolic signal in inflammation.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24361092',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Which is the most common measure of differences between dinucleotide relative abundance \"genomic signatures\"',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15046306',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15716010',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9294192',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10430918',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9190805',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7482779',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10066522',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7809131',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18953039'],\n",
       "   'ideal_answer': ['The concept of a genomic signature was introduced with the observation of species-type specific Dinucleotide Relative Abundance Profiles (DRAPs). The set of dinucleotide odds ratios or \\'general design\\' is a remarkably stable property of the DNA of an organism, and can be used to discriminate between sequences from different organisms. The average absolute dinucleotide relative abundance difference is termed delta-distance. Delta-distance is the most commonly used measure of differences bwetween \"genomic signatures\". Delta-distances between different genomic sequences in the same species are low, and are generally smaller than the between-species delta-distances.'],\n",
       "   'exact_answer': ['delta-distance'],\n",
       "   'type': 'factoid',\n",
       "   'id': '554356d0ed966d112c000005',\n",
       "   'snippets': [{'offsetInBeginSection': 594,\n",
       "     'offsetInEndSection': 780,\n",
       "     'text': \"The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature',\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18953039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 215,\n",
       "     'offsetInEndSection': 359,\n",
       "     'text': 'the concept of a genomic signature was introduced with the observation of species-type specific Dinucleotide Relative Abundance Profiles (DRAPs)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15716010',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 212,\n",
       "     'text': 'The dinucleotide relative abundance profile can be regarded as a genomic signature because, despite diversity between species, it varies little between 50 kilobase or longer windows on a given genome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 350,\n",
       "     'offsetInEndSection': 531,\n",
       "     'text': 'The profile is computed from the base step \"odds ratios\" that compare dinucleotide frequencies to those expected under the assumption of stochastic equilibrium (thorough shuffling).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 307,\n",
       "     'offsetInEndSection': 369,\n",
       "     'text': 'The genome signatures (dinucleotide relative abundance values)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10430918',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 521,\n",
       "     'offsetInEndSection': 643,\n",
       "     'text': \"the set of dinucleotide biases constitutes a 'genomic signature' that can discriminate sequences from different organisms.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10066522',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 669,\n",
       "     'offsetInEndSection': 841,\n",
       "     'text': 'the set of dinucleotide odds ratio (relative abundance) values constitute a signature of each DNA genome, which can discriminate between sequences from different organisms.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7482779',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 153,\n",
       "     'text': 'Genomic homogeneity is investigated for a broad base of DNA sequences in terms of dinucleotide relative abundance distances (abbreviated delta-distances)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7809131',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 218,\n",
       "     'offsetInEndSection': 423,\n",
       "     'text': 'delta-distances between different genomic sequences in the same species are low, only about 2 or 3 times the distance found in random DNA, and are generally smaller than the between-species delta-distances',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7809131',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 47,\n",
       "     'offsetInEndSection': 160,\n",
       "     'text': \"the set of dinucleotide odds ratios or 'general design' is a remarkably stable property of the DNA of an organism\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7482779',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 588,\n",
       "     'offsetInEndSection': 819,\n",
       "     'text': \"The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18953039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 200,\n",
       "     'text': 'The dinucleotide relative abundance profile can be regarded as a genomic signature because, despite diversity between species, it varies little between 50 kilobase or longer windows on a given genome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 365,\n",
       "     'offsetInEndSection': 601,\n",
       "     'text': 'The correlation between the genomic signature and DNA-DNA hybridisation values was high and taxa that showed less than 30% DNA-DNA binding will in general not have dinucleotide relative abundance dissimilarity (delta*) values below 40.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15046306',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 201,\n",
       "     'offsetInEndSection': 476,\n",
       "     'text': 'Previously, the concept of a genomic signature was introduced with the observation of species-type specific Dinucleotide Relative Abundance Profiles (DRAPs); dinucleotides were identified as the subsequences with the greatest bias in representation in a majority of genomes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15716010',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 133,\n",
       "     'offsetInEndSection': 488,\n",
       "     'text': 'Comparisons within and between species sample sequences are based on the profile of dinucleotide relative abundance values (The profile is rho*XY = f*XY/f*Xf*Y for all XY, where f*X denotes the frequency of the nucleotide X and f*XY denotes the frequency of the dinucleotide XY, both computed from the sequence concatenated with its inverted complement).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9294192',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 590,\n",
       "     'offsetInEndSection': 818,\n",
       "     'text': 'The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or genomic signature, between bacterial chromosomes and plasmids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18953039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 135,\n",
       "     'offsetInEndSection': 489,\n",
       "     'text': 'Comparisons within and between species sample sequences are based on the profile of dinucleotide relative abundance values (The profile is rho*XY = f*XY/f*Xf*Y for all XY, where f*X denotes the frequency of the nucleotide X and f*XY denotes the frequency of the dinucleotide XY, both computed from the sequence concatenated with its inverted complement).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9294192',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 953,\n",
       "     'offsetInEndSection': 1127,\n",
       "     'text': 'We demonstrate that the Mahalanobis distance is better than the delta-distance at measuring genomic signature differences between plasmids and chromosomes of potential hosts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18953039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 594,\n",
       "     'offsetInEndSection': 824,\n",
       "     'text': \"The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18953039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 594,\n",
       "     'offsetInEndSection': 823,\n",
       "     'text': \"The average absolute dinucleotide relative abundance difference, termed delta-distance, has been commonly used to measure differences in dinucleotide composition, or 'genomic signature', between bacterial chromosomes and plasmids\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18953039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 203,\n",
       "     'offsetInEndSection': 476,\n",
       "     'text': 'Previously, the concept of a genomic signature was introduced with the observation of species-type specific Dinucleotide Relative Abundance Profiles (DRAPs); dinucleotides were identified as the subsequences with the greatest bias in representation in a majority of genomes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15716010',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the incidence of Edwards syndrom in the european population?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/19911411',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18467377',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20584846',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18067809',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18156713',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21786507',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23421080',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19408854',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8248469',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23116348'],\n",
       "   'ideal_answer': ['Between 0.125 and 39 in every 1000 live births. Most probably 1:5000 of live-born.'],\n",
       "   'exact_answer': ['1:5000'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015994',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D014314',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:1085'],\n",
       "   'type': 'factoid',\n",
       "   'id': '52f350042059c6d71c000010',\n",
       "   'snippets': [{'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 102,\n",
       "     'text': \"dwards syndrome (trisomy 18) occurs in 1: 8000 live births and is closely related to the mother's age\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23421080',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1189,\n",
       "     'offsetInEndSection': 1274,\n",
       "     'text': 'Mean incidence was 390.06 (449.08 in boys and 327.93 in girls) per 10 000 live births',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23116348',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 340,\n",
       "     'offsetInEndSection': 396,\n",
       "     'text': 'The incidence of Edwards syndrome is 1:5000 of live-born',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21786507',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1121,\n",
       "     'offsetInEndSection': 1310,\n",
       "     'text': 'The OR for Patau syndrome was 1.10 (95% CI 0.83 to 1.45); for Edwards syndrome, 1.15 (0.96 to 1.38); for Klinefelter syndrome, 1.35 (1.02 to 1.79); and for XYY syndrome, 1.99 (0.75 to 5.26)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20584846',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 710,\n",
       "     'offsetInEndSection': 937,\n",
       "     'text': 'The live birth prevalence in the absence of prenatal screening and selective termination in England and Wales from 1997 to 2004 was 1.4 (95% CI: 1.2-1.6) per 10 000 births for trisomy 13 and 2.3 (95% CI: 2.1-2.5) for trisomy 18',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19911411',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1501,\n",
       "     'offsetInEndSection': 1997,\n",
       "     'text': 'During the period under the study, following total numbers, mean relative incidences (per 10,000 live births, in brackets) and mean prenatal diagnostics efficiency (in %) were found in following chromosomal syndromes: Down syndrome 2,244 (16.58) and 63.37%, Edwards syndrome 521 (3.85) and 79.93%, Patau syndrome 201 (1.49) and 68.87%, Turner syndrome 380 (2.81) and 79.89%, 47,XXX syndrome 61 (0.45) and 59.74%, Klinefelter syndrome 163 (1.20) and 73.65% and 47,XYY syndrome 22 (0.16) and 54.76%',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19408854',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1043,\n",
       "     'offsetInEndSection': 1278,\n",
       "     'text': 'There was evidence of space-time clustering for Down syndrome (fixed threshold of close in space: P = 0.01, NN threshold: P = 0.02), but little or no clustering for Patau (P = 0.57, P = 0.19) or Edwards (P = 0.37, P = 0.06) syndromes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18467377',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 542,\n",
       "     'offsetInEndSection': 836,\n",
       "     'text': \"Of the 49,806 pregnant women between 15 and 23 weeks' gestational age who received prenatal serum screening with a cut-off value (a risk of 1:270 for Down and 1:100 for Edwards syndrome), 2,116 (4.2%) and 196 (0.4%) were screen positive for Down syndrome and for Edwards syndrome, respectively.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18156713',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1051,\n",
       "     'offsetInEndSection': 1265,\n",
       "     'text': '26,803 of the 27,313 women (98%) were screened. The average was 25.1, and 1.7% of them were over 35. Serum screening showed that 1,244 (5%) were Down syndrome positive and 105 (0.4%) were Edwards syndrome positive.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18067809',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 362,\n",
       "     'offsetInEndSection': 728,\n",
       "     'text': '235 pregnancies of women delivered in units in the North West Thames region over a two-year period (1990-91) whose babies or fetuses were diagnosed as having Down, Edwards or Patau syndrome. RESULTS: 33% of Down syndrome, 68% of Edwards syndrome and 52% of Patau syndrome were diagnosed prenatally (before 28 weeks) in the region without the use of serum screening. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8248469',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the definition of autophagy?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24308968',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23159909',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20116986',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19323652',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20595626',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23774579',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20404488',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21778180',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23422284',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16271306'],\n",
       "   'ideal_answer': ['There are several definitions of autophagy. Among them, autophagy can be defined as a non- apoptotic programmed cell death that consists on a catabolic trafficking pathway for bulk destruction and turnover of long-lived proteins and organelles via regulated lysosomal degradation.'],\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006914',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D001343'],\n",
       "   'type': 'summary',\n",
       "   'id': '5341d15dc45e133714000018',\n",
       "   'snippets': [{'offsetInBeginSection': 494,\n",
       "     'offsetInEndSection': 604,\n",
       "     'text': 'autophagy, a process in which de novo formed membrane enclosed vesicles engulf and consume cellular components',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24308968',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 858,\n",
       "     'offsetInEndSection': 1112,\n",
       "     'text': 'As a general definition, autophagy encompasses a range of processes in which the cell degrades parts of itself within the lysosome (or the analogous organelle, the vacuole, in yeast and plants), followed by the release and reuse of the breakdown products',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23774579',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 20,\n",
       "     'offsetInEndSection': 202,\n",
       "     'text': 'definition of autophagy is the following: all processes in which intracellular material is degraded within the lysosome/vacuole and where the macromolecular constituents are recycled',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23159909',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 852,\n",
       "     'offsetInEndSection': 1023,\n",
       "     'text': 'Autophagic PCD in animals is defined as being accompanied by an increase in the number of autophagosomes, autolysosomes, and small lytic vacuoles produced by autolysosomes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21778180',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 182,\n",
       "     'text': 'utophagy is the endogenous, tightly regulated cellular \"housekeeping\" process responsible for the degradation of damaged and dysfunctional cellular organelles and protein aggregates',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20595626',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 156,\n",
       "     'text': 'utophagy has been predominantly studied as a nonselective self-digestion process that recycles macromolecules and produces energy in response to starvation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20404488',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 85,\n",
       "     'text': 'utophagy is a ubiquitous eukaryotic cytoplasmic quality and quantity control pathway',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20116986',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 153,\n",
       "     'text': 'utophagy is a catabolic trafficking pathway for bulk destruction and turnover of long-lived proteins and organelles via regulated lysosomal degradation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19323652',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 105,\n",
       "     'offsetInEndSection': 161,\n",
       "     'text': ' non-apoptotic programmed cell death, such as autophagy ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16271306',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 292,\n",
       "     'text': 'utophagy and senescence share a number of characteristics, which suggests that both responses could serve to collaterally protect the cell from the toxicity of external stress such as radiation and chemotherapy and internal forms of stress such as telomere shortening and oncogene activation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23422284',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is lambrolizumab effective for treatment of patients with melanoma ?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23724846',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24416617',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23847357',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24089441',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24499550',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23970885',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24516336',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23907003',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24348481',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23997828'],\n",
       "   'ideal_answer': [\"Lambrolizumab, a programmed death-1 receptor (PD-1)/its ligand (PD-L1) antibody, has been shown to be effective for treatment of patients with melanoma. High rate of sustained tumor regression with mainly minimal adverse effects in melanoma patients treated with lambrolizumab has been reported. Because of all these reasons PD-1/PD-L1 antibodies are considered 'drug of the year.\",\n",
       "    'Yes. In patients with advanced melanoma, including those who had had disease progression while they had been receiving ipilimumab, treatment with lambrolizumab resulted in a high rate of sustained tumor regression, with mainly grade 1 or 2 toxic effects.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D018328',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D013812',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D018326',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D016896',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:1909',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D008545'],\n",
       "   'type': 'yesno',\n",
       "   'id': '530cefaaad0bf1360c000008',\n",
       "   'snippets': [{'offsetInBeginSection': 683,\n",
       "     'offsetInEndSection': 1181,\n",
       "     'text': 'However, through parallel efforts that have showcased the efficacy of small-molecule BRAF and MAP-ERK kinase (MEK) inhibitors, as well as the immune checkpoint inhibitors, namely ipilimumab and the anti-PD1/PDL1 antibodies (lambrolizumab, nivolumab, MPDL3280), an opportunity exists to transform the treatment of melanoma specifically and cancer generally by exploring rational combinations of molecularly targeted therapies, immunotherapies, and molecular targeted therapies with immunotherapies. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24089441',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 187,\n",
       "     'text': 'Programmed death-1 receptor (PD-1)/its ligand (PD-L1) antibodies have changed the landscape in oncology in 2013. The most mature results have been obtained in advanced melanoma patients. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23907003',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 193,\n",
       "     'text': 'Merck\\'s lambrolizumab (MK-3475) monoclonal antibody received \"Breakthrough Therapy\" designation from the U.S. Food and Drug Administration in April for treating patients with advanced melanoma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23847357',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 263,\n",
       "     'text': 'The programmed death 1 (PD-1) receptor is a negative regulator of T-cell effector mechanisms that limits immune responses against cancer. We tested the anti-PD-1 antibody lambrolizumab (previously known as MK-3475) in patients with advanced melanoma. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23724846',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1745,\n",
       "     'offsetInEndSection': 1995,\n",
       "     'text': 'In patients with advanced melanoma, including those who had had disease progression while they had been receiving ipilimumab, treatment with lambrolizumab resulted in a high rate of sustained tumor regression, with mainly grade 1 or 2 toxic effects. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23724846',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 946,\n",
       "     'offsetInEndSection': 1031,\n",
       "     'text': \"Because of all these reasons PD-1/PD-L1 antibodies are considered 'drug of the year'.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23907003',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List human diseases involving genomic imprinting.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/15945483',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22960339',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12524013',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12444886',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16035043',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7868070',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15802919',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11568896',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20822494',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8565331'],\n",
       "   'ideal_answer': ['Prader Willi Syndrome\\nAngelman syndrome\\nBeckwith-Wiedemann syndrome\\nHydatidiform mole\\nCancer\\nSilver-Russell syndrome\\nDiabetes'],\n",
       "   'exact_answer': [['Prader Willi Syndrome'],\n",
       "    ['Angelman syndrome'],\n",
       "    ['Beckwith-Wiedemann syndrome'],\n",
       "    ['Hydatidiform mole'],\n",
       "    ['Cancer'],\n",
       "    ['Silver-Russell syndrome'],\n",
       "    ['Diabetes']],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:4',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015894',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:2000653',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D018392',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0071514',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D016678'],\n",
       "   'type': 'list',\n",
       "   'id': '54ff30dc6ad7dcbc1200000f',\n",
       "   'snippets': [{'offsetInBeginSection': 252,\n",
       "     'offsetInEndSection': 371,\n",
       "     'text': ' Here we describe the clinical case of a female patient with Prader Willi Syndrome (PWS), a genomic imprinting disorder',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22960339',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 775,\n",
       "     'offsetInEndSection': 977,\n",
       "     'text': 'Perturbation of the allelic DNA methylation at ICRs is causally involved in several human diseases, including the Beckwith-Wiedemann and Silver-Russell syndromes, associated with aberrant foetal growth.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20822494',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 978,\n",
       "     'offsetInEndSection': 1114,\n",
       "     'text': 'Perturbed imprinted gene expression is also implicated in the neuro-developmental disorders Prader-Willi syndrome and Angelman syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20822494',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 674,\n",
       "     'offsetInEndSection': 943,\n",
       "     'text': 'This epigenetic \"life cycle\" of imprinting (germline erasure, germline establishment, and somatic maintenance) can be disrupted in several human diseases, including Beckwith-Wiedemann syndrome (BWS), Prader-Willi syndrome (PWS), Angelman syndrome and Hydatidiform mole.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16035043',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 944,\n",
       "     'offsetInEndSection': 1044,\n",
       "     'text': 'In the neurodevelopmental Rett syndrome, the way the ICR mediates imprinted expression is perturbed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16035043',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 234,\n",
       "     'offsetInEndSection': 455,\n",
       "     'text': 'eregulation of imprinted genes has been observed in a number of human diseases as gestation trophoblastic disease, Prader-Willi, Angelmann and Beckwith-Wiedemann syndromes and plays significant role in the carcinogenesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15945483',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1014,\n",
       "     'offsetInEndSection': 1077,\n",
       "     'text': \"Angelman's syndrome families, which are known to be imprinted. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15802919',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 160,\n",
       "     'text': 'Deregulation of imprinted genes has been observed in a number of human diseases such as Beckwith-Wiedemann syndrome, Prader-Willi/Angelman syndromes and cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12524013',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 438,\n",
       "     'text': 'Genomic imprinting is the phenomenon whereby some genes preferentially produce mRNA transcripts from the gene copy derived from the parent of a specific sex. It has been implicated in a number of human diseases (most of them of endocrine interest), such as Prader-Willi/Angelman syndromes, Silver-Russell syndrome, Beckwith-Wiedemann syndrome, transient neonatal diabetes, the focal form of nesidioblastosis, and pseudohypoparathyroidism.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12444886',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 125,\n",
       "     'offsetInEndSection': 205,\n",
       "     'text': 'imprinting disorders like Beckwith-Wiedemann and Prader-Willi/Angelman syndromes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11568896',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 586,\n",
       "     'offsetInEndSection': 821,\n",
       "     'text': 'It is particularly interesting from the clinical point of view that a number of human diseases, such as the Beckwith-Wiedemann and Prader-Willi/Angelman syndromes, appear to involve unbalanced parental contributions of imprinted loci. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7868070',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 411,\n",
       "     'offsetInEndSection': 769,\n",
       "     'text': 'Examples include the Prader-Willi, Angelman, and Beckwith-Wiedemann syndromes [Nicholls (1994): Am J Hum Genet 54:733-740], malignancy [Sapienza (1990): Biochim Biophys Acta 1072:51-61; Feinberg (1993): Nat Genet 4:110-113], and insulin-dependent diabetes mellitus (IDDM) [Julier et al. (1994) Nature 354:155-159; Bennett et al. (1995) Nat Genet 9:284-292]. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8565331',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is there an association between bruxism and reflux?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22926484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15520695',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19089153',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23738993',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22156738',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24011800',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14655925',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21248360',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19830044',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/841409'],\n",
       "   'ideal_answer': ['Yes, bruxism is associated with reflux. Sleep bruxism is prevalent in GERD patients.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D020186',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:2846',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002012',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D005764',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D001244',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:8534'],\n",
       "   'type': 'yesno',\n",
       "   'id': '530cefaaad0bf1360c00000f',\n",
       "   'snippets': [{'offsetInBeginSection': 1564,\n",
       "     'offsetInEndSection': 1647,\n",
       "     'text': 'Sleep bruxism is prevalent in GERD patients, and GERD is highly associated with SB.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24011800',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1103,\n",
       "     'offsetInEndSection': 1219,\n",
       "     'text': 'There was a statistical trend towards tooth wear progression being associated with gastric risk factors (p < 0.05). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22156738',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 338,\n",
       "     'offsetInEndSection': 482,\n",
       "     'text': 'This article presents a case report of a 27-year-old male smoker with tooth wear and dentin sensitivity caused by GERD associated with bruxism. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23738993',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 143,\n",
       "     'text': 'The aim of this cross-over, randomized, single-blinded trial was to examine whether intra-esophageal acidification induces sleep bruxism (SB). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21248360',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 672,\n",
       "     'offsetInEndSection': 741,\n",
       "     'text': 'RMMA episodes including SB were induced by esophageal acidification. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21248360',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 224,\n",
       "     'text': 'Chronic regurgitation of gastric acids in patients with gastroesophageal reflux disease may cause dental erosion, which can lead in combination with attrition or bruxism to extensive loss of coronal tooth tissue.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19830044',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 234,\n",
       "     'offsetInEndSection': 489,\n",
       "     'text': 'This clinical report describes treatment of severe tooth wear of a gastroesophageal reflux disease patient who is 54-year-old Turkish male patient. After his medical treatment, severe tooth wear, bruxism and decreased vertical dimensions were determined. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19830044',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 956,\n",
       "     'offsetInEndSection': 1084,\n",
       "     'text': 'Gastroesophageal reflux disease by itself or in combination with attrition, abrasion or bruxism may be responsible for the loss.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19830044',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 355,\n",
       "     'offsetInEndSection': 910,\n",
       "     'text': 'The association between bruxism, feeding and smoking habits and digestive disorders may lead to serious consequences to dental and related structures, involving dental alterations (wear, fractures and cracks), periodontal signs (gingival recession and tooth mobility) and muscle-joint sensitivity, demanding a multidisciplinary treatment plan. This paper presents a case report in which bruxism associated with acid feeding, smoking habit and episodes of gastric reflow caused severe tooth wear and great muscular discomfort with daily headache episodes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19089153',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 518,\n",
       "     'offsetInEndSection': 683,\n",
       "     'text': 'The frequencies of RMMA, single short-burst, and clenching episodes were significantly higher during decreased esophageal pH episodes than those during other times. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15520695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 804,\n",
       "     'offsetInEndSection': 988,\n",
       "     'text': 'These results suggest that most jaw muscle activities, ie, RMMA, single short-burst, and clenching episodes, occur in relation to gastroesophageal reflux mainly in the supine position.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15520695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2192,\n",
       "     'offsetInEndSection': 2328,\n",
       "     'text': 'Nocturnal bruxism may be secondary to nocturnal gastroesophageal reflux, occurring via sleep arousal and often together with swallowing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14655925',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What molecule is targeted by brodalumab?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25093016',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26422722',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24646743',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24200404',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24552447',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24918373',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25599143',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25713988',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25246805',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22455412'],\n",
       "   'ideal_answer': ['Interleukin-17. Brodalumab is anti interleukin-17 monoclonal antibody.'],\n",
       "   'exact_answer': ['Interleukin-17'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56bc9268ac7ad1001900001b',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 159,\n",
       "     'text': 'BACKGROUND: Early clinical studies suggested that the anti-interleukin-17 receptor A monoclonal antibody brodalumab has efficacy in the treatment of psoriasis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26422722',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 364,\n",
       "     'offsetInEndSection': 559,\n",
       "     'text': 'METHODS: We reviewed the results of the phase II clinical trials for the anti-IL-17 agents secukinumab, ixekizumab and brodalumab in order to assess the efficacy and safety profile of each agent.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24552447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 421,\n",
       "     'offsetInEndSection': 618,\n",
       "     'text': 'These include the IL-17 antagonists, secukinumab, brodalumab and ixekizumab; the IL-23 antagonists, guselkumab and tildrakizumab; and the oral small molecule therapies, tofacitinib and apremilast. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25713988',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1021,\n",
       "     'offsetInEndSection': 1426,\n",
       "     'text': 'RECENT FINDINGS: New drugs that are designed to inhibit steps in this pathway, the IL12/IL23 inhibitor, ustekinumab, the IL17A inhibitors secukinumab and ixekizumab, the IL17A receptor inhibitor, brodalumab, and the IL23 inhibitors guselkumab and tildrakizumab, have demonstrated significant effectiveness in treating these diseases, particularly psoriasis, psoriatic arthritis and ankylosing spondylitis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25599143',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 71,\n",
       "     'text': 'Brodalumab, an anti-IL17RA monoclonal antibody, in psoriatic arthritis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24918373',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 67,\n",
       "     'text': 'Brodalumab, an anti-interleukin-17-receptor antibody for psoriasis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22455412',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 144,\n",
       "     'text': 'Randomized, double-blind, placebo-controlled study of brodalumab, a human anti-IL-17 receptor monoclonal antibody, in moderate to severe asthma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24200404',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 132,\n",
       "     'text': 'Gene expression profiles normalized in psoriatic skin by treatment with brodalumab, a human anti-IL-17 receptor monoclonal antibody.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24646743',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 403,\n",
       "     'offsetInEndSection': 614,\n",
       "     'text': 'The three new therapies with biologic drugs - brodalumab, secukinumab, and ixekizumab - all target the IL-17 signaling pathway. Secukinumab and ixekizumab neutralize IL-17A, while brodalumab blocks its receptor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25246805',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 448,\n",
       "     'offsetInEndSection': 520,\n",
       "     'text': 'Brodalumab is a human monoclonal antibody that targets IL-17 receptor A,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25093016',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can we detect DNA strand asymmetries using dinucleotide relative abundance \"genomic signatures\"?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/15046306',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15716010',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9294192',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18799480',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9520433',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10430918',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9190805',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7482779',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10066522'],\n",
       "   'ideal_answer': ['The set of dinucleotide relative abundances can be regarded as a genomic signature because, despite diversity between species, it varies little between 50 kilobase or longer windows on a given genome. Thus, dinucleotide relative abundance profiles are species-type specific. These profiles are computed from the base step \"odds ratios\" that compare dinucleotide frequencies to those expected under the assumption of stochastic equilibrium (thorough shuffling). Dinucleotide relative abundance \"genomic signatures\" are strand-independent second-order DNA features. Thus, they cannot be used to detect DNA strand asymmetries.'],\n",
       "   'exact_answer': 'no',\n",
       "   'type': 'yesno',\n",
       "   'id': '55435aeced966d112c000007',\n",
       "   'snippets': [{'offsetInBeginSection': 523,\n",
       "     'offsetInEndSection': 702,\n",
       "     'text': 'comparing the heterogeneities of bacterial genomes with respect to strand-independent first- and second-order features, (i) G + C content and (ii) dinucleotide relative abundance,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18799480',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 215,\n",
       "     'offsetInEndSection': 477,\n",
       "     'text': 'the concept of a genomic signature was introduced with the observation of species-type specific Dinucleotide Relative Abundance Profiles (DRAPs); dinucleotides were identified as the subsequences with the greatest bias in representation in a majority of genomes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15716010',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 54,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'dinucleotide relative abundance values (the genomic signature)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15046306',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 212,\n",
       "     'text': 'The dinucleotide relative abundance profile can be regarded as a genomic signature because, despite diversity between species, it varies little between 50 kilobase or longer windows on a given genome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 350,\n",
       "     'offsetInEndSection': 532,\n",
       "     'text': 'The profile is computed from the base step \"odds ratios\" that compare dinucleotide frequencies to those expected under the assumption of stochastic equilibrium (thorough shuffling). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 307,\n",
       "     'offsetInEndSection': 369,\n",
       "     'text': 'The genome signatures (dinucleotide relative abundance values)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10430918',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'Early biochemical experiments measuring nearest neighbor frequencies established that the set of dinucleotide relative abundance values (dinucleotide biases) is a remarkably stable property of the DNA of an organism.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10066522',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 521,\n",
       "     'offsetInEndSection': 643,\n",
       "     'text': \"the set of dinucleotide biases constitutes a 'genomic signature' that can discriminate sequences from different organisms.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10066522',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 669,\n",
       "     'offsetInEndSection': 773,\n",
       "     'text': 'the set of dinucleotide odds ratio (relative abundance) values constitute a signature of each DNA genome',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7482779',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 62,\n",
       "     'text': 'Dinucleotide relative abundance extremes: a genomic signature.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7482779',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 200,\n",
       "     'text': 'The dinucleotide relative abundance profile can be regarded as a genomic signature because, despite diversity between species, it varies little between 50 kilobase or longer windows on a given genome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12171605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 201,\n",
       "     'offsetInEndSection': 476,\n",
       "     'text': 'Previously, the concept of a genomic signature was introduced with the observation of species-type specific Dinucleotide Relative Abundance Profiles (DRAPs); dinucleotides were identified as the subsequences with the greatest bias in representation in a majority of genomes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15716010',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 133,\n",
       "     'offsetInEndSection': 488,\n",
       "     'text': 'Comparisons within and between species sample sequences are based on the profile of dinucleotide relative abundance values (The profile is rho*XY = f*XY/f*Xf*Y for all XY, where f*X denotes the frequency of the nucleotide X and f*XY denotes the frequency of the dinucleotide XY, both computed from the sequence concatenated with its inverted complement).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9294192',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 705,\n",
       "     'offsetInEndSection': 887,\n",
       "     'text': 'Dinucleotide relative abundances (i.e., dinucleotide representations normalized by the component nucleotide frequencies) are consonant with respect to the leading and lagging strands',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9520433',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What was the aim of the COSS (Carotid Occlusion Surgery Study) clinical trial?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/21772967',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22682265',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21960571',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22244016',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22645702',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22220280',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22068990',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23101451',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23909253',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24339571'],\n",
       "   'triples': [{'p': 'http://data.linkedct.org/resource/linkedct/description',\n",
       "     's': 'http://data.linkedct.org/resource/trials/NCT00029146',\n",
       "     'o': '\\n      The overall purpose of this research is to determine if a surgical operation called\\n      \"Extracranial-Intracranial Bypass\" can reduce the chance of a subsequent stroke in someone\\n      who has complete blockage in one main artery in the neck (the carotid artery) that supplies\\n      blood to the brain and has already suffered a small stroke. This surgery involves taking an\\n      artery from the scalp outside the skull, making a small hole in the skull and then connecting\\n      the scalp artery to a brain artery inside the skull. In this way the blockage of the carotid\\n      artery in the neck is bypassed and more blood can flow to the brain. In some people natural\\n      bypass arteries develop and the brain is already getting plenty of blood. These people have a\\n      low risk of stroke if they take medicine. In other people, no natural bypass arteries develop\\n      so less blood flows to their brains. This second group has a much higher risk of stroke while\\n      taking medicine, as high as 25-50% within the next two years. It is this second group of\\n      people who may benefit from having the bypass operation and who are the candidates for this\\n      study.\\n\\n      This bypass surgery is considered experimental because it is not generally performed for this\\n      condition and it is unknown whether it leads to a decrease, an increase or no change in the\\n      risk of stroke. In order to determine if people fit into this second group of people who may\\n      benefit from the bypass operation they need to have a test called a PET scan. The PET scan\\n      measures the amount of blood that is getting to the brain and the amount of oxygen that the\\n      brain is using. The PET scan uses radioactive oxygen and water and is experimental (not\\n      approved by the United States Food and Drug Administration). If the PET scan shows that less\\n      blood is getting to the brain, there will be a 50-50 chance (like a coin toss) of receiving\\n      the bypass surgery or not. There will then be follow-up visits to the clinic one month later\\n      and then every three months for two years to check on the appropriate medical treatment that\\n      everyone will receive and to determine who has had a stroke.\\n\\n      The study hypothesis is that extracranial-intracranial bypass surgery when added to best\\n      medical therapy can reduce by 40 percent  subsequent stroke within  two years in participants\\n      with recent TIA (\\'ministroke\") or stroke (</= 120 days) due to blockage of the carotid artery\\n      and reduced blood flow to the brain measured by PET.\\n    '},\n",
       "    {'p': 'http://data.linkedct.org/resource/linkedct/summary',\n",
       "     's': 'http://data.linkedct.org/resource/trials/NCT00029146',\n",
       "     'o': '\\n      The purpose of this study is to determine if extracranial-intracranial bypass surgery when\\n      added to best medical therapy can reduce the subsequent risk of ipsilateral stroke in\\n      high-risk patients with recently symptomatic carotid occlusion and increased cerebral oxygen\\n      extraction fraction measured by PET.\\n    '}],\n",
       "   'ideal_answer': ['The Carotid Occlusion Surgery Study (COSS) was conducted to determine if superficial temporal artery-middle cerebral artery (STA-MCA) bypass, when added to the best medical therapy, would reduce subsequent ipsilateral stroke in patients with complete internal carotid artery (ICA) occlusion and an elevated oxygen extraction fraction (OEF) in the cerebral hemisphere distal to the occlusion.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002986',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D016430'],\n",
       "   'type': 'summary',\n",
       "   'id': '5314c045dae131f847000007',\n",
       "   'snippets': [{'offsetInBeginSection': 8,\n",
       "     'offsetInEndSection': 399,\n",
       "     'text': 'The Carotid Occlusion Surgery Study (COSS) was conducted to determine if superficial temporal artery-middle cerebral artery (STA-MCA) bypass, when added to the best medical therapy, would reduce subsequent ipsilateral stroke in patients with complete internal carotid artery (ICA) occlusion and an elevated oxygen extraction fraction (OEF) in the cerebral hemisphere distal to the occlusion.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23101451',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': 'Extracranial-intracranial bypass surgery for stroke prevention in hemodynamic cerebral ischemia: the Carotid Occlusion Surgery Study randomized trial.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22068990',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 201,\n",
       "     'offsetInEndSection': 436,\n",
       "     'text': 'To test the hypothesis that extracranial-intracranial (EC-IC) bypass surgery, added to best medical therapy, reduces subsequent ipsilateral ischemic stroke in patients with recently symptomatic AICAO and hemodynamic cerebral ischemia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22068990',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 8,\n",
       "     'offsetInEndSection': 343,\n",
       "     'text': 'The Carotid Occlusion Surgery Study (COSS) was a large, prospective clinical trial that examined whether superficial temporal artery-middle cerebral artery (STA-MCA) bypass, in addition to best medical therapy, reduced the risk of ipsilateral ischemic stroke in patients with carotid artery occlusion and hemodynamic cerebral ischemia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23909253',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the treatment of neuropathic pain in children?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22401313',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22147611',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18363625',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7561230',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15265351',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22735246',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18820538',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21332246',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12712053',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11902308'],\n",
       "   'ideal_answer': ['It is unclear if any treatment is registered for pediatric use. The reported treatments are:\\nOxcarbazepine \\nOpioids alone, in rotations  or with Analgesics (e.g. Ketamine and Lidocaine infusion)\\nOpioids and Benzodiazepines\\nPregabalin -  is one of the first drugs registered for the treatment of neuropathic pain. It is unclear if Pregabalin is registered for the treatment of neuropathic pain in children specifically but it is being used in practice.\\nTricyclic Antidepressants\\nLidocaine 5% patches for chronic localized neuropathic pain'],\n",
       "   'exact_answer': [['Oxcarbazepine'],\n",
       "    ['Opioids alone, in rotations or with Analgesics (e.g. Ketamine and Lidocaine infusion)'],\n",
       "    ['Opioids and Benzodiazepines - for terminal care'],\n",
       "    ['Pregabalin'],\n",
       "    ['Tricyclic Antidepressants'],\n",
       "    ['Lidocaine 5% patches for chronic localized neuropathic pain'],\n",
       "    ['Ketamine']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D010146',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D013812',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:0060164'],\n",
       "   'type': 'list',\n",
       "   'id': '515da6e2298dcd4e51000010',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 336,\n",
       "     'text': 'Oxcarbazepine, a metabolite of carbamazepine, is used as an antiepileptic, analgesic for neuropathic pain and in the treatment of affective disorders. It has been approved by the Food and Drug Administration for partial seizures in adults as both adjunctive and monotherapy, and as adjunctive therapy in children aged from 2 to 16 years',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22735246',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': 'For difficult to treat neuropathic pain from cancer, adjuvant analgesics are often used with opioids',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22401313',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 448,\n",
       "     'offsetInEndSection': 593,\n",
       "     'text': 'Ketamine and lidocaine can be safely infused together with concomitant opioids for the treatment of refractory neuropathic pain caused by cancer.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22401313',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1245,\n",
       "     'offsetInEndSection': 1417,\n",
       "     'text': 'Patient-controlled analgesia with morphine as continuous subcutaneous or intravenous infusions and the possibility of a bolus injection is suited for children aged 6 years.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22147611',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1773,\n",
       "     'offsetInEndSection': 1881,\n",
       "     'text': 'In neuropathic pain or phantom pain coanalgetics should be used to effectively treat pain in young patients.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22147611',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 236,\n",
       "     'text': 'Pregabalin is one of the first drugs registered for the treatment of neuropathic pain. It is also indicated as adjuvant therapy in the treatment of epilepsy and for generalized anxiety disorder. Pregabalin is a GABA analogue',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21332246',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 694,\n",
       "     'offsetInEndSection': 1166,\n",
       "     'text': 'Treatment was typically initiated with a tricyclic antidepressant, and 5 of the 6 girls noted improvement in their symptoms, including 2 who had marked improvement, and another 3 with substantial improvement who were able to discontinue therapy without a recurrence. CONCLUSIONS: Vulvodynia does occur among young girls and, when treated as a neuropathic pain disorder, was found to dramatically improve or remit in the majority of those treated in this small case series.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18820538',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 288,\n",
       "     'text': 'We describe a case series of five adolescents who were managed with lidocaine 5% patches for chronic localized neuropathic pain from a variety of causes with minimal adverse effects. Treatment was effective in four of five patients with only one patient complaining of minimal pain relief',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18363625',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 317,\n",
       "     'offsetInEndSection': 1022,\n",
       "     'text': 'Twenty-two children or 14% of children on opioid therapy underwent 30 opioid rotations. Mucositis was the cause of pain in 19 (70%) children, bone pain in 3 (11%) children, and postoperative, visceral, or neuropathic pain in the remainder. The opioid was rotated either for excessive side effects with adequate analgesia (70%), excessive side effects with inadequate analgesia (16.7%), or tolerance (6.7%). Five (23%) children required two rotations, 3 during the same admission. The favored rotations were morphine to fentanyl in 20 (67%) children and fentanyl to hydromorphone in 6 (20%). Adverse opioid effects were resolved in 90% of cases, all failures occurred when morphine was rotated to fentanyl.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15265351',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 268,\n",
       "     'text': 'To test the hypothesis that children with terminal cancer and neuropathic pain require rapid increases of opioids and benzodiazepines immediately before death, we compared drug usage in the last 72 hours of life in children with and without neuropathic pain',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12712053',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1164,\n",
       "     'offsetInEndSection': 1394,\n",
       "     'text': 'Dying children with cancer and neuropathic pain have higher baseline requirements of morphine and benzodiazepines and require rapid increases of both drugs in the last 72 hours of life than dying children without neuropathic pain.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12712053',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1299,\n",
       "     'offsetInEndSection': 1580,\n",
       "     'text': 'ketamine treatment may be effective in children with severe neuropathic pain not responsive to other analgesics. This patient also demonstrates the feasibility of long-term ketamine treatment in pediatric oncology and that such treatment can be administered in a home care setting.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11902308',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 734,\n",
       "     'text': \"Oral amitriptyline has been used as an analgesic in a wide range of pain settings. Despite long-term availability of a parenteral form, the few reports about this formulation have been limited to pharmacokinetic studies in normal volunteers, trials in depressed patients, and analyses of electroencephalogram (EEG) activation. We retrospectively reviewed our experience using intravenous (IV) amitriptyline at Children's Hospital, Boston and at Children's Hospital at Stanford. Eight children (aged 5-16.6 years), who were unable to tolerate medications by the oral route, received IV amitriptyline for a variety of indications, including neuropathic pain, depression, sleep disturbance, and as an adjuvant agent for opioid analgesia.\",\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7561230',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Can exosomes be detected in urine?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24250247',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24315007',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24205503',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24101370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24196483',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24044569',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24060994',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24069349',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23585095',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24058411'],\n",
       "   'ideal_answer': ['Yes, urinary exosomes can be detected in urine.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D014556',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D055354'],\n",
       "   'type': 'yesno',\n",
       "   'id': '550895c12e93f0133a000002',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 370,\n",
       "     'text': 'Exosomes are nanovesicles secreted into the extracellular environment upon internal vesicle fusion with the plasma membrane. The molecular content of exosomes is a fingerprint of the releasing cell type and of its status. For this reason, and because they are released in easily accessible body fluids such as blood and urine, they represent a precious biomedical tool. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24044569',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 67,\n",
       "     'text': 'Exosomes are vesicles that are released from the kidney into urine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24060994',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 75,\n",
       "     'text': 'Quantification of human urinary exosomes by nanoparticle tracking analysis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24060994',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': 'Urinary extracellular vesicles (uEVs) are released by cells throughout the nephron and contain biomolecules from their cells of origin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24069349',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 66,\n",
       "     'text': 'Urinary exosomes have been proposed as potential diagnostic tools.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24058411',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 85,\n",
       "     'text': 'Urinary exosomes as a source of kidney dysfunction biomarker in renal transplantation',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24315007',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 73,\n",
       "     'offsetInEndSection': 231,\n",
       "     'text': '. Here we sought to optimize the methodologies for the isolation and quantification of urinary exosomal microRNA as a prelude to biomarker discovery studies. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24250247',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': 'Exosomes are small (30-150\\u2009nm) vesicles containing unique RNA and protein cargo, secreted by all cell types in culture. They are also found in abundance in body fluids including blood, saliva, and urine. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24205503',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 173,\n",
       "     'text': 'Urinary exosome-like vesicles (ELVs) are a heterogenous mixture (diameter 40-200\\u2009nm) containing vesicles shed from all segments of the nephron including glomerular podocytes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24196483',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': 'Exosomes are cytoplasm containing vesicles released by many cells that can be found in several biological fluids including urine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24101370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': 'Proteomic analysis of urinary exosomes in cardiovascular and associated kidney diseases by two-dimensional electrophoresis and LC-MS/MS',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23585095',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Which protein is causing Netherton syndrome?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23344365',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24015757',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24292773',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21697885',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21895535',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24138501',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23347305',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23407075',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24211642',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20657595'],\n",
       "   'triples': [{'p': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/diseasome/associatedGene',\n",
       "     's': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/diseases/3345',\n",
       "     'o': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/genes/SPINK5'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#exactMatch',\n",
       "     's': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/genes/SPINK5',\n",
       "     'o': 'http://www.dbpedia.org/resource/SPINK5'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/genes/SPINK5',\n",
       "     'o': 'SPINK5'},\n",
       "    {'p': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/diseasome/name',\n",
       "     's': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/diseases/3345',\n",
       "     'o': 'Netherton syndrome, 256500'},\n",
       "    {'p': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/diseasome/associatedGene',\n",
       "     's': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/diseases/3345',\n",
       "     'o': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/genes/LEKTI'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://www4.wiwiss.fu-berlin.de/diseasome/resource/genes/LEKTI',\n",
       "     'o': 'LEKTI'}],\n",
       "   'ideal_answer': ['Netherton syndrome (NS) is a serious inherited skin disorder caused by mutations in the gene SPINK5 (serine protease inhibitor Kazal type 5) which encodes for a serine protease inhibitor LEKTI (lymphoepithelial Kazal type-related inhibitor)'],\n",
       "   'exact_answer': ['LEKTI', 'lymphoepithelial Kazal type-related inhibitor'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D056770',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D013577',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:0050474',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:225'],\n",
       "   'type': 'factoid',\n",
       "   'id': '54ff45966ad7dcbc12000010',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 193,\n",
       "     'text': 'Netherton syndrome (NTS) is a rare genetic skin disease caused by mutations in the serine protease inhibitor Kazal-type 5 gene, which encodes the lympho-epithelial Kazal-type-related inhibitor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24292773',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 111,\n",
       "     'text': 'Netherton syndrome is caused by loss-of-function mutations in SPINK5 encoding the Kazal-type inhibitor LEKTI-1 ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24211642',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': 'Netherton syndrome (NS) is a serious inherited skin disorder caused by mutations in the gene SPINK5 (serine protease inhibitor Kazal type 5) which encodes for a serine protease inhibitor LEKTI (lymphoepithelial Kazal type-related inhibitor)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24138501',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 153,\n",
       "     'offsetInEndSection': 308,\n",
       "     'text': 'Mutations in the serine protease inhibitor Kazal type 5 (SPINK5) gene leading to lymphoepithelial Kazal-type-related inhibitor (LEKTI) deficiency cause NS.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24015757',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 323,\n",
       "     'offsetInEndSection': 502,\n",
       "     'text': 'NS is due to loss-of-function mutations in the SPINK5 gene and to the consequent lack of expression of its encoded protein LEKTI in the skin and all stratified epithelial tissues.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23407075',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 779,\n",
       "     'offsetInEndSection': 879,\n",
       "     'text': 'syndrome and caused by a genetic mutation in SPINK5, may be a facilitating factor for the infection.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23347305',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 177,\n",
       "     'offsetInEndSection': 382,\n",
       "     'text': 'NS is caused by loss-of-function mutations in SPINK5 (serine protease inhibitor of kazal type 5) encoding LEKTI-1 (lympho-epithelial kazal type related inhibitor type 5) expressed in stratified epithelia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23344365',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 345,\n",
       "     'offsetInEndSection': 445,\n",
       "     'text': 'Netherton syndrome, which arises due to mutations in serine protease inhibitor Kazal-type 5 (SPINK5)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21895535',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': 'Lympho-epithelial Kazal-type-related inhibitor (LEKTI) is the defective protein of the ichthyosiform condition Netherton syndrome (NS).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21697885',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 172,\n",
       "     'text': 'Deficiency in the serine protease inhibitor LEKTI is the etiological origin of Netherton syndrome, which causes detachment of the stratum corneum and chronic inflammation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20657595',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which proteins participate in the formation of the Notch transcriptional activation complex?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/20972443',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18758478',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11536431',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20118921',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21124806',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12644465',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12205678',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21245387',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16530044',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22325781'],\n",
       "   'ideal_answer': ['The Notch intracellular domain (NICD) forms a transcriptional activation complex with the DNA-binding factor CSL and a transcriptional co-activator of the Mastermind family (MAML). ICN binds to a highly conserved DNA-binding transcription factor called CSL (also known as RBP-Jkappa, CBF1, Suppressor of Hairless, and Lag-1) and recruits Mastermind-like transcriptional co-activators to form a transcriptional activation complex.',\n",
       "    'Although it is well understood that N(ICD) forms a transcriptional activation complex, little is known about how the complex is assembled. The Notch intracellular domain (NICD) forms a transcriptional activation complex with the DNA-binding factor CSL and a transcriptional co-activator of the Mastermind family (MAML).'],\n",
       "   'exact_answer': [['Notch intracellular domain (NICD)',\n",
       "     'NICD',\n",
       "     'Intracellular part of Notch',\n",
       "     'ICN',\n",
       "     'NIC'],\n",
       "    ['Transcription factor CSL',\n",
       "     'RBP-Jkappa',\n",
       "     'CBF1',\n",
       "     'Suppressor of Hairless',\n",
       "     'Lag-1'],\n",
       "    ['The coactivator protein Mastermind-like 1', 'MAML-1']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015533',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D051880'],\n",
       "   'type': 'list',\n",
       "   'id': '54fb6fb5d176fff445000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 335,\n",
       "     'text': 'The Notch intracellular domain (NICD) forms a transcriptional activation complex with the DNA-binding factor CSL and a transcriptional co-activator of the Mastermind family (MAML). The \"RAM\" region of NICD recruits Notch to CSL, facilitating the binding of MAML at the interface between the ankyrin (ANK) repeat domain of NICD and CSL.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22325781',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 152,\n",
       "     'offsetInEndSection': 623,\n",
       "     'text': 'Inappropriate release of the intracellular domain of Notch (N(ICD)) from the plasma membrane results in the accumulation of deregulated nuclear N(ICD) that has been linked to human cancers, notably T-cell acute lymphoblastic leukemia (T-ALL). Nuclear N(ICD) forms a transcriptional activation complex by interacting with the coactivator protein Mastermind-like 1 and the DNA binding protein CSL (for CBF-1/Suppressor of Hairless/Lag-1) to regulate target gene expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21245387',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 624,\n",
       "     'offsetInEndSection': 762,\n",
       "     'text': 'Although it is well understood that N(ICD) forms a transcriptional activation complex, little is known about how the complex is assembled.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21245387',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 640,\n",
       "     'text': 'Canonical Notch signaling is initiated when ligand binding induces proteolytic release of the intracellular part of Notch (ICN) from the cell membrane. ICN then travels into the nucleus where it drives the assembly of a transcriptional activation complex containing the DNA-binding transcription factor CSL, ICN, and a specialized co-activator of the Mastermind family. A consensus DNA binding site motif for the CSL protein was previously defined using selection-based methods, but whether subsequent association of Notch and Mastermind-like proteins affects the DNA binding preferences of CSL has not previously been examined.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21124806',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 166,\n",
       "     'offsetInEndSection': 414,\n",
       "     'text': 'We report here the crystal structure of a Notch transcriptional activation complex containing the ankyrin domain of human Notch1 (ANK), the transcription factor CSL on cognate DNA, and a polypeptide from the coactivator Mastermind-like-1 (MAML-1). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16530044',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 457,\n",
       "     'text': 'Ligand binding by Notch receptors triggers a series of proteolytic cleavages that liberate the intracellular portion of Notch (ICN) from the cell membrane, permitting it to translocate to the nucleus. Nuclear ICN binds to a highly conserved DNA-binding transcription factor called CSL (also known as RBP-Jkappa, CBF1, Suppressor of Hairless, and Lag-1) and recruits Mastermind-like transcriptional co-activators to form a transcriptional activation complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12644465',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1588,\n",
       "     'offsetInEndSection': 1709,\n",
       "     'text': 'On the basis of our results, we present a working structural model for the organization of the MAML1.ICN.CSL.DNA complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12644465',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1429,\n",
       "     'offsetInEndSection': 1587,\n",
       "     'text': 'The requirement for cooperative assembly of the MAML1.ICN.CSL.DNA complex suggests that a primary function of ICN is to render CSL competent for MAML loading.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12644465',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 369,\n",
       "     'offsetInEndSection': 642,\n",
       "     'text': 'We report here that the astrogliogenic role of Notch is in part mediated by direct binding of the Notch intracellular domain to the CSL DNA binding protein, forming a transcriptional activation complex onto the astrocyte marker gene, glial fibrillary acidic protein (GFAP).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12205678',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 870,\n",
       "     'offsetInEndSection': 1103,\n",
       "     'text': 'Importantly, although the classical CSL-dependent Notch signaling pathway is intact and able to activate the Notch canonical target promoter during the neurogenic phase, it is unable to activate the GFAP promoter during neurogenesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12205678',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 189,\n",
       "     'text': 'Mastermind (Mam) is a component of Notch pathway signaling. In combination with the intracellular domain of Notch and Suppressor of Hairless, Mam forms a transcriptional activation complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11536431',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 366,\n",
       "     'offsetInEndSection': 564,\n",
       "     'text': 'High levels of Notch-mediated transcriptional activation require the formation of a ternary complex consisting of NotchICD, CSL (CBF-1, suppressor of hairless, LAG-1) and a Mastermind family member.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20118921',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 161,\n",
       "     'offsetInEndSection': 353,\n",
       "     'text': 'This function relies on the formation of transcriptional activation complexes that include intracellular Notch, a Mastermind co-activator and the transcription factor CSL bound to cognate DNA.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20972443',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 161,\n",
       "     'offsetInEndSection': 352,\n",
       "     'text': 'This function relies on the formation of transcriptional activation complexes that include intracellular Notch, a Mastermind co-activator and the transcription factor CSL bound to cognate DNA',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20972443',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 366,\n",
       "     'offsetInEndSection': 563,\n",
       "     'text': 'High levels of Notch-mediated transcriptional activation require the formation of a ternary complex consisting of NotchICD, CSL (CBF-1, suppressor of hairless, LAG-1) and a Mastermind family member',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20118921',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 461,\n",
       "     'offsetInEndSection': 638,\n",
       "     'text': 'NotchIC, together with the transcriptional coactivator Mastermind, form a ternary complex with CSL that activates transcription from genes that are responsive to Notch signaling',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18758478',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 395,\n",
       "     'offsetInEndSection': 622,\n",
       "     'text': 'Nuclear N(ICD) forms a transcriptional activation complex by interacting with the coactivator protein Mastermind-like 1 and the DNA binding protein CSL (for CBF-1/Suppressor of Hairless/Lag-1) to regulate target gene expression',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21245387',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 201,\n",
       "     'offsetInEndSection': 456,\n",
       "     'text': 'Nuclear ICN binds to a highly conserved DNA-binding transcription factor called CSL (also known as RBP-Jkappa, CBF1, Suppressor of Hairless, and Lag-1) and recruits Mastermind-like transcriptional co-activators to form a transcriptional activation complex',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12644465',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What molecule is targeted by suvorexant?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22920041',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25489915',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25406050',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25533960',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25667197',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23197752',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24757363',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25397996',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21473737',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26478806'],\n",
       "   'ideal_answer': ['Suvorexant is a dual orexin receptor antagonist for the treatment of sleep onset and sleep maintenance insomnia.'],\n",
       "   'exact_answer': ['orexin'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56c1f003ef6e394741000039',\n",
       "   'snippets': [{'offsetInBeginSection': 605,\n",
       "     'offsetInEndSection': 773,\n",
       "     'text': 'Suvorexant is the first DORA to be approved and has demonstrated efficacy at decreasing both time to sleep onset and increasing total sleep time compared with placebo. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26478806',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 110,\n",
       "     'text': 'Suvorexant: a dual orexin receptor antagonist for the treatment of sleep onset and sleep maintenance insomnia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25667197',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1221,\n",
       "     'offsetInEndSection': 1329,\n",
       "     'text': 'CONCLUSION: Suvorexant is the first dual orexin receptor antagonist approved for the treatment of insomnia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25667197',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 89,\n",
       "     'text': 'Crystal structure of the human OX2 orexin receptor bound to the insomnia drug suvorexant.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25533960',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 362,\n",
       "     'offsetInEndSection': 594,\n",
       "     'text': ' The human OX2 receptor (OX2R) belongs to the β branch of the rhodopsin family of GPCRs, and can bind to diverse compounds including the native agonist peptides orexin-A and orexin-B and the potent therapeutic inhibitor suvorexant. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25533960',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 96,\n",
       "     'text': 'Orexin receptor antagonism for treatment of insomnia: a randomized clinical trial of suvorexant.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23197752',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 73,\n",
       "     'text': 'Promotion of sleep by suvorexant-a novel dual orexin receptor antagonist.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21473737',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 588,\n",
       "     'offsetInEndSection': 1041,\n",
       "     'text': 'The neurotransmitter systems on which the development of these agents were based included serotonin for ondansetron and lorcaserin, dopamine for varenicline, substance P (or neurokinin) for aprepitant, melatonin for ramelteon, and orexin for suvorexant. The indications were chemotherapy-induced nausea and vomiting for ondansetron and aprepitant, smoking cessation for varenicline, weight loss for lorcaserin, and insomnia for suvorexant and ramelteon.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25406050',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'Suvorexant, a dual orexin receptor antagonist for the management of insomnia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24757363',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'Suvorexant, a dual orexin receptor antagonist for the management of insomnia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24757363',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 302,\n",
       "     'text': 'The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25489915',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 207,\n",
       "     'offsetInEndSection': 283,\n",
       "     'text': 'Suvorexant helps in decreasing wakefulness by counteracting orexin activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25397996',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 303,\n",
       "     'text': ' The orexin-1 and orexin-2 receptors are two G protein-coupled receptors that bind the neuropeptides orexin-A and orexin-B. Dual antagonism of the receptors by small molecules is clinically efficacious in the treatment of insomnia, where the most advanced molecule suvorexant has recently been approved.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25489915',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 388,\n",
       "     'offsetInEndSection': 760,\n",
       "     'text': 'Hypnotic drug development has arguably become more focused in recent years, particularly upon the highly anticipated novel target, the orexin (hypocretin) system. Merck&apos;s suvorexant (MK-4305) is the first compound of the so-called dual orexin receptor antagonist (DORA) class expected to be submitted for FDA approval, with a new drug application anticipated in 2012.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22920041',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'How is OCT3 associated with serotonin?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/20402963',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19033200',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19280114',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19025979',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21636115',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24618127',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24246570',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23982114',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12584728',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19371745'],\n",
       "   'ideal_answer': ['OCT3 plays a role in serotonin clearance'],\n",
       "   'exact_answer': ['serotonin clearance'],\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4274509',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012701',\n",
       "    'http://www.biosemantics.org/jochem#http://www.biosemantics.org/jochem#:4274509',\n",
       "    'http://www.uniprot.org/uniprot/OCT3_ARATH'],\n",
       "   'type': 'factoid',\n",
       "   'id': '571e2beabb137a4b0c000006',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 268,\n",
       "     'text': 'The organic cation transporter 3 (OCT3) is a widely expressed transporter for endogenous and exogenous organic cations. Of particular interest is OCT3 expression and function in the brain, where it plays a role in serotonin clearance and influences mood and behavior. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23982114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 761,\n",
       "     'offsetInEndSection': 959,\n",
       "     'text': 'Interestingly, OCT3 mRNA is however also significantly up-regulated in the hippocampus of serotonin transporter knockout mice where it might serve as an alternative reuptake mechanism for serotonin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19280114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 266,\n",
       "     'offsetInEndSection': 413,\n",
       "     'text': 'Of particular interest is OCT3 expression and function in the brain, where it plays a role in serotonin clearance and influences mood and behavior.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23982114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 989,\n",
       "     'offsetInEndSection': 1152,\n",
       "     'text': 'However, OCT3 was determined to be a high-capacity and low-affinity transporter for the neurotransmitters dopamine (DA), norepinephrine (NE), and serotonin (5-HT).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20402963',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 206,\n",
       "     'text': 'Organic cation transporter 3 (OCT3) is a high-capacity, low-affinity transporter that mediates bidirectional, sodium-independent transport of dopamine, norepinephrine, epinephrine, serotonin, and histamine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19025979',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 361,\n",
       "     'text': 'The effect of blockade of either 5-hydroxytryptamine (5-HT)/serotonin transporter (SERT) with citalopram or the organic cation transporter 3 (OCT3)/plasma membrane monoamine transporter (PMAT) with decynium-22 (D-22) on spontaneous and evoked release of 5-HT in the nucleus tractus solitarius (NTS) was investigated in rat brainstem slices treated with gabazine',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24618127',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 120,\n",
       "     'offsetInEndSection': 266,\n",
       "     'text': 'Of particular interest is OCT3 expression and function in the brain, where it plays a role in serotonin clearance and influences mood and behavior',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23982114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 831,\n",
       "     'offsetInEndSection': 993,\n",
       "     'text': 'However, OCT3 was determined to be a high-capacity and low-affinity transporter for the neurotransmitters dopamine (DA), norepinephrine (NE), and serotonin (5-HT)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20402963',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 567,\n",
       "     'offsetInEndSection': 764,\n",
       "     'text': 'Interestingly, OCT3 mRNA is however also significantly up-regulated in the hippocampus of serotonin transporter knockout mice where it might serve as an alternative reuptake mechanism for serotonin',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19280114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 205,\n",
       "     'text': 'Organic cation transporter 3 (OCT3) is a high-capacity, low-affinity transporter that mediates bidirectional, sodium-independent transport of dopamine, norepinephrine, epinephrine, serotonin, and histamine',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19025979',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1008,\n",
       "     'offsetInEndSection': 1386,\n",
       "     'text': 'RESULTS: We found a drastic decrease in IL-4 production by stimulated basophils on exposure to serotonin (5-hydroxytryptamine [5-HT]) that is taken up by basophils through the specific high-affinity transporters serotonin transporter and the polyspecific, high-capacity organic cation transporter 3 (OCT3; or Slc22a3) but inhibits their function exclusively through the latter. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21636115',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 120,\n",
       "     'offsetInEndSection': 268,\n",
       "     'text': 'Of particular interest is OCT3 expression and function in the brain, where it plays a role in serotonin clearance and influences mood and behavior. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23982114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 831,\n",
       "     'offsetInEndSection': 995,\n",
       "     'text': 'However, OCT3 was determined to be a high-capacity and low-affinity transporter for the neurotransmitters dopamine (DA), norepinephrine (NE), and serotonin (5-HT). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20402963',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'Organic cation transporter 3: Keeping the brake on extracellular serotonin in serotonin-transporter-deficient mice.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19033200',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 118,\n",
       "     'text': 'Organic cation transporter capable of transporting serotonin is up-regulated in serotonin transporter-deficient mice.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12584728',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 568,\n",
       "     'offsetInEndSection': 766,\n",
       "     'text': 'Interestingly, OCT3 mRNA is however also significantly up-regulated in the hippocampus of serotonin transporter knockout mice where it might serve as an alternative reuptake mechanism for serotonin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19280114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 121,\n",
       "     'offsetInEndSection': 268,\n",
       "     'text': 'Of particular interest is OCT3 expression and function in the brain, where it plays a role in serotonin clearance and influences mood and behavior.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23982114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 832,\n",
       "     'offsetInEndSection': 995,\n",
       "     'text': 'However, OCT3 was determined to be a high-capacity and low-affinity transporter for the neurotransmitters dopamine (DA), norepinephrine (NE), and serotonin (5-HT).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20402963',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 977,\n",
       "     'offsetInEndSection': 1345,\n",
       "     'text': 'We found a drastic decrease in IL-4 production by stimulated basophils on exposure to serotonin (5-hydroxytryptamine [5-HT]) that is taken up by basophils through the specific high-affinity transporters serotonin transporter and the polyspecific, high-capacity organic cation transporter 3 (OCT3; or Slc22a3) but inhibits their function exclusively through the latter.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21636115',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the role of ELMO1 gene in cell migration?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/11595183',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22503503',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24819662',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23591873',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16377631',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20466982',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12029088',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15952790'],\n",
       "   'ideal_answer': ['ELMO proteins are also known to regulate actin cytoskeleton reorganization through activation of the small GTPbinding protein Rac via the ELMO-Dock180 complex. In mammalian cells, ELMO1 interacts with Dock180 as a component of the CrkII/Dock180/Rac pathway responsible for phagocytosis and cell migration.  We also show that Hck and ELMO1 interact in intact cells and that ELMO1 is heavily tyrosine-phosphorylated in cells that co-express Hck, suggesting that it is a substrate of Hck.The ELMO1/DOCK180 complex then forms a guanine nucleotide exchange factor for Rac1, regulating its activation during cell migration in different biological systems. Rac activation by the ELMO.Dock180 complex at discrete intracellular locations is mediated by the N-terminal 330 amino acids of ELMO1 rather than generalized Rac activation plays a role in cell migration.',\n",
       "    'Engulfment and cell motility 1 (Elmo1) has been reported to cooperate with dedicator of cytokinesis 1 (Dock180) and to be linked to the invasive phenotype of cancer cells through activating small G-protein Rac.'],\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/ELMO1_HUMAN',\n",
       "    'http://amigo.geneontology.org/amigo/term/GO:0016477'],\n",
       "   'type': 'summary',\n",
       "   'id': '56c2fbd91e69116444000001',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 82,\n",
       "     'text': 'Elmo1 helps dock180 to regulate Rac1 activity and cell migration of ovarian cancer',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24819662',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 220,\n",
       "     'text': 'Engulfment and cell motility 1 (Elmo1) has been reported to cooperate with dedicator of cytokinesis 1 (Dock180) and to be linked to the invasive phenotype of cancer cells through activating small G-protein Rac',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24819662',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1325,\n",
       "     'offsetInEndSection': 1513,\n",
       "     'text': 'Engulfment and cell motility 1 presents with synergetic action in helping Dock180 to activate Rac1 and promote cell motility, and thus promote untoward expansion and aggressiveness of SOC.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24819662',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 179,\n",
       "     'text': 'Elmo1 and Elmo2 are highly homologous cytoplasmic adaptor proteins that interact with Dock family guanine nucleotide exchange factors to promote activation of the small GTPase Rac',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1532,\n",
       "     'offsetInEndSection': 1711,\n",
       "     'text': 'This work provides valuable insights into the molecular regulation of Dock2 by Elmo1 that can be used to design improved inhibitors that target the Elmo-Dock-Rac signaling complex',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1228,\n",
       "     'offsetInEndSection': 1403,\n",
       "     'text': 'Taken together, these findings reveal a previously unknown, nonredundant role for Elmo1 in controlling Dock2 levels and Dock2-dependent T cell migration in primary lymphocytes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 410,\n",
       "     'offsetInEndSection': 491,\n",
       "     'text': 'Here we show that CXCL12 stimulation promotes interaction between Gαi2 and ELMO1.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23591873',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 492,\n",
       "     'offsetInEndSection': 762,\n",
       "     'text': 'Gi signalling and ELMO1 are both required for CXCL12-mediated actin polymerization, migration and invasion of breast cancer cells. CXCL12 triggers a Gαi2-dependent membrane translocation of ELMO1, which associates with Dock180 to activate small G-proteins Rac1 and Rac2.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23591873',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 899,\n",
       "     'offsetInEndSection': 1073,\n",
       "     'text': 'Our findings indicate that a chemokine-controlled pathway, consisting of Gαi2, ELMO1/Dock180, Rac1 and Rac2, regulates the actin cytoskeleton during breast cancer metastasis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23591873',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1138,\n",
       "     'offsetInEndSection': 1327,\n",
       "     'text': 'These findings suggest that clearance of apoptotic cells in living vertebrates is accomplished by the combined actions of apoptotic cell migration and elmo1-dependent macrophage engulfment.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22503503',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 689,\n",
       "     'offsetInEndSection': 928,\n",
       "     'text': 'Mechanistically, we identified Netrin-1 and its receptor Unc5B as upstream activators of the ELMO1/DOCK180 complex, regulating its functional interaction and leading to Rac1 activation in endothelial cells and vessel formation in zebrafish',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20466982',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 63,\n",
       "     'offsetInEndSection': 222,\n",
       "     'text': 'The ELMO1/DOCK180 complex forms a guanine nucleotide exchange factor for Rac1, regulating its activation during cell migration in different biological systems.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20466982',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 217,\n",
       "     'offsetInEndSection': 376,\n",
       "     'text': 'ELMO proteins are also known to regulate actin cytoskeleton reorganization through activation of the small GTPbinding protein Rac via the ELMO-Dock180 complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16377631',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1092,\n",
       "     'offsetInEndSection': 1591,\n",
       "     'text': 'Finally, in contrast to most other ERM-binding proteins, ELMO1 binding occurred independently of the state of radixin C-terminal phosphorylation, suggesting an ELMO1 interaction with both the active and inactive forms of ERM proteins and implying a possible role of ELMO in localizing or retaining ERM proteins in certain cellular sites. Together these data suggest that ELMO1-mediated cytoskeletal changes may be coordinated with ERM protein crosslinking activity during dynamic cellular functions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16377631',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 553,\n",
       "     'offsetInEndSection': 672,\n",
       "     'text': 'In mammalian fibroblasts, ELMO1 binds to Dock180, and functions upstream of Rac during phagocytosis and cell migration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15952790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 673,\n",
       "     'offsetInEndSection': 771,\n",
       "     'text': 'We previously showed that ELMO1 binds directly to the Hck SH3 domain and is phosphorylated by Hck.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15952790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1278,\n",
       "     'offsetInEndSection': 1473,\n",
       "     'text': 'The data suggest that Src family kinase mediated tyrosine phosphorylation of ELMO1 might represent an important regulatory mechanism that controls signaling through the ELMO1/Crk/Dock180 pathway.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15952790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 952,\n",
       "     'offsetInEndSection': 1045,\n",
       "     'text': 'We also found that ELMO1 regulated multiple Dock180 superfamily members to promote migration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1295,\n",
       "     'offsetInEndSection': 1522,\n",
       "     'text': 'This finding suggests that Rac activation by the ELMO.Dock180 complex at discrete intracellular locations mediated by the N-terminal 330 amino acids of ELMO1 rather than generalized Rac activation plays a role in cell migration',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 671,\n",
       "     'offsetInEndSection': 816,\n",
       "     'text': 'In mammalian cells, ELMO1 interacts with Dock180 as a component of the CrkII/Dock180/Rac pathway responsible for phagocytosis and cell migration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12029088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1120,\n",
       "     'offsetInEndSection': 1325,\n",
       "     'text': 'The binding of ELMO1 to Hck is specifically dependent on the interaction of a polyproline motif with the SH3 domain of Hck. Our results suggest that these proteins may be novel activators/effectors of Hck.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12029088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 941,\n",
       "     'offsetInEndSection': 1118,\n",
       "     'text': 'We also show that Hck and ELMO1 interact in intact cells and that ELMO1 is heavily tyrosine-phosphorylated in cells that co-express Hck, suggesting that it is a substrate of Hck',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12029088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1244,\n",
       "     'offsetInEndSection': 1403,\n",
       "     'text': 'these findings reveal a previously unknown, nonredundant role for Elmo1 in controlling Dock2 levels and Dock2-dependent T cell migration in primary lymphocytes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 64,\n",
       "     'text': 'Essential role of Elmo1 in Dock2-dependent lymphocyte migration.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 102,\n",
       "     'text': 'Dock180 and ELMO1 proteins cooperate to promote evolutionarily conserved Rac-dependent cell migration.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1228,\n",
       "     'offsetInEndSection': 1405,\n",
       "     'text': 'Taken together, these findings reveal a previously unknown, nonredundant role for Elmo1 in controlling Dock2 levels and Dock2-dependent T cell migration in primary lymphocytes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1289,\n",
       "     'offsetInEndSection': 1405,\n",
       "     'text': 'nonredundant role for Elmo1 in controlling Dock2 levels and Dock2-dependent T cell migration in primary lymphocytes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24821968',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 454,\n",
       "     'offsetInEndSection': 596,\n",
       "     'text': 'Here, we addressed the role of Dock180 and ELMO1 proteins, which function as a complex to mediate Rac activation, in mammalian cell migration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1325,\n",
       "     'offsetInEndSection': 1512,\n",
       "     'text': 'Engulfment and cell motility 1 presents with synergetic action in helping Dock180 to activate Rac1 and promote cell motility, and thus promote untoward expansion and aggressiveness of SOC',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24819662',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 453,\n",
       "     'offsetInEndSection': 594,\n",
       "     'text': 'Here, we addressed the role of Dock180 and ELMO1 proteins, which function as a complex to mediate Rac activation, in mammalian cell migration',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 453,\n",
       "     'offsetInEndSection': 596,\n",
       "     'text': 'Here, we addressed the role of Dock180 and ELMO1 proteins, which function as a complex to mediate Rac activation, in mammalian cell migration. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1295,\n",
       "     'offsetInEndSection': 1523,\n",
       "     'text': 'This finding suggests that Rac activation by the ELMO.Dock180 complex at discrete intracellular locations mediated by the N-terminal 330 amino acids of ELMO1 rather than generalized Rac activation plays a role in cell migration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14638695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 692,\n",
       "     'offsetInEndSection': 830,\n",
       "     'text': 'These studies identify CED-12/ELMO as an upstream regulator of Rac1 that affects engulfment and cell migration from C. elegans to mammals.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11595183',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 468,\n",
       "     'offsetInEndSection': 673,\n",
       "     'text': 'elegans, ced-12 is required for the engulfment of dying cells and for cell migration. In mammalian fibroblasts, ELMO1 binds to Dock180, and functions upstream of Rac during phagocytosis and cell migration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15952790',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is known about Vancomycin dosing in neonates?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/21862473',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22488303',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10103340',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3991250',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21455009',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10801244',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2630235',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21378399',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11378679',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22892931'],\n",
       "   'ideal_answer': ['Staphylococcus epidermis, including methicillin-resistant strains, are inhibited by vancomycin concentrations of 1-4 µg/ml. \\nStaphylococcus pyogenes, Streptococcus pneumonia, and Streptococcus viridans are susceptible to 2 µg/ml vancomycin. \\nBacillus spp. are inhibited by 2 µg/ml, Corynebacterium spp. by 0.04-3.1 µg/ml and Clostridium spp. by 0.39-6 µg/ml vancomycin. \\nPeak and trough concentrations of vancomycin should be 40 μg/ml and 10 μg/ml, respectively, to both be effective and avoid oto- or nephrotoxicity in adults. There is no ideal pattern of vancomycin dosing; vancomycin dosages must be individualized. Because vancomycin activity is primarily time-dependent, the 24-h area under the curve (AUC0-24h) divided by the minimum inhibitor concentration (MIC) value (AUC0-24h/MIC) is a better predictor of efficacy. In adults with MIC values less than 1 μg/ml, trough concentrations greater than 10 µg/ml result in AUC0-24h/MIC values 400\\nCompared with adults, neonates have a higher extracellular fluid volume and a limited renal elimination capacity resulting in different pharmacokinetics subject to maturation stage.  Infants weighing less than 1,000 gm had significantly larger volumes of drug distribution and consequently longer drug half-lives than larger premature infants, regardless of postconceptual or actual age. These differences alter the vancomycin dosing recommendations in these two groups of premature infants.\\nVancomycin-associated nephrotoxicity is rare in neonates, even with serum peak concentrations 40 microg/mL. Vancomycin is associated with ototoxicity.\\nThere is no consensus on vancomycin dosing in newborns and young infants, which leads to significant variation in vancomycin dosing regimens and TDM guidance across neonatal units. The development of standardized, evidence-based protocols should be prioritized.'],\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4272288',\n",
       "    'http://www.biosemantics.org/jochem#4272290',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D014640',\n",
       "    'http://www.biosemantics.org/jochem#4022642',\n",
       "    'http://www.biosemantics.org/jochem#4145076',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007231',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007234'],\n",
       "   'type': 'summary',\n",
       "   'id': '515d71ee298dcd4e5100000b',\n",
       "   'snippets': [{'offsetInBeginSection': 2117,\n",
       "     'offsetInEndSection': 2534,\n",
       "     'text': 'Monte Carlo simulations based on our population pharmacokinetic model suggest that vancomycin dosing guidelines based on serum creatinine concentration have a greater likelihood of achieving trough concentrations in the 5-15-mg/L range compared with other evaluated dosing regimens. None of the four dosing regimens is suitable to produce target trough concentration of 15-20 mg/L in an acceptable number of patients.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22488303',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 827,\n",
       "     'offsetInEndSection': 1039,\n",
       "     'text': 'There is significant variation in gentamicin and vancomycin dosing regimens and TDM guidance across a UK network of neonatal units. The development of standardized, evidence-based protocols should be prioritized.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21862473',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1323,\n",
       "     'offsetInEndSection': 1380,\n",
       "     'text': 'Vancomycin, by contrast, was associated with ototoxicity.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21455009',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 85,\n",
       "     'text': 'There is no consensus on vancomycin dosing in newborns and young infants.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21378399',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1244,\n",
       "     'offsetInEndSection': 1391,\n",
       "     'text': 'The modified regimen for a target vancomycin concentration of 25 mg/l consisted of a bolus of 20 mg/kg followed by continuous infusion of 30 mg/kg.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21378399',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1767,\n",
       "     'offsetInEndSection': 1857,\n",
       "     'text': 'There is no ideal pattern of vancomycin dosing; vancomycin dosages must be individualized.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11378679',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1598,\n",
       "     'offsetInEndSection': 1775,\n",
       "     'text': 'Recommended vancomycin schedules for term newborn infants with neonatal sepsis should be based on the weight and postconceptual age only to start antimicrobial therapy. There is',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11378679',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 927,\n",
       "     'offsetInEndSection': 1879,\n",
       "     'text': 'Targets were a trough concentration between 5 and 15 mg/L and a peak below 40 mg/L. In the prospective study, the optimal scheme was tested in 22 patients. RESULTS: Of the 108 patients, 34.3% of measured trough concentrations and 17.6% of peak concentrations were outside the desired therapeutic range. The model that best fitted the data included clearance and volume per kilogram and was independent of gestational age. Simulation of various dosing schemes showed that a dosing schedule of 30 mg/kg/day, irrespective of gestational age, in three doses was optimal, and this scheme was prospectively tested. Mean trough concentrations before the second dose were 8.2 +/- 2.2 mg/L versus a predicted trough of 8.9 +/- 2.5 mg/L. No peak levels higher than 40 mg/L were found. CONCLUSIONS: The use of the proposed schedule leads to adequate vancomycin trough serum concentrations, and there is no need for routine monitoring of peak serum concentrations.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10801244',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 2578,\n",
       "     'offsetInEndSection': 2686,\n",
       "     'text': 'Vancomycin-associated nephrotoxicity is rare in neonates, even with serum peak concentrations >40 microg/mL.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10103340',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 68,\n",
       "     'offsetInEndSection': 760,\n",
       "     'text': 'Infants weighing less than 1,000 gm had significantly larger volumes of drug distribution and consequently longer drug half-lives than larger premature infants, regardless of postconceptual or actual age. These differences alter the vancomycin dosing recommendations in these two groups of premature infants. We recommend initial dosage regimens consisting of a loading dose of vancomycin of 25 mg/kg followed by doses of 15 mg/kg every 12 hours for infants with weights less than 1,000 gm. Infants weighing over 1,000 gm should receive 10 mg/kg every 12 hours, with a loading dose of 12.5 mg/kg. Serum vancomycin concentration should be monitored, however, for final optimization of therapy.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3991250',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Which antiepileptic drug is most strongly associated with spina bifida? ',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/3939491',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10339792',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8075508',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23082254',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22051200',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17075842',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19490036',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11077457',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2707392',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21766433'],\n",
       "   'ideal_answer': ['Phenytoin is not used in pregnancy as it is associated with a severe fetal deformation. From the other anticonvulsants most studies report the higher association between use during pregnancy and spin bifida to occur with Valproate.'],\n",
       "   'exact_answer': ['Valproate'],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:0080016',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D000927'],\n",
       "   'type': 'factoid',\n",
       "   'id': '51588bb2d24251bc05000091',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 258,\n",
       "     'text': 'The teratogenicity of antiepilepsy drug valproic acid (VPA) mostly is found in genetic and somatic levels, causing teratogenesis involving neurotubular defects (NTDs), anencephaly, lumbosacral meningomyelocele, and leg dysfunction due to spina bifida aperta.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22051200',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 721,\n",
       "     'offsetInEndSection': 1062,\n",
       "     'text': 'The summary odds ratio estimate for the association between valproic acid and spina bifida was 11.9 (95% uncertainty interval (UI): 4.0-21.2); for valproic acid and cleft palate 5.8 (95% UI: 3.3-9.5); for carbamazepine and spina bifida 3.6 (95% UI: 1.3-7.8); and for carbamazepine and cleft palate 2.4 (95% UI: 1.1-4.5) in the United States.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21766433',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1030,\n",
       "     'offsetInEndSection': 1307,\n",
       "     'text': 'Increased risk for MCMs could be demonstrated only for exposure to valproate (5.6%, p = 0.005) and AED polytherapy (6.1%, p = 0.02). Neonatal spina bifida was not significantly increased, but was a major indication for elective pregnancy termination among women with epilepsy. ',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19490036',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 316,\n",
       "     'text': 'Valproic acid, a drug commonly used to treat seizures and other psychiatric disorders, causes neural tube defects (NTDs) in exposed fetuses at a rate 20 times higher than in the general population. Failure of the neural tube to close during development results in exencephaly or anencephaly, as well as spina bifida.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17075842',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1034,\n",
       "     'offsetInEndSection': 1095,\n",
       "     'text': 'Associations were found for spina bifida with valproic acid. ',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11077457',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 95,\n",
       "     'text': 'Fetal exposure to valproic acid or carbamazepine increases the risk of neural tube defect (NTD)',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10339792',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 684,\n",
       "     'text': 'Women with epilepsy giving birth during 1973 to 1991 were identified by record linkage of Swedish health registries. Among 3,625 identified infants, 9 had spina bifida. A nested case-control study was performed, comparing drugs used in early pregnancy in the 9 cases and in 18 controls, matched for year of delivery, maternal age, and parity. Six of the spina bifida mothers had used carbamazepine and two had used valproic acid. Among the controls, 5 women used carbamazepine and one valproic acid. There is an apparent excess risk for spina bifida after use of either of these two drugs, but it is not statistically significant when the analysis is restricted to drug-using women. T',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8075508',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 459,\n",
       "     'offsetInEndSection': 623,\n",
       "     'text': 'A significant association was seen between maternal use of valproic acid and spina bifida, and a weaker, non-significant one between carbamazepine and spina bifida.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2707392',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 768,\n",
       "     'offsetInEndSection': 988,\n",
       "     'text': 'A statistically significant association between Spina Bifida and Valproic Acid (odds ratio 22.7; Fisher p value = 0.0364) was observed: no other anticonvulsant tested showed any association with any type of malformation.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3939491',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'What is apelin?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25668242',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25362565',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25486928',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25711427',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25931124',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25965959',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25380625',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26491547',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25491175',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26149233'],\n",
       "   'ideal_answer': ['Apelin, a small regulatory peptide, is the endogenous ligand for the apelin receptor (APJ) receptor.'],\n",
       "   'exact_answer': ['Apelin, a small regulatory peptide, is the endogenous ligand for the apelin receptor (APJ) receptor.'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56e6dfc2edfc094c1f000003',\n",
       "   'snippets': [{'offsetInBeginSection': 115,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'Apelin, a small regulatory peptide, is the endogenous ligand for the apelin receptor (APJ) receptor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25965959',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 247,\n",
       "     'offsetInEndSection': 334,\n",
       "     'text': 'Apelin is an adipocyte-derived hormone that plays important roles in energy metabolism.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25931124',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 110,\n",
       "     'offsetInEndSection': 248,\n",
       "     'text': 'Apelin, as the endogenous ligand of G protein-coupled receptor APJ, participates in a number of physiological and pathological processes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26491547',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 69,\n",
       "     'text': 'Apelin is a novel bioactive peptide as the endogenous ligand for APJ.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26149233',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 36,\n",
       "     'offsetInEndSection': 53,\n",
       "     'text': 'adipokine apelin ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25491175',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 103,\n",
       "     'text': 'Apelin is the endogenous ligand of the APJ receptor, a member of the G protein-coupled receptor family.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25668242',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 857,\n",
       "     'offsetInEndSection': 1019,\n",
       "     'text': 'To date four adipokines (leptin, visfatin, apelin and ghrelin) have been investigated and all affect myometrial contractility, but some more potently than others.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25711427',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 130,\n",
       "     'text': 'The adipocytokine apelin is a peptide, Apelin and its receptor are abundantly expressed in the nervous and cardiovascular systems.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25362565',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 79,\n",
       "     'text': 'The aim of this study was to determine the levels of regulatory peptides apelin',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25380625',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 346,\n",
       "     'offsetInEndSection': 432,\n",
       "     'text': 'Apelin is a vaso-dilatory peptide that also has a modulatory role in pain processing. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25486928',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the function of the protein encoded by the gene PABPC4?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23938467',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22884093',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21300955',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23300856',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22530058',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21940797',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11328870',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20943973',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22896784',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23181716'],\n",
       "   'triples': [{'p': 'http://purl.uniprot.org/core/reviewed',\n",
       "     's': 'http://purl.uniprot.org/uniprot/P21187',\n",
       "     'o': 'true'},\n",
       "    {'p': 'http://purl.uniprot.org/core/shortName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_5032313138370015',\n",
       "     'o': 'Poly(A)-binding protein'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://purl.uniprot.org/intact/EBI-103658',\n",
       "     'o': 'pAbp'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#exactMatch',\n",
       "     's': 'http://purl.uniprot.org/intact/EBI-103658',\n",
       "     'o': 'http://purl.uniprot.org/uniprot/P21187'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_5032313138370015',\n",
       "     'o': 'Polyadenylate-binding protein'},\n",
       "    {'p': 'http://purl.uniprot.org/core/recommendedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/P21187',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_5032313138370015'},\n",
       "    {'p': 'http://purl.uniprot.org/core/shortName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_5032313138370015',\n",
       "     'o': 'PABP'}],\n",
       "   'ideal_answer': ['The main function of PABPC4 is in mRNA stability and translation initiation. PABPC4 may also play a role in chronic inflammation and in the pathogenesis of colorectal cancer.'],\n",
       "   'exact_answer': ['PABC4 is important for mRNA stability and translation initiation, in chronic inflammation and in the pathogenesis of colorectal cancer.'],\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006378',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D026723',\n",
       "    'http://www.uniprot.org/uniprot/PABP_DROME',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002352',\n",
       "    'http://www.uniprot.org/uniprot/PABP4_HUMAN',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D011485',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0043631',\n",
       "    'http://www.biosemantics.org/jochem#4249823',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0005515',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008143',\n",
       "    'http://www.biosemantics.org/jochem#4002540'],\n",
       "   'type': 'factoid',\n",
       "   'id': '531d744c267d7dd053000009',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 121,\n",
       "     'text': 'In testis mRNA stability and translation initiation are extensively under the control of poly(A)-binding proteins (PABP).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11328870',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 974,\n",
       "     'offsetInEndSection': 1081,\n",
       "     'text': 'Taken together, our findings indicate that PABPC4 may play a role in the pathogenesis of colorectal cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22884093',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1170,\n",
       "     'offsetInEndSection': 1447,\n",
       "     'text': 'Consistent with these biochemical activities, plus corresponding histological profiles, the identified RNA processing factors are predicted to collectively drive post-transcriptional expression of an alternative exome that fuels finishing steps of sperm maturation and fitness.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23938467',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1580,\n",
       "     'offsetInEndSection': 1719,\n",
       "     'text': 'This is the first evidence that PABPCs have a targeted role in hTERT regulation leading to a growth advantage in cells expressing HPV16 E6.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20943973',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 890,\n",
       "     'offsetInEndSection': 953,\n",
       "     'text': 'to play a role in chronic inflammation (PPP1R3B, SALL1, PABPC4,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21300955',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 385,\n",
       "     'offsetInEndSection': 495,\n",
       "     'text': 'and with the exception of PABP4, appear to be restricted in their expression to a small number of cell types. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21940797',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1198,\n",
       "     'offsetInEndSection': 1493,\n",
       "     'text': 'PABPC4 (p(Het) = 0.034) for HDL-C. Our findings suggest that some of the previously identified variants associate differently with lipid traits in adolescents compared to adults, either because of developmental changes or because of greater interactions with environmental differences in adults.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22530058',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1047,\n",
       "     'offsetInEndSection': 1115,\n",
       "     'text': 'In addition, PABP4 was associated with the poly(A) tract of pre-mRNA',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23300856',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What kind of enzyme is encoded by the proto-oncogene ABL1?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/19794087',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20841568',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12796783',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19290927',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21435002',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23842646',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9500553',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18528425',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18796434',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24012954'],\n",
       "   'ideal_answer': ['ABL-family proteins comprise one of the best conserved branches of the tyrosine kinases. Each ABL protein contains an SH3-SH2-TK (Src homology 3-Src homology 2-tyrosine kinase) domain cassette, which confers autoregulated kinase activity and is common among nonreceptor tyrosine kinases. This cassette is coupled to an actin-binding and -bundling domain, which makes ABL proteins capable of connecting phosphoregulation with actin-filament reorganization. Two vertebrate paralogs, ABL1 and ABL2, have evolved to perform specialized functions.   ',\n",
       "    'The Abelson (ABL) family of nonreceptor tyrosine kinases, ABL1 and ABL2, transduces diverse extracellular signals to protein networks that control proliferation, survival, migration and invasion. Constitutively activated mutants of the non-receptor tyrosine kinase (TK) ABL1 (Abelson murine leukemia viral (v-abl) homolog (1) protein) play a central role in the pathogenesis myeloproliferative disorders and in some cases of acute leukemia and lymphoma.'],\n",
       "   'exact_answer': ['Nonreceptor tyrosine kinase', 'Protein-Tyrosine Kinase'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D011519',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D011518'],\n",
       "   'type': 'factoid',\n",
       "   'id': '55241f9e2c8b63434a000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 336,\n",
       "     'text': 'The Abelson (ABL) family of nonreceptor tyrosine kinases, ABL1 and ABL2, transduces diverse extracellular signals to protein networks that control proliferation, survival, migration and invasion. ABL1 was first identified as an oncogene required for the development of leukaemias initiated by retroviruses or chromosome translocations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23842646',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 613,\n",
       "     'text': 'Chromosomal rearrangements involving the ABL1 gene, leading to a BCR-ABL1 fusion gene, have been mainly associated with chronic myeloid leukemia and B-cell acute lymphoblastic leukemia (ALL). At present, six other genes have been shown to fuse to ABL1. The kinase domain of ABL1 is retained in all chimeric proteins that are also composed of the N-terminal part of the partner protein that often includes a coiled-coil or a helix-loop-helix domain. These latter domains allow oligomerization of the protein that is required for tyrosine kinase activation, cytoskeletal localization, and neoplastic transformation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21435002',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 543,\n",
       "     'text': 'ABL-family proteins comprise one of the best conserved branches of the tyrosine kinases. Each ABL protein contains an SH3-SH2-TK (Src homology 3-Src homology 2-tyrosine kinase) domain cassette, which confers autoregulated kinase activity and is common among nonreceptor tyrosine kinases. This cassette is coupled to an actin-binding and -bundling domain, which makes ABL proteins capable of connecting phosphoregulation with actin-filament reorganization. Two vertebrate paralogs, ABL1 and ABL2, have evolved to perform specialized functions. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20841568',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 857,\n",
       "     'text': 'Protein tyrosine kinases form a large family of signaling proteins implicated in both normal and malignant cell signaling. The aim of this study was to identify protein tyro-sine kinases that can transform hematopoietic cells to growth factor independent proliferation when constitutively activated by homodimerization. We used a modified retroviral insertion mutagenesis screen with a retroviral vector containing the homodimerization domain of ETV6 followed by an artificial splice donor site. Integration of this retroviral vector within a gene of the host genome would generate a fusion transcript containing the dimerization domain and part of the disrupted gene. Using this strategy with the IL3 dependent Ba/F3 cell line, we identified 8 different protein tyrosine kinases (Abl1, Fgfr1, Hck, Jak2, Lck, Mertk, Mst1r, Tnk1) that transformed the cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19794087',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 341,\n",
       "     'offsetInEndSection': 952,\n",
       "     'text': 'Three families of tyrosine kinases have long been recognized to play critical roles in TCR-dependent signaling. They are the Src, zeta-associated protein of 70 kDa, and Tec families of kinases. More recently, the Abelson (Abl) tyrosine kinases have been shown to be activated by TCR engagement and to be required for maximal TCR signaling. Using T-cell conditional knockout mice deficient for Abl family kinases, Abl (Abl1) and Abl-related gene (Arg) (Abl2), it was recently shown that loss of Abl kinases results in defective T-cell development and a partial block in the transition to the CD4(+)CD8(+) stage. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19290927',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 332,\n",
       "     'text': 'ABL family tyrosine kinases are tightly regulated by autoinhibition and phosphorylation mechanisms. These kinases maintain an inactive conformation through intramolecular interactions involving SH3 and SH2 domains. RIN1, a downstream effector of RAS, binds to the ABL SH3 and SH2 domains and stimulates ABL tyrosine kinase activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18796434',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 333,\n",
       "     'offsetInEndSection': 471,\n",
       "     'text': 'RIN1 binding to the ABL2 kinase resulted in a large decrease in Km and a small increase in Vmax toward an ABL consensus substrate peptide.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18796434',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 859,\n",
       "     'offsetInEndSection': 1036,\n",
       "     'text': 'RIN1 increased the kinase activity of both ABL1 and ABL2, and this occurred in the presence or absence of ABL regulatory domains outside the SH3-SH2-tyrosine kinase domain core.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18796434',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 383,\n",
       "     'text': 'Constitutively activated mutants of the non-receptor tyrosine kinases (TK) ABL1 (Abelson murine leukemia viral (v-abl) homolog (1) protein) and JAK2 (JAnus Kinase 2 or Just Another Kinase 2) play a central role in the pathogenesis of clinically and morphologically distinct chronic myeloproliferative disorders but are also found in some cases of de novo acute leukemia and lymphoma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18528425',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 334,\n",
       "     'offsetInEndSection': 639,\n",
       "     'text': 'Abl1 (previously known as Abl) and the Abl1-related gene product Abl2 (previously known as Arg) define a family of tyrosine kinases that regulate actin structure and presynaptic axon guidance. Here we show that the Abl kinases are critical mediators of postsynaptic assembly downstream of agrin and MuSK. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12796783',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 202,\n",
       "     'text': 'The ABL1 proto-oncogene encodes a cytoplasmic and nuclear protein tyrosine kinase (c-Abl) that has been implicated in processes of cell differentiation, cell division, cell adhesion and stress response.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9500553',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 522,\n",
       "     'offsetInEndSection': 723,\n",
       "     'text': 'Two essential proteins, Proto-oncogene tyrosine-protein kinase ABL1 (c-ABL) and Heat shock 70kDa protein 4 (Apg-2), were confirmed by Western blot and showed consistent changes with proteomic results.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24012954',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 201,\n",
       "     'text': 'The ABL1 proto-oncogene encodes a cytoplasmic and nuclear protein tyrosine kinase (c-Abl) that has been implicated in processes of cell differentiation, cell division, cell adhesion and stress response',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9500553',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which proteins cause cytoplasmic sequestration of NF-kB?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22975329',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16136188',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12399470',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12972430',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20845110',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8692272',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10454561',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8804077',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9566872',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7809091'],\n",
       "   'ideal_answer': ['In unstimulated cells, NF-kB transcription factors are retained in the cytoplasm with the inhibitory activity of I-kBs, Sef, NF-kB1 (p105) and NF-kB2 (p100).'],\n",
       "   'exact_answer': [['I-kBs'], ['Sef'], ['NF-kB1 (p105)'], ['NF-kB2 (p100)']],\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0008588',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0035525',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0032088',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0042994',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0007253',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0051220',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D016328',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0071159',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0042347'],\n",
       "   'type': 'list',\n",
       "   'id': '530db83b38c1322806000002',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 103,\n",
       "     'text': 'Sef is an inhibitor of proinflammatory cytokine signaling, acting by cytoplasmic sequestration of NF-κB',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22975329',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 70,\n",
       "     'offsetInEndSection': 268,\n",
       "     'text': 'According to the classical model, NF-κB is retained in the cytoplasm of resting cells via binding to inhibitory, IκB proteins and translocates into the nucleus upon their ligand-induced degradation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22975329',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 583,\n",
       "     'offsetInEndSection': 649,\n",
       "     'text': 'Like IκBs, Sef sequesters NF-κB in the cytoplasm of resting cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22975329',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 122,\n",
       "     'offsetInEndSection': 233,\n",
       "     'text': 'The activity of NF-κB is tightly controlled through its cytoplasmic sequestration by specific inhibitors, IκBs.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20845110',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 128,\n",
       "     'text': 'The inhibitor of NF-kappaB (IkappaB) family of proteins is believed to regulate NF-kappaB activity by cytoplasmic sequestration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16136188',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 111,\n",
       "     'text': 'IkappaBalpha is an inhibitory molecule that sequesters NF-kappaB dimers in the cytoplasm of unstimulated cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12972430',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'p105.Ikappa Bgamma and prototypical Ikappa Bs use a similar mechanism to bind but a different mechanism to regulate the subcellular localization of NF-kappa B',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12399470',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1355,\n",
       "     'offsetInEndSection': 1501,\n",
       "     'text': 'We show that the death domain of p105 (also of IkappaBgamma) is essential for the cytoplasmic sequestration of NF-kappaB by p105 and IkappaBgamma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12399470',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'In unstimulated cells, NF-kappaB transcription factors are retained in the cytoplasm by inhibitory IkappaB proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10454561',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 230,\n",
       "     'text': 'The ability of the IkappaB alpha protein to sequester dimeric NF-kappaB/Rel proteins in the cytoplasm provides an effective mechanism for regulating the potent transcriptional activation properties of NF-kappaB/Rel family members.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9566872',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 925,\n",
       "     'offsetInEndSection': 1201,\n",
       "     'text': 'The presence of a discrete nuclear import sequence in IkappaB alpha suggests that cytoplasmic sequestration of the NF-kappaB/Rel-IkappaB alpha complex is a consequence of the mutual masking of the NLS within NF-kappaB/Rel proteins and the import sequence within IkappaB alpha.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9566872',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 266,\n",
       "     'text': 'The viral Tax protein, which is encoded by human T-cell leukaemia virus HTLV-I, activates nuclear translocation of the NF-kappa B/Rel transcription factors and relieves cytoplasmic sequestration of RelA and Rel by heterodimerization with NF-kappa B1/p1O5 (refs 1,2).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8692272',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 502,\n",
       "     'offsetInEndSection': 711,\n",
       "     'text': 'The I-kappa B protein, which is necessary for the cytoplasmic sequestration of the NF-kappa B transcription factor complex, was identified specifically in regions of limbic, hypothalamic, and autonomic nuclei.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8804077',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Human T-cell leukemia virus type I Tax-protein-mediated activation of NF-kappa B from p100 (NF-kappa B2)-inhibited cytoplasmic reservoirs',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7809091',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 407,\n",
       "     'offsetInEndSection': 668,\n",
       "     'text': 'The preexisting NF-kappa B proteins are retained in the cytoplasm of cells by association with inhibitory ankyrin-motif-containing I kappa B proteins, primarily I kappa B-alpha but also including the precursor proteins p105 (NF-kappa B1) and p100 (NF-kappa B2).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7809091',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which is the most common CFTR mutation in Caucasians?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/11014930',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8222279',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1376017',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17662673',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8825494',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24517344',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23148214',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22081250',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12357328',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19837664'],\n",
       "   'ideal_answer': ['The commonest CFTR mutation, deltaF508, is found in 74.1% of all CF chromosomes. In the Caucasian CF population, 57.5% are deltaF508 homozygotes but the UK ISC CF population with only 24.7%, has significantly fewer deltaF508 homozygotes patients (95% confidence interval (CI) 0.2-0.4).'],\n",
       "   'exact_answer': ['deltaF508'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D009154',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D003550',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D006801'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56c5feb75795f9a73e000006',\n",
       "   'snippets': [{'offsetInBeginSection': 1382,\n",
       "     'offsetInEndSection': 1497,\n",
       "     'text': 'Exposure to WCS caused a pronounced reduction in CFTR activity in both CFTR (+/+) cells and F508del CFTR (+/-) cell',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24517344',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 894,\n",
       "     'offsetInEndSection': 1096,\n",
       "     'text': 'Moreover, the common heterozygous F508del/5T and F508del/R117H were observed in 17 and 4% of CBAVD cases respectively, and the allele frequency in CBAVD was 17% for F508del, 25% for 5T and 3% for R117H.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22081250',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1268,\n",
       "     'offsetInEndSection': 1785,\n",
       "     'text': 'The most common mutations were p.F508del (DeltaF508) (18.1%), c.2183_2184delAAinsG (2183AA>G) (6.5%), p.S466X (5.8%), p.N1303K (4.3%), c.2789+5G>A (4.3%), p.G542X (3.6%), c.3120+1G>A (3.6%), p.R334W (2.9%) and c.3130delA (2.9%). These 9 types of mutant CFTR genes totaled for 52% of all CFTR genes derived from the 69 Iranian CF patients. Eight mutations, c.406-8T>C, p.A566D, c.2576delA, c.2752-1_2756delGGTGGCinsTTG, p.T1036I, p.W1145R, c.3850-24G>A, c.1342-?_1524+?del, were found for the first time in this study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17662673',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 831,\n",
       "     'offsetInEndSection': 1116,\n",
       "     'text': 'The commonest CFTR mutation, deltaF508, is found in 74.1% of all CF chromosomes. In the Caucasian CF population, 57.5% are deltaF508 homozygotes but the UK ISC CF population with only 24.7%, has significantly fewer deltaF508 homozygotes patients (95% confidence interval (CI) 0.2-0.4).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12357328',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 362,\n",
       "     'offsetInEndSection': 561,\n",
       "     'text': 'Although the major mutation that results in a single amino acid deletion (F508) accounts for 70% of the disease alleles, more than 550 additional mutant alleles of different forms have been detected.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8825494',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 479,\n",
       "     'offsetInEndSection': 678,\n",
       "     'text': 'Besides the major 3-bp deletion, delta F508 that was found on 73% of German CF chromosomes, more than 50 other missense, nonsense, frame-shift, and splice-site mutations have already been identified.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8222279',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 382,\n",
       "     'offsetInEndSection': 660,\n",
       "     'text': 'However, the CFTR mutation delta F508 is the most common reason for the frequently inherited disease among the Caucasian population. Maturation and processing of delta F508-CFTR is defective which leads to expression of only very little but functional CFTR in the cell membrane.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11014930',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1048,\n",
       "     'offsetInEndSection': 1151,\n",
       "     'text': 'The most common disease-causing mutation, DeltaF508, is found in 70% of patients with cystic fibrosis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19837664',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 221,\n",
       "     'text': ' Cystic fibrosis is the most common inherited lethal disease in Caucasians. It is caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR), of which the cftr ΔF508 mutation is the most common.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23148214',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 221,\n",
       "     'text': 'Cystic fibrosis is the most common inherited lethal disease in Caucasians. It is caused by mutations in the cystic fibrosis transmembrane conductance regulator (CFTR), of which the cftr ΔF508 mutation is the most common.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23148214',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 904,\n",
       "     'offsetInEndSection': 1196,\n",
       "     'text': 'Caucasians (eight of nine), Northern Irish (four of five), and Israelis (three of three) also occurred in other Caucasian groups. The preponderance of previously reported mutations in these three groups suggested that a subset of the non-delta F508 mutations occur in common among Caucasians.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1376017',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Are chromomethylases present in animal genomes?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/11487702',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20505370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17660570',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11459824',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12740729',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23021223',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9584105',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18488247',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12121623',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12151602'],\n",
       "   'triples': [{'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393937006',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/O64997',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393937006'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393937007',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393937006'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_423147594832004',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/B1GYH2',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_423147594832004'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_423147594832005',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_423147594832004'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_423147594832004',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_423147594831006',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/B1GYH1',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_423147594831006'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_423147594831007',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_423147594831006'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_423147594831006',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_42314759483300E',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/B1GYH3',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_42314759483300E'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_42314759483300F',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_42314759483300E'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_42314759483300E',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393935004',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/O64995',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393935004'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393935005',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393935004'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393935004',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0676094',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A1361869'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1361869',\n",
       "     'o': 'http://www.w3.org/2008/05/skos-xl#Label'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1361869',\n",
       "     'o': 'C112810'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1361869',\n",
       "     'o': 'MeSH'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1361869',\n",
       "     'o': 'chromomethylase'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0676094',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A1361869'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F343931343400F',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/O49144',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F343931343400F'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F34393134340010',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F343931343400F'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F343931343400F',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F353030333700F',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/O50037',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F353030333700F'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F35303033370010',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F353030333700F'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F353030333700F',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393936005',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/O64996',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393936005'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393936006',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393936005'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393936005',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_513057534338006',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q0WSC8',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_513057534338006'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_513057534338007',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_513057534338006'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_513057534338006',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://linkedlifedata.com/resource/pubmed/chemical/chromomethylase',\n",
       "     'o': 'chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393934005',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/O64994',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393934005'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393934006',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_4F3634393934005'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_4F3634393934005',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_42365A43513700D',\n",
       "     'o': 'Chromomethylase'},\n",
       "    {'p': 'http://purl.uniprot.org/core/submittedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/B6ZCQ7',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_42365A43513700D'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#subject',\n",
       "     's': 'http://linkedlifedata.com/resource/#_42365A43513700E',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_42365A43513700D'},\n",
       "    {'p': 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type',\n",
       "     's': 'http://linkedlifedata.com/resource/#_42365A43513700D',\n",
       "     'o': 'http://purl.uniprot.org/core/Structured_Name'}],\n",
       "   'ideal_answer': ['No. Multiple lines of experimental evidence suggest that chromomethylases (CMTs) have been hitherto identified in plant genomes(Arabidopsis, maize, tomato). CMTs maintain CpNpG (N = A, T, C, or G) methylation and they are unique to the plant kingdom. The lack of CMT homologs in animal genomes could be explained based on the fact that, in contrast to plants, animals maintain primarily CG methylation. Therefore, the presence of CMTs is not required in the animal genomes.'],\n",
       "   'exact_answer': 'no',\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/CMT3_ARATH',\n",
       "    'http://www.uniprot.org/uniprot/CMT2_ARATH',\n",
       "    'http://www.uniprot.org/uniprot/CMT1_ARATH'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5171833c8ed59a060a00000f',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 125,\n",
       "     'text': 'Many plant, animal, and fungal genomes contain cytosine DNA methylation in asymmetric sequence contexts (CpHpH, H = A, T, C).',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12151602',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 852,\n",
       "     'offsetInEndSection': 996,\n",
       "     'text': 'However, at the SUPERMAN locus, asymmetric methylation was only completely abolished in drm1 drm2 chromomethylase 3 (cmt3) triple mutant plants.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12151602',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1302,\n",
       "     'offsetInEndSection': 1486,\n",
       "     'text': 'Although neither the drm1 drm2 double mutants nor the cmt3 single mutants show morphological defects, drm1 drm2 cmt3 triple mutant plants show pleiotropic effects on plant development.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12151602',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 104,\n",
       "     'text': 'Arabidopsis cmt3 chromomethylase mutations block non-CG methylation and silencing of an endogenous gene.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11459824',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 558,\n",
       "     'offsetInEndSection': 705,\n",
       "     'text': 'The lack of CMT homologs in animal genomes could account for the observation that in contrast to plants, animals maintain primarily CG methylation.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11459824',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 108,\n",
       "     'text': 'Dual binding of chromomethylase domains to H3K9me2-containing nucleosomes directs DNA methylation in plants.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23021223',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 123,\n",
       "     'text': 'A role for CHROMOMETHYLASE3 in mediating transposon and euchromatin silencing during egg cell reprogramming in Arabidopsis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20505370',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'During embryogenesis there is a major switch from dependence upon maternally-deposited products to reliance on products of the zygotic genome.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20505370',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1204,\n",
       "     'offsetInEndSection': 1556,\n",
       "     'text': 'Expression analysis of eight putative tomato DNA methyltransferases encoding genes showed that one chromomethylase (CMT) and two rearranged methyltransferases (DRMs) are preferentially expressed in the pericarp during fruit growth and could be involved in the locus-specific increase of methylation observed at this developmental phase in the pericarp.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18488247',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 90,\n",
       "     'text': 'Natural variation for alleles under epigenetic control by the maize chromomethylase zmet2.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17660570',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 233,\n",
       "     'offsetInEndSection': 516,\n",
       "     'text': 'Arabidopsis has two types of methyltransferases with demonstrated maintenance activity: MET1, which maintains CpG methylation and is homologous to mammalian DNMT1, and CHROMOMETHYLASE 3 (CMT3), which maintains CpNpG (N = A, T, C, or G) methylation and is unique to the plant kingdom.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12121623',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 79,\n",
       "     'text': 'Maize chromomethylase Zea methyltransferase2 is required for CpNpG methylation.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11487702',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 298,\n",
       "     'text': 'A cytosine DNA methyltransferase containing a chromodomain, Zea methyltransferase2 (Zmet2), was cloned from maize. The sequence of ZMET2 is similar to that of the Arabidopsis chromomethylases CMT1 and CMT3, with C-terminal motifs characteristic of eukaryotic and prokaryotic DNA methyltransferases.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11487702',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 96,\n",
       "     'offsetInEndSection': 427,\n",
       "     'text': 'We have detected a chromodomain embedded within the catalytic region of a predicted Arabidopsis DNA methyltransferase that is diverged from other eukaryotic enzymes. The 791 residue \"chromomethylase\" (CMT1) is encoded by a floral transcript that is spliced from 20 exons and is present at only approximately 1/10(-7) of total mRNA.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9584105',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'List all reported treatment options for anxiety in autism spectrum disorder.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18437549',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22964266',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22735897',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23118256',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22588377',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21571763',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17171539',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20694508',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22299802',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22934167'],\n",
       "   'ideal_answer': ['The predominant approach is to use versions of cognitive behavioural therapies, such as:\\nMindfulness Based Therapy (MBT)\\nMultimodal Anxiety and Social Skills Intervention (MASSI) program\\nmodified version of the Coping Cat program, (cognitive-behavioral therapy; CBT)\\nFamily cognitive-behavioral therapy has been found to be more effective than Individual cognitive-behavioral therapy \\nConflict management for couples, even when conflict and family distress is low\\n\\nDrugtherapy: \\nSertraline'],\n",
       "   'exact_answer': [['Mindfulness Based Therapy (MBT)'],\n",
       "    ['Multimodal Anxiety and Social Skills Intervention (MASSI) program'],\n",
       "    ['modified version of the Coping Cat program',\n",
       "     '(cognitive-behavioral therapy; CBT'],\n",
       "    ['Family cognitive-behavioral therapy'],\n",
       "    ['Individual cognitive-behavioral therapy'],\n",
       "    ['Conflict management for couples',\n",
       "     'even when conflict and family distress is low'],\n",
       "    ['Sertraline']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002659',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:2030',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:12849',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D001007',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D001008',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D001321',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:0060041'],\n",
       "   'type': 'list',\n",
       "   'id': '515de993298dcd4e51000024',\n",
       "   'snippets': [{'offsetInBeginSection': 1127,\n",
       "     'offsetInEndSection': 1368,\n",
       "     'text': 'A parent report of comorbid diagnosis of attention-deficit/hyperactivity disorder, bipolar disorder, obsessive-compulsive disorder, depression, or anxiety was associated with a high rate of use, with 80% receiving ≥ 1 psychotropic medication',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23118256',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 129,\n",
       "     'offsetInEndSection': 269,\n",
       "     'text': 'Mindfulness-based therapy (MBT) has been found effective in reducing anxiety and depression symptoms, however research in autism is limited.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22964266',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 270,\n",
       "     'offsetInEndSection': 377,\n",
       "     'text': 'Therefore, we examined the effects of a modified MBT protocol (MBT-AS) in high-functioning adults with ASD.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22964266',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 715,\n",
       "     'offsetInEndSection': 823,\n",
       "     'text': 'the present study is the first controlled trial to demonstrate that adults with ASD can benefit from MBT-AS.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22964266',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 167,\n",
       "     'offsetInEndSection': 423,\n",
       "     'text': 'This pilot, randomized controlled trial evaluated the feasibility and preliminary outcomes of the Multimodal Anxiety and Social Skills Intervention (MASSI) program in a sample of 30 adolescents with ASD and anxiety symptoms of moderate or greater severity.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22735897',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 736,\n",
       "     'offsetInEndSection': 833,\n",
       "     'text': 'These findings suggest MASSI is a feasible treatment program and further evaluation is warranted.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22735897',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 188,\n",
       "     'text': 'The purpose of this pilot study was to evaluate whether a modified version of the Coping Cat program could be effective in reducing anxiety in children with autism spectrum disorder (ASD).',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22588377',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 315,\n",
       "     'offsetInEndSection': 373,\n",
       "     'text': 'the Coping Cat program (cognitive-behavioral therapy; CBT)',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22588377',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 572,\n",
       "     'offsetInEndSection': 791,\n",
       "     'text': 'Results provide preliminary evidence that a modified version of the Coping Cat program may be a feasible and effective program for reducing clinically significant levels of anxiety in children with high-functioning ASD.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22588377',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 313,\n",
       "     'text': 'The intent of this article is to explore the efficacy of both the literal and concrete externalization aspects within narrative therapy, and the implementation of interactive metaphors as a combined psychotherapeutic approach for decreasing anxiety with people who present with high-functioning autism.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22299802',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 752,\n",
       "     'offsetInEndSection': 989,\n",
       "     'text': 'This paper reports a case series of children and adolescents with ASD and an anxiety disorder who were treated with a standard cognitive behaviour therapy (CBT) rationale adapted to take account of the neuropsychological features of ASD.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21571763',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 249,\n",
       "     'offsetInEndSection': 408,\n",
       "     'text': 'children with moderate autistic symptomology (per SRS-P) were significantly more likely to improve from family CBT (FCBT) than individual CBT (ICBT; OR\\xa0=\\xa08.67)',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20694508',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 653,\n",
       "     'offsetInEndSection': 850,\n",
       "     'text': 'Though both treatments reduced anxiety, FCBT outperformed ICBT for children with moderate ASD symptoms, a benefit potentially linked to more at-home exposures and greater child involvement in FCBT.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20694508',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 412,\n",
       "     'offsetInEndSection': 577,\n",
       "     'text': 'A structural model-building approach was used to test the extent to which family and peer variables directly or indirectly affected ASD via child anxiety/depression.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18437549',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 950,\n",
       "     'offsetInEndSection': 1297,\n",
       "     'text': 'The key findings were that anxiety/depression and ASD symptomatology were significantly related, and family conflict was more predictive of ASD symptomatology than positive family/peer influences. The results point to the utility of expanding interventions to include conflict management for couples, even when conflict and family distress is low.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18437549',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 173,\n",
       "     'text': 'A family-based, cognitive behavioural treatment for anxiety in 47 children with comorbid anxiety disorders and High Functioning Autism Spectrum Disorder (HFA) was evaluated.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17171539',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 402,\n",
       "     'offsetInEndSection': 674,\n",
       "     'text': 'Following treatment, 71.4% of the treated participants no longer fulfilled diagnostic criteria for an anxiety disorder. Comparisons between the two conditions indicated significant reductions in anxiety symptoms as measured by self-report, parent report and teacher report',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17171539',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': \"List adenosine A2A receptor antagonists that are used for Parkinson's disease treatment.\",\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23748382',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20182024',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24687255',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15298067',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22585137',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25175961',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15974638',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23483627',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16004599',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23642267'],\n",
       "   'ideal_answer': [\"Istradefylline and preladenant are adenosine A2A receptor antagonists that are used for Parkinson's disease treatment.\"],\n",
       "   'exact_answer': [['istradefylline'], ['preladenant']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D058917',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D010300',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:14330'],\n",
       "   'type': 'list',\n",
       "   'id': '54fc9b236ad7dcbc12000005',\n",
       "   'snippets': [{'offsetInBeginSection': 177,\n",
       "     'offsetInEndSection': 323,\n",
       "     'text': 'Adenosine A2A antagonists, such as istradefylline, improve motor function in PD, but their effect on cognitive impairment has not been determined.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23748382',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 862,\n",
       "     'offsetInEndSection': 995,\n",
       "     'text': 'Both istradefylline and preladenant have demonstrated moderate efficacy in reducing off time in PD patients with motor fluctuations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22585137',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1067,\n",
       "     'offsetInEndSection': 1310,\n",
       "     'text': \" The available data also suggest that caffeine can improve the motor deficits of PD and that adenosine A2A receptor antagonists such as istradefylline reduces OFF time and dyskinesia associated with standard 'dopamine replacement' treatments. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20182024',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 955,\n",
       "     'offsetInEndSection': 1228,\n",
       "     'text': \"Istradefylline (KW-6002) is the first of several adenosine A2A receptor antagonists in development for PD to advance to phase III clinical trials. Initial studies indicate that in patients with motor fluctuations on levodopa, addition of istradefylline reduces 'off' time. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15974638',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 858,\n",
       "     'offsetInEndSection': 1240,\n",
       "     'text': 'These include alpha2 adrenergic receptor antagonists (eg, fipamezole), adenosine A2A receptor antagonists (eg, istradefylline), AMPA receptor antagonists (eg, talampanel), neuronal synchronization modulators (eg, levetiracetam) and agents that interact with serotonergic systems such as 5-hydroxytryptamine (5-HT)1A agonists (eg, sarizotan) and 5-HT2A antagonists (eg, quetiapine). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15298067',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 266,\n",
       "     'text': 'BACKGROUND: We evaluated the efficacy and safety of istradefylline, a selective adenosine A2A receptor antagonist administered as adjunctive treatment to levodopa for 12 weeks in a double-blind manner in Parkinsons disease patients with motor complications in Japan.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23483627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 99,\n",
       "     'text': 'Istradefylline, a novel adenosine A2A receptor antagonist, for the treatment of Parkinsons disease.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16004599',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': \"Istradefylline, a novel adenosine A2A receptor antagonist, for the treatment of Parkinson's disease.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16004599',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 431,\n",
       "     'offsetInEndSection': 606,\n",
       "     'text': \"In this article, the author discusses the potential role of A2A adenosine receptor antagonists in the treatment of Parkinson's disease through the evaluation of istradefylline\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23642267',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': \"Suitability of the adenosine antagonist istradefylline for the treatment of Parkinson's disease: pharmacokinetic and clinical considerations.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23642267',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'What pharmacological and non-pharmacological interventions can be considered as prophylactic therapies in Cluster Headache patients?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/11252143',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16628535',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15651299',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21284609',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17901920',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11026146',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2520442',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12390644',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20352587',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16041199'],\n",
       "   'ideal_answer': ['Verapamil, a calcium channel blocker, is considered the mainstay of prophylactic therapy of Cluster Headache patients. Lithium carbonate, topiramate, valproic acid, gabapentin, baclofen, methysergide, melatonin, ketoprofen and indomethacin can also be tried for prophylactic therapy of Cluster Headaches patients. Non-pharmacological prophylactic measures, such as peripheral (mostly occipital nerve) and central (hypothalamic) neurostimulation, ablative surgery, and botulinum toxin type-A (BTX-A) injection, can be also considered.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D003027',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D055502'],\n",
       "   'type': 'summary',\n",
       "   'id': '5148a2f6d24251bc05000035',\n",
       "   'snippets': [{'offsetInBeginSection': 563,\n",
       "     'offsetInEndSection': 767,\n",
       "     'text': 'The calcium channel blocker verapamil is the drug of choice for CH prevention. Other drugs that may be used for this purpose include lithium carbonate, topiramate, valproic acid, gabapentin, and baclofen.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21284609',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 967,\n",
       "     'offsetInEndSection': 1149,\n",
       "     'text': 'Recently, the therapeutic options for refractory CH patients have expanded with the emergence of both peripheral (mostly occipital nerve) and central (hypothalamic) neurostimulation.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21284609',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1150,\n",
       "     'offsetInEndSection': 1244,\n",
       "     'text': 'With the emergence of these novel treatments, the role of ablative surgery in CH has declined.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21284609',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1102,\n",
       "     'offsetInEndSection': 1338,\n",
       "     'text': 'The mainstay of prophylactic therapy is verapamil. Yet, other medications, including lithium, divalproex sodium, topiramate, methysergide, gabapentin, and even indomethacin, may be useful when the headache fails to respond to verapamil.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20352587',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1339,\n",
       "     'offsetInEndSection': 1471,\n",
       "     'text': 'For medically refractory patients, surgical interventions, occipital nerve stimulation, and deep brain stimulation remain an option.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20352587',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 192,\n",
       "     'text': 'The objective of this open single-centre study was to evaluate the efficacy and tolerability of botulinum toxin type-A (BTX-A) as add-on in the prophylactic treatment of cluster headache (CH).',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17901920',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 829,\n",
       "     'offsetInEndSection': 973,\n",
       "     'text': 'These findings provide evidence that BTX-A may be beneficial as an add-on prophylactic therapy for a limited number of patients with chronic CH.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17901920',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1253,\n",
       "     'offsetInEndSection': 1397,\n",
       "     'text': 'There are a variety of different medications for abortive and prophylactic therapy, accompanied by a variable amount of evidence-based medicine.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16041199',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1503,\n",
       "     'offsetInEndSection': 1639,\n",
       "     'text': 'Most procedures are directed against the sensory trigeminal nerve and associated ganglia, eg, anesthetizing the sphenopalatine ganglion.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16041199',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1060,\n",
       "     'offsetInEndSection': 1172,\n",
       "     'text': 'The mainstay of prophylactic therapy is verapamil. Lithium, divalproex sodium, or topiramate may also be useful.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16628535',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1154,\n",
       "     'offsetInEndSection': 1299,\n",
       "     'text': 'Based on our clinical experience, we recommended the combination of nasal sumatriptan for acute attacks and verapamil 240 mg/day for prophylaxis.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15651299',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 3204,\n",
       "     'offsetInEndSection': 3308,\n",
       "     'text': 'Topiramate also appears to be well tolerated and useful in the adjunctive treatment of cluster headache.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12390644',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1071,\n",
       "     'offsetInEndSection': 1276,\n",
       "     'text': 'The cornerstone of maintenance prophylaxis is verapamil, yet methysergide, lithium, and divalproex sodium may also be employed. In some patients, melatonin or topiramate may be useful adjunctive therapies.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11252143',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 600,\n",
       "     'offsetInEndSection': 685,\n",
       "     'text': 'Prophylactic therapy in most cases consisted of verapamil, also with a good response.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11026146',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 913,\n",
       "     'offsetInEndSection': 1079,\n",
       "     'text': 'Patients with chronic cluster headache may achieve good results from long-term treatment with other therapies, including lithium carbonate, verapamil, and ketoprofen.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2520442',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'List the components of mTOR Complex 2 (mTORC2).',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23455608',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22820188',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22595285',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22773877',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23049074',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22532249',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24077282',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23673367',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22678916',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24244675'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A10789460',\n",
       "     'o': 'C39159'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C1515673',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A7667725'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A7667725',\n",
       "     'o': 'mTOR Signaling Pathway'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C1515673',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A7667725'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C1515673',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A10789460'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A7667725',\n",
       "     'o': 'C39159'}],\n",
       "   'ideal_answer': ['Mammalian target of rapamycin complex 2 (mTORC2) is a kinase complex comprised of mTOR, Rictor, mSin1, mLST8/GβL and PRR5.'],\n",
       "   'exact_answer': [['mTOR'], ['Rictor'], ['mSin1'], ['mLST8/GβL'], ['PRR5']],\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/MTOR_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MTOR_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MTOR_RAT',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0070438'],\n",
       "   'type': 'list',\n",
       "   'id': '5505c9008e1671127b000002',\n",
       "   'snippets': [{'offsetInBeginSection': 104,\n",
       "     'offsetInEndSection': 192,\n",
       "     'text': 'mTOR is the major component of two protein complexes: mTOR complex 1 (mTORC1) and mTORC2',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24077282',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 222,\n",
       "     'offsetInEndSection': 259,\n",
       "     'text': ' Rictor, an mTORC2-specific component',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24244675',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 888,\n",
       "     'offsetInEndSection': 960,\n",
       "     'text': 'f Rictor (rapamycin-insensitive companion of mTOR), an mTORC2 component,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23673367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 248,\n",
       "     'offsetInEndSection': 398,\n",
       "     'text': ' The mammalian target of rapamycin (mTOR) complex 2 (mTORC2), which contains the regulatory protein Rictor (rapamycin-insensitive companion of mTOR), ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23455608',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 656,\n",
       "     'offsetInEndSection': 690,\n",
       "     'text': ' Rictor, a key component of mTORC2',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23049074',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 793,\n",
       "     'offsetInEndSection': 841,\n",
       "     'text': 'SIN1, a key component of mTOR complex 2 (mTORC2)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22820188',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 179,\n",
       "     'offsetInEndSection': 218,\n",
       "     'text': ' the essential mTORC2 component rictor ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22773877',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 144,\n",
       "     'offsetInEndSection': 203,\n",
       "     'text': 'Sin1 is an essential component of mTOR complex 2 (mTORC2). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22678916',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 782,\n",
       "     'offsetInEndSection': 844,\n",
       "     'text': ' SIN1 or Rictor, two key components of mTOR complex 2 (mTORC2)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22595285',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 174,\n",
       "     'offsetInEndSection': 297,\n",
       "     'text': ' Mammalian target of rapamycin complex 2 (mTORC2) is a kinase complex comprised of mTOR, Rictor, mSin1, mLST8/GβL and PRR5 ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22532249',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is sumoylation implicated in myogenesis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/16973431',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24344126',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23247248',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25002400',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16631162',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17202138',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16478538',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23637228',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16966324',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23754700'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A18469770',\n",
       "     'o': 'D058207'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A7814146',\n",
       "     'o': 'D024510'}],\n",
       "   'ideal_answer': ['Yes, sumoylation is implicated in myogenesis.',\n",
       "    'Yes,  protein sumoylation present in myoblasts is regulated in myogenesis.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D024510',\n",
       "    'http://amigo.geneontology.org/amigo/term/GO:0016925',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D058207'],\n",
       "   'type': 'yesno',\n",
       "   'id': '56cca4da5795f9a73e000034',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 170,\n",
       "     'text': 'Sentrin/small ubiquitin-like modifier (SUMO)-specific protease 2 (SENP2) has broad de-SUMOylation activities in vitro, which is essential for embryonic heart development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24344126',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 413,\n",
       "     'offsetInEndSection': 622,\n",
       "     'text': 'Silencing SENP2 can reduce myostatin expression and, therefore, promote myogenesis of skeletal muscle. These results reveal the important role of SENP2 in the regulation of myostatin expression and myogenesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24344126',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 290,\n",
       "     'offsetInEndSection': 426,\n",
       "     'text': 'Overexpression of c-Ski/SnoN also induces skeletal muscle differentiation, but how c-Ski/SnoN function in myogenesis is largely unknown.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17202138',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1028,\n",
       "     'offsetInEndSection': 1293,\n",
       "     'text': 'Notably, loss of sumoylation in the Lys-50 site (via a Lys-to-Arg point mutation) potently activates muscle-specific gene expression and enhances myotube formation. Our study suggests a novel role for SUMO modification in the regulation of myogenic differentiation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17202138',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1104,\n",
       "     'offsetInEndSection': 1415,\n",
       "     'text': 'Although this modification has little effect on SnoN repression of the plasminogen activator inhibitor-1 promoter and only modestly potentiates SnoN repression of the p21 promoter, SnoN sumoylation robustly augments the ability of SnoN to suppress transcription of the myogenesis master regulatory gene myogenin',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16966324',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1724,\n",
       "     'offsetInEndSection': 1861,\n",
       "     'text': 'Our study also points to a physiological role for SnoN sumoylation in the control of myogenin expression in differentiating muscle cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16966324',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 592,\n",
       "     'offsetInEndSection': 714,\n",
       "     'text': 'Here, we biochemically characterize SnoN sumoylation in detail and report the physiological function of the modification. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17202138',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 117,\n",
       "     'text': 'An essential role of small ubiquitin-like modifier (SUMO)-specific Protease 2 in myostatin expression and myogenesis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24344126',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 516,\n",
       "     'offsetInEndSection': 622,\n",
       "     'text': 'These results reveal the important role of SENP2 in the regulation of myostatin expression and myogenesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24344126',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'The E3 SUMO ligase Nse2 regulates sumoylation and nuclear-to-cytoplasmic translocation of skNAC-Smyd1 in myogenesis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25002400',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 157,\n",
       "     'text': 'Sumoylation of the basic helix-loop-helix transcription factor sharp-1 regulates recruitment of the histone methyltransferase G9a and function in myogenesis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23637228',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 516,\n",
       "     'offsetInEndSection': 637,\n",
       "     'text': ' We show that the overall load of sumoylated proteins present in myoblasts diminishes progressively throughout myogenesis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16631162',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 869,\n",
       "     'offsetInEndSection': 1059,\n",
       "     'text': 'These novel results suggest that protein sumoylation plays a pivotal role in myoblast differentiation and is required to regulate the activity of key targets downstream of MyoD and myogenin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16631162',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 167,\n",
       "     'offsetInEndSection': 435,\n",
       "     'text': 'a composite sequence motif has recently been identified that couples phosphorylation, sumoylation, and perhaps also deacetylation to control transcriptional repression in stress response, mitogen and nuclear hormone signaling, myogenesis, and neuronal differentiation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16973431',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 575,\n",
       "     'offsetInEndSection': 904,\n",
       "     'text': 'Mutation of these SUMO acceptor sites in Sharp-1 does not impact its subcellular localization but attenuates its ability to act as a transcriptional repressor and inhibit myogenic differentiation. Consistently, co-expression of the SUMO protease SENP1 with wild type Sharp-1 abrogates Sharp-1-dependent inhibition of myogenesis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23637228',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 89,\n",
       "     'text': 'Transforming growth factor-beta-independent regulation of myogenesis by SnoN sumoylation.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17202138',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 190,\n",
       "     'text': 'Ubiquitin Specific Protease 25 (USP25), a member of the deubiquitinase family, is involved in several disease-related signal pathways including myogenesis, immunity and protein degradation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23754700',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 918,\n",
       "     'offsetInEndSection': 1317,\n",
       "     'text': ' In addition, we show that the skNAC interaction partner Smyd1 contains a putative sumoylation motif and is sumoylated in muscle cells, with depletion of Mms21/Nse2 leading to reduced concentrations of sumoylated Smyd1. Taken together, our data suggest that the function, specifically the balance between the nuclear and cytosolic roles, of the skNAC-Smyd1 complex might be regulated by sumoylation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25002400',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Are there randomised controlled trials on sevoflurane?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/16867087',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21733178',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20683334',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12693995',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23452265',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22577917',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21675061',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15310345',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22103571',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11966554'],\n",
       "   'ideal_answer': ['Yes. There are < 10 studies reported, answering questions like : how to improve speed of recovery, relationship to dreaming and anesthetic experience, effect on cardiac troponin release, effect on myocardial injury, postoperative delirium,  haemodynamics & emergence and recovery characteristics of total intravenous anaesthesia,  costs of postoperative nausea and vomiting, pediatric conscious sedation for dental procedures'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4252326'],\n",
       "   'type': 'yesno',\n",
       "   'id': '515d9e5c298dcd4e5100000e',\n",
       "   'snippets': [{'offsetInBeginSection': 459,\n",
       "     'offsetInEndSection': 641,\n",
       "     'text': 'After Ethics Review Board approval, 44 ASA I-III patients undergoing elective gynaecological surgery were randomised after surgery to either hypercapnic hyperpnoea or control groups.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23452265',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1471,\n",
       "     'offsetInEndSection': 1617,\n",
       "     'text': 'Hypercapnic hyperpnoea in spontaneously breathing patients halves the time of recovery from sevoflurane-induced anaesthesia in the operating room.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23452265',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 329,\n",
       "     'offsetInEndSection': 569,\n",
       "     'text': 'A total of 200 women undergoing first trimester abortion (American Society of Anesthesiologists physical status I) participated in the study. Patients were randomly assigned to receive either sevoflurane or propofol for short-term sedation.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22577917',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 650,\n",
       "     'offsetInEndSection': 841,\n",
       "     'text': 'The results showed the incidence of dreaming was significantly different between anaesthesia groups with 60% (60/100) of the sevoflurane group and 33% (33/100) of the propofol group (P=0.000)',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22577917',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1269,\n",
       "     'offsetInEndSection': 1569,\n",
       "     'text': 'Anaesthesia administered had no effect on patient satisfaction. The results suggest that the incidence of dreaming was not affected by recovery time. Patient satisfaction was not influenced by choice of sedative and/or by the occurrence of dreaming during sevoflurane or propofol short-term sedation.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22577917',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'Prior reports suggest that dreaming during anaesthesia is dependent on recovery time. Dreaming during sedation may impact patient satisfaction',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22577917',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 105,\n",
       "     'text': 'Sevoflurane vs. propofol in patients with coronary disease undergoing mitral surgery: a randomised study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22103571',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 197,\n",
       "     'offsetInEndSection': 373,\n",
       "     'text': 'We therefore performed a randomised controlled trial (sevoflurane vs. propofol) to compare cardiac troponin release in patients with coronary disease undergoing mitral surgery.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22103571',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 146,\n",
       "     'text': 'Myocardial injury in remifentanil-based anaesthesia for off-pump coronary artery bypass surgery: an equipotent dose of sevoflurane versus propofol',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21675061',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 340,\n",
       "     'text': 'This randomised controlled trial compared the effect of equipotent anaesthetic doses of sevoflurane (S group) versus propofol (P group), during remifentanil-based anaesthesia for off-pump coronary artery bypass surgery, on myocardial injury. Either sevoflurane or propofol was titrated to maintain bispectral index values between 40 and 50.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21675061',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 365,\n",
       "     'offsetInEndSection': 545,\n",
       "     'text': 'This randomised, multicentre, parallel-group trial included 98 adult patients. Patients received intravenous propofol for induction followed by sevoflurane maintenance anaesthesia.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20683334',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 654,\n",
       "     'offsetInEndSection': 747,\n",
       "     'text': 'Patients were randomly allocated to receive sugammadex 2.0 mg kg(-1) or neostigmine 50 microg',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20683334',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 269,\n",
       "     'text': 'We compared the haemodynamics, emergence and recovery characteristics of total intravenous anaesthesia using propofol/remifentanil with sevoflurane/remifentanil anaesthesia, under bispectral index guidance, in 103 patients undergoing surgical procedures lasting > 3.5 h',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16867087',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 176,\n",
       "     'text': 'A randomised controlled trial of paediatric conscious sedation for dental treatment using intravenous midazolam combined with inhaled nitrous oxide or nitrous oxide/sevoflurane',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15310345',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1403,\n",
       "     'offsetInEndSection': 1643,\n",
       "     'text': 'Intravenous midazolam, especially in combination with inhaled nitrous oxide or sevoflurane and nitrous oxide, are effective techniques, with the combination of midazolam and sevoflurane the one most likely to result in successful treatment.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15310345',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 114,\n",
       "     'offsetInEndSection': 244,\n",
       "     'text': 'We randomly assigned 1063 adult and 322 paediatric elective patients to one of four (adult) or two (paediatric) anaesthesia groups',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12693995',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 842,\n",
       "     'offsetInEndSection': 1174,\n",
       "     'text': 'In both studies, there was no difference in postdischarge outcomes at Day 7. Sevoflurane/sevoflurane was more costly with higher PONV rates in both studies. In adults, the cost per extra episode of PONV avoided was pound 296 (propofol/propofol vs. propofol/ sevoflurane) and pound 333 (propofol/sevoflurane vs. propofol/isoflurane).',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12693995',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 178,\n",
       "     'text': 'Comparison of sevoflurane and nitrous oxide mixture with nitrous oxide alone for inhalation conscious sedation in children having dental treatment: a randomised controlled trial.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11966554',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'We studied 411 children aged 3-10 years who were referred for dental treatment. They were randomly allocated to have inhalation conscious sedation with either sevoflurane/nitrous oxide mixture or nitrous oxide alone',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11966554',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Which are the characteristics of the Meier-Gorlin syndrome?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23144622',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21358631',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14564153',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21358632',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23023959',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7981855',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11477602',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11807867',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10213048',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25059018'],\n",
       "   'ideal_answer': ['The Meier-Gorlin syndrome is a rare autosomal recessive disorder, characterized by the association of bilateral microtia, aplasia or hypoplasia of the patellae, and severe pre- and postnatal growth retardation.',\n",
       "    'The Meier-Gorlin syndrome (MGS) is a rare autosomal recessive disorder, characterized by bilateral microtia, aplasia or hypoplasia of the patellae, and severe intrauterine and post-natal growth retardation  ',\n",
       "    'The Meier-Gorlin syndrome (MGS) is a rare autosomal recessive disorder, characterized by bilateral microtia, aplasia or hypoplasia of the patellae, and severe intrauterine and post-natal growth retardation'],\n",
       "   'type': 'summary',\n",
       "   'id': '5537766dbc4f83e82800000d',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 146,\n",
       "     'text': 'Meier-Gorlin syndrome (MGS) is a rare autosomal recessive disorder characterized by primordial dwarfism, microtia, and patellar aplasia/hypoplasia',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23023959',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 205,\n",
       "     'text': 'The Meier-Gorlin syndrome (MGS) is a rare autosomal recessive disorder, characterized by bilateral microtia, aplasia or hypoplasia of the patellae, and severe intrauterine and post-natal growth retardation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14564153',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 259,\n",
       "     'text': 'The Meier-Gorlin syndrome or ear, patella, short stature syndrome (MIM 224690) is a rare autosomal recessive disorder, characterized by the association of bilateral microtia, aplasia/hypoplasia of the patellae, and severe pre- and postnatal growth retardation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11477602',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 194,\n",
       "     'text': 'Meier-Gorlin syndrome (ear, patella and short-stature syndrome) is an autosomal recessive primordial dwarfism syndrome characterized by absent or hypoplastic patellae and markedly small ears?⁻?.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21358632',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 74,\n",
       "     'text': 'The Meier-Gorlin syndrome, or ear-patella-short stature syndrome, in sibs.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10213048',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 972,\n",
       "     'offsetInEndSection': 1211,\n",
       "     'text': 'Seckel Syndrome is characterised by microcephaly and growth delay, features also displayed by several related disorders including Majewski (microcephalic) osteodysplastic primordial dwarfism (MOPD) type II and Meier-Gorlin Syndrome (MGS).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23144622',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 260,\n",
       "     'text': 'The Meier-Gorlin syndrome or ear, patella, short stature syndrome (MIM 224690) is a rare autosomal recessive disorder, characterized by the association of bilateral microtia, aplasia/hypoplasia of the patellae, and severe pre- and postnatal growth retardation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11477602',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 96,\n",
       "     'text': 'We report on an Italian boy with the Meier-Gorlin syndrome (ear-patella-short stature syndrome).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11807867',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 143,\n",
       "     'text': 'Meier-Gorlin syndrome (ear-patella-short stature syndrome) in an Italian patient: clinical evaluation and analysis of possible candidate genes.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11807867',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 227,\n",
       "     'text': 'The Meier-Gorlin syndrome (MGS) or ear, patella, short stature syndrome (MIM #224690) is a rare disorder with bilateral microtia, aplasia or hypoplasia of the patellae and severe intra-uterine and post-natal growth retardation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25059018',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 88,\n",
       "     'text': 'Further delineation of the ear, patella, short stature syndrome (Meier-Gorlin syndrome).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7981855',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 256,\n",
       "     'offsetInEndSection': 507,\n",
       "     'text': 'This combination of anomalies has many similarities to the six cases previously described with the Ear, Patellae, Short stature syndrome (Meier-Gorlin syndrome), which is distinguished by the triad of microtia, absent patellae and growth retardation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7981855',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 194,\n",
       "     'text': 'Meier-Gorlin syndrome (ear, patella and short-stature syndrome) is an autosomal recessive primordial dwarfism syndrome characterized by absent or hypoplastic patellae and markedly small ears�{�.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21358632',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 175,\n",
       "     'text': 'Meier-Gorlin syndrome is a rare autosomal recessive genetic condition whose primary clinical hallmarks include small stature, small external ears and small or absent patellae.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21358631',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1688,\n",
       "     'offsetInEndSection': 1933,\n",
       "     'text': 'While aberrant bone development was mild in the original ATR-SS patient, some of the patients described here display skeletal abnormalities including, in one patient, small patellae, a feature characteristically observed in Meier-Gorlin Syndrome',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23144622',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 979,\n",
       "     'offsetInEndSection': 1216,\n",
       "     'text': 'Seckel Syndrome is characterised by microcephaly and growth delay, features also displayed by several related disorders including Majewski (microcephalic) osteodysplastic primordial dwarfism (MOPD) type II and Meier-Gorlin Syndrome (MGS)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23144622',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List available methods for transmembrane protein topology prediction.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/19470175',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11590105',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18989393',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19812766',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24225132',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15111065',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17237066',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16597327',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19785723',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16568545'],\n",
       "   'ideal_answer': ['HMMpTM, MetaTM, Philius, HMM_RA, HMMTOP, MEMSAT3, HMM-TM, TMHMM, Phobius and SignalP.'],\n",
       "   'exact_answer': [['HMMpTM'],\n",
       "    ['MetaTM'],\n",
       "    ['Philius'],\n",
       "    ['HMM_RA'],\n",
       "    ['HMMTOP'],\n",
       "    ['MEMSAT3'],\n",
       "    ['HMM-TM'],\n",
       "    ['TMHMM'],\n",
       "    ['Phobius'],\n",
       "    ['SignalP']],\n",
       "   'type': 'list',\n",
       "   'id': '554143ad182542114d000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'HMMpTM: improving transmembrane protein topology prediction using phosphorylation and glycosylation site prediction',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24225132',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 716,\n",
       "     'offsetInEndSection': 1246,\n",
       "     'text': 'We report the development of a Hidden Markov Model based method, capable of predicting the topology of transmembrane proteins and the existence of kinase specific phosphorylation and N/O-linked glycosylation sites along the protein sequence. Our method integrates a novel feature in transmembrane protein topology prediction, which results in improved performance for topology prediction and reliable prediction of phosphorylation and glycosylation sites. The method is freely available at http://bioinformatics.biol.uoa.gr/HMMpTM',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24225132',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 74,\n",
       "     'text': 'MetaTM - a consensus method for transmembrane protein topology prediction.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19785723',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 647,\n",
       "     'offsetInEndSection': 1364,\n",
       "     'text': 'A novel TM consensus method, named MetaTM, is proposed in this work. MetaTM is based on support vector machine models and combines the results of six TM topology predictors and two signal peptide predictors. On a large data set comprising 1460 sequences of TM proteins with known topologies and 2362 globular protein sequences it correctly predicts 86.7% of all topologies. CONCLUSION: Combining several TM predictors in a consensus prediction framework improves overall accuracy compared to any of the individual methods. Our proposed SVM-based system also has higher accuracy than a previous consensus predictor. MetaTM is made available both as downloadable source code and as DAS server at http://MetaTM.sbc.su.se',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19785723',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 682,\n",
       "     'offsetInEndSection': 1419,\n",
       "     'text': 'We present a support vector machine-based (SVM) TM protein topology predictor that integrates both signal peptide and re-entrant helix prediction, benchmarked with full cross-validation on a novel data set of 131 sequences with known crystal structures. The method achieves topology prediction accuracy of 89%, while signal peptides and re-entrant helices are predicted with 93% and 44% accuracy respectively. An additional SVM trained to discriminate between globular and TM proteins detected zero false positives, with a low false negative rate of 0.4%. We present the results of applying these tools to a number of complete genomes. Source code, data sets and a web server are freely available from http://bioinf.cs.ucl.ac.uk/psipred/',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19470175',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 670,\n",
       "     'text': 'Hidden Markov models (HMMs) have been successfully applied to the tasks of transmembrane protein topology prediction and signal peptide prediction. In this paper we expand upon this work by making use of the more powerful class of dynamic Bayesian networks (DBNs). Our model, Philius, is inspired by a previously published HMM, Phobius, and combines a signal peptide submodel with a transmembrane submodel. We introduce a two-stage DBN decoder that combines the power of posterior decoding with the grammar constraints of Viterbi-style decoding. Philius also provides protein type, segment, and topology confidence metrics to aid in the interpretation of the predictions',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18989393',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 672,\n",
       "     'offsetInEndSection': 859,\n",
       "     'text': 'We report a relative improvement of 13% over Phobius in full-topology prediction accuracy on transmembrane proteins, and a sensitivity and specificity of 0.96 in detecting signal peptides',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18989393',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 86,\n",
       "     'text': 'HMM_RA: an improved method for alpha-helical transmembrane protein topology prediction',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19812766',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 241,\n",
       "     'offsetInEndSection': 614,\n",
       "     'text': 'This paper presents a Hidden Markov Model (referred to as HMM_RA) that can predict the topology of alpha-helical transmembrane proteins with improved performance. HMM_RA adopts the same structure as the HMMTOP method, which has five modules: inside loop, inside helix tail, membrane helix, outside helix tail and outside loop. Each module consists of one or multiple states',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19812766',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 633,\n",
       "     'offsetInEndSection': 833,\n",
       "     'text': 'A new method (MEMSAT3) for predicting transmembrane protein topology from sequence profiles is described and benchmarked with full cross-validation on a standard data set of 184 transmembrane proteins',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17237066',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1698,\n",
       "     'offsetInEndSection': 1867,\n",
       "     'text': 'The algorithms presented here, are easily implemented in any kind of a Hidden Markov Model, whereas the prediction method (HMM-TM) is freely available for academic users',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16597327',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 434,\n",
       "     'offsetInEndSection': 1056,\n",
       "     'text': \"TMHMM, the current state-of-the-art method, has less than 52% accuracy in topology prediction on one set of transmembrane proteins of known topology. Based on the observation that there are functional domains that occur preferentially internal or external to the membrane, we have extended the model of TMHMM to incorporate functional domains, using a probabilistic approach originally developed for computational gene finding. Our extension is better than TMHMM in predicting the topology of transmembrane proteins. As prediction of functional domain improves, our system's prediction accuracy will likely improve as well\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16568545',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 637,\n",
       "     'offsetInEndSection': 1594,\n",
       "     'text': 'Here, we present Phobius, a combined transmembrane protein topology and signal peptide predictor. The predictor is based on a hidden Markov model (HMM) that models the different sequence regions of a signal peptide and the different regions of a transmembrane protein in a series of interconnected states. Training was done on a newly assembled and curated dataset. Compared to TMHMM and SignalP, errors coming from cross-prediction between transmembrane segments and signal peptides were reduced substantially by Phobius. False classifications of signal peptides were reduced from 26.1% to 3.9% and false classifications of transmembrane helices were reduced from 19.0% to 7.7%. Phobius was applied to the proteomes of Homo sapiens and Escherichia coli. Here we also noted a drastic reduction of false classifications compared to TMHMM/SignalP, suggesting that Phobius is well suited for whole-genome annotation of signal peptides and transmembrane regions',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15111065',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'HMMpTM: improving transmembrane protein topology prediction using phosphorylation and glycosylation site prediction.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24225132',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 52,\n",
       "     'text': 'The HMMTOP transmembrane topology prediction server.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11590105',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Which pituitary adenoma is common cause of infertility is women?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/9152623',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23090264',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2803131',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12477530',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2738821',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10649814',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10649815',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2520800',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/6788711',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/6868876'],\n",
       "   'ideal_answer': ['Prolactinoma is a pituitary adenoma that is strongly associated with infertility in women mainly due to increased prolactin secretion causing hyperprolactinemia. Other pituitary lesions can also be associated with infertility.'],\n",
       "   'exact_answer': ['prolactinoma'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D014930',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015175',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007247',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007246',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:5394',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:5395',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:5223',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:3829',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D010911'],\n",
       "   'type': 'factoid',\n",
       "   'id': '514a51c2d24251bc0500005c',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 151,\n",
       "     'text': 'Prolactinoma is the most common secreting pituitary adenoma. It is typically diagnosed in women of reproductive age and is common cause of infertility.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23090264',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1060,\n",
       "     'offsetInEndSection': 1308,\n",
       "     'text': 'Examination of the tissue excised by transsphenoidal excision of the mass showed a pituitary adenoma that stained strongly for FSH. RESULTS: Regular menses resumed soon after excision of the gonadotroph adenoma, followed by a spontaneous pregnancy.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12477530',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1309,\n",
       "     'offsetInEndSection': 1529,\n",
       "     'text': 'CONCLUSIONS: Gonadotroph adenoma should be suspected in a reproductive age woman with oligomenorrhea or amenorrhea, infertility, multiple preovulatory follicles, and a persistently elevated serum estradiol concentration.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12477530',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 199,\n",
       "     'offsetInEndSection': 614,\n",
       "     'text': 'Hyperprolactinemia is the most common endocrine disorder of the hypothalamic-pituitary axis, occurring mostly in women and presenting most commonly with amenorrhea and galactorrhea. Causes of hyperprolactinemia include physiologic, pharmacologic and pathologic factors; pituitary adenoma is a common pathologic cause. Women may present with decreased libido, infertility, oligomenorrhea/amenorrhea and galactorrhea.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10649815',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 443,\n",
       "     'offsetInEndSection': 788,\n",
       "     'text': 'When specific treatable underlying causes have been eliminated and in cases of severe hyperprolactinemia, the most likely cause is a prolactin (PRL)-secreting pituitary adenoma. Microadenomas should be treated medically, with a dopamine agonist, if there is an indication for therapy (such as amenorrhea, infertility or bothersome galactorrhea).',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10649814',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 173,\n",
       "     'text': 'Pregnancy in a woman with active acromegaly is very rare, because amenorrhea, due to hyperprolactinemia and disturbed pituitary gonadotropin secretion may cause infertility.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9152623',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 943,\n",
       "     'offsetInEndSection': 1318,\n",
       "     'text': 'Of the remaining six patients who had been investigated for infertility, no demonstrable cause of infertility was found in three. Of the other three patients, one showed evidence of bilateral tubal occlusion secondary to pelvic inflammatory disease, one has had a right ectopic pregnancy followed by two abortions, and the third patient was found to have a pituitary adenoma.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2738821',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 257,\n",
       "     'text': 'Results in 136 hyperprolactinaemic women who presented with infertility, amenorrhoea, menstrual irregularities and/or galactorrhoea are reported. There was radiographic evidence of pituitary microadenoma in 21 (15.4%) patients and 5 (3.7%) had macroadenoma.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2803131',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 474,\n",
       "     'offsetInEndSection': 778,\n",
       "     'text': 'Patients with pituitary adenoma had a significantly higher (p less than 0.001) baseline serum prolactin level (182 +/- 4.6 ng/ml) than those with no adenoma (59.2 +/- 4.2 ng/ml). All patients in the study were treated with bromocriptine (2.5-10 mg) to normalize serum prolactin or to achieve a pregnancy.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2803131',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1349,\n",
       "     'offsetInEndSection': 1462,\n",
       "     'text': 'There was no significant difference in the pregnancy rate between the patients with or without pituitary adenoma.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2803131',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 199,\n",
       "     'text': 'Two hyperprolactinemic infertile women, one with and one without a pituitary adenoma, who were resistant to bromocriptine treatment, were treated orally with Hachimijiogan, a Chinese herbal medicine.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2520800',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 92,\n",
       "     'text': 'Infertility caused by hyperprolactinemic amenorrhea may be complicated by pituitary adenoma.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/6868876',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Which are the cardiac effects of thyronamines?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18954857',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21835056',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19016324',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20880963',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17204552',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19273499',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20739399',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17579492',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22073124',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18486124'],\n",
       "   'ideal_answer': ['Thyronamines have negative chronotropy, negative inotropy; in particular thyronamines are considered negative inotropic agents',\n",
       "    'In the heart, thyronamines cause negative chronotropy, negative inotropy,reduced cardiac output and resistance to ischemic injury.'],\n",
       "   'exact_answer': [['negative chronotropy', 'lower heart rate'],\n",
       "    ['negative inotropy', 'negative inotropism'],\n",
       "    ['resistance to ischemic injury'],\n",
       "    ['reduced cardiac output']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D006331',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D006321',\n",
       "    'http://www.biosemantics.org/jochem#4251308',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002302'],\n",
       "   'type': 'list',\n",
       "   'id': '534bb147aeec6fbd07000014',\n",
       "   'snippets': [{'offsetInBeginSection': 1383,\n",
       "     'offsetInEndSection': 1653,\n",
       "     'text': 'Most recently, thyroid hormone derivatives were identified, the thyronamines which are decarboxylated thyroid hormones initiating physiological actions like lowering body temperature and heart rate, thereby acting in opposite direction to the classical thyroid hormones.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21835056',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 641,\n",
       "     'offsetInEndSection': 947,\n",
       "     'text': 'Intraperitoneal or central injection of 3-T(1)AM or T(0)AM into mice, rats, or Djungarian hamsters caused various prompt effects, such as metabolic depression, hypothermia, negative chronotropy, negative inotropy, hyperglycemia, reduction of the respiratory quotient, ketonuria, and reduction of fat mass. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20880963',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 329,\n",
       "     'offsetInEndSection': 620,\n",
       "     'text': 'Functional effects have been observed after administration of exogenous T(1)AM: in the isolated heart, a negative inotropic and chronotropic action was produced, and the resistance to ischemic injury was increased, possibly as a consequence of an action on intracellular calcium homeostasis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19016324',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 561,\n",
       "     'offsetInEndSection': 904,\n",
       "     'text': 'Octopamine, beta-phenylethylamine, and tryptamine produced a dose-dependent negative inotropic effect as shown by reduced cardiac output (IC(50)=109 microM, 159 microM, and 242 microM, respectively). In the same preparation a similar effect was produced by thyronamine and 3-iodothyronamine, with IC(50)=94 microM and 27 microM, respectively. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18486124',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'A class of thyroid hormone metabolites has dramatic physiological effects on metabolism and heart rate by still-unknown mechanisms of action. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17579492',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 254,\n",
       "     'offsetInEndSection': 384,\n",
       "     'text': 'In the mouse, thyronamines act rapidly in a nongenomic fashion to initiate hypothermia and decrease cardiac output and heart rate.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17204552',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which is the most well-accepted method for Down syndrome non-invasive prenatal diagnosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25218787',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22500647',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22839575',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18207470',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12498419',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21303301',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22038362',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17697502',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23371439',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22773950'],\n",
       "   'ideal_answer': ['Currently, two applications for NIPD of Down syndrome have been developed with potential and have displayed positive results; the NIPD using next-generation sequencing technologies and the NIPD using the methylated DNA immunoprecipitation (MeDIP) real-time quantitative polymerase chain reaction (qPCR). This study examined the methylation difference in AIRE and RASSF1A between maternal and placental DNA, and the implication of this difference in the identification of free fetal DNA in maternal plasma and in prenatal diagnosis of trisomy 21. Diagnosis of trisomy 21 was established according to the ratio of fetal-specific AIRE to RASSF1A in maternal plasma. Both methods confirmed that AIRE and RASSF1A were hypomethylated in maternal blood cells but hypermethylated in placental or chorionic villus tissues. It was concluded that hypermethylated AIRE and RASSF1A may serve as fetal-specific markers for the identification of fetal DNA in maternal plasma and may be used for noninvasive prenatal diagnosis of trisomy 21.',\n",
       "    'This study examined the methylation difference in AIRE and RASSF1A between maternal and placental DNA, and the implication of this difference in the identification of free fetal DNA in maternal plasma and in prenatal diagnosis of trisomy 21. Similarly, cell-free fetal DNA can be reliably recovered from maternal plasma and assessed by quantitative PCR to detect fetal trisomy 21 and paternally derived single gene mutations. The presence of foetal DNA in the plasma of pregnant women has opened up new possibilities for non-invasive prenatal diagnosis. Moreover, the differential methylation for each locus could be seen during the whole pregnant period.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D004314',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:14250',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D011296'],\n",
       "   'type': 'summary',\n",
       "   'id': '56f403d009dd18d46b000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 108,\n",
       "     'text': 'Potential of syncytiotrophoblasts isolated from the cervical mucus for early non-invasive prenatal diagnosis',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25218787',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': 'Maternal serum protein profile and immune response protein subunits as markers for non-invasive prenatal diagnosis of trisomy 21, 18, and 13',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23371439',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1335,\n",
       "     'offsetInEndSection': 1565,\n",
       "     'text': 'Maternal serum protein profiling using proteomics may allow non-invasive diagnostic testing for the most common trisomies and may complement ultrasound-based methods to more accurately determine pregnancies with fetal aneuploidies',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23371439',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 316,\n",
       "     'text': 'The presence of foetal DNA in the plasma of pregnant women has opened up new possibilities for non-invasive prenatal diagnosis. The use of circulating foetal DNA for the non-invasive prenatal detection of foetal chromosomal aneuploidies is challenging as foetal DNA represents a minor fraction of maternal plasma DNA',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22773950',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 596,\n",
       "     'offsetInEndSection': 704,\n",
       "     'text': 'Using massively parallel sequencing, foetal trisomies 21, 13 and 18 have been detected from maternal plasma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22773950',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1017,\n",
       "     'offsetInEndSection': 1167,\n",
       "     'text': 'These developments suggest that the analysis of foetal DNA in maternal plasma would play an increasingly important role in future obstetrics practice.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22773950',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 91,\n",
       "     'offsetInEndSection': 393,\n",
       "     'text': 'Currently, two applications for NIPD of Down syndrome have been developed with potential and have displayed positive results; the NIPD using next-generation sequencing technologies and the NIPD using the methylated DNA immunoprecipitation (MeDIP) real-time quantitative polymerase chain reaction (qPCR)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22500647',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 337,\n",
       "     'offsetInEndSection': 644,\n",
       "     'text': 'Cell-free fetal RNA was extracted from the plasma of peripheral blood from 121 women 9-20 weeks of pregnancy. Five single nucleotide polymorphism (SNP) loci in PLAC4 gene were analyzed by reverse transcriptase multiplex ligation-dependent probe amplification (RT-MLPA), followed by capillary electrophoresis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21303301',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 128,\n",
       "     'text': 'Non-invasive prenatal diagnosis of trisomy 21 by dosage ratio of fetal chromosome-specific epigenetic markers in maternal plasma',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22038362',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 241,\n",
       "     'text': 'This study examined the methylation difference in AIRE and RASSF1A between maternal and placental DNA, and the implication of this difference in the identification of free fetal DNA in maternal plasma and in prenatal diagnosis of trisomy 21.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22038362',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 600,\n",
       "     'offsetInEndSection': 867,\n",
       "     'text': 'Diagnosis of trisomy 21 was established according to the ratio of fetal-specific AIRE to RASSF1A in maternal plasma. Both methods confirmed that AIRE and RASSF1A were hypomethylated in maternal blood cells but hypermethylated in placental or chorionic villus tissues.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22038362',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1606,\n",
       "     'offsetInEndSection': 1817,\n",
       "     'text': 'It was concluded that hypermethylated AIRE and RASSF1A may serve as fetal-specific markers for the identification of fetal DNA in maternal plasma and may be used for noninvasive prenatal diagnosis of trisomy 21.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22038362',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 764,\n",
       "     'offsetInEndSection': 1131,\n",
       "     'text': 'Preliminary reports indicate that the detection of fetal aneuploidies might be possible using epigenetically modified genes, e.g. maspin on chromosome 18. Additionally, an exiting recent development is that it might be feasible to detect Down syndrome via the quantitative assessment of placentally derived cell-free mRNA of chromosome-21-specific genes such as PLAC4',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18207470',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 84,\n",
       "     'text': 'Candidate epigenetic biomarkers for non-invasive prenatal diagnosis of Down syndrome',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17697502',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1124,\n",
       "     'offsetInEndSection': 1286,\n",
       "     'text': 'Next, it was shown that the methylation status of chorionic villus sample DNA from first trimester pregnancies matched the hypermethylated state of term placenta.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17697502',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 120,\n",
       "     'text': 'Non-invasive prenatal diagnosis of trisomy 21 by reverse transcriptase multiplex ligation-dependent probe amplification.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21303301',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 193,\n",
       "     'text': 'To study whether pregnant women would like to be informed if sex chromosomal abnormalities (SCA) were suspected with the non-invasive prenatal diagnosis of fetal Down syndrome (the NIFTY) test.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22839575',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Are conserved noncoding elements associated with developmental genes?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18282512',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16630819',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19562753',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21175683',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18279518',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16533910',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17387144',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19698106',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16859531',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19073165'],\n",
       "   'ideal_answer': ['Yes.  Numerous studies suggest that conserved noncoding elements span developmental regulatory genes and define regulatory domains.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0051094',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0048589',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D050437',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0050793',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0032502'],\n",
       "   'type': 'yesno',\n",
       "   'id': '553d02c1f321868558000012',\n",
       "   'snippets': [{'offsetInBeginSection': 143,\n",
       "     'offsetInEndSection': 345,\n",
       "     'text': 'Some characteristics of CNEs include their high frequency in mammalian genomes, their potential regulatory role in gene expression, and their enrichment in gene deserts nearby master developmental genes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21175683',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 519,\n",
       "     'offsetInEndSection': 786,\n",
       "     'text': 'we review recent findings that disruptions of CNEs, within or at long distance from the coding sequences of key genes involved in NCC development, result in neurocristopathies via the alteration of tissue- or stage-specific long-distance regulation of gene expression',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21175683',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 169,\n",
       "     'text': 'Genomic regulatory blocks are chromosomal regions spanned by long clusters of highly conserved noncoding elements devoted to long-range regulation of developmental genes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19698106',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1186,\n",
       "     'offsetInEndSection': 1609,\n",
       "     'text': 'Analysis of CNEs, at least some of which are candidate regulatory elements, suggests that ancestral CNEs partitioned between gene duplicates. These results help explain the evolutionary pathways by which the developmentally important family of FgfD molecules arose and the deduced principles that guided FgfD evolution are likely applicable to the evolution of developmental regulation in many vertebrate multigene families',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19562753',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 676,\n",
       "     'text': 'Pan-vertebrate developmental cis-regulatory elements are discernible as highly conserved noncoding elements (HCNEs) and are often dispersed over large areas around the pleiotropic genes whose expression they control. On the loci of two developmental transcription factor genes, SOX3 and PAX6, we demonstrate that HCNEs conserved between human and zebrafish can be systematically and reliably tested for their regulatory function in multiple stable transgenes in zebrafish, and their genomic reach estimated with confidence using synteny conservation and HCNE density along these loci. HCNEs of both human and zebrafish function as specific developmental enhancers in zebrafish',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19073165',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 678,\n",
       "     'offsetInEndSection': 875,\n",
       "     'text': 'We show that human HCNEs result in expression patterns in zebrafish equivalent to those in mouse, establishing zebrafish as a suitable model for large-scale testing of human developmental enhancers',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19073165',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1185,\n",
       "     'offsetInEndSection': 1390,\n",
       "     'text': 'HCNEs from the same area often drive overlapping patterns, suggesting that multiple regulatory inputs are required to achieve robust and precise complex expression patterns exhibited by developmental genes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19073165',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 90,\n",
       "     'text': 'Organization of conserved elements near key developmental regulators in vertebrate genomes',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18282512',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 480,\n",
       "     'offsetInEndSection': 647,\n",
       "     'text': 'Further positional analysis of these conserved noncoding elements (CNEs) in the genome demonstrates that they cluster around genes involved in developmental regulation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18282512',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 130,\n",
       "     'text': 'Ancora: a web resource for exploring highly conserved noncoding elements and their association with developmental regulatory genes',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18279518',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 149,\n",
       "     'text': 'Metazoan genomes contain arrays of highly conserved noncoding elements (HCNEs) that span developmental regulatory genes and define regulatory domains',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18279518',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 276,\n",
       "     'text': 'The most highly conserved noncoding elements (HCNEs) in mammalian genomes cluster within regions enriched for genes encoding developmentally important transcription factors (TFs). This suggests that HCNE-rich regions may contain key regulatory controls involved in development',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16630819',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 112,\n",
       "     'offsetInEndSection': 353,\n",
       "     'text': 'We found the largest mammal-teleost conserved chromosomal segments to be spanned by highly conserved noncoding elements (HCNEs), their developmental regulatory target genes, and phylogenetically and functionally unrelated \"bystander\" genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17387144',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'Ancora: a web resource for exploring highly conserved noncoding elements and their association with developmental regulatory genes.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18279518',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'Pan-vertebrate developmental cis-regulatory elements are discernible as highly conserved noncoding elements (HCNEs) and are often dispersed over large areas around the pleiotropic genes whose expression they control.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19073165',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': 'Metazoan genomes contain arrays of highly conserved noncoding elements (HCNEs) that span developmental regulatory genes and define regulatory domains.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18279518',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 476,\n",
       "     'offsetInEndSection': 645,\n",
       "     'text': 'Further positional analysis of these conserved noncoding elements (CNEs) in the genome demonstrates that they cluster around genes involved in developmental regulation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18282512',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 179,\n",
       "     'text': 'The most highly conserved noncoding elements (HCNEs) in mammalian genomes cluster within regions enriched for genes encoding developmentally important transcription factors (TFs).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16630819',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 86,\n",
       "     'text': 'Disruption of long-distance highly conserved noncoding elements in neurocristopathies.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21175683',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 305,\n",
       "     'text': 'Fish-mammal genomic comparisons have proved powerful in identifying conserved noncoding elements likely to be cis-regulatory in nature, and the majority of those tested in vivo have been shown to act as tissue-specific enhancers associated with genes involved in transcriptional regulation of development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16533910',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 139,\n",
       "     'offsetInEndSection': 422,\n",
       "     'text': 'Despite this, attempts at unearthing genome-wide regulatory elements conserved throughout the vertebrate lineage using BLAST-like approaches have thus far detected noncoding conservation in only a few hundred genes, mostly associated with regulation of transcription and development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16859531',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 480,\n",
       "     'offsetInEndSection': 648,\n",
       "     'text': 'Further positional analysis of these conserved noncoding elements (CNEs) in the genome demonstrates that they cluster around genes involved in developmental regulation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18282512',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 114,\n",
       "     'offsetInEndSection': 354,\n",
       "     'text': 'We found the largest mammal-teleost conserved chromosomal segments to be spanned by highly conserved noncoding elements (HCNEs), their developmental regulatory target genes, and phylogenetically and functionally unrelated \"bystander\" genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17387144',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 91,\n",
       "     'text': 'Organization of conserved elements near key developmental regulators in vertebrate genomes.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18282512',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'Pan-vertebrate developmental cis-regulatory elements are discernible as highly conserved noncoding elements (HCNEs) and are often dispersed over large areas around the pleiotropic genes whose expression they control',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19073165',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'In what proportion of children with heart failure has Enalapril been shown to be safe and effective?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/1318542',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23124387',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8110005',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12530495',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9315539',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7576410',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8512763',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14990637',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9381720',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12454107'],\n",
       "   'ideal_answer': ['In children with heart failure evidence of the effect of enalapril is empirical. Enalapril was clinically safe and effective in 50% to 80% of for children with cardiac failure secondary to congenital heart malformations before and after cardiac surgery,  impaired ventricular function , valvar regurgitation,  congestive cardiomyopathy,  , arterial hypertension, life-threatening arrhythmias coexisting with circulatory insufficiency.   \\nACE inhibitors have shown a transient beneficial effect on heart failure due to anticancer drugs and possibly a beneficial effect in muscular dystrophy-associated cardiomyopathy, which deserves further studies.'],\n",
       "   'exact_answer': ['50% to 80%'],\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4176194',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D006333',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:6000',\n",
       "    'http://www.biosemantics.org/jochem#4249241',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D054144',\n",
       "    'http://www.biosemantics.org/jochem#4175579',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D054143',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D004656',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015773',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002648',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D006321',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:9775',\n",
       "    'http://www.biosemantics.org/jochem#4250224',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:9651'],\n",
       "   'type': 'factoid',\n",
       "   'id': '515dd3d5298dcd4e5100001c',\n",
       "   'snippets': [{'offsetInBeginSection': 1407,\n",
       "     'offsetInEndSection': 1609,\n",
       "     'text': 'The responses to IV KCl were attenuated by concomitant furosemide (p\\xa0=\\xa00.01), amphotericin B (p\\xa0<\\xa00.01), and KCl in parenteral nutrition (p\\xa0<\\xa00.01). The responses were augmented by concomitant enalapril',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23124387',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 306,\n",
       "     'text': 'To determine whether an angiotensin-converting enzyme (ACE) inhibitor, enalapril, prevents cardiac function deterioration (defined using maximal cardiac index [MCI] on exercise testing or increase in left ventricular end-systolic wall stress [LVESWS]) in long-term survivors of pediatric cancer.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14990637',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1342,\n",
       "     'offsetInEndSection': 1441,\n",
       "     'text': 'Enalapril treatment did not influence exercise performance, but did reduce LVESWS in the first year',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14990637',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 233,\n",
       "     'text': 'Patients with intraatrial baffle procedure for transposition of the great arteries (TGA) have diastolic dysfunction, decreased exercise capacity, stroke volume response and elevated systemic vascular resistance (SVR) during exercise.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12530495',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1713,\n",
       "     'offsetInEndSection': 1887,\n",
       "     'text': 'We conclude that short-term (<1 year) use of enalapril does not improve exercise performance in patients with TGA in whom the intraatrial baffle procedure has been performed.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12530495',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 428,\n",
       "     'text': 'A common late effect of doxorubicin therapy for childhood cancer is reduced left-ventricular (LV) wall thickness resulting in elevated LV afterload and depressed LV function. Many children are given angiotensin-converting enzyme inhibitors, which have been studied primarily in adults. We document the long-term effects of angiotensin-converting enzyme inhibitors in doxorubicin-treated survivors of childhood cancer.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12454107',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1375,\n",
       "     'offsetInEndSection': 1676,\n",
       "     'text': 'In doxorubicin-treated long-term survivors of childhood cancer, enalapril-induced improvement in LV structure and function is transient. The primary defect, which is LV wall thinning, continues to deteriorate, and thus the short-term improvement was mostly related to lowered diastolic blood pressure.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12454107',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 208,\n",
       "     'offsetInEndSection': 417,\n",
       "     'text': 'Patients who have undergone the Fontan procedure have decreased cardiac output, increased systemic vascular resistance, abnormal diastolic function, and decreased exercise capacity compared with normal people.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9315539',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1884,\n",
       "     'offsetInEndSection': 2107,\n",
       "     'text': 'We conclude that enalapril administration for 10 weeks does not alter abnormal systemic vascular resistance, resting cardiac index, diastolic function, or exercise capacity in patients who have undergone a Fontan procedure.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9315539',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 642,\n",
       "     'text': 'Angiotensin convertase inhibitor (Enalapril) was used in 51 children aged 4 days up to 18 years (mean 4.3 +/- 5.5, years). As many as 27 subjects were newborns (4) and infants (23). The patients suffered from circulatory insufficiency due to congestive cardiomyopathy (13 cases). 6 treated subjects suffered from circulatory insufficiency due to congenital heart malformations before cardiac surgery and 22 after it (including complex malformations operated according to Fontan method). 10 children were treated because of arterial hypertension. 4 subjects suffered form life-threatening arrhythmias coexisting with circulatory insufficiency.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9381720',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 850,\n",
       "     'offsetInEndSection': 1233,\n",
       "     'text': '4 patients (8%) died during treatment but their deaths can not be related to angiotensin convertase inhibitor therapy. In the other children (82%) the beneficial influence of angiotensin convertase inhibitor use was found (improvement in comparison with the state before convertase inhibitor introduction). In 10% of subjects enalapril did not show any significant therapeutic effect',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9381720',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1183,\n",
       "     'offsetInEndSection': 1385,\n",
       "     'text': 'We conclude that the combination of ACE inhibitor and beta-blocker deserves further exploration for inclusion in any management regimen for the treatment of muscular dystrophy-associated cardiomyopathy.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7576410',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 943,\n",
       "     'offsetInEndSection': 1182,\n",
       "     'text': 'Addition of these medications, never before attempted in the management of cardiomyopathy associated with generalized myopathic disease, complemented each other in relieving symptoms and reversing signs of congestive heart failure and DCM.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7576410',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 864,\n",
       "     'offsetInEndSection': 1098,\n",
       "     'text': 'Enalapril was clinically safe and effective for children with cardiac failure secondary to ventricular impairment, valvar regurgitation, or after cardiac surgery. Renal failure was a problem in young infants with left-to-right shunts.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8110005',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 694,\n",
       "     'text': 'In a tertiary referral centre 63 patients underwent 67 treatment periods with enalapril. The median age was 5.4 months. All children had signs of heart failure: congestive cardiac failure with breathlessness at rest was present in 88%. Haemodynamic groups were left-to-right shunt (n = 15), impaired ventricular function (n = 14), after cardiac surgery (n = 23), valvar regurgitation (n = 12), and hypertension (n = 3). Serial clinical, radiological, and laboratory data were used to judge outcome. The mean (SD) maximal dose was 0.30 (0.21) mg/kg/day. Thirty nine (58%) patients improved, 20 (30%) showed no improvement, and eight (12%) had side effects requiring discontinuation of enalapril.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8110005',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 225,\n",
       "     'text': 'We studied the inhibition of angiotensin converting enzyme (ACE) in eight infants with congestive heart failure (CHF) poorly controlled with digoxin and diuretics, treated orally with 0.25 mg kg-1 enalapril maleate once a day',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8512763',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 584,\n",
       "     'offsetInEndSection': 680,\n",
       "     'text': 'In infants with CHF, mean baseline ACE activity was significantly higher than in control infants',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8512763',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1240,\n",
       "     'offsetInEndSection': 1397,\n",
       "     'text': 'Converting enzyme inhibitors may benefit \"heart failure\" associated with large ventricular septal defects and normal or mildly elevated pulmonary resistance.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1318542',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Which cell type has the protein Chromogranin A as marker?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25532001',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25294372',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25294889',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25394660',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24888775',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24897131',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25177680',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25099181',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25220535',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25501094'],\n",
       "   'triples': [{'p': 'http://purl.uniprot.org/core/reviewed',\n",
       "     's': 'http://purl.uniprot.org/uniprot/P33716',\n",
       "     'o': 'true'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_503333373136007',\n",
       "     'o': 'Chromogranin-A'},\n",
       "    {'p': 'http://purl.uniprot.org/core/recommendedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/P33716',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_503333373136007'},\n",
       "    {'p': 'http://purl.uniprot.org/core/classifiedWith',\n",
       "     's': 'http://purl.uniprot.org/uniprot/P33716',\n",
       "     'o': 'http://purl.uniprot.org/keywords/964'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://purl.uniprot.org/keywords/964',\n",
       "     'o': 'Secreted'}],\n",
       "   'ideal_answer': ['Chromogranin A is a marker for neuroendocrine cells'],\n",
       "   'exact_answer': ['neuroendocrine cells'],\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/CMGA_BOVIN',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D000943',\n",
       "    'http://www.biosemantics.org/jochem#4266983',\n",
       "    'http://www.biosemantics.org/jochem#4262122',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D053379',\n",
       "    'http://www.biosemantics.org/jochem#4218257',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002864',\n",
       "    'http://www.uniprot.org/uniprot/CMGA_HUMAN'],\n",
       "   'type': 'factoid',\n",
       "   'id': '54e262daae9738404b000018',\n",
       "   'snippets': [{'offsetInBeginSection': 562,\n",
       "     'offsetInEndSection': 642,\n",
       "     'text': 'Neuroendocrine differentiation (chromogranin A and/or synaptophysin positivity) ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25294889',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1029,\n",
       "     'offsetInEndSection': 1093,\n",
       "     'text': 'All paragangliomas were universally positive for chromogranin A ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25294372',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 9,\n",
       "     'offsetInEndSection': 92,\n",
       "     'text': 'chromogranin A (CGA) is a useful marker for pancreatic neuroendocrine tumors (pNET)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25220535',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 151,\n",
       "     'text': 'Together with Chromogranin B and Secretogranins, Chromogranin A (CGA) is stored in secretory (chromaffin) granules of the diffuse neuroendocrine system',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25177680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 21,\n",
       "     'offsetInEndSection': 168,\n",
       "     'text': 'Chromogranin A (CgA) is the most important general tumour marker used in the diagnosis and follow-up of patients with neuroendocrine tumours (NET).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25532001',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 197,\n",
       "     'text': 'Chromogranin A (CgA) not only plays an important role in pathologic diagnosis, but is also used as a circulating biomarker in patients with gastroenteropancreatic neuroendocrine neoplasm (GEP-NEN).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25501094',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1206,\n",
       "     'offsetInEndSection': 1274,\n",
       "     'text': 'numbers of chromogranin A (CHGA)-positive enteroendocrine cells (EEC',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25394660',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1761,\n",
       "     'offsetInEndSection': 1816,\n",
       "     'text': ' CgA is a reliable serum diagnostic biomarker for PNETs',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25099181',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 81,\n",
       "     'offsetInEndSection': 186,\n",
       "     'text': 'Chromogranin A (CgA) was considered as the most practical and useful serum tumor marker in PNET patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25099181',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 401,\n",
       "     'offsetInEndSection': 461,\n",
       "     'text': 'the neuroendocrine markers chromogranin A and synaptophysin;',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24888775',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is Lysine-specific demethylase 1 (LSD1) a critical regulator of hematopoiesis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/19497860',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19736520',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22801375',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23795291',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22310283',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17707228',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23147254',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24163373',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22399799'],\n",
       "   'ideal_answer': ['Yes. Lysine-specific demethylase 1 restricts hematopoietic progenitor proliferation and is essential for terminal differentiation of erythroid, granulomonocytic and megakaryocytic progenitors.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0035162',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0030097',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0033193'],\n",
       "   'type': 'yesno',\n",
       "   'id': '553c011af321868558000009',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'Histone lysine-specific demethylase 1 (LSD1) protein is involved in Sal-like protein 4 (SALL4)-mediated transcriptional repression in hematopoietic stem cells',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24163373',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1167,\n",
       "     'offsetInEndSection': 1317,\n",
       "     'text': 'shRNA-mediated knockdown of LSD1 in hematopoietic precursor cells resulted in altered SALL4 downstream gene expression and increased cellular activity',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24163373',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1325,\n",
       "     'offsetInEndSection': 1560,\n",
       "     'text': 'our data revealed that histone demethylase LSD1 may negatively regulate SALL4-mediated transcription, and the dynamic regulation of SALL4-associated epigenetic factors cooperatively modulates early hematopoietic precursor proliferation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24163373',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 124,\n",
       "     'text': 'Lysine-specific demethylase 1 restricts hematopoietic progenitor proliferation and is essential for terminal differentiation',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 326,\n",
       "     'offsetInEndSection': 405,\n",
       "     'text': ' LSD1 represents a central regulator of hematopoietic stem and progenitor cells',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 502,\n",
       "     'offsetInEndSection': 602,\n",
       "     'text': ' LSD1-kd led to an extensive expansion of granulomonocytic, erythroid and megakaryocytic progenitors',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1052,\n",
       "     'offsetInEndSection': 1123,\n",
       "     'text': 'LSD1-kd was associated with the upregulation of key hematopoietic genes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1352,\n",
       "     'offsetInEndSection': 1422,\n",
       "     'text': 'our findings distinguish LSD1 as a critical regulator of hematopoiesis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': 'A short Gfi-1B isoform controls erythroid differentiation by recruiting the LSD1-CoREST complex through the dimethylation of its SNAG domain',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22399799',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 113,\n",
       "     'text': 'Dynamic interaction between TAL1 oncoprotein and LSD1 regulates TAL1 function in hematopoiesis and leukemogenesis',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22310283',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 341,\n",
       "     'offsetInEndSection': 778,\n",
       "     'text': 'Here, we reported that protein kinase A (PKA)-mediated phosphorylation regulates TAL1 interaction with the lysine-specific demethylase (LSD1) that removes methyl group from methylated Lys 4 on histone H3 tails. Phosphorylation of serine 172 in TAL1 specifically destabilizes the TAL1-LSD1 interaction leading to promoter H3K4 hypermethylation and activation of target genes that have been suppressed in normal and malignant hematopoiesis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22310283',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 85,\n",
       "     'text': 'LSD1-mediated epigenetic modification is required for TAL1 function and hematopoiesis',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19497860',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 251,\n",
       "     'offsetInEndSection': 525,\n",
       "     'text': 'we show that TAL1 is associated with histone demethylase complexes containing lysine-specific demethylase 1 (LSD1), RE1 silencing transcription factor corepressor (CoREST), histone deacetylase 1 (HDAC1), and histone deacetylase 2 in erythroleukemia and T cell leukemia cells',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19497860',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 703,\n",
       "     'offsetInEndSection': 871,\n",
       "     'text': 'we demonstrate that the TAL1-associated LSD1, HDAC1, and their enzymatic activities are coordinately down-regulated during the early phases of erythroid differentiation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19497860',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 959,\n",
       "     'offsetInEndSection': 1090,\n",
       "     'text': 'TAL1 recruits LSD1 to the silenced p4.2 promoter in undifferentiated, but not in differentiated, murine erythroleukemia (MEL) cells',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19497860',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1385,\n",
       "     'offsetInEndSection': 1507,\n",
       "     'text': 'the dynamic regulation of TAL1-associated LSD1/HDAC1 complex may determine the onset of erythroid differentiation programs',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19497860',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': 'Epigenetic regulation of hematopoietic differentiation by Gfi-1 and Gfi-1b is mediated by the cofactors CoREST and LSD1',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17707228',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 592,\n",
       "     'offsetInEndSection': 740,\n",
       "     'text': 'Inhibition of CoREST and LSD1 perturbs differentiation of erythroid, megakaryocytic, and granulocytic cells as well as primary erythroid progenitors',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17707228',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 904,\n",
       "     'offsetInEndSection': 1140,\n",
       "     'text': 'we show that chromatin regulatory proteins CoREST and LSD1 mediate transcriptional repression by Gfi proteins. Lineage-restricted deployment of these cofactors through interaction with Gfi proteins controls hematopoietic differentiation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17707228',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1324,\n",
       "     'offsetInEndSection': 1489,\n",
       "     'text': 'Taken together, our findings distinguish LSD1 as a critical regulator of hematopoiesis and point to severe, but reversible, side effects of a LSD1-targeted therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 86,\n",
       "     'text': 'LSD1-mediated epigenetic modification is required for TAL1 function and hematopoiesis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19497860',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 114,\n",
       "     'text': 'Dynamic interaction between TAL1 oncoprotein and LSD1 regulates TAL1 function in hematopoiesis and leukemogenesis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22310283',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 547,\n",
       "     'offsetInEndSection': 775,\n",
       "     'text': 'Phosphorylation of serine 172 in TAL1 specifically destabilizes the TAL1-LSD1 interaction leading to promoter H3K4 hypermethylation and activation of target genes that have been suppressed in normal and malignant hematopoiesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22310283',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 159,\n",
       "     'text': 'Histone lysine-specific demethylase 1 (LSD1) protein is involved in Sal-like protein 4 (SALL4)-mediated transcriptional repression in hematopoietic stem cells.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24163373',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1336,\n",
       "     'offsetInEndSection': 1500,\n",
       "     'text': 'Taken together, our findings distinguish LSD1 as a critical regulator of hematopoiesis and point to severe, but reversible, side effects of a LSD1-targeted therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22699452',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Have 5q35 microdeletions been implicated in Sotos syndrome development?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25510705',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18505455',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12900893',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16140999',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23190751',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15805156',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24670087',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24192683',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22012791',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23913520'],\n",
       "   'ideal_answer': ['Loss-of-function mutations of NSD1 and 5q35 microdeletions encompassing NSD1 are a major cause of Sotos syndrome (Sos), which is characterized by overgrowth, macrocephaly, characteristic facies, and variable intellectual disability (ID).',\n",
       "    'Loss-of-function mutations of NSD1 and 5q35 microdeletions encompassing NSD1 are a major cause of Sotos syndrome (Sos), which is characterized by overgrowth, macrocephaly, characteristic facies, and variable intellectual disability (ID)'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '571f33f10fd6f91b68000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 236,\n",
       "     'text': 'Loss-of-function mutations of NSD1 and 5q35 microdeletions encompassing NSD1 are a major cause of Sotos syndrome (Sos), which is characterized by overgrowth, macrocephaly, characteristic facies, and variable intellectual disability (ID)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23913520',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 684,\n",
       "     'text': 'We observed a novel 3.5 Mb 5q subtelomeric deletion in a 3-year-old girl with developmental delay, hypotonia and multiple minor anomalies. Comparison of her phenotype with the few published patients with terminal 5q35 deletions revealed several overlapping features, but also showed remarkable differences such as shortness of stature versus macrosomia. After the report of 5q35.3 microdeletions in Sotos syndrome we integrated the published BACs into the public draft sequence and exactly mapped the deletion size in our patient by FISH analysis with 15 BAC probes. We demonstrated that the deletion in our patient is immediately adjacent to the reported Sotos syndrome deletion site',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12900893',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 172,\n",
       "     'text': 'Switch in FGFR3 and -4 expression profile during human renal development may account for transient hypercalcemia in patients with Sotos syndrome due to 5q35 microdeletions.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24670087',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 94,\n",
       "     'text': 'Multiple mechanisms are implicated in the generation of 5q35 microdeletions in Sotos syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15805156',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 564,\n",
       "     'offsetInEndSection': 776,\n",
       "     'text': 'After the report of 5q35.3 microdeletions in Sotos syndrome we integrated the published BACs into the public draft sequence and exactly mapped the deletion size in our patient by FISH analysis with 15 BAC probes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12900893',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 159,\n",
       "     'text': 'Clinical and genetic spectrum of 18 unrelated Korean patients with Sotos syndrome: frequent 5q35 microdeletion and identification of four novel NSD1 mutations.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23190751',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 302,\n",
       "     'offsetInEndSection': 479,\n",
       "     'text': 'Here we describe a new case of Sotos syndrome with a 5q35 microdeletion, affecting the fibroblast growth factor receptor 4 (FGFR4) gene, presenting with infantile hypercalcemia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24670087',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 216,\n",
       "     'offsetInEndSection': 330,\n",
       "     'text': 'Microdeletions at 5q35.3, encompassing NSD1, are responsible for approximately 10% of non-Japanese cases of Sotos.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15805156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 50,\n",
       "     'text': 'Alu-related 5q35 microdeletions in Sotos syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18505455',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 92,\n",
       "     'text': 'Most cases of Sotos syndrome are caused by intragenic NSD1 mutations or 5q35 microdeletions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16140999',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 93,\n",
       "     'text': 'Multiple mechanisms are implicated in the generation of 5q35 microdeletions in Sotos syndrome',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15805156',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'Clinical and genetic spectrum of 18 unrelated Korean patients with Sotos syndrome: frequent 5q35 microdeletion and identification of four novel NSD1 mutations',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23190751',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 103,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'Microdeletions at 5q35.3, encompassing NSD1, are responsible for approximately 10% of non-Japanese cases of Sotos',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15805156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'A case of Sotos syndrome with 5q35 microdeletion and novel clinical findings.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24192683',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 135,\n",
       "     'offsetInEndSection': 313,\n",
       "     'text': 'Here we describe a new case of Sotos syndrome with a 5q35 microdeletion, affecting the fibroblast growth factor receptor 4 (FGFR4) gene, presenting with infantile hypercalcemia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24670087',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 110,\n",
       "     'offsetInEndSection': 312,\n",
       "     'text': 'There are two types of mutations that cause NSD1 haploinsufficiency: mutations within the NSD1 gene (mutation type) and a 5q35 submicroscopic deletion encompassing the entire NSD1 gene (deletion type). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22012791',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1350,\n",
       "     'offsetInEndSection': 1458,\n",
       "     'text': 'aCGH and metaphase FISH are useful for rapid diagnosis of 5q35 microdeletion associated with Sotos syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25510705',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 95,\n",
       "     'text': 'Multiple mechanisms are implicated in the generation of 5q35 microdeletions in Sotos syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15805156',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 173,\n",
       "     'text': 'Switch in FGFR3 and -4 expression profile during human renal development may account for transient hypercalcemia in patients with Sotos syndrome due to 5q35 microdeletions.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24670087',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 78,\n",
       "     'text': 'A case of Sotos syndrome with 5q35 microdeletion and novel clinical findings.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24192683',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 93,\n",
       "     'text': 'Most cases of Sotos syndrome are caused by intragenic NSD1 mutations or 5q35 microdeletions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16140999',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1316,\n",
       "     'offsetInEndSection': 1424,\n",
       "     'text': 'aCGH and metaphase FISH are useful for rapid diagnosis of 5q35 microdeletion associated with Sotos syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25510705',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 51,\n",
       "     'text': 'Alu-related 5q35 microdeletions in Sotos syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18505455',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Is metabolic syndrome related with cardiovascular disease?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18945929',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24247648',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19196885',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24320036',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24320031',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24287796',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24328010',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21234418',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24320038',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24290090'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A2041867',\n",
       "     'o': 'Syndrome, Metabolic Cardiovascular'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A2029581',\n",
       "     'o': 'Cardiovascular Syndrome, Metabolic'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A2029582',\n",
       "     'o': 'Cardiovascular Syndromes, Metabolic'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A17989670',\n",
       "     'o': 'Metabolic Cardiovascular Syndrome'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A2035905',\n",
       "     'o': 'Metabolic Cardiovascular Syndrome'}],\n",
       "   'ideal_answer': ['The metabolic syndrome (MetS) is characterized by a cluster of risk factors including central obesity, hypertension, dyslipidemia and insulin resistance, The MetS is associated with an increased risk for cardiovascular disease (CVD) and type 2 diabetes mellitus (T2DM).'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D024821',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002318',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:14221',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:1287'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5328185dd6d3ac6a3400000e',\n",
       "   'snippets': [{'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 282,\n",
       "     'text': 'The metabolic syndrome (MetS) is characterized by a cluster of risk factors including central obesity, hypertension, dyslipidemia and insulin resistance, The MetS is associated with an increased risk for cardiovascular disease (CVD) and type 2 diabetes mellitus (T2DM). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24320038',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 260,\n",
       "     'offsetInEndSection': 622,\n",
       "     'text': 'As a molecular link between metabolic signals, inflammation, and vascular dysfunction, resistin can be proposed as playing a significant role in the heightened inflammatory state induced by metabolic stress linked to excessive caloric intake, thus contributing to the risk for metabolic syndrome (MetS), type 2 diabetes (T2DM), and cardiovascular disease (CVD). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24320036',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'The metabolic syndrome (MetS) is associated with a higher risk for both, type 2 diabetes mellitus and cardiovascular disease. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24320031',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 110,\n",
       "     'offsetInEndSection': 241,\n",
       "     'text': 'arotid intima-media thickness (CIMT) has been widely used as a surrogate marker of atherosclerosis and cardiovascular disease (CVD)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24328010',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is there any functional association during viral replication between flaviviridae viral RNA depended RNA polymerase and viral helicase?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23463199',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22040846',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17319152',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19951725',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21946389',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23157297',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17660525',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21898331',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15917225',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18179252'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2004/02/skos/core#broader',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0206510',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C0035691'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0206510',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A0363752'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0206510',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A0363753'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A7573489',\n",
       "     'o': 'RNA Virus'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0112753',\n",
       "     'o': 'RNA Viruses'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0206510',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A0363751'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#altLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0206510',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A2143236'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0206510',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A0363752'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0132798',\n",
       "     'o': 'Virus, RNA'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0363751',\n",
       "     'o': 'Flaviviridae'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A7574474',\n",
       "     'o': 'Viruses, RNA'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A0451316',\n",
       "     'o': 'RNA virus'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A2143236',\n",
       "     'o': 'Flavivirus (arbovirus group B)'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A7666306',\n",
       "     'o': 'Virus-RNA'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#broader',\n",
       "     's': 'http://linkedlifedata.com/resource/diseaseontology/id/DOID:0050303',\n",
       "     'o': 'http://linkedlifedata.com/resource/diseaseontology/id/DOID:1886'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/diseaseontology/id/DOID:1886',\n",
       "     'o': 'Flaviviridae infectious disease'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/diseaseontology/id/DOID:0050303',\n",
       "     'o': 'Hepacivirus infectious disease'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#broader',\n",
       "     's': 'http://linkedlifedata.com/resource/diseaseontology/id/DOID:993',\n",
       "     'o': 'http://linkedlifedata.com/resource/diseaseontology/id/DOID:1886'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/diseaseontology/id/DOID:993',\n",
       "     'o': 'Flavivirus infectious disease'}],\n",
       "   'ideal_answer': ['Several labs have obtained evidence for a protein complex that involves many of the nonstructural (NS) proteins encoded by the virus. NS3, NS4A, NS4B, NS5A, and NS5B appear to interact structurally and functionally. The interaction between the helicase, NS3, and the RNA polymerase, NS5B play a key role in viral replication. Pull-down experiments and surface plasmon resonance data indicate a direct interaction between NS3 and NS5B that is primarily mediated through the protease domain of NS3. This interaction reduces the basal ATPase activity of NS3. However, NS5B stimulates product formation in RNA unwinding experiments under conditions of excess nucleic acid substrate. When the concentrations of NS3 and NS5B are in excess of nucleic acid substrate, NS5B reduces the rate of NS3-catalyzed unwinding. Under pre-steady-state conditions, in which NS3 and substrate concentrations are similar, product formation increased in the presence of NS5B. The increase was consistent with 1:1 complex formed between the two proteins.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D018067',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012194',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D020365',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012324',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012367'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5321b8579b2d7acc7e000008',\n",
       "   'snippets': [{'offsetInBeginSection': 275,\n",
       "     'offsetInEndSection': 1527,\n",
       "     'text': 'Several labs have obtained evidence for a protein complex that involves many of the nonstructural (NS) proteins encoded by the virus. NS3, NS4A, NS4B, NS5A, and NS5B appear to interact structurally and functionally. In this study, we investigated the interaction between the helicase, NS3, and the RNA polymerase, NS5B. Pull-down experiments and surface plasmon resonance data indicate a direct interaction between NS3 and NS5B that is primarily mediated through the protease domain of NS3. This interaction reduces the basal ATPase activity of NS3. However, NS5B stimulates product formation in RNA unwinding experiments under conditions of excess nucleic acid substrate. When the concentrations of NS3 and NS5B are in excess of nucleic acid substrate, NS5B reduces the rate of NS3-catalyzed unwinding. Under pre-steady-state conditions, in which NS3 and substrate concentrations are similar, product formation increased in the presence of NS5B. The increase was consistent with 1:1 complex formed between the two proteins. A fluorescently labeled form of NS3 was used to investigate this interaction through fluorescence polarization binding assays. Results from this assay support interactions that include a 1:1 complex formed between NS3 and NS5B.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18179252',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 81,\n",
       "     'offsetInEndSection': 1037,\n",
       "     'text': 'Contradictory results have been reported regarding NS3 in RNA synthesis. To investigate the effect of NS3 on classical swine fever virus (CSFV) NS5B RNA-dependent RNA polymerase activity (RdRp) activity and NS3-NS5B interaction, RdRp reactions, GST-pull-down assays and co-immunoprecipitation analyses containing NS5B and either of NS3 protein and the different truncated NS3 mutants were performed, respectively. We found that NS3 stimulated NS5B RdRp activity in a dose-dependent manner by binding to NS5 through a NS3 protease domain. Furthermore, mapping important regions of the NS3 protease domain was carried out by deletion mutagenesis, associated with RdRp reactions, GST-pull-down assays and co-immunoprecipitation analyses. Results showed that stimulation of CSFV NS5B RdRp activity was obtained by NS3 binding to NS5B through a 31-amino acid fragment at the N-terminal end of NS3 protease domain, which mediated a specific NS3-NS5B interaction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19951725',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 113,\n",
       "     'offsetInEndSection': 474,\n",
       "     'text': 'The protocols detailed in this unit are used to purify three recombinant enzymes that are widely used in HCV research: the HCV NS3 protease domain, the helicase domain as an NS3+NS4A complex, and the NS5B RNA-dependent RNA polymerase. The active enzymes are purified to homogeneity by two-column chromatography to support a screening program for HCV inhibitors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21898331',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 518,\n",
       "     'offsetInEndSection': 945,\n",
       "     'text': 'Among potential targets are viral entry factors, including scavenger receptor type B1 (SR-B1) and CD81, as well as neutralizing antibodies against the viral glycoproteins. Popular targets related to translation and replication are the NS3/4A protease (inhibited by telaprevir and boceprevir) and the NS5B polymerase, as well as the NS2/3 autoprotease, the NS3 helicase, and nonenzymatic targets such as NS4B and NS5A proteins. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21946389',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1210,\n",
       "     'offsetInEndSection': 1501,\n",
       "     'text': 'The NS3 helicase domain competes with NS3 full-length for NS5 RdRp binding, with a K(d.) of 2.5μM. Since NS3 and NS5 are required for DENV replication, this fascile assay could be used to screen for non-nucleoside, allosteric inhibitors that disrupt the interaction between the two proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22040846',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'How functional connectivity of the default mode network changes in patients with disorders of consciousness?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23034909',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22418054',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22905075',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22623966',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21693087',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21191476',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22218274',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19289479',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22039473',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22563263'],\n",
       "   'ideal_answer': ['Functional connectivity in the default mode network (DMN) is reduced in patients with different disorders of consciousness, and correlates with the level of consciousness. \\nSpecifically, functional connectivity in the default mode network was shown to be absent in brain death patients, extremely low in vegetative state patients and slightly decreased in minimally conscious state patients when compared to healthy subjects. Therefore, functional connectivity in the default mode network was suggested to be valuable in differentiating patients with different disorders of consciousness. Clinically, functional connectivity in the default mode network was also shown to be an indicator of the extent of cortical disruption and predict reversible impairments in consciousness.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D059907',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D009415',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D003244'],\n",
       "   'type': 'summary',\n",
       "   'id': '514a2649d24251bc0500005a',\n",
       "   'snippets': [{'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 196,\n",
       "     'text': 'Functional connectivity in the default mode network (DMN) is known to be reduced in patients with disorders of consciousness, to a different extent depending on their clinical severity.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23034909',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 926,\n",
       "     'offsetInEndSection': 1167,\n",
       "     'text': 'Patients showed significant impairments in all of the pathways connecting cortical regions within this network, as well as the pathway connecting the posterior cingulate cortex/precuneus with the thalamus, relative to the healthy volunteers.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23034909',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1168,\n",
       "     'offsetInEndSection': 1476,\n",
       "     'text': \"Moreover, the structural integrity of this pathway, as well as that of those connecting the posterior areas of the network, was correlated with the patients' behavioral signs for awareness, being higher in EMCS patients than those in the upper and lower ranges of the MCS patients, and lowest in VS patients.\",\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23034909',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1490,\n",
       "     'offsetInEndSection': 1719,\n",
       "     'text': 'These results provide a possible neural substrate for the functional disconnection previously described in these patients, and reinforce the importance of the DMN in the genesis of awareness and the neural bases of its disorders.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23034909',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 193,\n",
       "     'offsetInEndSection': 394,\n",
       "     'text': 'Although the functional significance of the default-mode network remains a matter of debate, it has been suggested to be a candidate for the network subserving basic functions related to consciousness.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22418054',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 510,\n",
       "     'offsetInEndSection': 842,\n",
       "     'text': 'Despite resolution of corpus callosum lesion on magnetic resonance imaging (MRI) within 1 week, the patient persistently presented disturbance of consciousness. Resting-state functional MRI revealed that the posterior cingulate cortex/precuneus was functionally disconnected from other brain regions within the default-mode network.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22418054',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 843,\n",
       "     'offsetInEndSection': 1061,\n",
       "     'text': 'Our case report suggests that assessment of the functional connectivity in the resting-state default-mode network could be a useful marker of consciousness disturbance even in the presence of a reversible brain lesion.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22418054',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 487,\n",
       "     'offsetInEndSection': 666,\n",
       "     'text': 'A present and intact DMN was observed in controls and those patients who subsequently regained consciousness, but was disrupted in all patients who failed to regain consciousness.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22218274',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 680,\n",
       "     'offsetInEndSection': 770,\n",
       "     'text': 'The results suggest that the DMN is necessary but not sufficient to support consciousness.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22218274',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 771,\n",
       "     'offsetInEndSection': 915,\n",
       "     'text': 'Clinically, DMN connectivity may serve as an indicator of the extent of cortical disruption and predict reversible impairments in consciousness.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22218274',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 427,\n",
       "     'offsetInEndSection': 643,\n",
       "     'text': 'Recent studies on resting state activity in DOC, measured with functional magnetic resonance imaging (fMRI) techniques, show that functional connectivity is disrupted in the task-negative or the default mode network.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21693087',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 309,\n",
       "     'offsetInEndSection': 626,\n",
       "     'text': 'This report shows for the first time, in three patients, that the persistent vegetative state (PVS) is marked by a dysfunctional default mode network, with decreased connectivity in several brain regions, including the dorsolateral prefrontal cortex and anterior cingulated cortex, especially in the right hemisphere.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19289479',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'What is the pyroptotic pathway?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/26056317',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25043180',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21178113',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24667705',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25341033',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24376002',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23816851',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21057511',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25418070',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23637985'],\n",
       "   'ideal_answer': ['Pyroptosis is an inflammasome-mediated programmed cell death pathway.'],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:649',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:12858',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D000069292',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:1393',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:6576',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:6575'],\n",
       "   'type': 'summary',\n",
       "   'id': '56afa7fd0a360a5e45000016',\n",
       "   'snippets': [{'offsetInBeginSection': 1036,\n",
       "     'offsetInEndSection': 1058,\n",
       "     'text': 'pyroptotic cell death,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26056317',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 633,\n",
       "     'offsetInEndSection': 655,\n",
       "     'text': ' pyroptotic cell death',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23816851',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 164,\n",
       "     'text': 'Pyroptosis is an inflammasome-mediated programmed cell death pathway triggered in macrophages by a variety of stimuli, including intracellular bacterial pathogens. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24376002',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 744,\n",
       "     'offsetInEndSection': 761,\n",
       "     'text': 'pyroptotic death ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25043180',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 335,\n",
       "     'offsetInEndSection': 402,\n",
       "     'text': 'C. albicans triggers pyroptosis, a proinflammatory macrophage death',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24667705',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 82,\n",
       "     'text': 'Pyroptosis is a caspase-1-dependent pro-inflammatory form of programmed cell death',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25418070',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 160,\n",
       "     'offsetInEndSection': 230,\n",
       "     'text': 'associated with pyroptosis, the pro-inflammatory programmed cell death',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25341033',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 686,\n",
       "     'offsetInEndSection': 880,\n",
       "     'text': 'Our study here identified a novel cell death, pyroptosis in ox-LDL induced human macrophage, which may be implicated in lesion macrophages death and play an important role in lesion instability.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23637985',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 563,\n",
       "     'offsetInEndSection': 584,\n",
       "     'text': 'pyroptotic cell death',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21178113',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 467,\n",
       "     'offsetInEndSection': 508,\n",
       "     'text': ' caspase-1-induced pyroptotic cell death ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21057511',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 99,\n",
       "     'text': 'Caspase-1-induced pyroptosis is an innate immune effector mechanism against intracellular bacteria.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21057511',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'What is known about maternal smoking and brain tumor risk?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/7139628',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15191928',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24260161',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15801484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8470663',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12115571',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8850274',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11452935',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10620527'],\n",
       "   'ideal_answer': ['Findings regarding association of maternal smoking and brain tumor risk are mixed. It was shown that children of women who smoked during pregnancy had an increased incidence of brain tumors (hazard ratio = 1.24; 95% confidence interval: 1.01-1.53). The increase in risk was similar for benign and malignant tumors, and was most apparent for astrocytoma. However, other authors did not find association between maternal smoking and brain tumor risk.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012907',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:368',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D001932',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:1319',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D009035'],\n",
       "   'type': 'summary',\n",
       "   'id': '56c1d85fef6e394741000036',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 275,\n",
       "     'text': 'BACKGROUND: A recent meta-analysis suggested an association between exposure to paternal smoking during pregnancy and childhood brain tumor risk, but no studies have evaluated whether this association differs by polymorphisms in genes that metabolize tobacco-smoke chemicals.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24260161',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 225,\n",
       "     'text': 'OBJECTIVE: Studies of the effect of maternal smoking during pregnancy on development of brain tumors in the offspring generally have found no increase in risk but most have mainly relied on retrospective exposure assessment. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15801484',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 839,\n",
       "     'offsetInEndSection': 1092,\n",
       "     'text': 'Children of women who smoked during pregnancy had an increased incidence of brain tumors (hazard ratio = 1.24; 95% confidence interval: 1.01-1.53). The increase in risk was similar for benign and malignant tumors, and was most apparent for astrocytoma. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15801484',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1199,\n",
       "     'offsetInEndSection': 1321,\n",
       "     'text': 'CONCLUSIONS: These results support a role for maternal smoking during pregnancy in the etiology of childhood brain tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15801484',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1197,\n",
       "     'offsetInEndSection': 1360,\n",
       "     'text': 'Maternal occupational exposure to PAH before conception or during pregnancy was rare, and this exposure was not associated with any type of childhood brain tumor. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15191928',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 910,\n",
       "     'offsetInEndSection': 1185,\n",
       "     'text': \"There was no association between the risk of brain tumors in the child and parental smoking prior to pregnancy, maternal smoking or regular exposure to others' cigarette smoke during pregnancy at home or at work, or passive smoking by the child during the first year of life.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12115571',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 143,\n",
       "     'text': 'Maternal smoking during pregnancy and the risk of childhood brain tumors: a meta-analysis of 6566 subjects from twelve epidemiological studies.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 149,\n",
       "     'text': 'OBJECTIVE: Prior epidemiological studies suggest a possible association between maternal smoking during pregnancy and risk of childhood brain tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1485,\n",
       "     'offsetInEndSection': 1707,\n",
       "     'text': 'Pooling all twelve reports yielded an RRs of 1.05 (0.90-1.21), a non-statistically significant result suggesting no clear association between maternal smoking during pregnancy and risk of childhood brain tumor development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1911,\n",
       "     'offsetInEndSection': 2075,\n",
       "     'text': '.CONCLUSION: The available epidemiological data do not support a clear association between maternal smoking during pregnancy and pediatric brain tumor development. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 654,\n",
       "     'offsetInEndSection': 1024,\n",
       "     'text': 'We observed only few positive associations, namely, between CNS tumors and low birth weight [<2,500 g; odds ratio (OR), 1.73; 95% confidence interval (CI), 1.06-2.84], between ependymoma and maternal smoking during pregnancy (>10 cigarettes per day: OR, 4.71; 95% CI, 1.69-13.1), and between astrocytoma and exposure to wood preservatives (OR, 1.91; 95% CI, 1.22-3.01). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11452935',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 879,\n",
       "     'offsetInEndSection': 1068,\n",
       "     'text': 'The results on exposure to paternal tobacco smoke suggest an association with brain tumors (RR 1.22; CI, 1.05-1. 40; based on 10 studies) and lymphomas (RR 2.08; CI, 1.08-3.98; 4 studies). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10620527',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 850,\n",
       "     'offsetInEndSection': 1588,\n",
       "     'text': 'No association was found between the risk of CBTs and maternal or paternal smoking before pregnancy and there was no association between CBTs and maternal smoking during pregnancy [odds ratio (OR) = 0.98; 95% confidence interval (CI) = 0.72-1.3]. A slightly increased OR for CBTs was found for paternal smoking during pregnancy in the absence of maternal smoking (OR = 1.2; 95% CI = 0.90-1.5) and for maternal exposure to passive smoke from any source (OR = 1.2; 95% CI = 0.95-1.6). The results of this analysis are consistent with results from several prior epidemiological studies that showed no significant association between CBTs and maternal smoking before or during pregnancy or maternal exposure to passive smoke during pregnancy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8850274',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 359,\n",
       "     'offsetInEndSection': 1295,\n",
       "     'text': 'No significant differences in risks were found to be associated with maternal or paternal smoking at any time (odds ratio (OR) = 0.92 for mothers and 1.06 for fathers), during the year of birth of the child (which included both the prenatal and postnatal periods) (ORs = 0.84 for<1 pack/day and 1.0 for>or = 1 pack/day for mothers, and 0.68 for<1 pack/day and 1.07 for>or = 1 pack/day for fathers), or 2 years before the child was born, i.e., the pre-conception period (ORs = 0.75 for<1 pack/day and 1.01 for>or = 1 pack/day for mothers, and 0.90 for<1 pack/day and 1.15 for>or = 1 pack/day for fathers). Mothers were also specifically asked if they smoked during the pregnancy, and no association was found compared with never smokers (OR = 1.08, 95% confidence interval (CI) 0.80-1.45) or for ever-smokers who continued to smoke during pregnancy compared with those who stopped smoking during pregnancy (OR = 1.15, 95% CI 0.75-1.78). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8470663',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1489,\n",
       "     'offsetInEndSection': 1775,\n",
       "     'text': 'The lack of an effect of parental smoking was observed for both the major histologic types and locations of brain tumors. These findings and those from earlier studies provide no support for the hypothesis that parental cigarette smoking influences the risk of brain tumors in children.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8470663',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 334,\n",
       "     'offsetInEndSection': 758,\n",
       "     'text': 'Increased risk was associated with maternal contact with nitrosamine-containing substances such as burning incense (odds ratio, 3.3; p = 0.005), sidestream cigarette smoke (odds ratio, 1.5; p = 0.03), and face makeup (odds ratio, 1.6; p = 0.02); with maternal use of diuretics (odds ratio, 2.0; p = 0.03) and antihistamines (odds ratio, 3.4; p = 0.002); and with the level of maternal consumption of cured meats (p = 0.008).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7139628',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'Prior epidemiological studies suggest a possible association between maternal smoking during pregnancy and risk of childhood brain tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 213,\n",
       "     'text': 'Studies of the effect of maternal smoking during pregnancy on development of brain tumors in the offspring generally have found no increase in risk but most have mainly relied on retrospective exposure assessment.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15801484',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1092,\n",
       "     'offsetInEndSection': 1338,\n",
       "     'text': 'No association was found between the risk of CBTs and maternal or paternal smoking before pregnancy and there was no association between CBTs and maternal smoking during pregnancy [odds ratio (OR) = 0.98; 95% confidence interval (CI) = 0.72-1.3].',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8850274',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1176,\n",
       "     'offsetInEndSection': 1451,\n",
       "     'text': \"There was no association between the risk of brain tumors in the child and parental smoking prior to pregnancy, maternal smoking or regular exposure to others' cigarette smoke during pregnancy at home or at work, or passive smoking by the child during the first year of life.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12115571',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 879,\n",
       "     'offsetInEndSection': 1270,\n",
       "     'text': 'RESULTS: We found positive interaction odds ratios (ORs) for both maternal and paternal smoking during pregnancy, EPHX1 H139R, and childhood brain tumors (P(interaction)\\u200a=\\u200a0.02; 0.10), such that children with the high-risk (greater PAH activation) genotype were at a higher risk of brain tumors relative to children with the low-risk genotype when exposed to tobacco smoke during pregnancy. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24260161',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1311,\n",
       "     'offsetInEndSection': 1504,\n",
       "     'text': \"Finally, no significant increase in risk of brain tumors was found for the child's passive exposure to parental smoking during the period from birth to diagnosis of the brain tumor in the case.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8470663',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 139,\n",
       "     'text': 'Prior epidemiological studies suggest a possible association between maternal smoking during pregnancy and risk of childhood brain tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1532,\n",
       "     'offsetInEndSection': 1691,\n",
       "     'text': 'a non-statistically significant result suggesting no clear association between maternal smoking during pregnancy and risk of childhood brain tumor development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12125967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1176,\n",
       "     'offsetInEndSection': 1286,\n",
       "     'text': 'These results support a role for maternal smoking during pregnancy in the etiology of childhood brain tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15801484',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Global quantitative phosphoproteomic analyses are emerging. List the preferred technologies for the enrichment for phosphorylated peptides?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22345495',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20340162',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22369663',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20927383',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22499768',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21822884',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22499769',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23485197',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22906719',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17372656'],\n",
       "   'triples': [{'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0031684',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A0319731'}],\n",
       "   'ideal_answer': ['There are many different approaches to enrich for phosphorylated peptides: titanium dioxide, IMAC, simple derivatization through phosphoramidate chemistry and antibodies.'],\n",
       "   'exact_answer': [['enriched for phosphorylated peptides using titanium dioxide'],\n",
       "    ['sequential elution from IMAC'],\n",
       "    ['simple derivatization procedure based on phosphoramidate chemistry'],\n",
       "    ['antiphosphotyrosine antibodies']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D010748',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D020543',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D045163',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D040901'],\n",
       "   'type': 'list',\n",
       "   'id': '516281e6298dcd4e51000040',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 113,\n",
       "     'text': 'TiSH--a robust and sensitive global phosphoproteomics strategy employing a combination of TiO2, SIMAC, and HILIC.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22906719',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 429,\n",
       "     'offsetInEndSection': 740,\n",
       "     'text': 'An initial TiO(2) phosphopeptide pre-enrichment step is followed by post-fractionation using sequential elution from IMAC (SIMAC) to separate multi- and mono-phosphorylated peptides, and hydrophilic interaction liquid chromatography (HILIC) of the mono-phosphorylated peptides (collectively abbreviated \"TiSH\").',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22906719',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 618,\n",
       "     'offsetInEndSection': 729,\n",
       "     'text': 'After enrichment by anti-phosphotyrosine antibodies, we identified 29 potential novel c-Src substrate proteins.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22499769',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 579,\n",
       "     'offsetInEndSection': 845,\n",
       "     'text': 'Combining immunoprecipitation with an antiphosphotyrosine antibody, titanium dioxide phosphopeptide enrichment, isobaric tag for the relative and absolute quantitation methodology, and strong cation exchange separation, we were able to identify 2814 phosphopeptides.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22499768',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 443,\n",
       "     'offsetInEndSection': 710,\n",
       "     'text': 'Phosphopeptide fractionation by strong cation exchange chromatography combined with immobilized metal affinity chromatography (IMAC) enrichment enabled quantification of more than 8000 distinct phosphorylation sites in Ppt1 wild-type versus Ppt1-deficient yeast cells',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22369663',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 710,\n",
       "     'offsetInEndSection': 868,\n",
       "     'text': 'By employing titanium dioxide in addition to antiphosphotyrosine antibodies as enrichment methods, we identified 4164 phosphopeptides on 1670 phosphoproteins.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22345495',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 676,\n",
       "     'offsetInEndSection': 942,\n",
       "     'text': 'Our method combines the use of strong cation exchange (SCX) and titanium dioxide (TiO(2)) for phosphopeptide enrichment, high-resolution MS for peptide and protein identification, and stable isotope labeling by amino acids in cell culture (SILAC) for quantification.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21822884',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1007,\n",
       "     'offsetInEndSection': 1136,\n",
       "     'text': 'Phosphopeptides were isolated with high specificity through a simple derivatization procedure based on phosphoramidate chemistry.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17372656',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Which gene is mutated in a subtype of arrhythmogenic right ventricular cardiomyopathy known as Naxos disease?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18937352',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21880664',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16096717',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21789513',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12574890',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11691526',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15851108',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10902626',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17924338',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22315228'],\n",
       "   'ideal_answer': ['Identification of a deletion in plakoglobin in arrhythmogenic right ventricular cardiomyopathy with palmoplantar keratoderma and woolly hair (Naxos disease).Arrhythmogenic right ventricular cardiomyopathy (ARVC) is an inherited disorder associated with arrhythmias and sudden death. A recessive mutation in the gene encoding plakoglobin has been shown to cause Naxos disease, a cardiocutaneous syndrome characterized by ARVC and abnormalities of hair and skin.',\n",
       "    'Arrhythmogenic right ventricular cardiomyopathy (ARVC) is an inherited disorder associated with arrhythmias and sudden death. A recessive mutation in the gene encoding plakoglobin has been shown to cause Naxos disease, a cardiocutaneous syndrome characterized by ARVC and abnormalities of hair and skin.  ',\n",
       "    'A homozygous loss-of-function mutation of the Plakoglobin (Jup) gene, which encodes a major component of the desmosome and the adherens junction, had been identified in Naxos patients, although the underlying mechanism remained elusive.',\n",
       "    'A homozygous loss-of-function mutation of the Plakoglobin (Jup) gene, had been identified in Naxos Disease patients, a subset of ARVC, which is characterized by cutaneous disorder.'],\n",
       "   'exact_answer': ['The Plakoglobin gene', 'plakoglobin[jup]'],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:4',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D009202',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:0050431'],\n",
       "   'type': 'factoid',\n",
       "   'id': '54f9ae2506d9727f76000002',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 517,\n",
       "     'text': 'Arrhythmogenic right ventricular cardiomyopathy (ARVC) is an inheritable myocardial disorder associated with fibrofatty replacement of myocardium and ventricular arrhythmia. A subset of ARVC is categorized as Naxos disease, which is characterized by ARVC and a cutaneous disorder. A homozygous loss-of-function mutation of the Plakoglobin (Jup) gene, which encodes a major component of the desmosome and the adherens junction, had been identified in Naxos patients, although the underlying mechanism remained elusive.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21880664',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 181,\n",
       "     'text': 'Loss-of-function mutation of Jup has been associated with Naxos disease, which is characterized by arrhythmogenic cardiomyopathy and the cutaneous disorder palmoplantar keratoderma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22315228',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 182,\n",
       "     'offsetInEndSection': 337,\n",
       "     'text': 'Previously, we have shown that genetic ablation of Jup in cardiomyocytes in mice leads to arrhythmogenic cardiomyopathy similar to Naxos disease in humans.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22315228',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 309,\n",
       "     'offsetInEndSection': 520,\n",
       "     'text': 'As similar phenotypes have been described in Naxos disease and Carvajal syndrome, respectively, the genes for plakoglobin (JUP) and desmoplakin (DSP) were screened for mutations using direct genomic sequencing. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21789513',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 396,\n",
       "     'offsetInEndSection': 554,\n",
       "     'text': 'One is a C-terminal mutation causing Naxos disease, a recessive syndrome of arrhythmogenic right ventricular cardiomyopathy (ARVC) and abnormal skin and hair.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18937352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 271,\n",
       "     'offsetInEndSection': 394,\n",
       "     'text': 'In this study, we examined the effects of two different mutations in plakoglobin on cell migration, stiffness, and adhesion',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18937352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 303,\n",
       "     'text': 'Arrhythmogenic right ventricular cardiomyopathy (ARVC) is an inherited disorder associated with arrhythmias and sudden death. A recessive mutation in the gene encoding plakoglobin has been shown to cause Naxos disease, a cardiocutaneous syndrome characterized by ARVC and abnormalities of hair and skin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17924338',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1328,\n",
       "     'offsetInEndSection': 1517,\n",
       "     'text': 'Three other genes implicated for ARVC, plakoglobin (Naxos disease), desmoplakin (ARVC8) and plakophilin (ARVC9) have prompted the speculation that ARVC is primarily a disease of desmosomes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16096717',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 271,\n",
       "     'text': 'We tested the hypothesis that defective interactions between adhesion junctions and the cytoskeleton caused by the plakoglobin mutation in Naxos disease lead to remodeling of gap junctions and altered expression of the major gap junction protein, connexin43. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15851108',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 283,\n",
       "     'offsetInEndSection': 568,\n",
       "     'text': 'Naxos disease, a recessive form of arrhythmogenic right ventricular cardiomyopathy, is associated with a high incidence of arrhythmias and sudden cardiac death. Naxos disease is caused by a mutation in plakoglobin, a protein that links cell-cell adhesion molecules to the cytoskeleton.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15851108',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1463,\n",
       "     'offsetInEndSection': 1736,\n",
       "     'text': 'However, in Naxos disease, a recessive form of ARVC with coexpression of palmoplantar keratoderma and woolly hair, a mutation in the plakoglobin gene has recently been discovered, thus underscoring the potential role of genetic alterations in cytoskeletal proteins in ARVC.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12574890',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 274,\n",
       "     'offsetInEndSection': 490,\n",
       "     'text': 'Naxos disease is a recessively inherited ARVC caused by a mutation in the gene encoding plakoglobin (cell adhesion protein) in which the cardiac phenotype is associated with palmoplantar keratoderma and woolly hair. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11691526',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1545,\n",
       "     'offsetInEndSection': 1734,\n",
       "     'text': 'Autosomal recessive ARVC caused by a mutation in plakoglobin was 100% penetrant by adolescence. Affected subjects who were homozygous experienced progressive disease with adverse prognosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11691526',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 546,\n",
       "     'text': 'Arrhythmogenic right ventricular cardiomyopathy (ARVC) is an autosomal dominant heart muscle disorder that causes arrhythmia, heart failure, and sudden death. Previously we mapped the genetic locus for the triad of autosomal recessive ARVC, palmoplantar keratoderma, and woolly hair (Naxos disease) to chromosome 17q21, in which the gene for plakoglobin is encoded. This protein is a key component of desmosomes and adherens junctions, and is important for the tight adhesion of many cell types, including those in the heart and skin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10902626',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 157,\n",
       "     'text': 'Identification of a deletion in plakoglobin in arrhythmogenic right ventricular cardiomyopathy with palmoplantar keratoderma and woolly hair (Naxos disease).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10902626',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 136,\n",
       "     'text': 'Remodeling of myocyte gap junctions in arrhythmogenic right ventricular cardiomyopathy due to a deletion in plakoglobin (Naxos disease).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15851108',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Do selenoproteins and selenium play a role in prostate cancer prevention?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/19074884',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23133653',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19299660',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19690186',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20852007',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20424130',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22072582',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15875088',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16690748',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17160069'],\n",
       "   'ideal_answer': ['No, although initial epidemiological studies on humans and on animal and cell- based models indicated that selenoproteins may be protecting against prostate cancer, more research is needed to improve the understanding of selenium metabolism and requirements for optimal health.'],\n",
       "   'exact_answer': 'no',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D051140',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:10283'],\n",
       "   'type': 'yesno',\n",
       "   'id': '515ed87c298dcd4e51000032',\n",
       "   'snippets': [{'offsetInBeginSection': 1115,\n",
       "     'offsetInEndSection': 1402,\n",
       "     'text': 'The selenoprotein-deficient mice exhibited accelerated development of lesions associated with prostate cancer progression, implicating selenoproteins in cancer risk and development and raising the possibility that selenium prevents cancer by modulating the levels of these selenoproteins',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16690748',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1293,\n",
       "     'offsetInEndSection': 1564,\n",
       "     'text': 'Notably and in contrast to previous studies, RWPE-1 cells were significantly more sensitive to selenite than either of the prostate cancer cell lines. These results demonstrate that selenoproteins and selenium metabolism are regulated at multiple levels in prostate cells',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17160069',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1783,\n",
       "     'offsetInEndSection': 2110,\n",
       "     'text': 'In a low-selenium population, SOD2-Ala16+ men homozygous for SEPP1-Ala234 are at an increased risk of prostate cancer/aggressive prostate cancer especially if ever-smokers, because they are likely to produce more mitochondrial H(2)O(2) that they cannot remove, thereby promoting prostate tumor cell proliferation and migration.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19074884',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1417,\n",
       "     'offsetInEndSection': 1566,\n",
       "     'text': 'Our results support a role of selenium and polymorphisms in selenoenzymes in prostate cancer etiology, which warrants confirmation in future studies.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20852007',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1470,\n",
       "     'offsetInEndSection': 1767,\n",
       "     'text': 'This study provides evidence that SEP15 genetic variation may influence PCa mortality. Additionally, the association of selenium with PCa mortality was modified by a variant, suggesting the possibility that some men with PCa may benefit more from selenium than others, depending on their genotype.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20424130',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1619,\n",
       "     'offsetInEndSection': 1752,\n",
       "     'text': 'We conclude that decreased SEPP concentration in serum might represent an additional valuable marker for prostate cancer diagnostics.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19690186',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 289,\n",
       "     'text': 'The recently completed Selenium and Vitamin E Cancer Prevention Trial (SELECT) was one of the largest human cancer prevention trials ever undertaken. Its purpose was to assess the role of selenium and vitamin E in prostate cancer prevention, but SELECT found no decline in prostate cancer.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19299660',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 325,\n",
       "     'offsetInEndSection': 928,\n",
       "     'text': \"We studied Se levels in whole blood, plasma and prostate of 32 PC and 40 benign prostate hyperplasia (BPH) patients and in the control group composed of 39 healthy subjects. The selenoenzyme glutathione peroxidase (GSH-Px) was also measured in the patients' red cells, plasma and prostate tissue. Se concentration in whole blood and plasma in both groups of patients was lower as compared with controls, while in prostate gland it was significantly higher in PC than in BPH patients and controls. Red cell GSH-Px activity was the same in PC patients and controls but significantly lower in BPH patients.\",\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15875088',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1352,\n",
       "     'offsetInEndSection': 1553,\n",
       "     'text': 'Of particular interest was the positive correlation between tissue GPx activity and Gleason score, with this relationship achieving statistical significance among African-Americans (r\\u2009=\\u20090.67, P\\u2009=\\u20090.02)',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22072582',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Mutations of which genes have been associated with Catecholaminergic Polymorphic Ventricular Tachycardia (CPVT)?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/20301466',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22422768',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24370574',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23595086',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20807279',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22787013',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22589293',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16601229',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25480325',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23908839'],\n",
       "   'ideal_answer': ['Mutations in five genes – ryanodine receptor 2 (RYR2), calsequestrin 2(CASQ2), triadic (TRDN), calmodulin 1 (CALM1) and potassium channel, inwardly rectifying subfamily J, member 2 (KCNJ2)  – have been found to be associated with CPVT'],\n",
       "   'exact_answer': [['ryanodine receptor 2', 'RYR2'],\n",
       "    ['calsequestrin 2', 'CASQ2'],\n",
       "    ['triadin', 'TRDN'],\n",
       "    ['calmodulin 1', 'CALM1'],\n",
       "    ['potassium channel, inwardly rectifying subfamily J, member 2', 'KCNJ2']],\n",
       "   'type': 'list',\n",
       "   'id': '54c225bcf693c3b16b000002',\n",
       "   'snippets': [{'offsetInBeginSection': 1703,\n",
       "     'offsetInEndSection': 1921,\n",
       "     'text': 'Mutation in four genes – RYR2, CASQ2, TRDN, and CALM1 – is known to cause CPVT or related phenotypes of adrenergically induced life-threatening arrhythmias. The presence of other as-yet unidentified loci is postulated ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20301466',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 239,\n",
       "     'offsetInEndSection': 384,\n",
       "     'text': 'some KCNJ2 mutation carriers lack the ATS triad and sometimes share the phenotype of catecholaminergic polymorphic ventricular tachycardia (CPVT)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22589293',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 246,\n",
       "     'offsetInEndSection': 513,\n",
       "     'text': 'In 50 Japanese probands from unrelated families who satisfied clinical criteria for CPVT, genetic testing was conducted in all exons on 3 CPVT-related genes: cardiac ryanodine receptor 2 (RYR2), calsequestrin 2 (CASQ2) and inward rectifier potassium channel 2 (KCNJ2)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23595086',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 621,\n",
       "     'offsetInEndSection': 947,\n",
       "     'text': 'point mutations in RYR2, the gene encoding for the cardiac isoform of the RyR (RyR2), are associated with catecholaminergic polymorphic ventricular tachycardia (CPVT), an arrhythmogenic syndrome characterized by the development of adrenergically-mediated ventricular tachycardia in individuals with an apparently normal heart.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25480325',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 464,\n",
       "     'offsetInEndSection': 528,\n",
       "     'text': 'Recessive CPVT variants are due to mutations in the CASQ2 gene. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24370574',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 367,\n",
       "     'offsetInEndSection': 507,\n",
       "     'text': 'Recent studies have shown that CPVT is caused by mutations in the cardiac ryanodine receptor type 2 (RyR2) or calsequestrin 2 (CASQ2) genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23908839',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 689,\n",
       "     'offsetInEndSection': 817,\n",
       "     'text': 'We identified three mutations in triadin which cosegregated with the disease on a recessive mode of transmission in two families',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22422768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 206,\n",
       "     'text': 'Catecholaminergic polymorphic ventricular tachycardia (CPVT) is a familial arrhythmogenic disorder associated with mutations in the cardiac ryanodine receptor (RyR2) and cardiac calsequestrin (CASQ2) genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16601229',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 209,\n",
       "     'text': 'BACKGROUND: Catecholaminergic polymorphic ventricular tachycardia (CPVT) is an inherited arrhythmia syndrome associated with mutations in the cardiac ryanodine receptor gene (Ryr2) in the majority of patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22787013',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 238,\n",
       "     'text': 'BACKGROUND: Catecholaminergic polymorphic ventricular tachycardia (CPVT) is a familial arrhythmic syndrome caused by mutations in genes encoding the calcium-regulation proteins cardiac ryanodine receptor (RyR2) or calsequestrin-2 (CASQ2).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20807279',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 212,\n",
       "     'text': 'Catecholaminergic polymorphic ventricular tachycardia (CPVT) is an inherited arrhythmogenic disease so far related to mutations in the cardiac ryanodine receptor (RYR2) or the cardiac calsequestrin (CASQ2) genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22422768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 196,\n",
       "     'text': 'Catecholaminergic polymorphic ventricular tachycardia (CPVT) is an inherited arrhythmia syndrome associated with mutations in the cardiac ryanodine receptor gene (Ryr2) in the majority of patients',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22787013',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is selenium deficiency involved in autoimmune thyroid disease?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/15132715',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22009156',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20810577',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21896670',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23448365',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19034261',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11932302',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17160166',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23565426',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23786024'],\n",
       "   'ideal_answer': ['Selenium deficiency is likely to constitute a risk factor for a feedforward derangement of the immune system-thyroid interaction, while selenium supplementation appears to dampen the self-amplifying nature of this derailed interactionIn areas with severe selenium deficiency higher incidence of thyroiditis has been reported due to a decreased activity of selenium-dependent glutathione peroxidase enzyme within thyroid cells',\n",
       "    'The essential trace element selenium was recently recognized as being incorporated as selenocysteine in all three deiodinases of the thyroid gland. This has decisively confirmed the clear-cut link between selenium and thyroid function. Additionally, it has been established that the thyroid contains more selenium than any other tissue, and that selenium deficiency aggravates the manifestation of autoimmune thyroid disease.',\n",
       "    'The recent recognition that the essential trace element selenium is incorporated as selenocysteine in all three deiodinases has decisively confirmed the clear-cut link between selenium and thyroid function. It has additionally been established that the thyroid contains more selenium than any other tissue and that selenium deficiency aggravates the manifestation of endemic myxedematous cretinism and autoimmune thyroid disease  '],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D013967',\n",
       "    'http://www.biosemantics.org/jochem#4278041',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:7188'],\n",
       "   'type': 'yesno',\n",
       "   'id': '550c1ca3a103b78016000003',\n",
       "   'snippets': [{'offsetInBeginSection': 14,\n",
       "     'offsetInEndSection': 205,\n",
       "     'text': 'In areas with severe selenium deficiency higher incidence of thyroiditis has been reported due to a decreased activity of selenium-dependent glutathione peroxidase enzyme within thyroid cells',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23565426',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1028,\n",
       "     'offsetInEndSection': 1355,\n",
       "     'text': 'Of 30 patients in the selenium treated group, 6 patients were overtly hypothyroid, 15 were subclinical hypothyroid, 6 were euthyroid, and 3 were subclinical hyperthyroid. The mean TPOAb concentration decreased significantly by 49.5% (P < 0.013) in the selenium treated group versus 10.1% (P < 0.95) in the placebo-treated group',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23565426',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1369,\n",
       "     'offsetInEndSection': 1479,\n",
       "     'text': 'Selenium substitution has a significant impact on inflammatory activity in thyroid-specific autoimmune disease',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23565426',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 56,\n",
       "     'text': \"Serum selenium is low in newly diagnosed Graves' disease\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23448365',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 678,\n",
       "     'offsetInEndSection': 801,\n",
       "     'text': 'S-Se was lower in patients with GD than in controls (mean (SD), GD: 89·9\\xa0μg/l (18·4); controls: 98·8\\xa0μg/l (19·7), P\\xa0<\\xa00·01)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23448365',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1371,\n",
       "     'offsetInEndSection': 1603,\n",
       "     'text': 'Patients with newly diagnosed GD and AIH had significantly lower s-Se compared with random controls. Our observation supports the postulated link between inadequate selenium supply and overt autoimmune thyroid disease, especially GD',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23448365',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1618,\n",
       "     'offsetInEndSection': 1852,\n",
       "     'text': 'Selenium deficiency is likely to constitute a risk factor for a feedforward derangement of the immune system-thyroid interaction, while selenium supplementation appears to dampen the self-amplifying nature of this derailed interaction',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22009156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 9,\n",
       "     'offsetInEndSection': 437,\n",
       "     'text': 'The recent recognition that the essential trace element selenium is incorporated as selenocysteine in all three deiodinases has decisively confirmed the clear-cut link between selenium and thyroid function. It has additionally been established that the thyroid contains more selenium than any other tissue and that selenium deficiency aggravates the manifestation of endemic myxedematous cretinism and autoimmune thyroid disease',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20810577',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1669,\n",
       "     'offsetInEndSection': 1834,\n",
       "     'text': 'Maintenance of \"selenostasis\" via optimal intake not only aids preservation of general health but also contributes substantially to the prevention of thyroid disease',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20810577',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 520,\n",
       "     'offsetInEndSection': 844,\n",
       "     'text': 'Low birth weight, iodine excess and deficiency, selenium deficiency, parity, oral contraceptive use, reproductive span, fetal microchimerism, stress, seasonal variation, allergy, smoking, radiation damage to the thyroid gland, viral and bacterial infections all play a role in the development of autoimmune thyroid disorders',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15132715',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 205,\n",
       "     'offsetInEndSection': 428,\n",
       "     'text': 'It has additionally been established that the thyroid contains more selenium than any other tissue and that selenium deficiency aggravates the manifestation of endemic myxedematous cretinism and autoimmune thyroid disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20810577',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 111,\n",
       "     'text': 'Selenium deficiency may play an important role in the initiation and progression of autoimmune thyroid disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23448365',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': '[Selenium deficiency in celiac disease: risk of autoimmune thyroid diseases].',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19034261',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 516,\n",
       "     'offsetInEndSection': 842,\n",
       "     'text': 'Low birth weight, iodine excess and deficiency, selenium deficiency, parity, oral contraceptive use, reproductive span, fetal microchimerism, stress, seasonal variation, allergy, smoking, radiation damage to the thyroid gland, viral and bacterial infections all play a role in the development of autoimmune thyroid disorders.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15132715',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1083,\n",
       "     'offsetInEndSection': 1304,\n",
       "     'text': 'Some clinical studies have demonstrated that selenium-deficient patients with autoimmune thyroid disease benefit from selenium supplementation, although the data are conflicting and many parameters must still be defined.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22009156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1356,\n",
       "     'offsetInEndSection': 1489,\n",
       "     'text': 'Our observation supports the postulated link between inadequate selenium supply and overt autoimmune thyroid disease, especially GD.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23448365',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'Selenium deficiency in celiac disease: risk of autoimmune thyroid diseases].',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19034261',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 184,\n",
       "     'text': 'Selenoproteins contain the essential trace element selenium whose deficiency leads to major disorders including cancer, male reproductive system failure, or autoimmune thyroid disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21896670',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 216,\n",
       "     'offsetInEndSection': 438,\n",
       "     'text': 'It has additionally been established that the thyroid contains more selenium than any other tissue and that selenium deficiency aggravates the manifestation of endemic myxedematous cretinism and autoimmune thyroid disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20810577',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 920,\n",
       "     'offsetInEndSection': 1222,\n",
       "     'text': 'EVIDENCE SYNTHESIS: Evidence in support of selenium supplementation in thyroid autoimmune disease is evaluated, the results herein presented demonstrating the potential effectiveness of selenium in reducing the antithyroid peroxidase titer and improving the echostructure in the ultrasound examination.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20810577',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1139,\n",
       "     'offsetInEndSection': 1288,\n",
       "     'text': \"Some investigators suggest that selenium may be a useful adjunctive treatment for autoimmune thyroid diseases, such as Hashimoto and Graves' disease.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17160166',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 273,\n",
       "     'offsetInEndSection': 395,\n",
       "     'text': 'Therefore, even mild selenium deficiency may contribute to the development and maintenance of autoimmune thyroid diseases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11932302',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1091,\n",
       "     'offsetInEndSection': 1311,\n",
       "     'text': 'Some clinical studies have demonstrated that selenium-deficient patients with autoimmune thyroid disease benefit from selenium supplementation, although the data are conflicting and many parameters must still be defined.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22009156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1090,\n",
       "     'offsetInEndSection': 1309,\n",
       "     'text': 'Some clinical studies have demonstrated that selenium-deficient patients with autoimmune thyroid disease benefit from selenium supplementation, although the data are conflicting and many parameters must still be defined',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22009156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 145,\n",
       "     'text': 'High prevalence of hyperplastic and autoimmune diseases of thyroid in Ukrainian population is determined by endemic deficit of iodine and selenium',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23786024',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 207,\n",
       "     'offsetInEndSection': 429,\n",
       "     'text': 'It has additionally been established that the thyroid contains more selenium than any other tissue and that selenium deficiency aggravates the manifestation of endemic myxedematous cretinism and autoimmune thyroid disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20810577',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 273,\n",
       "     'offsetInEndSection': 394,\n",
       "     'text': 'Therefore, even mild selenium deficiency may contribute to the development and maintenance of autoimmune thyroid diseases',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11932302',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1139,\n",
       "     'offsetInEndSection': 1287,\n",
       "     'text': \"Some investigators suggest that selenium may be a useful adjunctive treatment for autoimmune thyroid diseases, such as Hashimoto and Graves' disease\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17160166',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which deiodinases are best known to be present in brain?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22294745',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16728541',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8663170',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7768329',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10426574',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22719854',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12072404',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8444882',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15062548',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1417847'],\n",
       "   'ideal_answer': ['All the 3 deiodinases (Type 1, Type 2 and Type 3 deiodinase) are present in the \"brain\" but Type 1 deiodinase  is only found in neurohypophysis that cannot be actually considered true \"brain tissue\".'],\n",
       "   'exact_answer': [['Type 2 deiodinase'], ['Type 3 deiodinase']],\n",
       "   'type': 'list',\n",
       "   'id': '517a8bab8ed59a060a000040',\n",
       "   'snippets': [{'offsetInBeginSection': 181,\n",
       "     'offsetInEndSection': 307,\n",
       "     'text': 'In brain, the presence of type III iodothyronine deiodinase (D3) seems to be important to maintain homeostasis of T(3) levels.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12072404',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1502,\n",
       "     'offsetInEndSection': 1750,\n",
       "     'text': 'We were able to detect a protein fragment corresponding to the expected molecular mass (30 kDa) for type III deiodinase by means of Western blot analysis. RT-PCR as well as Northern blot analysis confirmed the presence of D3 mRNA in the cerebellum.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12072404',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 345,\n",
       "     'offsetInEndSection': 583,\n",
       "     'text': \"The presence of 5D (type-III) together with our previous report of 5'-deiodinase (type-I in euthyroidism and type-II in hypothyroidism) shows that the isozymes of deiodinases in the neurohypophysis are quite similar to those in the brain.\",\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1417847',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Which enzymes are involved in global genome nucleotide excision repair (GG-NER) in bacteria?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/20004108',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19617358',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19933360',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24290807',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23963890',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23245695',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21145481',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19762288',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22559785',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25463394'],\n",
       "   'ideal_answer': ['Nucleotide excision repair (NER) is universally used to recognize and remove many types of DNA damage. In eubacteria, the NER system typically consists of UvrA, UvrB, UvrC, the UvrD helicase, DNA polymerase I, and ligase.  Damage recognition during bacterial NER depends upon UvrA, which binds to the damage and loads UvrB onto the DNA. Subsequently, UvrA, UvrB and UvrC form the excinuclease protein UvrABC endonuclease, a multi-enzymatic complex which carries out repair of damaged DNA in sequential manner. In some cases, Cho may be the effective nuclease for NER, rather than UvrC. UvrC nuclease and the short oligonucleotide that contains the DNA lesion are removed from the post-incision complex by UvrD, a superfamily 1A helicase. In gram-positive organisms, PcrA helicase can also displace UvrC and the excised oligonucleotide from a post-incision NER complex.'],\n",
       "   'exact_answer': [['UvrA'],\n",
       "    ['UvrB'],\n",
       "    ['UvrC nuclease'],\n",
       "    ['UvrD 1A helicase'],\n",
       "    ['DNA polymerase I'],\n",
       "    ['Ligase'],\n",
       "    ['Cho nuclease (instead of UvrC)'],\n",
       "    ['PcrA helicase (instead of Uvrd, in gram-positive bacteria)']],\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0000720',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0000109',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006289',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0070911',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D004260'],\n",
       "   'type': 'list',\n",
       "   'id': '5547a7e9f35db75526000006',\n",
       "   'snippets': [{'offsetInBeginSection': 169,\n",
       "     'offsetInEndSection': 455,\n",
       "     'text': 'we evaluated the in vivo role of NER in the repair of DNA adducts generated by psoralens (mono- or bi-functional) and UV-A light (PUVA) in E. coli. Cultures of wild-type E. coli K12 and mutants for uvrA, uvrB, uvrC or uvrAC genes were treated with PUVA and cell survival was determined.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20004108',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 221,\n",
       "     'text': 'Nucleotide excision repair (NER) is universally used to recognize and remove many types of DNA damage. In eubacteria, the NER system typically consists of UvrA, UvrB, UvrC, the UvrD helicase, DNA polymerase I, and ligase.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19933360',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 207,\n",
       "     'text': 'During nucleotide excision repair (NER) in bacteria the UvrC nuclease and the short oligonucleotide that contains the DNA lesion are removed from the post-incision complex by UvrD, a superfamily 1A helicase.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19762288',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 349,\n",
       "     'offsetInEndSection': 416,\n",
       "     'text': 'UvrD interacts with UvrB, a component of the post-incision complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19762288',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1162,\n",
       "     'offsetInEndSection': 1400,\n",
       "     'text': 'PcrA helicase from Bacillus stearothermophilus can also displace UvrC and the excised oligonucleotide from a post-incision NER complex, which supports the idea that PcrA performs a UvrD-like function during NER in gram-positive organisms.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19762288',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 957,\n",
       "     'offsetInEndSection': 1032,\n",
       "     'text': 'Cho rather than UvrC seems to be an effective nuclease for the NER of DPCs.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19617358',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 312,\n",
       "     'offsetInEndSection': 478,\n",
       "     'text': 'Nucleotide excision repair consists of excinuclease protein UvrABC endonuclease, multi-enzymatic complex which carries out repair of damaged DNA in sequential manner.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23963890',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 223,\n",
       "     'offsetInEndSection': 502,\n",
       "     'text': 'Escherichia coli K12 mutant strains deficient in nucleotide excision repair (NER) were submitted to increasing concentrations of cisplatin, and the results revealed that uvrA and uvrB mutants are sensitive to this agent, while uvrC and cho mutants remain as the wild type strain.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23245695',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 401,\n",
       "     'offsetInEndSection': 1194,\n",
       "     'text': 'we used a genetic approach to investigate the roles of nucleotide excision repair (NER) pathway components in H. pylori mutation and recombination. RESULTS: Inactivation of any of the four uvr genes strongly increased the susceptibility of H. pylori to DNA damage by ultraviolet light. Inactivation of uvrA and uvrB significantly decreased mutation frequencies whereas only the uvrA deficient mutant exhibited a significant decrease of the recombination frequency after natural transformation. A uvrC mutant did not show significant changes in mutation or recombination rates; however, inactivation of uvrC promoted the incorporation of significantly longer fragments of donor DNA (2.2-fold increase) into the recipient chromosome. A deletion of uvrD induced a hyper-recombinational phenotype.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22559785',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 586,\n",
       "     'text': 'Transcription-coupled DNA repair (TCR) is a subpathway of nucleotide excision repair (NER) that is triggered when RNA polymerase is stalled by DNA damage. Lesions targeted by TCR are repaired more quickly than lesions repaired by the transcription-independent \"global\" NER pathway, but the mechanism underlying this rate enhancement is not understood. Damage recognition during bacterial NER depends upon UvrA, which binds to the damage and loads UvrB onto the DNA. Bacterial TCR additionally requires the Mfd protein, a DNA translocase that removes the stalled transcription complexes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21145481',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is DNA methylation an epigenetic modification of chromatin related to gene expression?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25512710',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25514241',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25534027',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22359296',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22078060',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25437055',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25531272',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22922032',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23604474',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22992936'],\n",
       "   'ideal_answer': ['Epigenetic changes such as DNA methylation alter gene expression at the level of transcription by upregulating, downregulating, or silencing genes completely.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0010467',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0006306',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0000785'],\n",
       "   'type': 'yesno',\n",
       "   'id': '54d9f7894b1fd0d33c00000a',\n",
       "   'snippets': [{'offsetInBeginSection': 21,\n",
       "     'offsetInEndSection': 164,\n",
       "     'text': 'DNA methylation is a chemical modification of DNA involved in the regulation of gene expression by controlling the access to the DNA sequence. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25531272',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 163,\n",
       "     'text': 'Epigenetic marks such as DNA methylation play important biological roles in gene expression regulation and cellular differentiation during development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25534027',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 196,\n",
       "     'text': 'DNA methylation patterns are characterized by highly conserved developmental programs, but allow for divergent gene expression resulting from stochastic epigenetic drift or divergent environments.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25437055',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 76,\n",
       "     'text': 'DNA methylation plays a critical role in the regulation of gene expression. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25514241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 198,\n",
       "     'text': 'Epigenetic changes such as DNA methylation and histone methylation and acetylation alter gene expression at the level of transcription by upregulating, downregulating, or silencing genes completely.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25512710',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 199,\n",
       "     'offsetInEndSection': 358,\n",
       "     'text': 'Dysregulation of epigenetic events can be pathological, leading to cardiovascular disease, neurological disorders, metabolic disorders, and cancer development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25512710',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 122,\n",
       "     'text': 'DNA methylation is a pervasive epigenetic DNA modification that strongly affects chromatin regulation and gene expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22922032',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'Epigenetic control, which includes DNA methylation and histone modifications, leads to chromatin remodeling and regulated gene expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23604474',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 240,\n",
       "     'text': 'Epigenetic modifications on the DNA sequence (DNA methylation) or on chromatin-associated proteins (i.e., histones) comprise the \"cellular epigenome\"; together these modifications play an important role in the regulation of gene expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22992936',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 136,\n",
       "     'offsetInEndSection': 426,\n",
       "     'text': 'Epigenetics is a process involved in gene regulation, mediated via DNA methylation, histone modification, chromatin remodeling, and functional noncoding RNAs, which influences the accessibility of the underlying DNA to transcriptional regulatory factors that activate or repress expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22359296',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 101,\n",
       "     'offsetInEndSection': 322,\n",
       "     'text': 'Significant progress has been made in the basic understanding of how various epigenetic changes such as DNA methylation, histone modification, miRNA expression, and higher order chromatin structure affect gene expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22078060',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Does Chromatin Immunoprecipitation (ChIP) show a bias for highly expressed loci?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24173036',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17477493',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22248020',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22870189',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24255646',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21934668',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22499706',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17592629',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22522655',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21554688'],\n",
       "   'ideal_answer': ['Several issues in the processing and analysis of ChIP-chip data have not been resolved fully, including the effect of background (mock control) subtraction and normalization within and across arrays. We detected a chromatin-state bias: open chromatin regions yielded higher coverage, which led to false positives if not corrected. The localization of unrelated proteins, including the entire silencing complex, to the most highly transcribed genes was highly suggestive of a technical issue with the immunoprecipitations.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0010467',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D020224',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D059467',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D047369'],\n",
       "   'type': 'yesno',\n",
       "   'id': '52ef6da1c8da898910000011',\n",
       "   'snippets': [{'offsetInBeginSection': 179,\n",
       "     'offsetInEndSection': 386,\n",
       "     'text': 'However, several issues in the processing and analysis of ChIP-chip data have not been resolved fully, including the effect of background (mock control) subtraction and normalization within and across arrays',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17592629',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1444,\n",
       "     'offsetInEndSection': 1695,\n",
       "     'text': ' Proper normalization is essential for ChIP-chip experiments. The proposed normalization technique can correct systematic errors and compensate for the lack of mock control data, thus reducing the experimental cost and producing more accurate results.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17592629',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 930,\n",
       "     'offsetInEndSection': 1164,\n",
       "     'text': 'Subtraction of the mock (non-specific antibody or no antibody) control data is generally needed to eliminate the bias, but appropriate normalization obviates the need for mock experiments and increases the correlation among replicates',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17592629',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1718,\n",
       "     'offsetInEndSection': 1839,\n",
       "     'text': 'The proposed method can handle several control samples allowing for correction of multiple sources of bias simultaneously',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22248020',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 141,\n",
       "     'offsetInEndSection': 263,\n",
       "     'text': 'However, the data generated will always contain noise due to e.g. repetitive regions or non-specific antibody interactions',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22248020',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 320,\n",
       "     'offsetInEndSection': 466,\n",
       "     'text': 'The generation of high copy numbers of DNA fragments as an artifact of the PCR step in ChIP-seq is an important source of bias of this methodology',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21554688',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 510,\n",
       "     'offsetInEndSection': 671,\n",
       "     'text': 'Here we describe several technical aspects of the ChIP-Seq assay that diminish bias and background noise and allow the consistent generation of high-quality data',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21934668',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 622,\n",
       "     'offsetInEndSection': 838,\n",
       "     'text': ' This theoretical paper systematically characterizes the biases and properties of ChIP-seq data by comparing 62 separate publicly available datasets, using rigorous statistical models and signal processing techniques',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22499706',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 341,\n",
       "     'offsetInEndSection': 470,\n",
       "     'text': 'We detected a chromatin-state bias: open chromatin regions yielded higher coverage, which led to false positives if not corrected',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22522655',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 472,\n",
       "     'offsetInEndSection': 558,\n",
       "     'text': 'This bias had a greater effect on detection specificity than any base-composition bias',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22522655',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 294,\n",
       "     'offsetInEndSection': 442,\n",
       "     'text': 'This problem turns out to be surprisingly difficult, even in simple pairwise comparisons, because of the significant level of noise in ChIP-seq data',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22870189',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 664,\n",
       "     'offsetInEndSection': 787,\n",
       "     'text': 'We show that the ChIPnorm method removes most of the noise and bias in the data and outperforms other normalization methods',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22870189',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 281,\n",
       "     'offsetInEndSection': 498,\n",
       "     'text': 'We investigated the impact of library amplification bias on the identification of allele-specific (AS) molecular events from high-throughput sequencing data derived from chromatin immunoprecipitation assays (ChIP-seq)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24255646',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 675,\n",
       "     'offsetInEndSection': 938,\n",
       "     'text': 'The 238 loci, termed \"hyper-ChIPable\", were in highly expressed regions with strong polymerase II and polymerase III enrichment signals, and the correlation between transcription level and ChIP enrichment was not limited to these 238 loci but extended genome-wide',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24173036',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1122,\n",
       "     'offsetInEndSection': 1311,\n",
       "     'text': 'The localization of unrelated proteins, including the entire silencing complex, to the most highly transcribed genes was highly suggestive of a technical issue with the immunoprecipitations',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24173036',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can RG7112 inhibit MDM2?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23400593',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23808545',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24900694',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22753001',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23084521',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24869939',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24309210',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21391905',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25082860',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24812409'],\n",
       "   'ideal_answer': ['Yes, RG7112 is a small molecule MDM2 antagonist.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/MDM2_DANRE',\n",
       "    'http://www.uniprot.org/uniprot/MDM2_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MDM2_MESAU',\n",
       "    'http://www.uniprot.org/uniprot/MDM2_CANLF',\n",
       "    'http://www.uniprot.org/uniprot/MDM2_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MDM2_FELCA',\n",
       "    'http://www.uniprot.org/uniprot/MDM2_XENLA'],\n",
       "   'type': 'yesno',\n",
       "   'id': '56ed08062ac5ed1459000005',\n",
       "   'snippets': [{'offsetInBeginSection': 86,\n",
       "     'offsetInEndSection': 330,\n",
       "     'text': 'To assess the influence of the p53 regulatory pathway further, we studied the effect of RG7112, a small molecule MDM2 antagonist that activates p53 by preventing its interaction with MDM2, on normal megakaryocytopoiesis and platelet production.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24309210',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 609,\n",
       "     'offsetInEndSection': 723,\n",
       "     'text': 'RG7112 (2g) is the first clinical small-molecule MDM2 inhibitor designed to occupy the p53-binding pocket of MDM2.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24900694',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 182,\n",
       "     'text': 'Effect of the MDM2 antagonist RG7112 on the P53 pathway in patients with MDM2-amplified, well-differentiated or dedifferentiated liposarcoma: an exploratory proof-of-mechanism study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23084521',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 237,\n",
       "     'text': 'We report a proof-of-mechanism study of RG7112, a small-molecule MDM2 antagonist, in patients with chemotherapy-naive primary or relapsed well-differentiated or dedifferentiated MDM2-amplified liposarcoma who were eligible for resection.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23084521',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 329,\n",
       "     'offsetInEndSection': 573,\n",
       "     'text': 'To assess the influence of the p53 regulatory pathway further, we studied the effect of RG7112, a small molecule MDM2 antagonist that activates p53 by preventing its interaction with MDM2, on normal megakaryocytopoiesis and platelet production.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24309210',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'Discovery of RG7112: A Small-Molecule MDM2 Inhibitor in Clinical Development.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24900694',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 398,\n",
       "     'offsetInEndSection': 477,\n",
       "     'text': 'RG7112 was the first small-molecule p53-MDM2 inhibitor in clinical development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23808545',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 467,\n",
       "     'offsetInEndSection': 568,\n",
       "     'text': 'RG7112 binds MDM2 with high affinity (K(D) ~ 11 nmol/L), blocking its interactions with p53 in vitro.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23400593',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 176,\n",
       "     'text': 'RG7112 is a selective inhibitor of p53-MDM2 binding that frees p53 from negative control, activating the p53 pathway in cancer cells leading to cell cycle arrest and apoptosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22753001',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 676,\n",
       "     'offsetInEndSection': 864,\n",
       "     'text': 'The effects of RG7112 and Peg-IFNα 2a on MPN progenitor cells were dependent on blocking p53-MDM2 interactions and activating the p53 pathway, thereby increasing MPN CD34(+) cell apoptosis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24869939',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 128,\n",
       "     'text': 'The orally bioavailable MDM2 antagonist RG7112 and pegylated interferon α 2a target JAK2V617F-positive progenitor and stem cells',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24869939',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 89,\n",
       "     'text': 'Initial testing of the MDM2 inhibitor RG7112 by the Pediatric Preclinical Testing Program',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22753001',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 243,\n",
       "     'text': 'In this issue of Blood, Lu et al describe the cooperation between an orally bioavailable mouse double minute 2 (MDM2) antagonist (RG7112) and the pegylated interferon α (Peg-IFNα 2a) to target JAK2V617F hematopoietic progenitors and stem cells',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25082860',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 118,\n",
       "     'text': 'MDM2 small-molecule antagonist RG7112 activates p53 signaling and regresses human tumors in preclinical cancer models.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23400593',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 70,\n",
       "     'text': 'Activation of p53 by the MDM2 inhibitor RG7112 impairs thrombopoiesis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24309210',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 129,\n",
       "     'text': 'The orally bioavailable MDM2 antagonist RG7112 and pegylated interferon α 2a target JAK2V617F-positive progenitor and stem cells.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24869939',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 90,\n",
       "     'text': 'Initial testing of the MDM2 inhibitor RG7112 by the Pediatric Preclinical Testing Program.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22753001',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 614,\n",
       "     'offsetInEndSection': 802,\n",
       "     'text': 'The primary endpoint was to assess markers of RG7112-dependent MDM2 inhibition and P53 pathway activation (P53, P21, MDM2, Ki-67, macrophage inhibitory cytokine-1 [MIC-1], and apoptosis). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23084521',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 372,\n",
       "     'text': 'RG7112 is a selective inhibitor of p53-MDM2 binding that frees p53 from negative control, activating the p53 pathway in cancer cells leading to cell cycle arrest and apoptosis. RG7112 was selected for evaluation by the Pediatric Preclinical Testing Program (PPTP) due to the relatively low incidence of p53 mutations in pediatric cancers compared with adult malignancies.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22753001',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 136,\n",
       "     'offsetInEndSection': 401,\n",
       "     'text': 'However, the hydrophobic protein-protein interaction surface represents a significant challenge for the development of small-molecule inhibitors with desirable pharmacological profiles. RG7112 was the first small-molecule p53-MDM2 inhibitor in clinical development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23808545',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 303,\n",
       "     'offsetInEndSection': 866,\n",
       "     'text': 'Treatment with low doses of RG7112, an orally available small-molecule inhibitor of p53-MDM2, both alone and combined with pegylated interferon α 2a (Peg-IFNα 2a), significantly decreased MPN colony-forming unit-granulocyte macrophage and burst-forming unit-erythroid numbers and preferentially eliminated the total number of JAKV617F(+) MPN hematopoietic progenitor cells. The effects of RG7112 and Peg-IFNα 2a on MPN progenitor cells were dependent on blocking p53-MDM2 interactions and activating the p53 pathway, thereby increasing MPN CD34(+) cell apoptosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24869939',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 501,\n",
       "     'offsetInEndSection': 807,\n",
       "     'text': 'RG7112 (2g) is the first clinical small-molecule MDM2 inhibitor designed to occupy the p53-binding pocket of MDM2. In cancer cells expressing wild-type p53, RG7112 stabilizes p53 and activates the p53 pathway, leading to cell cycle arrest, apoptosis, and inhibition or regression of human tumor xenografts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24900694',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is known as Calcium Induced Calcium Release (CICR) and its role in cardiomyocyte contractility?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/17214508',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23123322',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2726431',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21718716',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15194743',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25377479',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21979103',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18538346',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15110149',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10779321'],\n",
       "   'ideal_answer': ['the cicr mechanism has been understood mainly based on binding of cytosolic ca(2+) with ryanodine receptors (ryrs) and inducing ca(2+) release from the sarcoplasmic reticulum (sr). l-type ca(2+) channels activate ryrs to produce cicr in smooth muscle cells in the form of ca(2+) sparks and propagated ca(2+) waves. in heart cells, a tight coupling between the gating of single l-type ca(2+) channels and ryanodine receptors (ryrs) underlies calcium release. the importance of ca-induced ca release in excitation-contraction coupling in the heart. waves of calcium-induced calcium release occur in a variety of cell types and have been implicated in the origin of cardiac arrhythmias. in mammals, ca(2+) influx as l-type ca(2+) current (ica) triggers the release of ca(2+) from sarcoplasmic reticulum (sr) and ca(2+)-induced ca(2+) release (cicr) is critical for excitation-contraction coupling. ',\n",
       "    'Cardiomyocyte contraction depends on rapid changes in intracellular Ca(2+). Ca(2+) influx as L-type Ca(2+) current (ICa) triggers the release of Ca(2+) from sarcoplasmic reticulum (SR) and Ca(2+)-induced Ca(2+) release (CICR) is critical for excitation-contraction coupling. Calcium-induced calcium-release in cardiac myocytes takes place in spatially restricted regions known as dyads, where discrete patches of junctional sarcoplasmic reticulum tightly associate with the t-tubule membrane. Calcium-induced calcium release (CICR) has been observed in cardiac myocytes as elementary calcium release events (calcium sparks) associated with the opening of L-type Ca(2+) channels. Waves of calcium-induced calcium release occur in a variety of cell types and have been implicated in the origin of cardiac arrhythmias.'],\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4277675'],\n",
       "   'type': 'summary',\n",
       "   'id': '54dc8ed6c0bb8dce23000002',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 286,\n",
       "     'text': 'Cardiomyocyte contraction depends on rapid changes in intracellular Ca(2+). In mammals, Ca(2+) influx as L-type Ca(2+) current (ICa) triggers the release of Ca(2+) from sarcoplasmic reticulum (SR) and Ca(2+)-induced Ca(2+) release (CICR) is critical for excitation-contraction coupling.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25377479',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 9,\n",
       "     'offsetInEndSection': 291,\n",
       "     'text': 'Release of calcium (Ca(2+)) from the sarcoplasmic reticulum (SR) induced by Ca(2+) influx through voltage-dependent sarcolemmal L-type Ca(2+) channels (CICR) in cardiac muscle cells has been implicated as a potential target contributing to anesthetic-induced myocardial depression. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21979103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 240,\n",
       "     'text': 'It is well known that calcium-induced calcium-release in cardiac myocytes takes place in spatially restricted regions known as dyads, where discrete patches of junctional sarcoplasmic reticulum tightly associate with the t-tubule membrane. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21718716',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 348,\n",
       "     'text': 'Calcium (Ca(2+))-induced Ca(2+) release (CICR) is widely accepted as the principal mechanism linking electrical excitation and mechanical contraction in cardiac cells. The CICR mechanism has been understood mainly based on binding of cytosolic Ca(2+) with ryanodine receptors (RyRs) and inducing Ca(2+) release from the sarcoplasmic reticulum (SR).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18538346',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Waves of calcium-induced calcium release occur in a variety of cell types and have been implicated in the origin of cardiac arrhythmias. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15194743',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 329,\n",
       "     'text': 'Calcium-induced calcium release (CICR) has been observed in cardiac myocytes as elementary calcium release events (calcium sparks) associated with the opening of L-type Ca(2+) channels. In heart cells, a tight coupling between the gating of single L-type Ca(2+) channels and ryanodine receptors (RYRs) underlies calcium release. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10779321',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 354,\n",
       "     'offsetInEndSection': 487,\n",
       "     'text': 'L-type Ca(2+) channels activate RYRs to produce CICR in smooth muscle cells in the form of Ca(2+) sparks and propagated Ca(2+) waves.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10779321',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 498,\n",
       "     'offsetInEndSection': 587,\n",
       "     'text': ' the importance of Ca-induced Ca release in excitation-contraction coupling in the heart.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2726431',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 242,\n",
       "     'text': 'Calcium-induced calcium release (CICR) is an inherently regenerative process due to the Ca(2+)-dependent gating of ryanodine receptors (RyRs) in the sarco/endoplasmic reticulum (SR) and is critical for cardiac excitation-contraction coupling.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23123322',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 290,\n",
       "     'text': 'PURPOSE: Release of calcium (Ca(2+)) from the sarcoplasmic reticulum (SR) induced by Ca(2+) influx through voltage-dependent sarcolemmal L-type Ca(2+) channels (CICR) in cardiac muscle cells has been implicated as a potential target contributing to anesthetic-induced myocardial depression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21979103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 78,\n",
       "     'offsetInEndSection': 249,\n",
       "     'text': 'The Ca(2+)-induced Ca(2+) release (CICR) process that governs cardiac contractility is defective in hypertrophy/heart failure, but the molecular mechanisms remain elusive.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17214508',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 77,\n",
       "     'offsetInEndSection': 247,\n",
       "     'text': 'The Ca(2+)-induced Ca(2+) release (CICR) process that governs cardiac contractility is defective in hypertrophy/heart failure, but the molecular mechanisms remain elusive',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17214508',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 196,\n",
       "     'text': 'The majority of contractile calcium in cardiac muscle is released from stores in the sarcoplasmic reticulum (SR), by a process of calcium-induced calcium release (CICR) through ryanodine receptors',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15110149',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which protein complexes contain mitofilin?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24030101',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23676277',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22252321',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21987634',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22575891',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22014515',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21944719',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22496419',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22918945',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23704930'],\n",
       "   'ideal_answer': ['mitochondrial inter-membrane space bridging (MIB) complex \\nmitochondrial inner membrane organizing system (MINOS)\\nMitOS for mitochondrial organizing structure'],\n",
       "   'exact_answer': [['MIB',\n",
       "     'mitochondrial inter-membrane space bridging complex'],\n",
       "    ['MINOS', 'mitochondrial inner membrane organizing system'],\n",
       "    ['MitOS']],\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0016592',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0043234',\n",
       "    'http://www.uniprot.org/uniprot/FCJ1_LODEL',\n",
       "    'http://www.uniprot.org/uniprot/FCJ1_SCLS1',\n",
       "    'http://www.uniprot.org/uniprot/FCJ1_CANAL',\n",
       "    'http://www.uniprot.org/uniprot/FCJ1_AJECN',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D046912',\n",
       "    'http://www.uniprot.org/uniprot/FCJ1_CANTT'],\n",
       "   'type': 'list',\n",
       "   'id': '53215fca9b2d7acc7e000005',\n",
       "   'snippets': [{'offsetInBeginSection': 915,\n",
       "     'offsetInEndSection': 1062,\n",
       "     'text': ' and the mitochondrial structural proteins-components of mitochondrial inter-membrane space bridging (MIB) complex (Mitofilin, Sam50, and ChChd3), ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24030101',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 330,\n",
       "     'offsetInEndSection': 362,\n",
       "     'text': 'Mitofilin/MINOS protein complex ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23704930',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 879,\n",
       "     'offsetInEndSection': 989,\n",
       "     'text': ' APOOL physically interacts with several subunits of the MINOS complex, namely Mitofilin, MINOS1, and SAMM50. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23704930',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1054,\n",
       "     'offsetInEndSection': 1086,\n",
       "     'text': 'Mitofilin/MINOS protein complex ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23704930',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 616,\n",
       "     'offsetInEndSection': 662,\n",
       "     'text': 'MINOS subunits (mitofilin, MINOS1, and CHCHD3)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23676277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 918,\n",
       "     'offsetInEndSection': 950,\n",
       "     'text': 'mitofilin, a core MINOS subunit,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23676277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 99,\n",
       "     'offsetInEndSection': 239,\n",
       "     'text': 'The mitochondrial inner membrane organizing system (MINOS) is a large protein complex required for maintaining inner membrane architecture. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22918945',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 731,\n",
       "     'offsetInEndSection': 795,\n",
       "     'text': 'Mitochondria lacking mitofilin, the large core subunit of MINOS,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22918945',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 160,\n",
       "     'text': 'The mitochondrial inner membrane contains a large protein complex crucial for membrane architecture, the mitochondrial inner membrane organizing system (MINOS).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22575891',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 343,\n",
       "     'offsetInEndSection': 557,\n",
       "     'text': ' To study if outer membrane interactions and maintenance of cristae morphology are directly coupled, we generated mutant forms of mitofilin/Fcj1 (formation of crista junction protein 1), a core component of MINOS. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22575891',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'The C-terminal domain of Fcj1 is required for formation of crista junctions and interacts with the TOB/SAM complex in mitochondria.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22496419',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 232,\n",
       "     'offsetInEndSection': 291,\n",
       "     'text': 'Mitofilin/Fcj1, the central component of the MINOS complex,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22014515',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 443,\n",
       "     'offsetInEndSection': 648,\n",
       "     'text': 'We provide evidence that the SAM components exist in a large protein complex together with the IMM proteins mitofilin and CHCHD3, which we term the mitochondrial intermembrane space bridging (MIB) complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22252321',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 523,\n",
       "     'offsetInEndSection': 680,\n",
       "     'text': 'which we term MitOS for mitochondrial organizing structure, comprised of Fcj1/Mitofilin, a conserved inner membrane protein, and five additional components. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21987634',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 342,\n",
       "     'offsetInEndSection': 599,\n",
       "     'text': 'Mitofilin is part of a large inner membrane complex, and we identify five partner proteins as constituents of the mitochondrial inner membrane organizing system (MINOS) that is required for keeping cristae membranes connected to the inner boundary membrane.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21944719',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 744,\n",
       "     'offsetInEndSection': 813,\n",
       "     'text': ' Our findings indicate that mitofilin is a central component\\xa0of MINOS',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21944719',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Does smoking increase risk for glioblastoma?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/2319291',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17372252',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19568697',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21742680',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19022673',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19494549',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19107440',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16217772',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20487573',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15685439'],\n",
       "   'ideal_answer': ['No. Smoking does not increase risk for glioblastoma.'],\n",
       "   'exact_answer': 'no',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012907',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:3068',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:3073'],\n",
       "   'type': 'yesno',\n",
       "   'id': '550e7d6aa103b7801600000e',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 144,\n",
       "     'text': 'Glioma risk has consistently been inversely associated with allergy history but not with smoking history despite putative biologic plausibility.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21742680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1159,\n",
       "     'offsetInEndSection': 1380,\n",
       "     'text': 'No relation was observed between glioma risk and smoking (odds ratio = 0.92, 95% confidence interval: 0.77, 1.10; P = 0.37), and there were no interactions for glioma risk of smoking history with any of the risk alleles. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21742680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1654,\n",
       "     'offsetInEndSection': 1926,\n",
       "     'text': 'Non-smokers with G/A and A/A genotype showed increased glioma risk compared with G/G genotype (adjusted OR = 1.72, 95%CI: 1.29-2.30, p = 0.0002 and adjusted OR = 1.81, 95%CI: 1.10-2.99, p = 0.020, respectively). This association was not found in ever- or current-smokers. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20487573',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1256,\n",
       "     'offsetInEndSection': 1363,\n",
       "     'text': 'There was no significant association between glioma and alcohol consumption, smoking and mobile phone use. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19494549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1145,\n",
       "     'offsetInEndSection': 1565,\n",
       "     'text': 'RESULTS: We found no associations between the GSTM3, GSTP1, NQO1, CYP1A1, GSTM1, or GSTT1 polymorphisms and adult brain tumor risk with the possible exception of a weak association between the G-C (Val-Ala) GSTP1 105/114 haplotype and glioma [odds ratio (OR), 0.73; 95% confidence interval (95% CI), 0.54, 0.99], nor was there an interaction between the effects of the GSTM3 or GSTP1 polymorphisms and cigarette smoking.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17372252',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 825,\n",
       "     'offsetInEndSection': 1029,\n",
       "     'text': 'We did not find any evidence for an association with life-style characteristics such as cigarette smoking, alcohol consumption, use of drugs of any kind, or dietary intake of cured or smoked meat or fish.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2319291',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1158,\n",
       "     'offsetInEndSection': 1230,\n",
       "     'text': 'No relation was observed between glioma risk and smoking (odds ratio = 0',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21742680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'Glioma risk has consistently been inversely associated with allergy history but not with smoking history despite putative biologic plausibility',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21742680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1608,\n",
       "     'offsetInEndSection': 1824,\n",
       "     'text': 'Compared with nonsmokers, duration of cigarette smoking, number of cigarettes smoked per day and pack-years of smoking were associated with increased glioma risk, although the increases in risk were relatively modest',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16217772',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 989,\n",
       "     'offsetInEndSection': 1210,\n",
       "     'text': 'Among ever smokers, women who reported having quit smoking had a 51% increase in risk of glioma compared with never smokers (HR = 1.51, 95% CI = 0.97-2.34), while current smokers did not appear to have an increase in risk',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16217772',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Are ACTA1 (alpha actin) and NEB (nebulin) genes related to nemaline myopathy?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22407809',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22358459',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15495263',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11257471',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15564032',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16945536',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12805120',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19648653',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20012312',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11166164'],\n",
       "   'ideal_answer': ['Yes, most nemaline myopathy patients have mutations in the nebulin (NEB) or skeletal muscle alpha-actin (ACTA1) genes.',\n",
       "    'Yes. Most nemaline myopathy patients have mutations in the nebulin (NEB) or skeletal muscle alpha-actin (ACTA1) genes. Mutations in six genes have been reported to cause NM: Nebulin (NEB Pelin 1999), alpha-skeletal muscle actin (ACTA1 Nowak 1999), alpha-slow tropomyosin (TPM3 Laing 1995), beta-tropomyosin (TPM2 Donner 2002), slow troponin T (TNNT1 Johnston 2000) and cofilin 2 (CFL2 Agrawal 2007).'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/NEBU_HUMAN',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:3191',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D017696',\n",
       "    'http://www.uniprot.org/uniprot/ACTS_CYPCA',\n",
       "    'http://www.uniprot.org/uniprot/ACTS_ORYLA',\n",
       "    'http://www.uniprot.org/uniprot/ACTS_ATRMM',\n",
       "    'http://www.uniprot.org/uniprot/ACTS_CARAU',\n",
       "    'http://www.uniprot.org/uniprot/ACTS_OREMO'],\n",
       "   'type': 'yesno',\n",
       "   'id': '552440452c8b63434a00000b',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 262,\n",
       "     'text': 'Nemaline myopathy (NM) is a group of congenital myopathies, characterized by the presence of distinct rod-like inclusions \"nemaline bodies\" in the sarcoplasm of skeletal muscle fibers. To date, ACTA1, NEB, TPM3, TPM2, TNNT1, and CFL2 have been found to cause NM.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22407809',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 356,\n",
       "     'text': 'Nemaline myopathy (NM) is the most common congenital myopathy and is caused by mutations in various genes including NEB (nebulin), TPM2 (beta-tropomyosin), TPM3 (gamma-tropomyosin), and ACTA1 (skeletal alpha-actin). 20-25% of NM cases carry ACTA1 defects and these particular mutations usually induce substitutions of single residues in the actin protein. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22358459',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 487,\n",
       "     'text': 'Nemaline myopathy (NM) is a genetically and clinically heterogenous muscle disorder, which is myopathologically characterized by nemaline bodies. Mutations in six genes have been reported to cause NM: Nebulin (NEB Pelin 1999), alpha-skeletal muscle actin (ACTA1 Nowak 1999), alpha-slow tropomyosin (TPM3 Laing 1995), beta-tropomyosin (TPM2 Donner 2002), slow troponin T (TNNT1 Johnston 2000) and cofilin 2 (CFL2 Agrawal 2007). The majority of cases are due to mutation in NEB and ACTA1. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20012312',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 635,\n",
       "     'text': 'Nemaline myopathy is a heterogenous form of congenital myopathy characterised by a variable spectrum of clinical features, predominated in the severe form by profound muscle hypotonia and weakness accompanied by respiratory insufficiency. The clinical variability, with differing age of onset and severity of symptoms makes the diagnosis of nemaline myopathy difficult in some cases. Severe forms of nemaline myopathy may be caused by mutation of a number of different genes: skeletal muscle actin (ACTA1), nebulin (NEB) and alpha-tropomyosin (TPM3), all of which encode components of the sarcomeric thin filaments of skeletal muscle. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19648653',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 114,\n",
       "     'text': 'Most nemaline myopathy patients have mutations in the nebulin (NEB) or skeletal muscle alpha-actin (ACTA1) genes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16945536',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 232,\n",
       "     'text': 'We report muscle MRI findings of 10 patients from 8 families with nemaline myopathy. Patients with involvement of the nebulin (NEB) gene showed a consistent pattern of selective muscle involvement corresponding to clinical severity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15564032',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 522,\n",
       "     'offsetInEndSection': 719,\n",
       "     'text': 'Patients with nemaline myopathy secondary to mutations in the skeletal muscle alpha-actin (ACTA1) gene showed diffuse involvement of thigh and leg muscles with relative sparing of the gastrocnemii.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15564032',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 584,\n",
       "     'text': 'Congenital myopathies are clinical and genetic heterogeneous disorders characterized by skeletal muscle weakness ranging in severity. Three major forms have been identified: actin myopathy, intranuclear rod myopathy, and nemaline myopathy. Nemaline myopathy is the most common of these myopathies and is further subdivided into seven groups according to severity, progressiveness, and age of onset. At present, five genes have been linked to congenital myopathies. These include alpha-actin (ACTA1), alpha- and beta-tropomyosin (TPM3 and TPM2), troponin T (TNNT1), and nebulin (NEB). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15495263',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 360,\n",
       "     'text': 'Nemaline myopathy is a structural congenital myopathy which may show both autosomal dominant and autosomal recessive inheritance patterns. Mutations in three different genes have been identified as the cause of nemaline myopathy: the gene for slow alpha-tropomyosin 3 (TPM3) at 1q22-23, the nebulin gene (NEB) at 2q21.1-q22, and the actin gene (ACTA1) at 1q42.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11257471',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 426,\n",
       "     'text': 'Nemaline myopathy is a clinically and genetically heterogeneous condition. The clinical spectrum ranges from severe cases with antenatal or neonatal onset and early death to late onset cases with only slow progression. Three genes are known to cause nemaline myopathy: the genes for nebulin (NEB) on chromosome 2q22, slow alpha-tropomyosin (TPM3) on chromosome 1q21 and skeletal muscle alpha-actin (ACTA1) on chromosome 1q42. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11166164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'Nemaline myopathy (NM) is the most common congenital myopathy and is caused by mutations in various genes including NEB (nebulin), TPM2 (beta-tropomyosin), TPM3 (gamma-tropomyosin), and ACTA1 (skeletal alpha-actin).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22358459',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 216,\n",
       "     'offsetInEndSection': 423,\n",
       "     'text': 'Three genes are known to cause nemaline myopathy: the genes for nebulin (NEB) on chromosome 2q22, slow alpha-tropomyosin (TPM3) on chromosome 1q21 and skeletal muscle alpha-actin (ACTA1) on chromosome 1q42.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11166164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 123,\n",
       "     'offsetInEndSection': 294,\n",
       "     'text': 'Five genes have now been associated with nemaline myopathy: alpha-tropomyosin-3 (TPM3), alpha-actin (ACTA1), nebulin (NEB), beta-tropomysin (TPM2) and troponin T (TNNT1).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12805120',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 113,\n",
       "     'text': 'Most nemaline myopathy patients have mutations in the nebulin (NEB) or skeletal muscle alpha-actin (ACTA1) genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16945536',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 384,\n",
       "     'offsetInEndSection': 634,\n",
       "     'text': 'Severe forms of nemaline myopathy may be caused by mutation of a number of different genes: skeletal muscle actin (ACTA1), nebulin (NEB) and alpha-tropomyosin (TPM3), all of which encode components of the sarcomeric thin filaments of skeletal muscle.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19648653',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 125,\n",
       "     'offsetInEndSection': 295,\n",
       "     'text': 'Five genes have now been associated with nemaline myopathy: alpha-tropomyosin-3 (TPM3), alpha-actin (ACTA1), nebulin (NEB), beta-tropomysin (TPM2) and troponin T (TNNT1).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12805120',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 219,\n",
       "     'offsetInEndSection': 425,\n",
       "     'text': 'Three genes are known to cause nemaline myopathy: the genes for nebulin (NEB) on chromosome 2q22, slow alpha-tropomyosin (TPM3) on chromosome 1q21 and skeletal muscle alpha-actin (ACTA1) on chromosome 1q42.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11166164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 139,\n",
       "     'offsetInEndSection': 360,\n",
       "     'text': 'Mutations in three different genes have been identified as the cause of nemaline myopathy: the gene for slow alpha-tropomyosin 3 (TPM3) at 1q22-23, the nebulin gene (NEB) at 2q21.1-q22, and the actin gene (ACTA1) at 1q42.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11257471',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 111,\n",
       "     'text': 'Most nemaline myopathy patients have mutations in the nebulin (NEB) or skeletal muscle alpha-actin (ACTA1) genes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16945536',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 124,\n",
       "     'offsetInEndSection': 293,\n",
       "     'text': 'Five genes have now been associated with nemaline myopathy: alpha-tropomyosin-3 (TPM3), alpha-actin (ACTA1), nebulin (NEB), beta-tropomysin (TPM2) and troponin T (TNNT1)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12805120',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 218,\n",
       "     'offsetInEndSection': 423,\n",
       "     'text': 'Three genes are known to cause nemaline myopathy: the genes for nebulin (NEB) on chromosome 2q22, slow alpha-tropomyosin (TPM3) on chromosome 1q21 and skeletal muscle alpha-actin (ACTA1) on chromosome 1q42',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11166164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 213,\n",
       "     'text': 'Nemaline myopathy (NM) is the most common congenital myopathy and is caused by mutations in various genes including NEB (nebulin), TPM2 (beta-tropomyosin), TPM3 (gamma-tropomyosin), and ACTA1 (skeletal alpha-actin)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22358459',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 125,\n",
       "     'offsetInEndSection': 294,\n",
       "     'text': 'Five genes have now been associated with nemaline myopathy: alpha-tropomyosin-3 (TPM3), alpha-actin (ACTA1), nebulin (NEB), beta-tropomysin (TPM2) and troponin T (TNNT1)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12805120',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 214,\n",
       "     'text': 'Nemaline myopathy (NM) is the most common congenital myopathy and is caused by mutations in various genes including NEB (nebulin), TPM2 (beta-tropomyosin), TPM3 (gamma-tropomyosin), and ACTA1 (skeletal alpha-actin)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22358459',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 139,\n",
       "     'offsetInEndSection': 359,\n",
       "     'text': 'Mutations in three different genes have been identified as the cause of nemaline myopathy: the gene for slow alpha-tropomyosin 3 (TPM3) at 1q22-23, the nebulin gene (NEB) at 2q21.1-q22, and the actin gene (ACTA1) at 1q42',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11257471',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 219,\n",
       "     'offsetInEndSection': 424,\n",
       "     'text': 'Three genes are known to cause nemaline myopathy: the genes for nebulin (NEB) on chromosome 2q22, slow alpha-tropomyosin (TPM3) on chromosome 1q21 and skeletal muscle alpha-actin (ACTA1) on chromosome 1q42',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11166164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 112,\n",
       "     'text': 'Most nemaline myopathy patients have mutations in the nebulin (NEB) or skeletal muscle alpha-actin (ACTA1) genes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16945536',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 384,\n",
       "     'offsetInEndSection': 633,\n",
       "     'text': 'Severe forms of nemaline myopathy may be caused by mutation of a number of different genes: skeletal muscle actin (ACTA1), nebulin (NEB) and alpha-tropomyosin (TPM3), all of which encode components of the sarcomeric thin filaments of skeletal muscle',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19648653',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Pridopidine has been tested for treatment of which disorder?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23446684',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21586914',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20616707',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22948856',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22071279',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20667452',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22560595',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23450660',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23468085',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20373247'],\n",
       "   'ideal_answer': ['Pridopidine is a dopaminergic stabilizer that has shown promising results for treatment of Huntington disease patients.'],\n",
       "   'exact_answer': ['Huntington disease'],\n",
       "   'type': 'factoid',\n",
       "   'id': '550ea8f1b305b40c5c000005',\n",
       "   'snippets': [{'offsetInBeginSection': 1892,\n",
       "     'offsetInEndSection': 2083,\n",
       "     'text': \"These effects of pridopidine may serve to strengthen the cortico-striatal communication and to improve motor control in Huntington's disease for which pridopidine is currently in development.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23468085',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 92,\n",
       "     'text': \"A randomized, double-blind, placebo-controlled trial of pridopidine in Huntington's disease.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23450660',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 207,\n",
       "     'text': \"We examined the effects of 3 dosages of pridopidine, a dopamine-stabilizing compound, on motor function and other features of Huntington's disease, with additional evaluation of its safety and tolerability. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23450660',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1304,\n",
       "     'offsetInEndSection': 1497,\n",
       "     'text': \"Although the primary analysis did not demonstrate a statistically significant treatment effect, the overall results suggest that pridopidine may improve motor function in Huntington's disease. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23450660',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 92,\n",
       "     'text': 'One-year safety and tolerability profile of pridopidine in patients with Huntington disease.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23446684',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': 'OBJECTIVE: To assess the 1-year safety profile of the dopaminergic stabilizer pridopidine in patients with Huntington disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23446684',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1430,\n",
       "     'offsetInEndSection': 1616,\n",
       "     'text': 'CLASSIFICATION OF EVIDENCE: This study provides Class IV evidence that pridopidine (≤90 mg/day) is generally safe and well-tolerated in patients with Huntington disease for up to 1 year.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23446684',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 211,\n",
       "     'text': \"Pridopidine is being developed for the treatment of impaired motor function associated with Huntington's disease and belongs to a new class of compounds known as dopidines, which act as dopaminergic stabilizers.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22948856',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 248,\n",
       "     'offsetInEndSection': 431,\n",
       "     'text': ' There is so far neither cure nor approved disease-slowing therapy for HD, though recent clinical studies have shown a beneficial long-term effect of pridopidine in patients with HD. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22560595',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 160,\n",
       "     'text': \"Pridopidine for the treatment of motor function in patients with Huntington's disease (MermaiHD): a phase 3, randomised, double-blind, placebo-controlled trial.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22071279',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 139,\n",
       "     'offsetInEndSection': 652,\n",
       "     'text': \"Pridopidine belongs to a new class of compounds known as dopaminergic stabilisers, and results from a small phase 2 study in patients with Huntington's disease suggested that this drug might improve voluntary motor function. We aimed to assess further the effects of pridopidine in patients with Huntington's disease. METHODS: We undertook a 6 month, randomised, double-blind, placebo-controlled trial to assess the efficacy of pridopidine in the treatment of motor deficits in patients with Huntington's disease.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22071279',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2432,\n",
       "     'offsetInEndSection': 2725,\n",
       "     'text': \"INTERPRETATION: This study did not provide evidence of efficacy as measured by the mMS, but a potential effect of pridopidine on the motor phenotype of Huntington's disease merits further investigation. Pridopidine up to 90 mg per day was well tolerated in patients with Huntington's disease. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22071279',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 109,\n",
       "     'text': 'Regional cerebral glucose metabolism after pridopidine (ACR16) treatment in patients with Huntington disease.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21586914',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 221,\n",
       "     'offsetInEndSection': 748,\n",
       "     'text': 'Pridopidine (ACR16) belongs to a novel class of central nervous system compounds in development for the treatment of Huntington disease. The objective of the study was to investigate the metabolic changes in patients with Huntington disease before and after pridopidine treatment. METHODS: [(18)F]Fluorodeoxyglucose positron emission tomographic imaging was used to measure the regional cerebral metabolic rate of glucose at baseline and after 14 days of open-label pridopidine treatment in 8 patients with Huntington disease. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21586914',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1163,\n",
       "     'offsetInEndSection': 1562,\n",
       "     'text': 'CONCLUSIONS: Our findings suggest that pridopidine induces metabolic changes in brain regions implicated as important for mediating compensatory mechanisms in Huntington disease. In addition, the finding of a strong relationship between clinical severity and metabolic activity after treatment also suggests that pridopidine treatment targets a Huntington disease-related metabolic activity pattern.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21586914',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1365,\n",
       "     'offsetInEndSection': 1574,\n",
       "     'text': \"The putative restoration of function in cortico-subcortical circuitry by pridopidine is likely to make it useful for ameliorating several neurological and psychiatric disorders, including Huntington's disease.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20667452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 109,\n",
       "     'text': \"Efficacy and safety of the dopaminergic stabilizer Pridopidine (ACR16) in patients with Huntington's disease.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20616707',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 294,\n",
       "     'text': \"OBJECTIVES: To evaluate the efficacy and safety of the dopaminergic stabilizer pridopidine (ACR16) in patients with Huntington's disease (HD). METHODS: In a randomized, double-blind, placebo-controlled, 4-week trial, patients with HD received pridopidine (50 mg/d, n = 28) or placebo (n = 30). \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20616707',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1151,\n",
       "     'offsetInEndSection': 1237,\n",
       "     'text': 'CONCLUSIONS: Pridopidine shows promise as a treatment for some of the symptoms of HD. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20616707',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 647,\n",
       "     'offsetInEndSection': 841,\n",
       "     'text': 'Investigational drugs discussed include ALN-HTT (Alnylam Pharmaceuticals Inc/Medtronic Inc), EPI-743 (Edison Pharmaceuticals Inc), LNK-754 (Link Medicine Corp) and pridopidine (NeuroSearch A/S).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20373247',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1869,\n",
       "     'offsetInEndSection': 2061,\n",
       "     'text': \"These effects of pridopidine may serve to strengthen the cortico-striatal communication and to improve motor control in Huntington's disease for which pridopidine is currently in development.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23468085',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1287,\n",
       "     'offsetInEndSection': 1508,\n",
       "     'text': 'In addition, the finding of a strong relationship between clinical severity and metabolic activity after treatment also suggests that pridopidine treatment targets a Huntington disease-related metabolic activity pattern.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21586914',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1109,\n",
       "     'offsetInEndSection': 1181,\n",
       "     'text': 'Pridopidine shows promise as a treatment for some of the symptoms of HD.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20616707',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 72,\n",
       "     'text': 'Pridopidine shows promise as a treatment for some of the symptoms of HD.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20616707',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 166,\n",
       "     'offsetInEndSection': 386,\n",
       "     'text': 'In addition, the finding of a strong relationship between clinical severity and metabolic activity after treatment also suggests that pridopidine treatment targets a Huntington disease-related metabolic activity pattern.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21586914',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': \"We examined the effects of 3 dosages of pridopidine, a dopamine-stabilizing compound, on motor function and other features of Huntington's disease, with additional evaluation of its safety and tolerability\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23450660',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1891,\n",
       "     'offsetInEndSection': 2081,\n",
       "     'text': \"These effects of pridopidine may serve to strengthen the cortico-striatal communication and to improve motor control in Huntington's disease for which pridopidine is currently in development\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23468085',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1302,\n",
       "     'offsetInEndSection': 1522,\n",
       "     'text': 'In addition, the finding of a strong relationship between clinical severity and metabolic activity after treatment also suggests that pridopidine treatment targets a Huntington disease-related metabolic activity pattern.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21586914',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which genes are known to be involved in Diamond-Blackfan anemia?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22689679',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22706301',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23257444',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18715690',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20378560',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22262766',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19773262',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20454576',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20655265',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23812780'],\n",
       "   'ideal_answer': ['Diamond-Blackfan anemia (DBA) is an inherited red blood cell aplasia that usually presents during the first year of life. The disease has been associated with point mutations and large deletions in ten ribosomal protein (RP) genes RPS19, RPS24, RPS17, RPL35A, RPL5, RPL11, RPS7, RPS10, RPS26, and RPL26, as well as in GATA1, in about 60-65 % of patients.'],\n",
       "   'exact_answer': [['RPS19'],\n",
       "    ['RPS24'],\n",
       "    ['RPS17'],\n",
       "    ['RPL35A'],\n",
       "    ['RPL5'],\n",
       "    ['RPL11'],\n",
       "    ['RPS7'],\n",
       "    ['RPS10'],\n",
       "    ['RPS26'],\n",
       "    ['RPL26'],\n",
       "    ['GATA1']],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:1339'],\n",
       "   'type': 'list',\n",
       "   'id': '550312b4e9bde6963400001c',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 645,\n",
       "     'text': 'Diamond-Blackfan anemia (DBA) is an inherited red blood cell aplasia that usually presents during the first year of life. The main features of the disease are normochromic and macrocytic anemia, reticulocytopenia, and nearly absent erythroid progenitors in the bone marrow. The patients also present with growth retardation and craniofacial, upper limb, heart and urinary system congenital malformations in ~30-50 % of cases. The disease has been associated with point mutations and large deletions in ten ribosomal protein (RP) genes RPS19, RPS24, RPS17, RPL35A, RPL5, RPL11, RPS7, RPS10, RPS26, and RPL26 and GATA1 in about 60-65 % of patients',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23812780',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 344,\n",
       "     'text': 'This study was aimed to explore the mutations of ribosomal protein (RP) genes in patients with Diamond Blackfan anemia (DBA). Twenty-one cases of DBA admitted in our hospital from Dec 2008 to Aug 2012 were screened by PCR for mutations in the nine known genes associated with DBA: RPS19, RPS24, RPS17, RPL5, RPL11, RPS7, RPL35a, RPS10 and RPS26',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23257444',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 346,\n",
       "     'offsetInEndSection': 602,\n",
       "     'text': 'The results found that 8 patients (38.1%) with DBA had mutations in the genes coding for ribosomal protein, in which RPS19 mutation was identified in 3 patients, RPS24, RPS7, RPL5, RPL11 and RPL35A mutations were identified respectively in 1 of the patient',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23257444',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 379,\n",
       "     'text': 'Diamond-Blackfan anemia (DBA) is a hypoplastic anemia characterized by impaired production of red blood cells, with approximately half of all cases attributed to ribosomal protein gene mutations. We performed exome sequencing on two siblings who had no known pathogenic mutations for DBA and identified a mutation in the gene encoding the hematopoietic transcription factor GATA1',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22706301',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 352,\n",
       "     'offsetInEndSection': 679,\n",
       "     'text': 'As a proof of concept, we designed a multiplex ligation-dependent probe amplification assay targeted to screen the six genes that are most frequently mutated in Diamond-Blackfan anemia patients: RPS17, RPS19, RPS26, RPL5, RPL11, and RPL35A. Using this assay we showed that deletions represent approximately 20% of all mutations',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22689679',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 439,\n",
       "     'offsetInEndSection': 658,\n",
       "     'text': 'DESIGN AND METHODS: We screened 49 Japanese patients with Diamond-Blackfan anemia (45 probands) for mutations in the six known genes associated with Diamond-Blackfan anemia: RPS19, RPS24, RPS17, RPL5, RPL11, and RPL35A.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20378560',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 338,\n",
       "     'offsetInEndSection': 492,\n",
       "     'text': 'Defects in the RPS19 gene, encoding the ribosomal protein S19, are the main known cause of Diamond-Blackfan anemia and account for more than 25% of cases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19773262',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 133,\n",
       "     'text': 'OBJECTIVE: Diamond-Blackfan anemia (DBA) is a rare congenital hypoplastic anemia caused by mutations in ribosomal protein (RP) genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18715690',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': 'Fifty percent of Diamond-Blackfan anemia (DBA) patients possess mutations in genes coding for ribosomal proteins (RPs).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22262766',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 234,\n",
       "     'offsetInEndSection': 429,\n",
       "     'text': 'Mutations in the gene coding for the ribosomal protein RPS19 have been identified in 25% of patients with DBA, with resulting impairment of 18S rRNA processing and 40S ribosomal subunit formation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20454576',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 428,\n",
       "     'offsetInEndSection': 626,\n",
       "     'text': 'We screened 49 Japanese patients with Diamond-Blackfan anemia (45 probands) for mutations in the six known genes associated with Diamond-Blackfan anemia: RPS19, RPS24, RPS17, RPL5, RPL11, and RPL35A',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20378560',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 326,\n",
       "     'offsetInEndSection': 479,\n",
       "     'text': 'Defects in the RPS19 gene, encoding the ribosomal protein S19, are the main known cause of Diamond-Blackfan anemia and account for more than 25% of cases',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19773262',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 745,\n",
       "     'offsetInEndSection': 855,\n",
       "     'text': 'The first ribosomal gene involved in DBA, ribosomal protein (RP) gene S19 (RPS19 gene), was identified in 1999',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20655265',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1260,\n",
       "     'offsetInEndSection': 1396,\n",
       "     'text': 'These data identify RPL15 as a new gene involved in DBA and further support the presence of large deletions in RP genes in DBA patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23812780',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can desvenlafaxine be used at a dose of 50mg/day?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23587982',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21067460',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23881185',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20107296',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18507895',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23517291',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22173281',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19407711',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23473348',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19407730'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15520455',\n",
       "     'o': '790287'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341627',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A15520455'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341627',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A15520455'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15520455',\n",
       "     'o': 'Desvenlafaxine 50 MG'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15551742',\n",
       "     'o': '790289'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341629',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A15551742'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341629',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A15551742'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15551742',\n",
       "     'o': 'Desvenlafaxine 50 MG [Pristiq]'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#broader',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341629',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C2341627'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15520455',\n",
       "     'o': 'RXNORM'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15551742',\n",
       "     'o': 'RXNORM'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A16715518',\n",
       "     'o': '845106'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2682645',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A16715518'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2682645',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A16715518'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A16715518',\n",
       "     'o': 'Desvenlafaxine 50 MG Extended Release Tablet'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#broader',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2682645',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C2341609'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341609',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A15532899'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341609',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A15532899'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A15532899',\n",
       "     'o': 'Desvenlafaxine Extended Release Tablet'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#narrower',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C2341627',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/id/C2341629'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A16715518',\n",
       "     'o': 'RXNORM'}],\n",
       "   'ideal_answer': ['Yes, desvenlafaxine can be at 50mg/day to treat patients with major depressive disorder.  Studies suggest that 50 mg is the minimum effective dose of desvenlafaxine for the treatment of major depressive disorder. The recommended dose of DVS ranges from 50 to 100 mg.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4266016',\n",
       "    'http://www.biosemantics.org/jochem#4243854'],\n",
       "   'type': 'yesno',\n",
       "   'id': '530cf4fe960c95ad0c00000c',\n",
       "   'snippets': [{'offsetInBeginSection': 1331,\n",
       "     'offsetInEndSection': 1466,\n",
       "     'text': 'Long-term use of desvenlafaxine was safe and well tolerated, with a clinical benefit/risk profile similar to that in other populations.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23587982',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2,\n",
       "     'offsetInEndSection': 286,\n",
       "     'text': 'e objective of this study was to evaluate the long-term safety of desvenlafaxine for continuation treatment of major depressive disorder (MDD) in Japanese patients. This was a phase 3, multicenter, 10-month, open-label study with flexible dosing of desvenlafaxine (25, 50, 100 mg/day)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23587982',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 13,\n",
       "     'offsetInEndSection': 270,\n",
       "     'text': 'n an effort to establish the lowest effective dose of desvenlafaxine (administered as desvenlafaxine succinate), we assessed the efficacy, safety, and tolerability of 10- and 50-mg/day desvenlafaxine vs placebo for the treatment of major depressive disorder',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23517291',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 893,\n",
       "     'offsetInEndSection': 1101,\n",
       "     'text': 'Change from baseline to final evaluation in adjusted HAM-D(17) total scores was not significantly different comparing desvenlafaxine 10 mg/day (-9.28) and desvenlafaxine 50 mg/day (-8.92) with placebo (-8.42)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23517291',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1359,\n",
       "     'offsetInEndSection': 1449,\n",
       "     'text': 'Overall rates of treatment-emergent adverse events with both doses were similar to placebo',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23517291',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1526,\n",
       "     'offsetInEndSection': 1783,\n",
       "     'text': 'However, in a companion study reported separately, desvenlafaxine 50 mg, but not 25 mg, separated from placebo. Taken together, these studies suggest that 50 mg is the minimum effective dose of desvenlafaxine for the treatment of major depressive disorder. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23517291',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 740,\n",
       "     'offsetInEndSection': 793,\n",
       "     'text': 'Desvenlafaxine XR was dosed at 50 mg/day for 10 days.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22173281',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 539,\n",
       "     'offsetInEndSection': 645,\n",
       "     'text': 'Desvenlafaxine therapy is initiated at the therapeutic dose (50 mg/day) without a need for dose titration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21067460',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 370,\n",
       "     'offsetInEndSection': 764,\n",
       "     'text': 'Clinical studies have investigated the efficacy of DVS in doses ranging from 50 to 400 mg/day for the treatment of MDD in adult outpatients. The effects of DVS 50 mg/day have been clearly distinguished from placebo in the reduction of MDD symptoms in such clinical trials. No additional therapeutic benefits were found at doses > 50 mg/day. The recommended dose of DVS ranges from 50 to 100 mg.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20107296',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 540,\n",
       "     'offsetInEndSection': 668,\n",
       "     'text': 'Adult outpatients with major depressive disorder received desvenlafaxine doses ranging from 50-400 mg/day or placebo for 8 weeks',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19407730',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2422,\n",
       "     'offsetInEndSection': 2533,\n",
       "     'text': ' At the recommended therapeutic dose of 50 mg/day, discontinuation due to adverse events was similar to placebo',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19407730',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 341,\n",
       "     'offsetInEndSection': 483,\n",
       "     'text': 'atients received fixed (50, 100, 200, or 400 mg/day; n=1,342) or flexible doses (100-400 mg/day; n=463) of desvenlafaxine or placebo (n=1,108)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19407711',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1244,\n",
       "     'offsetInEndSection': 1493,\n",
       "     'text': 'Desvenlafaxine demonstrated short-term efficacy for treating major depressive disorder across the range of doses studied. No evidence of greater efficacy was observed with doses >50 mg/day; a strong dose-response effect on tolerability was observed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19407711',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 249,\n",
       "     'text': 'To assess the efficacy, safety, and tolerability of 50- and 100-mg/day doses of desvenlafaxine (administered as desvenlafaxine succinate), a serotonin-norepinephrine reuptake inhibitor, for the treatment of major depressive disorder (MDD)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18507895',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 280,\n",
       "     'offsetInEndSection': 581,\n",
       "     'text': 'Patients with Diagnostic and Statistical Manual of Mental Disorders, Fourth Edition (DSM-IV) MDD and 17-item Hamilton Rating Scale for Depression (HAM-D(17)) scores > or =20 were randomly assigned to double-blind placebo or desvenlafaxine treatment (fixed dose of 50 mg/day or 100 mg/day) for 8 weeks.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18507895',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 931,\n",
       "     'offsetInEndSection': 1093,\n",
       "     'text': 'Desvenlafaxine 50 mg was associated with a significantly greater adjusted mean change from baseline on the HAM-D(17) (-11.5) compared with placebo (-9.5, p=0.018)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18507895',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1701,\n",
       "     'offsetInEndSection': 1809,\n",
       "     'text': ' These results demonstrate efficacy, safety, and tolerability of desvenlafaxine 50 mg/day for treating MDD. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18507895',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1720,\n",
       "     'offsetInEndSection': 1956,\n",
       "     'text': 'CONCLUSIONS: Desvenlafaxine at the recommended dose of 50 mg/d was effective in relapse prevention of depression during a 6-month period in patients who demonstrated stable response after 20 weeks of open-label desvenlafaxine treatment.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23473348',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'what is the role of GATA-4 in regeneration of the heart after myocardial infarction?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22748493',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19591228',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21344679',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20870802',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22898045',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20200331',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16304579',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18290252',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15883211',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20971132'],\n",
       "   'ideal_answer': ['GATA-4 is implicated in cardiogenic differentiation of cardiac c-kit+AT2+ cells that represent approximately 0.19% of total cardiac cells in infarcted heart. GATA-4 overexpression in mesenchymal stem cells increases both survival and angiogenic potential in ischemic myocardium and may therefore represent a novel and efficient therapeutic approach for postinfarct regenaration. In addition, interventions, such as hypergravity and 5-Aza treatment, induce expression of early muscle and cardiac markers like GATA-4 in BMSCs. A subpopulation of primitive cells from rat heart, expressing c-kit and myogenic transcriptional factors, GATA-4 and MEF 2C, exhibit a high in vitro proliferative potential. Progeny of these implanted cells have been shown to migrate along the infarcted scar, reconstitute regenerated cardiomyocytes with incorporation into host myocardium, and inhibit cardiac remodeling with decreased scar formation. In another study, TGF-beta has been shown to conduct the myogenic differentiation of stem cells by upregulating GATA-4 and NKx-2.5 expression and the intramyocardial implantation of TGF-beta-preprogrammed stem cells effectively assisted the myocardial regeneration. Furthermore, G-CSF treatment in postinfarcted murine hearts appears to be an effective approach for treating heart failure and also leads to induction of GATA-4 resulting in expression of various sarcomeric proteins.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D006321',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D050980',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D009203',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012038'],\n",
       "   'type': 'summary',\n",
       "   'id': '532f03e6d6d3ac6a34000022',\n",
       "   'snippets': [{'offsetInBeginSection': 1704,\n",
       "     'offsetInEndSection': 2038,\n",
       "     'text': 'TGF-beta conducted the myogenic differentiation of CD117+ stem cells by upregulating GATA-4 and NKx-2.5 expression. Therefore, the intramyocardial implantation of TGF-beta-preprogrammed CD117+ cells effectively assisted the myocardial regeneration and induced therapeutic angiogenesis, contributing to functional cardiac regeneration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15883211',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 765,\n",
       "     'offsetInEndSection': 965,\n",
       "     'text': 'G-CSF treatment also led to activation of signal transducer and activator of transcription-3 and induction of GATA-4 and various sarcomeric proteins such as myosin heavy chain, troponin I and desmin. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16304579',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 769,\n",
       "     'offsetInEndSection': 1516,\n",
       "     'text': 'We describe here the identification of a subpopulation of primitive cells from rat heart, processing stem cell marker, c-kit and myogenic transcriptional factors, GATA-4 and MEF 2C, and cardiac specific proteins, troponin-I, alpha-sarcomeric actinin and connexin-43. They exhibited a high in vitro proliferative potential. These findings strongly suggest that these cells are putative cardiomyocyte precursors. After transplantation, they were able to be retained and proliferate (13.63 +/- 5.97% after 2 weeks) within the ischemic heart. Progeny of implanted cells migrated along the infarcted scar, reconstituted regenerated cardiomyocytes with incorporation into host myocardium, and inhibited cardiac remodeling with decreased scar formation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18290252',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1369,\n",
       "     'offsetInEndSection': 1822,\n",
       "     'text': 'Histological studies at week 12 post-transplantation demonstrated engraftment of the pre DiI-stained hCLCs into the scarred myocardium and their expression of human specific alpha-cardiac actin. Human alpha cardiac actin-positive cells also expressed cardiac nuclear factors; nkx2.5 and GATA-4. Our results suggest that intracoronary artery transplantation of hCLCs is a potentially effective therapeutic strategy for future cardiac tissue regeneration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22898045',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 810,\n",
       "     'offsetInEndSection': 988,\n",
       "     'text': 'This pattern of colocalization was also found in developing embryonic myocardium. Cardiac transcription factors Nkx-2.5 and GATA-4 were strongly expressed in the muscle bundles. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22748493',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1178,\n",
       "     'offsetInEndSection': 1318,\n",
       "     'text': 'After hypergravity and 5-Aza treatment, BMSCs showed positive for the early muscle and cardiac markers GATA-4, MEF-2, and Nkx2-5 with RT-PCR',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21344679',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 923,\n",
       "     'offsetInEndSection': 1309,\n",
       "     'text': 'Experiments in murine models of myocardial infarction (MI) demonstrated population of VSELs expressed also early markers of cardiac and endothelial lineages (GATA-4, Nkx2.5/Csx, VE-cadherin, von Willebrand factor) which migrated to stromal-derived factor-1 (SDF-1) and other chemoattractant gradient and underwent rapid mobilization into peripheral blood in experimental MI mice models.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20971132',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1607,\n",
       "     'offsetInEndSection': 1828,\n",
       "     'text': 'We conclude that GATA-4 overexpression in MSCs increased both MSC survival and angiogenic potential in ischemic myocardium and may therefore represent a novel and efficient therapeutic approach for postinfarct remodeling.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20870802',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 798,\n",
       "     'offsetInEndSection': 1087,\n",
       "     'text': 'These cardiac c-kit+AT2+ cells, represented approximately 0.19% of total cardiac cells in infarcted heart, were characterized by upregulated transcription factors implicated in cardiogenic differentiation (Gata-4, Notch-2, Nkx-2.5) and genes required for self-renewal (Tbx-3, c-Myc, Akt). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19591228',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1608,\n",
       "     'offsetInEndSection': 1738,\n",
       "     'text': 'these findings suggest G-CSF administration could be an effective approach to treating chronic heart failure following a large MI.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16304579',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the purpose of the Tokuhashi scoring system?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24212518',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17593839',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23328875',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21223698',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21772622',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21796024',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22973387',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25085251',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25035829',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17094910'],\n",
       "   'ideal_answer': ['Tokuhashi scoring system was developed to predict life expectancy of patients with spinal metastases. The revised Tokuhashi score has been widely used to evaluate indications for surgery and predict survival in patients with metastatic spinal disease.'],\n",
       "   'type': 'summary',\n",
       "   'id': '56bcd422d36b5da378000005',\n",
       "   'snippets': [{'offsetInBeginSection': 409,\n",
       "     'offsetInEndSection': 746,\n",
       "     'text': ' Those referring to representative scoring systems about predicting the survival of patients with metastatic spine tumors were used. The significance and limits of these scoring systems, and the future perspectives were described. Tokuhashi score, Tomita score, Baur score, Linden score, Rades score, and Katagiri score were introduced. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25035829',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 276,\n",
       "     'text': 'BACKGROUND: We sought to identify preoperative factors significantly correlated with survival. We also aimed to evaluate the validity of the prognostic scores in the Tomita and Tokuhashi systems and discuss several aspects to improve the predictive accuracy of these systems. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25085251',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 122,\n",
       "     'text': 'Accuracy of the revised Tokuhashi score in predicting survival in patients with metastatic spinal cord compression (MSCC).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23328875',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'PURPOSE: The revised Tokuhashi score has been widely used to evaluate indications for surgery and predict survival in patients with metastatic spinal disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23328875',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2501,\n",
       "     'offsetInEndSection': 2910,\n",
       "     'text': 'CONCLUSION: We would conclude that although the predictive value of the Tokuhashi score in terms of survival time is at best modest (66 %), the fact that there were statistically significant differences in survival between the groups looked at in this paper indicates that the scoring system, and the components which it consists of, are important in the evaluation of these patients when considering surgery.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23328875',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'Tokuhashi Scoring System has limited applicability in the majority of patients with spinal cord compression secondary to vertebral metastasis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24212518',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 166,\n",
       "     'offsetInEndSection': 235,\n",
       "     'text': 'The Tokuhashi Scoring System (TSS) is a widely used prognostic tool. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24212518',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 178,\n",
       "     'text': 'Predictive value of Tokuhashi scoring systems in spinal metastases, focusing on various primary tumor groups: evaluation of 448 patients in the Aarhus spinal metastases database.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21796024',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 125,\n",
       "     'offsetInEndSection': 294,\n",
       "     'text': 'OBJECTIVE: To determine the specific predictive value of the Tokuhashi scoring system (T12) and its revised version (T15) in spinal metastases of various primary tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21796024',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 120,\n",
       "     'text': 'Is Tokuhashi score suitable for evaluation of life expectancy before surgery in Iranian patients with spinal metastases?',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22973387',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 354,\n",
       "     'text': 'BACKGROUND: One of the most important selection criteria for spinal metastases surgery is life expectancy and the most important system for this prediction has been proposed by Tokuhashi. The aim of this study was to evaluate predictive value of the Tokuhashi score for life expectancy in Iranian patients with spinal metastases one year after diagnosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22973387',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1319,\n",
       "     'offsetInEndSection': 1508,\n",
       "     'text': 'CONCLUSIONS: Present study showed that the Tokuhashi revised scoring system may be practicable and highly predictive preoperative scoring system for patients with spinal metastases in Iran.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22973387',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 393,\n",
       "     'offsetInEndSection': 546,\n",
       "     'text': 'This study was to evaluate Tomita and Tokuhashi scoring systems in selecting surgical procedure and predicting prognosis of extradural spinal metastases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17094910',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 154,\n",
       "     'text': 'Analysis of the predictive role and new proposal for surgical strategies based on the modified Tomita and Tokuhashi scoring systems for spinal metastasis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25085251',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 183,\n",
       "     'offsetInEndSection': 306,\n",
       "     'text': 'We compared the Tokuhashi and Tomita scoring systems, two commonly used scoring systems for prognosis in spinal metastases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21772622',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 120,\n",
       "     'text': '[The predictive value of the Tokuhashi revised scoring system for the survival time of patients with spinal metastases].',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17593839',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 132,\n",
       "     'text': 'To evaluate the predictive value of the Tokuhashi revised scoring system for the life expectancy of patients with spinal metastases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17593839',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 269,\n",
       "     'offsetInEndSection': 427,\n",
       "     'text': 'To determine the specific predictive value of the Tokuhashi scoring system (T12) and its revised version (T15) in spinal metastases of various primary tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21796024',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 180,\n",
       "     'text': 'To evaluate the predictive values of Tokuhashi score, revised Tokuhashi score and Tomita score systems for life expectancy and treatment options in patients with spinal metastasis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21223698',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': 'Tokuhashi Scoring System has limited applicability in the majority of patients with spinal cord compression secondary to vertebral metastasis',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24212518',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': '[The predictive value of the Tokuhashi revised scoring system for the survival time of patients with spinal metastases]',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17593839',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'To evaluate the predictive value of the Tokuhashi revised scoring system for the life expectancy of patients with spinal metastases',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17593839',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 61,\n",
       "     'offsetInEndSection': 183,\n",
       "     'text': 'We compared the Tokuhashi and Tomita scoring systems, two commonly used scoring systems for prognosis in spinal metastases',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21772622',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 159,\n",
       "     'text': 'PURPOSE: The revised Tokuhashi score has been widely used to evaluate indications for surgery and predict survival in patients with metastatic spinal disease. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23328875',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which transcription factors are known as the four (4) \"Yamanaka factors\" that have been used to create induced pluripotent stem cells  (iPSCs)?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/21839145',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23149977',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23704989',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23272148',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23266889',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24706886',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23658991',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23939864',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23104133',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23166588'],\n",
       "   'ideal_answer': ['Fibroblasts can be reprogrammed into induced pluripotent stem cells (iPSCs) by the application of Yamanaka factors (OSKM). In particular, the mechanisms how the Yamanaka factors (Oct4, Sox2, Klf4, and c-Myc) directly drive reprogramming and which additional components are involved are still not yet understood.',\n",
       "    'Through the ectopic expression of four transcription factors, Oct4, Klf4, Sox2 and cMyc, human somatic cells can be converted to a pluripotent state, generating so-called induced pluripotent stem cells (iPSCs)(1-4).  Delivery of the transcription factors Oct4, Klf4, Sox2 and c-Myc via integrating viral vectors has been widely employed to generate induced pluripotent stem cell (iPSC) lines from both normal and disease-specific somatic tissues, providing an invaluable resource for medical research and drug development.  ',\n",
       "    'Through the ectopic expression of four transcription factors, Oct4, Klf4, Sox2 and cMyc, human somatic cells can be converted to a pluripotent state, generating so-called induced pluripotent stem cells (iPSCs)(1-4).Fibroblasts can be reprogrammed into induced pluripotent stem cells (iPSCs) by the application of Yamanaka factors (OSKM), but the mechanisms underlying this reprogramming remain poorly understood.'],\n",
       "   'exact_answer': [['Oct4', 'Oct 3/4'], ['Sox2'], ['Klf4'], ['c-Myc']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D057026',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D039904'],\n",
       "   'type': 'list',\n",
       "   'id': '54fb5720d176fff445000002',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 189,\n",
       "     'text': 'The recently established reprogramming of somatic cells into induced pluripotent stem cells (iPSCs) by Takahashi and Yamanaka represents a valuable tool for future therapeutic applications.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23939864',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 265,\n",
       "     'offsetInEndSection': 453,\n",
       "     'text': 'In particular, the mechanisms how the Yamanaka factors (Oct4, Sox2, Klf4, and c-Myc) directly drive reprogramming and which additional components are involved are still not yet understood.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23939864',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 444,\n",
       "     'offsetInEndSection': 638,\n",
       "     'text': 'Here we show that Yamanaka factors (OCT4, SOX2, MYC, and KLF4)-expressing EV can also reprogram adult peripheral blood mononuclear cells (PBMNCs) into pluripotency, yet at a very low efficiency.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23704989',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 698,\n",
       "     'offsetInEndSection': 941,\n",
       "     'text': 'The CytoTune™-iPS Reprogramming Kit contains four SeV-based reprogramming vectors, each capable of expressing one of the four Yamanaka factors (i.e., Oct4, Sox2, Klf4, and c-Myc) and are optimized for generating iPSCs from human somatic cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23658991',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 305,\n",
       "     'text': 'Delivery of the transcription factors Oct4, Klf4, Sox2 and c-Myc via integrating viral vectors has been widely employed to generate induced pluripotent stem cell (iPSC) lines from both normal and disease-specific somatic tissues, providing an invaluable resource for medical research and drug development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23272148',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 771,\n",
       "     'offsetInEndSection': 1031,\n",
       "     'text': 'We demonstrate that residual expression of the Yamanaka factors prevents iPSCs from acquiring the transcriptional program exhibited by embryonic stem cells (ESCs) and that the expression profiles of iPSCs generated with and without c-Myc are indistinguishable.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23272148',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 197,\n",
       "     'text': 'Fibroblasts can be reprogrammed into induced pluripotent stem cells (iPSCs) by the application of Yamanaka factors (OSKM), but the mechanisms underlying this reprogramming remain poorly understood.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23266889',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 733,\n",
       "     'offsetInEndSection': 889,\n",
       "     'text': 'We also discovered that HMGA1 enhances cellular reprogramming of somatic cells to iPSCs together with the Yamanaka factors (OCT4, SOX2, KLF4, cMYC - OSKM). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23166588',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'Through the ectopic expression of four transcription factors, Oct4, Klf4, Sox2 and cMyc, human somatic cells can be converted to a pluripotent state, generating so-called induced pluripotent stem cells (iPSCs)(1-4).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23149977',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 506,\n",
       "     'offsetInEndSection': 720,\n",
       "     'text': 'We have shown the generation of transgene-free human iPSCs from patients with different lung diseases using a single excisable polycistronic lentiviral Stem Cell Cassette (STEMCCA) encoding the Yamanaka factors(6).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23149977',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 915,\n",
       "     'offsetInEndSection': 1050,\n",
       "     'text': 'These iPSCs were established in a feeder-free system by lentiviral transduction of the Yamanaka factors, Oct3/4, Sox2, Klf4, and c-Myc.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21839145',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 227,\n",
       "     'text': 'Induced pluripotent stem cells (iPSCs) are created by the reprogramming of somatic cells via overexpression of certain transcription factors, such as the originally described Yamanaka factors: Oct4, Sox2, Klf4, and c-Myc (OSKM)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24706886',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 398,\n",
       "     'offsetInEndSection': 638,\n",
       "     'text': 'iPSC production was first achieved by transducing, with the use of retroviral vectors, four specific transcription factors: Oct4, Klf4, Sox2 and c-Myc (OKSM), into primary cells in culture Takahashi and Yamanaka, (Cell 126(4):663-676, 2006)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23104133',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the effect of the absence of Saccharomyces cerevisiae Rrm3p?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18725402',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12050116',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14690605',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21087929',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16418273',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19560424',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15037547',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19414561',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19277716',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15060144'],\n",
       "   'ideal_answer': [\"The Saccharomyces cerevisiae RRM3 gene encodes a 5' to 3' DNA helicase. While replication of most of the yeast genome was not dependent upon Rrm3p, in its absence, replication forks paused and often broke at an estimated 1400 discrete sites, including tRNA genes, centromeres, inactive replication origins, and transcriptional silencers. These replication defects were associated with activation of the intra-S phase checkpoint. Activation of the checkpoint was critical for viability of rrm3Delta cells, especially at low temperatures.\",\n",
       "    'The Saccharomyces cerevisiae RRM3 gene encodes a 5  to 3  DNA helicase. While replication of most of the yeast genome was not dependent upon Rrm3p, in its absence, replication forks paused and often broke at an estimated 1400 discrete sites, including tRNA genes, centromeres, inactive replication origins, and transcriptional silencers. These replication defects were associated with activation of the intra-S phase checkpoint. Activation of the checkpoint was critical for viability of rrm3Delta cells, especially at low temperatures  '],\n",
       "   'exact_answer': ['Lack of the yeast Rrm3p DNA helicase causes replication defects at multiple sites within ribosomal DNA (rDNA), including at the replication fork barrier (RFB).'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D012441'],\n",
       "   'type': 'factoid',\n",
       "   'id': '553ca8d8f32186855800000d',\n",
       "   'snippets': [{'offsetInBeginSection': 585,\n",
       "     'offsetInEndSection': 929,\n",
       "     'text': 'Twice as many pause sites were identified in rrm3 compared with wild-type cells, as pausing in this strain occurred at both highly transcribed RNA polymerase II genes and the previously identified protein DNA complexes. ORFs of highly transcribed RNA polymerase II genes are a class of natural pause sites that are not exacerbated in rrm3 cells',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19560424',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 457,\n",
       "     'text': 'The DNA helicase Rrm3 promotes replication fork progression through >1000 discrete genomic regions and represses the cDNA-mediated mobility of the Ty1 retrotransposon. We explored the connection between DNA replication and Ty1 retromobility by investigating the basis of increased retromobility in an rrm3 mutant. Even though Ty1 cDNA levels are increased in the absence of RRM3, neither the level nor target-site specificity of cDNA integration was altered',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19414561',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1095,\n",
       "     'offsetInEndSection': 1261,\n",
       "     'text': 'We propose that RNA:DNA hybrid regions within nascent retrotransposition events block replication in an rrm3 mutant, leading to chromosome breaks within Ty1 sequences',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19414561',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 150,\n",
       "     'offsetInEndSection': 447,\n",
       "     'text': 'We demonstrate that the inefficient mtDNA replication process of mutant yeast cells lacking the PIF1 DNA helicase is partly rescued in the absence of the DNA helicase RRM3. The rescue effect is likely due to the increase in the deoxynucleoside triphosphates (dNTPs) pool caused by the lack of RRM3',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19277716',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 972,\n",
       "     'offsetInEndSection': 1309,\n",
       "     'text': ' However, the essential nuclear function of Pfh1p could be supplied by Rrm3p. Expression of Rrm3p suppressed the accumulation of DNA damage foci but not the hydroxyurea sensitivity of cells depleted of nuclear Pfh1p. Together, these data demonstrate that Pfh1p has essential roles in the replication of both nuclear and mitochondrial DNA',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18725402',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 845,\n",
       "     'offsetInEndSection': 1100,\n",
       "     'text': ' In the absence of the Rrm3p helicase, there was a slight enhancement of fork arrest at the Ter sites. Simultaneous deletions of the TOF1 (or CSM3), and the RRM3 genes restored fork arrest by removing both the fork-releasing and fork-protection activities',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16418273',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 235,\n",
       "     'text': \"Rrm3p is a 5'-to-3' DNA helicase that helps replication forks traverse protein-DNA complexes. Its absence leads to increased fork stalling and breakage at over 1,000 specific sites located throughout the Saccharomyces cerevisiae genome\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15060144',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 877,\n",
       "     'offsetInEndSection': 1043,\n",
       "     'text': 'These data suggest a model in which the stalled and broken forks generated in rrm3 cells activate a checkpoint response that provides time for fork repair and restart',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15060144',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'Lack of the yeast Rrm3p DNA helicase causes replication defects at multiple sites within ribosomal DNA (rDNA), including at the replication fork barrier (RFB)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15037547',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 535,\n",
       "     'text': \"The Saccharomyces cerevisiae RRM3 gene encodes a 5' to 3' DNA helicase. While replication of most of the yeast genome was not dependent upon Rrm3p, in its absence, replication forks paused and often broke at an estimated 1400 discrete sites, including tRNA genes, centromeres, inactive replication origins, and transcriptional silencers. These replication defects were associated with activation of the intra-S phase checkpoint. Activation of the checkpoint was critical for viability of rrm3Delta cells, especially at low temperatures\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14690605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 754,\n",
       "     'offsetInEndSection': 926,\n",
       "     'text': 'These data indicate that the Rrm3p DNA helicase helps replication forks traverse protein-DNA complexes, naturally occurring impediments that are encountered in each S phase',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14690605',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 232,\n",
       "     'text': \"In wild-type Saccharomyces cerevisiae, replication forks slowed during their passage through telomeric C(1-3)A/TG(1-3) tracts. This slowing was greatly exacerbated in the absence of RRM3, shown here to encode a 5' to 3' DNA helicase\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12050116',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 397,\n",
       "     'offsetInEndSection': 579,\n",
       "     'text': ' Loss of Rrm3p also resulted in replication fork pausing at specific sites in subtelomeric DNA, such as at inactive replication origins, and at internal tracts of C(1-3)A/TG(1-3) DNA',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12050116',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 157,\n",
       "     'text': 'Local chromatin structure at the ribosomal DNA causes replication fork pausing and genome instability in the absence of the S. cerevisiae DNA helicase Rrm3p.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15037547',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'What are the functions of the ESCRT machinery?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25637630',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25624505',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24148098',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26339479',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25099357',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24482116',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26040713',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24641493',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25690770',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24109596'],\n",
       "   'ideal_answer': ['The endosomal sorting complexes required for transport (ESCRT) are needed for three distinct cellular functions in higher eukaryotes: (i) Multivesicular body formation for the degradation of transmembrane proteins in lysosomes, (ii) midbody abscission during cytokinesis and (iii) retroviral budding.'],\n",
       "   'type': 'summary',\n",
       "   'id': '570a8dcfcf1c325851000027',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 412,\n",
       "     'text': 'The endosomal sorting complexes required for transport (ESCRTs) collectively comprise a machinery that was first known for its function in the degradation of transmembrane proteins in the endocytic pathway of eukaryotic cells. Since their discovery, however, ESCRTs have been recognized as playing important roles at the plasma membrane, which appears to be the original site of function for the ESCRT machinery.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26339479',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 169,\n",
       "     'text': 'Endosomal sorting complexes required for transport (ESCRTs) are involved in the formation of multivesicular bodies and sorting of targeted proteins to the yeast vacuole.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25690770',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 191,\n",
       "     'text': 'Disassembly of the endosomal sorting complex required for transport (ESCRT) machinery from biological membranes is a critical final step in cellular processes that require the ESCRT function.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25637630',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 99,\n",
       "     'offsetInEndSection': 259,\n",
       "     'text': 'Evidence has accumulated revealing that efficient autophagic degradation requires functional endosomal sorting complex required for transport (ESCRT) machinery.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25624505',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 176,\n",
       "     'text': 'HIV-1 virions assemble at the plasma membrane of mammalian cells and recruit the endosomal sorting complex required for transport (ESCRT) machinery to enable particle release. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25099357',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 213,\n",
       "     'offsetInEndSection': 386,\n",
       "     'text': 'We found that endosomal sorting complex required for transport (ESCRT), involved previously in membrane budding and fission, plays a critical role in plasma membrane repair.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24482116',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 195,\n",
       "     'text': 'The ESCRT (endosomal sorting complex required for transport) machinery is known to sort ubiquitinated transmembrane proteins into vesicles that bud into the lumen of multivesicular bodies (MVBs).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24148098',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 301,\n",
       "     'text': 'The endosomal sorting complexes required for transport (ESCRT) are needed for three distinct cellular functions in higher eukaryotes: (i) Multivesicular body formation for the degradation of transmembrane proteins in lysosomes, (ii) midbody abscission during cytokinesis and (iii) retroviral budding. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24641493',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 274,\n",
       "     'text': 'Recently it was shown that both recycling endosome and endosomal sorting complex required for transport (ESCRT) components are required for cytokinesis, in which they are believed to act in a sequential manner to bring about secondary ingression and abscission, respectively',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24109596',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which are the enzymes involved in the control of tubulin acetylation?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23798680',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21677656',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17574768',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22700584',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23126280',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22046262',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18697214',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17868033',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20520769',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22972992'],\n",
       "   'ideal_answer': ['Acetyltransferase MEC-17, and deacetylases SIRT2 (Sirtuin 2), HDAC6 (histone deacetylase 6) and dTip60 are known to control the levels of tubulin acetylation.'],\n",
       "   'exact_answer': [['MEC-17'],\n",
       "    ['SIRT2 (Sirtuin 2)'],\n",
       "    ['HDAC6 (histone deacetylase 6)'],\n",
       "    ['dTip60']],\n",
       "   'concepts': ['http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0007021',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0015631',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D014404',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0071929',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D000107',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0045298'],\n",
       "   'type': 'list',\n",
       "   'id': '5322d3cb9b2d7acc7e000010',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'Inhibition of HDAC6 deacetylase activity increases its binding with microtubules and suppresses microtubule dynamic instability in MCF-7 cells',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23798680',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 333,\n",
       "     'offsetInEndSection': 454,\n",
       "     'text': 'HDAC6, the tubulin deacetylase, plays a key role in maintaining typical distribution of acetylated microtubules in cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23798680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 645,\n",
       "     'offsetInEndSection': 924,\n",
       "     'text': 'We found that whereas both pharmacological inhibition of HDAC6 as well as its depletion enhance microtubule acetylation, only pharmacological inhibition of HDAC6 activity leads to an increase in microtubule stability against cold and nocodazole-induced depolymerizing conditions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23798680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1397,\n",
       "     'offsetInEndSection': 1546,\n",
       "     'text': 'The evidence presented in this study indicated that the increased binding of HDAC6, rather than the acetylation per se, causes microtubule stability.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23798680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1547,\n",
       "     'offsetInEndSection': 1727,\n",
       "     'text': 'The results are in support of a hypothesis that in addition to its deacetylase function, HDAC6 might function as a MAP that regulates microtubule dynamics under certain conditions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23798680',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 527,\n",
       "     'offsetInEndSection': 684,\n",
       "     'text': 'Moreover, acetylation of α-tubulin is under the control of the acetyltransferase MEC-17 and deacetylases SIRT2 (Sirtuin 2) and HDAC6 (histone deacetylase 6).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23126280',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 99,\n",
       "     'text': 'MEC-17 deficiency leads to reduced α-tubulin acetylation and impaired migration of cortical neurons',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22972992',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 387,\n",
       "     'offsetInEndSection': 546,\n",
       "     'text': 'MEC-17 is a newly discovered α-tubulin acetyltransferase that has been found to play a major role in the acetylation of α-tubulin in different species in vivo.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22972992',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1277,\n",
       "     'offsetInEndSection': 1516,\n",
       "     'text': 'Thus, MEC-17, which regulates the acetylation of α-tubulin, appears to control the migration and morphological transition of cortical neurons. This finding reveals the importance of MEC-17 and α-tubulin acetylation in cortical development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22972992',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1248,\n",
       "     'offsetInEndSection': 1327,\n",
       "     'text': 'NT secretion is prevented by overexpression of HDAC6, an α-tubulin deacetylase.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22700584',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1135,\n",
       "     'offsetInEndSection': 1296,\n",
       "     'text': 'Moreover, α-tubulin acetylation levels of microtubules specifically extending into the terminal synaptic boutons are reduced in response to dTip60 HAT reduction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22046262',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 262,\n",
       "     'offsetInEndSection': 550,\n",
       "     'text': 'In this study, angiotensin II induced disassembly and deacetylation of α-tubulin, which were blocked by pretreatment with an angiotensin II type 1 receptor blocker losartan and a sirtuin class deacetylase inhibitor sirtinol, and by depletion of a deacetylase SIRT2 using RNA interference.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21677656',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1565,\n",
       "     'offsetInEndSection': 1793,\n",
       "     'text': 'These data show that angiotensin II and mechanical stretch stimulate microtubule redistribution and deacetylation via SIRT2 in endothelial cells, suggesting the emerging role of SIRT2 in hypertension-induced vascular remodeling.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21677656',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 282,\n",
       "     'text': 'Tubulin is a major substrate of the cytoplasmic class II histone deacetylase HDAC6. Inhibition of HDAC6 results in higher levels of acetylated tubulin and enhanced binding of the motor protein kinesin-1 to tubulin, which promotes transport of cargoes along microtubules.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20520769',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': 'Alcohol-induced alterations in hepatic microtubule dynamics can be explained by impaired histone deacetylase 6 function',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18697214',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 892,\n",
       "     'offsetInEndSection': 1051,\n",
       "     'text': 'Addition of trichostatin A (TSA), an HDAC6 inhibitor, induced microtubule acetylation to the same extent as in ethanol-treated cells (approximately threefold).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18697214',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1528,\n",
       "     'offsetInEndSection': 1763,\n",
       "     'text': 'Interestingly, HDAC6 from ethanol-treated cells was able to bind and deacetylate exogenous tubulin to the same extent as control, suggesting that ethanol-induced tubulin modifications prevented HDAC6 binding to endogenous microtubules.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18697214',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1776,\n",
       "     'offsetInEndSection': 1918,\n",
       "     'text': 'We propose that lower HDAC6 levels combined with decreased microtubule binding lead to increased tubulin acetylation in ethanol-treated cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18697214',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1464,\n",
       "     'offsetInEndSection': 1744,\n",
       "     'text': 'The results also showed that both pan-HDAC and class-I-specific inhibitor treatment resulted in increased acetylation of histones, but only pan-HDAC inhibitor treatment resulted in increased tubulin acetylation, which is in agreement with their activity towards the HDAC6 isoform.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17868033',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 575,\n",
       "     'offsetInEndSection': 969,\n",
       "     'text': 'Nicotinamide but not 3-aminobenzamide, an inhibitor for poly(ADP)ribose polymerase, enhanced tubulin acetylation and resistance to axonal degeneration in cultured cerebellar granule cells from wild-type (WT) mice, suggesting that mammalian Sir2-related protein (SIRT) 2, a nicotinamide adenine dinucleotide (NAD)--dependent tubulin deacetylase, could modulate resistance to axonal degeneration.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17574768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1065,\n",
       "     'offsetInEndSection': 1188,\n",
       "     'text': 'Moreover, SIRT2 overexpression abrogated microtubule hyperacetylation and resistance to axonal degeneration in these cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17574768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1189,\n",
       "     'offsetInEndSection': 1371,\n",
       "     'text': 'Conversely, SIRT2 knockdown by using a lentiviral vector expressing small interfering RNA, enhanced microtubule acetylation and resistance to axonal degeneration in WT granule cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17574768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1372,\n",
       "     'offsetInEndSection': 1563,\n",
       "     'text': 'Taken together, these results suggest that SIRT2-mediated tubulin deacetylation is involved in both microtubule hyperacetylation and resistance to axonal degeneration in Wld(S) granule cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17574768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 846,\n",
       "     'offsetInEndSection': 954,\n",
       "     'text': 'Compared to control cultures, higher levels of acetylated tubulin were found in neurons treated with tubacin',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20520769',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which are the most commonly reported pathological states associated with the formation of DNA G-quadruplexes?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23264878',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23175609',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18426915',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16652154',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15138591',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22303960',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22673230',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23423380',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22065584',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23161677'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2004/02/skos/core#definition',\n",
       "     's': 'http://linkedlifedata.com/resource/geneontology/id/GO:0071919',\n",
       "     'o': '\"A DNA metabolic process that results in the formation of G-quadruplex DNA structures, in which groups of four guanines adopt a flat, cyclic Hoogsteen hydrogen-bonding arrangement known as a guanine tetrad or G-quartet. The stacking of several layers of G-quartets forms G-quadruplexes, in which one or more DNA single strands are assembled in parallel and/or antiparallel, with interactions that can be either intra- or intermolecular in nature.\" [GOC:sre, PMID:20098422]'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#definition',\n",
       "     's': 'http://linkedlifedata.com/resource/geneontology/id/GO:0002151',\n",
       "     'o': '\"Interacting selectively and non-covalently with G-quadruplex RNA structures, in which groups of four guanines adopt a flat, cyclic hydrogen-bonding arrangement known as a guanine tetrad.\" [PMID:18294969, PMID:18568163, PMID:19330720]'}],\n",
       "   'ideal_answer': [\"There is a growing recognition for the profound role of G-quadruplexes in a wide spectrum of diseases, such as cancer, diabetes and cardiovascular disease. Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) were recently shown to be caused by expansion of a (GGGGCC)n/(GGCCCC)n repeat in the C9ORF72 gene.  Treatment with a G-quadruplex interactive ligand was shown to achieve antifibrotic action. G-quadruplex forming sequences have also been linked with ADAM10 a primary candidate for anti-amyloidogenic activity in Alzheimer's. A G-quadruplex-interactive agent, telomestatin (SOT-095), induces telomere shortening with apoptosis and enhances chemosensitivity in acute myeloid leukemia.\"],\n",
       "   'exact_answer': [['Amyotrophic lateral sclerosis (ALS)'],\n",
       "    ['frontotemporal dementia (FTD)'],\n",
       "    [\"Alzheimer's disease\"],\n",
       "    ['fibrosis'],\n",
       "    ['acute myeloid leukemia']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D054856',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0002151',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0071919',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:5353',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0051880',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:3463'],\n",
       "   'type': 'list',\n",
       "   'id': '51600ab3298dcd4e51000036',\n",
       "   'snippets': [{'offsetInBeginSection': 157,\n",
       "     'offsetInEndSection': 324,\n",
       "     'text': 'Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) were recently shown to be caused by expansion of a (GGGGCC)n/(GGCCCC)n repeat in the C9ORF72 gene',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23423380',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 714,\n",
       "     'text': 'Myocardial fibrosis is a key pathological change in a variety of heart diseases contributing to the development of heart failure, arrhythmias, and sudden death. Recent studies have shown that relaxin prevents and reverses cardiac fibrosis. Endogenous expression of relaxin was elevated in the setting of heart disease; the extent of such up-regulation, however, is insufficient to exert compensatory actions, and the mechanism regulating relaxin expression is poorly defined. In the rat relaxin-1 (RLN1, Chr1) gene promoter region we found presence of repeated guanine (G)-rich sequences, which allowed formation and stabilization of G-quadruplexes with the addition of a G-quadruplex interactive ligand berberine.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22673230',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 98,\n",
       "     'text': 'Up-regulating relaxin expression by G-quadruplex interactive ligand to achieve antifibrotic action',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22673230',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1685,\n",
       "     'offsetInEndSection': 1804,\n",
       "     'text': 'Our findings document a novel therapeutic strategy for fibrosis through up-regulating expression of endogenous relaxin.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22673230',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 368,\n",
       "     'offsetInEndSection': 468,\n",
       "     'text': 'We identified a G-rich sequence within exon 3 of BACE1 involved in controlling splice site selection',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22303960',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 263,\n",
       "     'text': 'β-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is the transmembrane aspartyl protease that catalyzes the first cleavage step in the proteolysis of the APP to the amyloid β-protein (Aβ), a process involved in the pathogenesis of Alzheimer disease',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22303960',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 347,\n",
       "     'text': 'Anti-amyloidogenic processing of the amyloid precursor protein APP by α-secretase prevents formation of the amyloid-β peptide, which accumulates in senile plaques of Alzheimer disease patients. α-Secretase belongs to the family of a disintegrin and metalloproteases (ADAMs), and ADAM10 is the primary candidate for this anti-amyloidogenic activity',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22065584',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 700,\n",
       "     'offsetInEndSection': 937,\n",
       "     'text': \"Using circular dichroism spectroscopy, we demonstrate that a G-rich region between nucleotides 66 and 94 of the ADAM10 5'-UTR forms a highly stable, intramolecular, parallel G-quadruplex secondary structure under physiological conditions\",\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22065584',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'FANCJ helicase defective in Fanconia anemia and breast cancer unwinds G-quadruplex DNA to defend genomic stability.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18426915',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'Telomerase inhibition with a novel G-quadruplex-interactive agent, telomestatin: in vitro and in vivo studies in acute leukemia',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16652154',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 120,\n",
       "     'offsetInEndSection': 304,\n",
       "     'text': 'Recently, we have demonstrated that treatment with a G-quadruplex-interactive agent, telomestatin reproducibly inhibited telomerase activity in the BCR-ABL-positive leukemic cell lines',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16652154',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 156,\n",
       "     'text': 'A G-quadruplex-interactive agent, telomestatin (SOT-095), induces telomere shortening with apoptosis and enhances chemosensitivity in acute myeloid leukemia',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15138591',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 432,\n",
       "     'offsetInEndSection': 612,\n",
       "     'text': 'We examined G-quadruplex interactive agent, telomestatin (SOT-095), for its ability to inhibit the proliferation of human leukemia cells, including freshly obtained leukemia cells.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15138591',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'Does MVIIA and MVIIC bind to the same calcium channel?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/8898826',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9063691',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10694509',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20820758',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10648826',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8853221',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16546143',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9315745',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11273667',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7804605'],\n",
       "   'ideal_answer': ['No, the omega-conotoxin MVIIC blocks P/Q-type calcium channels with high affinity and N-type calcium channels with low affinity, while the highly homologous omega-conotoxin MVIIA blocks only N-type calcium channels.'],\n",
       "   'exact_answer': 'no',\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2016/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015220'],\n",
       "   'type': 'yesno',\n",
       "   'id': '56cf2a153975bb303a000004',\n",
       "   'snippets': [{'offsetInBeginSection': 450,\n",
       "     'offsetInEndSection': 757,\n",
       "     'text': 'We examined the post-pubertal behavioral effects of neonatal (postnatal day 7) medial prefrontal cortex infusion of either vehicle or N-type and P/Q-type presynaptic voltage-dependent calcium channel blockers (omega-conotoxins MVIIA and MVIIC respectively; 6.8 and 45 pmol infused respectively) in rat pups.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16546143',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 354,\n",
       "     'offsetInEndSection': 614,\n",
       "     'text': 'Additionally, the number of binding sites for radioligands labelling L- ([3H]nitrendipine), N- ([125I]omega-conotoxin MVIIA) and P/Q-type ([125I]omega-conotoxin MVIIC) Ca2+ channels was assessed in the rat retina and, for further comparison, in the rat cortex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11273667',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'Omega-conotoxin MVIIC (MVIIC) blocks P/Q-type calcium channels with high affinity and N-type calcium channels with low affinity, while the highly homologous omega-conotoxin MVIIA blocks only N-type calcium channels.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10648826',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1726,\n",
       "     'offsetInEndSection': 1939,\n",
       "     'text': 'However, omega-conotoxin MVIIC seems to bind to sites different from those recognised by omega-conotoxin GVIA and MVIIA, which are markedly differentiated by their Ca2+ requirements for binding to their receptors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9063691',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 159,\n",
       "     'text': 'Despite their high sequence homology, the peptide neurotoxins omega-conotoxin MVIIA and MVIIC selectively block N- and P/Q-type calcium channels, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9315745',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 209,\n",
       "     'text': 'Replacement of the N-terminal half of omega-conotoxin MVIIC, a peptide blocker of P/Q-type calcium channels, with that of omega-conotoxin MVIIA significantly increased the affinity for N-type calcium channels.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10694509',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 405,\n",
       "     'text': ' Omega-conotoxin MVIIC (MVIIC) blocks P/Q-type calcium channels with high affinity and N-type calcium channels with low affinity, while the highly homologous omega-conotoxin MVIIA blocks only N-type calcium channels. We wished to obtain MVIIC analogues more selective for P/Q-type calcium channels than MVIIC to elucidate structural differences among the channels, which discriminate the omega-conotoxins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10648826',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1532,\n",
       "     'offsetInEndSection': 1643,\n",
       "     'text': 'omega-conotoxin MVIIC seems to bind to sites different from those recognised by omega-conotoxin GVIA and MVIIA,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9063691',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the mode of inheritance in Fanconi anemia?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/8987005',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3369448',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17924555',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22959745',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15502827',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/939547',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19549413',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11128621',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22693659',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16675878'],\n",
       "   'ideal_answer': ['Fanconi anemia (FA) is a rare inherited syndrome with diverse clinical symptoms including developmental defects, short stature, bone marrow failure, and a high risk of malignancies. Fifteen genetic subtypes have been distinguished so far. The mode of inheritance for all subtypes is autosomal recessive, except for FA-B, which is X-linked  ',\n",
       "    'Fanconi anemia (FA) is a rare inherited syndrome. So far, fifteen genetic subtypes have been distinguished. The mode of inheritance for all subtypes is autosomal recessive, except for FANCB, which is X-linked.'],\n",
       "   'exact_answer': ['Autosomal recessive, except for FANCB, which is X-linked.'],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:1062',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D005199',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:13636'],\n",
       "   'type': 'factoid',\n",
       "   'id': '54ede95c94afd6150400000a',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 338,\n",
       "     'text': 'Fanconi anemia (FA) is a rare inherited syndrome with diverse clinical symptoms including developmental defects, short stature, bone marrow failure, and a high risk of malignancies. Fifteen genetic subtypes have been distinguished so far. The mode of inheritance for all subtypes is autosomal recessive, except for FA-B, which is X-linked',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22693659',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'Fanconi anemia (FA) is a recessively inherited syndrome with predisposition to bone marrow failure and malignancies',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17924555',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 433,\n",
       "     'offsetInEndSection': 626,\n",
       "     'text': 'Original Fanconi anemia data, for which no information about the ascertainment was available, were then analyzed, with results that confirmed a monogenic autosomal recessive mode of inheritance',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3369448',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 309,\n",
       "     'text': \"The formal genetics of Fanconi's anemia were investigated on the basis of 21 families from different European countries, and of 69 families from the literature. CONCLUSIONS: 1. The result of segregation analysis is compatible with the hypothesis of a simple autosomal recessive mode of inheritance\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/939547',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 429,\n",
       "     'offsetInEndSection': 624,\n",
       "     'text': ' Original Fanconi anemia data, for which no information about the ascertainment was available, were then analyzed, with results that confirmed a monogenic autosomal recessive mode of inheritance.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3369448',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 63,\n",
       "     'text': 'X-linked inheritance of Fanconi anemia complementation group B.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15502827',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 129,\n",
       "     'text': 'Fanconi anemia (FA) is a rare genetic disease with both autosomal and X-linked inheritance, characterized by genomic instability.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19549413',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 860,\n",
       "     'offsetInEndSection': 1001,\n",
       "     'text': ' X-linked inheritance has important consequences for genetic counseling of families with Fanconi anemia belonging to complementation group B.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15502827',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 433,\n",
       "     'offsetInEndSection': 627,\n",
       "     'text': 'Original Fanconi anemia data, for which no information about the ascertainment was available, were then analyzed, with results that confirmed a monogenic autosomal recessive mode of inheritance.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3369448',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 264,\n",
       "     'text': 'Fanconi anemia (FA), a recessive syndrome with both autosomal and X-linked inheritance, features diverse clinical symptoms, such as progressive bone marrow failure, hypersensitivity to DNA cross-linking agents, chromosomal instability and susceptibility to cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16675878',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 865,\n",
       "     'offsetInEndSection': 1210,\n",
       "     'text': 'X-linked inheritance has important consequences for genetic counseling of families with Fanconi anemia belonging to complementation group B. Its presence as a single active copy and essentiality for a functional Fanconi anemia-BRCA pathway make FANCB a potentially vulnerable component of the cellular machinery that maintains genomic integrity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15502827',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 90,\n",
       "     'text': 'Fanconi anemia (FA) is a rare genetic disease, transmitted in an autosomal recessive mode.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22959745',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 77,\n",
       "     'offsetInEndSection': 200,\n",
       "     'text': 'Report of two early-onset cases with Fanconis anemia-like phenotypes suggesting an autosomal-recessive inheritance pattern.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11128621',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 264,\n",
       "     'offsetInEndSection': 436,\n",
       "     'text': 'In the first family, an association with Fanconis anemia was observed in three of seven pregnancies (2 boys, 1 girl) suggesting an autosomal recessive mode of transmission.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8987005',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 239,\n",
       "     'offsetInEndSection': 339,\n",
       "     'text': 'The mode of inheritance for all subtypes is autosomal recessive, except for FA-B, which is X-linked.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22693659',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 867,\n",
       "     'offsetInEndSection': 1007,\n",
       "     'text': 'X-linked inheritance has important consequences for genetic counseling of families with Fanconi anemia belonging to complementation group B.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15502827',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 239,\n",
       "     'offsetInEndSection': 338,\n",
       "     'text': 'The mode of inheritance for all subtypes is autosomal recessive, except for FA-B, which is X-linked',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22693659',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 264,\n",
       "     'offsetInEndSection': 436,\n",
       "     'text': \"In the first family, an association with Fanconi's anemia was observed in three of seven pregnancies (2 boys, 1 girl) suggesting an autosomal recessive mode of transmission\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8987005',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 867,\n",
       "     'offsetInEndSection': 1006,\n",
       "     'text': 'X-linked inheritance has important consequences for genetic counseling of families with Fanconi anemia belonging to complementation group B',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15502827',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 89,\n",
       "     'text': 'Fanconi anemia (FA) is a rare genetic disease, transmitted in an autosomal recessive mode',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22959745',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is arimoclomol a co-inducer of the heat shock response?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22591194',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18551622',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20582873',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19183864',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17656567',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24853414',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23978556',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18673445',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19938902',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23393146'],\n",
       "   'ideal_answer': ['Yes, arimoclomol is a hydroxylamine derivative, a group of compounds which have unique properties as co-inducers of heat shock protein expression, but only under conditions of cellular stress.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '56f7d9dd09dd18d46b000014',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 188,\n",
       "     'text': 'Arimoclomol is a hydroxylamine derivative, a group of compounds which have unique properties as co-inducers of heat shock protein expression, but only under conditions of cellular stress. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23978556',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 628,\n",
       "     'offsetInEndSection': 805,\n",
       "     'text': 'In this review we summarize the evidence for the neuroprotective effects of enhanced heat shock protein expression by Arimoclomol and other inducers of the Heat Shock Response. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23978556',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1061,\n",
       "     'offsetInEndSection': 1121,\n",
       "     'text': 'arimoclomol, a co-inducer of the heat shock stress response,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23393146',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1329,\n",
       "     'offsetInEndSection': 1440,\n",
       "     'text': 'The heat-shock response (HSR) was activated in P23H retinae, and this was enhanced with arimoclomol treatment. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24853414',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 143,\n",
       "     'offsetInEndSection': 246,\n",
       "     'text': ' We also assessed these functions in mice treated with a known heat shock protein inducer, arimoclomol.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22591194',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 736,\n",
       "     'offsetInEndSection': 897,\n",
       "     'text': ' Under conditions of excessive stress, arimoclomol induces amplification of the cytoprotective heat shock response in order to protect motor neurons from death. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20582873',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 416,\n",
       "     'offsetInEndSection': 487,\n",
       "     'text': 'Although both arimoclomol and celastrol induced the expression of Hsp70',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19183864',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 201,\n",
       "     'text': 'Arimoclomol, an amplifier of heat shock protein expression involved in cellular stress response, has emerged as a potential therapeutic candidate in amyotrophic lateral sclerosis (ALS) in recent years.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19938902',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1337,\n",
       "     'offsetInEndSection': 1485,\n",
       "     'text': 'The mechanism of action of arimoclomol involves potentiation of the heat shock response, and treatment with arimoclomol increased Hsp70 expression. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18673445',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 148,\n",
       "     'text': 'Arimoclomol is an investigational drug for amyotrophic lateral sclerosis (ALS) that amplifies heat shock protein gene expression during cell stress.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18551622',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 359,\n",
       "     'offsetInEndSection': 554,\n",
       "     'text': 'Arimoclomol, a coinducer of heat shock proteins, delayed progression of amyotrophic lateral sclerosis (ALS) in a mouse model in which motor neurons in the spinal cord and motor cortex degenerate.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17656567',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the association between NT-proBNP and cognitive function?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25142900',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22479261',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24942833',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24333505',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21526197',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22973461',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21683832',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23579182',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23384944',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20727348'],\n",
       "   'ideal_answer': ['Greater NT-proBNP serum concentration is associated with poorer cognitive function and cognitive decline. In community-dwelling older adults, greater NT-proBNP levels were strongly associated with poor cognitive function independently from age, sex, education, hypertension, body mass index, exercise, alcohol use, smoking, low density lipoprotein cholesterol, creatinine clearance, and previous cardiovascular disease. However, other authors did not find an association between NT-proBNP and severe cognitive impairment (SCI).'],\n",
       "   'type': 'summary',\n",
       "   'id': '55032785e9bde69634000031',\n",
       "   'snippets': [{'offsetInBeginSection': 1027,\n",
       "     'offsetInEndSection': 1175,\n",
       "     'text': 'RESULTS: Measures of cognitive decline were significantly associated with stroke, NT-proBNP, ABI, and cIMT, but not with nonstroke vascular events. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23579182',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 990,\n",
       "     'offsetInEndSection': 1185,\n",
       "     'text': 'SCI was present at baseline more often in NYHA IV patients compared with NYHA II [odds ratio 2.94; 95% confidence interval (CI) 1.15-7.51, P = 0.025], but it was not related to NT-proBNP levels. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23384944',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1022,\n",
       "     'offsetInEndSection': 2049,\n",
       "     'text': \"Comparing extreme quintiles of NT-proBNP, subjects in the highest quintile were more likely to have reduced cognitive ability (within the lowest tertile of 'g') and 'possible' depression (HADS depression ≥8) (OR 1.80; 95% CI: 1.20, 2.70; p\\u200a=\\u200a0.005 and OR 2.18; 95% CI: 1.28, 3.71; p\\u200a=\\u200a0.004, respectively). Associations persisted when pre-morbid ability was adjusted for, but as expected were no longer statistically significant following the adjustment for diabetes-related and vascular co-variates (β -0.02, 95% CI -0.07 to 0.03, p>0.05 for 'g'; β 0.03, 95% CI -0.02 to 0.07, p>0.05 for depression scores). CONCLUSION: Raised plasma NT-proBNP was weakly but statistically significantly associated with poorer cognitive function and depression. The prospective phases of the ET2DS will help determine whether or not NT-proBNP can be considered a risk marker for subsequent cognitive impairment and incident depression and whether it provides additional information over and above traditional risk factors for these conditions.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22973461',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 430,\n",
       "     'offsetInEndSection': 711,\n",
       "     'text': 'RESULTS AND CONCLUSION: Patients with vascular disease and elevated serum NT-proBNP level had a lower cognition level, shorter survival time, lower renal function and a higher percentage of pathological brain imaging than patients with vascular disease and normal NT-proBNP level. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22479261',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 224,\n",
       "     'text': 'BACKGROUND: Natriuretic peptides have prognostic value across a wide spectrum of cardiovascular diseases and may predict cognitive dysfunction in patients with cardiovascular disease, even in the absence of previous stroke. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21683832',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1114,\n",
       "     'offsetInEndSection': 1711,\n",
       "     'text': 'In unadjusted analyses, all 3 cognitive function test scores were significantly associated with NT-proBNP levels (P<.001). After adjusting for age, sex, education, hypertension, body mass index, exercise, alcohol use, smoking, low density lipoprotein cholesterol, creatinine clearance, and previous cardiovascular disease, elevated NT-proBNP levels remained independently associated with poor cognitive performance on MMSE (odds ratio [OR] 2.0; 95% confidence interval [CI], 1.1-3.6; P=.02) and Trails B (OR 1.7; 95% CI, 1.2-2.7; P=.01), but not Category Fluency (OR 1.4; 95% CI, 0.9-2.2; P=.19). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21683832',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1798,\n",
       "     'offsetInEndSection': 1936,\n",
       "     'text': ' CONCLUSIONS: NT-proBNP levels were strongly and independently associated with poor cognitive function in community-dwelling older adults.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21683832',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 798,\n",
       "     'offsetInEndSection': 876,\n",
       "     'text': 'However, NT-proBNP level did not predict cognition as assessed by MMSE score. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20727348',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 778,\n",
       "     'offsetInEndSection': 897,\n",
       "     'text': 'However, N-terminal pro-brain natriuretic peptide (NT-proBNP) was significantly enhanced both in MCI and AD patients (1',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24333505',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 904,\n",
       "     'offsetInEndSection': 1113,\n",
       "     'text': 'In a second analysis of a sample of 110 subjects including younger healthy controls, we confirmed that NT-proBNP has the potential to be a stable candidate protein for both diagnosis and AD disease progression',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24333505',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1585,\n",
       "     'offsetInEndSection': 1708,\n",
       "     'text': 'Raised plasma NT-proBNP was weakly but statistically significantly associated with poorer cognitive function and depression',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22973461',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1086,\n",
       "     'offsetInEndSection': 1207,\n",
       "     'text': 'In unadjusted analyses, all 3 cognitive function test scores were significantly associated with NT-proBNP levels (P<.001)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21683832',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1772,\n",
       "     'offsetInEndSection': 1896,\n",
       "     'text': 'NT-proBNP levels were strongly and independently associated with poor cognitive function in community-dwelling older adults.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21683832',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'what is the role of MEF-2 in cardiomyocyte differentiation?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/19498465',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17158926',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8366095',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22850285',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23261540',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12663654',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22199256',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24091702',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10790371',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9857019'],\n",
       "   'triples': [{'p': 'http://purl.uniprot.org/core/classifiedWith',\n",
       "     's': 'http://purl.uniprot.org/uniprot/A2ICN5',\n",
       "     'o': 'http://purl.uniprot.org/go/0030154'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#exactMatch',\n",
       "     's': 'http://purl.uniprot.org/go/0030154',\n",
       "     'o': 'http://www.geneontology.org/go#GO:0030154'},\n",
       "    {'p': 'http://purl.uniprot.org/core/recommendedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/A2ICN5',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_413249434E3500C'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://purl.uniprot.org/go/0030154',\n",
       "     'o': 'cell differentiation'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_413249434E3500C',\n",
       "     'o': 'Myocyte-specific enhancer factor 2A'},\n",
       "    {'p': 'http://purl.uniprot.org/core/classifiedWith',\n",
       "     's': 'http://purl.uniprot.org/uniprot/A2VDZ3',\n",
       "     'o': 'http://purl.uniprot.org/go/0030154'},\n",
       "    {'p': 'http://purl.uniprot.org/core/recommendedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/A2VDZ3',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_413256445A3300B'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_413256445A3300B',\n",
       "     'o': 'Myocyte-specific enhancer factor 2A'},\n",
       "    {'p': 'http://purl.uniprot.org/core/classifiedWith',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q5REW7',\n",
       "     'o': 'http://purl.uniprot.org/go/0030154'},\n",
       "    {'p': 'http://purl.uniprot.org/core/recommendedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q5REW7',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_51355245573700A'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_51355245573700A',\n",
       "     'o': 'Myocyte-specific enhancer factor 2A'},\n",
       "    {'p': 'http://purl.uniprot.org/core/classifiedWith',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q9W6U8',\n",
       "     'o': 'http://purl.uniprot.org/go/0030154'},\n",
       "    {'p': 'http://purl.uniprot.org/core/recommendedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q9W6U8',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_513957365538009'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_513957365538009',\n",
       "     'o': 'Myocyte-specific enhancer factor 2A'},\n",
       "    {'p': 'http://purl.uniprot.org/core/classifiedWith',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q8CFN5',\n",
       "     'o': 'http://purl.uniprot.org/go/0003211'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#exactMatch',\n",
       "     's': 'http://purl.uniprot.org/go/0003211',\n",
       "     'o': 'http://www.geneontology.org/go#GO:0003211'},\n",
       "    {'p': 'http://purl.uniprot.org/core/recommendedName',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q8CFN5',\n",
       "     'o': 'http://linkedlifedata.com/resource/#_513843464E35001D'},\n",
       "    {'p': 'http://purl.uniprot.org/core/fullName',\n",
       "     's': 'http://linkedlifedata.com/resource/#_513843464E35001D',\n",
       "     'o': 'Myocyte-specific enhancer factor 2C'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://purl.uniprot.org/go/0003211',\n",
       "     'o': 'cardiac ventricle formation'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/relationontology/participateInBiologicalProcess',\n",
       "     's': 'http://purl.uniprot.org/uniprot/A2ICN5',\n",
       "     'o': 'http://linkedlifedata.com/resource/geneontology/id/GO:0030154'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/geneontology/id/GO:0030154',\n",
       "     'o': 'cell differentiation'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/relationontology/participateInBiologicalProcess',\n",
       "     's': 'http://purl.uniprot.org/uniprot/A2VDZ3',\n",
       "     'o': 'http://linkedlifedata.com/resource/geneontology/id/GO:0030154'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/relationontology/participateInBiologicalProcess',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q5REW7',\n",
       "     'o': 'http://linkedlifedata.com/resource/geneontology/id/GO:0030154'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/relationontology/participateInBiologicalProcess',\n",
       "     's': 'http://purl.uniprot.org/uniprot/Q9W6U8',\n",
       "     'o': 'http://linkedlifedata.com/resource/geneontology/id/GO:0030154'}],\n",
       "   'ideal_answer': ['The myocyte enhancer factor-2 (MEF2) proteins are MADS-box transcription factors that are essential for differentiation of all muscle lineages but their mechanisms of action remain largely undefined. MEF2C expression initiates cardiomyogenesis, resulting in the up-regulation of Brachyury T, bone morphogenetic protein-4, Nkx2-5, GATA-4, cardiac alpha-actin, and myosin heavy chain expression. Inactivation of the MEF2C gene causes cardiac developmental arrest and severe downregulation of a number of cardiac markers including atrial natriuretic factor (ANF). BMP-2, a regulator of cardiac development during embryogenesis, was shown to increase PI 3-kinase activity in cardiac precursor cells, resulting in increased expression of sarcomeric myosin heavy chain (MHC) and MEF-2A. Furthermore, expression of MEF-2A increased MHC expression in a PI 3-kinase-dependent manner. Other studies showed that Gli2 and MEF2C proteins form a complex, capable of synergizing on cardiomyogenesis-related promoters. Dominant interference of calcineurin/mAKAP binding blunts the increase in MEF2 transcriptional activity seen during myoblast differentiation, as well as the expression of endogenous MEF2-target genes. These findings show that MEF-2 can direct early stages of cell differentiation into a cardiomyogenic pathway.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D032383',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0035051',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D064326'],\n",
       "   'type': 'summary',\n",
       "   'id': '532f043ed6d3ac6a34000023',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 297,\n",
       "     'text': 'The growth and differentiation factor bone morphogenetic protein-2 (BMP-2) regulates cardiac development during vertebrate embryogenesis. In cardiac precursor cells, BMP-2 has recently been shown to induce expression of cardiac transcription factors, including myocyte enhancer factor 2A (MEF-2A).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12663654',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 525,\n",
       "     'offsetInEndSection': 689,\n",
       "     'text': 'BMP-2 increased PI 3-kinase activity in these cells in a time-dependent manner, resulting in increased expression of sarcomeric myosin heavy chain (MHC) and MEF-2A.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12663654',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1545,\n",
       "     'offsetInEndSection': 1637,\n",
       "     'text': 'Furthermore expression of MEF-2A increased MHC expression in a PI 3-kinase-dependent manner.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12663654',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 465,\n",
       "     'offsetInEndSection': 773,\n",
       "     'text': 'Gene expression profiling revealed decreased levels of transcription factors transforming growth factor-β/bone morphogenetic protein, serum response factor, GATA4, and myocyte enhancer factor 2, linked to increased Ca(2+)-dependent calcineurin activity and Notch1 signaling that impaired ESC differentiation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24091702',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 895,\n",
       "     'offsetInEndSection': 1221,\n",
       "     'text': 'Dominant interference of calcineurin/mAKAP binding blunts the increase in MEF2 transcriptional activity seen during myoblast differentiation, as well as the expression of endogenous MEF2-target genes. Furthermore, disruption of calcineurin binding to mAKAP in cardiac myocytes inhibits adrenergic-induced cellular hypertrophy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23261540',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 731,\n",
       "     'offsetInEndSection': 1088,\n",
       "     'text': 'Hypertrophic agonist stimulation of neonatal ventricular cardiomyocytes increased Mef2 expression by enhancing its translation, without changing its transcription or blocking degradation of the protein. MEF2 abundance was increased by Calcineurin overexpression in vivo and was reduced by Calcineurin inhibition in vitro, without affecting Mef2 mRNA levels.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22850285',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 735,\n",
       "     'offsetInEndSection': 1321,\n",
       "     'text': \"In addition, chromatin immunoprecipitation (ChIP) revealed association of Gli2 to the Mef2c gene, and of MEF2C to the Gli2 gene in differentiating P19 cells. Finally, co-immunoprecipitation studies showed that Gli2 and MEF2C proteins formed a complex, capable of synergizing on cardiomyogenesis-related promoters containing both Gli- and MEF2-binding elements. We propose a model whereby Gli2 and MEF2C bind each other's regulatory elements, activate each other's expression and form a protein complex that synergistically activates transcription, enhancing cardiac muscle development. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22199256',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 412,\n",
       "     'offsetInEndSection': 635,\n",
       "     'text': 'MC1568 arrests myogenesis by (i) decreasing myocyte enhancer factor 2D (MEF2D) expression, (ii) by stabilizing the HDAC4-HDAC3-MEF2D complex, and (iii) paradoxically, by inhibiting differentiation-induced MEF2D acetylation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19498465',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 331,\n",
       "     'text': 'The myocyte enhancer factor 2 (MEF2) family of transcription factors is not only important for controlling gene expression in normal cellular programs, like muscle differentiation, T-cell apoptosis, neuronal survival, and synaptic differentiation, but has also been linked to cardiac hypertrophy and other pathological conditions. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17158926',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1215,\n",
       "     'offsetInEndSection': 1534,\n",
       "     'text': 'Furthermore, the nuclear receptor corepressor SMRT (silencing mediator of retinoid acid and thyroid hormone receptor) stimulated the deacetylase activity of HDAC3 towards MEF2 and PCAF. Supporting the physical interaction and deacetylase activity, HDAC3 repressed MEF2-dependent transcription and inhibited myogenesis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17158926',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 200,\n",
       "     'text': 'The myocyte enhancer factor-2 (MEF2) proteins are MADS-box transcription factors that are essential for differentiation of all muscle lineages but their mechanisms of action remain largely undefined. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10790371',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 331,\n",
       "     'offsetInEndSection': 497,\n",
       "     'text': 'Inactivation of the MEF2C gene causes cardiac developmental arrest and severe downregulation of a number of cardiac markers including atrial natriuretic factor (ANF).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10790371',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 733,\n",
       "     'offsetInEndSection': 967,\n",
       "     'text': 'We provide evidence that MEF2 proteins are recruited to target promoters by the cell-specific GATA transcription factors, and that MEF2 potentiates the transcriptional activity of this family of tissue-restricted zinc finger proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10790371',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 585,\n",
       "     'offsetInEndSection': 737,\n",
       "     'text': 'During differentiation into cardiac muscle, Nkx2-5 expression resulted in the activation of myocyte enhancer factor 2C (MEF2C), but not MEF2A, -B, or -D',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9857019',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 901,\n",
       "     'offsetInEndSection': 1113,\n",
       "     'text': 'Similar to Nkx2-5, MEF2C expression initiated cardiomyogenesis, resulting in the up-regulation of Brachyury T, bone morphogenetic protein-4, Nkx2-5, GATA-4, cardiac alpha-actin, and myosin heavy chain expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9857019',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1114,\n",
       "     'offsetInEndSection': 1314,\n",
       "     'text': 'These findings indicate the presence of a positive regulatory network between Nkx2-5 and MEF2C and show that both factors can direct early stages of cell differentiation into a cardiomyogenic pathway.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9857019',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is targeted by Palbociclib?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/21815704',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21679088',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20197484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25701171',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25636162',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21806477',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21109448',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24216225',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26045340',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23300028'],\n",
       "   'ideal_answer': ['Necitumumab is a fully human IgG(1) monoclonal antibody directed at the epidermal growth factor receptor (EGFR). It is used for treatment of nonsmall cell lung cancer.'],\n",
       "   'exact_answer': ['epidermal growth factor receptor'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56c1f01cef6e394741000044',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': 'BACKGROUND: Necitumumab is a second-generation, recombinant, human immunoglobulin G1 EGFR antibody. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26045340',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 104,\n",
       "     'text': 'Epidermal growth factor receptor-directed monoclonal antibodies in nonsmall cell lung cancer: an update.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25636162',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 505,\n",
       "     'offsetInEndSection': 762,\n",
       "     'text': ' A Southwest Oncology Group study currently prospectively evaluates the predictive biomarkers for cetuximab. In the SQUIRE phase III trial, necitumumab added to cisplatin and gemcitabine increased the survival in patients with advanced squamous cell NSCLC. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25636162',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 152,\n",
       "     'text': 'BACKGROUND: Necitumumab is a second-generation recombinant human immunoglobulin G1 EGFR monoclonal antibody that competitively inhibits ligand binding. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25701171',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1231,\n",
       "     'offsetInEndSection': 1703,\n",
       "     'text': 'Numerous other drugs are in earlier stages of development for HNSCC treatment, including novel anti-EGFR mAbs (MEHD7945A, necitumumab, and RO5083945), small-molecule TKIs (vandetanib, icotinib, and CUDC-101), EGFR antisense, various add-on therapies to radiation and chemotherapy (bevacizumab, interleukin-12, lenalidomide, alisertib, and VTX-2337), and drugs (temsirolimus, everolimus, OSI-906, dasatinib, and PX-866) intended to overcome resistance to anti-EGFR agents. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24216225',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1072,\n",
       "     'offsetInEndSection': 1233,\n",
       "     'text': 'Necitumumab, a fully human monoclonal antibody, is currently evaluated in combination with chemotherapy in two phase III trials in patients with advanced NSCLC. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23300028',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1188,\n",
       "     'offsetInEndSection': 1291,\n",
       "     'text': 'Necitumumab is currently evaluated in combination with chemotherapy in two randomized phase III trials.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21109448',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 412,\n",
       "     'text': 'Joining cetuximab, sorafenib, afatinib, intedanib, and crizotinib in phase III development for non-small cell lung cancer (NSCLC) are ramucirumab (developed by ImClone, a subsidiary of Lilly), necitumumab (developed by ImClone and Bristol-Myers Squibb), and tivantinib (ARQ 197, developed by ArQule and Daiichi Sankyo). Necitumumab is a second-generation anti-EGFR monoclonal antibody (mAb) similar to cetuximab.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21815704',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 332,\n",
       "     'offsetInEndSection': 564,\n",
       "     'text': 'Necitumunab is a fully human IgG1 monoclonal antibody targeting EGFR, having the potential benefit of lower hypersensitivity reaction risk as compared with cetuximab and also equivalent antibody-dependent cell-mediated cytotoxicity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21679088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 123,\n",
       "     'text': 'Necitumumab, a monoclonal antibody directed against EGFR, is currently under development as a treatment for advanced NSCLC.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21806477',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 163,\n",
       "     'text': 'A phase I pharmacologic study of necitumumab (IMC-11F8), a fully human IgG1 monoclonal antibody directed against EGFR in patients with advanced solid malignancies.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20197484',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 362,\n",
       "     'text': 'PURPOSE: This study aimed to determine a maximum tolerated dose (MTD) and recommended dose for disease-directed studies of necitumumab (IMC-11F8), a fully human IgG(1) monoclonal antibody directed at the epidermal growth factor receptor, and to characterize the safety profile, pharmacokinetics, preliminary antitumor activity, and immunogenicity of necitumumab.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20197484',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Does neuroglobin has neuroprotective properties in the setting of traumatic brain injury?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23658829',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24491879',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20345238',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20476562',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20544857',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25624825',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21915648',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22703519',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22664218',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24642455'],\n",
       "   'ideal_answer': ['Yes, neuroglobin has neuroprotective properties in the setting of traumatic brain injury.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '56c1d846ef6e39474100002e',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 164,\n",
       "     'text': 'Neuroglobin has shown rich neuroprotective effects against cerebral hypoxia, and therefore has the potential to impact outcomes after traumatic brain injury (TBI). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20345238',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': 'Neuroglobin (Ngb) is proposed to be a neuron-specific, hypoxia-responsive, neuroprotective protein. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20544857',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 796,\n",
       "     'offsetInEndSection': 1081,\n",
       "     'text': 'CONCLUSION: The increased expression of neuroglobin in traumatic brain injury informed us that neuroglobin had anti-apoptosis action in post-injury neuron. It could protect the neuron from traumatic stress and secondary ischemia and hypoxia insults during ultra-early and acute stages.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20476562',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 71,\n",
       "     'text': 'Neuroglobin-overexpression reduces traumatic brain lesion size in mice.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22703519',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 153,\n",
       "     'text': 'BACKGROUND: Accumulating evidence has demonstrated that over-expression of Neuroglobin (Ngb) is neuroprotective against hypoxic/ischemic brain injuries. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22703519',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1071,\n",
       "     'offsetInEndSection': 1201,\n",
       "     'text': 'CONCLUSION: Ngb over-expression reduced traumatic lesion volume, which might partially be achieved by decreasing oxidative stress.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22703519',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 74,\n",
       "     'text': 'Neuroglobin upregulation offers neuroprotection in traumatic brain injury.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22664218',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 228,\n",
       "     'text': 'OBJECTIVES: The aim of this study was to investigate rat neuroglobin (rNGB) expression level after traumatic brain injury (TBI) and further study its neuroprotective effects in TBI when it was overexpressed in adenoviral vector.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22664218',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 955,\n",
       "     'offsetInEndSection': 1060,\n",
       "     'text': 'CONCLUSIONS: NGB was upregulated in TBI and overexpressed rNGB had a significant neuroprotection in TBI. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22664218',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1101,\n",
       "     'offsetInEndSection': 1189,\n",
       "     'text': 'This study suggested that rNGB overexpression may be a new strategy for treating of TBI.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22664218',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 163,\n",
       "     'text': 'Neuroglobin has shown rich neuroprotective effects against cerebral hypoxia, and therefore has the potential to impact outcomes after traumatic brain injury (TBI).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20345238',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'The aim of this study was to investigate rat neuroglobin (rNGB) expression level after traumatic brain injury (TBI) and further study its neuroprotective effects in TBI when it was overexpressed in adenoviral vector.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22664218',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': 'Accumulating evidence has demonstrated that over-expression of Neuroglobin (Ngb) is neuroprotective against hypoxic/ischemic brain injuries.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22703519',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 171,\n",
       "     'offsetInEndSection': 334,\n",
       "     'text': 'Various studies seem to indicate that neuroglobin is a neuroprotective agent when overexpressed, acting as a potent inhibitor of oxidative and nitrosative stress. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24491879',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Does triiodothyronine play a regulatory role in insulin secretion from pancreas?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/17408701',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21099301',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1357067',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/619228',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19021014',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20730704',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3310493',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1537314',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21914860',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2242013'],\n",
       "   'ideal_answer': ['YES'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/INS_DANRE',\n",
       "    'http://www.uniprot.org/uniprot/INS_CAIMO',\n",
       "    'http://www.biosemantics.org/jochem#4054676',\n",
       "    'http://www.uniprot.org/uniprot/INS_HORSE',\n",
       "    'http://www.uniprot.org/uniprot/INS_LOPAM',\n",
       "    'http://www.uniprot.org/uniprot/INS_CANFA',\n",
       "    'http://www.uniprot.org/uniprot/INS_PANBU',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0050796',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0046676',\n",
       "    'http://www.biosemantics.org/jochem#4275389',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D011988',\n",
       "    'http://www.uniprot.org/uniprot/INS_APLCA',\n",
       "    'http://www.biosemantics.org/jochem#4249447',\n",
       "    'http://www.uniprot.org/uniprot/INS_CHICH',\n",
       "    'http://www.uniprot.org/uniprot/INS_CHICK',\n",
       "    'http://www.uniprot.org/uniprot/INS_HYSCR',\n",
       "    'http://www.uniprot.org/uniprot/INS_ORENI',\n",
       "    'http://www.uniprot.org/uniprot/INS_LAMFL',\n",
       "    'http://www.uniprot.org/uniprot/INS_VERMO',\n",
       "    'http://www.uniprot.org/uniprot/INS_LOPPI',\n",
       "    'http://www.uniprot.org/uniprot/INS_CAVPO',\n",
       "    'http://www.uniprot.org/uniprot/INS_AOTTR',\n",
       "    'http://www.uniprot.org/uniprot/INS_OCTDE',\n",
       "    'http://www.uniprot.org/uniprot/INS_GADMC',\n",
       "    'http://www.uniprot.org/uniprot/INS_CAPHI',\n",
       "    'http://www.uniprot.org/uniprot/INS_RODSP',\n",
       "    'http://www.uniprot.org/uniprot/INS_SQUAC',\n",
       "    'http://www.uniprot.org/uniprot/INS_CAMDR',\n",
       "    'http://www.uniprot.org/uniprot/INS_CALMI',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D061385',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0030073',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=0032024',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D014284',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D010179',\n",
       "    'http://www.uniprot.org/uniprot/INS_SHEEP',\n",
       "    'http://www.uniprot.org/uniprot/INS_MYOSC',\n",
       "    'http://www.uniprot.org/uniprot/INS_PIAME',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D050417',\n",
       "    'http://www.uniprot.org/uniprot/INS_ORNAN',\n",
       "    'http://www.uniprot.org/uniprot/INS_ATRSP',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2012/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007328',\n",
       "    'http://www.uniprot.org/uniprot/INS_PHYMC'],\n",
       "   'type': 'yesno',\n",
       "   'id': '51682382298dcd4e51000066',\n",
       "   'snippets': [{'offsetInBeginSection': 1105,\n",
       "     'offsetInEndSection': 1320,\n",
       "     'text': 'Our findings establish that p43 is an important regulator of glucose homeostasis and pancreatic β-cell function and provide evidence for the first time of a physiological role for a mitochondrial endocrine receptor.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21914860',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 492,\n",
       "     'offsetInEndSection': 648,\n",
       "     'text': 'The p43(-/-) mice had a major defect in insulin secretion both in vivo and in isolated pancreatic islets and a loss of glucose-stimulated insulin secretion.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21914860',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 914,\n",
       "     'offsetInEndSection': 1083,\n",
       "     'text': 'We demonstrated that treatment of primary cultures of rat pancreatic islets with T3 results in augmented β-cell vitality with an increase of their functional properties.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21099301',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1266,\n",
       "     'offsetInEndSection': 1344,\n",
       "     'text': 'Nonetheless, the insulin secretion is sensibly augmented after T3 stimulation.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21099301',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 681,\n",
       "     'offsetInEndSection': 946,\n",
       "     'text': 'Plasma glucose concentration of the fetal hypothyroid group during intravenous glucose tolerance test was significantly higher (p=0.003) at 5-20 min as compared to the control group, whereas plasma insulin concentration was significantly lower (p=0.012) at 5-20 min',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20730704',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1182,\n",
       "     'offsetInEndSection': 1340,\n",
       "     'text': 'Although adult offspring born from hypothyroid mothers were euthyroid, their glucose tolerance and glucose stimulated insulin secretion of islets were altered',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20730704',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1,\n",
       "     'offsetInEndSection': 87,\n",
       "     'text': 'hyroid hormones modulate the immune system and metabolism, influence insulin secretion',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19021014',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 999,\n",
       "     'offsetInEndSection': 1190,\n",
       "     'text': 'Only T(3) concentrations higher than 250 microM were able to decrease cell viability and proliferation rate, to increase the rate of apoptosis and to reduce glucose-induced insulin secretion.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17408701',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 359,\n",
       "     'offsetInEndSection': 735,\n",
       "     'text': 'Islets preincubated with glucose (3.3 mmol/l) and glucagon (1.4 mumol/l) plus theophylline (10 mmol/l), ACTH (0.11 nmol/l), bovine GH (0.46 mumol/l), prolactin (0.2 mumol/l) or tri-iodothyronine (1.0 nmol/l) have significantly lower Ca(2+)-ATPase activity than those preincubated with only 3.3 mmol glucose/l. All these hormones increased the release of insulin significantly.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1357067',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 1326,\n",
       "     'offsetInEndSection': 1476,\n",
       "     'text': 'T3 (0.2 nM) did not affect insulin secretion in the absence or presence of glucose or in the presence of secretagogues (potassium and glyceraldehyde).',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1537314',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 202,\n",
       "     'offsetInEndSection': 373,\n",
       "     'text': \"In the perfused rat pancreas, the addition of thyroxine (10 micrograms/dL) or 3,5,3'-triiodothyronine (150 ng/dL) to the perfusing medium did not affect insulin secretion.\",\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2242013',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 374,\n",
       "     'offsetInEndSection': 541,\n",
       "     'text': 'The administration of thyroxine (40 micrograms/kg, s.c.) in vivo increased the plasma insulin level from 11 +/- 2 microUnits/mL (mean +/- SD) to 30 +/- 7 microUnits/mL',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2242013',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 626,\n",
       "     'offsetInEndSection': 1003,\n",
       "     'text': 'Addition of T3 to the incubation medium, significantly modified the insulin release, but its effect varied according to the glucose concentration in the medium, i.e. it enhanced the insulin release at a glucose concentration between 2 to 8 mmol/l; it has no effect at 12 mmol/glucose, and significantly inhibited the secretion of insulin in the presence of 16.6 mmol/l glucose.',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3310493',\n",
       "     'endSection': 'sections.0'},\n",
       "    {'offsetInBeginSection': 588,\n",
       "     'offsetInEndSection': 630,\n",
       "     'text': 'Both T3 and T4 inhibited insulin secretion',\n",
       "     'beginSection': 'sections.0',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/619228',\n",
       "     'endSection': 'sections.0'}]},\n",
       "  {'body': 'What is the use of emulsion PCR in Next Generation Sequencing?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23398507',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22668416',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23900195',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23148498',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21513157',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21543404',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23546785',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23744319',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23665194',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22355625'],\n",
       "   'ideal_answer': ['Prior to Next Generation Sequencing reactions, DNA libraries are constructed, amplified with emulsion PCR, and enriched with the use of enrichment beads. The library samples are then loaded to a sequencing chip and analyzed on an NGS platform.'],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D016133',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D004655',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D017422',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D017421'],\n",
       "   'type': 'summary',\n",
       "   'id': '5319a83bb166e2b80600002c',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 130,\n",
       "     'text': 'Emulsion PCR-coupled target enrichment: an effective fishing method for high-throughput sequencing of poorly preserved ancient DNA',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23900195',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 325,\n",
       "     'offsetInEndSection': 680,\n",
       "     'text': 'We present a unified strategy in which emulsion PCR is coupled with target enrichment followed by next-generation sequencing. The method made it possible to obtain efficiently non-duplicated reads mapped to target sequences of interest, and this can achieve deep and reliable sequencing of ancient DNA from typical materials, even though poorly preserved.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23900195',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 492,\n",
       "     'offsetInEndSection': 900,\n",
       "     'text': 'Three multiplex PCR reactions were carried out to amplify the coding exons of 328 genes including FBN1, TGFBR1 and TGFBR2. DNA fragments from different samples were ligated with barcoded sequencing adaptors. Template preparation and emulsion PCR, and Ion Sphere Particles enrichment were carried out using an Ion One Touch system. The ion sphere particles were sequenced on a 318 chip using the PGM platform.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23744319',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 802,\n",
       "     'offsetInEndSection': 1156,\n",
       "     'text': 'In our NGS approach, the 16,569-bp mtDNA is enriched by long-range PCR and the 108 nuclear genes (which represent 1301 amplicons and 680 kb) are enriched by RainDance emulsion PCR. Sequencing is performed on Illumina HiSeq 2000 or MiSeq platforms, and bioinformatics analysis is performed using commercial and in-house developed bioinformatics pipelines.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23665194',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 461,\n",
       "     'offsetInEndSection': 684,\n",
       "     'text': 'However, pyrosequencing of emulsion PCR reactions, amplifying from only one molecule at a time, can generate megabases of clonally amplified loci at high coverage, thereby greatly simplifying allelic sequence determination.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23546785',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 861,\n",
       "     'offsetInEndSection': 1319,\n",
       "     'text': \"During genomic PCR, in this 4-primer system, the outer set of primers containing the MID and the 454 adaptor sequences are incorporated into an amplicon generated by the inner HLA target-specific primers each containing a common sequence tag at the 5' end of the forward and reverse primers. Pools of the resulting amplicons are used for emulsion PCR and clonal sequencing on the 454 Life Sciences GS FLX System, followed by genotyping with Conexio software.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23398507',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 428,\n",
       "     'offsetInEndSection': 724,\n",
       "     'text': 'The purpose of our study was to compare NGS enrichment methods for a clinical assay targeting the nine genes known to be associated with aortopathy. RainDance emulsion PCR and SureSelect RNA-bait hybridization capture enrichment methods were directly compared by enriching DNA from eight samples.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23148498',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 872,\n",
       "     'offsetInEndSection': 1073,\n",
       "     'text': 'We show the efficacy by comparing the method to a commercial kit and further demonstrate that emulsion PCR can be used for bias free amplification and virtual immortalization of DNA template libraries.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22668416',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 281,\n",
       "     'offsetInEndSection': 403,\n",
       "     'text': 'Here, we describe a novel approach for normalization of multiplex next generation sequencing libraries after emulsion PCR.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22355625',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 404,\n",
       "     'offsetInEndSection': 607,\n",
       "     'text': 'Briefly, amplified libraries carrying unique barcodes are prepared by fluorescent tagging of complementary sequences and then resolved by high-speed flow cytometric sorting of labeled emulsion PCR beads.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22355625',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 608,\n",
       "     'offsetInEndSection': 959,\n",
       "     'text': 'The protocol is simple and provides an even sequence distribution of multiplex libraries when sequencing the flow-sorted beads. Moreover, since many empty and mixed emulsion PCR beads are removed, the approach gives rise to a substantial increase in sequence quality and mean read length, as compared to that obtained by standard enrichment protocols.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22355625',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 156,\n",
       "     'offsetInEndSection': 307,\n",
       "     'text': 'Based on the mechanism of emulsion PCR, a unique DNA template would only generate a unique sequence read after being amplified and sequenced on GS FLX.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21543404',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 763,\n",
       "     'offsetInEndSection': 1015,\n",
       "     'text': 'Prior to SOLiD sequencing reaction, the libraries were amplified with emulsion PCR and enriched with the P2 enrichment beads. The library samples were loaded to sequencing Chip for Work Flow Analysis (WFA) or sequencing running with default parameters.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21513157',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which histone modifications distinguish between promoters and enhancers?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24038352',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21937433',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20008934',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18441229',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19094206',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22421546',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23547170',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12461564',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19295514',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21106759'],\n",
       "   'ideal_answer': ['H3K27ac is a marker of active enhancers. An enhancer chromatin state signature associated with active developmental enhancers may be defined by high levels of H3K27ac marking, nucleosome displacement, hypersensitivity to sonication, and strong depletion of H3K27me3.'],\n",
       "   'exact_answer': [['H3K27ac enrichment'], ['H3K27me3 depletion']],\n",
       "   'type': 'list',\n",
       "   'id': '56ddbbaf51531f7e33000009',\n",
       "   'snippets': [{'offsetInBeginSection': 464,\n",
       "     'offsetInEndSection': 744,\n",
       "     'text': 'Here, we used chromatin immunoprecipitation followed by high-throughput sequencing (ChIP-seq) to measure genome-wide changes in histone H3 acetylation at lysine 27 (H3K27ac), a marker of active enhancers, in unstimulated HUVECs and HUVECs stimulated with VEGFA for 1, 4, and 12 h.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23547170',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 365,\n",
       "     'offsetInEndSection': 544,\n",
       "     'text': 'Here we use whole-transcriptome analysis coupled with genome-wide profiling of H3K27ac and H3K27me3 to map chromatin states and enhancers in mouse embryonic forelimb and hindlimb.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22421546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 752,\n",
       "     'offsetInEndSection': 999,\n",
       "     'text': 'Using H3K27ac profiles, we identified 28,377 putative enhancers, many of which are likely to be limb specific based on strong enrichment near genes highly expressed in the limb and comparisons with tissue-specific EP300 sites and known enhancers. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22421546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1011,\n",
       "     'offsetInEndSection': 1219,\n",
       "     'text': 'a chromatin state signature associated with active developmental enhancers, defined by high levels of H3K27ac marking, nucleosome displacement, hypersensitivity to sonication, and strong depletion of H3K27me3',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22421546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1221,\n",
       "     'offsetInEndSection': 1434,\n",
       "     'text': 'We also find that some developmental enhancers exhibit components of this signature, including hypersensitivity, H3K27ac enrichment, and H3K27me3 depletion, at lower levels in tissues in which they are not active.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22421546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1355,\n",
       "     'offsetInEndSection': 1472,\n",
       "     'text': 'At some locations, H3K27 monomethylation was also found to be associated with chromatin signatures of gene enhancers.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21937433',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 287,\n",
       "     'offsetInEndSection': 413,\n",
       "     'text': 'Individual chromatin marks, such as H3K27ac and H3K27me3, have been identified to distinguish active from inactive enhancers. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24038352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 718,\n",
       "     'offsetInEndSection': 843,\n",
       "     'text': 'We find that histone H3K27ac distinguishes active enhancers from inactive/poised enhancer elements containing H3K4me1 alone. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21106759',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 80,\n",
       "     'offsetInEndSection': 413,\n",
       "     'text': 'Although certain chromatin features, such as the histone acetyltransferase P300 and the histone modification H3K4me1, indicate the presence of enhancers, only a fraction of enhancers are functionally active. Individual chromatin marks, such as H3K27ac and H3K27me3, have been identified to distinguish active from inactive enhancers.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24038352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 719,\n",
       "     'offsetInEndSection': 843,\n",
       "     'text': 'We find that histone H3K27ac distinguishes active enhancers from inactive/poised enhancer elements containing H3K4me1 alone.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21106759',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which mutations in the cardiac isoform of the ryanodine receptor (RyR2) have been found to be related to CPVT?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23498838',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21659649',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12919952',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15890976',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23152493',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18483626',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22828895',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14571276',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20676041',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23746327'],\n",
       "   'ideal_answer': ['Recently, a novel CPVT RyR2 mutation, G230C, was found to increase the cytosolic, but not the luminal, Ca2+ sensitivity of single RyR2 channels in lipid bilayers. The novel RYR2-S4153R mutation has been implicated as a cause of CPVT and atrial fibrillation. A novel RyR2-V2475F mutation is associated with CPVT in humans. 3 CPVT mouse models are: RyR2-R2474S+/-, RyR2-N2386I+/-, and RyR2-L433P+/-. The E189D RyR2 mutation is causative for CPVT. A knock-in mouse model carrier of the R4496C mutation is the mouse equivalent to the R4497C mutations identified in CPVT families. Scanning of 12 Finnish CPVT probands identified three novel RYR2 mutations (V2306I, P4902L, R4959Q), which were absent in unaffected and control individuals. Three CPVT-linked human RyR2 (hRyR2) mutations are: S2246L, N4104K, and R4497C.'],\n",
       "   'exact_answer': [['RyR2-G230C'],\n",
       "    ['RyR2-S4153R'],\n",
       "    ['RyR2-V2475F'],\n",
       "    ['RyR2-R2474S'],\n",
       "    ['RyR2-N2386I'],\n",
       "    ['RyR2-L433P'],\n",
       "    ['RyR2-E189D'],\n",
       "    ['RyR2-R4497C (RyR2-R4496C in the mouse)'],\n",
       "    ['RyR2-V2306I,'],\n",
       "    ['RyR2-P4902L'],\n",
       "    ['RyR2-R4959Q'],\n",
       "    ['RyR2-S2246L'],\n",
       "    ['RyR2-N4104K']],\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/RYR2_HUMAN'],\n",
       "   'type': 'list',\n",
       "   'id': '550712739d0faaa527000008',\n",
       "   'snippets': [{'offsetInBeginSection': 575,\n",
       "     'offsetInEndSection': 737,\n",
       "     'text': 'Recently, a novel CPVT RyR2 mutation, G230C, was found to increase the cytosolic, but not the luminal, Ca2+ sensitivity of single RyR2 channels in lipid bilayers.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23746327',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1513,\n",
       "     'offsetInEndSection': 1743,\n",
       "     'text': 'These data suggest that the G230C mutation enhances the propensity for SOICR by sensitizing the channel to luminal and cytosolic Ca2+ activation, and that G230C has an intrinsic structural impact on the N-terminal domains of RyR2.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23746327',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 118,\n",
       "     'offsetInEndSection': 765,\n",
       "     'text': 'The novel RYR2-S4153R mutation has been implicated as a cause of CPVT and atrial fibrillation. The mutation has been functionally characterized via store-overload-induced Ca(2+) release (SOICR) and tritium-labelled ryanodine ([(3)H]ryanodine) binding assays. The S4153R mutation enhanced propensity for spontaneous Ca(2+) release and reduced SOICR threshold but did not alter Ca(2+) activation of [(3)H]ryanodine binding, a common feature of other CPVT gain-of-function RYR2 mutations. We conclude that the S4153R mutation is a gain-of-function RYR2 mutation associated with a clinical phenotype characterized by both CPVT and atrial fibrillation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23498838',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 345,\n",
       "     'offsetInEndSection': 535,\n",
       "     'text': 'To determine the molecular and cellular mechanisms by which a novel RyR2-V2475F mutation associated with CPVT in humans triggers Ca(2+)-dependent arrhythmias in whole hearts and intact mice.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23152493',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1752,\n",
       "     'offsetInEndSection': 1888,\n",
       "     'text': 'The mutation RyR2-V2475F is phenotypically strong among other CPVT mutations and produces heterogeneous mechanisms of RyR2 dysfunction. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23152493',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 546,\n",
       "     'offsetInEndSection': 1117,\n",
       "     'text': 'We generated 3 knock-in mice with mutations introduced into RyR2 that result in leaky channels and cause exercise induced polymorphic ventricular tachycardia in humans [catecholaminergic polymorphic ventricular tachycardia (CPVT)]. We examined AF susceptibility in these three CPVT mouse models harboring RyR2 mutations to explore the role of diastolic SR Ca2+ leak in AF. AF was stimulated with an intra-esophageal burst pacing protocol in the 3 CPVT mouse models (RyR2-R2474S+/-, 70%; RyR2-N2386I+/-, 60%; RyR2-L433P+/-, 35.71%) but not in wild-type (WT) mice (P<0.05).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22828895',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1300,\n",
       "     'offsetInEndSection': 1917,\n",
       "     'text': 'The RyR2-G230C mutant exhibits similar biophysical defects compared with previously characterized CPVT mutations: decreased binding of the stabilizing subunit calstabin2 and a leftward shift in the Ca(2+) dependence for activation under conditions that simulate exercise, consistent with a \"leaky\" channel. Both RyR2-G230C and RyR2-P2328S channels exhibit normal luminal Ca(2+) activation. Thus, diastolic sarcoplasmic reticulum Ca(2+) leak caused by reduced calstabin2 binding and a leftward shift in the Ca(2+) dependence for activation by diastolic levels of cytosolic Ca(2+) is a common mechanism underlying CPVT.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21659649',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1058,\n",
       "     'offsetInEndSection': 1527,\n",
       "     'text': 'A novel CPVT mutation, E189D, was identified. The E189D mutation does not alter the affinity of the channel for FKBP12.6, but it increases the propensity for store-overload-induced Ca(2+) release (SOICR). Furthermore, the E189D mutation enhances the basal channel activity of RyR2 and its sensitivity to activation by caffeine. The E189D RyR2 mutation is causative for CPVT and functionally increases the propensity for SOICR without altering the affinity for FKBP12.6.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20676041',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 468,\n",
       "     'offsetInEndSection': 731,\n",
       "     'text': 'We found that mice heterozygous for the R2474S mutation in Ryr2 (Ryr2-R2474S mice) exhibited spontaneous generalized tonic-clonic seizures (which occurred in the absence of cardiac arrhythmias), exercise-induced ventricular arrhythmias, and sudden cardiac death. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18483626',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 952,\n",
       "     'offsetInEndSection': 1085,\n",
       "     'text': ' Thus, CPVT-associated mutant leaky Ryr2-R2474S channels in the brain can cause seizures in mice, independent of cardiac arrhythmias.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18483626',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 313,\n",
       "     'offsetInEndSection': 503,\n",
       "     'text': 'We screened 12 Finnish CPVT probands for mutations in these genes and identified three novel RYR2 mutations (V2306I, P4902L, R4959Q), which were absent in unaffected and control individuals.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14571276',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 149,\n",
       "     'offsetInEndSection': 567,\n",
       "     'text': 'RyR2 mutations suggested to cause defective Ca2+ channel function have recently been identified in catecholaminergic polymorphic ventricular tachycardia (CPVT) and arrhythmogenic right ventricular dysplasia (ARVD) affected individuals. We report expression of three CPVT-linked human RyR2 (hRyR2) mutations (S2246L, N4104K, and R4497C) in HL-1 cardiomyocytes displaying correct targeting to the endoplasmic reticulum. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12919952',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 683,\n",
       "     'offsetInEndSection': 942,\n",
       "     'text': 'We developed a conditional knock-in mouse model carrier of the R4496C mutation, the mouse equivalent to the R4497C mutations identified in CPVT families, to evaluate if the animals would develop a CPVT phenotype and if beta blockers would prevent arrhythmias.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15890976',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1466,\n",
       "     'offsetInEndSection': 1645,\n",
       "     'text': 'These data provide the first experimental demonstration that the R4496C RyR2 mutation predisposes the murine heart to VT and VF in response caffeine and/or adrenergic stimulation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15890976',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 682,\n",
       "     'offsetInEndSection': 940,\n",
       "     'text': 'We developed a conditional knock-in mouse model carrier of the R4496C mutation, the mouse equivalent to the R4497C mutations identified in CPVT families, to evaluate if the animals would develop a CPVT phenotype and if beta blockers would prevent arrhythmias',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15890976',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List components of the CRSP/Med complex.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/14759369',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12050112',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15175162',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9989412',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12509459',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24820420',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15195149',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10377381',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19571180',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11834832'],\n",
       "   'ideal_answer': ['Mediator of RNA polymerase II transcription subunit 7\\nMediator of RNA polymerase II transcription subunit 14\\nMediator of RNA polymerase II transcription subunit 17\\nMediator of RNA polymerase II transcription subunit 23\\nMediator of RNA polymerase II transcription subunit 24\\nMediator of RNA polymerase II transcription subunit 26\\nMediator of RNA polymerase II transcription subunit 27'],\n",
       "   'exact_answer': [['Mediator of RNA polymerase II transcription subunit 7'],\n",
       "    ['Mediator of RNA polymerase II transcription subunit 14'],\n",
       "    ['Mediator of RNA polymerase II transcription subunit 17'],\n",
       "    ['Mediator of RNA polymerase II transcription subunit 23'],\n",
       "    ['Mediator of RNA polymerase II transcription subunit 26'],\n",
       "    ['Mediator of RNA polymerase II transcription subunit 27'],\n",
       "    ['Mediator of RNA polymerase II transcription subunit 24']],\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/MED17_XENTR',\n",
       "    'http://www.uniprot.org/uniprot/MED27_BOVIN',\n",
       "    'http://www.uniprot.org/uniprot/MED7_PIG',\n",
       "    'http://www.uniprot.org/uniprot/MED23_DANRE',\n",
       "    'http://www.uniprot.org/uniprot/MED7_BOVIN',\n",
       "    'http://www.uniprot.org/uniprot/MED7_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MED23_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MED27_XENTR',\n",
       "    'http://www.uniprot.org/uniprot/MED17_DANRE',\n",
       "    'http://www.uniprot.org/uniprot/MED26_DANRE',\n",
       "    'http://www.uniprot.org/uniprot/MED26_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MED17_BOVIN',\n",
       "    'http://www.uniprot.org/uniprot/MED7_XENTR',\n",
       "    'http://www.uniprot.org/uniprot/MED27_DANRE',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D016329',\n",
       "    'http://www.uniprot.org/uniprot/MED27_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MED23_RAT',\n",
       "    'http://www.uniprot.org/uniprot/MED27_PIG',\n",
       "    'http://www.uniprot.org/uniprot/MED14_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MED7_DANRE',\n",
       "    'http://www.uniprot.org/uniprot/MED26_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MED27_PONAB',\n",
       "    'http://www.uniprot.org/uniprot/MED14_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MD27A_XENLA',\n",
       "    'http://www.uniprot.org/uniprot/MED7_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MED17_CHICK',\n",
       "    'http://www.uniprot.org/uniprot/MED26_XENLA',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2015/MB_cgi?field=uid&exact=Find+Exact+Term&term=D003067',\n",
       "    'http://www.uniprot.org/uniprot/MED17_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MED7A_XENLA',\n",
       "    'http://www.uniprot.org/uniprot/MED17_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MED27_MOUSE',\n",
       "    'http://www.uniprot.org/uniprot/MED23_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MED24_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/MED7B_XENLA',\n",
       "    'http://www.uniprot.org/uniprot/MD27B_XENLA',\n",
       "    'http://www.uniprot.org/uniprot/MED26_BOVIN'],\n",
       "   'type': 'list',\n",
       "   'id': '54e0dfa91388e8454a000018',\n",
       "   'snippets': [{'offsetInBeginSection': 622,\n",
       "     'offsetInEndSection': 1322,\n",
       "     'text': ' Here we describe a new human factor, CRSP, that is required together with the TAF(II)s for transcriptional activation by Sp1. Purification of CRSP identifies a complex of approximate relative molecular mass 700,000 (M(r) approximately 700K) that contains nine subunits with M(r) values ranging from 33K to 200K. Cloning of genes encoding CRSP subunits reveals that CRSP33 is a homologue of the yeast mediator subunit Med7, whereas CRSP150 contains a domain conserved in yeast mediator subunit Rgr1. CRSP p200 is identical to the nuclear hormone-receptor co-activator subunit TRIP2/PBP. CRSPs 34, 77 and 130 are new proteins, but the amino terminus of CRSP70 is homologous to elongation factor TFIIS.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9989412',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 805,\n",
       "     'offsetInEndSection': 905,\n",
       "     'text': 'CRSP contains unique subunits as well as polypeptides that are shared with other cofactor complexes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10377381',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 432,\n",
       "     'text': 'The multi-subunit, human CRSP coactivator-also known as Mediator (Med)-regulates transcription by mediating signals between enhancer-bound factors (activators) and the core transcriptional machinery. Interestingly, different activators are known to bind distinct subunits within the CRSP/Med complex. We have isolated a stable, endogenous CRSP/Med complex (CRSP/Med2) that specifically lacks both the Med220 and the Med70 subunits. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15175162',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the basic secondary structure of the variable domains of a typical antibody?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18513436',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9502319',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12566463',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3115977',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8683598',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8589603',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15186484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3929246',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11297418',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/402951'],\n",
       "   'ideal_answer': [\"The variable domains of heavy and light chains of antibodies consist of two β-sheets. The first one is composed of four strands, A, B, E and D, and the second one is composed of six strands, named A', G, F, C, C' and C''. The antigen binding site is formed by the inter-strand links BC, C′C″ and FG from each domain.\"],\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4000002',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D017433',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007136',\n",
       "    'http://amigo.geneontology.org/cgi-bin/amigo/term_details?term=GO:0042571',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D000906'],\n",
       "   'type': 'summary',\n",
       "   'id': '532ad531d6d3ac6a34000013',\n",
       "   'snippets': [{'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 281,\n",
       "     'text': 'Antibodies are prototypes of multimeric proteins and consist of structurally similar domains. The two variable domains of an antibody (VH and VL) interact through a large hydrophobic interface and can be expressed as covalently linked single-chain Fv (scFv) fragments. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9502319',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 564,\n",
       "     'offsetInEndSection': 1280,\n",
       "     'text': 'Secondary structure formation, measured by H/D-exchange protection, of a mutant scFv fragment of an antibody after short incubation in 6 M guanidinium chloride was shown to be multiphasic. NMR analysis shows that an intermediate with significant proton protection is observed within the dead time of the manual mixing experiments. Subsequently, the folding reaction proceeds via a biphasic reaction and mass spectrometry analyses of the exchange experiments confirm the existence of two parallel pathways. In the presence of cyclophilin, however, the faster of the two phases vanishes (when followed by intrinsic tryptophan fluorescence), while the slower phase is not significantly enhanced by equimolar cyclophilin',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9502319',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1297,\n",
       "     'offsetInEndSection': 1778,\n",
       "     'text': \"e formation of an early intermediate, which shows amide-proton exchange protection, is independent of proline isomerization. Subsequently, a proline cis-trans isomerization reaction in the rapidly formed intermediate, producing 'non-native' isomers, competes with the fast formation of native species. Interface formation in a folding intermediate of the scFv fragment is proposed to prevent the back-isomerization of these prolines from being efficiently catalyzed by cyclophilin.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9502319',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 8,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': ' heavy chain variable domains (VH) lacking their light chain (VL) partner are prime candidates for the design of minimum-size immunoreagents. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8589603',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Where does CTCF colocalize with cohesin?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23150255',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22780989',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20219941',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21628529',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19369356',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18219272',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21606361',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20159591',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19109133',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21550623'],\n",
       "   'ideal_answer': ['Cohesin subunits associate with viral and cellular CTCF sites involved in complex gene regulation and chromatin organization. Cohesin cobinds across the genome with transcription factors independently of CTCF, plays a functional role in estrogen-regulated transcription, and may help to mediate tissue-specific transcriptional responses via long-range chromosomal interactions.\\nNumerous CTCF/cohesin sites potentially form the bases of the multiloop rosette structures at the Igh locus that compact during Ig heavy chain rearrangement. Cohesins colocalize with CTCF at two additional imprinted loci, the Dlk1-Dio3 and the Kcnq1/Kcnq1ot1 loci.'],\n",
       "   'exact_answer': ['In imprinted and repressed, highly compacted loci that are prone to recombination events.'],\n",
       "   'concepts': ['http://www.uniprot.org/uniprot/CTCF_HUMAN',\n",
       "    'http://www.uniprot.org/uniprot/CTCF_RAT',\n",
       "    'http://amigo.geneontology.org/amigo/term/GO:0008278'],\n",
       "   'type': 'factoid',\n",
       "   'id': '56c5c7915795f9a73e000004',\n",
       "   'snippets': [{'offsetInBeginSection': 773,\n",
       "     'offsetInEndSection': 958,\n",
       "     'text': 'In human cells, Pol III genes and transcription factors have also been shown to colocalize with cohesin and the transcription regulator and genome organizer CCCTC-binding factor (CTCF).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23150255',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 348,\n",
       "     'offsetInEndSection': 560,\n",
       "     'text': 'To investigate cohesin-non-CTCF (CNC) binding events in vivo we mapped cohesin and CTCF, as well as a collection of tissue-specific and ubiquitous transcriptional regulators using ChIP-seq in primary mouse liver.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22780989',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 666,\n",
       "     'offsetInEndSection': 887,\n",
       "     'text': ' In contrast to regions of the genome where cohesin and CTCF colocalize, CNC sites coincide with the binding of master regulators and enhancer-markers and are significantly associated with liver-specific expressed genes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22780989',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1232,\n",
       "     'offsetInEndSection': 1402,\n",
       "     'text': 'Finally, we observe that the presence of mirrored CTCF binding events at promoters and their nearby cohesin-bound enhancers is associated with elevated expression levels.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22780989',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 98,\n",
       "     'offsetInEndSection': 305,\n",
       "     'text': 'Recently, cohesins have been implicated in transcriptional regulation and insulation through genome-wide colocalization with the insulator protein CTCF, including involvement at the imprinted H19/Igf2 locus.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21628529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 415,\n",
       "     'offsetInEndSection': 542,\n",
       "     'text': 'Here we report that cohesins colocalize with CTCF at two additional imprinted loci, the Dlk1-Dio3 and the Kcnq1/Kcnq1ot1 loci. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21628529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 698,\n",
       "     'offsetInEndSection': 871,\n",
       "     'text': ' To determine the functional importance of the binding of CTCF and cohesins at the three imprinted loci, CTCF and cohesins were depleted in mouse embryonic fibroblast cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21628529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1144,\n",
       "     'offsetInEndSection': 1283,\n",
       "     'text': 'Results of these experiments demonstrate an unappreciated role for CTCF and cohesins in the repression of imprinted genes in somatic cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21628529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 235,\n",
       "     'offsetInEndSection': 383,\n",
       "     'text': 'Here, we show in different cell types that cohesin functionally behaves as a tissue-specific transcriptional regulator, independent of CTCF binding.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20219941',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 384,\n",
       "     'offsetInEndSection': 596,\n",
       "     'text': 'By performing matched genome-wide binding assays (ChIP-seq) in human breast cancer cells (MCF-7), we discovered thousands of genomic sites that share cohesin and estrogen receptor alpha (ER) yet lack CTCF binding',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20219941',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 598,\n",
       "     'offsetInEndSection': 834,\n",
       "     'text': 'By use of human hepatocellular carcinoma cells (HepG2), we found that liver-specific transcription factors colocalize with cohesin independently of CTCF at liver-specific targets that are distinct from those found in breast cancer cells',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20219941',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1246,\n",
       "     'offsetInEndSection': 1526,\n",
       "     'text': 'Together, our data show that cohesin cobinds across the genome with transcription factors independently of CTCF, plays a functional role in estrogen-regulated transcription, and may help to mediate tissue-specific transcriptional responses via long-range chromosomal interactions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20219941',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 685,\n",
       "     'offsetInEndSection': 1004,\n",
       "     'text': ' Because cohesin can colocalize with CTCF, we performed chromatin immunoprecipitation for the cohesin subunit Rad21 and found lineage and stage-specific Rad21 recruitment to CTCF in all Ig loci. The differential binding of cohesin to CTCF sites may promote multiple loop formation and thus effective V(D)J recombination',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19109133',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 626,\n",
       "     'offsetInEndSection': 789,\n",
       "     'text': 'These numerous CTCF/cohesin sites potentially form the bases of the multiloop rosette structures at the Igh locus that compact during Ig heavy chain rearrangement.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21606361',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1406,\n",
       "     'offsetInEndSection': 1532,\n",
       "     'text': 'We conclude that cohesins interact with CTCF in mid-S phase and repress CTCF-regulated genes in a cell cycle-dependent manner.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19369356',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1533,\n",
       "     'offsetInEndSection': 1788,\n",
       "     'text': 'We propose that the CTCF-cohesin complex plays a critical role in regulating the cell cycle control of viral gene expression during latency and that failure to maintain cell cycle control of latent transcripts inhibits host cell proliferation and survival',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19369356',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1060,\n",
       "     'offsetInEndSection': 1202,\n",
       "     'text': 'We conclude that cohesin subunits associate with viral and cellular CTCF sites involved in complex gene regulation and chromatin organization.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18219272',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 875,\n",
       "     'offsetInEndSection': 1068,\n",
       "     'text': 'Because cohesin can colocalize with CTCF, we performed chromatin immunoprecipitation for the cohesin subunit Rad21 and found lineage and stage-specific Rad21 recruitment to CTCF in all Ig loci.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19109133',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 954,\n",
       "     'offsetInEndSection': 1139,\n",
       "     'text': 'In human cells, Pol III genes and transcription factors have also been shown to colocalize with cohesin and the transcription regulator and genome organizer CCCTC-binding factor (CTCF).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23150255',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 450,\n",
       "     'offsetInEndSection': 705,\n",
       "     'text': 'Here, we show that ATRX, MeCP2, and cohesin, chromatin regulators implicated in ATR-X, RTT, and CdLS syndromes, respectively, interact in the brain and colocalize at the H19 imprinting control region (ICR) with preferential binding on the maternal allele.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20159591',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1072,\n",
       "     'offsetInEndSection': 1173,\n",
       "     'text': 'Subcellular distribution of CTCF and colocalization with cohesins also varied across the cell cycle. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19369356',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 449,\n",
       "     'offsetInEndSection': 626,\n",
       "     'text': 'ChIP sequencing was performed on pro-B cells, revealing colocalization of CTCF and Rad21 binding at ~60 sites throughout the V(H) region and 2 other sites within the Igh locus. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21606361',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 98,\n",
       "     'offsetInEndSection': 306,\n",
       "     'text': 'Recently, cohesins have been implicated in transcriptional regulation and insulation through genome-wide colocalization with the insulator protein CTCF, including involvement at the imprinted H19/Igf2 locus. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21628529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 369,\n",
       "     'offsetInEndSection': 426,\n",
       "     'text': 'CTCF colocalizes with cohesin but not RNA Polymerase II. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21550623',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 687,\n",
       "     'offsetInEndSection': 881,\n",
       "     'text': 'Because cohesin can colocalize with CTCF, we performed chromatin immunoprecipitation for the cohesin subunit Rad21 and found lineage and stage-specific Rad21 recruitment to CTCF in all Ig loci. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19109133',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 691,\n",
       "     'offsetInEndSection': 1010,\n",
       "     'text': 'Because cohesin can colocalize with CTCF, we performed chromatin immunoprecipitation for the cohesin subunit Rad21 and found lineage and stage-specific Rad21 recruitment to CTCF in all Ig loci. The differential binding of cohesin to CTCF sites may promote multiple loop formation and thus effective V(D)J recombination.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19109133',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 416,\n",
       "     'offsetInEndSection': 499,\n",
       "     'text': 'Here we report that cohesins colocalize with CTCF at two additional imprinted loci,',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21628529',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What are the main benefits of pharmacophore models?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24245803',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24212027',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24119198',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24287559',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24112046',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22192589',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24294969',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23662282',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20838973',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24140950'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1308602',\n",
       "     'o': 'CRISP Thesaurus'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/prefMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0599740',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A1308602'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1308603',\n",
       "     'o': 'pharmacophore'},\n",
       "    {'p': 'http://linkedlifedata.com/resource/umls/altMetaMap',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0599740',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A1308603'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#note',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1308603',\n",
       "     'o': 'CRISP Thesaurus'},\n",
       "    {'p': 'http://www.w3.org/2004/02/skos/core#notation',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1308603',\n",
       "     'o': '2573-1500'}],\n",
       "   'ideal_answer': ['As researchers continue to search for new targets of therapeutic interest, transmembrane and G-protein coupled receptors are of ever-increasing importance. However, crystal structures for these targets may be impossible to resolve, posing great challenges in rational drug design. Structure-based virtual screening is not an option when the active site geometry is unknown, but assaying an entire library for hits is an inefficient and expensive proposition.\\nPharmacophore modeling solves this problem by determining the spatial arrangement of chemical features that confer drug activity toward a target receptor. Having established the chemical space occupied by active ligands, pharmacophore modeling software allows researchers to create 3-D structure-activity relationships, screen databases, and generate hits without the benefit of a receptor structure.'],\n",
       "   'exact_answer': [['They represent chemical functions valid not only for the existing bounds but also for unknown molecules.'],\n",
       "    ['Due their simplicity they are proper for large scale virtual screening'],\n",
       "    ['They are comprehensive and editable, so by changing chemical feature contains information can be easily trace back.']],\n",
       "   'concepts': ['http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D003198',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D008956',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D015195',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D008961',\n",
       "    'http://www.nlm.nih.gov/cgi/mesh/2014/MB_cgi?field=uid&exact=Find+Exact+Term&term=D008954'],\n",
       "   'type': 'list',\n",
       "   'id': '532c0ac4d6d3ac6a3400001a',\n",
       "   'snippets': [{'offsetInBeginSection': 149,\n",
       "     'offsetInEndSection': 561,\n",
       "     'text': 'To discover novel RNAP inhibitors, a pharmacophore based on the alignment of described inhibitors was used for virtual screening. In an optimization process of hit compounds, novel derivatives with improved in vitro potency were discovered. Investigations concerning the molecular mechanism of RNAP inhibition reveal that they prevent the protein-protein interaction (PPI) between σ(70) and the RNAP core enzyme.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24112046',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 777,\n",
       "     'offsetInEndSection': 1190,\n",
       "     'text': 'the primary structural features required for estrogen receptor binding, the phenols, are not required for inhibiting parasitic growth. Significantly, the most active antileishmanial benzothiophenes lack the pharmacophore for estrogen receptor activity and therefore address potential concerns about the undesirable effects of using selective estrogen receptor modulators in women and children with leishmaniasis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24119198',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2,\n",
       "     'offsetInEndSection': 255,\n",
       "     'text': 'series of novel diastereoisomeric σ ligands 3 was designed, synthesized and pharmacologically evaluated. The highly rigid [4.3.3]propellane scaffold was used to fix the three dimensional orientation of the pharmacophoric moieties required for σ affinity',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24140950',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 143,\n",
       "     'offsetInEndSection': 988,\n",
       "     'text': 'This study was performed to identify natural iNOS inhibitors from traditional Chinese herbs through a combination of pharmacophore modeling, molecular docking and virtual screening. First, the pharmacophore models were generated though six known iNOS inhibitors and validated by a test database. The pharmacophore model_017 showed good performance in external validation and was employed to screen Traditional Chinese Medicine Database (Version 2009), which resulting in a hit list of 498 compounds with matching score (QFIT) above 40. Then, the hits were subjected to molecular docking for further refinement. An empirical scoring function was used to evaluate the affinity of the compounds and the target protein. Parts of compounds with high docking scores have been reported to have the related pharmacological activity from the literatures.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24212027',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 327,\n",
       "     'offsetInEndSection': 850,\n",
       "     'text': 'Hit compound 8 was identified as an inhibitor of tankyrases through a combination of substructure searching of the Amgen compound collection based on a minimal binding pharmacophore hypothesis and high-throughput screening. Herein we report the structure- and property-based optimization of compound 8 leading to the identification of more potent and selective tankyrase inhibitors 22 and 49 with improved pharmacokinetic properties in rodents, which are well suited as tool compounds for further in vivo validation studies',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24294969',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List selective estrogen receptor degraders.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/21501600',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25609068',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27005559',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25879485',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27178332',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26289836',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27151773',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26162914',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24601804',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26164151'],\n",
       "   'ideal_answer': ['Selective estrogen receptor degraders (SERD) are fulvestrant, RAD1901 and ARN-810. Fulvestrant is the only SERD approved for the treatment of breast cancer.'],\n",
       "   'exact_answer': [['fulvestrant'], ['RAD1901'], ['ARN-810']],\n",
       "   'concepts': ['https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D011960'],\n",
       "   'type': 'list',\n",
       "   'id': '58919aad621ea6ff7e00000d',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 311,\n",
       "     'text': 'For women with hormone receptor-positive advanced breast cancer, endocrine therapies, including the selective estrogen receptor modulator tamoxifen, the aromatase inhibitors anastrozole, letrozole, and exemestane, and the selective estrogen receptor degrader fulvestrant, are recommended in clinical guidelines.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27151773',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 629,\n",
       "     'offsetInEndSection': 828,\n",
       "     'text': 'Here we describe and characterize a novel, orally bioavailable small-molecule selective estrogen receptor degrader, RAD1901, and evaluate its therapeutic potential for the treatment of breast cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26164151',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 187,\n",
       "     'text': 'Identification of GDC-0810 (ARN-810), an Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) that Demonstrates Robust Activity in Tamoxifen-Resistant Breast Cancer Xenografts.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25879485',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 447,\n",
       "     'offsetInEndSection': 633,\n",
       "     'text': 'Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-α and is active in patients who have progressed on antihormonal agents. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25879485',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 96,\n",
       "     'text': 'Evaluation of the pharmacological activities of RAD1901, a selective estrogen receptor degrader.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26162914',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1528,\n",
       "     'offsetInEndSection': 1694,\n",
       "     'text': 'However, being the only SERD currently known to access the brain, RAD1901 merits evaluation as a targeted therapy for the treatment of breast cancer brain metastases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26162914',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 800,\n",
       "     'offsetInEndSection': 991,\n",
       "     'text': 'Tamoxifen, toremifene, and the selective estrogen receptor degrader fulvestrant are used to treat breast cancer, and tamoxifen and raloxifene protect against breast cancer in high-risk women.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26289836',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 167,\n",
       "     'text': 'The turnover of estrogen receptor α by the selective estrogen receptor degrader (SERD) fulvestrant is a saturable process that is not required for antagonist efficacy.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21501600',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 305,\n",
       "     'offsetInEndSection': 409,\n",
       "     'text': 'Currently, ICI 182,780 (ICI, fulvestrant) is the only SERD approved for the treatment of breast cancer. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21501600',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 447,\n",
       "     'offsetInEndSection': 632,\n",
       "     'text': 'Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-á and is active in patients who have progressed on antihormonal agents.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25879485',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 447,\n",
       "     'offsetInEndSection': 631,\n",
       "     'text': 'Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-α and is active in patients who have progressed on antihormonal agents',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25879485',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 146,\n",
       "     'text': 'RAD1901: a novel, orally bioavailable selective estrogen receptor degrader that demonstrates antitumor activity in breast cancer xenograft models.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26164151',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 829,\n",
       "     'offsetInEndSection': 954,\n",
       "     'text': 'RAD1901 selectively binds to and degrades the ER and is a potent antagonist of ER-positive breast cancer cell proliferation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26164151',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1113,\n",
       "     'offsetInEndSection': 1430,\n",
       "     'text': 'The selective estrogen receptor degrader, fulvestrant, significantly reduced the anchorage-independent growth of ESR1 mutant-expressing cells, while combination treatments with the mTOR inhibitor everolimus, or an inhibitor blocking IGF1R, and the insulin receptor significantly enhanced anti-proliferative responses.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27178332',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 445,\n",
       "     'offsetInEndSection': 677,\n",
       "     'text': 'SUM-44\\xa0cells do not require exogenous estrogen for growth in\\xa0vitro; however, they are dependent on ERα expression, as ESR1 knock-down or exposure to the selective estrogen receptor degrader fulvestrant resulted in growth inhibition.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27005559',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 97,\n",
       "     'text': 'Evaluation of the pharmacological activities of RAD1901, a selective estrogen receptor degrader.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26162914',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 447,\n",
       "     'offsetInEndSection': 632,\n",
       "     'text': 'Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-α and is active in patients who have progressed on antihormonal agents.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25879485',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': \"What is the benserazide's mechanism of function when co-administered with L-DOPA in patients with Parkinson's Disease?\",\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/20303948',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3768865',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12703659',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11166288',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8584079',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11587490',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25511986',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/6437857',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9372552',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/452602'],\n",
       "   'ideal_answer': [\"Benserazide is a peripheral decarboxylase inhibitor. Benserazide in combination with L-DOPA constitutes a slow-release formulation of L-DOPA in patients with Parkinson's disease (PD), it reduces peaks and rapid fluctuations of L-DOPA plasma levels (possibly responsible for peak-dose dyskinesias and end-of-dose deterioration) and considered as the best available therapy.\"],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:14330',\n",
       "    'http://www.biosemantics.org/jochem#http://www.biosemantics.org/jochem#:4275540',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D010300',\n",
       "    'http://www.biosemantics.org/jochem#4275540'],\n",
       "   'type': 'summary',\n",
       "   'id': '58a5723c60087bc10a00001e',\n",
       "   'snippets': [{'offsetInBeginSection': 160,\n",
       "     'offsetInEndSection': 776,\n",
       "     'text': \"Recent human and animal biochemical investigations clearly confirm and extend previous findings indicating that benserazide is much more potent than carbidopa as peripheral decarboxylase inhibitor. L-DOPA in combination with benserazide or carbidopa constitutes the best available therapy for Parkinson's disease (PD). To reduce peaks and rapid fluctuations of L-DOPA plasma levels (possibly responsible for peak-dose dyskinesias and end-of-dose deterioration) a slow-release formulation of L-DOPA in combination with benserazide or with benserazide plus catechol-O-methyltransferase inhibitors should be developed. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/6437857',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 322,\n",
       "     'text': \"In an open label study 63 patients with idiopathic Parkinson's disease suffering from end-of-dose akinesia were switched from a treatment with a L-DOPA standard formulation to a combined therapy of L-DOPA standard in the morning and L-DOPA slow release (levodopa, benserazide, Madopar Depot) at the remaining single doses.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8584079',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 206,\n",
       "     'offsetInEndSection': 438,\n",
       "     'text': 'In the present study, L-dopa and benserazide were loaded by poly (lactic-co-glycolic acid) microspheres (LBM), which can release levodopa and benserazide in a sustained manner in order to continuous stimulate dopaminergic receptors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25511986',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which fimA genotypes are associated with disease?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22549664',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20219603',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22466890',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23264452',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26387644',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22053966',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23646850',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24466164',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26937292',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26600627'],\n",
       "   'ideal_answer': ['FimA has been characterized as an important virulence factor for P. gingivalis, and many studies, both animal experiments and clinical investigations, have characterized fimA genotypes II, Ib, and IV to be associated with disease (periodontitis and cardiovascular disease)'],\n",
       "   'exact_answer': [['genotypes II'], ['genotypes Ib'], ['genotypes IV']],\n",
       "   'type': 'list',\n",
       "   'id': '58a94483ee23e0236b000003',\n",
       "   'snippets': [{'offsetInBeginSection': 476,\n",
       "     'offsetInEndSection': 750,\n",
       "     'text': ' FimA has been characterized as an important virulence factor for P. gingivalis, and many studies, both animal experiments and clinical investigations, have characterized fimA genotypes II, Ib, and IV to be associated with disease (periodontitis and cardiovascular disease) ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26387644',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 11,\n",
       "     'offsetInEndSection': 179,\n",
       "     'text': 'Long fimbriae (FimA) are important virulence factors of Porphyromonas gingivalis. Based on the diversity of the fimA gene, this species is classified into 6 genotypes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20219603',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 782,\n",
       "     'offsetInEndSection': 1066,\n",
       "     'text': ' In cases of chronic apical periodontitis, P. gingivalis variant type IV was the most prevalent (24%), followed by types I (20%), II (16%), and III (8%). In acute abscess samples, variant type II was the most prevalent (12%), followed by types III and IV (8% of each) and type I (4%).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20219603',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 959,\n",
       "     'offsetInEndSection': 1082,\n",
       "     'text': 'We show that P. gingivalis strains with genotype I and II of FimA are efficient in interaction with saliva or S. gordonii. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22053966',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1238,\n",
       "     'offsetInEndSection': 1435,\n",
       "     'text': 'A strong association between Porphyromonas gingivalis fimA genotypes II and Ib and chronic periodontitis exists in the Spanish population. The most prevalent genotype in periodontal patients is II.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22549664',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1022,\n",
       "     'offsetInEndSection': 1206,\n",
       "     'text': 'Our results suggest that the presence of P. gingivalis is associated with periodontal diseases, and that the type II, IV and Ib/II combination are the most common among fimA genotypes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22466890',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1218,\n",
       "     'offsetInEndSection': 1361,\n",
       "     'text': 'Statistical analysis, however, revealed that a more significant correlation was found between periodontitis and the occurrence of type Ib fimA.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23264452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1667,\n",
       "     'offsetInEndSection': 1755,\n",
       "     'text': ' Type II of fimA was the most prevalent genotype of P. gingivalis in patients with AgP. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23646850',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1211,\n",
       "     'offsetInEndSection': 1409,\n",
       "     'text': 'The population of Tregs further decreased in patients with type II FimA compared with the other types. P.gingivlias FimA genotype II was the dominant type associated with decreased Treg population. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24466164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1099,\n",
       "     'offsetInEndSection': 1220,\n",
       "     'text': 'A trend toward a greater frequency of FimA II genotype in patients with moderate and severe periodontitis was determined.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26600627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1251,\n",
       "     'offsetInEndSection': 1441,\n",
       "     'text': 'The fimA type Ib genotype of P. gingivalis was found to play a critical role in the destruction of peri-implant tissue, suggesting that it may be a distinct risk factor for peri-implantitis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26937292',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Viliuisk encephalomyelitis is diagnosed in which geographical area?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/7848104',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20010208',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19618339',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1393513',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15182325',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18379734',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9223130',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24586232',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18252102',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2894813'],\n",
       "   'ideal_answer': ['Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeast Siberia and generally considered to be a chronic encephalomyelitis of unknown origin actually spreading in the Sakha (Yakutian) Republic.'],\n",
       "   'exact_answer': ['Northeast Siberia'],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:640',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D004679'],\n",
       "   'type': 'factoid',\n",
       "   'id': '588f9f83ed9bbee70d000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 229,\n",
       "     'text': 'BACKGROUND: Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeast Siberia and generally considered to be a chronic encephalomyelitis of unknown origin actually spreading in the Sakha (Yakutian) Republic.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24586232',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 139,\n",
       "     'text': 'Communicating hydrocephalus following eosinophilic meningitis is pathogenic for chronic Viliuisk encephalomyelitis in Northeastern Siberia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24586232',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 62,\n",
       "     'text': 'Epidemiology of Viliuisk encephalomyelitis in Eastern Siberia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20010208',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 245,\n",
       "     'offsetInEndSection': 350,\n",
       "     'text': '. An epidemic of this disease has been spreading throughout the Yakut Republic of the Russian Federation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20010208',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 502,\n",
       "     'offsetInEndSection': 729,\n",
       "     'text': 'METHODS: Detailed clinical, pathologic, laboratory, and epidemiologic studies have identified 414 patients with definite Viliuisk encephalomyelitis in 15 of 33 administrative regions of the Yakut Republic between 1940 and 1999.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20010208',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 70,\n",
       "     'text': 'Viliuisk encephalomyelitis in Eastern Siberia - analysis of 390 cases.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19618339',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 114,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is a unique disease occurring in the Yakut (Sakha) population of Eastern Siberia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19618339',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 99,\n",
       "     'text': 'Viliuisk encephalomyelitis in Northeastern Siberia is not caused by Borrelia burgdorferi infection.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18379734',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 167,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeastern Siberia and generally believed to be a chronic encephalomyelitis of unknown origin. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18379734',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 166,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeastern Siberia and generally believed to be a chronic encephalomyelitis of unknown origin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18379734',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 217,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeast Siberia and generally considered to be a chronic encephalomyelitis of unknown origin actually spreading in the Sakha (Yakutian) Republic.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24586232',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 58,\n",
       "     'text': 'Viliuisk encephalomyelitis in the Iakut people of Siberia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1393513',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 727,\n",
       "     'offsetInEndSection': 983,\n",
       "     'text': 'Initially identified in a small Yakut-Evenk population on the Viliui River of eastern Siberia, the disease subsequently spread through human contacts to new geographic areas, thus characterizing Viliuisk encephalomyelitis as an emerging infectious disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18252102',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is an unique neurological disease occurring in the Iakut (Sakha) people of Siberia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9223130',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 124,\n",
       "     'text': 'Viliuisk encephalomyelitis (VEM) appears to be endemic disease, affecting native population in Yakutia (Yakut, Even, Evenk).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7848104',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 534,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeast Siberia and generally considered to be a chronic encephalomyelitis of unknown origin actually spreading in the Sakha (Yakutian) Republic.In search for the pathophysiology and causative agent of VE, we performed a cross-sectional study on clinical, serological and neuroimaging data on chronic VE patients during two medical expeditions to three villages within the Viliuiski river basin in the Republic of Sakha in 2000 and to the capital Yakutsk in 2006',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24586232',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 732,\n",
       "     'offsetInEndSection': 987,\n",
       "     'text': 'Initially identified in a small Yakut-Evenk population on the Viliui River of eastern Siberia, the disease subsequently spread through human contacts to new geographic areas, thus characterizing Viliuisk encephalomyelitis as an emerging infectious disease',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18252102',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 112,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is a unique disease occurring in the Yakut (Sakha) population of Eastern Siberia',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19618339',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 98,\n",
       "     'text': 'Viliuisk encephalomyelitis in Northeastern Siberia is not caused by Borrelia burgdorferi infection',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18379734',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 165,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeastern Siberia and generally believed to be a chronic encephalomyelitis of unknown origin',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18379734',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 732,\n",
       "     'offsetInEndSection': 988,\n",
       "     'text': 'Initially identified in a small Yakut-Evenk population on the Viliui River of eastern Siberia, the disease subsequently spread through human contacts to new geographic areas, thus characterizing Viliuisk encephalomyelitis as an emerging infectious disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18252102',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 535,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE) is an endemic neurological disease in Northeast Siberia and generally considered to be a chronic encephalomyelitis of unknown origin actually spreading in the Sakha (Yakutian) Republic.In search for the pathophysiology and causative agent of VE, we performed a cross-sectional study on clinical, serological and neuroimaging data on chronic VE patients during two medical expeditions to three villages within the Viliuiski river basin in the Republic of Sakha in 2000 and to the capital Yakutsk in 2006.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24586232',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 447,\n",
       "     'offsetInEndSection': 926,\n",
       "     'text': 'IgG antibodies to HTLV-I were measured in the sera and/or cerebrospinal fluid from 82 Guamanian patients with amyotrophic lateral sclerosis and parkinsonism-dementia, 164 Guamanian normal controls, 10 patients with kuru from the Eastern Highlands of Papua New Guinea, 4 patients with Viliuisk encephalomyelitis from the Iakut region of eastern Siberia, 45 Italian patients with multiple sclerosis, and 56 patients with polymyositis (49 from the United States and 7 from Jamaica).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2894813',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 193,\n",
       "     'text': 'Viliuisk encephalomyelitis (VE), a progressive neurological disorder with a fatal outcome usually in several months to 6 yrs after disease onset, is seen only among the Iakut people of Siberia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1393513',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 559,\n",
       "     'offsetInEndSection': 731,\n",
       "     'text': 'Transmission to unrelated persons was documented in a densely populated region around the city of Yakutsk in which Viliuisk encephalomyelitis had not been previously known.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18252102',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 732,\n",
       "     'offsetInEndSection': 989,\n",
       "     'text': 'Initially identified in a small Yakut-Evenk population on the Viliui River of eastern Siberia, the disease subsequently spread through human contacts to new geographic areas, thus characterizing Viliuisk encephalomyelitis as an emerging infectious disease..',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18252102',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 63,\n",
       "     'text': 'Epidemiology of Viliuisk encephalomyelitis in Eastern Siberia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20010208',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 59,\n",
       "     'text': 'Viliuisk encephalomyelitis in the Iakut people of Siberia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1393513',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 71,\n",
       "     'text': 'Viliuisk encephalomyelitis in Eastern Siberia - analysis of 390 cases.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19618339',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Describe Exploding head syndrome.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24703829',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24907167',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20112796',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1896728',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20726288',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25773787',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11309216',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2030791',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2769286',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23467433'],\n",
       "   'ideal_answer': ['Exploding head syndrome is characterized by the perception of abrupt, loud noises when going to sleep or waking up.'],\n",
       "   'concepts': ['https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D013577',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:225'],\n",
       "   'type': 'summary',\n",
       "   'id': '588f9756ed9bbee70d000001',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 118,\n",
       "     'text': 'Exploding head syndrome is characterized by the perception of loud noises during sleep-wake or wake-sleep transitions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25773787',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'Exploding head syndrome is characterized by the perception of abrupt, loud noises when going to sleep or waking up.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24703829',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 206,\n",
       "     'text': 'BACKGROUND: Exploding head syndrome (EHS) is characterized by attacks of a sudden noise or explosive feeling experienced in the head occurring during the transition from wake to sleep or from sleep to wake.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24907167',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 147,\n",
       "     'text': 'INTRODUCTION: Exploding head syndrome (EHS) is a rare parasomnia in which affected individuals awaken from sleep with the sensation of a loud bang.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23467433',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 194,\n",
       "     'text': 'Exploding head syndrome (EHS) is characterized by attacks of a sudden noise or explosive feeling experienced in the head occurring during the transition from wake to sleep or from sleep to wake.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24907167',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 171,\n",
       "     'text': 'Exploding head syndrome (EHS) attacks are characterized by the sensation of sudden loud banging noises, and are occasionally accompanied by the sensation of a flash light.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20112796',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 270,\n",
       "     'text': 'Exploding head syndrome (EHS) is characterized by attacks of a sudden noise or explosive feeling experienced in the head occurring during the transition from wake to sleep or from sleep to wake.We present six new cases extending the clinical experience with the syndrome',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24907167',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 132,\n",
       "     'text': 'Exploding head syndrome (EHS) is a rare parasomnia in which affected individuals awaken from sleep with the sensation of a loud bang',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23467433',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 172,\n",
       "     'text': 'Exploding head syndrome (EHS) attacks are characterized by the sensation of sudden loud banging noises, and are occasionally accompanied by the sensation of a flash light. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20112796',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 271,\n",
       "     'text': 'Exploding head syndrome (EHS) is characterized by attacks of a sudden noise or explosive feeling experienced in the head occurring during the transition from wake to sleep or from sleep to wake.We present six new cases extending the clinical experience with the syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24907167',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is H4K20 methylation associated with DNA replication?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24049080',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24013172',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23152447',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20735237',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23924899',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23345616',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26598646',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27268234',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23754963'],\n",
       "   'ideal_answer': ['We employed genetic, cytological, and genomic approaches to better understand the role of PR-Set7 and H4K20 methylation in regulating DNA replication and genome stability in Drosophila cells. Thus, coordinating the status of H4K20 methylation is pivotal for the proper selection of DNA replication origins in higher eukaryotes. The methylation state of lysine 20 on histone H4 (H4K20) has been linked to chromatin compaction, transcription, DNA repair and DNA replication. Histone turnover is often associated with various histone modifications such as H3K56 acetylation (H3K56Ac), H3K36 methylation (H3K36me), and H4K20 methylation (H4K20me). We review the signaling pathways and functions associated with a single residue, H4K20, as a model chromatin and clinically important mark that regulates biological processes ranging from the DNA damage response and DNA replication to gene expression and silencing. <CopyrightInformation>© 2016 by The American Society for Biochemistry and Molecular Biology, Inc.</C In particular, the methylation states of H3K4, H3K36 and H4K20 have been associated with establishing active, repressed or poised origins depending on the timing and extent of methylation. 5BrC and 5ClC may cause aberrant methylation of cytosine during DNA replication and mimic the endogenous methylation signal associated with gene silencing.',\n",
       "    'H4K20 methylation regulates quiescence and chromatin compaction. H4K20 methylation status plays a direct role in recruiting ORC through the binding properties of ORC1 and ORCA/LRWD1. Thus, coordinating the status of H4K20 methylation is pivotal for the proper selection of DNA replication origins in higher eukaryotes.  Intense research during the past few years has revealed histone H4 lysine 20 methylation (H4K20me) as critically important for the biological processes that ensure genome integrity, such as DNA damage repair, DNA replication and chromatin compaction.',\n",
       "    'The methylation state of lysine 20 on histone H4 (H4K20) has been linked to chromatin compaction, transcription, DNA repair and DNA replication. Disruption of these H4K20-specific histone methyltransferases leads to genomic instability, demonstrating the important functions of H4K20 methylation in genome maintenance.',\n",
       "    'Thus, coordinating the status of H4K20 methylation is pivotal for the proper selection of DNA replication origins in higher eukaryotes. The methylation state of lysine 20 on histone H4 (H4K20) has been linked to chromatin compaction, transcription, DNA repair and DNA replication.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '58adbe999ef3c34033000005',\n",
       "   'snippets': [{'offsetInBeginSection': 923,\n",
       "     'offsetInEndSection': 1178,\n",
       "     'text': 'It seems likely that continued study of the methylation of H4K20 will yield extremely valuable insights concerning the regulation of histone modifications before and during cell division and the impact of these modifications on subsequent gene expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20735237',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 505,\n",
       "     'offsetInEndSection': 629,\n",
       "     'text': 'Aberrant rereplication correlates with decreased levels of H4K20me1 and increased levels of H4K20 trimethylation (H4K20me3).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23152447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 980,\n",
       "     'offsetInEndSection': 1255,\n",
       "     'text': 'Consistent with this, H4K20 methylation status plays a direct role in recruiting ORC through the binding properties of ORC1 and ORCA/LRWD1. Thus, coordinating the status of H4K20 methylation is pivotal for the proper selection of DNA replication origins in higher eukaryotes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23152447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 64,\n",
       "     'text': 'H4K20 methylation regulates quiescence and chromatin compaction.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23924899',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 371,\n",
       "     'offsetInEndSection': 580,\n",
       "     'text': 'Mass spectrometry analysis of histone modifications reveals that H4K20me2 and H4K20me3 increase in quiescence and other histone modifications are present at similar levels in proliferating and quiescent cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23924899',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 581,\n",
       "     'offsetInEndSection': 726,\n",
       "     'text': 'Analysis of cells in S, G2/M, and G1 phases shows that H4K20me1 increases after S phase and is converted to H4K20me2 and H4K20me3 in quiescence. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23924899',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 898,\n",
       "     'offsetInEndSection': 1249,\n",
       "     'text': 'Overexpression of Suv4-20h1, the enzyme that creates H4K20me2 from H4K20me1, results in G2 arrest, consistent with a role for H4K20me1 in mitosis. The results suggest that the same lysine on H4K20 may, in its different methylation states, facilitate mitotic functions in M phase and promote chromatin compaction and cell cycle exit in quiescent cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23924899',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 91,\n",
       "     'text': 'Histone H4 lysine 20 methylation: key player in epigenetic regulation of genomic integrity.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23345616',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 360,\n",
       "     'offsetInEndSection': 611,\n",
       "     'text': ' Intense research during the past few years has revealed histone H4 lysine 20 methylation (H4K20me) as critically important for the biological processes that ensure genome integrity, such as DNA damage repair, DNA replication and chromatin compaction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23345616',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 820,\n",
       "     'offsetInEndSection': 994,\n",
       "     'text': 'Disruption of these H4K20-specific histone methyltransferases leads to genomic instability, demonstrating the important functions of H4K20 methylation in genome maintenance. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23345616',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 113,\n",
       "     'text': 'Both H4K20 mono-methylation and H3K56 acetylation mark transcription-dependent histone turnover in fission yeast.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27268234',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 121,\n",
       "     'offsetInEndSection': 291,\n",
       "     'text': 'Histone turnover is often associated with various histone modifications such as H3K56 acetylation (H3K56Ac), H3K36 methylation (H3K36me), and H4K20 methylation (H4K20me).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27268234',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 974,\n",
       "     'offsetInEndSection': 1121,\n",
       "     'text': 'These results together indicate that H4K20me1 as well as H3K56Ac are bona fide marks for transcription-dependent histone turnover in fission yeast.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27268234',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 120,\n",
       "     'text': 'Methylation of histone H4 lysine 20 by PR-Set7 ensures the integrity of late replicating sequence domains in Drosophila.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 453,\n",
       "     'offsetInEndSection': 636,\n",
       "     'text': 'However, these studies were limited to only a handful of mammalian origins, and it remains unclear how PR-Set7 and H4K20 methylation impact the replication program on a genomic scale.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 254,\n",
       "     'text': 'The methylation state of lysine 20 on histone H4 (H4K20) has been linked to chromatin compaction, transcription, DNA repair and DNA replication. Monomethylation of H4K20 (H4K20me1) is mediated by the cell cycle-regulated histone methyltransferase PR-Set7',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 828,\n",
       "     'offsetInEndSection': 1066,\n",
       "     'text': ' We find that deregulation of H4K20 methylation had no impact on origin activation throughout the genome. Instead, depletion of PR-Set7 and loss of H4K20me1 results in the accumulation of DNA damage and an ATR-dependent cell cycle arrest.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 610,\n",
       "     'offsetInEndSection': 875,\n",
       "     'text': 'We review the signaling pathways and functions associated with a single residue, H4K20, as a model chromatin and clinically important mark that regulates biological processes ranging from the DNA damage response and DNA replication to gene expression and silencing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26598646',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1113,\n",
       "     'offsetInEndSection': 1248,\n",
       "     'text': 'Thus, coordinating the status of H4K20 methylation is pivotal for the proper selection of DNA replication origins in higher eukaryotes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23152447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 633,\n",
       "     'offsetInEndSection': 824,\n",
       "     'text': 'We employed genetic, cytological, and genomic approaches to better understand the role of PR-Set7 and H4K20 methylation in regulating DNA replication and genome stability in Drosophila cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 144,\n",
       "     'text': 'The methylation state of lysine 20 on histone H4 (H4K20) has been linked to chromatin compaction, transcription, DNA repair and DNA replication.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 450,\n",
       "     'offsetInEndSection': 633,\n",
       "     'text': 'However, these studies were limited to only a handful of mammalian origins, and it remains unclear how PR-Set7 and H4K20 methylation impact the replication program on a genomic scale.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 205,\n",
       "     'offsetInEndSection': 351,\n",
       "     'text': 'Methylation of histone H4 lysine-20 has been implicated in DNA repair, transcriptional silencing, genomic stability and regulation of replication.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24049080',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1120,\n",
       "     'offsetInEndSection': 1254,\n",
       "     'text': 'Thus, coordinating the status of H4K20 methylation is pivotal for the proper selection of DNA replication origins in higher eukaryotes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23152447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Methylation of histone H3 on lysine 79 associates with a group of replication origins and helps limit DNA replication once per cell cycle',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23754963',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 637,\n",
       "     'offsetInEndSection': 828,\n",
       "     'text': 'We employed genetic, cytological, and genomic approaches to better understand the role of PR-Set7 and H4K20 methylation in regulating DNA replication and genome stability in Drosophila cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27131378',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 610,\n",
       "     'offsetInEndSection': 976,\n",
       "     'text': 'We review the signaling pathways and functions associated with a single residue, H4K20, as a model chromatin and clinically important mark that regulates biological processes ranging from the DNA damage response and DNA replication to gene expression and silencing. <CopyrightInformation>© 2016 by The American Society for Biochemistry and Molecular Biology, Inc.</C',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26598646',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1120,\n",
       "     'offsetInEndSection': 1256,\n",
       "     'text': 'Thus, coordinating the status of H4K20 methylation is pivotal for the proper selection of DNA replication origins in higher eukaryotes..',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23152447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 361,\n",
       "     'offsetInEndSection': 611,\n",
       "     'text': 'Intense research during the past few years has revealed histone H4 lysine 20 methylation (H4K20me) as critically important for the biological processes that ensure genome integrity, such as DNA damage repair, DNA replication and chromatin compaction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23345616',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which treatment methods were compared in the EXCEL Trial?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/21547994',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25983143',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26159652',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25583761',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21796086',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22941121',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22075415',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27639738',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22511269',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23995725'],\n",
       "   'ideal_answer': ['EXCEL trial compared Everolimus Eluting Stent vs. Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization.'],\n",
       "   'exact_answer': [['Everolimus Eluting Stent'],\n",
       "    ['Coronary Artery Bypass Surgery']],\n",
       "   'concepts': ['https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D017428',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D002986',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D016430',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D017326'],\n",
       "   'type': 'list',\n",
       "   'id': '5880b583c872c95565000005',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': 'Long-term forecasting and comparison of mortality in the Evaluation of the Xience Everolimus Eluting Stent vs. Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial: prospective validation of the SYNTAX Score II.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25583761',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 247,\n",
       "     'text': 'AIMS: To prospectively validate the SYNTAX Score II and forecast the outcomes of the randomized Evaluation of the Xience Everolimus-Eluting Stent Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) Trial.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25583761',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1385,\n",
       "     'offsetInEndSection': 1665,\n",
       "     'text': 'Based on 4-year mortality predictions in EXCEL, clinical characteristics shifted long-term mortality predictions either in favour of PCI (older age, male gender and COPD) or CABG (younger age, lower creatinine clearance, female gender, reduced left ventricular ejection fraction).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25583761',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 463,\n",
       "     'offsetInEndSection': 797,\n",
       "     'text': 'Given that the completed randomized trials did not include contemporary DESs, the upcoming results of the ongoing trials evaluating the performance of new-generation DES compared with CABG (such as the EXCEL trial), may further help to clarify the current role and future recommendations of PCI for left main coronary artery disease. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26159652',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 531,\n",
       "     'offsetInEndSection': 686,\n",
       "     'text': 'The ongoing EXCEL trial will help elucidate the role of ULMCA PCI in the treatment of left main disease compared with coronary artery bypass graft surgery.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23995725',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1151,\n",
       "     'offsetInEndSection': 1412,\n",
       "     'text': 'The ongoing Evaluation of XIENCE V Everolimus Eluting Stent System Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial is expected to provide a better answer on the optimal treatment strategy for LMCAD patients. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22941121',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 496,\n",
       "     'text': 'The Evaluation of Xience Prime or Xience V versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial is a multicenter, ongoing trial conducted in patients with left main disease and SYNTAX score ≤ 32 to establish the presumptive advantage of percutaneous coronary intervention (PCI) versus bypass surgery in patients with less complex coronary artery disease than those enrolled in the Synergy between PCI with Taxus and Cardiac Surgery (SYNTAX) trial. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22075415',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 965,\n",
       "     'offsetInEndSection': 1095,\n",
       "     'text': 'The EXCEL trial will address the value of PCI in relation to CABG for the treatment of ULMCA stenosis in more than 2000 patients. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22511269',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 478,\n",
       "     'text': 'OBJECTIVES: The aim of this study is to verify the study hypothesis of the EXCEL trial by comparing percutaneous coronary intervention (PCI) and coronary artery bypass graft (CABG) in an EXCEL-like population of patients.BACKGROUND: The upcoming EXCEL trial will test the hypothesis that left main patients with SYNTAX score ≤ 32 experience similar rates of 3-year death, myocardial infarction (MI), or cerebrovascular accidents (CVA) following revascularization by PCI or CABG.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21547994',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1403,\n",
       "     'offsetInEndSection': 1562,\n",
       "     'text': 'CONCLUSIONS: In an EXCEL-like cohort of patients with left main disease, there seems to be a clinical equipoise between PCI and CABG in terms of death/MI/CVA. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21547994',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 209,\n",
       "     'text': 'The aim of this study is to verify the study hypothesis of the EXCEL trial by comparing percutaneous coronary intervention (PCI) and coronary artery bypass graft (CABG) in an EXCEL-like population of patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21547994',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 730,\n",
       "     'text': 'To prospectively validate the SYNTAX Score II and forecast the outcomes of the randomized Evaluation of the Xience Everolimus-Eluting Stent Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) Trial.Evaluation of the Xience Everolimus Eluting Stent vs. Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization is a prospective, randomized multicenter trial designed to establish the efficacy and safety of percutaneous coronary intervention (PCI) with the everolimus-eluting stent compared with coronary artery bypass graft (CABG) surgery in subjects with unprotected left-main coronary artery (ULMCA) disease and low-intermediate anatomical SYNTAX scores (&lt;33).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25583761',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 326,\n",
       "     'text': 'OBJECTIVES: The aim of this study is to verify the study hypothesis of the EXCEL trial by comparing percutaneous coronary intervention (PCI) and coronary artery bypass graft (CABG) in an EXCEL-like population of patients.BACKGROUND: The upcoming EXCEL trial will test the hypothesis that left main patients with SYNTAX score ?',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21547994',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 679,\n",
       "     'offsetInEndSection': 902,\n",
       "     'text': 'The results of the ongoing EXCEL trial, which compares left main percutaneous coronary intervention with drug-eluting stents and CABG, will provide insight regarding the ideal revascularization strategy for these patients..',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21796086',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 463,\n",
       "     'offsetInEndSection': 796,\n",
       "     'text': 'Given that the completed randomized trials did not include contemporary DESs, the upcoming results of the ongoing trials evaluating the performance of new-generation DES compared with CABG (such as the EXCEL trial), may further help to clarify the current role and future recommendations of PCI for left main coronary artery disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26159652',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 363,\n",
       "     'offsetInEndSection': 904,\n",
       "     'text': 'The introduction of the newer-generation drug-eluting stents (DES) -with documented improvements in both safety and efficacy- has prompted the interventional community to design two new dedicated randomised trials comparing CABG and PCI: the NOBLE (Coronary Artery Bypass Grafting Vs Drug Eluting Stent Percutaneous Coronary Angioplasty in the Treatment of Unprotected Left Main Stenosis) and EXCEL (Evaluation of XIENCE Everolimus Eluting Stent Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization) trials.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25983143',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 965,\n",
       "     'offsetInEndSection': 1094,\n",
       "     'text': 'The EXCEL trial will address the value of PCI in relation to CABG for the treatment of ULMCA stenosis in more than 2000 patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22511269',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1108,\n",
       "     'offsetInEndSection': 1368,\n",
       "     'text': 'The ongoing Evaluation of XIENCE V Everolimus Eluting Stent System Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial is expected to provide a better answer on the optimal treatment strategy for LMCAD patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22941121',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Have studies shown that there is no link between DNA methylation patterns and Post Traumatic Stress Disorder?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27886370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21508515',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25065861',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21714072',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26361058',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22332656',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21306736',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26305478',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26447678',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19793403'],\n",
       "   'ideal_answer': ['Studies do show a correlation of PTSD-related accelerated aging in DNA methylation patterns.'],\n",
       "   'exact_answer': 'no',\n",
       "   'concepts': ['https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D040921',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:2055',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D013313',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D019175'],\n",
       "   'type': 'yesno',\n",
       "   'id': '58e2422c6fddd3e83e000011',\n",
       "   'snippets': [{'offsetInBeginSection': 918,\n",
       "     'offsetInEndSection': 1502,\n",
       "     'text': 'Using pre-deployment SKA2 methylation levels and childhood trauma exposure, we found that the previously published suicide prediction rule significantly predicted post-deployment PTSD symptoms (AUC=0.66, 95% CI: 0.53-0.79) with an optimal sensitivity of 0.81 and specificity of 0.91. Permutation analysis using random methylation loci supported these findings. Together, these data establish the importance of SKA2 for cortisol stress responsivity and the development of PTSD and provide further evidence that SKA2 is a promising biomarker for stress-related disorders including PTSD.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26361058',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1126,\n",
       "     'offsetInEndSection': 1307,\n",
       "     'text': ' Results provide novel support for PTSD-related accelerated aging in DNAm and extend the evidence base of known DNAm age correlates to the domains of neural integrity and cognition.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26447678',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 234,\n",
       "     'text': 'We investigated serum DNA methylation patterns in genomic repetitive elements, LINE-1 and Alu, for post-traumatic stress disorder (PTSD) cases and controls who were US military service members recently deployed to Afghanistan or Iraq.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22332656',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 97,\n",
       "     'offsetInEndSection': 321,\n",
       "     'text': 'In light of its role in glucocorticoid receptor transactivation, we investigated whether SKA2 DNA methylation influences cortisol stress reactivity and is involved in the development of post-traumatic stress disorder (PTSD).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26361058',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1398,\n",
       "     'offsetInEndSection': 1579,\n",
       "     'text': 'These results suggest that alterations in global methylation pattern are involved in behavioural adaptation to environmental stress and pinpoint Dlgap2 as a possible target in PTSD.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19793403',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 146,\n",
       "     'offsetInEndSection': 286,\n",
       "     'text': 'Here we examined whether there was a link between an established rat model of post-traumatic stress disorder (PTSD) and Bdnf DNA methylation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21306736',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 287,\n",
       "     'text': 'We investigated serum DNA methylation patterns in genomic repetitive elements, LINE-1 and Alu, for post-traumatic stress disorder (PTSD) cases and controls who were US military service members recently deployed to Afghanistan or Iraq.Cases (n = 75) had a postdeployment diagnosis of PTSD',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22332656',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': 'DNA methylation in repetitive elements and post-traumatic stress disorder: a case-control study of US military service members',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22332656',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 146,\n",
       "     'offsetInEndSection': 288,\n",
       "     'text': 'Here we examined whether there was a link between an established rat model of post-traumatic stress disorder (PTSD) and Bdnf DNA methylation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21306736',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'DNA methylation in repetitive elements and post-traumatic stress disorder: a case-control study of US military service members.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22332656',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 240,\n",
       "     'text': 'AIM: We investigated serum DNA methylation patterns in genomic repetitive elements, LINE-1 and Alu, for post-traumatic stress disorder (PTSD) cases and controls who were US military service members recently deployed to Afghanistan or Iraq. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22332656',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1221,\n",
       "     'offsetInEndSection': 1398,\n",
       "     'text': 'Together, these results suggest that psychosocial stress may alter global and gene-specific DNA methylation patterns potentially associated with peripheral immune dysregulation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21714072',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 134,\n",
       "     'text': 'DNA methylation in vulnerability to post-traumatic stress in rats: evidence for the role of the post-synaptic density protein Dlgap2.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19793403',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 763,\n",
       "     'offsetInEndSection': 885,\n",
       "     'text': 'Subjects with PTSD showed a higher DNA methylation of four CpG sites at the BDNF promoter compared with those without PTSD',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27886370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1427,\n",
       "     'offsetInEndSection': 1606,\n",
       "     'text': 'Cumulatively, the data suggest that epigenetic variation at SKA2 mediates vulnerability to suicidal behaviors and PTSD through dysregulation of the HPA axis in response to stress.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26305478',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is autophagy the process where bacteria ingest viral particles?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24906121',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23121192',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27814601',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21946213',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23388380',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21175768',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27027951',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24198400',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24384599',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27812870'],\n",
       "   'ideal_answer': ['Autophagy, a cellular degradation process'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D001343'],\n",
       "   'type': 'yesno',\n",
       "   'id': '58dd2cb08acda34529000029',\n",
       "   'snippets': [{'offsetInBeginSection': 255,\n",
       "     'offsetInEndSection': 296,\n",
       "     'text': 'Autophagy, a cellular degradation process',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27812870',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 155,\n",
       "     'text': 'Autophagy, a form of lysosomal degradation capable of eliminating dysfunctional proteins and organelles, is a cellular process associated with homeostasis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27814601',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 362,\n",
       "     'text': 'Autophagy, a programmed process in which cell contents are delivered to lysosomes for degradation, appears to have both tumor-suppressive and tumor-promoting functions; both stimulation and inhibition of autophagy have been reported to induce cancer cell death, and particular genes and proteins have been associated both positively and negatively with autophagy',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24906121',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': 'Autophagy is a lysosome-mediated catabolic process involving the degradation of intracellular contents (e.g., proteins and organelles) as well as invading microbes (e.g., parasites, bacteria and viruses).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23388380',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 187,\n",
       "     'text': 'Autophagy is a cellular process that targets proteins, lipids and organelles to lysosomes for degradation, but it has also been shown to combat infection with various pathogenic bacteria.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24384599',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 163,\n",
       "     'text': 'Autophagy, an intracellular degradation process highly conserved from yeast to humans, is viewed as an important defence mechanism to clear intracellular bacteria.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23121192',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 89,\n",
       "     'offsetInEndSection': 274,\n",
       "     'text': 'Autophagy has intracellular anti-viral and anti-bacterial functions, and plays a role in the initiation of innate and adaptive immune system responses to viral and bacterial infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21175768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1562,\n",
       "     'offsetInEndSection': 1756,\n",
       "     'text': 'documented abundant autophagy within VZV-infected cells throughout the infectious cycle but also demonstrated that VZV-induced autophagy facilitated VZV glycoprotein biosynthesis and processing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24198400',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': 'Autophagy is a highly conserved process by which cells can recycle organelles and proteins by degrading them in the lysosomes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27027951',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is intraoperative radiotherapy used for treatment of glioblastoma?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25535398',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8092110',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/6328061',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7623073',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26824195',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3014858',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2852827',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25724425',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16044217',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7709332'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0338240',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A4352743'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A4352743',\n",
       "     'o': 'intraoperative radiotherapy'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0017636',\n",
       "     'o': 'GBM'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#altLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0338240',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A7582156'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A7582156',\n",
       "     'o': 'radiotherapy, intraoperative'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0338240',\n",
       "     'o': 'intraoperative radiation therapy'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#altLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0017636',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A18647416'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A18647416',\n",
       "     'o': 'glioblastoma'}],\n",
       "   'ideal_answer': ['Yes, intraoperative radiotherapy (IORT) is being used for treatment of glioblastoma. IORT combined with extensive tumor removal has an acceptable toxicity in previously irradiated patients and can be effective for selected recurrent malignant brain tumors.'],\n",
       "   'exact_answer': 'no',\n",
       "   'concepts': ['https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D005909',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:3068',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D011878',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:3073',\n",
       "    'https://www.nlm.nih.gov/cgi/mesh/2017/MB_cgi?field=uid&exact=Find+Exact+Term&term=D007430'],\n",
       "   'type': 'yesno',\n",
       "   'id': '589a245478275d0c4a000022',\n",
       "   'snippets': [{'offsetInBeginSection': 181,\n",
       "     'offsetInEndSection': 294,\n",
       "     'text': ' 1) Intraoperative radiotherapy ( IOR , 1,000-2,000 rad) was applied in 13 cases; the 2-year survival was 41.6%. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/6328061',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 588,\n",
       "     'offsetInEndSection': 837,\n",
       "     'text': 'The median survival time after IORT was 12 months for 9 patients with glioblastoma or anaplastic astrocytoma, while it was 51 months for 8 patients with less infiltrative tumors (ependymoma, anaplastic ependymoma, and anaplastic oligodendroglioma). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8092110',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1147,\n",
       "     'offsetInEndSection': 1339,\n",
       "     'text': 'It is concluded that IORT combined with extensive tumor removal has an acceptable toxicity in previously irradiated patients and can be effective for selected recurrent malignant brain tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8092110',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1467,\n",
       "     'offsetInEndSection': 1642,\n",
       "     'text': 'CONCLUSIONS: The results of this study indicate that IORT can contribute to successful tumor treatment while neither increasing peri-operative morbidity nor subacute sequelae.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7709332',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 512,\n",
       "     'offsetInEndSection': 694,\n",
       "     'text': 'This review compiles preclinical and clinical evidence for a dedicated treatment of both residual cancer cells and regional microenvironment using intraoperative radiotherapy (IORT).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26824195',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 380,\n",
       "     'offsetInEndSection': 624,\n",
       "     'text': ' Intraoperative radiotherapy (IORT) is a pragmatic and effective approach to sterilize the margins from persistent tumor cells, abrogate post-injury proliferative stimuli and to bridge the therapeutic gap between surgery and radiochemotherapy. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25535398',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': '[Surgical treatment and radiation therapy for glioblastoma multiforme, with special reference to intraoperative radiotherapy].',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/6328061',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 58,\n",
       "     'text': 'Rationale for intraoperative radiotherapy in glioblastoma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26824195',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 149,\n",
       "     'text': 'Intraoperative radiotherapy (IORT) was performed in 20 of 36 patients with glioma; 11 glioblastomas, 7 malignant astrocytomas, 2 benign astrocytomas.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7623073',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 125,\n",
       "     'text': '[Surgical treatment and radiation therapy for glioblastoma multiforme, with special reference to intraoperative radiotherapy]',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/6328061',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 98,\n",
       "     'text': 'INTRAGO: intraoperative radiotherapy in glioblastoma multiforme—a phase I/II dose escalation study',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25535398',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 61,\n",
       "     'text': 'Intraoperative cobalt-60 treatment of glioblastoma multiforme',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2852827',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 168,\n",
       "     'text': 'An intraoperative remote afterloading endocurietherapy (ECT) technique with high-activity 60cobalt (60Co) for the treatment of glioblastoma multiforme (GM) is described',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2852827',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': '[Surgical treatment and radiation therapy for glioblastoma multiforme, with special reference to intraoperative radiotherapy].',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/6328061',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 59,\n",
       "     'text': 'Rationale for intraoperative radiotherapy in glioblastoma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26824195',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 63,\n",
       "     'text': 'Intraoperative cobalt-60 treatment of glioblastoma multiforme.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2852827',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': 'INTRAGO: intraoperative radiotherapy in glioblastoma multiforme—a phase I/II dose escalation study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25535398',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 122,\n",
       "     'text': 'Intraoperative remote afterloading endocurietherapy with high-activity 60cobalt for treatment of glioblastoma multiforme.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3014858',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 122,\n",
       "     'text': 'Combining intraoperative carmustine wafers and Stupp regimen in multimodal first-line treatment of primary glioblastomas.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25724425',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1243,\n",
       "     'offsetInEndSection': 1378,\n",
       "     'text': 'The intraoperative use of carmustine wafers in combination with Stupp regimen is a viable first-line treatment option of glioblastomas.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25724425',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 169,\n",
       "     'text': 'An intraoperative remote afterloading endocurietherapy (ECT) technique with high-activity 60cobalt (60Co) for the treatment of glioblastoma multiforme (GM) is described.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2852827',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 181,\n",
       "     'text': 'The study investigated if intraoperative use of carmustine wafers, particularly in combination with Stupp regimen, is a viable and safe first-line treatment option of glioblastomas.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25724425',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List clinical features of the IMAGe syndrome.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25517553',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25861374',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16835919',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25258553',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22634751',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25541901',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24313804',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24617583',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14760276',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21108398'],\n",
       "   'triples': [{'p': 'http://www.w3.org/2008/05/skos-xl#altLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0683325',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A18567231'},\n",
       "    {'p': 'http://www.w3.org/2000/01/rdf-schema#label',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0683325',\n",
       "     'o': 'clinical aspects'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#altLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0683325',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A18697112'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A18697112',\n",
       "     'o': 'clinical aspects'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#prefLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0683325',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A1810832'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A1810832',\n",
       "     'o': 'clinical aspects'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#altLabel',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/id/C0683325',\n",
       "     'o': 'http://linkedlifedata.com/resource/umls/label/A18585749'},\n",
       "    {'p': 'http://www.w3.org/2008/05/skos-xl#literalForm',\n",
       "     's': 'http://linkedlifedata.com/resource/umls/label/A18585749',\n",
       "     'o': 'aspects clinical'}],\n",
       "   'ideal_answer': ['Clinical features of IMAGe syndrome include intra-uterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita and genital abnormalities. It is s caused by gain-of-function mutations of maternally expressed gene CDKN1C on chromosome 11p15.5.'],\n",
       "   'exact_answer': [['intra-uterine growth restriction'],\n",
       "    ['metaphyseal dysplasia'],\n",
       "    ['adrenal hypoplasia congenita'],\n",
       "    ['genital abnormalities']],\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:0050885'],\n",
       "   'type': 'list',\n",
       "   'id': '5895e4637d9090f353000012',\n",
       "   'snippets': [{'offsetInBeginSection': 171,\n",
       "     'offsetInEndSection': 461,\n",
       "     'text': 'Recently, our group described gain-of-function mutations in the PCNA-binding site of CDKN1C that result in an undergrowth syndrome called IMAGe Syndrome (Intrauterine Growth Restriction, Metaphyseal dysplasia, Adrenal hypoplasia, and Genital anomalies), with life-threatening consequences. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25861374',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 220,\n",
       "     'text': 'IMAGe syndrome is an exceedingly rare condition first described in 1999. Components of the syndrome are intrauterine growth retardation (IUGR), metaphyseal dysplasia, congenital adrenal hypoplasia and genital anomalies. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25541901',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 280,\n",
       "     'text': 'OBJECTIVE: Arboleda et al. have recently shown that IMAGe (intra-uterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita and genital abnormalities) syndrome is caused by gain-of-function mutations of maternally expressed gene CDKN1C on chromosome 11p15.5. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24313804',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 275,\n",
       "     'offsetInEndSection': 493,\n",
       "     'text': 'Intriguing is that CDKN1C gain-of-function variations were recently found in patients with IMAGe syndrome (intrauterine growth restriction, metaphyseal dysplasia, congenital adrenal hypoplasia, and genital anomalies). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25258553',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which proteins control the degradation of cryptic unstable transcripts (CUTs) in yeast?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25989903',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16973436',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25210768',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19854134',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25680078',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24106327',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21826286',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19169244',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21878619',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27492286'],\n",
       "   'ideal_answer': ['Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs). Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene. These results suggest that transcription termination of CUTs directed by Nrd1 and Nab3 is a prerequisite for rapid degradation by the nuclear exosome. ',\n",
       "    'Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs). These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. These transcripts are targeted for degradation immediately after synthesis by the action of the Nrd1-exosome-TRAMP complexes. Cryptic unstable transcripts (CUTs) were recently described as a principal class of RNA polymerase II transcripts in Saccharomyces cerevisiae. ',\n",
       "    \"The exosome and its nuclear specific subunit Rrp6 form a 3'-5' exonuclease complex that regulates diverse aspects of RNA biology including 3' end processing and degradation of a variety of noncoding RNAs (ncRNAs) and unstable transcripts. Known targets of the nuclear exosome include short (<1000 bp) RNAPII transcripts such as small noncoding RNAs (snRNAs), cryptic unstable transcripts (CUTs), and some stable unannotated transcripts (SUTs) that are terminated by an Nrd1, Nab3, and Sen1 (NNS) dependent mechanism. The MTREC complex physically interacts with the nuclear exosome and with various RNA-binding and RNA-processing complexes, coupling RNA processing to the RNA degradation machinery.\",\n",
       "    'Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3 These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene.',\n",
       "    'Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene. These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. These results suggest that transcription termination of CUTs directed by Nrd1 and Nab3 is a prerequisite for rapid degradation by the nuclear exosome. Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs). ',\n",
       "    'Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3',\n",
       "    'Cryptic unstable transcripts (CUTs) were recently described as a principal class of RNA polymerase II transcripts in Saccharomyces cerevisiae. These transcripts are targeted for degradation immediately after synthesis by the action of the Nrd1-exosome and Trf4/5-Air1/2-Mtr4 polyadenylation (TRAMP) complexes. The termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. Known targets of the nuclear exosome include short (<1000 bp) RNAPII transcripts such as small noncoding RNAs (snRNAs), cryptic unstable transcripts (CUTs), and some stable unannotated transcripts (SUTs) that are terminated by an Nrd1, Nab3, and Sen1 (NNS) dependent mechanism. Recent work suggests Nrd1 is necessary for transcriptome surveillance, regulating promoter directionality and suppressing antisense transcription independently of, or prior to, Rrp6 activity.'],\n",
       "   'exact_answer': [['Rrp6'],\n",
       "    ['Nrd1'],\n",
       "    ['Nab3'],\n",
       "    ['TRAMP complex', 'Trf4/5-Air1/2-Mtr4 polyadenylation'],\n",
       "    ['Sen1']],\n",
       "   'type': 'list',\n",
       "   'id': '58adca6d9ef3c34033000007',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 99,\n",
       "     'text': 'Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16973436',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 118,\n",
       "     'offsetInEndSection': 374,\n",
       "     'text': 'These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16973436',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 906,\n",
       "     'offsetInEndSection': 1057,\n",
       "     'text': ' These results suggest that transcription termination of CUTs directed by Nrd1 and Nab3 is a prerequisite for rapid degradation by the nuclear exosome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16973436',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 121,\n",
       "     'offsetInEndSection': 234,\n",
       "     'text': 'Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25210768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1068,\n",
       "     'offsetInEndSection': 1450,\n",
       "     'text': 'Lack of TRAMP activity stabilises ∼ 1600 CUTs in meiotic cells, which occupy 40% of the binding capacity of the nuclear cap binding complex (CBC). CBC mutants display defects in the formation of meiotic double strand breaks (DSBs), and we see similar defects in TRAMP mutants, suggesting that a key function of the nuclear exosome is to prevent saturation of the CBC complex by CUTs',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25210768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 641,\n",
       "     'offsetInEndSection': 895,\n",
       "     'text': 'We find that TRAMP mutants produce high levels of CUTs during meiosis that are undetectable in wild-type cells, showing that the nuclear exosome remains functional for CUT degradation, and we further report that the meiotic exosome complex contains Rrp6.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25210768',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 118,\n",
       "     'text': 'The exosome component Rrp6 is required for RNA polymerase II termination at specific targets of the Nrd1-Nab3 pathway.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25680078',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 239,\n",
       "     'offsetInEndSection': 515,\n",
       "     'text': 'Known targets of the nuclear exosome include short (<1000 bp) RNAPII transcripts such as small noncoding RNAs (snRNAs), cryptic unstable transcripts (CUTs), and some stable unannotated transcripts (SUTs) that are terminated by an Nrd1, Nab3, and Sen1 (NNS) dependent mechanism',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25680078',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 628,\n",
       "     'offsetInEndSection': 819,\n",
       "     'text': 'Recent work suggests Nrd1 is necessary for transcriptome surveillance, regulating promoter directionality and suppressing antisense transcription independently of, or prior to, Rrp6 activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25680078',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 171,\n",
       "     'offsetInEndSection': 439,\n",
       "     'text': 'Cryptic unstable transcripts (CUTs) were recently described as a principal class of RNA polymerase II transcripts in Saccharomyces cerevisiae. These transcripts are targeted for degradation immediately after synthesis by the action of the Nrd1-exosome-TRAMP complexes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19169244',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 279,\n",
       "     'text': 'In Saccharomyces cerevisiae, non-coding RNAs, including cryptic unstable transcripts (CUTs), are subject to degradation by the exosome. The Trf4/5-Air1/2-Mtr4 polyadenylation (TRAMP) complex in S. cerevisiae is a nuclear exosome cofactor that recruits the exosome to degrade RNAs',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21878619',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 432,\n",
       "     'offsetInEndSection': 652,\n",
       "     'text': ' The MTREC complex specifically binds to CUTs, meiotic mRNAs and unspliced pre-mRNA transcripts and targets these RNAs for degradation by the nuclear exosome, while the TRAMP complex has only a minor role in this process',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25989903',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 227,\n",
       "     'text': 'The eukaryotic exosome exoribonuclease Rrp6 forms a complex with Rrp47 that functions in nuclear RNA quality control mechanisms, the degradation of cryptic unstable transcripts (CUTs), and in the 3 end maturation of stable RNAs',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24106327',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 630,\n",
       "     'offsetInEndSection': 795,\n",
       "     'text': 'Nrd1 and Nab3 colocalize to regions of the genome expressing antisense CUTs, and these transcripts require yeast nuclear exosome and TRAMP components for degradation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16973436',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 299,\n",
       "     'text': 'Genome-wide studies have identified abundant small, noncoding RNAs, including small nuclear RNAs, small nucleolar RNAs (snoRNAs), cryptic unstable transcripts (CUTs), and upstream regulatory RNAs (uRNAs), that are transcribed by RNA polymerase II (pol II) and terminated by an Nrd1-dependent pathway',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19854134',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 631,\n",
       "     'offsetInEndSection': 797,\n",
       "     'text': 'Nrd1 and Nab3 colocalize to regions of the genome expressing antisense CUTs, and these transcripts require yeast nuclear exosome and TRAMP components for degradation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16973436',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 511,\n",
       "     'offsetInEndSection': 775,\n",
       "     'text': 'Here, by examining the overlap of stable (SUTs, stable unannotated transcripts) and unstable (CUTs, cryptic unstable transcripts) transcripts with protein-coding genes, we show that the predicted Nrd1 and Nab3-binding site sequences occur at differing frequencies.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27492286',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 367,\n",
       "     'offsetInEndSection': 510,\n",
       "     'text': 'Examples are the consensus binding site sequences of the RNA-binding proteins Nrd1 and Nab3 that target non-coding transcripts for degradation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27492286',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16973436',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 187,\n",
       "     'offsetInEndSection': 350,\n",
       "     'text': 'Some are designated Cryptic Unstable Transcripts (CUTs) because they are terminated by the Nrd1-Nab3-Sen1 pathway and then rapidly degraded by the nuclear exosome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21826286',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'How may CTCF mediate splicing?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/21964334',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24534946',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26154016',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22976956',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24657531',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26929370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25934638',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27083996',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25837375',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26711177'],\n",
       "   'ideal_answer': ['Two different mechanisms convey DNA methylation information into the regulation of alternative splicing. The first involves modulation of the elongation rate of RNA polymerase II (Pol II) by CCCTC-binding factor (CTCF) and methyl-CpG binding protein 2 (MeCP2); the second involves the formation of a protein bridge by heterochromatin protein 1 (HP1) that recruits splicing factors onto transcribed alternative exons.',\n",
       "    'CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing. Intragenic 5-methylcytosine and CTCF mediate opposing effects on pre-mRNA splicing: CTCF promotes inclusion of weak upstream exons through RNA polymerase II pausing, whereas 5-methylcytosine evicts CTCF, leading to exon exclusion.',\n",
       "    'CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing.Here we provide the first evidence that a DNA-binding protein, CCCTC-binding factor (CTCF), can promote inclusion of weak upstream exons by mediating local RNA polymerase II pausing both in a mammalian model system for alternative splicing, CD45, and genome-wideCTCF also functions in RNA polymerase II regulation and in doing so can influence co-transcriptional splicing events.CTCF is ubiquitously expressed and plays diverse roles in gene regulation, imprinting, insulation, intra/interchromosomal interactions, nuclear compartmentalisation, and alternative splicing.In particular, the CCCTC-binding factor (CTCF), the Argonaute protein AGO1, and members of the heterochromatin protein 1 (HP1) family have been implicated in the regulation of splicing associated with chromatin and the elongation of RNAPII.Furthermore, we found mutations in the splicing factor SFPQ and in the nonclassic regulators of mRNA processing CTCF and RAD21.However, there was no correlation between the KTS+/KTS- splicing variants of WT1 and the methylation status of the CpGs of the CTCF binding site.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5a43a933966455904c00000b',\n",
       "   'snippets': [{'offsetInBeginSection': 328,\n",
       "     'offsetInEndSection': 744,\n",
       "     'text': 'Two different mechanisms convey DNA methylation information into the regulation of alternative splicing. The first involves modulation of the elongation rate of RNA polymerase II (Pol II) by CCCTC-binding factor (CTCF) and methyl-CpG binding protein 2 (MeCP2); the second involves the formation of a protein bridge by heterochromatin protein 1 (HP1) that recruits splicing factors onto transcribed alternative exons.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25837375',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 957,\n",
       "     'offsetInEndSection': 1084,\n",
       "     'text': 'Furthermore, we found mutations in the splicing factor SFPQ and in the nonclassic regulators of mRNA processing CTCF and RAD21.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22976956',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1094,\n",
       "     'offsetInEndSection': 1257,\n",
       "     'text': ' We show that CTCF bound to HTLV-1 acts as an enhancer blocker, regulates HTLV-1 mRNA splicing, and forms long-distance interactions with flanking host chromatin. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26929370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1290,\n",
       "     'offsetInEndSection': 1573,\n",
       "     'text': ' Many CTCF sites are also involved in other functions such as modulation of RNA splicing and specific regulation of gene expression, and the relationship between these activities and loop formation is another unanswered question that should keep investigators occupied for some time.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27083996',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'TET-catalyzed oxidation of intragenic 5-methylcytosine regulates CTCF-dependent alternative splicing.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26711177',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 230,\n",
       "     'text': 'Intragenic 5-methylcytosine and CTCF mediate opposing effects on pre-mRNA splicing: CTCF promotes inclusion of weak upstream exons through RNA polymerase II pausing, whereas 5-methylcytosine evicts CTCF, leading to exon exclusion.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26711177',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 325,\n",
       "     'offsetInEndSection': 515,\n",
       "     'text': 'Here, we reveal the methylcytosine dioxygenases TET1 and TET2 as active regulators of CTCF-mediated alternative splicing through conversion of 5-methylcytosine to its oxidation derivatives. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26711177',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 106,\n",
       "     'text': 'A chromatin code for alternative splicing involving a putative association between CTCF and HP1α proteins.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25934638',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 150,\n",
       "     'offsetInEndSection': 534,\n",
       "     'text': 'Recent experiments have suggested a new complex network of splicing regulation involving chromatin, transcription and multiple protein factors. In particular, the CCCTC-binding factor (CTCF), the Argonaute protein AGO1, and members of the heterochromatin protein 1 (HP1) family have been implicated in the regulation of splicing associated with chromatin and the elongation of RNAPII.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25934638',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1716,\n",
       "     'offsetInEndSection': 2017,\n",
       "     'text': 'Our results show that a considerable number of alternative splicing events could have a chromatin-dependent regulation involving the association of HP1α and CTCF near regulated exons. Additionally, we find further evidence for the involvement of HP1α and AGO1 in chromatin-related splicing regulation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25934638',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 591,\n",
       "     'offsetInEndSection': 709,\n",
       "     'text': 'CTCF also functions in RNA polymerase II regulation and in doing so can influence co-transcriptional splicing events. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26154016',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 60,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': ' CTCF is ubiquitously expressed and plays diverse roles in gene regulation, imprinting, insulation, intra/interchromosomal interactions, nuclear compartmentalisation, and alternative splicing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24657531',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 778,\n",
       "     'offsetInEndSection': 923,\n",
       "     'text': 'However, there was no correlation between the KTS+/KTS- splicing variants of WT1 and the methylation status of the CpGs of the CTCF binding site.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24534946',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 74,\n",
       "     'text': 'CTCF-promoted RNA polymerase II pausing links DNA methylation to splicing.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21964334',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 565,\n",
       "     'offsetInEndSection': 827,\n",
       "     'text': 'Here we provide the first evidence that a DNA-binding protein, CCCTC-binding factor (CTCF), can promote inclusion of weak upstream exons by mediating local RNA polymerase II pausing both in a mammalian model system for alternative splicing, CD45, and genome-wide',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21964334',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is Kummell’s disease an avascular necrosis of the vertebral body?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22220246',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19534241',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23814399',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19124637',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19949820',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19321060',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21811863',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28913640',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11915453',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28160398'],\n",
       "   'ideal_answer': ['Yes, Kummell’s disease is an avascular necrosis of the vertebral body.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['https://meshb.nlm.nih.gov/record/ui?ui=D010020',\n",
       "    'http://www.disease-ontology.org/api/metadata/DOID:6603'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5a787544faa1ab7d2e00000b',\n",
       "   'snippets': [{'offsetInBeginSection': 13,\n",
       "     'offsetInEndSection': 326,\n",
       "     'text': \" Kummell's disease is an avascular necrosis of the vertebral body, secondary to a vertebral compression fracture. This entity is characterised by the gradual development in time of a vertebral body collapse following a trivial spinal trauma, involving a worsening back pain associated with a progressive kyphosis.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28913640',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': \"Kummell's disease is a rare spinal disorder characterized as avascular necrosis of a vertebral body occurring in a delayed fashion after minor trauma.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11915453',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 132,\n",
       "     'text': \"Kummell's disease is a spinal disorder characterized by delayed post-traumatic collapse of a vertebral body with avascular necrosis.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22220246',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 265,\n",
       "     'text': 'Kummell disease, or avascular necrosis of a vertebral body, presents as vertebral osteonecrosis typically affecting a thoracic vertebra with compression deformity, intravertebral vacuum cleft, and exaggerated kyphosis weeks to months after a minor traumatic injury.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23814399',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 125,\n",
       "     'text': \"INTRODUCTION Kummell's disease is an avascular necrosis of the vertebral body, secondary to a vertebral compression fracture.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28913640',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 113,\n",
       "     'text': 'Kummel disease is the eponym for avascular necrosis of the vertebral body after a vertebral compression fracture.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19124637',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'kummell s disease delayed post traumatic osteonecrosis of the vertebral body',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19949820',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 99,\n",
       "     'text': \"Kummell's disease, caused by osteonecrosis of the vertebral body, is a cause of vertebral collapse.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28160398',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 91,\n",
       "     'offsetInEndSection': 152,\n",
       "     'text': \"Kummell's disease is a post-traumatic vertebral body collapse\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19534241',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 168,\n",
       "     'text': \"Kummell's disease is a rare, delayed posttraumatic collapse of a vertebral body that can occur several months or even years after an osteoporotic compression fracture. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21811863',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 236,\n",
       "     'text': \"Avascular necrosis of a vertebral body, a relatively uncommon entity, is caused by malignancy, infection, radiation, systemic steroid treatment, trauma, and the like.1 Vertebral osteonecrosis induced by trauma is called Kvmell's disease\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19321060',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can a circRNA be translated into protein?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28903484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27617908',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27255916',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26649774',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27612318',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29028266',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26874353',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28344082',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28344080',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27892769'],\n",
       "   'ideal_answer': ['Circ-ZNF609 is associated with heavy polysomes, and it is translated into a protein in a splicing-dependent and cap-independent manner, providing an example of a protein-coding circRNA in eukaryotes.\\nCircular RNAs (circRNAs) represent a large class of noncoding RNAs (ncRNAs) that have recently emerged as regulators of gene expression\\t\\nHowever, whether circRNAs encode functional proteins remains elusive, although translation of several circRNAs was recently reported',\n",
       "    'Μost circRNAs were not associated with translating ribosomes, therefore, circRNAs were deemed to be noncoding. However, the latest research findings revealed that some circRNAs could generate proteins in vivo, which expands the landscape of transcriptome and proteome'],\n",
       "   'type': 'summary',\n",
       "   'id': '5a895a96fcd1d6a10c000003',\n",
       "   'snippets': [{'offsetInBeginSection': 224,\n",
       "     'offsetInEndSection': 356,\n",
       "     'text': 'However, whether circRNAs encode functional proteins remains elusive, although translation of several circRNAs was recently reported',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28903484',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'Circular RNAs (circRNAs) are long, non-coding RNAs that result from the non-canonical splicing of linear pre-mRNAs. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26649774',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 148,\n",
       "     'text': 'Circular RNAs (circRNAs) are a large type of noncoding RNAs characterized by their circular shape resulting from covalently closed continuous loops.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26874353',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 585,\n",
       "     'offsetInEndSection': 720,\n",
       "     'text': 'They are known to serve as microRNA (miRNA) sponges, regulators of alternative splicing, transcription factors and encode for proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26874353',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 215,\n",
       "     'offsetInEndSection': 306,\n",
       "     'text': 'Covalently closed loop structures elevate the stability of this new type of noncoding RNA. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27255916',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': 'Circular RNAs (circRNAs) are a novel class of non-coding RNA that assumes a covalently closed continuous conformation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27617908',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': 'Circular RNAs (circRNAs) represent a large class of noncoding RNAs (ncRNAs) that have recently emerged as regulators of gene expression',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27612318',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Circular RNAs (circRNAs) are highly abundant and evolutionarily conserved non-coding RNAs produced by circularization of specific exons. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27892769',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 81,\n",
       "     'text': 'Circ-ZNF609 Is a Circular RNA that Can Be Translated and Functions in Myogenesis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28344082',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 906,\n",
       "     'offsetInEndSection': 1106,\n",
       "     'text': ' Circ-ZNF609 is associated with heavy polysomes, and it is translated into a protein in a splicing-dependent and cap-independent manner, providing an example of a protein-coding circRNA in eukaryotes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28344082',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 856,\n",
       "     'offsetInEndSection': 996,\n",
       "     'text': 'Altogether, our study provides strong evidence for translation of circRNAs, revealing the existence of an unexplored layer of gene activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28344080',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 364,\n",
       "     'offsetInEndSection': 834,\n",
       "     'text': 'In addition, most circRNAs were not associated with translating ribosomes, therefore, circRNAs were deemed to be noncoding. However, the latest research findings revealed that some circRNAs could generate proteins in vivo, which expands the landscape of transcriptome and proteome. To gain insights into the new area of circRNA translation, we introduce an integrated tool capable of detecting circRNAs with protein-coding potential from high-throughput sequencing data.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29028266',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What protein is the most common cause of hereditary  renal amyloidosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29142973',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19073821',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19013120',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8097946',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26299174',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15523923',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9389696',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23551149',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25331409',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8113408'],\n",
       "   'ideal_answer': ['The most common cause of hereditary  renal amyloidosis is over expression of a mutant form of the Fibrinogen A Alpha protein'],\n",
       "   'exact_answer': ['Fibrinogen A Alpha protein'],\n",
       "   'concepts': ['https://meshb.nlm.nih.gov/record/ui?ui=D028226',\n",
       "    'https://meshb.nlm.nih.gov/record/ui?ui=D000686'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5a6a02a3b750ff4455000021',\n",
       "   'snippets': [{'offsetInBeginSection': 15,\n",
       "     'offsetInEndSection': 227,\n",
       "     'text': 'ibrinogen A alpha chain amyloidosis is an autosomal dominant disease associated with mutations in the fibrinogen A alpha chain (FGA) gene, and it is the most common cause of hereditary renal amyloidosis in the UK',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29142973',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 117,\n",
       "     'text': 'Hereditary renal amyloidosis is an autosomal dominant condition with considerable overlap with other amyloidosis type',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26299174',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 427,\n",
       "     'offsetInEndSection': 605,\n",
       "     'text': 'We suspected amyloidosis with fibrinogen A alpha chain deposits, which is the most frequent cause of hereditary amyloidosis in Europe, with a glomerular preferential affectation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26299174',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 147,\n",
       "     'text': 'Three members of a family who died with renal amyloidosis were found to share a single nucleotide substitution in the fibrinogen alpha-chain gene. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8097946',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': 'A French kindred with autosomal dominant hereditary renal amyloidosis was found to have a novel mutation in the fibrinogen Aalpha-chain gene.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9389696',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 120,\n",
       "     'text': 'Two families with hereditary renal amyloidosis were found to have a novel mutation in the fibrinogen A alpha chain gene.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8113408',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 148,\n",
       "     'text': 'The predominant cause of hereditary renal amyloidosis is a mutation of the fibrinogen Aalpha chain (AFib), the most common being the E526V mutation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19013120',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1108,\n",
       "     'offsetInEndSection': 1295,\n",
       "     'text': 'Discovery of this new mutation confirms the association between fibrinogen A alpha chain variant and hereditary renal amyloidosis and establishes a new biochemical subtype of amyloidosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8113408',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'Hereditary renal amyloidosis associated with a mutant fibrinogen alpha-chain.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8097946',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Mutations in the fibrinogen Aα-chain genes are the most common cause of hereditary renal amyloidosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25331409',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 103,\n",
       "     'text': 'Mutant fibrinogen A-alpha-chain associated with hereditary renal amyloidosis and peripheral neuropathy.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15523923',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'Mutations in the fibrinogen A alpha-chain gene are the most common cause of hereditary renal amyloidosis in the United Kingdom.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19073821',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 227,\n",
       "     'text': 'Introduction Fibrinogen A alpha chain amyloidosis is an autosomal dominant disease associated with mutations in the fibrinogen A alpha chain (FGA) gene, and it is the most common cause of hereditary renal amyloidosis in the UK.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29142973',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 102,\n",
       "     'text': 'Mutations in the fibrinogen Aα-chain genes are the most common cause of hereditary renal amyloidosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25331409',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 144,\n",
       "     'text': 'Hereditary amyloidosis with predominant renal disease can be caused by mutations in the gene encoding the fibrinogen Aα-chain (AFib)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23551149',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is cilengitide effective for treatment of glioblastoma?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/26918452',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28514722',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24442484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25163906',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28259301',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26792571',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28643756',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26717039',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27296952',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26935578'],\n",
       "   'ideal_answer': ['No, cilengitide does not improve survival of glioblastoma (GBM) patients. Cilengitide is a cyclic pentapeptide that demonstrated efficacy for GBM treatment by targeting the integrins avβ3 and avβ5 over-expressed on GBM cells. However, randomized phase III CENTRIC and phase II CORE trials explored failed to meet their primary endpoints. However, in CORE, higher αvβ3 levels in tumor cells were associated with improved progression-free survival by central review and with improved overall survival in patients treated with cilengitide. Analysis of randomized clinical trials of antiangiogenic drugs (including cilengitide) showed no improvement in overall survival and a trend for an inferior outcome, in terms of overall survival, in patients receiving antiangiogenic drug alone compared to cytotoxic drug alone.'],\n",
       "   'exact_answer': 'no',\n",
       "   'concepts': ['https://meshb.nlm.nih.gov/record/ui?ui=D005909',\n",
       "    'https://meshb.nlm.nih.gov/record/ui?ui=D016896',\n",
       "    'http://www.biosemantics.org/jochem#4242009'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5a76080683b0d9ea66000015',\n",
       "   'snippets': [{'offsetInBeginSection': 542,\n",
       "     'offsetInEndSection': 1013,\n",
       "     'text': 'RESULTS: fourteen randomized clinical trials were identified (7 with bevacizumab, 2 cilengitide, 1 enzastaurin, 1 dasatinib, 1 vandetanib, 1 temsirolimus, 1 cediranib) including 4330 patients. Antiangiogenic drugs showed no improvement in overall survival with a pooled HR of 1.00, a trend for an inferior outcome, in terms of overall survival, was observed in the group of patients receiving antiangiogenic drug alone compared to cytotoxic drug alone (HR=1.24, p=0.056).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28259301',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1713,\n",
       "     'offsetInEndSection': 1878,\n",
       "     'text': 'However, we could not conclusively confirm whether cilengitide 2000mg/5/week was the optimum regime, as only one trial using this protocol was included in our study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28514722',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 97,\n",
       "     'offsetInEndSection': 254,\n",
       "     'text': 'Cilengitide (CGT) is a cyclic pentapeptide that demonstrated efficacy for GBM treatment by targeting the integrins avβ3 and avβ5 over-expressed on GBM cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26792571',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 752,\n",
       "     'offsetInEndSection': 842,\n",
       "     'text': 'Cilengitide is recently failed in Phase III CENTRIC trial in unselected patients with GBM.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26717039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1188,\n",
       "     'offsetInEndSection': 1432,\n",
       "     'text': 'In conclusion, we demonstrate that EGFRvIII/integrin β3 complexes promote GBM progression and metastasis in the environment of hypoxia and vitronectin-enrichment, and cilengitide may serve as a promising therapeutics for EGFRvIII-positive GBMs.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26717039',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1242,\n",
       "     'offsetInEndSection': 1442,\n",
       "     'text': 'he addition of molecularly targeted drugs to TEM\\u2009+\\u2009RAD did not improve the OS of patients with GBM; however, it did improve PFS in patients treated by cilengitide who could not get improvement in OS. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27296952',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 161,\n",
       "     'offsetInEndSection': 449,\n",
       "     'text': ' The randomized phase III CENTRIC and phase II CORE trials explored the integrin inhibitor cilengitide in patients with newly diagnosed glioblastoma with versus without O6-methylguanine DNA methyltransferase (MGMT) promoter methylation. These trials failed to meet their primary endpoints',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26918452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 915,\n",
       "     'offsetInEndSection': 1106,\n",
       "     'text': '. In CORE, higher αvβ3 levels in tumor cells were associated with improved progression-free survival by central review and with improved overall survival in patients treated with cilengitide.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26918452',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1253,\n",
       "     'offsetInEndSection': 1461,\n",
       "     'text': 'Cilengitide combined with metronomic temozolomide and procarbazine in MGMT-promoter unmethylated glioblastoma did not improve survival compared with historical data and does not warrant further investigation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26935578',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2477,\n",
       "     'offsetInEndSection': 2629,\n",
       "     'text': 'The addition of cilengitide to temozolomide chemoradiotherapy did not improve outcomes; cilengitide will not be further developed as an anticancer drug.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25163906',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 103,\n",
       "     'text': 'Cilengitide treatment of newly diagnosed glioblastoma patients does not alter patterns of progression.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24442484',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1525,\n",
       "     'offsetInEndSection': 1702,\n",
       "     'text': 'It may be proposed that the combination therapy of NG2 suppression and cilengitide treatment showed no considerable effect on glioblastoma compared to cilengitide therapy alone.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28643756',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Describe King–Kopetzky syndrome.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/14570238',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10997453',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20500033',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20380613',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14558893',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8738633',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10380733',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11603771',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16562562',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8985564'],\n",
       "   'ideal_answer': ['The principal symptom of subjects suffering from King-Kopetzky syndrome (Obscure Auditory Dysfunction) is perceived difficulty in recognizing and understanding speech in noisy backgrounds. For some patients, minor disturbances in auditory function, e.g. a deteriorated signal-to-noise ratio for speech, can be demonstrated; for others, all measurements of hearing are normal.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5a7602be83b0d9ea6600000f',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 265,\n",
       "     'text': 'At least 10% of people who present for help with hearing difficulties will be found to have normal hearing thresholds. These cases are clinically categorized as King-Kopetzky syndrome (KKS), obscure auditory dysfunction (OAD), or auditory processing disorder (APD).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20380613',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Illness perceptions and hearing difficulties in King-Kopetzky syndrome: what determines help seeking?',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20500033',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 328,\n",
       "     'text': 'The present study explored illness perceptions of hearing difficulties amongst one hundred participants who reported experiencing hearing difficulties despite normal audiometric thresholds. This experience is referred to as King-Kopetzky syndrome (KKS), obscure auditory dysfunction (OAD), or auditory processing disorder (APD).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20500033',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 922,\n",
       "     'offsetInEndSection': 1155,\n",
       "     'text': 'Decreases in DPOAE level appear to represent evidence of minor cochlear pathology, and provide a pathological basis for the difficulty of hearing speech in the presence of background noise, which characterizes King-Kopetzky syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16562562',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 347,\n",
       "     'text': 'The principal symptom of subjects suffering from King-Kopetzky syndrome is a perceived difficulty in recognizing and understanding speech in noisy backgrounds. For some patients, minor disturbances in auditory function, e.g. a deteriorated signal-to-noise ratio for speech, can be demonstrated; for others, all measurements of hearing are normal. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14570238',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'King Kopetzky Syndrome (KKS) is a common condition in which individuals with normal audiograms complain of hearing difficulties, particularly in noisy places.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11603771',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 317,\n",
       "     'text': 'King-Kopetzky syndrome (Obscure Auditory Dysfunction, OAD) has been recognized as a clinically distinct condition in audiological and ENT clinics. It is characterized by normal hearing thresholds on pure tone audiometry (PTA) but complaints of difficulties in understanding speech in the presence of background noise.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10997453',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 191,\n",
       "     'text': 'King-Kopetzky syndrome is characterized by auditory disability with a clinically normal hearing threshold. The main reported disability is hearing speech in the presence of background noise. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8738633',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 90,\n",
       "     'text': 'Hearing complaints of patients with King-Kopetzky syndrome (obscure auditory dysfunction).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8985564',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 132,\n",
       "     'offsetInEndSection': 386,\n",
       "     'text': \"The main findings are that complaints were commonly focused on the categories of 'live speech' and 'electronic speech' difficulties in patients with King-Kopetzky syndrome, particularly the auditory difficulties of speech in noise and group conversation.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8985564',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 292,\n",
       "     'text': 'The condition in which individuals with normal pure tone audiograms complain of hearing difficulties, especially in the presence of background noise, (normal pure tone audiograms), has had a number of different names. The present term King-Kopetzky Syndrome was coined by Hinchcliffe in 1992.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14558893',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 159,\n",
       "     'text': 'The principal symptom of subjects suffering from King-Kopetzky syndrome is a perceived difficulty in recognizing and understanding speech in noisy backgrounds.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14570238',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 922,\n",
       "     'offsetInEndSection': 1159,\n",
       "     'text': 'Decreases in DPOAE level appear to represent evidence of minor cochlear pathology, and provide a pathological basis for the difficulty of hearing speech in the presence of background noise, which characterizes King-Kopetzky syndrome.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16562562',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 131,\n",
       "     'offsetInEndSection': 386,\n",
       "     'text': \"The main findings are that complaints were commonly focused on the categories of 'live speech' and 'electronic speech' difficulties in patients with King-Kopetzky syndrome, particularly the auditory difficulties of speech in noise and group conversation.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8985564',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 621,\n",
       "     'text': 'The condition in which individuals with normal pure tone audiograms complain of hearing difficulties, especially in the presence of background noise, (normal pure tone audiograms), has had a number of different names. The present term King-Kopetzky Syndrome was coined by Hinchcliffe in 1992. This is a common condition reported in 5 - 10% of those attending clinics complaining of hearing problems. A dominant genetic aetiology has been found in a proportion of cases. It may be associated with minor peripheral or central auditory dysfunction, and frequently the individuals exhibit anxious or depressive personalities.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14558893',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 346,\n",
       "     'text': 'The principal symptom of subjects suffering from King-Kopetzky syndrome is a perceived difficulty in recognizing and understanding speech in noisy backgrounds. For some patients, minor disturbances in auditory function, e.g. a deteriorated signal-to-noise ratio for speech, can be demonstrated; for others, all measurements of hearing are normal.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14570238',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can canagliflozin cause euglycemic diabetic ketoacidosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27375734',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28634592',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27088018',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28090050',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27928503',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29039237',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29099159',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27717592',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27471597',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26912914'],\n",
       "   'ideal_answer': ['Yes, canagliflozin use can cause euglycemic diabetic ketoacidosis. In May 2015, the FDA issued a warning of ketoacidosis with use of sodium-glucose cotransporter-2 (SGLT-2) drug class.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['https://meshb.nlm.nih.gov/record/ui?ui=D000068896'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5a6f922fb750ff4455000059',\n",
       "   'snippets': [{'offsetInBeginSection': 408,\n",
       "     'offsetInEndSection': 1464,\n",
       "     'text': 'CASE REPORT: We present a case of a 57-year-old woman with type 2 diabetes mellitus taking a combination of canagliflozin and metformin who presented with progressive altered mental status over the previous 2\\xa0days. Her work-up demonstrated a metabolic acidosis with an anion gap of 38 and a venous serum pH of 7.08. The serum glucose was 168\\xa0mg/dL. The urinalysis showed glucose>500\\xa0mg/dL and ketones of 80\\xa0mg/dL. Further evaluation demonstrated an elevated serum osmolality of 319 mOsm/kg and an acetone concentration of 93\\xa0mg/dL. She was treated with intravenous insulin and fluids, and the metabolic abnormalities and her altered mental status resolved within 36\\xa0h. This was the first episode of diabetic ketoacidosis (DKA) for this patient. WHY SHOULD AN EMERGENCY PHYSICIAN BE AWARE OF THIS?: Diabetic patients on SGLT2 inhibitor medications are at risk for ketoacidosis. Due to the renal glucose-wasting properties of these drugs, they may present with ketoacidosis with only mild elevations in serum glucose, potentially complicating the diagnosis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27717592',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 85,\n",
       "     'text': 'Euglycemic Diabetic Ketoacidosis with Persistent Diuresis Treated with Canagliflozin.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28090050',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 175,\n",
       "     'offsetInEndSection': 442,\n",
       "     'text': 'We herein report the case of a 27-year-old Asian woman with type 2 diabetes who was treated with a sodium-glucose cotransporter 2 (SGLT2) inhibitor (canagliflozin) who developed euglycemic diabetic ketoacidosis and persistent diuresis in the absence of hyperglycemia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28090050',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 664,\n",
       "     'offsetInEndSection': 930,\n",
       "     'text': 'Canagliflozin raised the risk of amputations and the rate of fractures in the CANVAS trial, although more data are necessary before drawing definite conclusions. The risk of euglycemic diabetic ketoacidosis seems to be minimal when the drugs are prescribed properly.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29039237',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 79,\n",
       "     'text': 'Severe Ketoacidosis Associated with Canagliflozin (Invokana): A Safety Concern.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27088018',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 332,\n",
       "     'offsetInEndSection': 479,\n",
       "     'text': ' However, some serious side effects, including severe anion gap metabolic acidosis and euglycemic diabetic ketoacidosis (DKA), have been reported. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27088018',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 104,\n",
       "     'offsetInEndSection': 380,\n",
       "     'text': 'At present, the Food and Drug Administration (FDA) has only approved three medications (canagliflozin, dapagliflozin and empagliflozin) in this drug class for the management of Type 2 diabetes. In May 2015, the FDA issued a warning of ketoacidosis with use of this drug class.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27471597',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 554,\n",
       "     'offsetInEndSection': 664,\n",
       "     'text': 'We present a case of euglycemic diabetic ketoacidosis secondary to canagliflozin in a type 2 diabetic patient.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27928503',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 302,\n",
       "     'text': 'Nonconvulsive Status Epilepticus in Elderly Patients Receiving SSRIs; Euglycemic Diabetic Ketoacidosis Associated with Canagliflozin Use in a Type 1 Diabetic Patient; Duloxetine-Induced Galactorrhea; Canagliflozin-Associated Severe Hypercalcemia and Hypernatremia; Vemurafenib-Induced Fanconi Syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26912914',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 162,\n",
       "     'text': 'Euglycemic Diabetic Ketoacidosis in a 27 year-old female patient with type-1-Diabetes treated with sodium-glucose cotransporter-2 (SGLT2) inhibitor Canagliflozin.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27375734',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 182,\n",
       "     'text': 'We are reporting a timely case of atypical euglycemic diabetic ketoacidosis in a type 1 diabetic patient treated with sodium-glucose cotransporter-2 (SGLT-2) inhibitor canagliflozin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27375734',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 508,\n",
       "     'offsetInEndSection': 596,\n",
       "     'text': 'Euglycemic ketoacidosis did not recur in our patient after discontinuing canagliflozin. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27375734',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 124,\n",
       "     'text': 'Euglycemic Diabetic Ketoacidosis With Prolonged Glucosuria Associated With the Sodium-Glucose Cotransporter-2 Canagliflozin.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28634592',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 822,\n",
       "     'offsetInEndSection': 975,\n",
       "     'text': 'In this article, we present a case of a 50-year-old woman with type 2 diabetes who developed euglycemic DKA after initiating therapy with canagliflozin. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28634592',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1103,\n",
       "     'offsetInEndSection': 1233,\n",
       "     'text': 'SGLT2 inhibitors such as canagliflozin may predispose patients not only to diabetic ketoacidosis but also to prolonged glucosuria.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28634592',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 554,\n",
       "     'offsetInEndSection': 668,\n",
       "     'text': 'We present a case of euglycemic diabetic ketoacidosis secondary to canagliflozin in a type 2 diabetic patient.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27928503',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 558,\n",
       "     'offsetInEndSection': 668,\n",
       "     'text': 'We present a case of euglycemic diabetic ketoacidosis secondary to canagliflozin in a type 2 diabetic patient.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27928503',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 561,\n",
       "     'offsetInEndSection': 660,\n",
       "     'text': 'CONCLUSION Treatment with canagliflozin was associated with development of euglycemic ketoacidosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29099159',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 86,\n",
       "     'text': 'Euglycemic Diabetic Ketoacidosis with Persistent Diuresis Treated with Canagliflozin.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28090050',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 554,\n",
       "     'offsetInEndSection': 665,\n",
       "     'text': 'We present a case of euglycemic diabetic ketoacidosis secondary to canagliflozin in a type 2 diabetic patient..',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27928503',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 125,\n",
       "     'text': 'Euglycemic Diabetic Ketoacidosis With Prolonged Glucosuria Associated With the Sodium-Glucose Cotransporter-2 Canagliflozin.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28634592',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 163,\n",
       "     'text': 'Euglycemic Diabetic Ketoacidosis in a 27 year-old female patient with type-1-Diabetes treated with sodium-glucose cotransporter-2 (SGLT2) inhibitor Canagliflozin.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27375734',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'What are check point inhibitors?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24485523',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27403706',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28363334',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26942589',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27528363',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28878676',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26321371',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28351171',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27654928',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28865357'],\n",
       "   'ideal_answer': ['Immune checkpoint blocking monoclonal antibodies are heralded as a promising therapeutic approach in clinical oncology. These mAbs do not directly attack the malignant cells as most anticancer mAbs; rather, they enhance the anti-tumor response of the immune system by targeting immune regulatory pathways.'],\n",
       "   'exact_answer': ['monoclonal antibody drugs that regulate the immune response to cancer'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5a8718c861bb38fb24000008',\n",
       "   'snippets': [{'offsetInBeginSection': 489,\n",
       "     'offsetInEndSection': 795,\n",
       "     'text': ' Immune checkpoint blocking monoclonal antibodies are heralded as a promising therapeutic approach in clinical oncology. These mAbs do not directly attack the malignant cells as most anticancer mAbs; rather, they enhance the anti-tumor response of the immune system by targeting immune regulatory pathways.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28363334',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 179,\n",
       "     'offsetInEndSection': 550,\n",
       "     'text': ' These next generation agents with novel mechanisms of action as well as targeted therapies are being used both in the preclinical and clinical settings for patients with myeloma. These agents include monoclonal antibodies, deacetylase inhibitors, kinase inhibitors, agents affecting various signaling pathways, immune check point inhibitors, and other targeted therapies',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27654928',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 844,\n",
       "     'offsetInEndSection': 972,\n",
       "     'text': 'Immune check point inhibitors may be of critical importance for the design of future immunotherapy approaches in GBM management.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27528363',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 984,\n",
       "     'offsetInEndSection': 1071,\n",
       "     'text': 'Immune check-point inhibitors should be considered a promising treatment option in GBM.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27528363',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 232,\n",
       "     'text': 'Immune check-point inhibitors are now employed as single-agents in current practice for the treatment of advanced non-small cell lung cancer (NSCLC), while combinations of different inhibitors are being evaluated in clinical trials.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28351171',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 236,\n",
       "     'offsetInEndSection': 423,\n",
       "     'text': 'Check point antibody inhibitors, such as anti-PD-1/PD-L1, are a novel class of inhibitors that function as a tumor suppressing factor via modulation of immune cell-tumor cell interaction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28878676',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 188,\n",
       "     'text': 'The recent successes of immune check point targeting therapies in treating cancer patients has driven a resurgence of interest in targeting these pathways in chronically infected patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28865357',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 424,\n",
       "     'offsetInEndSection': 587,\n",
       "     'text': 'These checkpoint blockers are rapidly becoming a highly promising cancer therapeutic approach that yields remarkable antitumor responses with limited side effects.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28878676',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 68,\n",
       "     'offsetInEndSection': 235,\n",
       "     'text': 'The survival of tumors in several cases is assisted by checkpoint immunomodulation to maintain the imbalance between immune surveillance and cancer cell proliferation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28878676',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1269,\n",
       "     'offsetInEndSection': 1391,\n",
       "     'text': 'Check point inhibitors have promising activity in several solid tumors and have demonstrated a favorable toxicity profile.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26942589',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 14,\n",
       "     'offsetInEndSection': 320,\n",
       "     'text': 'The treatment of melanoma is evolving rapidly over the past few years. Patients with BRAFv600 mutations can be treated with a combination of a BRAF-inhibitor and an MEK-inhibitor. Patients with BRAF wild-type tumors and BRAFv600 mutated tumors can be treated with immunotherapy i.e. check point inhibitors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27403706',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 359,\n",
       "     'text': 'Till now, the prognosis of advanced gastric cancer looked dreadful; thus the search for newer better approaches for this lethal disease has been a strategic target for cancer researchers. In recent years, important immunobiological aspects of the tumor have been revealed with the subsequent proposal of immune check point inhibitors to target these pathways.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26321371',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 486,\n",
       "     'text': 'Inhibitory receptors on immune system cells respond to membrane-bound and soluble ligands to abort or mitigate the intensity of immune responses by raising thresholds of activation, halting proliferation, favoring apoptosis or inhibiting/deviating effector function differentiation. Such evolutionarily selected inhibitory mechanisms are termed check-points and therefore check-point inhibitors empower any ongoing anti-cancer immune response that might have been too weak or exhausted.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24485523',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Does wheat belongs to the genus Avena, yes or no?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29146257',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10984842',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27079356',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11908651',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11536867',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24226111',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26618715',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28132155',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28132141',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20658121'],\n",
       "   'ideal_answer': ['oat seedlings (Avena sativa)',\n",
       "    'Wheat belongs to the species Triticum not to the genus Avena.',\n",
       "    'oat seedlings (Avena sativa). '],\n",
       "   'exact_answer': 'no',\n",
       "   'type': 'yesno',\n",
       "   'id': '5a79d25dfaa1ab7d2e00000f',\n",
       "   'snippets': [{'offsetInBeginSection': 84,\n",
       "     'offsetInEndSection': 112,\n",
       "     'text': 'oat seedlings (Avena sativa)',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29146257',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 19,\n",
       "     'offsetInEndSection': 49,\n",
       "     'text': 'wild green-oat (Avena sativa) ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26618715',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 74,\n",
       "     'offsetInEndSection': 96,\n",
       "     'text': ' Oat (Avena sativa L.)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28132141',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 38,\n",
       "     'offsetInEndSection': 59,\n",
       "     'text': 'Oat (Avena sativa L.)',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28132155',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 46,\n",
       "     'offsetInEndSection': 58,\n",
       "     'text': 'Avena (Oats)',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24226111',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 116,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'wild oats (Avena fatua L.)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27079356',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 41,\n",
       "     'offsetInEndSection': 59,\n",
       "     'text': 'oats (genus Avena)',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10984842',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 46,\n",
       "     'offsetInEndSection': 88,\n",
       "     'text': 'Avena sativa L. and A. byzantina C. Koch) ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20658121',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 81,\n",
       "     'offsetInEndSection': 104,\n",
       "     'text': ' oat (Avena sativa L.).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11908651',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 16,\n",
       "     'offsetInEndSection': 66,\n",
       "     'text': 'oat (Avena sativa L.) and wheat (Triticum aestivum',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11536867',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which disease can be diagnosed with the \"probe to bone\" test?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28440774',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20622159',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27369321',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17259493',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26804367',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18285592',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26009740',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24857262',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23001730'],\n",
       "   'ideal_answer': ['Probe-to-bone test is used for the diagnosis of diabetic foot osteomyelitis. The test has good sensitivity and specificity. Other diagnostic tests of diabetic foot osteomyelitis are plain films and magnetic resonance imaging.'],\n",
       "   'exact_answer': ['diabetic foot osteomyelitis'],\n",
       "   'concepts': ['https://meshb.nlm.nih.gov/record/ui?ui=D003933',\n",
       "    'https://meshb.nlm.nih.gov/record/ui?ui=D004194'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5a6f853ab750ff4455000055',\n",
       "   'snippets': [{'offsetInBeginSection': 552,\n",
       "     'offsetInEndSection': 698,\n",
       "     'text': 'Clinical inflammatory signs, probe-to-bone test, and plain X-rays are postulated as the basic tests for clinical diagnosis when DFO is suspected. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28440774',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1596,\n",
       "     'offsetInEndSection': 1773,\n",
       "     'text': 'In patients with a new DFU, we recommend probe to bone test and plain films to be followed by magnetic resonance imaging if a soft tissue abscess or osteomyelitis is suspected. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26804367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 103,\n",
       "     'text': 'Diagnostic Accuracy of Probe to Bone to Detect Osteomyelitis in the Diabetic Foot: A Systematic Review.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27369321',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 279,\n",
       "     'text': 'The probe-to-bone (PTB) test is a commonly used clinical test for osteomyelitis (OM), but its utility has been questioned in clinical settings where the prevalence of OM is low. This article aims to systematically review the accuracy of the PTB test to diagnose diabetic foot OM.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27369321',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 602,\n",
       "     'offsetInEndSection': 878,\n",
       "     'text': 'Pooled sensitivity and specificity for the PTB test was 0.87 (95% confidence interval [CI], .75-.93) and 0.83 (95% CI, .65-.93), respectively. We conclude that the PTB test can accurately rule in diabetic foot OM in the high-risk patients and rule out OM in low-risk patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27369321',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 143,\n",
       "     'text': 'Inter-observer reproducibility of diagnosis of diabetic foot osteomyelitis based on a combination of probe-to-bone test and simple radiography.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24857262',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 340,\n",
       "     'text': 'Probe-to-bone test and simple X-rays are both standard tests for the diagnosis of diabetic foot osteomyelitis. This study demonstrates the importance of considering jointly clinical information (probe-to-bone test) and diagnostic tests (simple radiography) to increase agreement among clinicians on diagnosis of diabetic foot osteomyelitis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24857262',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 473,\n",
       "     'text': 'Magnetic resonance imaging (MRI) has a higher sensitivity and specificity (90% and 79%) than plain radiography (54% and 68%) for diagnosing diabetic foot osteomyelitis. MRI performs somewhat better than any of several common tests--probe to bone (PTB), erythrocyte sedimentation rate (ESR)>70 mm/hr, C-reactive protein (CRP)>14 mg/L, procalcitonin>0.3 ng/mL, and ulcer size>2 cm²--although PTB has the highest specificity of any test and is commonly used together with MRI.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26009740',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'AIMS: To investigate the accuracy of the sequential combination of the probe-to-bone test and plain X-rays for diagnosing osteomyelitis in the foot of patients with diabetes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 935,\n",
       "     'offsetInEndSection': 1149,\n",
       "     'text': 'RESULTS: Overall, 72.4% of patients had histologically proven osteomyelitis, 85.2% of whom had positive bone culture. The performance characteristics of both the probe-to-bone test and plain X-rays were excellent. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1464,\n",
       "     'offsetInEndSection': 1745,\n",
       "     'text': 'CONCLUSIONS: Clinicians seeing patients in a setting similar to ours (specialized diabetic foot unit with a high prevalence of osteomyelitis) can confidently diagnose diabetic foot osteomyelitis when either the probe-to-bone test or a plain X-ray, or especially both, are positive.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 192,\n",
       "     'offsetInEndSection': 469,\n",
       "     'text': 'The tests assessed were probe-to-bone (PTB), clinical signs of infection, radiography signs of osteomyelitis, and ulcer specimen culture.<br><b>RESEARCH DESIGN AND METHODS</b>: A prospective study was performed on patients with foot ulcers referred to our diabetic foot clinic.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20622159',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1135,\n",
       "     'offsetInEndSection': 1720,\n",
       "     'text': 'Among the tests compared, the best results were yielded by the PTB test including an efficiency of 94%, sensitivity of 98%, specificity of 78%, positive predictive value of 95%, and negative predictive value of 91% (P < 0.001, κ 0.803); the positive likelihood ratio was 4.41, and the negative likelihood ratio was 0.02 (95% CI).<br><b>CONCLUSIONS</b>: In our outpatient population with a high prevalence of osteomyelitis, the PTB test was of greatest diagnostic value, especially for neuropathic ulcers, and proved to be efficient for detecting osteomyelitis in the diabetic foot.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20622159',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 81,\n",
       "     'text': 'Probe-to-bone test for diagnosing diabetic foot osteomyelitis: reliable or relic?',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17259493',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 82,\n",
       "     'text': 'Diagnostic Accuracy of Probe to Bone to Detect Osteomyelitis in the Diabetic Foot:',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27369321',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 175,\n",
       "     'text': 'OBJECTIVE: We sought to assess the accuracy of the probe-to-bone (PTB) test in diagnosing foot osteomyelitis in a cohort of diabetic patients with bone culture proven disease.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17259493',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 108,\n",
       "     'text': 'Validating the probe-to-bone test and other tests for diagnosing chronic osteomyelitis in the diabetic foot.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20622159',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 2757,\n",
       "     'offsetInEndSection': 3022,\n",
       "     'text': 'An ulcer area larger than 2 cm2, a positive probe-to-bone test result, an erythrocyte sedimentation rate of more than 70 mm/h, and an abnormal plain radiograph result are helpful in diagnosing the presence of lower extremity osteomyelitis in patients with diabetes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18285592',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 177,\n",
       "     'text': 'The probe-to-bone (PTB) test is a commonly used clinical test for osteomyelitis (OM), but its utility has been questioned in clinical settings where the prevalence of OM is low.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27369321',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1470,\n",
       "     'offsetInEndSection': 1750,\n",
       "     'text': 'CONCLUSIONS Clinicians seeing patients in a setting similar to ours (specialized diabetic foot unit with a high prevalence of osteomyelitis) can confidently diagnose diabetic foot osteomyelitis when either the probe-to-bone test or a plain X-ray, or especially both, are positive.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 173,\n",
       "     'text': 'AIMS To investigate the accuracy of the sequential combination of the probe-to-bone test and plain X-rays for diagnosing osteomyelitis in the foot of patients with diabetes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'OBJECTIVE We sought to assess the accuracy of the probe-to-bone (PTB) test in diagnosing foot osteomyelitis in a cohort of diabetic patients with bone culture proven disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17259493',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1286,\n",
       "     'offsetInEndSection': 1681,\n",
       "     'text': 'Wound area and depth were not found to be statistically significantly different between groups.<br><b>CONCLUSIONS</b>: Positive probe-to-bone test results and erythrocyte sedimentation rates greater than 70 mm/h provide some support for the diagnosis of diabetic foot osteomyelitis, but it is not strong; magnetic resonance imaging or bone biopsy will probably be required in cases of doubt.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23001730',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 629,\n",
       "     'text': '<b>BACKGROUND</b>: We investigated the validity of probe-to-bone testing in the diagnosis of osteomyelitis in a selected subgroup of patients clinically suspected of having diabetic foot osteomyelitis.<br><b>METHODS</b>: Between January 1, 2007, and December 31, 2008, inpatients and outpatients with a diabetic foot ulcer were prospectively evaluated, and those having a clinical diagnosis of foot infection and at least one of the osteomyelitis clinical suspicion criteria were consecutively included in this study.<br><b>RESULTS</b>: Sixty-five patients met the inclusion criteria and were prospectively enrolled in the study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23001730',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 164,\n",
       "     'text': 'We sought to assess the accuracy of the probe-to-bone (PTB) test in diagnosing foot osteomyelitis in a cohort of diabetic patients with bone culture proven disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17259493',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 82,\n",
       "     'text': 'Probe-to-bone test for diagnosing diabetic foot osteomyelitis: reliable or relic?',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17259493',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 109,\n",
       "     'text': 'Validating the probe-to-bone test and other tests for diagnosing chronic osteomyelitis in the diabetic foot.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20622159',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': 'Diagnosing diabetic foot osteomyelitis: is the combination of probe-to-bone test and plain radiography sufficient for high-risk inpatients?',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 168,\n",
       "     'text': 'To investigate the accuracy of the sequential combination of the probe-to-bone test and plain X-rays for diagnosing osteomyelitis in the foot of patients with diabetes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1443,\n",
       "     'offsetInEndSection': 1712,\n",
       "     'text': 'Clinicians seeing patients in a setting similar to ours (specialized diabetic foot unit with a high prevalence of osteomyelitis) can confidently diagnose diabetic foot osteomyelitis when either the probe-to-bone test or a plain X-ray, or especially both, are positive..',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21219428',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 110,\n",
       "     'text': 'Probe-to-bone test and simple X-rays are both standard tests for the diagnosis of diabetic foot osteomyelitis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24857262',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can doxycycline cause photosensitivity?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/26816569',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17315050',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10233667',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29083100',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17160610',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15262663',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23182067',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26299894',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26189561'],\n",
       "   'ideal_answer': ['Yes, one of the most important dermatologic side effects of doxycycline is photosensitivity. Clinical symptoms vary from light sunburn-like sensation (burning, erythema) to large-area photodermatitis.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4069681',\n",
       "    'http://www.biosemantics.org/jochem#4272840',\n",
       "    'https://meshb.nlm.nih.gov/record/ui?ui=D004318'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5a69031bb750ff445500001e',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'Phototoxicity of Doxycycline: A Systematic Review on Clinical Manifestations, Frequency, Cofactors, and Prevention.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 100,\n",
       "     'text': 'BACKGROUND: One of the most important dermatologic side effects of doxycycline is photosensitivity. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 276,\n",
       "     'offsetInEndSection': 402,\n",
       "     'text': 'While there are many publications on the phototoxicity of tetracyclines in general, only a few exist focusing on doxycycline. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 803,\n",
       "     'offsetInEndSection': 911,\n",
       "     'text': 'Clinical symptoms vary from light sunburn-like sensation (burning, erythema) to large-area photodermatitis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1233,\n",
       "     'offsetInEndSection': 1407,\n",
       "     'text': 'CONCLUSION: Evidence base must be improved for giving advice on appropriate prevention measures to travelers taking doxycycline and having a risk of significant sun exposure.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 5967,\n",
       "     'offsetInEndSection': 6364,\n",
       "     'text': 'Based on the available evidence, our best estimates of absolute effect for mefloquine versus doxycyline were: 2% versus 2% for discontinuation, 12% versus 3% for insomnia, 31% versus 3% for abnormal dreams, 18% versus 1% for anxiety, 11% versus 1% for depressed mood, 4% versus 14% for dyspepsia, 2% versus 19% for photosensitivity, 1% versus 5% for vomiting, and 2% versus 16% for vaginal thrush.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29083100',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 370,\n",
       "     'offsetInEndSection': 564,\n",
       "     'text': 'Many drugs are responsible for this phototoxic reaction, especially tetracyclines, psoralens, chloramphenicol, non-steroidal anti-inflammatory drugs, fluoroquinolones, and, rarely, doxycycline. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26816569',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': 'OBJECTIVES: Many patients undergoing long-term doxycycline treatment do not regularly take their treatment because of photosensitivity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26299894',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 114,\n",
       "     'text': 'Modulation of Melanogenesis and Antioxidant Status of Melanocytes in Response to Phototoxic Action of Doxycycline.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26189561',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 236,\n",
       "     'text': 'Doxycycline is a commonly used tetracycline antibiotic showing the broad spectrum of antibacterial action. However, the use of this antibiotic is often connected with the risk of phototoxic reactions that lead to various skin disorders.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26189561',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 840,\n",
       "     'offsetInEndSection': 1034,\n",
       "     'text': 'The results obtained in vitro may explain the mechanisms of phototoxic reactions that occur in normal human epidermal melanocytes in vivo after exposure of skin to doxycycline and UVA radiation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26189561',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1948,\n",
       "     'offsetInEndSection': 2119,\n",
       "     'text': 'Treatment with doxycycline is cheap and relatively safe, but gastrointestinal symptoms and photosensitivity reactions can be expected more often than with ceftriaxone.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17160610',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': '<b>OBJECTIVES</b>: Many patients undergoing long-term doxycycline treatment do not regularly take their treatment because of photosensitivity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26299894',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1417,\n",
       "     'offsetInEndSection': 1569,\n",
       "     'text': 'Photosensitivity reactions and gastrointestinal symptoms were noted more often among patients receiving doxycycline than in those receiving ceftriaxone.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17160610',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 443,\n",
       "     'offsetInEndSection': 830,\n",
       "     'text': 'Thus, the action spectra of the drug photosensitivity patients were plotted and compared with those of 12 nonphotosensitive control patients: 10 patients were found to be photosensitive in the UVA range; the implicated drugs included quinine, sparfloxacin, amiodarone, doxycycline, mefenamic acid, nalidixic acid, fenbrufen, diclofenac, enalapril, diltiazem and prochlorperazine maleate.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10233667',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 831,\n",
       "     'offsetInEndSection': 908,\n",
       "     'text': 'One patient on doxycycline was photosensitive in both the UVA and UVB ranges.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10233667',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1913,\n",
       "     'offsetInEndSection': 2080,\n",
       "     'text': 'Treatment with doxycycline is cheap and relatively safe, but gastrointestinal symptoms and photosensitivity reactions can be expected more often than with ceftriaxone.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17160610',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1381,\n",
       "     'offsetInEndSection': 1632,\n",
       "     'text': 'Anti-inflammatory-dose doxycycline should not be used by individuals with known hypersensitivity to tetracyclines or increased photosensitivity, or by pregnant or nursing women (anti-inflammatory-dose doxycycline is a pregnancy category-D medication).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17315050',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 106,\n",
       "     'text': '<b>BACKGROUND</b>: One of the most important dermatologic side effects of doxycycline is photosensitivity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 87,\n",
       "     'text': 'One of the most important dermatologic side effects of doxycycline is photosensitivity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291967',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1346,\n",
       "     'offsetInEndSection': 1434,\n",
       "     'text': 'One patient experienced mild photosensitivity from doxycycline but continued to take it.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23182067',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1524,\n",
       "     'offsetInEndSection': 1616,\n",
       "     'text': 'Participants in the doxycycline group had a higher incidence of nausea and photosensitivity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15262663',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1347,\n",
       "     'offsetInEndSection': 1499,\n",
       "     'text': 'Photosensitivity reactions and gastrointestinal symptoms were noted more often among patients receiving doxycycline than in those receiving ceftriaxone.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17160610',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the most common feature of the Doege–Potter syndrome?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25636632',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27376978',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23302323',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28784156',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27932370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28523334',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28228892',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28109372',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22206790',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28410845'],\n",
       "   'ideal_answer': ['Doege-Potter syndrome is a paraneoplastic syndrome characterized by hypoglycemia secondary to a solitary fibrous tumor of the pleura.'],\n",
       "   'exact_answer': ['hypoglycemia'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5a761ac3aacfb9cd4c000002',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 181,\n",
       "     'text': 'We describe a patient with Doege-Potter syndrome (solitary fibrous tumor of the pleura presenting with hypoglycemia) and illustrate several important lessons learned from the case. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28109372',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 185,\n",
       "     'text': 'Solitary fibrous tumour of the pleura (SFTP) is a rare primary tumour of the pleura associated with 4% of cases with a paraneoplastic hypoglycaemia, termed Doege-Potter syndrome (DPS). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27932370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 110,\n",
       "     'text': 'Metastatic extrapleural malignant solitary fibrous tumor presenting with hypoglycemia (Doege-Potter syndrome).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28228892',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': 'Doege-Potter syndrome\\xa0is a paraneoplastic\\xa0syndrome\\xa0characterized by non-islet cell tumor hypoglycemia secondary to a solitary fibrous tumor. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28410845',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 146,\n",
       "     'text': 'AIM: Doege-Potter syndrome is a rare condition consisting of a mesenchymal tumor, either benign or malignant, accompanied by severe hypoglycemia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28523334',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 75,\n",
       "     'text': 'Large pleural tumor revealed by severe hypoglycemia: Doege-Potter syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28523334',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 270,\n",
       "     'offsetInEndSection': 377,\n",
       "     'text': 'f hypoglycemia is associated with a solitary fibrous tumor, it is referred to as the Doege-Potter syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28784156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 890,\n",
       "     'offsetInEndSection': 1018,\n",
       "     'text': 'Her course was complicated by the development of recurrent fasting hypoglycemia, most likely secondary to Doege-Potter syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23302323',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 161,\n",
       "     'text': '<b>INTRODUCTION</b>: Doege-Potter syndrome is a paraneoplastic syndrome characterized by non-islet cell tumor hypoglycemia secondary to a solitary fibrous tumor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23302323',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 469,\n",
       "     'offsetInEndSection': 614,\n",
       "     'text': 'We present a case of Doege-Potter syndrome whose interest is to consider the TFSP as a cause of hypoglycemia in patients with pleural tumors.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22206790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': 'Syndrome Doege-Potter is a paraneoplastic syndrome in which hypoglycemia is the result of tumors producing insulin growth factor-like (IGF-II) it is most often solitary fibrous tumor of the pleura (TFSP).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22206790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 110,\n",
       "     'text': 'A common problem in the elderly with an uncommon cause: hypoglycaemia secondary to the Doege-Potter syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25636632',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': 'Doege-Potter syndrome is a paraneoplastic syndrome characterized by non-islet cell tumor hypoglycemia secondary to a solitary fibrous tumor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23302323',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': 'Doege-Potter syndrome is a paraneoplastic syndrome characterized by tumor-associated hypoglycemia secondary to a solitary fibrous tumor of the pleura.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27376978',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can CD55 deficiency cause thrombosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/26205796',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23020629',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25237200',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22077430',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28657829',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11843289',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22696589',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23765390',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27812245',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28516949'],\n",
       "   'ideal_answer': ['Yes, loss of CD55 is associated with thrombosis in patients with Paroxysmal nocturnal hemoglobinuria. CD55 deficiency with hyperactivation of complement, angiopathic thrombosis, and protein-losing enteropathy (the CHAPLE syndrome) is caused by abnormal complement activation due to biallelic loss-of-function mutations in CD55'],\n",
       "   'exact_answer': 'yes',\n",
       "   'concepts': ['http://www.disease-ontology.org/api/metadata/DOID:0060903',\n",
       "    'https://meshb.nlm.nih.gov/record/ui?ui=D013927'],\n",
       "   'type': 'yesno',\n",
       "   'id': '5a723cd12dc08e987e00000b',\n",
       "   'snippets': [{'offsetInBeginSection': 883,\n",
       "     'offsetInEndSection': 1056,\n",
       "     'text': ' The loss of CD55 and CD59 renders PNH erythrocytes susceptible to intravascular haemolysis, which can lead to thrombosis and to much of the morbidity and mortality of PNH. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28516949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 72,\n",
       "     'text': 'CD55 Deficiency, Early-Onset Protein-Losing Enteropathy, and Thrombosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28657829',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1236,\n",
       "     'offsetInEndSection': 1475,\n",
       "     'text': 'CONCLUSIONS: CD55 deficiency with hyperactivation of complement, angiopathic thrombosis, and protein-losing enteropathy (the CHAPLE syndrome) is caused by abnormal complement activation due to biallelic loss-of-function mutations in CD55. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28657829',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 175,\n",
       "     'offsetInEndSection': 854,\n",
       "     'text': 'It is caused by the expansion of a hematopoietic progenitor cell that has acquired a mutation in the X-linked phosphatidylinositol glycan class A (PIGA) gene that results in deficiency of the glycosylphosphatidylinositol anchor structure responsible for fixing a wide spectrum of proteins particularly CD55 and CD59. The clinical features of this disease arise as a result of complement-mediated hemolysis in unprotected red cells, leukocytes, and platelets as well as the release of free hemoglobin. Patients may present with a variety of clinical manifestations, such as anemia, thrombosis, kidney disease, smooth muscle dystonias, abdominal pain, dyspnea, and extreme fatigue.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27812245',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 273,\n",
       "     'offsetInEndSection': 445,\n",
       "     'text': 'The lack of one of the GPI-AP complement regulatory proteins (CD55, CD59) leads to hemolysis. The disease is diagnosed with hemolytic anemia, marrow failure and thrombosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26205796',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 353,\n",
       "     'text': 'Paroxysmal nocturnal hemoglobinuria (PNH) is a rare bone marrow failure disorder that manifests with hemolytic anemia, thrombosis, and peripheral blood cytopenias. The absence of two glycosylphosphatidylinositol (GPI)-anchored proteins, CD55 and CD59, leads to uncontrolled complement activation that accounts for hemolysis and other PNH manifestations.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25237200',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 954,\n",
       "     'offsetInEndSection': 1164,\n",
       "     'text': 'RESULTS: CD55 and/or CD59 deficiencies were found in 1.6% (2/127) of patients with primary BCS, 1.0% (1/100) of non-malignant and non-cirrhotic patients with PVT, and 4.7% (4/85) of cirrhotic patients with PVT.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23020629',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 819,\n",
       "     'offsetInEndSection': 983,\n",
       "     'text': 'Data of this study indicate that the PNH defect as detected with CD55, CD59, and CD16 is not an important cause of intra-abdominal thrombosis in northwestern India.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22696589',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 873,\n",
       "     'offsetInEndSection': 1146,\n",
       "     'text': 'PNH testing of red blood cells revealed a CD55 and CD59 deficiency consistent with PNH in both cases. The systemic complications typically associated with thrombosis were not observed for the following several months with early conservative treatments including eculizumab.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23765390',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 193,\n",
       "     'offsetInEndSection': 426,\n",
       "     'text': 'Deficiency of the GPI-anchored complement inhibitors CD55 and CD59 on erythrocytes leads to intravascular hemolysis upon complement activation. Apart from hemolysis, another prominent feature is a highly increased risk of thrombosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22077430',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1131,\n",
       "     'offsetInEndSection': 1514,\n",
       "     'text': 'Genetic reconstitution of CD55 or treatment with a complement-inhibitory therapeutic antibody reversed abnormal complement activation.<br><b>CONCLUSIONS</b>: CD55 deficiency with hyperactivation of complement, angiopathic thrombosis, and protein-losing enteropathy (the CHAPLE syndrome) is caused by abnormal complement activation due to biallelic loss-of-function mutations in CD55.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28657829',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 298,\n",
       "     'text': 'Paroxysmal nocturnal hemoglobinuria (PNH) is an acquired clonal disorder characterized by a decrease or absence of glycosylphosphatidylinositol (GPI)-anchored molecules such as CD55 and CD59 from the surface of affected cells, resulting in intravascular hemolysis, cytopenia, and venous thrombosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11843289',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1242,\n",
       "     'offsetInEndSection': 1479,\n",
       "     'text': 'CONCLUSIONS CD55 deficiency with hyperactivation of complement, angiopathic thrombosis, and protein-losing enteropathy (the CHAPLE syndrome) is caused by abnormal complement activation due to biallelic loss-of-function mutations in CD55.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28657829',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1209,\n",
       "     'offsetInEndSection': 1434,\n",
       "     'text': 'CD55 deficiency with hyperactivation of complement, angiopathic thrombosis, and protein-losing enteropathy (the CHAPLE syndrome) is caused by abnormal complement activation due to biallelic loss-of-function mutations in CD55.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28657829',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 73,\n",
       "     'text': 'CD55 Deficiency, Early-Onset Protein-Losing Enteropathy, and Thrombosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28657829',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Describe Vanishing lung syndrome.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24511405',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28219569',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28975065',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24977015',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26259424',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27695175',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25322795',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26770619',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19704328',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22937434'],\n",
       "   'ideal_answer': ['Vanishing lung syndrome, also known as idiopathic giant bullous emphysema, is a rare disease characterized by giant emphysematous bullae. It is a rare radiological syndrome in which the lungs appear to be disappearing on X-ray. It  typically occurs in young, thin male smokers with large bullae in one or more upper lobes occupying at least one-third of the hemithorax. This syndrome is associated with significant morbidity and mortality.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5a7376e33b9d13c708000009',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 130,\n",
       "     'text': 'Vanishing lung syndrome is a clinical presentation of giant bullous emphysema associated with significant morbidity and mortality.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28219569',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 129,\n",
       "     'offsetInEndSection': 306,\n",
       "     'text': 'A small number of case reports have described a pathological entity called vanishing lung syndrome (VLS), which is a rare bullous lung disease usually caused by tobacco smoking.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28975065',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 191,\n",
       "     'text': 'Giant bullous emphysema, or vanishing lung syndrome, typically occurs in young, thin male smokers with large bullae in one or more upper lobes occupying at least one-third of the hemithorax. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26770619',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'Vanishing lung syndrome, also known as idiopathic giant bullous emphysema, is a rare disease characterized by giant emphysematous bullae. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25322795',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 117,\n",
       "     'text': 'Vanishing lung syndrome (VLS) is a rare radiological syndrome in which the lungs appear to be disappearing on X-ray. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26259424',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 147,\n",
       "     'text': \"Giant bullous emphysema (GBE), known as 'vanishing lung syndrome', usually occurs in association with chronic obstructive pulmonary disease (COPD).\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24977015',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 249,\n",
       "     'text': 'Vanishing lung syndrome (VLS) is a rare and distinct clinical syndrome that usually affects young men. VLS leads to severe progressive dyspnea and is characterized by extensive, asymmetric, peripheral, and predominantly upper lobe giant lung bullae.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24511405',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 275,\n",
       "     'text': 'We describe here a rare case of Idiopathic Bullous Emphysema/Vanishing Lung Syndrome (VLS) in a 33-year-old male patient with a history of marijuana abuse who presents to the hospital with pleuritic chest pain thought to be due to pneumothorax based on the chest radiograph. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22937434',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Vanishing lung syndrome (giant bullous emphysema): CT findings in 7 patients and a literature review.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19704328',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 695,\n",
       "     'offsetInEndSection': 792,\n",
       "     'text': 'Eventually, his CT chest revealed the diagnosis of giant bullous disease/vanishing lung syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26259424',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'Vanishing lung syndrome (VLS) is a rare radiological syndrome in which the lungs appear to be disappearing on X-ray.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26259424',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Vanishing lung syndrome, also known as idiopathic giant bullous emphysema, is a rare disease characterized by giant emphysematous bullae.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25322795',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'The Mantoux test detects what latent infection/disease?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28337457',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12238571',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27187878',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16307510',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28276801',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17663901',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11089393',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27529924',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25401733',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21822042'],\n",
       "   'ideal_answer': ['screened for TB infection with a Mantoux tuberculin skin testtuberculin skin test (TST) performed according to the Mantoux method.',\n",
       "    'screened for TB infection with a Mantoux tuberculin skin test ',\n",
       "    'The Mantoux tuberculin skin test tests for latent tuberculosis',\n",
       "    'screened for TB infection with a Mantoux tuberculin skin test. '],\n",
       "   'exact_answer': ['tuberculosis'],\n",
       "   'concepts': ['https://meshb.nlm.nih.gov/record/ui?ui=D055985',\n",
       "    'https://meshb.nlm.nih.gov/record/ui?ui=D012882',\n",
       "    'https://meshb.nlm.nih.gov/record/ui?ui=D014374'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5ab147edfcf4565872000013',\n",
       "   'snippets': [{'offsetInBeginSection': 659,\n",
       "     'offsetInEndSection': 721,\n",
       "     'text': 'screened for TB infection with a Mantoux tuberculin skin test ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28276801',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 447,\n",
       "     'offsetInEndSection': 606,\n",
       "     'text': 'The present study reports the results of a three-year TB surveillance among HCW in a large teaching hospital in Rome, using TST (by standard Mantoux technique)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28337457',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1447,\n",
       "     'offsetInEndSection': 1594,\n",
       "     'text': 'Laboratory diagnostic testing of cellular immunity against pathogenic mycobacteria employs the tuberculin skin test (TST, Mantoux tuberculin test) ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27187878',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 421,\n",
       "     'offsetInEndSection': 491,\n",
       "     'text': 'tuberculin skin test (TST) performed according to the Mantoux method. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25401733',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 273,\n",
       "     'text': 'Diaskintest® test and Mantoux test with 2TE PPD-L were concurrently carried out in 300 children and adolescents with tuberculosis and followed up in risk groups at a tuberculosis dispensary to determine the sensitivity of the new skin test in active tuberculosis infection.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27529924',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1545,\n",
       "     'offsetInEndSection': 1624,\n",
       "     'text': 'Latent tuberculosis treatment was recommended in all Mantoux-positive contacts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17663901',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 236,\n",
       "     'offsetInEndSection': 440,\n",
       "     'text': 'The prevalence of tuberculosis infection among hospitalized patients in a pneumological department of an inner-city hospital was evaluated, using the intradermal tuberculin skin test (Mantoux technique). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11089393',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 491,\n",
       "     'offsetInEndSection': 657,\n",
       "     'text': 'he 100-year-old tuberculin skin test (Mantoux) has been the only available diagnostic test for latent tuberculosis infection, despite its many well-known limitations.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16307510',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 288,\n",
       "     'text': 'In order to determine the prevalence of latent infection due to Mycobacterium tuberculosis in drug users and to provide centres for drug users with a practical tool for tuberculosis screening, 237 drug users were subjected to the Monotest and, for reference purposes, to the Mantoux test.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12238571',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is caused by the ectopic expression of CTCF?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28862757',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15941718',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28161276',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26268681',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11507042',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14980504',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21465478',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19369356',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21870268',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9591631'],\n",
       "   'ideal_answer': ['ectopic expression of CTCF in K562 cells led to growth retardation and promotion of differentiation into the erythroid lineage;',\n",
       "    'Enforced ectopic expression of CTCF inhibits cell growth in culture. ectopic expression of CTCF in K562 cells led to growth retardation and promotion of differentiation into the erythroid lineage;',\n",
       "    'Ectopic expression of CTCF results in severe cell growth inhibition at multiple points within the cell cycle.',\n",
       "    'ectopic expression of CTCF in K562 cells led to growth retardation and promotion of differentiation into the erythroid lineage;. ',\n",
       "    'Enforced ectopic expression of CTCF inhibits cell growth in culture.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5abcf6b9fcf4565872000027',\n",
       "   'snippets': [{'offsetInBeginSection': 852,\n",
       "     'offsetInEndSection': 920,\n",
       "     'text': 'Enforced ectopic expression of CTCF inhibits cell growth in culture.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9591631',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 506,\n",
       "     'offsetInEndSection': 656,\n",
       "     'text': 'Here we show that ectopic expression of CTCF in many cell types inhibits cell clonogenicity by causing profound growth retardation without apoptosis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11507042',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 213,\n",
       "     'offsetInEndSection': 381,\n",
       "     'text': 'Ectopic expression of CTCF results in severe cell growth inhibition at multiple points within the cell cycle, indicating that CTCF levels must be stringently monitored.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14980504',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 640,\n",
       "     'offsetInEndSection': 767,\n",
       "     'text': 'ectopic expression of CTCF in K562 cells led to growth retardation and promotion of differentiation into the erythroid lineage;',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15941718',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 120,\n",
       "     'offsetInEndSection': 310,\n",
       "     'text': 'Ectopic expression of CTCF in various normal and tumoral human cell lines inhibits cell division and clonogenicity, with the consequence to consider CTCF a potential tumor-suppressor factor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21465478',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 833,\n",
       "     'offsetInEndSection': 1065,\n",
       "     'text': 'Depletion of the BORIS gene leads to altered transcription of a large number of genes and the differentiation of K562 cells, while the ectopic expression of this CTCF paralog leads to specific changes in transcription in MCF7 cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26268681',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 903,\n",
       "     'offsetInEndSection': 1219,\n",
       "     'text': 'The role of GATA-1 in interaction between CTCF sites was revealed by its ectopic expression in 293 cells and by deletion of a GATA-1 site in the LCR HS2. These findings indicate that erythroid specific activator GATA-1 acts at CTCF sites around the β-globin locus to establish tissue-specific chromatin organization.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28161276',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 502,\n",
       "     'text': 'By using these model systems we demonstrated that: (i) ectopic expression of CTCF in K562 cells led to growth retardation and promotion of differentiation into the erythroid lineage; (ii) CTCF knock-down significantly inhibited differentiation of K562 cells into erythroid lineage; (iii) differentiation of K562 into the megakaryocytic lineage was not significantly affected; and (iv) down-regulation of MYC has been identified as one of the mechanisms by which CTCF promotes erythroid differentiation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15941718',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 355,\n",
       "     'offsetInEndSection': 505,\n",
       "     'text': 'To determine how these varying effects would integrate in vivo, we engineered a variety of expression systems to study effects of CTCF on cell growth.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11507042',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 506,\n",
       "     'offsetInEndSection': 655,\n",
       "     'text': 'Here we show that ectopic expression of CTCF in many cell types inhibits cell clonogenicity by causing profound growth retardation without apoptosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11507042',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1014,\n",
       "     'offsetInEndSection': 1315,\n",
       "     'text': 'While MYC is the well-characterized CTCF target, the inhibitory effects of CTCF on cell growth could not be ascribed solely to repression of MYC, suggesting that additional CTS-driven genes involved in growth-regulatory circuits, such as p19ARF, are likely to contribute to CTCF-induced growth arrest.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11507042',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 676,\n",
       "     'offsetInEndSection': 783,\n",
       "     'text': 'Deletion of the CTCF-cohesin binding site caused an inhibition of cell growth and viral genome instability.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19369356',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 585,\n",
       "     'offsetInEndSection': 1087,\n",
       "     'text': 'By using these model systems we demonstrated that: (i) ectopic expression of CTCF in K562 cells led to growth retardation and promotion of differentiation into the erythroid lineage; (ii) CTCF knock-down significantly inhibited differentiation of K562 cells into erythroid lineage; (iii) differentiation of K562 into the megakaryocytic lineage was not significantly affected; and (iv) down-regulation of MYC has been identified as one of the mechanisms by which CTCF promotes erythroid differentiation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15941718',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Please list 3 diseases treated with Valtrex(valacyclovir)',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/16771614',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8593015',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22670895',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15482124',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25749332',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14505192',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15550990',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15989601',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11772333',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12556212'],\n",
       "   'ideal_answer': ['Valtrex (valacyclovir) is an antiviral medication used to treat infections with: herpes zoster (shingles), herpes simplex genitalis (genital herpes),\\nand herpes labialis (cold sores).'],\n",
       "   'exact_answer': [['Herpes zoster or Shingles'],\n",
       "    ['Herpes genitalis, or genital herpes'],\n",
       "    ['Herpes labialis or cold sores']],\n",
       "   'concepts': ['http://www.biosemantics.org/jochem#4273236',\n",
       "    'http://www.biosemantics.org/jochem#4265772'],\n",
       "   'type': 'list',\n",
       "   'id': '5abcf755fcf4565872000028',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 239,\n",
       "     'text': 'Valaciclovir (Valtrex, Zelitrex), the L-valine ester of aciclovir, increases aciclovir bioavailability by 3- to 5-fold over that achievable with oral aciclovir. It addresses many unmet needs of currently available anti-herpesvirus therapie',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15989601',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1355,\n",
       "     'offsetInEndSection': 1550,\n",
       "     'text': 'In controlled clinical trials in herpes zoster, valaciclovir (1000 mg three times daily) is superior to aciclovir in speeding the resolution of zoster-associated pain and post-herpetic neuralgia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15989601',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1169,\n",
       "     'offsetInEndSection': 1403,\n",
       "     'text': 'alacyclovir is the only antiviral drug approved for a once-daily dose of suppressive therapy for genital herpes, as well as the only antiviral drug US FDA approved for a 3-day regimen of episodic treatment of recurrent genital herpes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16771614',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1417,\n",
       "     'offsetInEndSection': 1568,\n",
       "     'text': 'valacyclovir is also indicated in the reduction of the sexual transmission of herpes simplex virus infection and for the treatment of herpes labialis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16771614',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1569,\n",
       "     'offsetInEndSection': 1784,\n",
       "     'text': 'n herpes zoster, valacyclovir is more effective than acyclovir or placebo (and as equally effective as famciclovir) in shortening the length and severity of herpes zoster-associated pain and postherpetic neuralgia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16771614',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 606,\n",
       "     'offsetInEndSection': 823,\n",
       "     'text': 'valacyclovir and famciclovir are efficacious and safe for the treatment of the first episode and recurrent genital herpes and are useful as suppressive therapy for individuals with frequent genital herpes recurrences.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15482124',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 867,\n",
       "     'offsetInEndSection': 1073,\n",
       "     'text': 'valacyclovir and famciclovir have been shown to speed the healing of herpes zoster, and data suggests that these agents also decrease associated acute and chronic pain in people of 50 years of age or older.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15482124',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 149,\n",
       "     'text': 'Valaciclovir (Valtrex), the L-valyl ester of acyclovir, is undergoing clinical development for the treatment and suppression of herpesviral diseases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8593015',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 93,\n",
       "     'text': 'Valacyclovir is frequently prescribed to the elderly to treat diseases such as herpes zoster.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25749332',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 158,\n",
       "     'text': 'The study group comprised 86 immunocompetent adult patients suffering from herpes zoster, who were treated with either famciclovir or valacyclovir for 7 days.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22670895',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 233,\n",
       "     'offsetInEndSection': 702,\n",
       "     'text': 'Over the past two decades, the treatment of these conditions has been transformed by guanosine nucleoside antivirals such as valacyclovir (Valtrex, a highly bioavailable prodrug of acyclovir (Zovirax, and famciclovir (Famvir), a highly bioavailable prodrug of penciclovir (Denavir).<br><b>OBJECTIVE</b>: We describe the pharmacology, pharmacokinetics, and clinical efficacy of valacyclovir for the treatment of herpes simplex, herpes zoster, and other viral infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14505192',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1014,\n",
       "     'offsetInEndSection': 1228,\n",
       "     'text': 'In herpes zoster, valacyclovir is as effective as famciclovir and more effective than either placebo or acyclovir at facilitating cutaneous healing and healing of zoster-associated pain and post-herpetic neuralgia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11772333',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 509,\n",
       "     'offsetInEndSection': 685,\n",
       "     'text': 'OBJECTIVE: We describe the pharmacology, pharmacokinetics, and clinical efficacy of valacyclovir for the treatment of herpes simplex, herpes zoster, and other viral infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14505192',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1229,\n",
       "     'offsetInEndSection': 1536,\n",
       "     'text': 'Valacyclovir is well tolerated, with convenient dosing frequencies for the treatment of genital herpes or herpes zoster, it also has the option for use as a short course therapy in the episodic treatment of recurrent genital herpes, all of which are important benefits in the management of these conditions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11772333',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 370,\n",
       "     'offsetInEndSection': 620,\n",
       "     'text': 'Acyclovir (Zoviraxr, GlaxoSmithKline), valacyclovir (Valtrex, GlaxoSmithKline) or famciclovir (Famvir, Novartis) can be used to treat herpes zoster, and all three have been shown to reduce the duration of the herpetic rash and zoster-associated pain.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15550990',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List proteins with HEAT repeats',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28512144',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27072897',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24120762',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28415797',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27066066',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26676747',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27549742',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27742610',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22315229',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28668119'],\n",
       "   'ideal_answer': ['mTOR,\\nTOG5, \\nDNA-PKcs,\\nHEATR1,\\nRif1,\\nB56γ,\\nPR65/A,\\nSF3b155,\\nPds5B'],\n",
       "   'exact_answer': [['mTOR'],\n",
       "    ['TOG5'],\n",
       "    ['DNA-PKcs'],\n",
       "    ['HEATR1'],\n",
       "    ['Rif1'],\n",
       "    ['B56γ'],\n",
       "    ['PR65/A'],\n",
       "    ['SF3b155'],\n",
       "    ['Pds5B']],\n",
       "   'type': 'list',\n",
       "   'id': '5abd44b1fcf4565872000030',\n",
       "   'snippets': [{'offsetInBeginSection': 835,\n",
       "     'offsetInEndSection': 869,\n",
       "     'text': 'mTOR N-terminal heat repeat domain',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28415797',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 689,\n",
       "     'offsetInEndSection': 726,\n",
       "     'text': 'TOG5-specific N-terminal HEAT repeat ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28512144',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 245,\n",
       "     'text': 'DNA-dependent protein kinase catalytic subunit (DNA-PKcs) is central to the regulation of the DNA damage response and repair through nonhomologous end joining. The structure has proved challenging due to its large size and multiple HEAT repeats.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28668119',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 223,\n",
       "     'offsetInEndSection': 265,\n",
       "     'text': 'HEAT repeat-containing protein 1 (HEATR1) ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26676747',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 419,\n",
       "     'offsetInEndSection': 716,\n",
       "     'text': \"The first 1,300 residues of Tor form a HEAT repeat-containing α-solenoid with four distinct segments: a highly curved 800-residue N-terminal 'spiral', followed by a 400-residue low-curvature 'bridge' and an extended 'railing' running along the bridge leading to the 'cap' that links to FAT region.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27072897',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 467,\n",
       "     'offsetInEndSection': 591,\n",
       "     'text': 'Rif1 proteins are large and characterized by N-terminal HEAT repeats, predicted to form an elongated alpha-helical structure',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27066066',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 574,\n",
       "     'offsetInEndSection': 689,\n",
       "     'text': 'B56γ contains 18 α-helices that are organized into eight HEAT (Huntington-elongation-A subunit-TOR) repeat motifs. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22315229',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 94,\n",
       "     'text': 'Here, we reveal a remarkable complexity in the unfolding of giant HEAT-repeat protein PR65/A, ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24120762',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 84,\n",
       "     'text': 'A HEAT-repeat superhelix within SF3b155 plays a key role in branch site recognition.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27742610',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 629,\n",
       "     'offsetInEndSection': 658,\n",
       "     'text': 'The HEAT-repeat protein Pds5B',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27549742',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is ibudilast effective for multiple sclerosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28258674',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30255442',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29627873',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15471363',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27501293',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30157388',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19929708',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29779852',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20200338',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27521810'],\n",
       "   'ideal_answer': ['Yes, Ibudilast was shown to be effective for progressive multiple sclerosis. In a phase 2 trial involving patients with progressive multiple sclerosis, ibudilast was associated with slower progression of brain atrophy than placebo but was associated with higher rates of gastrointestinal side effects, headache, and depression.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '5c58ac2286df2b917400000f',\n",
       "   'snippets': [{'offsetInBeginSection': 1155,\n",
       "     'offsetInEndSection': 1244,\n",
       "     'text': 'Ibudilast slowed brain atrophy in PPMS and SPMS patients in a multicenter phase 2b study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29627873',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 76,\n",
       "     'offsetInEndSection': 312,\n",
       "     'text': ' Ibudilast inhibits several cyclic nucleotide phosphodiesterases, macrophage migration inhibitory factor, and toll-like receptor 4 and can cross the blood-brain barrier, with potential salutary effects in progressive multiple sclerosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30157388',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1631,\n",
       "     'offsetInEndSection': 1894,\n",
       "     'text': 'CONCLUSIONS: In a phase 2 trial involving patients with progressive multiple sclerosis, ibudilast was associated with slower progression of brain atrophy than placebo but was associated with higher rates of gastrointestinal side effects, headache, and depression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30157388',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1148,\n",
       "     'offsetInEndSection': 1350,\n",
       "     'text': 'Specifically, the current evidence regarding treatment of progressive MS with ocrelizumab, simvastatin, ibudilast, alpha-lipoic acid, high-dose biotin, siponimod, and cell-based therapies are discussed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30255442',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 100,\n",
       "     'offsetInEndSection': 324,\n",
       "     'text': 'Based on our knowledge of pathophysiology, three therapeutic strategies are proposed: anti-inflammatory (ocrelizumab, siponimod…); remyelinating (opicinumab); and neuroprotective (high-dose biotin, ibudilast, simvastatin…). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29779852',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1333,\n",
       "     'offsetInEndSection': 1649,\n",
       "     'text': 'Current article provides an overview of the pharmacology of IBD with a focus on preclinical and clinical data supporting its potential neuroprotective benefits for neurological conditions, including multiple sclerosis, neuropathic pain, medication overuse headache, stroke, opioid, alcohol and methamphetamine abuse.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28258674',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': 'Design, rationale, and baseline characteristics of the randomized double-blind phase II clinical trial of ibudilast in progressive multiple sclerosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27521810',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 681,\n",
       "     'offsetInEndSection': 786,\n",
       "     'text': 'METHODS: SPRINT-MS is a randomized, placebo-controlled, phase II trial of ibudilast in patients with PMS.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27521810',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1492,\n",
       "     'offsetInEndSection': 1738,\n",
       "     'text': 'CONCLUSION: SPRINT-MS is designed to evaluate the safety and efficacy of ibudilast as a treatment for PMS while simultaneously validating five different imaging biomarkers as outcome metrics for use in future phase II proof-of-concept PMS trials.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27521810',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 50,\n",
       "     'text': 'Ibudilast for the treatment of multiple sclerosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27501293',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 593,\n",
       "     'offsetInEndSection': 1065,\n",
       "     'text': 'AREAS COVERED: This article reviews various studies looking at ibudilast as a potential therapy for MS. It summarizes prior and current clinical trials of ibudilast in MS as well as its pharmacology.EXPERT OPINION: Although ibudilast has not been found to decrease the focal inflammatory activity in relapsing MS, it was shown to have an effect on preserving brain volume and disability progression. Ibudilast may have a role in the treatment of progressive MS phenotypes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27501293',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1569,\n",
       "     'offsetInEndSection': 1934,\n",
       "     'text': 'Adverse events with ibudilast included gastrointestinal symptoms, headache, and depression.<br><b>CONCLUSIONS</b>: In a phase 2 trial involving patients with progressive multiple sclerosis, ibudilast was associated with slower progression of brain atrophy than placebo but was associated with higher rates of gastrointestinal side effects, headache, and depression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30157388',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 715,\n",
       "     'offsetInEndSection': 1021,\n",
       "     'text': 'It summarizes prior and current clinical trials of ibudilast in MS as well as its pharmacology.<br><b>EXPERT OPINION</b>: Although ibudilast has not been found to decrease the focal inflammatory activity in relapsing MS, it was shown to have an effect on preserving brain volume and disability progression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27501293',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which molecule is targeted by Caplacizumab?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23307200',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29209322',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29807376',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28139813',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28445600',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28110841',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28645643',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30298461',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30046703',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26863353'],\n",
       "   'ideal_answer': ['Caplacizumab is anti-von Willebrand factor (VWF) antibody that blocks the interaction between VWF and platelets. It is used for treatment of acquired thrombotic thrombocytopenic purpura (aTTP).'],\n",
       "   'exact_answer': ['von Willebrand factor'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5c55d9b707647bbc4b000009',\n",
       "   'snippets': [{'offsetInBeginSection': 527,\n",
       "     'offsetInEndSection': 766,\n",
       "     'text': 'Caplacizumab, a nanobody that blocks the interaction between VWF and platelets, has shown promising results in decreasing the time to recover from the acute events that will hopefully translate into long-term clinical benefit for patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30046703',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 178,\n",
       "     'text': 'Ablynx, a Sanofi Company, has developed the anti-von Willebrand factor Nanobody® caplacizumab (Cablivi™) for the treatment of acquired thrombotic thrombocytopenic purpura (aTTP).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30298461',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1204,\n",
       "     'offsetInEndSection': 1381,\n",
       "     'text': 'Promising agents under evaluation include N-acetylcysteine, bortezomib, recombinant ADAMTS13 and caplacizumab, an inhibitor of the glycoprotein-Ib/IX-von Willebrand factor axis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28110841',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 376,\n",
       "     'offsetInEndSection': 625,\n",
       "     'text': 'The most promising candidates for future treatment of TTP are: rituximab for termination of the autoimmune process, caplacizumab for prevention of platelet-VWF-interaction, and recombinant ADAMTS13 for replacement of the inhibited or missing enzyme.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28139813',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1276,\n",
       "     'offsetInEndSection': 1470,\n",
       "     'text': 'Promising agents under evaluation include caplacizumab (an inhibitor of the glycoprotein-Ib/IX-Von-Willebrand factor axis), N-acetylcysteine, recombinant ADAMTS13, and anti-plasmocyte compounds.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28645643',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 680,\n",
       "     'offsetInEndSection': 885,\n",
       "     'text': ' In the phase II TITAN study, treatment with caplacizumab, an anti-von Willebrand factor Nanobody® was shown to reduce the time to confirmed platelet count normalization and exacerbations during treatment.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28445600',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1112,\n",
       "     'offsetInEndSection': 1326,\n",
       "     'text': 'It is now anticipated that the first VHH-based antibody drug, Caplacizumab, a bivalent anti-vWF antibody for treating rare blood clotting disorders, may be approved and commercialized in 2018 or shortly thereafter.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29209322',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 261,\n",
       "     'offsetInEndSection': 610,\n",
       "     'text': 'Daily plasma exchange and immunosuppressive therapies induce remission, but mortality and morbidity due to microthrombosis remain high.<br><b>METHODS</b>: Caplacizumab, an anti-von Willebrand factor humanized single-variable-domain immunoglobulin (Nanobody), inhibits the interaction between ultralarge von Willebrand factor multimers and platelets.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26863353',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is Tisagenlecleucel effective for B-Cell Lymphoma?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29247018',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29385370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30213399',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30111196',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30501490',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29914976',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30309857',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29451276',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29499750',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30190371'],\n",
       "   'ideal_answer': ['Yes,  CD19-targeting CAR T-cell therapy tisagenlecleucel produces durable responses in patients with relapsed and refractory diffuse large B-cell lymphoma.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '5c56033607647bbc4b00000c',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 193,\n",
       "     'text': 'The phase II JULIET trial suggests that the CD19-targeting CAR T-cell therapy tisagenlecleucel produces durable responses in patients with relapsed and refractory diffuse large B-cell lymphoma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29247018',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 81,\n",
       "     'text': 'Tisagenlecleucel in Children and Young Adults with B-Cell Lymphoblastic Leukemia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29385370',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 336,\n",
       "     'text': 'BACKGROUND: In a single-center phase 1-2a study, the anti-CD19 chimeric antigen receptor (CAR) T-cell therapy tisagenlecleucel produced high rates of complete remission and was associated with serious but mainly reversible toxic effects in children and young adults with relapsed or refractory B-cell acute lymphoblastic leukemia (ALL).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29385370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1676,\n",
       "     'offsetInEndSection': 1943,\n",
       "     'text': 'CONCLUSIONS: In this global study of CAR T-cell therapy, a single infusion of tisagenlecleucel provided durable remission with long-term persistence in pediatric and young adult patients with relapsed or refractory B-cell ALL, with transient high-grade toxic effects.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29385370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 263,\n",
       "     'text': 'Chimeric antigen receptor T cells demonstrate efficacy in B-cell malignancies, leading to US Food and Drug Administration approval of axicabtagene ciloleucel (October 2017) and tisagenlecleucel (May 2018) for large B-cell lymphomas after 2 prior lines of therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29914976',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 442,\n",
       "     'offsetInEndSection': 605,\n",
       "     'text': 'This article reviews data of current diffuse large B-cell lymphoma management, focusing on axicabtagene ciloleucel, tisagenlecleucel, and lisocabtagene maraleucel.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29914976',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 442,\n",
       "     'offsetInEndSection': 609,\n",
       "     'text': 'This article reviews data of current diffuse large B-cell lymphoma management, focusing on axicabtagene ciloleucel, tisagenlecleucel, and lisocabtagene maraleucel.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29914976',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 195,\n",
       "     'offsetInEndSection': 698,\n",
       "     'text': 'The chimeric antigen receptor (CAR) T-cell therapy tisagenlecleucel targets and eliminates CD19-expressing B cells and showed efficacy against B-cell lymphomas in a single-center, phase 2a study.<br><b>METHODS</b>: We conducted an international, phase 2, pivotal study of centrally manufactured tisagenlecleucel involving adult patients with relapsed or refractory diffuse large B-cell lymphoma who were ineligible for or had disease progression after autologous hematopoietic stem-cell transplantation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30501490',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1818,\n",
       "     'offsetInEndSection': 2154,\n",
       "     'text': 'No differences between response groups in tumor expression of CD19 or immune checkpoint-related proteins were found.<br><b>CONCLUSIONS</b>: In this international study of CAR T-cell therapy in relapsed or refractory diffuse large B-cell lymphoma in adults, high rates of durable responses were produced with the use of tisagenlecleucel.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30501490',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 299,\n",
       "     'offsetInEndSection': 463,\n",
       "     'text': 'This was a milestone in tumor immunology on account of the significant antitumor effect of tisagenlecleucel for the treatment of relapsed/refractory B-ALL patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29451276',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 298,\n",
       "     'text': \"On August 30, 2017, the U.S. Food and Drug Administration (FDA) approved Novartis' tisagenlecleucel (CTL-019, Kymriah), which is a synthetic bioimmune product of anti-CD19 chimeric antigen receptor (CAR) T cells, for the treatment of relapsed/refractory B-cell acute lymphoblastic leukemia (B-ALL).\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29451276',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 244,\n",
       "     'offsetInEndSection': 593,\n",
       "     'text': 'Within the last one year, two anti-CD19 CAR T-cell therapy products, axicabtagene ciloleucel and tisagenlecleucel, were approved by the United States Food and Drug Administration for the treatment of relapsed or refractory large B-cell lymphoma after at least two lines of systemic therapy based on multicenter single-arm phase two clinical trials. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30213399',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 144,\n",
       "     'offsetInEndSection': 382,\n",
       "     'text': 'On August 30, 2017, the U.S. Food and Drug Administration approved tisagenlecleucel for treatment of patients up to 25 years of age with B-cell precursor acute lymphoblastic leukemia (ALL) that is refractory or in second or later relapse.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30309857',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 74,\n",
       "     'text': 'Tisagenlecleucel for the treatment of B-cell acute lymphoblastic leukemia.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30111196',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 225,\n",
       "     'text': 'Background: Tisagenlecleucel is an anti-CD19 chimeric antigen receptor (CAR19) T-cell therapy approved for the treatment of children and young adults with relapsed/refractory (r/r) B-cell acute lymphoblastic leukemia (B-ALL).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30190371',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What membrane proteins constitute TAM family of receptor tyrosine kinases (RTKs)?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25568918',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24184575',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27780404',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28734578',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28034848',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18620092',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27895032',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28904208',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22749189',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18374195'],\n",
       "   'ideal_answer': ['Tyro3, Axl, and Mer are integral membrane proteins that constitute TAM family of receptor tyrosine kinases (RTKs).',\n",
       "    'TAM-family RTKs AXL and Mer (MerTK).'],\n",
       "   'exact_answer': [['Tyro3'], ['Axl'], ['Mer']],\n",
       "   'type': 'list',\n",
       "   'id': '5c5315727e3cb0e231000014',\n",
       "   'snippets': [{'offsetInBeginSection': 677,\n",
       "     'offsetInEndSection': 714,\n",
       "     'text': 'TAM-family RTKs AXL and Mer (MerTK). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27895032',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 46,\n",
       "     'text': 'Deregulation of the TAM (TYRO3, AXL, and MERTK',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28034848',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 983,\n",
       "     'offsetInEndSection': 1015,\n",
       "     'text': 'TAM family members AXL or TYRO3 ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28904208',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 669,\n",
       "     'offsetInEndSection': 748,\n",
       "     'text': 'receptors of the Tyro3/Axl/Mer (TAM) family of receptor tyrosine kinases (RTKs)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28734578',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 195,\n",
       "     'text': 'Tyro-3, Axl, and Mer constitute the TAM family of receptor tyrosine kinases (RTKs) characterized by a conserved sequence within the kinase domain and adhesion molecule-like extracellular domains.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18620092',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'The TYRO3, AXL (also known as UFO) and MERTK (TAM) family of receptor tyrosine kinases (RTKs) are aberrantly expressed in multiple haematological and epithelial malignancies.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25568918',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 112,\n",
       "     'offsetInEndSection': 376,\n",
       "     'text': 'Gas6 was cloned and characterized in 1993 and found to be similar to the plasma anticoagulant protein S. Soon after it was recognized as a growth factor-like molecule, as it interacted with receptor tyrosine kinases (RTKs) of the TAM family; Tyro3, Axl, and MerTK.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18374195',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'The TAM subfamily of Receptor Tyrosine Kinases (RTKs) contains three human proteins of therapeutical interest, Axl, Mer, and Tyro3.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22749189',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 175,\n",
       "     'offsetInEndSection': 383,\n",
       "     'text': 'Increasing evidence shows that the TAM (TYRO3, AXL and MERTK) family of RTKs (receptor tyrosine kinases), which is expressed in macrophages, alleviates inflammatory responses through a negative feedback loop.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27780404',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 660,\n",
       "     'offsetInEndSection': 830,\n",
       "     'text': 'Among the TAM family of RTKs, AXL (AXL receptor tyrosine kinase) only induces autophagy in macrophages after interaction with its ligand, GAS6 (growth arrest specific 6).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27780404',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 443,\n",
       "     'offsetInEndSection': 790,\n",
       "     'text': 'A series of recent studies now consolidates the view that phosphatidylserine (PtdSer), a common phospholipid constituent of membrane bilayers, is similarly repurposed for use as a signal between cells and that the ligands and receptors of the Tyro3/Axl/Mer (TAM) family of receptor tyrosine kinases (RTKs) are prominent transducers of this signal.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28734578',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 263,\n",
       "     'text': 'The TAM family: phosphatidylserine sensing receptor tyrosine kinases gone awry in cancer.The TYRO3, AXL (also known as UFO) and MERTK (TAM) family of receptor tyrosine kinases (RTKs) are aberrantly expressed in multiple haematological and epithelial malignancies. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25568918',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 566,\n",
       "     'offsetInEndSection': 724,\n",
       "     'text': 'The TAM family of RTKs, consisting of Tyro3, Axl, and MerTK, is prominently expressed during the development and function of the central nervous system (CNS). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24184575',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 469,\n",
       "     'offsetInEndSection': 627,\n",
       "     'text': 'The TAM family of RTKs, consisting of Tyro3, Axl, and MerTK, is prominently expressed during the development and function of the central nervous system (CNS).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24184575',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is evaluated with the SAD PERSONS scale?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21546092',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29699523',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26346049',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24533537',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29665120',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28471534',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28302702',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28723978',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24884399'],\n",
       "   'ideal_answer': ['SAD PERSONS scale was developed to evaluate suicide risk.'],\n",
       "   'exact_answer': ['suicide risk'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5c73ad027c78d6947100008d',\n",
       "   'snippets': [{'offsetInBeginSection': 330,\n",
       "     'offsetInEndSection': 582,\n",
       "     'text': 'METHOD: We compared the predictive accuracy of the Manchester Self-Harm Rule (MSHR), ReACT Self-Harm Rule (ReACT), SAD PERSONS Scale (SPS) and Modified SAD PERSONS Scale (MSPS) in an unselected sample of patients attending hospital following self-harm.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29699523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 359,\n",
       "     'offsetInEndSection': 653,\n",
       "     'text': 'Clinicians completed a standard suicide risk instrument (modified SAD PERSONS scale), a 10-point Likert scale assessment of judgment of patient suicide risk (Clinician Prediction Scale), and a measure of their emotional responses to the patient (Therapist Response Questionnaire-Suicide Form). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29665120',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 133,\n",
       "     'offsetInEndSection': 412,\n",
       "     'text': 'AimsTo evaluate the performance of risk scales (Manchester Self-Harm Rule, ReACT Self-Harm Rule, SAD PERSONS scale, Modified SAD PERSONS scale, Barratt Impulsiveness Scale); and patient and clinician estimates of risk in identifying patients who repeat self-harm within 6 months.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28302702',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 46,\n",
       "     'text': 'Predicting suicide with the SAD PERSONS scale.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28471534',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 128,\n",
       "     'offsetInEndSection': 382,\n",
       "     'text': 'The SAD PERSONS suicide risk assessment scale is widely implemented in clinical settings despite limited supporting evidence. This article aims to determine the ability of the SAD PERSONS scale (SPS) to predict future suicide in the emergency department.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28471534',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1379,\n",
       "     'offsetInEndSection': 1603,\n",
       "     'text': 'CONCLUSIONS: Although widely used in educational and clinical settings, these findings do not support the use of the SPS and Modified SPS to predict suicide in adults seen by psychiatric services in the emergency department.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28471534',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 867,\n",
       "     'offsetInEndSection': 1206,\n",
       "     'text': ' For the outcome suicide attempt SAD PERSONS Scale had a sensitivity of 15% (95% CI 8-24) and specificity of 97% (96-98), and the Manchester Self-Harm Rule (MSHR) a sensitivity of 97% (97-97) and a specificity of 20% (20-21). ReACT, which is a modification of MSHR, had a similar low specificity, as did the Sodersjukhuset Self Harm Rule. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28723978',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 129,\n",
       "     'text': 'OBJECTIVE\\nThe SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1283,\n",
       "     'offsetInEndSection': 1442,\n",
       "     'text': 'SAD PERSONS did not predict suicide attempts better than chance (area under the curve =0.572; 95% confidence interval [CI], 0.51-0.64; P value nonsignificant).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1780,\n",
       "     'offsetInEndSection': 1885,\n",
       "     'text': 'CONCLUSION\\nIn their current form, SAD PERSONS and MSPS do not accurately predict future suicide attempts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1767,\n",
       "     'offsetInEndSection': 1896,\n",
       "     'text': 'CONCLUSIONS\\nThe Chinese SAD PERSONS Scale is a brief instrument with acceptable psychometric properties for self-harm prediction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24533537',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 127,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': 'The SAD PERSONS suicide risk assessment scale is widely implemented in clinical settings despite limited supporting evidence.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28471534',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 129,\n",
       "     'text': 'OBJECTIVE The SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1767,\n",
       "     'offsetInEndSection': 1896,\n",
       "     'text': 'CONCLUSIONS The Chinese SAD PERSONS Scale is a brief instrument with acceptable psychometric properties for self-harm prediction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24533537',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 108,\n",
       "     'text': 'The SAD PERSONS scale (SPS) is widely used for suicide risk assessment in clinical and educational settings.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24884399',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 765,\n",
       "     'offsetInEndSection': 929,\n",
       "     'text': \"Beck's Medical Lethality Scale (BMLS) was administered to assess the degree of medical injury, and the SAD PERSONS mnemonic scale was used to evaluate suicide risk.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21546092',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 179,\n",
       "     'text': 'The SAD PERSONS scale for suicide risk assessment: a systematic review.The SAD PERSONS scale (SPS) is widely used for suicide risk assessment in clinical and educational settings. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24884399',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 195,\n",
       "     'offsetInEndSection': 320,\n",
       "     'text': 'The SAD PERSONS suicide risk assessment scale is widely implemented in clinical settings despite limited supporting evidence. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28471534',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2048,\n",
       "     'offsetInEndSection': 2225,\n",
       "     'text': '<AbstractText Label=\"CONCLUSIONS\" NlmCategory=\"CONCLUSIONS\">The Chinese SAD PERSONS Scale is a brief instrument with acceptable psychometric properties for self-harm prediction. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24533537',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 255,\n",
       "     'text': 'Predicting suicide attempts with the SAD PERSONS scale: a longitudinal analysis.<AbstractText Label=\"OBJECTIVE\" NlmCategory=\"OBJECTIVE\">The SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 194,\n",
       "     'text': 'Predicting suicide with the SAD PERSONS scale.<AbstractText Label=\"BACKGROUND\">Suicide is a major public health issue, and a priority requirement is accurately identifying high-risk individuals. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28471534',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 528,\n",
       "     'offsetInEndSection': 720,\n",
       "     'text': 'Subjects completed a series of depression/suicide screening tools: the Columbia Suicide Severity Scale, SAD PERSONS scale, Patient Health Questionnaire 9, and Beck Scale for Suicidal Ideation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26346049',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': '<b>OBJECTIVE</b>: The SAD PERSONS scale is a widely used risk assessment tool for suicidal behavior despite a paucity of supporting data.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1051,\n",
       "     'offsetInEndSection': 1136,\n",
       "     'text': 'Both SAD PERSONS and MSPS showed poor predictive ability for future suicide attempts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1309,\n",
       "     'offsetInEndSection': 1468,\n",
       "     'text': 'SAD PERSONS did not predict suicide attempts better than chance (area under the curve =0.572; 95% confidence interval [CI], 0.51-0.64; P value nonsignificant).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22795212',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is GeneWeaver used for?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24233775',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26834590',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23195309',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22080549',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24731198',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25374613',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26656951',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26092690',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24923803'],\n",
       "   'ideal_answer': ['GeneWeaver: a web-based system for integrative functional genomics.',\n",
       "    'GeneWeaver is a freely available resource for storing, curating and analyzing sets of genes from heterogeneous data sources. GeneWeaver enables researchers to store, share, analyze, and compare results of their own genome-wide functional genomics experiments in an environment containing rich companion data obtained from major curated repositories, including the Mouse Genome Database and other model organism databases, along with derived data from highly specialized resources, publications, and user submissions.',\n",
       "    'GeneWeaver: a web-based system for integrative functional genomics. The freely available GeneWeaver (http://www.GeneWeaver.org) powered by the Ontological Discovery Environment is a curated repository of genomic experimental results with an accompanying tool set for dynamic integration of these data sets, enabling users to interactively address questions about sets of biological functions and their relations to sets of genes. The GeneWeaver.org system provides a platform for cross-species integration and interrogation of heterogeneous curated and experimentally derived functional genomics data.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5c6fff2e7c78d6947100005b',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 67,\n",
       "     'text': 'GeneWeaver: a web-based system for integrative functional genomics.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22080549',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 872,\n",
       "     'offsetInEndSection': 1233,\n",
       "     'text': 'The freely available GeneWeaver (http://www.GeneWeaver.org) powered by the Ontological Discovery Environment is a curated repository of genomic experimental results with an accompanying tool set for dynamic integration of these data sets, enabling users to interactively address questions about sets of biological functions and their relations to sets of genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22080549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 336,\n",
       "     'offsetInEndSection': 507,\n",
       "     'text': 'The GeneWeaver.org system provides a platform for cross-species integration and interrogation of heterogeneous curated and experimentally derived functional genomics data.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26092690',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 508,\n",
       "     'offsetInEndSection': 899,\n",
       "     'text': 'GeneWeaver enables researchers to store, share, analyze, and compare results of their own genome-wide functional genomics experiments in an environment containing rich companion data obtained from major curated repositories, including the Mouse Genome Database and other model organism databases, along with derived data from highly specialized resources, publications, and user submissions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26092690',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 180,\n",
       "     'text': 'The GeneWeaver data and analytics website (www.geneweaver.org) is a publically available resource for storing, curating and analyzing sets of genes from heterogeneous data sources.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26656951',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1030,\n",
       "     'offsetInEndSection': 1200,\n",
       "     'text': 'GeneWeaver empowers users with the ability to construct data driven descriptions of shared and unique biological processes, diseases and traits within and across species.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26656951',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 420,\n",
       "     'offsetInEndSection': 602,\n",
       "     'text': 'Our interactive web-based software system, Gene Weaver (http://www.geneweaver.org), couples curated results from genomic studies to graph-theoretical tools for combinatorial analysis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26834590',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 388,\n",
       "     'offsetInEndSection': 533,\n",
       "     'text': 'GeneWeaver is an online database and suite of tools at www.geneweaver.org that allows for fast aggregation and analysis of gene set-centric data.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1003,\n",
       "     'offsetInEndSection': 1187,\n",
       "     'text': 'Here we use GeneWeaver to find genes common to multiple gene sets, prioritize candidate genes from a quantitative trait locus, and characterize a set of differentially expressed genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 729,\n",
       "     'offsetInEndSection': 841,\n",
       "     'text': 'These can be entered directly into GeneWeaver or transferred from widely used resources such as GeneNetwork.org.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 534,\n",
       "     'offsetInEndSection': 728,\n",
       "     'text': 'GeneWeaver contains curated experimental data together with resource-level data such as GO annotations, MP annotations, and KEGG pathways, along with persistent stores of user entered data sets.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 874,\n",
       "     'offsetInEndSection': 1200,\n",
       "     'text': 'By enumerating the common and distinct biological molecules associated with all subsets of curated or user submitted groups of gene sets and gene networks, GeneWeaver empowers users with the ability to construct data driven descriptions of shared and unique biological processes, diseases and traits within and across species.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26656951',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 186,\n",
       "     'text': 'The GeneWeaver bipartite data model provides an efficient means to evaluate shared molecular components from sets derived across diverse species, disease states and biological processes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25374613',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 860,\n",
       "     'offsetInEndSection': 1049,\n",
       "     'text': 'Interrogating common relationships among biological networks and conventional GeneWeaver gene lists will increase functional specificity and reliability of the shared biological components.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25374613',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 420,\n",
       "     'offsetInEndSection': 603,\n",
       "     'text': 'Our interactive web-based software system, Gene Weaver (http://www.geneweaver.org), couples curated results from genomic studies to graph-theoretical tools for combinatorial analysis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26834590',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1581,\n",
       "     'offsetInEndSection': 1757,\n",
       "     'text': 'A robust implementation has been incorporated into GeneWeaver, an online tool for integrating and analyzing functional genomics experiments, available at http://geneweaver.org.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24731198',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 777,\n",
       "     'offsetInEndSection': 959,\n",
       "     'text': 'The GeneWeaver web-based software system brings together a large data archive from diverse functional genomics data with a suite of combinatorial tools in an interactive environment.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24233775',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 159,\n",
       "     'offsetInEndSection': 361,\n",
       "     'text': 'Data from multiple such publications and public data resources have been incorporated in the GeneWeaver database with >60,000 gene sets including 285 alcohol withdrawal and preference-related gene sets.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24923803',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 317,\n",
       "     'offsetInEndSection': 526,\n",
       "     'text': 'Data from multiple such publications and public data resources have been incorporated in the GeneWeaver database with &amp;gt;60,000 gene sets including 285 alcohol withdrawal and preference-related gene sets. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24923803',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 800,\n",
       "     'offsetInEndSection': 912,\n",
       "     'text': 'These can be entered directly into GeneWeaver or transferred from widely used resources such as GeneNetwork.org. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1878,\n",
       "     'offsetInEndSection': 2054,\n",
       "     'text': 'A robust implementation has been incorporated into GeneWeaver, an online tool for integrating and analyzing functional genomics experiments, available at http://geneweaver.org. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24731198',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 939,\n",
       "     'offsetInEndSection': 1300,\n",
       "     'text': 'The freely available GeneWeaver (http://www.GeneWeaver.org) powered by the Ontological Discovery Environment is a curated repository of genomic experimental results with an accompanying tool set for dynamic integration of these data sets, enabling users to interactively address questions about sets of biological functions and their relations to sets of genes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22080549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 512,\n",
       "     'offsetInEndSection': 661,\n",
       "     'text': 'GeneWeaver.org is a web-based software system that employs many of these techniques and has been used in the study of complex behavior and addiction. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23195309',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 263,\n",
       "     'text': 'GeneWeaver: data driven alignment of cross-species genomics in biology and disease.The GeneWeaver data and analytics website (www.geneweaver.org) is a publically available resource for storing, curating and analyzing sets of genes from heterogeneous data sources. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26656951',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 605,\n",
       "     'offsetInEndSection': 799,\n",
       "     'text': 'GeneWeaver contains curated experimental data together with resource-level data such as GO annotations, MP annotations, and KEGG pathways, along with persistent stores of user entered data sets. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 363,\n",
       "     'offsetInEndSection': 645,\n",
       "     'text': \"Since the previous review in the 2012 Nucleic Acids Research Database issue, GeneWeaver's underlying analytics platform has been enhanced, its number and variety of publically available gene set data sources has been increased, and its advanced search mechanisms have been expanded.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26656951',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 707,\n",
       "     'offsetInEndSection': 967,\n",
       "     'text': \"A search of over 60,000 gene sets in GeneWeaver's database revealed alcohol-related experimental results including genes identified in mouse genetic mapping studies, alcohol selected Drosophila lines, Rattus differential expression, and human alcoholic brains.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26834590',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1607,\n",
       "     'offsetInEndSection': 1783,\n",
       "     'text': 'A robust implementation has been incorporated into GeneWeaver, an online tool for integrating and analyzing functional genomics experiments, available at http://geneweaver.org.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24731198',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 71,\n",
       "     'text': 'Integrative Functional Genomics for Systems Genetics in GeneWeaver.org.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27933523',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'There is no drug available to prevent HIV infection, Pre-exposure prophylaxis (PrEP), yes or no?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/26746652',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27900502',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28370177',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21799568',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29278542',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27177804',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28657199',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23570850',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23972284',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25987851'],\n",
       "   'ideal_answer': ['Pre-exposure prophylaxis (PrEP) with the drug combination Truvada can substantially decrease HIV transmission in individuals at risk.',\n",
       "    'The antiviral agent tenofovir is highly effective for the treatment of HIV and hepatitis B virus infections, and the older prodrug tenofovir disoproxil fumarate (TDF) is also a component of daily preexposure prophylaxis (PrEP) to reduce the risk of HIV infection in high-risk populations.'],\n",
       "   'exact_answer': 'no',\n",
       "   'type': 'yesno',\n",
       "   'id': '5c5f2cef1a4c55d80b000022',\n",
       "   'snippets': [{'offsetInBeginSection': 26,\n",
       "     'offsetInEndSection': 169,\n",
       "     'text': 'pre-exposure prophylaxis with generic tenofovir disoproxil fumarate/emtrictabine in London - analysis of pharmacokinetics, safety and outcomes.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28657199',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 19,\n",
       "     'offsetInEndSection': 308,\n",
       "     'text': 'The antiviral agent tenofovir is highly effective for the treatment of HIV and hepatitis B virus infections, and the older prodrug tenofovir disoproxil fumarate (TDF) is also a component of daily preexposure prophylaxis (PrEP) to reduce the risk of HIV infection in high-risk populations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29278542',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 352,\n",
       "     'text': 'Following US Food and Drugs Administration approval in July 2012 of daily oral tenofovir and emtricitabine for pre-exposure prophylaxis (PrEP) to prevent HIV infection in high-risk individuals in the USA, there has been much controversy about the implementation of this PrEP regimen in other countries throughout the world, and in Europe in particular.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23972284',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': 'Daily oral pre-exposure prophylaxis (PrEP) is the use of antiretroviral drugs by HIV-negative people to prevent HIV infection.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27900502',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 154,\n",
       "     'offsetInEndSection': 332,\n",
       "     'text': 'HIV pre-exposure prophylaxis (PrEP) is a new approach that involves the ongoing use of antiretroviral medications by HIV-negative individuals to reduce the risk of HIV infection.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25987851',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1107,\n",
       "     'offsetInEndSection': 1254,\n",
       "     'text': 'CONCLUSIONS Combined ART + PrEP is likely to prevent more HIV infections than either strategy alone, but with higher prevalence of drug resistance.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23570850',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 151,\n",
       "     'text': 'INTRODUCTION Use of pre-exposure prophylaxis (PrEP) among people who inject drugs (PWID) has been shown to be effective in preventing HIV transmission.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27177804',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 239,\n",
       "     'text': 'Pre-exposure prophylaxis (PrEP) is an experimental approach to HIV prevention and consists of antiretroviral drugs to be taken before potential HIV exposure in order to reduce the risk of HIV infection and continued during periods of risk.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21799568',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'HIV pre-exposure prophylaxis (PrEP) is the use of one or more antiretroviral medications (in combination) to prevent HIV infection.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28370177',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 132,\n",
       "     'offsetInEndSection': 183,\n",
       "     'text': 'The most commonly used PrEP medication (Truvada<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28370177',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 102,\n",
       "     'text': 'The use of antiretrovirals as pre-exposure prophylaxis (PrEP) is highly efficacious in HIV prevention.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26746652',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'List two indications of Letermovir?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28675594',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30004790',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29578577',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28967706',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29290671',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29746444',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29914965',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29361041',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29288370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29998227'],\n",
       "   'ideal_answer': ['Letermovir is approved for the prophylaxis of CMV infection and disease in adult CMV-seropositive recipients of an allogeneic haematopoietic stem cell transplant (HSCT).'],\n",
       "   'exact_answer': [['prophylaxis of CMV infection'],\n",
       "    ['CMV-seropositive recipients of an allogeneic haematopoietic stem cell transplant']],\n",
       "   'type': 'list',\n",
       "   'id': '5c6e01af7c78d69471000048',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 307,\n",
       "     'text': 'Latanoprostene bunod ophthalmic solution (Vyzulta) for the reduction of intraocular pressure; letermovir (Prevymis) for prophylaxis of cytomegalovirus infection and disease after stem cell transplant; benralizumab (Fasenra) for the add-on maintenance treatment of severe asthma of an eosinophilic phenotype.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29290671',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 221,\n",
       "     'offsetInEndSection': 418,\n",
       "     'text': 'Letermovir has been approved in Canada and the USA for the prophylaxis of CMV infection and disease in adult CMV-seropositive recipients of an allogeneic haematopoietic stem cell transplant (HSCT).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29288370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 619,\n",
       "     'offsetInEndSection': 865,\n",
       "     'text': 'This article summarizes the milestones in the development of letermovir leading to its first global approval in Canada as well as the USA for the prophylaxis of CMV infection and disease in adult CMV-seropositive recipients of an allogeneic HSCT.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29288370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 680,\n",
       "     'offsetInEndSection': 828,\n",
       "     'text': 'A new terminase inhibitor, letermovir, recently proved effective against HCMV in phase III clinical trials, but the mechanism of action is unclear. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29361041',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 93,\n",
       "     'text': 'Letermovir is being developed for human cytomegalovirus infection treatment and prophylaxis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28967706',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': 'Letermovir is a human cytomegalovirus terminase inhibitor for cytomegalovirus infection prophylaxis in hematopoietic stem cell transplant recipients. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29578577',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 121,\n",
       "     'text': 'Role of letermovir for prevention of cytomegalovirus infection after allogeneic haematopoietic stem cell transplantation.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29746444',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 369,\n",
       "     'offsetInEndSection': 766,\n",
       "     'text': 'In this article, the role of the newly approved antiviral compound, letermovir, is reviewed.RECENT FINDINGS: Letermovir inhibits CMV by interfering viral terminase complex. In a phase 3 randomized placebo-controlled clinical study that enrolled 495 CMV-seropositive HSCT recipients, the primary end point of clinically significant CMV infection was significantly reduced by letermovir prophylaxis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29746444',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1054,\n",
       "     'offsetInEndSection': 1167,\n",
       "     'text': 'SUMMARY: Letermovir is an important addition to the current strategies for CMV prevention after allogeneic HSCT. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29746444',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 132,\n",
       "     'text': 'Letermovir is a human cytomegalovirus (CMV) terminase inhibitor recently approved as prophylaxis in stem cell transplant recipients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29914965',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 145,\n",
       "     'text': 'Letermovir is a novel antiviral in clinical development for prophylaxis against human cytomegalovirus in immunocompromised transplant recipients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28675594',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1106,\n",
       "     'offsetInEndSection': 1334,\n",
       "     'text': 'Expert commentary: With the introduction of letermovir, prevention of CMV infection in allo-HCT recipients may shift considerably, from a predominantly preemptive strategy to one that utilizes this novel therapy for prophylaxis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30004790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 688,\n",
       "     'offsetInEndSection': 913,\n",
       "     'text': 'The positive results from the recently published Phase III study of letermovir for prevention of CMV infection in CMV-seropositive allo-HCT recipients led to its approval as a prophylactic agent for CMV in multiple countries.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30004790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1277,\n",
       "     'offsetInEndSection': 1445,\n",
       "     'text': 'Letermovir has shown promising clinical efficacy and is generally well tolerated, thus providing a favorable new option in the prophylaxis of CMV infection and disease.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29998227',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Describe the mechanism of action of apalutamide.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28636139',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29381490',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30426794',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29606109',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29420164',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29856649',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29695920',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29734647',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29626324',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27160947'],\n",
       "   'ideal_answer': ['Apalutamide is a second-generation antiandrogen that inhibits the binding of androgen to androgen receptor (AR), nuclear translocation of the androgen-AR complex, and binding of AR transcription complex to DNA-binding sites and transcription elements. It does not show antagonist-to-agonist switch like bicalutamide. It is emerging as additional new option to treat castration-resistant prostate cancer.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5c6dfba77c78d69471000046',\n",
       "   'snippets': [{'offsetInBeginSection': 440,\n",
       "     'offsetInEndSection': 614,\n",
       "     'text': 'Second-generation NSAA receptor antagonists (enzalutamide, apalutamide and darolutamide) are emerging as additional new options to treat castration-resistant prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28636139',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 133,\n",
       "     'text': 'BACKGROUND: Apalutamide, a competitive inhibitor of the androgen receptor, is under development for the treatment of prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29420164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 157,\n",
       "     'text': 'Apalutamide (ARN-509) is an antiandrogen that binds selectively to androgen receptors (AR) and does not show antagonist-to-agonist switch like bicalutamide. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29381490',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 320,\n",
       "     'offsetInEndSection': 415,\n",
       "     'text': 'Apalutamide is a pure androgen antagonist with a very high affinity with the androgen receptor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29606109',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 376,\n",
       "     'offsetInEndSection': 750,\n",
       "     'text': 'Apalutamide (ARN-509), a second-generation androgen receptor antagonist, recently received approval in the nonmetastatic (M0) CRPC space. Similar to enzalutamide, apalutamide inhibits the binding of androgen to androgen receptor (AR), nuclear translocation of the androgen-AR complex, and binding of AR transcription complex to DNA-binding sites and transcription elements. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29695920',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 255,\n",
       "     'text': 'Apalutamide (ErleadaTM) is a next-generation oral androgen receptor (AR) inhibitor that is being developed by Janssen for the treatment of prostate cancer (PC). It binds directly to the ligand-binding domain of the AR and blocks the effects of androgens. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29626324',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 677,\n",
       "     'offsetInEndSection': 1079,\n",
       "     'text': 'Ongoing clinical trials for different subgroups of prostate cancer patients include the evaluation of the second-generation AR antagonists enzalutamide, apalutamide and darolutamide, of inhibitors of the phosphatidylinositol-4,5-bisphosphate 3-kinase (PI3K) pathway, of inhibitors of DNA damage response, of targeted alpha therapy and of prostate-specific membrane antigen (PSMA) targeting approaches. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29734647',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 395,\n",
       "     'offsetInEndSection': 502,\n",
       "     'text': ' One of these agents is apalutamide, which seems to be a promising AR antagonist for the treatment of CRPC.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29856649',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 719,\n",
       "     'offsetInEndSection': 921,\n",
       "     'text': 'Expert opinion: Apalutamide is an oral, investigational, AR antagonist that targets the AR ligand-binding domain and prevents AR nuclear translocation, DNA binding, and transcription of AR gene targets.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29856649',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 191,\n",
       "     'text': 'Phase 2 Study of the Safety and Antitumor Activity of Apalutamide (ARN-509), a Potent Androgen Receptor Antagonist, in the High-risk Nonmetastatic Castration-resistant Prostate Cancer Cohort.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27160947',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 200,\n",
       "     'text': 'BACKGROUND\\nApalutamide is a potent androgen receptor (AR) antagonist that targets the AR ligand-binding domain and prevents AR nuclear translocation, DNA binding, and transcription of AR gene targets.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27160947',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 132,\n",
       "     'text': 'BACKGROUND\\nApalutamide, a competitive inhibitor of the androgen receptor, is under development for the treatment of prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29420164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 319,\n",
       "     'offsetInEndSection': 414,\n",
       "     'text': 'Apalutamide is a pure androgen antagonist with a very high affinity with the androgen receptor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29606109',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 514,\n",
       "     'offsetInEndSection': 749,\n",
       "     'text': 'Similar to enzalutamide, apalutamide inhibits the binding of androgen to androgen receptor (AR), nuclear translocation of the androgen-AR complex, and binding of AR transcription complex to DNA-binding sites and transcription elements.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29695920',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 376,\n",
       "     'offsetInEndSection': 513,\n",
       "     'text': 'Apalutamide (ARN-509), a second-generation androgen receptor antagonist, recently received approval in the nonmetastatic (M0) CRPC space.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29695920',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 160,\n",
       "     'text': 'Apalutamide (ErleadaTM) is a next-generation oral androgen receptor (AR) inhibitor that is being developed by Janssen for the treatment of prostate cancer (PC).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29626324',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 705,\n",
       "     'offsetInEndSection': 907,\n",
       "     'text': 'Expert opinion: Apalutamide is an oral, investigational, AR antagonist that targets the AR ligand-binding domain and prevents AR nuclear translocation, DNA binding, and transcription of AR gene targets.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29856649',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 216,\n",
       "     'offsetInEndSection': 349,\n",
       "     'text': 'Apalutamide, an androgen receptor inhibitor, has substantial clinical response in nonmetastatic castration-resistant prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30426794',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 200,\n",
       "     'text': 'BACKGROUND Apalutamide is a potent androgen receptor (AR) antagonist that targets the AR ligand-binding domain and prevents AR nuclear translocation, DNA binding, and transcription of AR gene targets.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27160947',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 132,\n",
       "     'text': 'BACKGROUND Apalutamide, a competitive inhibitor of the androgen receptor, is under development for the treatment of prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29420164',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the role of metalloproteinase-17 (ADAM17) in NK cells?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23625205',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24941379',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17510296',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30009514',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25816339',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19395875',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21480393',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27077118',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20871631',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29978334'],\n",
       "   'ideal_answer': ['The metalloproteinase-17 (ADAM17) is involved in CD16A cleavage and acts as a regulatory checkpoint in NK cells'],\n",
       "   'type': 'summary',\n",
       "   'id': '5c6ffc5b7c78d69471000057',\n",
       "   'snippets': [{'offsetInBeginSection': 953,\n",
       "     'offsetInEndSection': 1142,\n",
       "     'text': ' To address this concern, bispecific or trispecific engagers that target NK cells to the tumour and an ADAM17 inhibitor that prevents CD16 shedding after NK cell activation are being tested',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30009514',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1356,\n",
       "     'offsetInEndSection': 1566,\n",
       "     'text': 'ese findings support ADAM17 as a dynamic inhibitory checkpoint of the potent activating receptor CD16A, which can be targeted by MEDI3622 to potentially increase the efficacy of anti-tumor therapeutic antibodie',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29978334',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1148,\n",
       "     'offsetInEndSection': 1320,\n",
       "     'text': 'Our findings provide further characterization of CD16 cleavage by ADAM17 and they demonstrate that a non-cleavable version of CD16a can be expressed in engineered NK cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25816339',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 258,\n",
       "     'text': 'CD16a and CD16b are IgG Fc receptors expressed by human natural killer (NK) cells and neutrophils, respectively. Both CD16 isoforms undergo a rapid down-regulation in expression by ADAM17-mediated proteolytic cleavage upon cell activation by various stimuli.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25816339',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 728,\n",
       "     'offsetInEndSection': 1013,\n",
       "     'text': 'Bispecific or trispecific killer engagers that target CD16 on NK cells to enhance recognition of tumor antigens, and desintegrin and metalloproteinase 17 (ADAM17) inhibition that prevents CD16 shedding after NK-cell activation should promote enhanced killing of cancer with specificity',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24941379',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': 'Role of ADAM17 in the ectodomain shedding of TNF-alpha and its receptors by neutrophils and macrophages.TNF-alpha and its receptors TNFRI and TNFRII are cleaved from the surface of leukocytes by a proteolytic process referred to as ectodomain shedding. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17510296',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': 'A disintegrin and metalloproteinase-17 (ADAM17) is involved in proteolytic ectodomain shedding of several membrane-bound growth factors and cytokines.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21480393',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 150,\n",
       "     'text': 'A disintegrin and metalloproteinase-17 (ADAM17) is involved in proteolytic ectodomain shedding of several membrane-bound growth factors and cytokines.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19395875',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 171,\n",
       "     'text': 'A disintegrin and metalloproteinase-17 (ADAM17) is a member of the metalloproteinase superfamily and involved in the cleavage of ectodomain of many transmembrane proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23625205',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 728,\n",
       "     'offsetInEndSection': 1014,\n",
       "     'text': 'Bispecific or trispecific killer engagers that target CD16 on NK cells to enhance recognition of tumor antigens, and desintegrin and metalloproteinase 17 (ADAM17) inhibition that prevents CD16 shedding after NK-cell activation should promote enhanced killing of cancer with specificity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24941379',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 285,\n",
       "     'text': 'ADAM17 (a disintegrin and metalloproteinase 17, also referred to as TNFα converting enzyme or TACE) is a cell-surface metalloproteinase that regulates signaling via the epidermal growth factor receptor (EGFR) and has important roles in diseases such as cancer and rheumatoid arthritis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20871631',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 88,\n",
       "     'offsetInEndSection': 360,\n",
       "     'text': 'Here we demonstrate that a disintegrin and metalloproteinase-17 (ADAM17), a main sheddase for tumor necrosis factor (TNF)-α, is essential for defensive epithelial properties against UC by promoting epithelial cell growth and goblet cell differentiation in mouse and human.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27077118',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is Lasmiditan effective for migraine?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29488143',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29352859',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22430431',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22459549',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29563831',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20855362',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28103158',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30323656',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29098075'],\n",
       "   'ideal_answer': ['Yes, Lasmiditan is effective for treatment of migraine. This has been demonstrated in clinical trials.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '5c6f6ae37c78d69471000054',\n",
       "   'snippets': [{'offsetInBeginSection': 1191,\n",
       "     'offsetInEndSection': 1370,\n",
       "     'text': 'Amongst the ditans, lasmiditan: (i) fails to constrict human coronary arteries; and (ii) is effective for the acute treatment of migraine in preliminary Phase III clinical trials.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29352859',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1443,\n",
       "     'offsetInEndSection': 1634,\n",
       "     'text': 'Although ongoing phase III clinical trials are needed to confirm its efficacy and safety, lasmiditan might offer an alternative to treat acute migraine with no associated cardiovascular risk.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29488143',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1136,\n",
       "     'offsetInEndSection': 1255,\n",
       "     'text': 'Lasmiditan is considered to be the first member of a new drug category, the neurally acting anti-migraine agent (NAAMA)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29563831',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 97,\n",
       "     'text': 'Lasmiditan for the treatment of acute migraine: a review and potential role in clinical practice.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30323656',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 523,\n",
       "     'offsetInEndSection': 951,\n",
       "     'text': 'Lasmiditan, a highly selective 5-HT1F agonist, has completed two Phase III randomized, double blind, placebo-controlled clinical trials, with a third - a long-term, open-label safety study - still underway. Research to date suggests lasmiditan lacks vasoconstrictive properties and may be a safe and effective treatment option in patients refractory to current acute migraine medications or who have cardiovascular risk factors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30323656',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 84,\n",
       "     'text': 'Lasmiditan is an effective acute treatment for migraine: A phase 3 randomized study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 640,\n",
       "     'offsetInEndSection': 1192,\n",
       "     'text': 'Compared with placebo, more patients dosed with lasmiditan 200 mg were free of headache pain at 2 hours after dosing (32.2% vs 15.3%; odds ratio [OR] 2.6, 95% confidence interval [CI] 2.0-3.6, p< 0.001), similar to those dosed with lasmiditan 100 mg (28.2%; OR 2.2, 95% CI 1.6-3.0, p< 0.001). Furthermore, compared with those dosed with placebo, more patients dosed with lasmiditan 200 mg (40.7% vs 29.5%; OR 1.6, 95% CI 1.3-2.1, p< 0.001) and lasmiditan 100 mg (40.9%; OR 1.7, 95% CI, 1.3-2.2, p< 0.001) were free of their MBS at 2 hours after dosing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1250,\n",
       "     'offsetInEndSection': 1431,\n",
       "     'text': 'CONCLUSIONS: Lasmiditan dosed at 200 and 100 mg was efficacious and well tolerated in the treatment of acute migraine among patients with a high level of cardiovascular risk factors',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1474,\n",
       "     'offsetInEndSection': 1697,\n",
       "     'text': 'CLASSIFICATION OF EVIDENCE: This study provides Class I evidence that for adult patients with migraine, lasmiditan increases the proportion of subjects who are headache pain free at 2 hours after treating a migraine attack.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1163,\n",
       "     'offsetInEndSection': 1350,\n",
       "     'text': 'For the understanding of migraine pathophysiology, it is very important to note that a selective 5-HT(1F) receptor agonist like lasmiditan is effective in the acute treatment of migraine. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22430431',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1430,\n",
       "     'offsetInEndSection': 1584,\n",
       "     'text': 'While lasmiditan most likely is effective in the treatment of migraine attacks it had, unfortunately, a high incidence of CNS related AEs in the oral RCT. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22430431',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 224,\n",
       "     'text': 'Acute treatment of migraine with the selective 5-HT1F receptor agonist lasmiditan--a randomised proof-of-concept trial.At intravenous doses of 20 mg and higher, lasmiditan proved effective in the acute treatment of migraine. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20855362',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 97,\n",
       "     'text': 'Lasmiditan for the treatment of acute migraine: a review and potential role in clinical practice. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30323656',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 235,\n",
       "     'text': 'BACKGROUND\\nLasmiditan (COL-144) is a novel, centrally acting, highly selective 5-HT(1F) receptor agonist without vasoconstrictor activity that seemed effective when given as an intravenous infusion in a proof-of-concept migraine study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22459549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2485,\n",
       "     'offsetInEndSection': 2582,\n",
       "     'text': 'INTERPRETATION\\nOral lasmiditan seems to be safe and effective in the acute treatment of migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22459549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 912,\n",
       "     'offsetInEndSection': 1124,\n",
       "     'text': 'The 5-HT1F receptor agonist lasmiditan, a drug acting through non-vasoconstrictive mechanisms, represents a promising safe, effective and tolerated acute migraine therapy also for patients at cardiovascular risk.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28103158',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1013,\n",
       "     'offsetInEndSection': 1200,\n",
       "     'text': 'For the understanding of migraine pathophysiology, it is very important to note that a selective 5-HT(1F) receptor agonist like lasmiditan is effective in the acute treatment of migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22430431',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 136,\n",
       "     'text': 'The 5-HT1F receptor agonist lasmiditan as a potential treatment of migraine attacks: a review of two placebo-controlled phase II trials.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22430431',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 647,\n",
       "     'offsetInEndSection': 1016,\n",
       "     'text': 'Within the past few years, new and promising drugs such as more specific 5-HT 1F receptor agonists (that is, lasmiditan) and monoclonal calcitonin gene-related peptide (CGRP) receptor antibodies entered advanced development phases while non-invasive neuromodulatory approaches were suggested to be potentially effective as non-pharmaceutical interventions for migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29098075',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1484,\n",
       "     'offsetInEndSection': 1706,\n",
       "     'text': 'CLASSIFICATION OF EVIDENCE\\nThis study provides Class I evidence that for adult patients with migraine, lasmiditan increases the proportion of subjects who are headache pain free at 2 hours after treating a migraine attack.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 235,\n",
       "     'text': 'BACKGROUND Lasmiditan (COL-144) is a novel, centrally acting, highly selective 5-HT(1F) receptor agonist without vasoconstrictor activity that seemed effective when given as an intravenous infusion in a proof-of-concept migraine study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22459549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2485,\n",
       "     'offsetInEndSection': 2582,\n",
       "     'text': 'INTERPRETATION Oral lasmiditan seems to be safe and effective in the acute treatment of migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22459549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 730,\n",
       "     'offsetInEndSection': 951,\n",
       "     'text': 'Research to date suggests lasmiditan lacks vasoconstrictive properties and may be a safe and effective treatment option in patients refractory to current acute migraine medications or who have cardiovascular risk factors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30323656',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2310,\n",
       "     'offsetInEndSection': 2510,\n",
       "     'text': 'The non-vascular, neural mechanism of action of lasmiditan may offer an alternative means to treat migraine especially in patients who have contra-indications for agents with vasoconstrictor activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20855362',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 669,\n",
       "     'offsetInEndSection': 1346,\n",
       "     'text': 'Compared with placebo, more patients dosed with lasmiditan 200 mg were free of headache pain at 2 hours after dosing (32.2% vs 15.3%; odds ratio [OR] 2.6, 95% confidence interval [CI] 2.0-3.6, <br><b>CONCLUSIONS</b>: Lasmiditan dosed at 200 and 100 mg was efficacious and well tolerated in the treatment of acute migraine among patients with a high level of cardiovascular risk factors.<br><b>CLINICALTRIALSGOV IDENTIFIER</b>: NCT02439320.<br><b>CLASSIFICATION OF EVIDENCE</b>: This study provides Class I evidence that for adult patients with migraine, lasmiditan increases the proportion of subjects who are headache pain free at 2 hours after treating a migraine attack.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2389,\n",
       "     'offsetInEndSection': 2617,\n",
       "     'text': 'The most common adverse events were CNS related and included dizziness, fatigue, vertigo, paraesthesia, and somnolence.<br><b>INTERPRETATION</b>: Oral lasmiditan seems to be safe and effective in the acute treatment of migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22459549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2012,\n",
       "     'offsetInEndSection': 2238,\n",
       "     'text': 'Dizziness, paresthesia and sensations of heaviness (usually limb) were more common on lasmiditan.<br><b>CONCLUSIONS</b>: At intravenous doses of 20 mg and higher, lasmiditan proved effective in the acute treatment of migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20855362',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 243,\n",
       "     'text': '<b>BACKGROUND</b>: Lasmiditan (COL-144) is a novel, centrally acting, highly selective 5-HT(1F) receptor agonist without vasoconstrictor activity that seemed effective when given as an intravenous infusion in a proof-of-concept migraine study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22459549',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1019,\n",
       "     'offsetInEndSection': 1206,\n",
       "     'text': 'For the understanding of migraine pathophysiology, it is very important to note that a selective 5-HT(1F) receptor agonist like lasmiditan is effective in the acute treatment of migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22430431',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1415,\n",
       "     'offsetInEndSection': 1610,\n",
       "     'text': 'This study provides Class I evidence that for adult patients with migraine, lasmiditan increases the proportion of subjects who are headache pain free at 2 hours after treating a migraine attack.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30446595',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1286,\n",
       "     'offsetInEndSection': 1440,\n",
       "     'text': 'While lasmiditan most likely is effective in the treatment of migraine attacks it had, unfortunately, a high incidence of CNS related AEs in the oral RCT.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22430431',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What the chromsomal location of the gene that is deleted in Potocki-Shaffer syndrome?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/23239541',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15852040',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28571721',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22822387',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28127865',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15666301',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20140962',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26333423',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19222835',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25653495'],\n",
       "   'ideal_answer': ['In Potocki-Shaffer syndrome (PSS), the full phenotypic spectrum is manifested when deletions are at least 2.1\\u2009Mb in size at 11p11.2',\n",
       "    'Potocki-Shaffer syndrome (PSS) is a rare contiguous gene deletion syndrome caused by heterozygous deletion of 11p11.2p12.'],\n",
       "   'exact_answer': ['11p11.2p12'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5c72b7277c78d69471000073',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'In Potocki-Shaffer syndrome (PSS), the full phenotypic spectrum is manifested when deletions are at least 2.1\\u2009Mb in size at 11p11.2',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26333423',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 153,\n",
       "     'text': 'Potocki-Shaffer syndrome (PSS, OMIM #601224) is a rare contiguous gene deletion syndrome caused by haploinsufficiency of genes located on the 11p11.2p12.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25653495',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 187,\n",
       "     'text': 'Potocki-Shaffer syndrome (PSS) is a contiguous gene deletion syndrome that results from haploinsufficiency of at least two genes within the short arm of chromosome 11[del(11)(p11.2p12)]. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15852040',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 136,\n",
       "     'text': 'Construction of a natural panel of 11p11.2 deletions and further delineation of the critical region involved in Potocki-Shaffer syndrome',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15852040',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 262,\n",
       "     'text': 'Potocki-Shaffer syndrome is a rare contiguous gene deletion syndrome due to haploinsufficiency of the 11p11.2p12 region and is characterized by craniofacial abnormalities, developmental delay, intellectual disability, multiple exostoses, and biparietal foramina.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20140962',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 566,\n",
       "     'offsetInEndSection': 710,\n",
       "     'text': 'This is also the first report describing deletion of 11p11.12-p11.2 and neocentromere formation resulting in inherited Potocki-Shaffer syndrome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15666301',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 186,\n",
       "     'text': 'Potocki-Shaffer syndrome (PSS) is a contiguous gene deletion syndrome that results from haploinsufficiency of at least two genes within the short arm of chromosome 11[del(11)(p11.2p12)].',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15852040',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'Potocki-Shaffer Syndrome is a rare neurodevelopmental syndrome associated with microdeletion of a region of Chromosome 11p11.2.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28571721',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 275,\n",
       "     'text': 'Potocki-Shaffer syndrome is a contiguous gene deletion syndrome involving 11p11.2p12 and characterized by multiple exostoses, biparietal foramina, genitourinary anomalies in males, central nervous system abnormalities, intellectual disability, and craniofacial abnormalities.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28127865',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'Interstitial deletion 11(p11.12p11.2) and analphoid marker formation results in inherited Potocki-Shaffer syndrome.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15666301',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 152,\n",
       "     'text': 'Potocki-Shaffer syndrome (PSS) is a rare disorder caused by haploinsufficiency of genes located on the proximal short arm of chromosome 11 (11p11.2p12).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23239541',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 323,\n",
       "     'text': 'Construction of a natural panel of 11p11.2 deletions and further delineation of the critical region involved in Potocki-Shaffer syndrome.Potocki-Shaffer syndrome (PSS) is a contiguous gene deletion syndrome that results from haploinsufficiency of at least two genes within the short arm of chromosome 11[del(11)(p11.2p12)]. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15852040',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 609,\n",
       "     'offsetInEndSection': 785,\n",
       "     'text': 'The deletion (11p13-p12) located in the area between the deletions associated with the WAGR and Potocki-Shaffer syndromes had a maximum size of 8.5 Mb and encompasses 44 genes. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19222835',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 333,\n",
       "     'text': 'Delayed Diagnosis of Potocki-Shaffer Syndrome in a Woman with Multiple Exostoses and Mental Retardation.We describe the case of an adult patient affected by multiple exostoses, severe mental retardation, epilepsy and facial dysmorphisms with a deletion of ∼2.3 Mb on chromosome 11p11.21, correlated to Potocki-Shaffer syndrome (PSS). ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22822387',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 566,\n",
       "     'offsetInEndSection': 714,\n",
       "     'text': 'This is also the first report describing deletion of 11p11.12-p11.2 and neocentromere formation resulting in inherited Potocki-Shaffer syndrome.<br>',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15666301',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 359,\n",
       "     'text': \" WAGR syndrome (Wilms' tumor, aniridia, genitourinary abnormalities and mental retardation) and Potocki-Shaffer syndrome are rare contiguous gene deletion syndromes caused by deletions of the 11p14-p12 chromosome region.We present a patient with mental retardation, unilateral cataract, bilateral ptosis, genital abnormalities, seizures and a dysmorphic face.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19222835',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 467,\n",
       "     'offsetInEndSection': 643,\n",
       "     'text': 'The deletion (11p13-p12) located in the area between the deletions associated with the WAGR and Potocki-Shaffer syndromes had a maximum size of 8.5 Mb and encompasses 44 genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19222835',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Please list 2 antitoxin antibodies approved by the FDA for reducing the recurrence of Clostridium difficile infection',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28232034',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27905086',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29760164',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28636484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28480750',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28865041',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29234211',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25486992',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28730660',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27757389'],\n",
       "   'ideal_answer': ['The antitoxin antibodies actoxumab and bezlotoxumab bind to and neutralize TcdA and TcdB, respectively. Bezlotoxumab was recently approved by the FDA for reducing the recurrence of CDI.'],\n",
       "   'exact_answer': [['bezlotoxumab'], ['actoxumab']],\n",
       "   'type': 'list',\n",
       "   'id': '5c851c2675a4a5d219000006',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 297,\n",
       "     'text': 'The exotoxins toxin A (TcdA) and toxin B (TcdB) are produced by the bacterial pathogen Clostridium difficile and are responsible for the pathology associated with C. difficile infection (CDI). The antitoxin antibodies actoxumab and bezlotoxumab bind to and neutralize TcdA and TcdB, respectively. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28232034',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 297,\n",
       "     'offsetInEndSection': 378,\n",
       "     'text': 'Bezlotoxumab was recently approved by the FDA for reducing the recurrence of CDI.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28232034',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 164,\n",
       "     'text': 'The Monoclonal Antitoxin Antibodies (Actoxumab-Bezlotoxumab) Treatment Facilitates Normalization of the Gut Microbiota of Mice with Clostridium difficile Infection.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27757389',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 193,\n",
       "     'offsetInEndSection': 296,\n",
       "     'text': 'The antitoxin antibodies actoxumab and bezlotoxumab bind to and neutralize TcdA and TcdB, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28232034',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 131,\n",
       "     'text': 'Mechanisms of protection against Clostridium difficile infection by the monoclonal antitoxin antibodies actoxumab and bezlotoxumab.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25486992',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 104,\n",
       "     'text': 'Bezlotoxumab: anti-toxin B monoclonal antibody to prevent recurrence of Clostridium difficile infection.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28636484',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 118,\n",
       "     'text': 'Bezlotoxumab (Zinplava™) is a human monoclonal antibody against Clostridium difficile toxin B developed by Merck & Co.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27905086',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 212,\n",
       "     'text': 'Bezlotoxumab (Zinplava™) is a fully human monoclonal antibody against Clostridium difficile toxin B indicated for the prevention of C. difficile infection (CDI) recurrence in patients with a high recurrence risk.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28865041',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 122,\n",
       "     'text': 'Bezlotoxumab (Zinplava™) is a human monoclonal antibody against Clostridium difficile toxin B developed by Merck &amp; Co.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27905086',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which diseases are treated with netarsudil?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29199013',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28609185',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30323550',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27072905',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29469601',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30007591',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29622939',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30209441',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30028313',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29453668'],\n",
       "   'ideal_answer': ['In 2 large, randomized, double-masked trials, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma.'],\n",
       "   'exact_answer': [['ocular hypertension'], ['open-angle glaucoma']],\n",
       "   'type': 'list',\n",
       "   'id': '5c73ad0f7c78d69471000096',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 207,\n",
       "     'text': 'Two Phase 3 Clinical Trials Comparing the Safety and Efficacy of Netarsudil to Timolol in Patients With Elevated Intraocular Pressure: Rho Kinase\\xa0Elevated IOP Treatment Trial 1 and 2 (ROCKET-1 and ROCKET-2).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29199013',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1449,\n",
       "     'offsetInEndSection': 1685,\n",
       "     'text': 'CONCLUSIONS: In 2 large, randomized, double-masked trials reported here, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29199013',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 439,\n",
       "     'text': 'Netarsudil ophthalmic solution 0.02% (hereafter referred to as netarsudil 0.02%) [Rhopressa®] is a Rho-associated protein kinase inhibitor that is thought to lower intraocular pressure (IOP) by increasing aqueous humour outflow through the trabecular meshwork. It has been developed by Aerie Pharmaceuticals and was recently approved in the USA for the reduction of elevated IOP in patients with open-angle glaucoma or ocular hypertension.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29453668',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': 'PURPOSE: Netarsudil, an inhibitor of Rho kinase and a norepinephrine transporter, has been shown to lower elevated intraocular pressure (IOP) in controlled studies of patients with open-angle glaucoma and ocular hypertension, and in healthy volunteers.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29469601',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1501,\n",
       "     'offsetInEndSection': 1648,\n",
       "     'text': 'CONCLUSIONS: Once-daily dosing of netarsudil ophthalmic solution 0.02% lowered IOP through increasing trabecular outflow facility and reducing EVP.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29469601',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': 'Netarsudil ophthalmic solution (Rhopressa) for the reduction of elevated intraocular pressure; ertugliflozin (Steglatro) for adults with type-2 diabetes; and lutetium lu 177 dotatate (Lutathera) for certain gastroenteropancreatic neuroendocrine tumors.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29622939',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1209,\n",
       "     'offsetInEndSection': 1452,\n",
       "     'text': 'Two recently approved multi-pharmacophoric drugs (e.g., rho kinase inhibitor, Netarsudil; a drug conjugate, Latanoprostene Bunod) and novel aqueous humor drainage devices (iStent and CyPass) are also gaining acceptance for treating POAG/ NTG. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30028313',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 513,\n",
       "     'offsetInEndSection': 764,\n",
       "     'text': 'Ultimately, a search for Rho kinase inhibitors led to the discovery of several molecules of therapeutic interest, leaving us today with 2 new ocular hypotensive agents approved for clinical use: ripasudil in Japan and netarsudil in the United States. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30007591',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'Netarsudil ophthalmic solution 0.02% for the treatment of patients with open-angle glaucoma or ocular hypertension.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30209441',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 210,\n",
       "     'text': 'Once-daily (p.m.) netarsudil ophthalmic solution 0.02% (Rhopressa) is approved in the United States for lowering elevated intraocular pressure (IOP) in patients with open-angle glaucoma or ocular hypertension. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30209441',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1457,\n",
       "     'offsetInEndSection': 1691,\n",
       "     'text': 'CONCLUSIONS\\nIn 2 large, randomized, double-masked trials reported here, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29199013',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 116,\n",
       "     'text': 'Discovery and Preclinical Development of Netarsudil, a Novel Ocular Hypotensive Agent for the Treatment of Glaucoma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28609185',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 234,\n",
       "     'text': 'Netarsudil ophthalmic solution is a novel topical intraocular pressure (IOP)-lowering agent that has recently been approved by the US Food and Drug Administration (FDA) for the treatment of ocular hypertension and open-angle glaucoma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30323550',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 209,\n",
       "     'text': 'Once-daily (p.m.) netarsudil ophthalmic solution 0.02% (Rhopressa) is approved in the United States for lowering elevated intraocular pressure (IOP) in patients with open-angle glaucoma or ocular hypertension.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30209441',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 593,\n",
       "     'offsetInEndSection': 796,\n",
       "     'text': 'This article summarizes the milestones in the development of netarsudil 0.02% leading to this first approval for the reduction of elevated IOP in patients with open-angle glaucoma or ocular hypertension.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29453668',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 84,\n",
       "     'text': 'Discovery of the ROCK inhibitor netarsudil for the treatment of open-angle glaucoma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27072905',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 352,\n",
       "     'text': 'Profile of netarsudil ophthalmic solution and its potential in the treatment of open-angle glaucoma: evidence to date.Netarsudil ophthalmic solution is a novel topical intraocular pressure (IOP)-lowering agent that has recently been approved by the US Food and Drug Administration (FDA) for the treatment of ocular hypertension and open-angle glaucoma. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30323550',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 349,\n",
       "     'text': 'Discovery and Preclinical Development of Netarsudil, a Novel Ocular Hypotensive Agent for the Treatment of Glaucoma.Netarsudil is a novel ROCK/NET inhibitor with high potency in biochemical and cell-based assays, an ability to produce large and durable IOP reductions in animal models, and favorable pharmacokinetic and ocular tolerability profiles.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28609185',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 118,\n",
       "     'text': 'Profile of netarsudil ophthalmic solution and its potential in the treatment of open-angle glaucoma: evidence to date.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30323550',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1427,\n",
       "     'offsetInEndSection': 1649,\n",
       "     'text': 'In 2 large, randomized, double-masked trials reported here, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29199013',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 223,\n",
       "     'text': 'To evaluate the efficacy and ocular and systemic safety of netarsudil 0.02% ophthalmic solution, a rho-kinase inhibitor and norepinephrine transporter inhibitor, in patients with open-angle glaucoma and ocular hypertension.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29199013',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Reslizumab is a humanized monoclonal antibody to treat what specific type of asthma?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29554826',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27445482',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28344579',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27458609',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29059618',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28683596',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30346831',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29486600',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28421429',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23326187'],\n",
       "   'ideal_answer': ['Reslizumab in the treatment of severe eosinophilic asthma:\\xa0an update.',\n",
       "    'Reslizumab is a humanized monoclonal antibody to treat eosinophilic asthma'],\n",
       "   'exact_answer': ['eosinophilic asthma'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5ca0bf00ecadf2e73f000045',\n",
       "   'snippets': [{'offsetInBeginSection': 131,\n",
       "     'offsetInEndSection': 398,\n",
       "     'text': 'Since IL-5 plays an important role in the maturation, survival and migration of eosinophils, hence the pathogenesis of eosinophilic asthma, biotherapeutics targeting IL-5/IL-5Rα have been developed and/or marketed, including Mepolizumab, Reslizumab, and Benralizumab.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29059618',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 14,\n",
       "     'offsetInEndSection': 383,\n",
       "     'text': 'Reslizumab, a humanized mAb against IL-5, reduces the number of eosinophils in the blood and lungs. Based on efficacy and safety data from pivotal RCTs, reslizumab had been approved for use as an add-on maintenance treatment of severe asthma with an eosinophilic phenotype in adults who have a history of exacerbations despite receiving their current asthma medicines. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29486600',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 57,\n",
       "     'text': 'Reslizumab in the treatment of severe eosinophilic asthma',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29554826',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 311,\n",
       "     'offsetInEndSection': 547,\n",
       "     'text': 'Reslizumab is a humanized monoclonal (immunoglobulin G4/κ) antibody that binds with high affinity to circulating human IL-5 and downregulates the IL-5 signaling pathway, potentially disrupting the maturation and survival of eosinophils.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28683596',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 154,\n",
       "     'text': 'Reslizumab (Cinqaero®; Cinqair®) is a humanized monoclonal antibody against interleukin-5 (IL-5), a cytokine mediator of eosinophilic airway inflammation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28421429',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 164,\n",
       "     'text': 'BACKGROUND\\nReslizumab is a humanized anti-interleukin-5 monoclonal antibody used as add-on maintenance treatment for patients with uncontrolled eosinophilic asthma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30346831',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 522,\n",
       "     'offsetInEndSection': 920,\n",
       "     'text': 'In particular, reslizumab is a humanized anti-IL-5 monoclonal antibody that has been found to be an effective and safe add-on therapy, capable of decreasing asthma exacerbations and significantly improving disease control and lung function in patients experiencing persistent allergic or nonallergic eosinophilic asthma, despite the regular use of moderate-to-high doses of inhaled corticosteroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27445482',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 297,\n",
       "     'offsetInEndSection': 483,\n",
       "     'text': 'Reslizumab (Cinquil™) is a humanized monoclonal antibody with potent interleukin-5 neutralizing effects, which represents a potential treatment for poorly controlled eosinophilic asthma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23326187',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 155,\n",
       "     'offsetInEndSection': 316,\n",
       "     'text': 'Reslizumab is indicated as an add-on treatment for severe eosinophilic asthma in adults, on the basis of data from the BREATH phase III clinical trial programme.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28421429',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 220,\n",
       "     'text': 'Reslizumab in Eosinophilic Asthma: A Review.Reslizumab (Cinqaero<sup>®</sup>; Cinqair<sup>®</sup>) is a humanized monoclonal antibody against interleukin-5 (IL-5), a cytokine mediator of eosinophilic airway inflammation. ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28421429',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 477,\n",
       "     'offsetInEndSection': 713,\n",
       "     'text': 'Reslizumab is a humanized monoclonal (immunoglobulin G4/κ) antibody that binds with high affinity to circulating human IL-5 and downregulates the IL-5 signaling pathway, potentially disrupting the maturation and survival of eosinophils. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28683596',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 419,\n",
       "     'offsetInEndSection': 605,\n",
       "     'text': 'Reslizumab (Cinquil™) is a humanized monoclonal antibody with potent interleukin-5 neutralizing effects, which represents a potential treatment for poorly controlled eosinophilic asthma. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23326187',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Reslizumab: Maintenance treatment for eosinophilic asthma inadequately controlled on corticosteroids.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27458609',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'What is Quorum Sensing in bacteria?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/22825856',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29130162',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29130160',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22269240',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15151251',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29095463',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27634359',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23125205',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27688408',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20819164'],\n",
       "   'ideal_answer': ['In many pathogenic microorganisms, communication systems, collectively termed quorum sensing (QS),',\n",
       "    'In many pathogenic microorganisms, communication systems, collectively termed quorum sensing (QS), have been observed',\n",
       "    'In most bacteria, a global level of regulation, termed quorum sensing (QS), exists involving intercellular communication via the production and response to cell density-dependent signal molecules.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5c978924ecadf2e73f000023',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'Many Proteobacteria synthesize acyl-homoserine lactone (AHL) molecules for use as signals in cell density-dependent gene regulation known as quorum sensing (QS) and response.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29130162',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 197,\n",
       "     'text': 'In most bacteria, a global level of regulation, termed quorum sensing (QS), exists involving intercellular communication via the production and response to cell density-dependent signal molecules. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29130160',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 358,\n",
       "     'offsetInEndSection': 596,\n",
       "     'text': ' In many pathogenic microorganisms, communication systems, collectively termed quorum sensing (QS), have been observed to control a number of bacterial behaviours including expression of virulence factors and the development of biofilms. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29095463',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 825,\n",
       "     'offsetInEndSection': 1049,\n",
       "     'text': 'Bacteria can communicate within their own species (intra-species) but also between species (inter-species), for which they employ an autoinducer-2 quorum sensing system which is called the universal language of the bacteria.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27688408',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 217,\n",
       "     'text': 'Quorum sensing is a signaling mechanism through which bacteria modulate a number of cellular functions (genes), including sporulation, biofilm formation, bacteriocin production, virulence responses, as well as others.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15151251',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 218,\n",
       "     'offsetInEndSection': 395,\n",
       "     'text': 'Quorum sensing is a mechanism of cell-to-cell communication and is mediated by extracellular chemical signals generated by the bacteria when specific cell densities are reached.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15151251',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 117,\n",
       "     'text': 'Quorum sensing is used by a large variety of bacteria to regulate gene expression in a cell-density-dependent manner.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22825856',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'BACKGROUND\\nQuorum sensing or the bacterial information flow in an orchestrated manner is an essential feature of many pathogenic bacteria.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27634359',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 931,\n",
       "     'offsetInEndSection': 1056,\n",
       "     'text': 'Quorum sensing is the means of communication between these bacteria to regulate a wide range of behavior patterns among them.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22269240',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 155,\n",
       "     'text': 'Quorum sensing is a process of cell-cell communication that allows bacteria to share information about cell density and adjust gene expression accordingly.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23125205',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 103,\n",
       "     'text': 'Quorum sensing is the phenomenon, whereby bacteria use signal molecules to communicate with each other.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20819164',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 153,\n",
       "     'text': \"The term 'quorum sensing' describes intercellular bacterial communication which regulates bacterial gene expression according to population cell density.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27688408',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which receptor is inhibited by bimagrumab?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27167138',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25381300',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24298022',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29226558',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31761834',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26506009',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27462398',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28905498',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30095981'],\n",
       "   'ideal_answer': ['Bimagrumab is a fully human monoclonal antibody that blocks the activin type II receptors, preventing the activity of myostatin and other negative skeletal muscle regulators.'],\n",
       "   'exact_answer': ['activin type II receptors'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5e44b04c48dab47f26000016',\n",
       "   'snippets': [{'offsetInBeginSection': 189,\n",
       "     'offsetInEndSection': 313,\n",
       "     'text': 'Bimagrumab is a human anti-ActRII antibody which was found to increase muscle mass and function by blocking ActRII signaling',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29226558',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 228,\n",
       "     'text': 'BACKGROUND: Bimagrumab is a human monoclonal antibody inhibitor of activin type II receptors (ActRII), with anabolic action on skeletal muscle mass by blocking binding of myostatin and other negative regulators of muscle growth.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1376,\n",
       "     'offsetInEndSection': 1504,\n",
       "     'text': 'CONCLUSION: Bimagrumab alters the function of pituitary gonadotroph cells, consistent with blockade of activin on local ActRII. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 185,\n",
       "     'text': 'RATIONALE: Bimagrumab is a fully human monoclonal antibody that blocks the activin type II receptors, preventing the activity of myostatin and other negative skeletal muscle regulators.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30095981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1397,\n",
       "     'offsetInEndSection': 1644,\n",
       "     'text': 'CONCLUSIONS: Blocking the action of negative muscle regulators through the activin type II receptors with bimagrumab treatment safely increased skeletal muscle mass but did not improve functional capacity in patients with COPD and low muscle mass.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30095981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 293,\n",
       "     'offsetInEndSection': 552,\n",
       "     'text': 'This study aimed to explore the clinical potential of bimagrumab, a human monoclonal antibody targeting the activin type II receptor, for the recovery of skeletal muscle volume from disuse atrophy using an experimental model of lower extremity immobilization.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28905498',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 94,\n",
       "     'text': 'Effects of bimagrumab, an activin receptor type II inhibitor, on pituitary neurohormonal axes.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 168,\n",
       "     'text': 'Bimagrumab (BYM338) is a novel fully human monoclonal antibody that exerts strong promyogenic effects on skeletal muscle by blocking activin type II receptors (ActRII).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27167138',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 228,\n",
       "     'text': 'BACKGROUND\\n\\nBimagrumab is a human monoclonal antibody inhibitor of activin type II receptors (ActRII), with anabolic action on skeletal muscle mass by blocking binding of myostatin and other negative regulators of muscle growth.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 185,\n",
       "     'text': 'RATIONALE\\n\\nBimagrumab is a fully human monoclonal antibody that blocks the activin type II receptors, preventing the activity of myostatin and other negative skeletal muscle regulators.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30095981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 591,\n",
       "     'offsetInEndSection': 806,\n",
       "     'text': 'CDD866 (murinized version of bimagrumab) is a neutralizing antibody against the activin receptor type II (ActRII) preventing binding of ligands such as myostatin and activin A, which are involved in cancer cachexia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27462398',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 317,\n",
       "     'offsetInEndSection': 483,\n",
       "     'text': 'We have developed a novel , human anti-ActRII antibody ( bimagrumab , or BYM338 ) to prevent binding of ligands to the receptors and thus inhibit downstream signaling',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24298022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 284,\n",
       "     'offsetInEndSection': 544,\n",
       "     'text': 'This study aimed to explore the clinical potential of bimagrumab , a human monoclonal antibody targeting the activin type II receptor , for the recovery of skeletal muscle volume from disuse atrophy using an experimental model of lower extremity immobilization',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28905498',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 217,\n",
       "     'text': 'Bimagrumab is a human monoclonal antibody inhibitor of activin type II receptors ( ActRII) , with anabolic action on skeletal muscle mass by blocking binding of myostatin and other negative regulators of muscle growth',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 658,\n",
       "     'offsetInEndSection': 905,\n",
       "     'text': 'Novel antiresorptive and anabolic therapies are emerging for osteoporosis as well as drugs for sarcopenia , cancer cachexia or muscle wasting disorders , including antibodies against myostatin or activin receptor type IIA and IIB ( e.g. bimagrumab',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26506009',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 346,\n",
       "     'text': \"A global , randomized , double-blind placebo-controlled study was conducted to confirm that BYM338 ( bimagrumab) , an anti-activin type II receptor antibody , improves motor function in patients with sporadic inclusion body myositis after 52 weeks' treatment consisting of intravenous administration every 4 weeks at doses of 10 , 3 , and 1 mg/kg\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31761834',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 169,\n",
       "     'text': 'Bimagrumab ( BYM338 ) is a novel fully human monoclonal antibody that exerts strong promyogenic effects on skeletal muscle by blocking activin type II receptors ( ActRII',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27167138',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 169,\n",
       "     'text': 'Bimagrumab (BYM338) is a novel fully human monoclonal antibody that exerts strong promyogenic effects on skeletal muscle by blocking activin type II receptors (ActRII).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27167138',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 228,\n",
       "     'text': 'BACKGROUND\\nBimagrumab is a human monoclonal antibody inhibitor of activin type II receptors (ActRII), with anabolic action on skeletal muscle mass by blocking binding of myostatin and other negative regulators of muscle growth.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 357,\n",
       "     'offsetInEndSection': 506,\n",
       "     'text': 'Bimagrumab also blocks other endogenous ActRII ligands, such as activins, which act on the neurohormonal axes, pituitary, gonads and adrenal glands.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 185,\n",
       "     'text': 'RATIONALE\\nBimagrumab is a fully human monoclonal antibody that blocks the activin type II receptors, preventing the activity of myostatin and other negative skeletal muscle regulators.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30095981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 358,\n",
       "     'offsetInEndSection': 506,\n",
       "     'text': 'Bimagrumab also blocks other endogenous ActRII ligands, such as activins, which act on the neurohormonal axes, pituitary, gonads and adrenal glands.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 311,\n",
       "     'offsetInEndSection': 474,\n",
       "     'text': 'We have developed a novel, human anti-ActRII antibody (bimagrumab, or BYM338) to prevent binding of ligands to the receptors and thus inhibit downstream signaling.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24298022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'Bimagrumab is a human monoclonal antibody inhibitor of activin type II receptors (ActRII), with anabolic action on skeletal muscle mass by blocking binding of myostatin and other negative regulators of muscle growth.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29566437',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'Bimagrumab is a fully human monoclonal antibody that blocks the activin type II receptors, preventing the activity of myostatin and other negative skeletal muscle regulators.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30095981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1442,\n",
       "     'offsetInEndSection': 1676,\n",
       "     'text': 'Blocking the action of negative muscle regulators through the activin type II receptors with bimagrumab treatment safely increased skeletal muscle mass but did not improve functional capacity in patients with COPD and low muscle mass.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30095981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 281,\n",
       "     'offsetInEndSection': 540,\n",
       "     'text': 'This study aimed to explore the clinical potential of bimagrumab, a human monoclonal antibody targeting the activin type II receptor, for the recovery of skeletal muscle volume from disuse atrophy using an experimental model of lower extremity immobilization.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28905498',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 569,\n",
       "     'offsetInEndSection': 784,\n",
       "     'text': 'CDD866 (murinized version of bimagrumab) is a neutralizing antibody against the activin receptor type II (ActRII) preventing binding of ligands such as myostatin and activin A, which are involved in cancer cachexia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27462398',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 307,\n",
       "     'offsetInEndSection': 452,\n",
       "     'text': 'We tested inhibition of activin receptors IIA and IIB (ActRII) in 14 patients with sIBM using one dose of bimagrumab (n = 11) or placebo (n = 3).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25381300',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What classes of drugs does Retapamulin belong to?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28874907',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22777229',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23793314',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19436611',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18041900',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17350985',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18341664',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18389088',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16940066',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28533232'],\n",
       "   'ideal_answer': ['Retapamulin is a member of the pleuromutilin family of antibiotics.',\n",
       "    'Pleuromutilins have a potential to be developed as a new class of antibiotics for  use in humans. This class includes valnemulin, tiamulin, and retapamulin.',\n",
       "    'Retapamulin belongs to the class of gentamycin-resistant antibiotics.'],\n",
       "   'exact_answer': ['antibiotics'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5e5e50751af46fc13000000b',\n",
       "   'snippets': [{'offsetInBeginSection': 321,\n",
       "     'offsetInEndSection': 458,\n",
       "     'text': 'Retapamulin 1% ointment is a unique topical antibiotic formulation that may be a suitable option for the treatment of clinically infected',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22777229',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1139,\n",
       "     'offsetInEndSection': 1328,\n",
       "     'text': 'Retapamulin 1% is effective for the treatment of atopic dermatitis infected with S aureus, and demonstrates efficacy against both methicillin-susceptible and methicillin-resistant strains. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22777229',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 318,\n",
       "     'offsetInEndSection': 741,\n",
       "     'text': 'Pleuromutilins have a potential to be developed as a new class of antibiotics for systemic use in humans. In the current study, we investigated the relationship between pleuromutilins, including valnemulin, tiamulin, and retapamulin, and 13 other antibiotics representing different mechanisms of action, against methicillin-susceptible and -resistant S. aureus both in vitro and in an experimental Galleria mellonella model',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28874907',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 779,\n",
       "     'offsetInEndSection': 958,\n",
       "     'text': 'he eight antibiotics, azithromycin, clarithromycin, erythromycin, fusidic acid, mupirocin, retapamulin, rifampin, and telithromycin, had diverse targets and mechanisms of action. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28533232',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 322,\n",
       "     'offsetInEndSection': 435,\n",
       "     'text': 'Retapamulin belongs to a newly developed class of antibiotics for the treatment of uncomplicated skin infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18341664',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 40,\n",
       "     'text': 'Retapamulin: a newer topical antibiotic.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23793314',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 89,\n",
       "     'text': 'Retapamulin: a new topical antibiotic for the treatment of uncomplicated skin infections.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18389088',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': 'Retapamulin is the first agent in the new pleuromutilin class of antibacterials to become commercially available for clinical use in humans.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18389088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 111,\n",
       "     'text': 'Retapamulin is a novel semisynthetic pleuromutilin antibiotic specifically designed for use as a topical agent.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19436611',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 321,\n",
       "     'offsetInEndSection': 434,\n",
       "     'text': 'Retapamulin belongs to a newly developed class of antibiotics for the treatment of uncomplicated skin infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18341664',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 234,\n",
       "     'text': 'Retapamulin is a semisynthetic pleuromutilin compound with in vitroactivity against Gram-positive bacteria, no cross-resistance to other classes of antimicrobial agents in current use and a low potential for development of resistance.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18041900',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'Retapamulin is a semisynthetic pleuromutilin derivative being developed as a topical antibiotic for treating bacterial infections of the skin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16940066',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'OBJECTIVES\\n\\nRetapamulin is the first agent of the pleuromutilin class formulated as a topical antibacterial for treating skin infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17350985',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'Retapamulin is the first agent of the pleuromutilin class formulated as a topical antibacterial for treating skin infections . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17350985',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': 'Retapamulin is the first agent in the new pleuromutilin class of antibacterials to become commercially available for clinical use in humans.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18389088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 112,\n",
       "     'text': 'Retapamulin is a novel semisynthetic pleuromutilin antibiotic specifically designed for use as a topical agent.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19436611',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 143,\n",
       "     'text': 'Retapamulin is a semisynthetic pleuromutilin derivative being developed as a topical antibiotic for treating bacterial infections of the skin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16940066',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 110,\n",
       "     'text': 'In vitro activity against anaerobes of retapamulin, a new topical antibiotic for treatment of skin infections.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17350985',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 125,\n",
       "     'text': 'Retapamulin is the first agent of the pleuromutilin class formulated as a topical antibacterial for treating skin infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17350985',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What are Syndecans?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25572401',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11900484',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8875948',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16620374',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29510059',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29931674',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23384311',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29226950',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15936998',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26149933'],\n",
       "   'ideal_answer': ['Syndecans are transmembrane proteoglycans with heparan and chondroitin sulfate chains attached to their extracellular domain. Like many proteoglycans, they interact with a large number of ligands, such as growth factors, adhesion receptors, soluble small molecules, proteinases, and other extracellular matrix proteins to initiate downstream signaling pathways. Syndecans play a major role in inflammation, mainly by regulating leukocyte extravasation and cytokine function.',\n",
       "    'Syndecans are important mediators of signalling by transmitting external stimuli into the cells.  Syndecans (SDCs) are a family of heparan sulfate proteoglycans (HSPGs) glycoproteins ubiquitously expressed on the cell surfaces and extracellular matrix of all mammalian tissues Syndecans are transmembrane proteoglycans that, together with integrins, control cell interactions with extracellular matrix components. ',\n",
       "    'Syndecans are transmembrane proteoglycans that, together with integrins, control cell interactions with extracellular matrix components. '],\n",
       "   'type': 'summary',\n",
       "   'id': '5e6e9689c6a8763d23000006',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 178,\n",
       "     'text': 'Syndecans (SDCs) are a family of heparan sulfate proteoglycans (HSPGs) glycoproteins ubiquitously expressed on the cell surfaces and extracellular matrix of all mammalian tissues',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29226950',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Syndecans are transmembrane proteoglycans that, together with integrins, control cell interactions with extracellular matrix components. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29510059',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 146,\n",
       "     'text': 'Syndecans are cell surface heparan sulfate proteoglycans that serve as co-receptors and modulate the actions of a number of extracellular ligands.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15936998',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 197,\n",
       "     'text': 'The syndecans are a type of cell surface adhesion receptor that initiates intracellular signaling events through receptor clustering mediated by their highly conserved transmembrane domains (TMDs).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25572401',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 246,\n",
       "     'text': 'BACKGROUND\\n\\nThe syndecans are the major family of transmembrane proteoglycans in animals and are known for multiple roles in cell interactions and growth factor signalling during development, inflammatory response, wound-repair and tumorigenesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16620374',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 145,\n",
       "     'text': 'Syndecans are transmembrane proteoglycans implicated in the regulation of cell growth and differentiation , by interacting with growth factors . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11900484',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 177,\n",
       "     'text': 'Syndecans are transmembrane heparan sulfate proteoglycans involved in the regulation of cell growth , differentiation , adhesion , neuronal development , and lipid metabolism . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26149933',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 209,\n",
       "     'text': 'Syndecans are a family of four cell surface proteoglycans that bind to various components of the extracellular environment and can regulate many cellular behaviors including growth , adhesion , and movement . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8875948',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'Syndecans are transmembrane heparan sulfate proteoglycans with roles in cell proliferation , differentiation , adhesion , and migration . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23384311',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 171,\n",
       "     'text': 'Syndecans are transmembrane heparan sulfate proteoglycans involved in the regulation of cell growth, differentiation, adhesion, neuronal development, and lipid metabolism.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26149933',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is bortezomib a Proteasome inhibitor?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/26288836',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22702336',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28409734',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16278210',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31088925',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22027222',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21247388',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22216088',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28860152',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28317148'],\n",
       "   'ideal_answer': ['The proteasome inhibitor bortezomib is effective for a variety of tumors, but not for GBM. Proteasome inhibitor bortezomib',\n",
       "    'Yes, bortezomib is a Proteasome inhibitor.',\n",
       "    'Yes, bortezomib is a potent and specific reversible ubiquitin/proteasome inhibitor.',\n",
       "    'yes, The proteasome-inhibitor bortezomib',\n",
       "    'Yes bortezomib is a Proteasome inhibitor.',\n",
       "    'proteasome inhibitor bortezomib',\n",
       "    'The proteasome inhibitor bortezomib is effective for a variety of tumors, but not for gbm.',\n",
       "    'Proteasome inhibitor bortezomib'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '5e3c6850b5b409ea5300001f',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 35,\n",
       "     'text': 'The proteasome-inhibitor bortezomib',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28317148',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 158,\n",
       "     'offsetInEndSection': 249,\n",
       "     'text': 'The proteasome inhibitor bortezomib is effective for a variety of tumors, but not for GBM. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28409734',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 32,\n",
       "     'text': 'Proteasome inhibitor bortezomib ',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28860152',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 176,\n",
       "     'text': 'The proteasome inhibitor bortezomib, registered for Multiple Myeloma treatment, is currently explored for activity in solid tumors including non-small cell lung cancer (NSCLC).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22027222',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 82,\n",
       "     'text': 'Regulation of osteoblastic differentiation by the proteasome inhibitor bortezomib.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22702336',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 216,\n",
       "     'offsetInEndSection': 446,\n",
       "     'text': 'The proteasome inhibitor bortezomib (also known as Velcade and PS-341) is a clinically effective antineoplastic drug that is FDA approved for treatment of hematologic malignancies such as multiple myeloma and mantle cell lymphoma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31088925',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Bortezomib as the first proteasome inhibitor anticancer drug: current status and future perspectives.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21247388',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 78,\n",
       "     'text': 'The proteasome inhibitor bortezomib is emerging as a potent anti-cancer agent.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16278210',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': 'Bortezomib (Velcade™) is a reversible proteasome inhibitor that is approved for the treatment of multiple myeloma (MM).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22216088',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 75,\n",
       "     'text': 'The proteasome inhibitor Bortezomib is used to treat multiple myeloma (MM).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26288836',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is another name for acid sphingomyelinase deficiency (ASMD)?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27884455',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28259515',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27340749',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28801223',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22614361',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30795770',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27198631',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28228103',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26049896',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31080679'],\n",
       "   'ideal_answer': ['Acid sphingomyelinase deficiency(ASMD) is also known as Niemann-Pick disease type A and type B.',\n",
       "    'Historically, ASMD has been classified as Niemann-Pick disease (NPD) types A (NPD A) and B (NPD B). The clinical spectrum distinguishes a severe infantile neurological form (type A), a non-neurological visceral form (type B) and a rare intermediate neurovisceral form.',\n",
       "    'Acid sphingomyelinase deficiency (ASMD) is an autosomal recessive disease with a clinical spectrum ranging from a neurovisceral infantile form (Niemann-Pick disease type A) to a chronic visceral form also encountered in adults (Niemann-Pick disease type B, NP-B)',\n",
       "    'niemann-pick disease'],\n",
       "   'exact_answer': ['Niemann-Pick disease type A and type B'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5e36dc8cb5b409ea5300000d',\n",
       "   'snippets': [{'offsetInBeginSection': 14,\n",
       "     'offsetInEndSection': 276,\n",
       "     'text': 'Acid sphingomyelinase deficiency (ASMD) is an autosomal recessive disease with a clinical spectrum ranging from a neurovisceral infantile form (Niemann-Pick disease type A) to a chronic visceral form also encountered in adults (Niemann-Pick disease type B, NP-B)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27884455',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 65,\n",
       "     'text': 'Acid sphingomyelinase deficiency (ASMd, Niemann-Pick disease A/B)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28259515',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 255,\n",
       "     'text': 'Acid sphingomyelinase deficiency (ASMD), a rare lysosomal storage disease, is an autosomal recessive genetic disorder caused by different SMPD1 mutations. Historically, ASMD has been classified as Niemann-Pick disease (NPD) types A (NPD A) and B (NPD B). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28228103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 304,\n",
       "     'text': 'Acid sphingomyelinase (ASM) deficient Niemann-Pick disease is a lysosomal storage disorder resulting from mutations in the SMPD1 gene. The clinical spectrum distinguishes a severe infantile neurological form (type A), a non-neurological visceral form (type B) and a rare intermediate neurovisceral form. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28801223',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 212,\n",
       "     'text': 'BACKGROUND\\n\\nAcid sphingomyelinase deficiency (ASMD), [Niemann-Pick Disease Types A and B (NPD A and B)], is an inherited metabolic disorder resulting from deficiency of the lysosomal enzyme acid sphingomyelinase.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27198631',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 148,\n",
       "     'text': 'Introduction\\n\\nAcid sphingomyelinase deficiency (ASMD, also known as Niemann-Pick Type A and Type B disease) is a rare, inherited metabolic disorder.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31080679',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 155,\n",
       "     'offsetInEndSection': 255,\n",
       "     'text': 'Historically, ASMD has been classified as Niemann-Pick disease (NPD) types A (NPD A) and B (NPD B).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28228103',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 276,\n",
       "     'text': 'Niemann-Pick disease (types A and B), or acid sphingomyelinase deficiency, is an inherited deficiency of acid sphingomyelinase, resulting in intralysosomal accumulation of sphingomyelin in cells throughout the body, particularly within those of the reticuloendothelial system.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22614361',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 263,\n",
       "     'text': 'Acid sphingomyelinase deficiency (ASMD) is an autosomal recessive disease with a clinical spectrum ranging from a neurovisceral infantile form (Niemann-Pick disease type A) to a chronic visceral form also encountered in adults (Niemann-Pick disease type B, NP-B).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27884455',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 134,\n",
       "     'text': 'Acid sphingomyelinase deficiency (ASMD, also known as Niemann-Pick Type A and Type B disease) is a rare, inherited metabolic disorder.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31080679',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 299,\n",
       "     'text': 'Acid sphingomyelinase deficiency (ASMD), due to mutations in the sphingomyelin phosphodiesterase 1 (SMPD1) gene, is divided into infantile neurovisceral ASMD (Niemann-Pick type A), chronic neurovisceral ASMD (intermediate form, Niemann-Pick type A/B) and chronic visceral ASMD (Niemann-Pick type B).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30795770',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 200,\n",
       "     'text': 'Acid sphingomyelinase deficiency (ASMD), [Niemann-Pick Disease Types A and B (NPD A and B)], is an inherited metabolic disorder resulting from deficiency of the lysosomal enzyme acid sphingomyelinase.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27198631',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which molecule is targeted by Asciminib?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/31006307',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28329763',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29568367',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31826340',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30927708',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29325229',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31543464',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30059193',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29522367',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30137981'],\n",
       "   'ideal_answer': ['Asciminib is an orally administered allosteric inhibitor of the BCR-ABL tyrosine kinase.'],\n",
       "   'exact_answer': ['BCR-ABL tyrosine kinase'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5e30e80bfbd6abf43b00003b',\n",
       "   'snippets': [{'offsetInBeginSection': 1091,\n",
       "     'offsetInEndSection': 1265,\n",
       "     'text': 'Asciminib, an allosteric ABL1 inhibitor, could demonstrate a higher capacity in overcoming common TKIs resistant mutations, including T315I, but clinical findings are needed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29522367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 274,\n",
       "     'offsetInEndSection': 473,\n",
       "     'text': 'Moreover, allosteric targeting has been identified to consequentially inhibit Bcr-Abl activity, which led to the recent development of ABL-001 (asciminib) that selectively binds the myristoyl pocket.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29325229',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 266,\n",
       "     'text': 'Asciminib (previously ABL001), which binds the myristate-binding pocket of the Bcr-Abl kinase domain, is in phase I clinical trials as monotherapy and in combination with imatinib, nilotinib and dasatinib for the treatment of patients with refractory CML or Ph+ ALL.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29568367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 97,\n",
       "     'text': 'Asciminib (ABL001) is an orally administered allosteric inhibitor of the BCR-ABL tyrosine kinase.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30059193',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Discovery of Asciminib (ABL001), an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30137981',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 321,\n",
       "     'offsetInEndSection': 548,\n",
       "     'text': 'We describe the discovery of asciminib (ABL001), the first allosteric BCR-ABL1 inhibitor to reach the clinic. Asciminib binds to the myristate pocket of BCR-ABL1 and maintains activity against TKI-resistant ATP-site mutations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30137981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 632,\n",
       "     'offsetInEndSection': 865,\n",
       "     'text': 'Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clinical development testing in patients with CML and Philadelphia chromosome-positive (Ph+) acute lymphoblastic leukaemia. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28329763',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 321,\n",
       "     'offsetInEndSection': 430,\n",
       "     'text': 'We describe the discovery of asciminib (ABL001), the first allosteric BCR-ABL1 inhibitor to reach the clinic.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30137981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 431,\n",
       "     'offsetInEndSection': 547,\n",
       "     'text': 'Asciminib binds to the myristate pocket of BCR-ABL1 and maintains activity against TKI-resistant ATP-site mutations.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30137981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 96,\n",
       "     'text': 'Disposition of asciminib, a potent BCR-ABL1 tyrosine kinase inhibitor, in healthy male subjects.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31006307',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 272,\n",
       "     'text': 'Asciminib ( previously ABL001) , which binds the myristate-binding pocket of the Bcr-Abl kinase domain , is in phase I clinical trials as monotherapy and in combination with imatinib , nilotinib and dasatinib for the treatment of patients with refractory CML or Ph+ ALL . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29568367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'Asciminib is an allosteric inhibitor that binds a myristoyl site of the BCR-ABL1 protein , locking BCR-ABL1 into an inactive conformation through a mechanism distinct from those for all other ABL kinase inhibitors . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31826340',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 104,\n",
       "     'text': 'Discovery of Asciminib ( ABL001) , an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1 .',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30137981',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 431,\n",
       "     'offsetInEndSection': 548,\n",
       "     'text': 'Asciminib binds to the myristate pocket of BCR-ABL1 and maintains activity against TKI-resistant ATP-site mutations.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30137981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 321,\n",
       "     'offsetInEndSection': 431,\n",
       "     'text': 'We describe the discovery of asciminib (ABL001), the first allosteric BCR-ABL1 inhibitor to reach the clinic.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30137981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 267,\n",
       "     'text': 'Asciminib (previously ABL001), which binds the myristate-binding pocket of the Bcr-Abl kinase domain, is in phase I clinical trials as monotherapy and in combination with imatinib, nilotinib and dasatinib for the treatment of patients with refractory CML or Ph+ ALL.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29568367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 156,\n",
       "     'text': 'Asciminib, a highly selective non-ATP competitive inhibitor of BCR-ABL, has demonstrated to be a promising drug for patients with chronic myeloid leukemia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30927708',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 548,\n",
       "     'offsetInEndSection': 720,\n",
       "     'text': 'The obtained results indicate that the mutations have adversely influence on the binding of Asciminib to BCR-ABL, as the nonpolar contributions decline in the two mutants.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30927708',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 215,\n",
       "     'text': 'Asciminib is a potent, specific BCR-ABL1 inhibitor being developed for the treatment of patients with chronic myelogenous leukemia (CML) and Philadelphia chromosome positive acute lymphoblastic leukemia (Ph\\u2009+\\u2009ALL).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31006307',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 352,\n",
       "     'offsetInEndSection': 530,\n",
       "     'text': 'Asciminib, an allosteric inhibitor targeting the myristoyl-binding pocket of BCR-ABL1, is active against most single mutants but ineffective against all tested compound mutants.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31543464',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 632,\n",
       "     'offsetInEndSection': 831,\n",
       "     'text': 'Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clinical development testing in patients with CML and Philadelphia chromosome-positive (Ph',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28329763',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1077,\n",
       "     'offsetInEndSection': 1251,\n",
       "     'text': 'Asciminib, an allosteric ABL1 inhibitor, could demonstrate a higher capacity in overcoming common TKIs resistant mutations, including T315I, but clinical findings are needed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29522367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 352,\n",
       "     'offsetInEndSection': 529,\n",
       "     'text': 'Asciminib, an allosteric inhibitor targeting the myristoyl-binding pocket of BCR-ABL1, is active against most single mutants but ineffective against all tested compound mutants.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31543464',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is SATB1 positioned close to AT-rich sequences?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24729451',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24047082',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18408014',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28179318',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24118100',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10629043',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8049444',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29306014',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9548713',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27590341'],\n",
       "   'ideal_answer': ['Yes, SATB1 is preferentially located at the start of an AT-rich sequence and is associated with other, more diffuse AT- rich sequences in the genome.',\n",
       "    \"Special AT-rich sequence-binding protein 1 (SATB1), a DNA-binding protein expressed predominantly in thymocytes, recognizes an ATC sequence context that consists of a cluster of sequence stretches with well-mixed A's, T's, and C's without G's on one strand. SATB1 (special AT-rich sequence-binding protein-1) provides a key link between DNA loop organization, chromatin modification/remodeling, and association of transcription factors at matrix attachment regions (MARs).\",\n",
       "    \"Special AT-rich sequence-binding protein 1 (SATB1), a DNA-binding protein expressed predominantly in thymocytes, recognizes an ATC sequence context that consists of a cluster of sequence stretches with well-mixed A's, T's, and C's without G's on one strand.\",\n",
       "    'We have purified and identified one of the core factors as the matrix attachment region (MAR) binding protein, SATB1, which is known to bind to AT-rich sequences with a high propensity to unwind'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '5d38826ea1e1595105000016',\n",
       "   'snippets': [{'offsetInBeginSection': 416,\n",
       "     'offsetInEndSection': 585,\n",
       "     'text': ' Tryptic cleavage and peptide sequence analysis demonstrated that the 98-kD protein is identical to a recently cloned protein, special A-T-rich binding protein 1 (SATB1)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8049444',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 258,\n",
       "     'text': \"Special AT-rich sequence-binding protein 1 (SATB1), a DNA-binding protein expressed predominantly in thymocytes, recognizes an ATC sequence context that consists of a cluster of sequence stretches with well-mixed A's, T's, and C's without G's on one strand. \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9548713',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 778,\n",
       "     'offsetInEndSection': 973,\n",
       "     'text': ' We have purified and identified one of the core factors as the matrix attachment region (MAR) binding protein, SATB1, which is known to bind to AT-rich sequences with a high propensity to unwind',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10629043',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 214,\n",
       "     'text': 'SATB1 (special AT-rich sequence-binding protein-1) provides a key link between DNA loop organization, chromatin modification/remodeling, and association of transcription factors at matrix attachment regions (MARs).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18408014',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 166,\n",
       "     'text': 'Over-expression of the special AT rich sequence binding protein 1 (SATB1) promotes the progression of nasopharyngeal carcinoma: association with EBV LMP-1 expression.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24047082',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 13,\n",
       "     'offsetInEndSection': 133,\n",
       "     'text': 'pecial AT rich sequence binding protein 1 (SATB1) plays a crucial role in the biology of various types of human cancer. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24047082',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 58,\n",
       "     'text': 'Loss of special AT-rich sequence-binding protein 1 (SATB1)',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24118100',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 5,\n",
       "     'offsetInEndSection': 164,\n",
       "     'text': 'Special AT-rich sequence-binding protein 1 (SATB1) is a cell type-specific matrix attachment region binding protein, functioning as a global genome organizer. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24118100',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': 'SATB1 (special AT-rich binding protein 1) is a global chromatin organizer regulating the expression of a large number of genes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24729451',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 108,\n",
       "     'text': 'Special AT-rich Sequence-binding Protein 1 (SATB1) Functions as an Accessory Factor in Base Excision Repair.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27590341',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 79,\n",
       "     'text': 'Rearrangement of the Chromatin Organizer Special AT-rich Binding Protein 1 Gene',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28179318',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'The Special AT-rich Sequence Binding Protein 1 (SATB1) exerts multiple functions, by influencing the structural organization of chromatin and interacting with several co-activators and co-repressors of transcription.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29306014',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 84,\n",
       "     'text': 'The Special AT-rich Sequence Binding Protein 1 (SATB1) and its role in solid tumors.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29306014',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': \"Is Huntington's disease caused by a dominate or recessive gene?\",\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29413175',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31286142',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2881213',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22119622',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25356969',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14526190',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15764008',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28927719',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29134321',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11723754'],\n",
       "   'ideal_answer': [\"Huntington's Disease (HD) is an autosomal dominant neurodegenerative disease\",\n",
       "    \"Huntington's disease is caused by an autosomal dominant gene.\",\n",
       "    \"Huntington's disease (HD) is an inherited NDD caused by autosomal-dominant expanded CAG trinucleotide repeat mutation in the gene coding for Huntingtin (Htt).\"],\n",
       "   'exact_answer': ['dominant'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5e31cbd4fbd6abf43b00004f',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 252,\n",
       "     'text': \"Huntington's Disease (HD) is an autosomal dominant neurodegenerative disease characterized by gradual deterioration of motor and cognitive functions and development of psychiatric deficits. Animal models provide powerful means to study the pathological\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28927719',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 197,\n",
       "     'offsetInEndSection': 356,\n",
       "     'text': \"Huntington's disease (HD) is an inherited NDD caused by autosomal-dominant expanded CAG trinucleotide repeat mutation in the gene coding for Huntingtin (Htt). \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29134321',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 176,\n",
       "     'offsetInEndSection': 346,\n",
       "     'text': \" Huntington's disease (HD) is an autosomal-dominant, progressive neurodegenerative disease characterized by movement disorder, psychiatric symptoms and cognitive decline.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29413175',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1029,\n",
       "     'offsetInEndSection': 1264,\n",
       "     'text': \"The D4S10 locus, defined by the probe G8 and linked to the gene for Huntington's disease (HD), has permitted us to identify individuals with a high probability of being homozygous for this autosomal dominant neurodegenerative disorder.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2881213',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 510,\n",
       "     'offsetInEndSection': 644,\n",
       "     'text': \"Huntington's disease is a fatal autosomal dominant neurodegenerative disease caused by an abnormal CAG expansion in huntingtin's gene.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31286142',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 510,\n",
       "     'offsetInEndSection': 643,\n",
       "     'text': \"Huntington's disease is a fatal autosomal dominant neurodegenerative disease caused by an abnormal CAG expansion in huntingtin's gene\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31286142',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 144,\n",
       "     'text': \"Huntington's disease is an autosomal dominant neurodegenerative disease caused by a toxic gain of function mutation in the huntingtin gene (Htt)\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22119622',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 206,\n",
       "     'text': \"Huntington's disease (HD) is a late onset, incurable, autosomal dominantly-inherited, progressive neuropsychiatric disease, characterised by chorea, changes in personality, mood and behaviour, and dementia.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11723754',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': \"Huntington's disease (HD) is a genetically dominant neurodegenerative condition caused by an unique mutation in the disease gene huntingtin.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14526190',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 152,\n",
       "     'text': 'Huntington disease is an autosomal-dominant neurodegenerative disease of mid-life onset caused by expansion of a polymorphic trinucleotide (CAG) repeat.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25356969',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 130,\n",
       "     'text': 'Huntington disease (HD) is a well-defined autosomal dominant neurodegenerative disease caused by CAG repeat expansions in HD gene.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15764008',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is Amyand hernia?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29875977',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24303691',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29455120',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17990042',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23591313',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30049909',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19478627',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24765286',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30151100'],\n",
       "   'ideal_answer': ['An Amyand hernia is a rare disease where the appendix is found within an inguinal hernia sac, which may or may not contain other abdominal contents or pathologic inflammatory changes.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5e2ad60e76af173751000003',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 241,\n",
       "     'text': \"INTRODUCTION: An Amyand's hernia is a heterogeneous clinical condition defined by the presence of the vermiform appendix within an inguinal hernia sac, which may or may not contain other abdominal contents or pathologic inflammatory changes.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29455120',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 108,\n",
       "     'text': 'INTRODUCTION: An Amyand hernia is a rare disease where the appendix is found within an inguinal hernia sac. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1438,\n",
       "     'offsetInEndSection': 1660,\n",
       "     'text': \"CONCLUSION: This case report reviews a rare entity known as an Amyand's hernia that presented as an incarcerated hernia that was diagnosed intraoperatively with an inflamed appendix, recognized as a type 2 Amyand's hernia.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': \"Claudius Amyand's hernia is defined as the incarceration of the vermiform appendix in the hernia sac.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29875977',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 144,\n",
       "     'offsetInEndSection': 278,\n",
       "     'text': \"Amyand's hernia is an inguinal hernia; a protrusion of abdominal cavity content through the inguinal canal, with a vermiform appendix.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30049909',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 196,\n",
       "     'text': 'The presence of an incarcerated vermiform appendix within a femoral hernia defect, a De Garengeot hernia, is distinctly different than an inguinal hernia containing the appendix, an Amyand hernia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30151100',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 107,\n",
       "     'text': 'INTRODUCTION\\n\\nAn Amyand hernia is a rare disease where the appendix is found within an inguinal hernia sac.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 106,\n",
       "     'text': 'INTRODUCTION An Amyand hernia is a rare disease where the appendix is found within an inguinal hernia sac.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'The term Amyand hernia refers to presence of appendix within inguinal hernia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24765286',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 79,\n",
       "     'text': \"BACKGROUND Amyand's hernia is an inguinal hernia containing vermiform appendix.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17990042',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 65,\n",
       "     'text': 'Appendix-containing inguinal hernias are known as Amyand hernias.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19478627',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 95,\n",
       "     'text': 'An Amyand hernia is a rare disease where the appendix is found within an inguinal hernia sac . ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 143,\n",
       "     'offsetInEndSection': 278,\n",
       "     'text': \"Amyand's hernia is an inguinal hernia; a protrusion of abdominal cavity content through the inguinal canal, with a vermiform appendix.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30049909',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 217,\n",
       "     'text': \"The eponym 'Amyand hernia' coined in recognition of Claudius Amyand, the first surgeon to successfully perform appendectomy in a hernia sac containing appendix, refers to the vermiform appendix within inguinal hernia.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23591313',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 112,\n",
       "     'text': 'The finding of the appendix inside an hernial sac is called \"Amyand hernia\": The global incidence is 0.28 to 1%.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24303691',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 391,\n",
       "     'offsetInEndSection': 498,\n",
       "     'text': ' Introduction An Amyand hernia is a rare disease where the appendix is found within an inguinal hernia sac.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1463,\n",
       "     'offsetInEndSection': 1621,\n",
       "     'text': \"Discussion The current generally accepted treatment algorithm for Amyand's hernia is essentially contingent on the appendix's condition within the hernia sac.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29753277',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Does radiotherapy for prostate cancer increase bladder cancer risk?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27432454',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28499661',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25900243',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19553822',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27844130',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27370205',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26936410',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27629559',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30293908'],\n",
       "   'ideal_answer': ['Yes, radiotherapy for prostate cancer is associated with increased bladder cancer risk.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '5e33916afbd6abf43b000061',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'External Beam Radiotherapy Increases the Risk of Bladder Cancer When Compared with Radical Prostatectomy in Patients Affected by Prostate Cancer: A Population-based Analysis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30293908',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1817,\n",
       "     'offsetInEndSection': 2035,\n",
       "     'text': 'On multivariable competing risk regression analyses, treatment with EBRT was independently associated with the risk of developing a second primary BCa (hazard ratio: 1.35, CI: 1.18-1.55; p<0.001), but not RCa (p=0.4). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30293908',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2180,\n",
       "     'offsetInEndSection': 2456,\n",
       "     'text': 'CONCLUSIONS: Patients treated with EBRT are at increased risk of developing a second primary BCa compared with those treated with RP. However, no differences were found considering RCa incidence in patients treated with RP or EBRT within the first 5 yr after primary therapy. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30293908',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2777,\n",
       "     'offsetInEndSection': 2914,\n",
       "     'text': 'We found that those treated with external beam radiotherapy are at an increased risk of developing a second primary bladder cancer tumor.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30293908',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 708,\n",
       "     'offsetInEndSection': 1018,\n",
       "     'text': 'All radiation modalities were found to have an increased RR of developing BlCa after 10\\xa0years, with brachytherapy having a significantly higher RR than external beam radiation (EBRT) or combined EBRT and brachytherapy in Caucasian men and a significantly higher RR than EBRT in men of other/unknown ethnicity. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27629559',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1168,\n",
       "     'offsetInEndSection': 1427,\n",
       "     'text': 'CONCLUSIONS: The increased risk of BlCa after prostate radiation occurs predominantly after 10\\xa0years, regardless of ethnicity. The RR of developing BlCa after 10\\xa0years is significantly higher following brachytherapy than after EBRT or EBRT and brachytherapy. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27629559',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 328,\n",
       "     'offsetInEndSection': 716,\n",
       "     'text': 'Based on the data in the literature, there is a\\xa0consistently increased risk of bladder cancer (HR: 1.67, 95% CI 1.55-1.80), rectal cancer (HR: 1.79, 95% CI 1.34-2.38), and colorectal cancer (HR: 1.79, 95% CI 1.34-23.8) following percutaneous radiation therapy. Following brachytherapy only an increased for the development of bladder cancer (HR: 2.14, 95% CI 1.03-3.94) has been observed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27844130',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 816,\n",
       "     'offsetInEndSection': 956,\n",
       "     'text': 'When comparing with a matched general French population, the standard incidence ratio (SIR) for bladder cancer was 1.02 (95% CI: 0.46-1.93).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28499661',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1637,\n",
       "     'offsetInEndSection': 1734,\n",
       "     'text': 'LDR resulted in lower bladder cancer risks than HDR, and lower or similar risks of rectal cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27370205',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1807,\n",
       "     'offsetInEndSection': 1914,\n",
       "     'text': 'Compared to external beam techniques, second rectal and bladder cancer risks were lowest for brachytherapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27370205',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 241,\n",
       "     'text': 'OBJECTIVE: Although it is well known that radiotherapy for prostate cancer increases comorbid rate of secondary bladder cancer, the effect of aging and smoking with radiotherapy on incidence rate of secondary bladder cancer remains unknown. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27432454',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 738,\n",
       "     'offsetInEndSection': 978,\n",
       "     'text': 'RESULTS: During the median follow-up period of 4.3 and 3.1 years, secondary bladder cancer occurred in 11 (3.4%) and 5 (1.1%) of patients with prostate cancer treated with external beam radiotherapy and radical prostatectomy, respectively. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27432454',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1745,\n",
       "     'offsetInEndSection': 1951,\n",
       "     'text': 'CONCLUSIONS\\n\\nMen who receive radiotherapy for localized prostate cancer have an increased risk of bladder cancer compared to patients undergoing radical prostatectomy and compared to the general population.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': 'Radiation therapy for prostate cancer increases subsequent risk of bladder and rectal cancer: a population based cohort study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 814,\n",
       "     'offsetInEndSection': 1035,\n",
       "     'text': 'RESULTS\\n\\nThe relative risk of bladder cancer developing after external beam radiotherapy, brachytherapy and external beam radiotherapy-brachytherapy compared to radical prostatectomy was 1.88, 1.52 and 1.85, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1036,\n",
       "     'offsetInEndSection': 1308,\n",
       "     'text': 'Compared to the general United States population the standardized incidence ratio for bladder cancer developing after radical prostatectomy, external beam radiotherapy, brachytherapy and external beam radiotherapy-brachytherapy was 0.99, 1.42, 1.10 and 1.39, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 240,\n",
       "     'text': 'OBJECTIVE\\n\\nAlthough it is well known that radiotherapy for prostate cancer increases comorbid rate of secondary bladder cancer, the effect of aging and smoking with radiotherapy on incidence rate of secondary bladder cancer remains unknown.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27432454',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 240,\n",
       "     'text': 'OBJECTIVE\\nAlthough it is well known that radiotherapy for prostate cancer increases comorbid rate of secondary bladder cancer, the effect of aging and smoking with radiotherapy on incidence rate of secondary bladder cancer remains unknown.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27432454',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1745,\n",
       "     'offsetInEndSection': 1951,\n",
       "     'text': 'CONCLUSIONS\\nMen who receive radiotherapy for localized prostate cancer have an increased risk of bladder cancer compared to patients undergoing radical prostatectomy and compared to the general population.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 170,\n",
       "     'text': 'PURPOSE\\nPre-prostate specific antigen era series demonstrated an increased risk of bladder cancer and rectal cancer in men who received radiotherapy for prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 814,\n",
       "     'offsetInEndSection': 1035,\n",
       "     'text': 'RESULTS\\nThe relative risk of bladder cancer developing after external beam radiotherapy, brachytherapy and external beam radiotherapy-brachytherapy compared to radical prostatectomy was 1.88, 1.52 and 1.85, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1551,\n",
       "     'offsetInEndSection': 1835,\n",
       "     'text': 'CONCLUSIONS: Taken together, these findings suggest that smoking history might be one of criteria to choose radical prostatectomy than external beam radiotherapy for prostate cancer, and that age would not be a criterion for therapeutic selection in terms of secondary bladder cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27432454',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 170,\n",
       "     'text': 'PURPOSE: Pre-prostate specific antigen era series demonstrated an increased risk of bladder cancer and rectal cancer in men who received radiotherapy for prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 810,\n",
       "     'offsetInEndSection': 1031,\n",
       "     'text': 'RESULTS: The relative risk of bladder cancer developing after external beam radiotherapy, brachytherapy and external beam radiotherapy-brachytherapy compared to radical prostatectomy was 1.88, 1.52 and 1.85, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1032,\n",
       "     'offsetInEndSection': 1304,\n",
       "     'text': 'Compared to the general United States population the standardized incidence ratio for bladder cancer developing after radical prostatectomy, external beam radiotherapy, brachytherapy and external beam radiotherapy-brachytherapy was 0.99, 1.42, 1.10 and 1.39, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 161,\n",
       "     'text': 'Pre-prostate specific antigen era series demonstrated an increased risk of bladder cancer and rectal cancer in men who received radiotherapy for prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1701,\n",
       "     'offsetInEndSection': 1894,\n",
       "     'text': 'Men who receive radiotherapy for localized prostate cancer have an increased risk of bladder cancer compared to patients undergoing radical prostatectomy and compared to the general population.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 780,\n",
       "     'offsetInEndSection': 992,\n",
       "     'text': 'The relative risk of bladder cancer developing after external beam radiotherapy, brachytherapy and external beam radiotherapy-brachytherapy compared to radical prostatectomy was 1.88, 1.52 and 1.85, respectively.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18801517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 108,\n",
       "     'text': 'Radiotherapy for prostate cancer is associated with an increased incidence of secondary bladder cancer (BC).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25900243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 884,\n",
       "     'offsetInEndSection': 977,\n",
       "     'text': 'Radiation therapy for prostate cancer is associated with an increased risk of bladder cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19553822',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2204,\n",
       "     'offsetInEndSection': 2429,\n",
       "     'text': 'Radiotherapy for prostate cancer was associated with higher risks of developing second malignancies of the bladder, colon, and rectum compared with patients unexposed to radiotherapy, but the reported absolute rates were low.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26936410',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the function of WAPL protein on cohesin?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/24055153',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27797072',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17141150',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29447171',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29217591',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28424523',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17112726',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23034634',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17113138',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28475897'],\n",
       "   'ideal_answer': [\"Human Wapl is a cohesin-binding protein that promotes sister-chromatid resolution in mitotic prophase We show that the human ortholog of Wapl is a cohesin-binding protein that facilitates cohesin's timely release from chromosome arms during prophase.\",\n",
       "    \"Wapl is a cohesin-binding protein that facilitates cohesin's timely release from chromosome arms during prophase.It promotes the release of cohesin from chromosomes during both interphase and mitosis.\",\n",
       "    \"Human Wapl is a cohesin-binding protein that promotes sister-chromatid resolution in mitotic prophase. We show that the human ortholog of Wapl is a cohesin-binding protein that facilitates cohesin's timely release from chromosome arms during prophase.\"],\n",
       "   'exact_answer': ['Wapl is a cohesin unloading factor'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5d35be1cb3a6380763000005',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Human Wapl is a cohesin-binding protein that promotes sister-chromatid resolution in mitotic prophase',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17112726',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 761,\n",
       "     'offsetInEndSection': 909,\n",
       "     'text': \"We show that the human ortholog of Wapl is a cohesin-binding protein that facilitates cohesin's timely release from chromosome arms during prophase.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17112726',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1264,\n",
       "     'offsetInEndSection': 1419,\n",
       "     'text': 'Conversely, overexpression of Wapl causes premature separation of sister chromatids. Wapl physically associates with cohesin in HeLa-cell nuclear extracts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17112726',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 63,\n",
       "     'text': 'Wapl controls the dynamic association of cohesin with chromatin',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17113138',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 379,\n",
       "     'offsetInEndSection': 997,\n",
       "     'text': 'Here, we show that the interaction between cohesin and chromatin is controlled by Wapl, a protein implicated in heterochromatin formation and tumorigenesis. Wapl is associated with cohesin throughout the cell cycle, and its depletion blocks cohesin dissociation from chromosomes during the early stages of mitosis and prevents the resolution of sister chromatids until anaphase, which occurs after a delay. Wapl depletion also increases the residence time of cohesin on chromatin in interphase. Our data indicate that Wapl is required to unlock cohesin from a particular state in which it is stably bound to chromatin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17113138',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 108,\n",
       "     'offsetInEndSection': 386,\n",
       "     'text': 'In recent articles published in Cell and Current Biology, the characterization of Wapl, a newly identified cohesin-interacting protein, suggests that a dynamic interaction between the cohesin complex and chromatin is important for normal regulation of sister chromatid cohesion.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17141150',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 139,\n",
       "     'text': 'Wapl protein regulates binding of the cohesin complex to chromosomes during interphase and helps remove cohesin from chromosomes at mitosis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23034634',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1088,\n",
       "     'offsetInEndSection': 1316,\n",
       "     'text': 'Wapl-AG was found to increase the stability of cohesin binding to polytene chromosomes. Our data suggest that increasing cohesin stability interferes with PcG silencing at genes that are co-regulated by cohesin and PcG proteins.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23034634',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 110,\n",
       "     'offsetInEndSection': 236,\n",
       "     'text': '. First, during prophase and prometaphase, the bulk of cohesin is driven from chromosome arms by the cohesin antagonist WAPL. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24055153',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 807,\n",
       "     'offsetInEndSection': 1245,\n",
       "     'text': 's a consequence, WAPL-depleted cells undergo anaphase with segregation errors, including both lagging chromosomes and catenanes, resulting in micronuclei and DNA damage. Stable WAPL depletion arrests cells in a p53-dependent manner but causes p53-deficient cells to become highly aneuploid. Our data show that the WAPL-dependent prophase pathway is essential for proper chromosome segregation and is crucial to maintain genomic integrity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24055153',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 417,\n",
       "     'offsetInEndSection': 621,\n",
       "     'text': 'cohesin suppresses compartments but is required for TADs and loops, that CTCF defines their boundaries, and that the cohesin unloading factor WAPL and its PDS5 binding partners control the length of loops',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29217591',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 907,\n",
       "     'offsetInEndSection': 1181,\n",
       "     'text': 'ohesin has an essential genome-wide function in mediating long-range chromatin interactions and support the hypothesis that cohesin creates these by loop extrusion, until it is delayed by CTCF in a manner dependent on PDS5 proteins, or until it is released from DNA by WAPL.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29217591',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 67,\n",
       "     'text': 'The Cohesin Release Factor WAPL Restricts Chromatin Loop Extension.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28475897',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 352,\n",
       "     'offsetInEndSection': 481,\n",
       "     'text': \"Cohesin's DNA release factor WAPL restricts this loop extension and also prevents looping between incorrectly oriented CTCF sites\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28475897',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 915,\n",
       "     'offsetInEndSection': 1027,\n",
       "     'text': 'We conclude that the balanced activity of SCC2/SCC4 and WAPL enables cohesin to correctly structure chromosomes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28475897',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 75,\n",
       "     'text': 'Cohesin is positioned in mammalian genomes by transcription, CTCF and Wapl.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28424523',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 602,\n",
       "     'offsetInEndSection': 737,\n",
       "     'text': 'the distribution of cohesin in the mouse genome depends on transcription, CTCF and the cohesin release factor Wings apart-like (Wapl). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28424523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 934,\n",
       "     'offsetInEndSection': 1100,\n",
       "     'text': \"In the absence of both CTCF and Wapl, cohesin accumulates in up to 70 kilobase-long regions at 3'-ends of active genes, in particular if these converge on each other.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28424523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 393,\n",
       "     'offsetInEndSection': 505,\n",
       "     'text': 'the cohesin regulator Wapl promotes the release of cohesin from chromosomes during both interphase and mitosis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27797072',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 88,\n",
       "     'text': 'Brca2, Pds5 and Wapl differentially control cohesin chromosome association and function.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29447171',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1089,\n",
       "     'offsetInEndSection': 1404,\n",
       "     'text': 'Pds5 and Wapl, but not Brca2, limit the distance that cohesin extends from origins, thereby determining which active genes, enhancers and silencers bind cohesin. Using RNA-seq we find that Brca2, Pds5 and Wapl influence the expression of most genes sensitive to Nipped-B and cohesin, largely in the same direction. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29447171',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which histone mark distinguishes active from inactive enhancers?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27851968',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24038352',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15448640',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22920947',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28732206',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24565409',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21160473',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26305225',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21106759',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25984238'],\n",
       "   'ideal_answer': [\"Histone H3K27ac separates active from poised enhancers and predicts developmental state . In contrast, elements of the second class 'poised enhancers' are linked to genes inactive in hESCs . They are involved in orchestrating early steps in embryogenesis, such as gastrulation, mesoderm formation and neurulation .\",\n",
       "    \"Enhancers cause a high level of transcription and activation of chromatin structure at target genes . Individual chromatin marks, such as H3K27ac, have been identified to distinguish active from inactive enhancers . In contrast, elements of the second class, which we term 'poised enhancers', are distinguished by the absence of H327ac and enrichment of histone H3 lysine 27 trimethylation .\",\n",
       "    'Conversion of inactive enhancers to an active state is marked by accumulation of H3K4me1 and H3K27ac histone marks.',\n",
       "    \"Hyperacetylation of histones H3 and H4, a mark of active chromatin, is established broadly across target loci by enhancers that function over long distances . In contrast, elements of the second class 'poised enhancers' are distinguished by the absence of H3K27ac and enrichment of histone H3 lysine 27 trimethylation . They are linked to genes inactive in hESCs and instead are involved in orchestrating early steps in embryogenesis, such as gastrulation and mesoderm formation .\",\n",
       "    'Monomethylation of histone H3 on Lys 27 (H3K27) is associated with active and inactive enhancers, respectively. An enhancer chromatin state signature associated withactive enhancers may be defined by high levels of H3 K27 acetylation, nucleosome displacement, hypersensitivity to sonication, and strong suppression of enhancer activity by DNase I.',\n",
       "    'We demonstrate that UTX, in a demethylase activity-independent manner, facilitates conversion of inactive enhancers in embryonic stem cells to an active (H3K4me1+/H3K27ac+) state by recruiting and coupling the enzymatic functions of MLL4 and p300. This work reveals a previously unrecognized cooperativity among enhancer-associated chromatin modulators, including a unique function for UTX, in establishing an \"active enhancer landscape\" and defines a detailed mechanism for the joint deposition of H3K4me1 and H3K27ac.'],\n",
       "   'exact_answer': ['H3K27ac', 'Histone 3 Lysine 27 acetylation'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5fe30e06a43ad31278000037',\n",
       "   'snippets': [{'offsetInBeginSection': 226,\n",
       "     'offsetInEndSection': 474,\n",
       "     'text': 'We demonstrate that UTX, in a demethylase activity-independent manner, facilitates conversion of inactive enhancers in embryonic stem cells to an active (H3K4me1+/H3K27ac+) state by recruiting and coupling the enzymatic functions of MLL4 and p300. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28732206',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 896,\n",
       "     'offsetInEndSection': 1167,\n",
       "     'text': 'This work reveals a previously unrecognized cooperativity among enhancer-associated chromatin modulators, including a unique function for UTX, in establishing an \"active enhancer landscape\" and defines a detailed mechanism for the joint deposition of H3K4me1 and H3K27ac.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28732206',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 813,\n",
       "     'offsetInEndSection': 1168,\n",
       "     'text': \"In contrast, elements of the second class, which we term 'poised enhancers', are distinguished by the absence of H3K27ac, enrichment of histone H3 lysine 27 trimethylation (H3K27me3), and are linked to genes inactive in hESCs and instead are involved in orchestrating early steps in embryogenesis, such as gastrulation, mesoderm formation and neurulation.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21160473',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 287,\n",
       "     'offsetInEndSection': 412,\n",
       "     'text': 'Individual chromatin marks, such as H3K27ac and H3K27me3, have been identified to distinguish active from inactive enhancers.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24038352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 718,\n",
       "     'offsetInEndSection': 842,\n",
       "     'text': 'We find that histone H3K27ac distinguishes active enhancers from inactive/poised enhancer elements containing H3K4me1 alone.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21106759',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1016,\n",
       "     'offsetInEndSection': 1121,\n",
       "     'text': 'rthermore, like H3K27ac, H3K9ac and H3K14ac can also differentiate active enhancers from inactive ones. A',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22920947',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1783,\n",
       "     'offsetInEndSection': 1914,\n",
       "     'text': ' observation suggests that histone acetyl transferases (HATs) prime inactive genes by H3K14ac for stimuli dependent activation. In ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22920947',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 971,\n",
       "     'offsetInEndSection': 1167,\n",
       "     'text': 'e level of binding signal for TAL1 or GATA1 did not help distinguish TAL1-bound DNA segments as active versus inactive enhancers, nor did the density of regulation-related histone modifications. A',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25984238',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 718,\n",
       "     'offsetInEndSection': 882,\n",
       "     'text': 'Moreover, repressed regions are trimethylated at lysines 9 and 27, suggesting that these histone modifications represent a mark for inactive PcG-controlled regions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15448640',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 291,\n",
       "     'offsetInEndSection': 417,\n",
       "     'text': 'vidual chromatin marks, such as H3K27ac and H3K27me3, have been identified to distinguish active from inactive enhancers. Howe',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24038352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1309,\n",
       "     'offsetInEndSection': 1589,\n",
       "     'text': 'he active modification H3K4m3 are characterized by lower nucleosome occupancy before the transcription start site compared to genes marked by the inactive modification H3K27m3, while bivalent domains, genes associated with both marks, lie exactly in the middle. Combinatorial patt',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26305225',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What causes yellowing of the skin and eyes, also known as jaundice, in patients with liver failure?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27904243',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21498981',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8666325',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30441460',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29187150',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24834192',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26015697',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26052974',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/14765767',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30266131'],\n",
       "   'ideal_answer': ['Jaundice refers to yellow coloration of the skin and the sclera (white of the eyes) of newborn babies that result from the accumulation of bilirubin in the skin and mucous membranes.'],\n",
       "   'type': 'summary',\n",
       "   'id': '5e67b8bb1af46fc13000001b',\n",
       "   'snippets': [{'offsetInBeginSection': 153,\n",
       "     'offsetInEndSection': 394,\n",
       "     'text': 'n enzyme required to convert bilirubin into a more soluble form that can then be removed from the body. Absence or severe deficiency of this enzyme can lead to bilirubin accumulation in the body resulting in yellow skin and eyes (jaundice). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30266131',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 20,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': ' jaundice refers to yellow coloration of the skin and the sclera (whites of the eyes) of newborn babies that result from the accumulation of bilirubin in the skin and mucous membranes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26015697',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 654,\n",
       "     'offsetInEndSection': 879,\n",
       "     'text': 'however, are hindered by the relatively limited knowledge about the impact of different amounts of extravascular bilirubin on skin spectral responses and on the onset of jaundice, the resulting yellow-tinted skin appearance. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30441460',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 173,\n",
       "     'text': 'Febrile jaundice results clinically in generalized yellow coloration of the teguments and mucous membranes due to excess plasma bilirubin, accompanied by fever. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29187150',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 128,\n",
       "     'text': 'Jaundice is a yellowish pigmentation of skin and mucous membranes caused by hyperbilirubinemia, which itself has various causes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21498981',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 717,\n",
       "     'offsetInEndSection': 803,\n",
       "     'text': ' signs of jaundice occur when the serum bilirubin level exceeds 2.5 to 3 mg/dL. In all',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27904243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'Jaundice is a clinical manifestation of disorders of underlying bilirubin metabolism, hepatocellular dysfunction, or biliary obstruction. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27904243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Jaundice is a clinical manifestation of disorders of underlying bilirubin metabolism, hepatocellular dysfunction, or biliary obstruction.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27904243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 709,\n",
       "     'offsetInEndSection': 793,\n",
       "     'text': 'Clinical signs of jaundice occur when the serum bilirubin level exceeds 2.5 to 3 mg/',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27904243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 235,\n",
       "     'offsetInEndSection': 375,\n",
       "     'text': 'Prehepatic causes of jaundice include hemolysis and hematoma resorption, which lead to elevated levels of unconjugated (indirect) bilirubin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/14765767',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 861,\n",
       "     'offsetInEndSection': 1021,\n",
       "     'text': 'The most common causes of jaundice were drug-induced hepatitis, occurring in 11 patients (31 percent), and alcoholic liver disease, occurring in 5 (13 percent).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8666325',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 165,\n",
       "     'text': 'Jaundice is one of the notable markers of liver malfunction in our body, revealing a significant rise in the concentration of an endogenous yellow pigment bilirubin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26052974',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 709,\n",
       "     'offsetInEndSection': 796,\n",
       "     'text': 'Clinical signs of jaundice occur when the serum bilirubin level exceeds 2.5 to 3 mg/dL.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27904243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 196,\n",
       "     'offsetInEndSection': 452,\n",
       "     'text': 'This destruction has two consequences: leaking ALT and AST liver enzymes which increases during the course of disease and accumulation of bilirubin- a red pigmented compound released from dead red cells- which causes the yellow coloration of eyes and skin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24834192',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the chemical structure of Etanercept (ETN)?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29411222',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26665003',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24308717',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12510367',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23961669',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29020515',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24611432',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29887576',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28638112',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11336570'],\n",
       "   'ideal_answer': ['Etanercept (ETN) is a soluble fusion protein of the tumor necrosis factor receptor (TNFR) extracellular domain, linked to an Fc part of IgG1. It possesses three N- and 13 O-glycosylation sites, which form a complex with the plasma membrane protein Enbrel. Etanercept has been tested for treatment of solid cancers, including  glioblastoma and neuroblastoma, and liver fibrosis.',\n",
       "    'Etanercept (ETN) is a soluble fusion protein of the tumor necrosis factor receptor (TNFR) extracellular domain, linked to an Fc part of IgG1. It possesses three N- and 13 O-glycosylation sites. Etanercept has the ability to bind to TNF-α and TGF-β, and thus is a potential novel therapeutic option for the treatment of cancer.',\n",
       "    'Etanercept is a soluble fusion protein of the tumor necrosis factor receptor (TNFR) extracellular domain, linked to an Fc part of IgG1. It possesses three N- and 13 O-glycosylation sites.',\n",
       "    'Etanercept has the best retention rate in rheumatic diseases, but is less or not effective in granulomatous diseases, such as inflammatory bowel diseases or uveitis. Etanercept is a highly glycosylated therapeutic Fc-fusion protein that contains multiple N- and O-glycosylation sites.',\n",
       "    'Etanercept has the best retention rate in rheumatic diseases, but is less or not effective in granulomatous diseases, such as inflammatory bowel diseases or uveitis. Etanercept is a highly glycosylated therapeutic Fc-fusion protein that contains multiple N- and O-glycosylation sites. Etanercept is a TNFα receptor Fc fusion protein used for the treatment of rheumatic disease and psoriasis. Etanercept is a soluble fusion protein of the tumor necrosis factor receptor (TNFR) extracellular domain, linked to an Fc part of IgG1.',\n",
       "    'Etanercept has the best retention rate in rheumatic diseases, but is less effective in granulomatous diseases, such as inflammatory bowel diseases or uveitis . It possesses three N- and 13 O-glycosylation sites, with multiple N-and-IgG1-N-G sites . It is a fusion protein, composed of the Fc portion of IgG1 and the extracellular domain of the TNF receptor .'],\n",
       "   'type': 'summary',\n",
       "   'id': '5fe31306a43ad3127800003c',\n",
       "   'snippets': [{'offsetInBeginSection': 412,\n",
       "     'offsetInEndSection': 577,\n",
       "     'text': 'Etanercept has the best retention rate in rheumatic diseases, but is less or not effective in granulomatous diseases, such as inflammatory bowel diseases or uveitis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24611432',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 118,\n",
       "     'text': 'Etanercept is a highly glycosylated therapeutic Fc-fusion protein that contains multiple N- and O-glycosylation sites.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24308717',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 107,\n",
       "     'text': 'Etanercept is a TNFα receptor Fc fusion protein used for the treatment of rheumatic disease and psoriasis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28638112',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 188,\n",
       "     'text': 'Etanercept is a soluble fusion protein of the tumor necrosis factor receptor (TNFR) extracellular domain, linked to an Fc part of IgG1. It possesses three N- and 13 O-glycosylation sites. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29411222',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 448,\n",
       "     'offsetInEndSection': 569,\n",
       "     'text': 'Etanercept is a fusion protein, composed of the Fc portion of IgG1 and the extracellular domain of the TNF receptor (p75)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11336570',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 190,\n",
       "     'text': 'Etanercept is a dimeric genetic recombinant glycoprotein consisting of Fc domain of human Immunoglobulin G1 and the extracellular domain of human tumor necrosis factor (TNF) receptor type II',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29887576',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 126,\n",
       "     'text': 'Etanercept (ETN) is a fusion protein of the receptor (CD120b) for tumor necrosis factor (TNF) and the Fc portion of IgG1. A ph',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23961669',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 367,\n",
       "     'offsetInEndSection': 466,\n",
       "     'text': 'Etanercept (ETN), a recombinant TNF receptor (p75)-Fc fusion protein, competitively inhibits TNF-α.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26665003',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 121,\n",
       "     'text': 'Etanercept (ETN) is a fusion protein of the receptor (CD120b) for tumor necrosis factor (TNF) and the Fc portion of IgG1.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23961669',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 147,\n",
       "     'offsetInEndSection': 324,\n",
       "     'text': 'ETN is synthesized in Chinese hamster ovary cells by recombinant DNA technology as a fusion protein, with a fully human TNFRII ectodomain linked to the Fc portion of human IgG1.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29020515',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 191,\n",
       "     'text': 'Etanercept is a dimeric genetic recombinant glycoprotein consisting of Fc domain of human Immunoglobulin G1 and the extracellular domain of human tumor necrosis factor (TNF) receptor type II.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29887576',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 448,\n",
       "     'offsetInEndSection': 570,\n",
       "     'text': 'Etanercept is a fusion protein, composed of the Fc portion of IgG1 and the extracellular domain of the TNF receptor (p75).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11336570',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 119,\n",
       "     'text': 'Etanercept is a protein comprised of the extracellular domains of two TNF receptors attached to a Fc portion of an IgG.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12510367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': 'Etanercept is a soluble fusion protein of the tumor necrosis factor receptor (TNFR) extracellular domain, linked to an Fc part of IgG1.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29411222',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Does an antiphlogistic promotes inflammation?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/16756015',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/9595260',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/125891',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28208246',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/8254024',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10321035',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2806531',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7286205',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28972949',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17704978'],\n",
       "   'ideal_answer': ['Antiinflammatory agents: new series of N-substituted amino acids with complex pyrimidine structures endowed with antiphlogistic activity.',\n",
       "    'An  antiphlogistic drug counteracts inflammation'],\n",
       "   'exact_answer': 'no',\n",
       "   'type': 'yesno',\n",
       "   'id': '601c18921cb411341a00000f',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 464,\n",
       "     'text': 'The therapeutic effect of olipiphate was demonstrated for chronic inflammation of advanced arthritis and concanavalin A-related acute edema. The best systemic effect was obtained with 50 mg/kg, symptomatic--100 mg/kg. Skin wounds treated with 5% olipiphate (26 + 2) healed faster than those treated with 2% solcoseryl (30 + 0.8) or in control (33 + 0.6). It was shown histologically that the proliferative and antiphlogistic effect of olipiphate involved no scars.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16756015',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 333,\n",
       "     'offsetInEndSection': 558,\n",
       "     'text': 'Moreover, we observed an in vitro-inhibition of human neutrophil elastase, a protease involved in the inflammatory process, by extracts and fractions from yarrow, which suggests additional mechanisms of antiphlogistic action.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17704978',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 788,\n",
       "     'offsetInEndSection': 1076,\n",
       "     'text': 'Blood 5-HT in adrenalectomized rats with inflammationadrenalectomized rats 42 days and 3 months old with inflammation after injection of phenylbutazone an increase of 5-HT was observed, but in 18-month-old animals in which antiphlogistic action is highest a decrease of 5-HT was observed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/125891',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1385,\n",
       "     'offsetInEndSection': 1559,\n",
       "     'text': 'These results indicate that methotrexate is a nonsteroidal antiinflammatory agent, the antiphlogistic action of which is due to increased adenosine release at inflamed sites.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/8254024',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 224,\n",
       "     'text': \"The antiphlogistic Ibuprofen incorporated in liposomes caused a decrease of the inflammatory edema induced by Carrageenan in the distal part of the rat's hind leg after both the intramuscular and percutaneous administration.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9595260',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 440,\n",
       "     'offsetInEndSection': 622,\n",
       "     'text': 'Enhancement of the immunoreactivity inhibition caused by the drugs was not proportional to the increase in their antiphlogistic effects determined by the Selye model of inflammation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7286205',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Antiinflammatory agents: new series of N-substituted amino acids with complex pyrimidine structures endowed with antiphlogistic activity.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10321035',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 875,\n",
       "     'offsetInEndSection': 1031,\n",
       "     'text': ' investigate whether the antiphlogistic ingredient may suppress the inflammatory response to ultraviolet (UV) irradiation, the SPF was determined in vivo. F',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28972949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 402,\n",
       "     'offsetInEndSection': 502,\n",
       "     'text': 'Antiphlogistics were found to enhance the membrane viscosity both in control and under inflammation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2806531',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1595,\n",
       "     'offsetInEndSection': 1954,\n",
       "     'text': 'e in vivo determination of the SPF. Evidence of anti-inflammatory activity of the sunscreen antiphlogistics bisabolol and panthenol was also not apparent in the UV model over a time course of 48 h. Conlusion: The antiphlogistic ingredients panthenol and bisabolol incorporated in the tested sunscreen formula do not interfere with erythema reddening and thus ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28972949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 846,\n",
       "     'offsetInEndSection': 1002,\n",
       "     'text': 'nts was analyzed in vitro. To investigate whether the antiphlogistic ingredient may suppress the inflammatory response to ultraviolet (UV) irradiation, the ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28972949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 234,\n",
       "     'offsetInEndSection': 486,\n",
       "     'text': 'The aim of this study was to analyze the formation of the most relevant inflammation mediators including proteins and lipids in human fibroblasts upon inflammatory stimulation and subsequent treatment with dexamethasone, a powerful antiphlogistic drug.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28208246',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which epigenetic marks are deposited by PRC1?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27410265',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27630184',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21311219',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25071008',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19462008',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25754661',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26151332',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18974828',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26564795',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23706298'],\n",
       "   'ideal_answer': ['PRC2 induces histone H3 lysine 27 (H3K27) trimethylation (H3K27me3), which is subsequently read by PRC1 that further catalyzes H2A monoubiquitination (H2Aub1), creating a transcriptional silent chromatin conformation.',\n",
       "    'H2A monoubiquitination (H2Aub1), catalyzed by Polycomb-Repressive Complex1 (PRC1) is a key epigenetic mark in Polycomb silencing . Stable X chromosome inactivation involves the PRC1 Polycomb complex and requires histone MACROH2A1 and the CULLIN3/SPOP ubiquitin E3 ligase .',\n",
       "    'Histone H2A monoubiquitination (H2Aub1), catalyzed by Polycomb-Repressive Complex1 (PRC1), is a key epigenetic mark in Polycomb silencing . Stable X chromosome inactivation involves the PRC1 Polycomb complex and requires histone MACROH2A1 and the CULLIN3/SPOP ubiquitin E3 ligase .'],\n",
       "   'exact_answer': ['H2Aub1'],\n",
       "   'type': 'factoid',\n",
       "   'id': '5d35ee08b3a6380763000012',\n",
       "   'snippets': [{'offsetInBeginSection': 395,\n",
       "     'offsetInEndSection': 571,\n",
       "     'text': ' In Arabidopsis thaliana, LHP1 co-localizes with H3K27me3 epigenetic marks throughout the genome and interacts with PRC1 and PRC2 members as well as with a long noncoding RNA. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27410265',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 536,\n",
       "     'offsetInEndSection': 694,\n",
       "     'text': 'When cells enter senescence the binding to RD of both PRC1 and PRC2 complexes is lost leading to a decreased level of histone H3K27 trimethylation (H3K27me3).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19462008',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 300,\n",
       "     'offsetInEndSection': 518,\n",
       "     'text': 'PRC2 induces histone H3 lysine 27 (H3K27) trimethylation (H3K27me3), which is subsequently read by PRC1 that further catalyzes H2A monoubiquitination (H2Aub1), creating a transcriptional silent chromatin conformation. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23706298',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 138,\n",
       "     'text': 'Histone H2A monoubiquitination (H2Aub1), catalyzed by Polycomb-Repressive Complex1 (PRC1), is a key epigenetic mark in Polycomb silencing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27630184',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 117,\n",
       "     'offsetInEndSection': 415,\n",
       "     'text': 'For a long time, the PcG mechanism has been proposed to follow a hierarchical recruitment of PcG repressive complexes (PRCs) to target genes in which the binding of PRC2 and the incorporation of H3 lysine 27 trimethyl marks led to recruitment of PRC1, which in turn mediated H2A monoubiquitination.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25754661',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': 'Histone H2A monoubiquitination (H2Aub1), catalyzed by Polycomb-Repressive Complex1 (PRC1), is a key epigenetic mark in Polycomb silencing. Ho',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27630184',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1060,\n",
       "     'offsetInEndSection': 1352,\n",
       "     'text': 'Intriguingly, many promoters were co-regulated by all three histone marks, becoming hypermethylated with loss of H3K4me3 or H3K27me3 and hypomethylated with depletion of H2AK119ub, and many of these co-regulated loci were among those commonly targeted for aberrant hypermethylation in cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25071008',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Givosiran is used for treatment of which disease?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/31792921',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32521132',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33043761',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32034693',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31994716',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30847674',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32592692',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33275677',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30726693',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33139979'],\n",
       "   'ideal_answer': ['Givosiran is approved for treatment of porphyria.'],\n",
       "   'exact_answer': ['porphyria'],\n",
       "   'type': 'factoid',\n",
       "   'id': '601c4b231cb411341a000020',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 111,\n",
       "     'text': 'Cost savings with hemin versus givosiran for the treatment of patients with acute intermittent porphyria (AIP).',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33043761',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 98,\n",
       "     'offsetInEndSection': 187,\n",
       "     'text': ' In 2019, FDA approved givosiran for the treatment of adults with acute hepatic porphyria',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33043761',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1796,\n",
       "     'offsetInEndSection': 1914,\n",
       "     'text': 'CONCLUSIONS: Hemin is expected to provide cost savings compared to givosiran for all AIP patients and subpopulations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33043761',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 756,\n",
       "     'offsetInEndSection': 936,\n",
       "     'text': 'New treatment options include givosiran (siRNA) to down-regulate ALA synthase-1 (ALAS1) and the messenger RNA of PBGD (PBGD mRNA) delivered to the liver cells of patients with AIP.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33139979',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 543,\n",
       "     'offsetInEndSection': 837,\n",
       "     'text': 'Management of AIP includes awareness and avoidance of triggering factors, infusions of hemin for severe acute attacks, and, if indicated for chronic suppressive therapy, maintenance treatment with hemin or givosiran, a small interfering RNA molecule that antagonizes ALA synthase 1 transcripts.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33275677',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 790,\n",
       "     'text': 'BACKGROUND: Induction of delta aminolevulinic acid synthase 1 ( ALAS1) gene expression and accumulation of neurotoxic intermediates result in neurovisceral attacks and disease manifestations in patients with acute intermittent porphyria, a rare inherited disease of heme biosynthesis. Givosiran is an investigational RNA interference therapeutic agent that inhibits hepatic ALAS1 synthesis.METHODS: We conducted a phase 1 trial of givosiran in patients with acute intermittent porphyria. In part A of the trial, patients without recent porphyria attacks (i.e., no attacks in the 6 months before baseline) were randomly assigned to receive a single subcutaneous injection of one of five ascending doses of givosiran (0.035, 0.10, 0.35, 1.0, or 2.5 mg per kilogram of body weight) or placebo.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30726693',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2029,\n",
       "     'offsetInEndSection': 2374,\n",
       "     'text': 'CONCLUSIONS: Once-monthly injections of givosiran in patients who had recurrent porphyria attacks resulted in mainly low-grade adverse events, reductions in induced ALAS1 mRNA levels, nearly normalized levels of the neurotoxic intermediates delta aminolevulinic acid and porphobilinogen, and a lower attack rate than that observed with placebo. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30726693',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 297,\n",
       "     'text': 'In November 2019 givosiran became the second small interfering RNA (siRNA)-based drug to receive US Food and Drug Administration (FDA) approval, it has been developed for the treatment of acute intermittent porphyria (AIP), a disorder characterized by life-threatening acute neurovisceral attacks.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31792921',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1466,\n",
       "     'offsetInEndSection': 1692,\n",
       "     'text': 'The clinical performance of givosiran revealed that suppression of ALAS1 by GalNac-decorated siRNAs represents an additional approach for the treatment of patients with AIP that manifests recurrent acute neurovisceral attacks.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31792921',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 77,\n",
       "     'text': 'Phase 3 Trial of RNAi Therapeutic Givosiran for Acute Intermittent Porphyria.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32521132',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 373,\n",
       "     'offsetInEndSection': 480,\n",
       "     'text': 'Givosiran is being developed by Alnylam Pharmaceuticals for the treatment of acute hepatic porphyria (AHP).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32034693',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 155,\n",
       "     'text': 'Givosiran is a small interfering ribonucleic acid agent that was recently approved in the United States for the treatment of acute hepatic porphyria (AHP).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31994716',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 229,\n",
       "     'offsetInEndSection': 487,\n",
       "     'text': 'This technology has led to the approval of givosiran for the treatment of acute hepatic porphyria, and there are another seven conjugates in registrational review or phase 3 trials and at least another 21 conjugates at earlier stages of clinical development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32592692',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 481,\n",
       "     'offsetInEndSection': 649,\n",
       "     'text': 'In November 2019, givosiran was approved in the USA for the treatment of adults with AHP based on the positive results from the multinational, phase III ENVISION trial.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32034693',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1296,\n",
       "     'offsetInEndSection': 1465,\n",
       "     'text': 'In preliminary results, givosiran achieved clinical endpoints for AIP, reducing urinary ALA levels, and presented a safety profile that enabled further drug development.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31792921',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 156,\n",
       "     'offsetInEndSection': 360,\n",
       "     'text': 'This phase I study evaluated the safety, pharmacokinetic, and pharmacodynamic profile of subcutaneously (SC) administered givosiran in patients with acute intermittent porphyria, the most common AHP type.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31994716',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1130,\n",
       "     'offsetInEndSection': 1308,\n",
       "     'text': 'Givosiran, another RNAi therapeutic, targeting 5-aminolevulinic acid synthase, has been positively tested in acute intermittent porphyria in phase 1/2 and ongoing phase 3 trials.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30847674',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What is the role of elagolix in treatment of uterine fibroids?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/30194661',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30570832',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30774352',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28579415',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31695514',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31650182',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33045114',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29476499',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32702363',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31971678'],\n",
       "   'ideal_answer': ['Elagolix is approved for the treatment of moderate to severe pain caused by endometriosis. Elagolix is also effective for heavy bleeding caused by uterine fibroids.'],\n",
       "   'type': 'summary',\n",
       "   'id': '6024a25c1cb411341a0000a1',\n",
       "   'snippets': [{'offsetInBeginSection': 1072,\n",
       "     'offsetInEndSection': 1152,\n",
       "     'text': 'One of them, Elagolix, is in the early stages of testing in women with fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30774352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1250,\n",
       "     'offsetInEndSection': 1534,\n",
       "     'text': 'Recently, elagolix received US FDA approval for the treatment of moderate to severe pain caused by endometriosis. Several clinical trials assessed the efficacy of elagolix for the treatment of heavy bleeding caused by UFs and the definitive results of Phase III studies are expected. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31695514',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 258,\n",
       "     'text': 'INTRODUCTION: Elagolix is a novel, orally active, non-peptide, competitive gonadotropin-releasing hormone (GnRH) receptor antagonist in development for the management of endometriosis with associated pain and heavy menstrual bleeding due to uterine fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29476499',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 261,\n",
       "     'offsetInEndSection': 388,\n",
       "     'text': 'In July 2018, the US FDA approved elagolix tablets for the management of moderate to severe pain associated with endometriosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30194661',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 667,\n",
       "     'offsetInEndSection': 872,\n",
       "     'text': 'Elagolix with and without low-dose hormone add-back therapy is also undergoing phase III clinical development for heavy menstrual bleeding associated with uterine fibroids in the aforementioned locations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30194661',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 137,\n",
       "     'text': 'Elagolix for the management of heavy menstrual bleeding associated with uterine fibroids: results from a phase 2a proof-of-concept study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28579415',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1564,\n",
       "     'offsetInEndSection': 1658,\n",
       "     'text': 'CONCLUSION(S): Elagolix significantly reduced heavy menstrual bleeding in women with fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28579415',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 69,\n",
       "     'text': 'Elagolix for Heavy Menstrual Bleeding in Women with Uterine Fibroids.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31971678',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1919,\n",
       "     'offsetInEndSection': 2045,\n",
       "     'text': 'CONCLUSIONS: Elagolix with add-back therapy was effective in reducing heavy menstrual bleeding in women with uterine fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31971678',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 867,\n",
       "     'offsetInEndSection': 1170,\n",
       "     'text': 'In Elaris Uterine Fibroids 1 and Uterine Fibroids 2, 2 replicate multicenter, double-blind, randomized, placebo-controlled, phase 3 studies, treatment of premenopausal women with elagolix with hormonal add-back therapy demonstrated reduction in heavy menstrual bleeding associated with uterine fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32702363',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 229,\n",
       "     'text': 'CONTEXT: Elagolix is an oral gonadotropin-releasing hormone (GnRH) antagonist recently approved for the treatment of endometriosis-associated pain and being developed for heavy menstrual bleeding associated with uterine fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31650182',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 316,\n",
       "     'offsetInEndSection': 486,\n",
       "     'text': 'Elagolix coadministered with estradiol/norethindrone acetate is in late-stage development for the management of heavy menstrual bleeding associated with uterine fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33045114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 223,\n",
       "     'text': 'CONTEXT: Elagolix is an oral gonadotropin-releasing hormone (GnRH) antagonist recently approved for the treatment of endometriosis-associated pain and being developed for heavy menstrual bleeding associated with uterine fib',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31650182',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 319,\n",
       "     'text': 'Elagolix is an oral gonadotropin-releasing hormone receptor antagonist indicated for the management of endometriosis-associated pain and in combination with estradiol/norethindrone acetate indicated for the management of heavy menstrual bleeding associated with uterine leiomyomas (fibroids) in premenopausal women. Ela',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33045114',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 221,\n",
       "     'text': 'CONTEXT: Elagolix is an oral gonadotropin-releasing hormone (GnRH) antagonist recently approved for the treatment of endometriosis-associated pain and being developed for heavy menstrual bleeding associated with uterine f',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31650182',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 867,\n",
       "     'offsetInEndSection': 1171,\n",
       "     'text': 'In Elaris Uterine Fibroids 1 and Uterine Fibroids 2, 2 replicate multicenter, double-blind, randomized, placebo-controlled, phase 3 studies, treatment of premenopausal women with elagolix with hormonal add-back therapy demonstrated reduction in heavy menstrual bleeding associated with uterine fibroids.O',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32702363',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 855,\n",
       "     'offsetInEndSection': 1732,\n",
       "     'text': ' ovulation. In Elaris Uterine Fibroids 1 and Uterine Fibroids 2, 2 replicate multicenter, double-blind, randomized, placebo-controlled, phase 3 studies, treatment of premenopausal women with elagolix with hormonal add-back therapy demonstrated reduction in heavy menstrual bleeding associated with uterine fibroids.OBJECTIVE: This analysis aimed to evaluate the safety and efficacy of elagolix (300 mg twice a day) with add-back therapy (1 mg estradiol/0.5 mg norethindrone acetate once a day) in reducing heavy menstrual bleeding associated with uterine fibroids in various subgroups of women over 6 months of treatment.STUDY DESIGN: Data were pooled from Elaris Uterine Fibroid-1 and Uterine Fibroid-2 studies, which evaluated premenopausal women (18-51 years) with heavy menstrual bleeding (>80 mL menstrual blood loss per cycle, alkaline hematin methodology) and ultrasound',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32702363',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 837,\n",
       "     'text': 'CONTEXT: Elagolix is an oral gonadotropin-releasing hormone (GnRH) antagonist recently approved for the treatment of endometriosis-associated pain and being developed for heavy menstrual bleeding associated with uterine fibroids.OBJECTIVE: The objective was to evaluate the effects of elagolix on ovulation and ovarian sex hormones.DESIGN AND SETTING: This was a randomized, open-label, multicenter study.PARTICIPANTS: Participants were healthy ovulatory women aged 18 to 40 years.INTERVENTIONS: Elagolix was administered orally for 3 continuous 28-day dosing intervals at 100 to 200 mg once daily (QD), 100 to 300 mg twice daily (BID), and 300 mg BID plus estradiol/norethindrone acetate (E2/NETA) 1/0.5 mg QD.MAIN OUTCOME MEASURES: The main outcomes measures were ovulation rates measured by transvaginal ultrasound, progesterone conce',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31650182',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1534,\n",
       "     'offsetInEndSection': 1813,\n",
       "     'text': 'Available data on elagolix and UFs showed that the drug, with or without low-dose hormone add-back therapy, is able to significantly reduce menstrual blood loss, lead to amenorrhea and improve haemoglobin concentrations in the majority of participants in comparison with placebo.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31695514',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which eukaryote genomes contain operons?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/20382830',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19740764',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19712444',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12075352',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17174523',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/23668349',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18218978',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12949121',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24429814',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24931407'],\n",
       "   'ideal_answer': ['Genes in nematode and ascidian genomes frequently occur in operons such genes comprise 15-20% of the coding genome for Caenorhabditis elegans and Ciona intestinalis We find that birth-death models of operon evolution reasonably describe the relative abundance of operons of different sizes in the C. elegans and Ciona genomes and generate predictions about the number of monocistronic, nonoperon genes that likely participate in the birth-death process',\n",
       "    'The eukaryote genomes contain operons, which can be classified into boader families based, particularly, on the presence of other types of genes. Operons have been found to be present in the following genomes: caenorhabditis elegans, ciona, c. elegans, trichinella spiralis, trichuris muris and ciona intestinalis. Operons have also been reported to exist in the non-coding regions of the human and plant genomes.',\n",
       "    'Genes in nematode and ascidian genomes frequently occur in operons. Such genes comprise 15-20% of the coding genome for Caenorhabditis elegans and Ciona intestinalis.  The organization of genes into operons, clusters of genes that are co-transcribed to produce polycistronic pre-mRNAs, is a trait found in a wide range of eukaryotic groups, including multiple animal phyla.',\n",
       "    'Operons are widespread in prokaryotes, but are uncommon in eukaryotes, except nematode worms, where approximately 15% of genes reside in over 1100 operons in the model organism Caenorhabditis elegans',\n",
       "    'Operons are widespread in prokaryotes, but are uncommon in eukaryotes, except nematode worms, where approximately 15% of genes reside in over 1100 operons in the model organism Caenorhabditis elegans. Birth-death models of operon evolution reasonably describe the relative abundance of operons of different sizes in the C. elegans and Ciona genomes'],\n",
       "   'exact_answer': [['C. elegans'],\n",
       "    ['C. intestinalis'],\n",
       "    ['C. briggsae'],\n",
       "    ['C. remanei'],\n",
       "    ['C. brenneri']],\n",
       "   'type': 'list',\n",
       "   'id': '5e9eaf3b0d431b5f73000004',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 66,\n",
       "     'text': 'Genes in nematode and ascidian genomes frequently occur in operons',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20382830',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 161,\n",
       "     'offsetInEndSection': 258,\n",
       "     'text': 'such genes comprise 15-20% of the coding genome for Caenorhabditis elegans and Ciona intestinalis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20382830',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 816,\n",
       "     'offsetInEndSection': 1103,\n",
       "     'text': 'We find that birth-death models of operon evolution reasonably describe the relative abundance of operons of different sizes in the C. elegans and Ciona genomes and generate predictions about the number of monocistronic, nonoperon genes that likely participate in the birth-death process',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20382830',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 51,\n",
       "     'text': 'Operons are a conserved feature of nematode genomes',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24931407',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': 'The organization of genes into operons, clusters of genes that are co-transcribed to produce polycistronic pre-mRNAs, is a trait found in a wide range of eukaryotic groups, including multiple animal phyla',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24931407',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 206,\n",
       "     'offsetInEndSection': 288,\n",
       "     'text': 'Operons are present in the class Chromadorea, one of the two main nematode classes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24931407',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1391,\n",
       "     'offsetInEndSection': 1632,\n",
       "     'text': ' Our data suggest that operons and \"spliced leader\" (SL) trans-splicing predate the radiation of the nematode phylum, an inference which is supported by the phylogenetic profile of proteins known to be involved in nematode SL trans-splicing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24931407',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1681,\n",
       "     'offsetInEndSection': 1795,\n",
       "     'text': 'Instead operons, polycistronic transcriptional units common in derived nematodes, seemingly adopted their function',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19712444',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 199,\n",
       "     'text': 'Operons are widespread in prokaryotes, but are uncommon in eukaryotes, except nematode worms, where approximately 15% of genes reside in over 1100 operons in the model organism Caenorhabditis elegans',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18218978',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 481,\n",
       "     'offsetInEndSection': 754,\n",
       "     'text': 'To test this hypothesis, we analyzed the presence and absence of C. elegans operons in Caenorhabditis briggsae, Caenorhabditis remanei, and Caenorhabditis brenneri, using Pristionchus pacificus and Brugia malayi as outgroups, and identified numerous operon gains and losses',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18218978',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 88,\n",
       "     'text': 'In the nematode Caenorhabditis elegans, up to 15% of the genes are organized in operons.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12949121',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 257,\n",
       "     'offsetInEndSection': 441,\n",
       "     'text': 'Here we address this issue by analysing transcription termination at the end of single protein expressing genes and genes located within operons in the nematode Caenorhabditis elegans.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19740764',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 816,\n",
       "     'offsetInEndSection': 1104,\n",
       "     'text': 'We find that birth-death models of operon evolution reasonably describe the relative abundance of operons of different sizes in the C. elegans and Ciona genomes and generate predictions about the number of monocistronic, nonoperon genes that likely participate in the birth-death process.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20382830',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1105,\n",
       "     'offsetInEndSection': 1239,\n",
       "     'text': 'This theory, and applications to C. elegans and Ciona, motivates several new and testable hypotheses about eukaryote operon evolution.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20382830',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 259,\n",
       "     'text': 'Genes in nematode and ascidian genomes frequently occur in operons--multiple genes sharing a common promoter to generate a polycistronic primary transcript--and such genes comprise 15-20% of the coding genome for Caenorhabditis elegans and Ciona intestinalis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20382830',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 797,\n",
       "     'offsetInEndSection': 981,\n",
       "     'text': 'e operon structure in the eukaryote Caenorhabditis elegans genome is a valuable, though underserved, tool for identifying physically or functionally interacting proteins. Based on the ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24429814',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 107,\n",
       "     'text': 'The genome of the nematode Caenorhabditis elegans is unusual among eukaryotes, in that it contains operons.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17174523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 466,\n",
       "     'offsetInEndSection': 546,\n",
       "     'text': 'This is one of the largest operons identified thus far in the C. elegans genome.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17174523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 94,\n",
       "     'text': 'The frataxin-encoding operon of Caenorhabditis elegans shows complex structure and regulation.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17174523',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 394,\n",
       "     'offsetInEndSection': 465,\n",
       "     'text': 'The C. elegans frataxin gene, frh-1, is encoded in the operon CEOP2232.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17174523',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 167,\n",
       "     'offsetInEndSection': 272,\n",
       "     'text': 'There are instances of operons found in C. elegans, Drosophila melanogaster and other eukaryotic species.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/23668349',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1062,\n",
       "     'offsetInEndSection': 1197,\n",
       "     'text': 'Our evidence indicates that the genome contains at least 1,000 operons, 2 8 genes long, that contain about 15% of all C. elegans genes.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12075352',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 538,\n",
       "     'offsetInEndSection': 661,\n",
       "     'text': 'Although only 28 operons have been reported, the complete sequence of the C. elegans genome reveals numerous gene clusters.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12075352',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is ofatumumab effective for multiple sclerosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/33092190',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33107072',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30985372',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30604390',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29695594',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30539801',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33090003',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30632834',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24453078',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32757523'],\n",
       "   'ideal_answer': ['Ofatumumab, a fully human anti-CD20 monoclonal antibody, is effective for relapsing forms of multiple sclerosis.'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '60234bd51cb411341a000093',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 81,\n",
       "     'text': 'Anti-CD20 Agents for Multiple Sclerosis: Spotlight on Ocrelizumab and Ofatumumab.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33092190',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1204,\n",
       "     'offsetInEndSection': 1367,\n",
       "     'text': 'Ofatumumab, a fully human anti-CD20 monoclonal antibody, that binds a distinct epitope, has been further investigated in phase 3 trials for relapsing forms of MS. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33092190',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 481,\n",
       "     'offsetInEndSection': 591,\n",
       "     'text': 'Conclusion: Ofatumumab offers beneficial outcomes for RMS\\xa0by reducing relapse and disability progression risk.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33090003',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 659,\n",
       "     'offsetInEndSection': 837,\n",
       "     'text': 'Currently, new therapies are emerging that promise more convenience and an improved safety profile (ofatumumab) or remyelinating potential with clinical improvement (opicinumab).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30604390',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 601,\n",
       "     'offsetInEndSection': 812,\n",
       "     'text': 'The emerging B-cell depleting therapies, particularly anti-CD20 agents such as rituximab, ocrelizumab, as well as the fully human ofatumumab, have shown promising clinical and magnetic resonance imaging benefit.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30539801',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 256,\n",
       "     'offsetInEndSection': 435,\n",
       "     'text': 'AREAS COVERED: In this manuscript, we review mechanisms of action, efficacy, safety, and tolerance of anti-CD20 therapies for MS, including rituximab, ocrelizumab, and ofatumumab.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30632834',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 737,\n",
       "     'offsetInEndSection': 791,\n",
       "     'text': 'Another CD20 directed mAb, ofatumumab, is in phase 3. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30985372',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 493,\n",
       "     'offsetInEndSection': 591,\n",
       "     'text': 'Ofatumumab offers beneficial outcomes for RMS\\xa0by reducing relapse and disability progression risk.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33090003',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1296,\n",
       "     'offsetInEndSection': 1493,\n",
       "     'text': 'CONCLUSION: Imaging showed that all subcutaneous ofatumumab doses demonstrated efficacy (most robust: cumulative doses ≥30 mg/12 wk), with a safety profile consistent with existing ofatumumab data.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29695594',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2318,\n",
       "     'offsetInEndSection': 2453,\n",
       "     'text': 'CONCLUSIONS: Among patients with multiple sclerosis, ofatumumab was associated with lower annualized relapse rates than teriflunomide. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32757523',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Please list 3 small molecule CGRP-Receptor antagonists for migraine',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/32799325',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32020557',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19939188',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31932515',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30725283',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28644160',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31096904',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32173558',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31020659',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32058712'],\n",
       "   'ideal_answer': ['Rimegepant and ubrogepant have been developed for acute migraine treatment, while atogepant is studied for migraine prophylaxis.'],\n",
       "   'exact_answer': [['ubrogepant'], ['rimegepant'], ['atogepant']],\n",
       "   'type': 'list',\n",
       "   'id': '601f1bbd1cb411341a000074',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 227,\n",
       "     'text': 'Ubrogepant (Ubrelvy™) is an orally administered, small molecule, highly-selective, calcitonin gene-related peptide (CGRP) antagonist that was developed by Allergan under license to Merck & Co. as an acute treatment for migraine',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32020557',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 359,\n",
       "     'offsetInEndSection': 524,\n",
       "     'text': ' This article summarizes the milestones in the development of ubrogepant leading to its first global approval for the acute treatment of migraine (±\\u2009aura) in adults.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32020557',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 104,\n",
       "     'offsetInEndSection': 382,\n",
       "     'text': ' a small molecule CGRP receptor antagonist and an anti-CGRP receptor antibody, can be used concomitantly to treat refractory migraine.METHODS: Case reports are presented of 2 patients participating in a long-term safety study of rimegepant 75 mg oral tablets for acute treatment',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31932515',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 462,\n",
       "     'text': 'The pivotal role of calcitonin gene-related peptide (CGRP) in migraine pathophysiology was identified over 30 years ago, but the successful clinical development of targeted therapies has only recently been realized. This Perspective traces the decades long evolution of medicinal chemistry required to advance small molecule CGRP receptor antagonists, also called gepants, including the current clinical agents rimegepant, vazegepant, ubrogepant, and atogepant. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32058712',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1167,\n",
       "     'offsetInEndSection': 1305,\n",
       "     'text': 'Currently, rimegepant and ubrogepant have been developed for acute migraine treatment, while atogepant is studied for migraine prophylaxis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32173558',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1047,\n",
       "     'offsetInEndSection': 1309,\n",
       "     'text': 'Small molecule CGRP receptor antagonists, gepants, are effective for acute relief of migraine headache, whereas monoclonal antibodies against CGRP (Eptinezumab, Fremanezumab, and Galcanezumab) or the CGRP receptor (Erenumab) effectively prevent migraine attacks.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30725283',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1530,\n",
       "     'offsetInEndSection': 1773,\n",
       "     'text': 'vel classes of drugs, i.e., the 5-HT1F receptor agonists (lasmiditan) and the small molecule CGRP receptor antagonists (gepants). Data on the effects of 5-HT1F receptor agonism in relation to MOH, as well as the effects of chronic CGRP recepto',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31096904',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 513,\n",
       "     'offsetInEndSection': 659,\n",
       "     'text': 'small-molecule CGRP receptor antagonist (ubrogepant, MK-1602) is currently in phase 3 studies for the acute treatment of migraine. Two of these an',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28644160',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 101,\n",
       "     'text': 'Safety of Rimegepant, an Oral CGRP Receptor Antagonist, Plus CGRP Monoclonal Antibodies for Migraine.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32799325',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 267,\n",
       "     'text': 'OBJECTIVE: Evaluate the safety and tolerability of oral rimegepant when used for acute treatment concomitantly with a monoclonal antibody (mAb) targeting the calcitonin gene-related peptide (CGRP) ligand or receptor (CGRP mAb) for the preventive treatment of migraine',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32799325',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 500,\n",
       "     'offsetInEndSection': 643,\n",
       "     'text': 'Meanwhile, 1 small-molecule CGRP receptor antagonist (ubrogepant, MK-1602) is currently in phase 3 studies for the acute treatment of migraine.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28644160',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 716,\n",
       "     'offsetInEndSection': 1088,\n",
       "     'text': 'ithin the past decade. The commentary is supplemented by information presented in journal articles and focuses on the activity of several major pharmaceutical companies in the field.CONCLUSION: Two small molecule CGRP receptor antagonists, olcegepant and telcagepant, have been shown to be clinically efficacious in the treatment of migraine, and thus provide validation o',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19939188',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1932,\n",
       "     'offsetInEndSection': 2204,\n",
       "     'text': 'Recently, orally administered next-generation small molecule CGRP-RAs have been shown to have safety and efficacy in acute treatment (ubrogepant and rimegepant) and prevention (atogepant) of migraine, giving additional CGRP-based therapeutic options for migraine patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31020659',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Describe a cytokine release syndrome.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/32776808',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32678378',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32166291',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33262810',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18382853',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32474885',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32983172',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32446778',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31755797',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/17471824'],\n",
       "   'ideal_answer': ['The major factor responsible for acute respiratory distress syndrome is the so-called \"cytokine storm,\" which is an aberrant response from the host immune system that induces an exaggerated release of proinflammatory cytokines/chemokines.',\n",
       "    'Cytokine release syndrome is defined by the release of cytokines from sarcoplasmic reticulum (SR) and is associated with retinal vein occlusions, fatigue and dyspnea.',\n",
       "    'Cytokine release syndrome (CRS) is an acute systemic inflammatory syndrome characterized by fever and multiple organ dysfunction that is associated with chimeric antigen receptor (CAR)-T cell therapy, therapeutic antibodies, and haploidentical allogeneic transplantation.',\n",
       "    'cytokine storm is an aberrant response from the host immune system that induces an exaggerated release of proinflammatory cytokines/chemokines.',\n",
       "    'Cytokine release syndrome is characterized by the release of cytokines from the cytosol into the cerebrospinal fluid (CSF) and cytokine production from the sarcoplasmic reticulum (SR).',\n",
       "    'the so-called \"cytokine storm,\" which is an aberrant response from the host immune system that induces an exaggerated release of proinflammatory cytokines/chemokines.',\n",
       "    '\"Cytokine storm,\"  is an aberrant response from the host immune system that induces an exaggerated release of proinflammatory cytokines/chemokines. I',\n",
       "    'A severe immune response in patients with coronavirus disease 2019 (COVID-19) can cause a potentially lethal unconstrained inflammatory cytokine storm, known as cytokine release syndrome (CRS).'],\n",
       "   'type': 'summary',\n",
       "   'id': '6032722f1cb411341a00013d',\n",
       "   'snippets': [{'offsetInBeginSection': 434,\n",
       "     'offsetInEndSection': 674,\n",
       "     'text': 'The major factor responsible for acute respiratory distress syndrome is the so-called \"cytokine storm,\" which is an aberrant response from the host immune system that induces an exaggerated release of proinflammatory cytokines/chemokines. I',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32983172',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 194,\n",
       "     'text': 'A severe immune response in patients with coronavirus disease 2019 (COVID-19) can cause a potentially lethal unconstrained inflammatory cytokine storm, known as cytokine release syndrome (CRS). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33262810',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 546,\n",
       "     'text': 'Immune cells secrete small protein molecules that aim for cell-cell communications. These small molecules are called cytokines. Targeting cancer cells with administration of bispecific antibodies and natural extracts results in elevated circulating levels of inflammatory cytokines, including interferon-γ and interleukin (IL)-6, which lead to cell toxicity. Sustained release of cytokines due to immunotherapy or hormonal issues causes various diseases. Novel T cell-engaging therapies and monoclonal antibodies cause cytokine release syndrome. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31755797',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 137,\n",
       "     'offsetInEndSection': 377,\n",
       "     'text': 'hile immunologic complications such as macrophage activation syndrome also known as secondary hemophagocytic lymphohistiocytosis, resulting in cytokine storm syndrome and acute\\xa0respiratory distress syndrome, may also occur in some patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32474885',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 662,\n",
       "     'offsetInEndSection': 912,\n",
       "     'text': 'Available evidence indicate that the so called \"cytokine storm\" an uncontrolled over-production of soluble markers of inflammation which, in turn, sustain an aberrant systemic inflammatory response, is a major responsible for the occurrence of ARDS. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32446778',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 218,\n",
       "     'offsetInEndSection': 465,\n",
       "     'text': 'ts: A 14-year-old boy with refractory B cell precursor acute lymphoblastic leukemia given chimeric antigen receptor cells developed severe cytokine release syndrome 7 days after the drug product infusion with progressive respiratory failure. He wa',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32166291',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 480,\n",
       "     'offsetInEndSection': 900,\n",
       "     'text': 'heless, CAR-T cell treatment is associated with toxicities such as cytokine release syndrome, which can range in severity from mild flu-like symptoms to life-threatening vasodilatory shock, and a neurological syndrome termed ICANS (immune effector cell-associated neurotoxicity syndrome), which can also range in severity from a temporary cognitive deficit lasting only a few hours to lethal cerebral edema. In this revi',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32776808',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 217,\n",
       "     'text': 'Acute cytokine release syndromes are associated with some therapeutic antibodies in man, leading to a spectrum of clinical signs from nausea, chills and fever to more serious dose limiting hypotension and tachycardia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18382853',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 561,\n",
       "     'offsetInEndSection': 680,\n",
       "     'text': 'However, some patients may experience severe, life-threatening reactions that result from massive release of cytokines.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17471824',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 281,\n",
       "     'offsetInEndSection': 472,\n",
       "     'text': 'When cytokines are released into the circulation, systemic symptoms such as fever, nausea, chills, hypotension, tachycardia, asthenia, headache, rash, scratchy throat, and dyspnea can result.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/17471824',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'Cytokine release syndrome is a systemic inflammatory condition that may occur after treatment with some types of immunotherapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32678378',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Which disease can be treated with Relugolix.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/32674208',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32674616',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30741797',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30937733',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32469183',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32911575',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32273183',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32912633',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31461087',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31594635'],\n",
       "   'ideal_answer': ['Relugolix has a role in treatment of prostate cancer, uterine fibroids, endometriosis and uterine myomas.'],\n",
       "   'exact_answer': [['prostate cancer'],\n",
       "    ['uterine fibroids'],\n",
       "    ['endometriosis'],\n",
       "    ['uterine myomas']],\n",
       "   'type': 'list',\n",
       "   'id': '602496791cb411341a00009d',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 201,\n",
       "     'text': 'Relugolix, a novel oral gonadotropin-releasing hormone antagonist, in the treatment of pain symptoms associated with uterine fibroids: a randomized, placebo-controlled, phase 3 study in Japanese women.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31594635',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1697,\n",
       "     'offsetInEndSection': 1786,\n",
       "     'text': 'CONCLUSION(S): Relugolix improved uterine fibroid-associated pain and was well tolerated.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31594635',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 76,\n",
       "     'text': 'Oral Relugolix for Androgen-Deprivation Therapy in Advanced Prostate Cancer.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32469183',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 2023,\n",
       "     'offsetInEndSection': 2271,\n",
       "     'text': 'CONCLUSIONS: In this trial involving men with advanced prostate cancer, relugolix achieved rapid, sustained suppression of testosterone levels that was superior to that with leuprolide, with a 54% lower risk of major adverse cardiovascular events. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32469183',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 270,\n",
       "     'text': 'The Oral Gonadotropin-releasing Hormone Receptor Antagonist Relugolix as Neoadjuvant/Adjuvant Androgen Deprivation Therapy to External Beam Radiotherapy in Patients with Localised Intermediate-risk Prostate Cancer: A Randomised, Open-label, Parallel-group Phase 2 Trial.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32273183',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 2227,\n",
       "     'offsetInEndSection': 2526,\n",
       "     'text': 'CONCLUSIONS: Relugolix achieved testosterone suppression to castrate levels within days and maintained it over 24 wk with a safety profile consistent with its mechanism of action.PATIENT SUMMARY: Oral once-daily relugolix may be a novel oral alternative to injectable androgen deprivation therapies.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32273183',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 83,\n",
       "     'text': 'Phase 3 HERO Trial Finds Relugolix to Be Superior to Leuprolide in Prostate Cancer.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32674208',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 442,\n",
       "     'text': 'Results from the phase 3 HERO trial(NCT03085095), presented during the 2020 American Society of Clinical Oncology Virtual Scientific Program, indicated that relugolix (Relumina) demonstrated superiority over leuprolide (Lupron) in sustained testosterone suppression through 48 weeks, fast testosterone recovery after discontinuation, and a 50% reduction in major adverse cardiovascular events (MACE) in patients with advanced prostate cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32674208',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 48,\n",
       "     'text': 'Relugolix for the treatment of uterine fibroids.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32674616',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 430,\n",
       "     'offsetInEndSection': 581,\n",
       "     'text': 'AREAS COVERED: The authors review the current options available for the treatment of women with UF, with a special focus on the newest one, relugolix. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32674616',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1006,\n",
       "     'offsetInEndSection': 1101,\n",
       "     'text': 'EXPERT OPINION: Relugolix is a promising drug for the non-surgical treatment of women with UF. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32674616',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1404,\n",
       "     'offsetInEndSection': 1639,\n",
       "     'text': 'CONCLUSION(S): Oral administration of relugolix alleviated endometriosis-associated pain in a dose-response manner and was generally well tolerated. Relugolix 40 mg demonstrated efficacy and safety comparable with those of leuprorelin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32912633',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 189,\n",
       "     'text': 'Relugolix, an oral gonadotropin-releasing hormone receptor antagonist, reduces endometriosis-associated pain in a dose-response manner: a randomized, double-blind, placebo-controlled study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32912633',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 2313,\n",
       "     'offsetInEndSection': 2650,\n",
       "     'text': 'CONCLUSION: In women with uterine leiomyomas, once-daily treatment with relugolix, an oral gonadotropin-releasing hormone antagonist, demonstrated noninferiority to monthly leuprorelin for improvement of heavy menstrual bleeding at 6-12 weeks of treatment, had a more rapid effect on menstrual bleeding, and was generally well tolerated.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30741797',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 475,\n",
       "     'text': 'The orally active nonpeptide gonadotropin-releasing hormone (GnRH)-receptor antagonist relugolix (Relumina) is being developed by Takeda and ASKA Pharmaceutical as a treatment for various sex hormone related disorders. Relugolix was recently approved for marketing in Japan as a treatment for symptoms associated with uterine fibroids, and studies evaluating the efficacy of the drug as treatment for endometriosis-associated pain and prostate cancer are currently underway. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30937733',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 48,\n",
       "     'text': 'Relugolix for the treatment of uterine fibroids.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31461087',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 673,\n",
       "     'offsetInEndSection': 822,\n",
       "     'text': 'Relugolix is an orally active nonpeptide GnRH-ant, recently licensed for marketing in Japan for the treatment of symptoms related to uterine myomas. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31461087',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 673,\n",
       "     'offsetInEndSection': 821,\n",
       "     'text': 'Relugolix is an orally active nonpeptide GnRH-ant, recently licensed for marketing in Japan for the treatment of symptoms related to uterine myomas.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31461087',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 219,\n",
       "     'offsetInEndSection': 474,\n",
       "     'text': 'Relugolix was recently approved for marketing in Japan as a treatment for symptoms associated with uterine fibroids, and studies evaluating the efficacy of the drug as treatment for endometriosis-associated pain and prostate cancer are currently underway.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30937733',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 475,\n",
       "     'offsetInEndSection': 640,\n",
       "     'text': 'This article summarizes the milestones in the development of relugolix leading to this first approval for the treatment of symptoms associated with uterine fibroids.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30937733',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 181,\n",
       "     'text': 'We here describe a case of the prolapse of pedunculated submucosal leiomyoma through the cervix during the treatment of a gonadotropin-releasing hormone (GnRH) antagonist relugolix.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32911575',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Unlike DNA, RNA is not methylated, yes or no?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/18567810',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34694621',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27375676',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/4992370',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16424344',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19808971',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16111635',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33526437',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34694619',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28349450'],\n",
       "   'ideal_answer': ['In addition to DNA methylation, reversible epigenetic modification occurring in RNA has been discovered recently. The most abundant type of RNA methylation is N6-methyladenosine (m6A) modification, which is dynamically regulated by methylases (\"writers\"), demethylases (\"erasers\") and m6A-binding proteins (\"readers\")',\n",
       "    'N6-methyladenosine (m6A) is the most abundant internal modification on messenger RNAs (mRNAs) and long noncoding RNAs (lncRNAs) in eukaryotes'],\n",
       "   'exact_answer': 'no',\n",
       "   'type': 'yesno',\n",
       "   'id': '621fd6083a8413c653000066',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 187,\n",
       "     'text': 'Mapping the position and quantifying the level of 5-methylcytosine (m5C) as a modification in different types of cellular RNA is an important objective in the field of epitranscriptomics.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34694621',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 141,\n",
       "     'text': 'N6-methyladenosine (m6A) is the most abundant internal modification on messenger RNAs (mRNAs) and long noncoding RNAs (lncRNAs) in eukaryotes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34694619',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 129,\n",
       "     'text': 'Gluten-induced RNA methylation changes regulate intestinal inflammation via allele-specific XPO1 translation in epithelial cells.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33526437',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 419,\n",
       "     'offsetInEndSection': 603,\n",
       "     'text': 'nogaster was methylated to a lower degree, but in the case of Dictyostelium, there was no difference in the methylation of RNA isolated from wild-type and Dnmt2 knock-out strains. Meth',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18567810',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 220,\n",
       "     'text': 'The detection and quantification of methylated RNA can be beneficial to understand certain cellular regulation processes such as transcriptional modulation of gene expression, immune response, or epigenetic alterations. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28349450',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 137,\n",
       "     'offsetInEndSection': 420,\n",
       "     'text': 'ombined genetic and biochemical approach revealed that human DNMT2 did not methylate DNA but instead methylated a small RNA; mass spectrometry showed that this RNA is aspartic acid transfer RNA (tRNA(Asp)) and that DNMT2 specifically methylated cytosine 38 in the anticodon loop. The',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16424344',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 179,\n",
       "     'offsetInEndSection': 340,\n",
       "     'text': 'ted nucleosides in naturally occurring RNA are also methylated or otherwise modified, but the immunomodulatory effects of these alterations remain untested. We s',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16111635',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 923,\n",
       "     'offsetInEndSection': 1129,\n",
       "     'text': '-induced loss of RNA methylation seemed specific for DNMT2 target sites because we did not observe any significant demethylation at sites known to be methylated by other RNA methyltransferases. Our results ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19808971',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 816,\n",
       "     'offsetInEndSection': 920,\n",
       "     'text': 'Crude extracts of this organism possess RNA methylase activity but no detectable DNA methylase activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/4992370',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 134,\n",
       "     'offsetInEndSection': 416,\n",
       "     'text': 'A combined genetic and biochemical approach revealed that human DNMT2 did not methylate DNA but instead methylated a small RNA; mass spectrometry showed that this RNA is aspartic acid transfer RNA (tRNA(Asp)) and that DNMT2 specifically methylated cytosine 38 in the anticodon loop.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16424344',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 291,\n",
       "     'offsetInEndSection': 405,\n",
       "     'text': 'Here, we propose that RNA methylation began prior to DNA methylation in the early forms of life evolving on Earth.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27375676',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is there a genetic cause of craniostenosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29579021',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3221205',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/15791470',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29037998',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/7400783',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/964994',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11977182',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26463893',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28523332',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/19893477'],\n",
       "   'ideal_answer': ['9There a a number of different genetic mutations or syndromes(Saethre-Chotzen, Aperts, Crouzon, Pfeiffer) associated with craniostenosis.',\n",
       "    'Saethre-Chotzen syndrome is an autosomal dominant disease characterized by craniosynostosis, ptosis, and limb and external ear abnormalities'],\n",
       "   'exact_answer': 'yes',\n",
       "   'type': 'yesno',\n",
       "   'id': '601d73261cb411341a00003a',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 324,\n",
       "     'text': 'Apert syndrome - acrocephalosyndactyly - is a rare autosomal dominant disorder representing 1:65 000 cases of living newborns. Characteristic malformations of the Apert syndrome are early craniostenosis, microviscerocranium and II-V finger syndactyly of hand and toes with proximal phalanx of the bilateral thumb \"in delta\".',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28523332',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 254,\n",
       "     'text': 'A 3-year-old child with tertiary trisomy (14 (+14q--), daughter of a mother with a balanced reciprocal translocation [46,XX,t(14;16)(q11;q24) is presented. Craniostenosis and developmental retardation were the primary presenting features in this patient.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/964994',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 140,\n",
       "     'text': 'Saethre-Chotzen syndrome is an autosomal dominant disease characterized by craniosynostosis, ptosis, and limb and external ear abnormalities',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11977182',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 117,\n",
       "     'offsetInEndSection': 452,\n",
       "     'text': '. For 98 patients (15%) a syndrome is associated. Third part of them has Apert syndrome, an other third part has Crouzon syndrome, and for the last third more exceptional acrocephalosyndactyly syndrome (Saethre-Chotzen, Pfeiffer) or others atypical associations, sometimes not yet described, but with an autosomal dominant inheritance.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3221205',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 597,\n",
       "     'offsetInEndSection': 786,\n",
       "     'text': 'Coronal craniostenosis seems to be a dominant autosomal character, when scaphocephaly is more often sporadic; for both, an autosomal dominant inheritance is not excluded for some pedigrees.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3221205',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 189,\n",
       "     'offsetInEndSection': 537,\n",
       "     'text': ' genetic origins are not completely clear although mutations in the genes that code for fibroblast growth factor receptors have been described; depending upon the gene involved, the type of mutation and the embryological period in which the mutation itself occurs, a type of craniosynostosis arises that may involve one or more cranial sutures. The',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19893477',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 142,\n",
       "     'text': 'Saethre-Chotzen syndrome is an autosomal dominant disease characterized by craniosynostosis, ptosis, and limb and external ear abnormalities. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11977182',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 109,\n",
       "     'text': 'Identification and analysis of the genetic causes in nine unrelated probands with syndromic craniosynostosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29037998',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 144,\n",
       "     'offsetInEndSection': 438,\n",
       "     'text': 'To identify and analyze causative genetic variants in nine unrelated probands mainly manifested as syndromic craniosynostosis, we reviewed the relevant medical information of the patients and performed the whole exome sequencing, further verified with Sanger sequencing and parental background.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29037998',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 135,\n",
       "     'text': '[Genetic counseling in craniostenosis. Results of a prospective study performed with a group of studies on craniofacial malformations].',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3221205',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 195,\n",
       "     'text': 'Constitutional 11q interstitial deletion syndrome presents with congenital anomalies including microcephaly with craniostenosis, minor dysmorphic features, vitreoretinopathy, and renal anomalies.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26463893',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 204,\n",
       "     'text': 'A suckling baby with microcephaly, craniostenosis, downward slanting palpebral fissues, malformed ears, cerebral, cardiac and intestinal malformation, and partial 6q25 leads to 6qter trisomy is presented.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/7400783',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 186,\n",
       "     'offsetInEndSection': 533,\n",
       "     'text': 'Its genetic origins are not completely clear although mutations in the genes that code for fibroblast growth factor receptors have been described; depending upon the gene involved, the type of mutation and the embryological period in which the mutation itself occurs, a type of craniosynostosis arises that may involve one or more cranial sutures.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/19893477',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1085,\n",
       "     'offsetInEndSection': 1421,\n",
       "     'text': 'through which skull growth abnormalities are seen. It is becoming clearer that in most patients with craniosynostosis, there is regional imbalance of skull growth, which co-exists with a variety of other equally important factors, such as genetic defects, raised intracranial pressure, venous hypertension, and other brain parenchymal a',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/15791470',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1389,\n",
       "     'offsetInEndSection': 1692,\n",
       "     'text': 'Recent genetic studies have identified several novel genes and pathways that cause nonsyndromic craniosynostosis, providing genetic evidence linking the causes of syndromic and nonsyndromic craniosynostoses, and allowing for genotype-based prediction of risk of recurrence in some nonsyndromic families.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29579021',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What are the diagnostic criteria for hemophagocytic lymphohistiocytosis?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/32300936',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/16937360',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34850652',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24583560',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26512263',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30086677',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33658450',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32588191',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1561489',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21773661'],\n",
       "   'ideal_answer': ['Hemophagocytic syndrome (HS) is a severe hyper inflammatory condition whose cardinal symptoms are prolonged fever, cytopenia, hepatosplenomegaly, and hemophagocytosis by activated, morphologically benign macrophages.',\n",
       "    'Hemophagocytic lymphohistiocytosis (HLH), diagnosis is based on five criteria (fever, splenomegaly, bicytopenia, hypertriglyceridemia and/or hypofibrinogenemia, and hemophagocytosis).'],\n",
       "   'exact_answer': [['fever'],\n",
       "    ['splenomegaly'],\n",
       "    ['bicytopenia'],\n",
       "    ['hypertriglyceridemia'],\n",
       "    ['hypofibrinogenemia'],\n",
       "    ['hemophagocytosis'],\n",
       "    ['hyperferritinemia'],\n",
       "    ['excessive cytokine production'],\n",
       "    ['hepatosplenomegaly'],\n",
       "    ['Hemophagocytic lymphohistiocytosis is a rare and aggressive syndrome of excessive cytokine requiring prompt recognition and aggressive therapy.']],\n",
       "   'type': 'list',\n",
       "   'id': '625ebd06e764a53204000031',\n",
       "   'snippets': [{'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 161,\n",
       "     'text': 'Hemophagocytic lymphohistiocytosis (HLH) is a rare and aggressive syndrome of excessive cytokine requiring prompt recognition and aggressive therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32300936',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 408,\n",
       "     'text': 'Hemophagocytic lymphohistiocytosis (HLH) is a rare and life-threatening condition that is characterized by an overactive response of the immune system with excessive production of proinflammatory cytokines. Initial presentation of this condition often mimics and overlaps with many diseases including infections, sepsis, and multiorgan failure syndrome, which makes diagnosis the diagnosis of HLH challenging',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34850652',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 222,\n",
       "     'text': 'Hemophagocytic syndrome (HPS) or hemophagocytic lymphohistiocytosis (HLH) is an acute and rapidly progressive systemic inflammatory disorder characterized by cytopenia, excessive cytokine production, and hyperferritinemia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32588191',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 217,\n",
       "     'text': 'Hemophagocytic syndrome (HS) is a severe hyper inflammatory condition whose cardinal symptoms are prolonged fever, cytopenia, hepatosplenomegaly, and hemophagocytosis by activated, morphologically benign macrophages. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21773661',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 59,\n",
       "     'offsetInEndSection': 310,\n",
       "     'text': 'Acquired hemophagocytic lymphohistiocytosis comprises a heterogenous group of hyperinflammatory immunoreactions often resulting in uncontrolled immune responses, mainly throughout proliferation of cytotoxic T cells and hemophagocytosis by macrophages.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30086677',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 910,\n",
       "     'offsetInEndSection': 1136,\n",
       "     'text': ' and symptoms fulfilled the five criteria for a diagnosis of hemophagocytic lymphohistiocytosis, including fever, cytopenia, hemophagocytosis, hyperferritinemia, and high soluble interleukin-2 receptor levels. Therefore, the d',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33658450',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 500,\n",
       "     'offsetInEndSection': 902,\n",
       "     'text': 'ostic criteria of hemophagocytic lymphohistiocytosis are the following: fever, splenomegaly, cytopenias affecting at least two of the 3 lineages in peripheral blood, hypertriglyceridemia and hyperferritinemia, elevated serum level of soluble interleukin-2 receptor (sCD25), hypofibrinogenemia, hemophagocytosis in bone marrow and decreased cytotoxic T cell and natural killer cell activity. In this cas',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24583560',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 257,\n",
       "     'text': 'In HLH-94, the first prospective international treatment study for hemophagocytic lymphohistiocytosis (HLH), diagnosis was based on five criteria (fever, splenomegaly, bicytopenia, hypertriglyceridemia and/or hypofibrinogenemia, and hemophagocytosis). In HL',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16937360',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 301,\n",
       "     'offsetInEndSection': 451,\n",
       "     'text': 'Diagnostic criteria include idiopathic fever, splenomegaly, cytopenias, hypertriglyceridemia, hypofibrinogenemia, and the presence of hemophagocytosis',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1561489',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 201,\n",
       "     'text': 'Hemophagocytic lymphohistiocytosis (HLH) is characterized by fever, splenomegaly, jaundice, and pathologic findings of hemophagocytosis in bone marrow or other tissues such as the lymph nodes and liver',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26512263',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 495,\n",
       "     'offsetInEndSection': 890,\n",
       "     'text': 'Diagnostic criteria of hemophagocytic lymphohistiocytosis are the following: fever, splenomegaly, cytopenias affecting at least two of the 3 lineages in peripheral blood, hypertriglyceridemia and hyperferritinemia, elevated serum level of soluble interleukin-2 receptor (sCD25), hypofibrinogenemia, hemophagocytosis in bone marrow and decreased cytotoxic T cell and natural killer cell activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24583560',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 301,\n",
       "     'offsetInEndSection': 452,\n",
       "     'text': 'Diagnostic criteria include idiopathic fever, splenomegaly, cytopenias, hypertriglyceridemia, hypofibrinogenemia, and the presence of hemophagocytosis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1561489',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 251,\n",
       "     'text': 'In HLH-94, the first prospective international treatment study for hemophagocytic lymphohistiocytosis (HLH), diagnosis was based on five criteria (fever, splenomegaly, bicytopenia, hypertriglyceridemia and/or hypofibrinogenemia, and hemophagocytosis).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/16937360',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What links immune response pathways to ALS?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/27812125',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29568058',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25889790',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32594542',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/24170856',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34402459',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20406178',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27308305',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33918092',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25344935'],\n",
       "   'ideal_answer': ['Microglia, which are the primary immune cells of the central nervous system, are strongly implicated in ALS, their activation being correlated with various clinical features, and inflammatory microglial responses being correlated withe disease progression. The immune response may be implicated in other ways with ALS molecular pathology. such as through inflammatory regulation and circulating interleukins. It is possible the T cell receptor signalling and activation is involved. It is also possible that the innate / non-specific immune system is involved - i.e. immune protection against foreign substances, viruses, and bacteria.'],\n",
       "   'type': 'summary',\n",
       "   'id': '6273a2a0e764a53204000045',\n",
       "   'snippets': [{'offsetInBeginSection': 1469,\n",
       "     'offsetInEndSection': 1598,\n",
       "     'text': 'Neuromuscular pathology in ALS involves the systemic regulation of inflammatory markers mostly active on T-cell immune responses.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27308305',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 150,\n",
       "     'offsetInEndSection': 359,\n",
       "     'text': 'At the sites of motor neuron injury, accumulation of activated microglia, the primary immune cells of the central nervous system, is commonly observed in both human post mortem studies and animal models of MND',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32594542',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 361,\n",
       "     'offsetInEndSection': 499,\n",
       "     'text': 'Microglial activation has been found to correlate with many clinical features and importantly, the speed of disease progression in humans.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32594542',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 500,\n",
       "     'offsetInEndSection': 641,\n",
       "     'text': 'Both anti-inflammatory and pro-inflammatory microglial responses have been shown to influence disease progression in humans and models of MND',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32594542',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 652,\n",
       "     'offsetInEndSection': 753,\n",
       "     'text': 'microglia could both contribute to and protect against inflammatory mechanisms of pathogenesis in MND',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32594542',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 469,\n",
       "     'offsetInEndSection': 664,\n",
       "     'text': 'ral pathways were found to be upregulated in patients with ALS, among which the toll-like receptor (TLR) and NOD-like receptor (NLR) signaling pathways are related to the immune response. Genes-t',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27812125',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 122,\n",
       "     'offsetInEndSection': 319,\n",
       "     'text': 'he adaptive immune response has been implicated in disease processes of amyotrophic lateral sclerosis (ALS), but it remains unknown if innate immune signaling also contributes to ALS progression. H',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29568058',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 161,\n",
       "     'text': 'BACKGROUND: The peripheral immune system is implicated in modulating microglial activation, neurodegeneration and disease progression in amyotrophic lateral scle',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25889790',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 542,\n",
       "     'offsetInEndSection': 750,\n",
       "     'text': 'Besides its connection to inflammation, NF-κB activity can be linked to several genes associated to familial forms of ALS, and many of the environmental risk factors of the disease stimulate NF-κB activation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33918092',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1465,\n",
       "     'offsetInEndSection': 1700,\n",
       "     'text': 'Our data suggest that the TLR and NLR signaling pathways are involved in pathological innate immunity and neuroinflammation associated with ALS and that TOLLIP, MAPK9, IL-1β, IL-8, and CXCL1 play a role in ALS-specific immune responses',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27812125',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 465,\n",
       "     'offsetInEndSection': 656,\n",
       "     'text': 'Several pathways were found to be upregulated in patients with ALS, among which the toll-like receptor (TLR) and NOD-like receptor (NLR) signaling pathways are related to the immune response.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27812125',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1465,\n",
       "     'offsetInEndSection': 1701,\n",
       "     'text': 'Our data suggest that the TLR and NLR signaling pathways are involved in pathological innate immunity and neuroinflammation associated with ALS and that TOLLIP, MAPK9, IL-1β, IL-8, and CXCL1 play a role in ALS-specific immune responses.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27812125',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 420,\n",
       "     'offsetInEndSection': 746,\n",
       "     'text': 'Although innate and adaptive immune responses are associated with progressive neurodegeneration, in the early stages of ALS immune activation pathways are primarily considered to be beneficial promoting neuronal repair of the damaged tissues, though a harmful effect of T cells at this stage of disease has also been observed.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25344935',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 171,\n",
       "     'offsetInEndSection': 477,\n",
       "     'text': 'This includes microglial activation, lymphocyte infiltration, and the induction of C1q, the initiating component of the classic complement system that is the protein-based arm of the innate immune response, in motor neurons of multiple ALS mouse models expressing dismutase active or inactive SOD1 mutants.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/24170856',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is lumbar puncture the first test that should be performed on a patient with increased intracranial pressure?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/25685562',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21129599',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10608260',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/2277190',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11985377',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21541088',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/5763958',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3550721',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28965126',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/3520526'],\n",
       "   'ideal_answer': ['No. A lumbar puncture is contraindicated in any patient with signs of increased intracranial pressure because it may precipitate cerebral herniation and death. For this reason, a computed tomography (CT) or magnetic resonance imaging (MRI) scan is done first. When the findings of the scan are normal, a lumbar puncture can be performed, if needed.',\n",
       "    'Lumbar puncture (LP) is usually contra-indicated in situations where the ICP is suspected to be high.'],\n",
       "   'exact_answer': 'no',\n",
       "   'type': 'yesno',\n",
       "   'id': '627a6ef856bf9aee6f000020',\n",
       "   'snippets': [{'offsetInBeginSection': 158,\n",
       "     'offsetInEndSection': 462,\n",
       "     'text': 'As an emergency diagnostic procedure, spinal puncture is indicated when CNS infection is suspected or to establish the diagnosis of subarachnoid hemorrhage when results of cranial computed tomography are normal. The major contraindication is elevated intracranial pressure with evidence of a mass lesion.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3520526',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 583,\n",
       "     'offsetInEndSection': 849,\n",
       "     'text': 'Brain shift is a contraindication to LP, whether CSF pressure is raised or not, and whether papilloedema is present or not. Subsequently, recommendations are offered for indications to perform CT before LP, grouped according to the safety and clinical utility of LP.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11985377',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 210,\n",
       "     'text': 'Death following lumbar puncture (LP) is feared by physicians. Many opinions are found in literature on the question whether computed cranial tomography (CT) should be performed before LP, to prevent herniation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11985377',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 210,\n",
       "     'offsetInEndSection': 625,\n",
       "     'text': 'Headache, caused by cerebrospinal fluid (CSF) hypotension, is a frequent complication of lumbar puncture; hematic patch is a therapeutic option for severe cases. The most serious complication is cerebral herniation and, for its prevention, computed tomography (CT) or cerebral magnetic resonance imaging (MRI) must always be performed before lumbar puncture: a lesion with evident mass effect is a contraindication.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21129599',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 238,\n",
       "     'offsetInEndSection': 339,\n",
       "     'text': 'Lumbar puncture (LP) is usually contra-indicated in situations where the ICP is suspected to be high.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10608260',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 436,\n",
       "     'text': 'Low-dose cranial computed tomography (LD-CCT) based on iterative reconstruction has been shown to have sufficient image quality to assess cerebrospinal fluid spaces (CSF) and midline structures but not to exclude subtle parenchymal pathologies. Patients without focal neurological deficits often undergo CCT before lumbar puncture (LP) to exclude contraindications to LP including brain herniation or increased CSF pressure.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28965126',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 225,\n",
       "     'text': 'Lumbar puncture is performed routinely for diagnostic and therapeutic purposes in idiopathic intracranial hypertension, despite lumbar puncture being classically contraindicated in the setting of raised intracranial pressure.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25685562',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 209,\n",
       "     'offsetInEndSection': 410,\n",
       "     'text': 'Although generally considered innocuous, there may be considerable danger when lumbar puncture is performed in the presence of increased intracranial pressure, especially when a mass lesion is present.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/3550721',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1360,\n",
       "     'offsetInEndSection': 1619,\n",
       "     'text': 'Lumbar puncture should be avoided if focal neurologic findings suggest concomitant mass lesion, as in brain abscess, and lumbar puncture should be approached with great caution if meningitis is accompanied by evidence of significant intracranial hypertension.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/2277190',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1604,\n",
       "     'offsetInEndSection': 1789,\n",
       "     'text': 'There are few abnormal CT scans presenting a contraindication for lumbar puncture and the majority of these patients usually have clinical signs to suggest raised intracranial pressure.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21541088',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Summarize the function of DEAH helicase DHX36 and its role in G-quadruplex-dependent processes.',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/29899445',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33857359',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33857358',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34862880',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33021960',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34025941',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29269411',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30910870',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25653156',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27797375'],\n",
       "   'ideal_answer': ['DEAH-Box helicase 36 (DHX36), a member of the large DExD/H box helicase family, enzymatically unwinds both G4 DNA and G4 RNA. RNA helicases of the DEAH/RHA family form a large and conserved class of enzymes that remodel RNA protein complexes (RNPs) by translocating along the RNA'],\n",
       "   'type': 'summary',\n",
       "   'id': '6220cf383a8413c653000069',\n",
       "   'snippets': [{'offsetInBeginSection': 286,\n",
       "     'offsetInEndSection': 410,\n",
       "     'text': 'DEAH-Box helicase 36 (DHX36), a member of the large DExD/H box helicase family, enzymatically unwinds both G4 DNA and G4 RNA',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34862880',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 154,\n",
       "     'text': 'RNA helicases of the DEAH/RHA family form a large and conserved class of enzymes that remodel RNA protein complexes (RNPs) by translocating along the RNA.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33857358',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 136,\n",
       "     'text': 'DHX36 is a member of the DExD/H box helicase family, which comprises a large number of proteins involved in various cellular functions. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33021960',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 227,\n",
       "     'offsetInEndSection': 365,\n",
       "     'text': ' G4s are alternative nucleic acid structures, which influence many cellular pathways on a transcriptional and post-transcriptional level. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33021960',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 531,\n",
       "     'offsetInEndSection': 701,\n",
       "     'text': 'We also show that GSEC binds to the DEAH box polypeptide 36 (DHX36) RNA helicase via its G-quadruplex-forming sequence and inhibits DHX36 G-quadruplex unwinding activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27797375',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 168,\n",
       "     'offsetInEndSection': 293,\n",
       "     'text': 'DHX36, also known as RHAU or G4R1, is a DEAH-box ATP-dependent helicase highly specific for DNA and RNA G-quadruplexes (G4s).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25653156',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 978,\n",
       "     'offsetInEndSection': 1244,\n",
       "     'text': \"Together, our results animate recent DHX36 crystal structures, suggesting a model in which the DSM recruits G4s in a modular and flexible manner by contacting the 5' face early in binding, prior to rate-limiting capture and disruption of the G4 by the helicase core.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33857359',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 140,\n",
       "     'offsetInEndSection': 291,\n",
       "     'text': 'he DEAH helicase DHX37 (Dhr1 in yeast) is activated by the ribosome biogenesis factor UTP14A to facilitate maturation of the small ribosomal subunit. W',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30910870',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 154,\n",
       "     'text': 'DHX36 is a eukaryotic DEAH/RHA family helicase that disrupts G-quadruplex structures (G4s) with high specificity, contributing to regulatory roles of G4s.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33857359',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 718,\n",
       "     'offsetInEndSection': 944,\n",
       "     'text': 'DHX36\\xa0is a multi-functional helicase that has been implicated in G-quadruplex-mediated transcriptional and post-transcriptional regulation, and is essential for heart development, haematopoiesis, and embryogenesis in mice9-12.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29899445',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 423,\n",
       "     'offsetInEndSection': 717,\n",
       "     'text': 'DHX36 (also known as RHAU and G4R1), a member of the DEAH/RHA family of helicases, binds both DNA and RNA G-quadruplexes with extremely high affinity4-6, is consistently found bound to G-quadruplexes in cells7,8, and is a major source of G-quadruplex unfolding activity in HeLa cell lysates 6 .',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29899445',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 286,\n",
       "     'offsetInEndSection': 411,\n",
       "     'text': 'DEAH-Box helicase 36 (DHX36), a member of the large DExD/H box helicase family, enzymatically unwinds both G4 DNA and G4 RNA.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34862880',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 200,\n",
       "     'offsetInEndSection': 339,\n",
       "     'text': 'One example is DHX36, a DEAH-box helicase, which participates in gene expression and replication by recognizing and unwinding parallel G4s.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34025941',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 129,\n",
       "     'text': 'The G-quadruplex (G4) resolvase DHX36 efficiently and specifically disrupts DNA G4s via a translocation-based helicase mechanism.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29269411',\n",
       "     'endSection': 'title'}]},\n",
       "  {'body': 'Which disease is treated with Emapalumab?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/33484058',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31537529',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33686916',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32374962',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32648854',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34096649',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32447592',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33424859',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30623346',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32817285'],\n",
       "   'ideal_answer': ['Emapalumab is a human monoclonal antibody directed against interferon-γ (IFN-γ) that was approved by the Food and Drug Administration for primary hemophagocytic lymphohistiocytosis (HLH).'],\n",
       "   'exact_answer': ['primary hemophagocytic lymphohistiocytosis'],\n",
       "   'type': 'factoid',\n",
       "   'id': '60234dc61cb411341a000094',\n",
       "   'snippets': [{'offsetInBeginSection': 1117,\n",
       "     'offsetInEndSection': 1283,\n",
       "     'text': 'All patients who received the complement blocker eculizumab in addition to the interferon gamma blocker emapalumab had complete resolution of their TMA and survived. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32447592',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 71,\n",
       "     'text': 'Emapalumab in Children with Primary Hemophagocytic Lymphohistiocytosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32374962',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 2045,\n",
       "     'offsetInEndSection': 2167,\n",
       "     'text': 'CONCLUSIONS: Emapalumab was an efficacious targeted therapy for patients with primary hemophagocytic lymphohistiocytosis. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32374962',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 67,\n",
       "     'text': 'Emapalumab for the treatment of hemophagocytic lymphohistiocytosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648854',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 216,\n",
       "     'text': 'Emapalumab-Igsz (Gamifant) is a human monoclonal antibody directed against interferon-γ (IFN-γ), and the first Food and Drug Administration (FDA)-approved therapy for primary hemophagocytic lymphohistiocytosis (HLH).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648854',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 424,\n",
       "     'offsetInEndSection': 809,\n",
       "     'text': 'Emapalumab is approved for treatment of primary HLH that is refractory, recurrent, progressing or intolerant to current HLH treatments in both adult and pediatric patients. FDA approval was based on the results of a phase II/III clinical trial evaluating the safety and efficacy of emapalumab in 34 pediatric patients with primary HLH, 27 of whom were refractory to current therapies. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648854',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 306,\n",
       "     'text': 'Emapalumab, a fully human anti-IFNγ monoclonal antibody, has been approved in the US as second-line treatment of primary hemophagocytic lymphohistiocytosis (HLH) patients and has shown promise in patients with graft failure (GF) requiring a second allogeneic hematopoietic stem cell transplantation (HSCT).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32817285',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 348,\n",
       "     'text': 'Emapalumab is a fully human immunoglobulin G1 monoclonal antibody directed against interferon-γ (IFN-γ), which in November 2018 received the first global approval for the treatment of pediatric and adult patients with primary hemophagocytic lymphohistiocytosis (HLH) with refractory, recurrent, or progressive disease or intolerance to HLH therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31537529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1263,\n",
       "     'offsetInEndSection': 1498,\n",
       "     'text': 'Available data support the use of emapalumab for treatment of pHLH patients with refractory, recurrent or progressive disease, or intolerance to conventional therapy, recently, leading to FDA approval of the drug for these indications.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33424859',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 297,\n",
       "     'offsetInEndSection': 497,\n",
       "     'text': 'Emapalumab, a monoclonal antibody directed against IFN-γ, is the first target therapy approved for primary HLH with refractory, recurrent or progressive disease or intolerance to conventional therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33686916',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 761,\n",
       "     'offsetInEndSection': 849,\n",
       "     'text': 'RT OPINION: Emapalumab is an effective treatment for HLH with a good safety profile. Its',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33686916',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 87,\n",
       "     'text': 'Emapalumab for the treatment of relapsed/refractory hemophagocytic lymphohistiocytosis.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31537529',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 287,\n",
       "     'offsetInEndSection': 581,\n",
       "     'text': 'Recently, the therapeutic potential of IFN-γ inhibition has been documented; emapalumab, an anti-IFN-γ monoclonal antibody, has been approved in the United States for treatment of primary HLH that is refractory, recurrent or progressive, or in patients with intolerance to conventional therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33484058',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 311,\n",
       "     'text': 'Emapalumab, a fully human anti-IFNγ monoclonal antibody, has been approved in the US as second-line treatment of primary hemophagocytic lymphohistiocytosis (HLH) patients and has shown promise in patients with graft failure (GF) requiring a second allogeneic hematopoietic stem cell transplantation (HSCT). The ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32817285',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 291,\n",
       "     'offsetInEndSection': 493,\n",
       "     'text': 'role. Emapalumab, a monoclonal antibody directed against IFN-γ, is the first target therapy approved for primary HLH with refractory, recurrent or progressive disease or intolerance to conventional ther',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33686916',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 353,\n",
       "     'text': 'Emapalumab is a fully human immunoglobulin G1 monoclonal antibody directed against interferon-γ (IFN-γ), which in November 2018 received the first global approval for the treatment of pediatric and adult patients with primary hemophagocytic lymphohistiocytosis (HLH) with refractory, recurrent, or progressive disease or intolerance to HLH therapy. This',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31537529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 220,\n",
       "     'text': 'Emapalumab-Igsz (Gamifant) is a human monoclonal antibody directed against interferon-γ (IFN-γ), and the first Food and Drug Administration (FDA)-approved therapy for primary hemophagocytic lymphohistiocytosis (HLH). HLH',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648854',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 295,\n",
       "     'offsetInEndSection': 593,\n",
       "     'text': ', the therapeutic potential of IFN-γ inhibition has been documented; emapalumab, an anti-IFN-γ monoclonal antibody, has been approved in the United States for treatment of primary HLH that is refractory, recurrent or progressive, or in patients with intolerance to conventional therapy. Moreover, r',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33484058',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 224,\n",
       "     'offsetInEndSection': 430,\n",
       "     'text': 'a disorder characterized by hypercytokinemia in the setting of unbridled immune activation, and emapalumab represents the first therapeutic developed to address the underlying pathophysiology of HLH. Emapal',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648854',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 86,\n",
       "     'text': 'Emapalumab for the treatment of relapsed/refractory hemophagocytic lymphohistiocytosis',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31537529',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 83,\n",
       "     'text': 'Emapalumab for adult and pediatric patients with hemophagocytic lymphohistiocytosis',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33686916',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 313,\n",
       "     'offsetInEndSection': 580,\n",
       "     'text': 'ember 2018, emapalumab received its first global approval in the USA, for the treatment of paediatric (newborn and older) and adult patients with primary HLH, who have refractory, recurrent or progressive disease or intolerance to conventional HLH therapy. Emapalumab',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30623346',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1290,\n",
       "     'offsetInEndSection': 1528,\n",
       "     'text': 'use of emapalumab for treatment of pHLH patients with refractory, recurrent or progressive disease, or intolerance to conventional therapy, recently, leading to FDA approval of the drug for these indications. Additional data are needed to',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33424859',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 424,\n",
       "     'offsetInEndSection': 596,\n",
       "     'text': 'Emapalumab is approved for treatment of primary HLH that is refractory, recurrent, progressing or intolerant to current HLH treatments in both adult and pediatric patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648854',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1342,\n",
       "     'offsetInEndSection': 1500,\n",
       "     'text': 'The novel agent, emapalumab (an anti-IFN-γ monoclonal antibody) can be used to treat EBV-HLH cases to avoid the risk of secondary malignancy due to etoposide.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34096649',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 283,\n",
       "     'offsetInEndSection': 581,\n",
       "     'text': 'hogenic role. Emapalumab, a monoclonal antibody directed against IFN-γ, is the first target therapy approved for primary HLH with refractory, recurrent or progressive disease or intolerance to conventional therapy.AREAS COVERED: We reviewed the pharmacological characteristics, safety, efficacy and',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33686916',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 171,\n",
       "     'offsetInEndSection': 306,\n",
       "     'text': 'Emapalumab is being developed by Novimmune and Swedish Orphan Biovitrum for the treatment of haemophagocytic lymphohistiocytosis (HLH).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30623346',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 307,\n",
       "     'offsetInEndSection': 569,\n",
       "     'text': 'In November 2018, emapalumab received its first global approval in the USA, for the treatment of paediatric (newborn and older) and adult patients with primary HLH, who have refractory, recurrent or progressive disease or intolerance to conventional HLH therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30623346',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What are the main clinical components of the brain death exam?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/33273410',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32312535',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32667440',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21604079',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/18514082',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31456011',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32648194',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/22522447',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32524528',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/20724258'],\n",
       "   'ideal_answer': ['The three essential findings in brain death are coma, absence of brain stem reflexes by neurologic exam including ancillary testing, and apnea. Studies can highlight the variability in practice in regard to the AT and supports the use of ancillary tests to determine BD in patients. Coma and brain stem reflexes can be determined by several ancillary tests  Ancillary tests include electroencephalography,  brainstem auditory evoked potentials, included somatosensory evoked potentials, transcranial Doppler ultrasonography, conventional cerebral angiography, and  nuclear medicine flow study'],\n",
       "   'exact_answer': [['coma'],\n",
       "    ['neurologic exam', 'neurologic criteria'],\n",
       "    ['Apnea test'],\n",
       "    ['Ancillary test'],\n",
       "    ['somatosensory evoked potentials'],\n",
       "    ['electroencephalography'],\n",
       "    ['transcranial Doppler ultrasonography'],\n",
       "    ['nuclear medicine flow study'],\n",
       "    ['conventional cerebral angiography'],\n",
       "    ['brainstem auditory evoked potentials']],\n",
       "   'type': 'list',\n",
       "   'id': '6235ee453a8413c6530000ac',\n",
       "   'snippets': [{'offsetInBeginSection': 979,\n",
       "     'offsetInEndSection': 1348,\n",
       "     'text': 'A total of 157 ancillary tests were performed, including electroencephalogram (62%), computed tomography angiography (22%), transcranial Doppler ultrasound (6%), cerebral blood flow nuclear study (5%), cerebral angiography (4%), and other (1%). Forty-seven patients (53% of patients with AT) with confirmatory AT still underwent additional ancillary for BD confirmation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32312535',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1349,\n",
       "     'offsetInEndSection': 1469,\n",
       "     'text': ' Only 21 patients (12% of all patients) were declared brain-dead using confirmatory ATs alone without ancillary testing.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32312535',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1588,\n",
       "     'offsetInEndSection': 1724,\n",
       "     'text': 'Our study highlights the variability in practice in regard to the AT and supports the use of ancillary tests to determine BD in patients',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32312535',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 407,\n",
       "     'offsetInEndSection': 1047,\n",
       "     'text': 'Details provided on the performance and interpretation of ancillary tests for determination of BD/DNC were variably provided and inconsistent. Approximately half of all protocols that included each ancillary test provided details about study performance: 63% of protocols that included conventional cerebral angiography, 55% of protocols that included electroencephalography, 50% of protocols that included somatosensory evoked potentials, 48% of protocols that included transcranial Doppler ultrasonography, 43% of protocols that included nuclear medicine flow study and 41% of protocols that included brainstem auditory evoked potentials.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648194',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1566,\n",
       "     'offsetInEndSection': 1660,\n",
       "     'text': ' Diagnostic requirements for ancillary testing in BD/DNC determination vary around the world. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32648194',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'Apnea is one of the three cardinal findings in brain death (BD). Apnea testing (AT) is physiologically and practically complex.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32524528',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 51,\n",
       "     'text': 'Apnea Testing for the Determination of Brain Death:',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32524528',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1584,\n",
       "     'offsetInEndSection': 1750,\n",
       "     'text': ' When conducting the neurologic examination and apnea test required for death by neurologic criteria determination in patients with suspected or confirmed coronavirus',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33273410',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 196,\n",
       "     'text': 'To discuss the challenges of conducting a death by neurologic criteria or brain death evaluation in the coronavirus disease 2019 era and provide guidance to mitigate viral transmission',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33273410',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1847,\n",
       "     'offsetInEndSection': 1996,\n",
       "     'text': 'he final exam, negative brain flow was demonstrated 12 of 15 patients with craniectomy (80%) and 111 of 123 patients without craniectomy (90.2%). The',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31456011',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 109,\n",
       "     'text': 'Use of CPAP as an alternative to the apnea test during the determination of brain death in hypoxemic patients',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32667440',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 168,\n",
       "     'text': 'The apnea test, which involves disconnection from the mechanical ventilator, presents risks during the determination of brain death, especially in hypoxemic patients. W',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32667440',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 584,\n",
       "     'offsetInEndSection': 770,\n",
       "     'text': 'The 2 main radionuclidic techniques used in evaluation of brain death are radionuclide angiography with nonlipophilic radiopharmaceuticals and parenchymal imaging with lipophilic agents.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18514082',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 2143,\n",
       "     'offsetInEndSection': 2552,\n",
       "     'text': 'are not a substitute for the neurologic examination. Ancillary studies may be used to assist the clinician in making the diagnosis of brain death (a) when components of the examination or apnea testing cannot be completed safely due to the underlying medical condition of the patient; (b) if there is uncertainty about the results of the neurologic examination; (c) if a medication effect may be present; or (',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/22522447',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 538,\n",
       "     'text': 'BACKGROUND: Although the new Practice Parameters for brain death support a single examination, there is paucity of data comparing its impact to dual brain death (DBD) examinations.METHODS: We reviewed all brain deaths in our hospital over a 39-month period and compared the optional single brain death (SBD) exam requiring an apnea and a mandatory confirmatory blood flow test to the DBD for organ function at the time of death, rate of donation, and cost.RESULTS: Thirty-six patients had a SBD and 59 DBD exams, without any of them regai',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21604079',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 212,\n",
       "     'text': 'BACKGROUND: In Canada, ancillary tests, such as selective four vessels angiography (S4VA), are sometimes necessary for brain death (BD) diagnosis when the clinical exam cannot be completed or confounding factors ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/20724258',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1237,\n",
       "     'offsetInEndSection': 1571,\n",
       "     'text': 'We hypothesize that the existence of these apparent contradictions is related to differences in sensitivity of the physical examination and the confirmatory examinations, differences in localization of the physical examination and confirmatory tests, and differences between blood flow and cerebral function as markers of brain death.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18514082',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 984,\n",
       "     'offsetInEndSection': 1127,\n",
       "     'text': 'Nonetheless, on occasion patients clinically diagnosed with brain death will exhibit persistent intracranial blood flow or electrical activity.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/18514082',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Idecabtagene vicleucel can be used for treatment of which disease?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/34461271',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34256668',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33896344',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34527606',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34625232',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33626253',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33598857',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34854741',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34104374',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34145225'],\n",
       "   'ideal_answer': ['Idecabtagene vicleucel was shown to be effective for Relapsed and Refractory Multiple Myeloma.'],\n",
       "   'exact_answer': ['Multiple Myeloma'],\n",
       "   'type': 'factoid',\n",
       "   'id': '61f7ce26882a024a1000002d',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 342,\n",
       "     'text': 'B-cell maturation antigen (BCMA) has become a key target for antibody-drug conjugates, bispecific antibodies, chimeric antigen receptor T-cell therapies, and other immunotherapies in multiple myeloma. Some of these agents such as belantamab mafodotin and idecabtagene vicleucel have already received regulatory approval in the United States. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34461271',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1113,\n",
       "     'offsetInEndSection': 1321,\n",
       "     'text': 'These include isatuximab (also anti-CD38) and, in the past year, the antibody-drug conjugate (ADC) belantamab mafodotin and the chimeric antigen receptor (CAR) T-cell product idecabtagene vicleucel (ide-cel).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34527606',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 219,\n",
       "     'text': 'Matching-adjusted indirect comparison of efficacy outcomes for ciltacabtagene autoleucel in CARTITUDE-1 versus idecabtagene vicleucel in KarMMa for the treatment of patients with relapsed or refractory multiple myeloma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34256668',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 117,\n",
       "     'text': 'KarMMa-RW: comparison of idecabtagene vicleucel with real-world outcomes in relapsed and refractory multiple myeloma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34145225',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 233,\n",
       "     'offsetInEndSection': 406,\n",
       "     'text': 'Idecabtagene vicleucel (ide-cel, bb2121), a BCMA-directed CAR T-cell therapy, demonstrated efficacy in triple-class exposed RRMM patients in the KarMMa trial (NCT03361748). ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34145225',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 67,\n",
       "     'text': 'Idecabtagene Vicleucel in Relapsed and Refractory Multiple Myeloma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33626253',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 1917,\n",
       "     'offsetInEndSection': 2100,\n",
       "     'text': 'CONCLUSIONS: Ide-cel induced responses in a majority of heavily pretreated patients with refractory and relapsed myeloma; MRD-negative status was achieved in 26% of treated patients. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33626253',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1633,\n",
       "     'offsetInEndSection': 1822,\n",
       "     'text': 'Here, we will discuss the preclinical and clinical development of idecabtagene vicleucel and its future role in the changing treatment landscape of relapsed and refractory multiple myeloma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34104374',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1207,\n",
       "     'offsetInEndSection': 1632,\n",
       "     'text': 'After the recent publication of the results of a phase II trial confirming a notable efficacy and acceptable safety profile, idecabtagene vicleucel is the first CAR-T to gain regulatory US Food and Drug Administration approval to treat refractory multiple myeloma patients who have already been exposed to antibodies against CD38, proteasome inhibitors, and immunomodulatory agents and who are refractory to the last therapy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34104374',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 337,\n",
       "     'text': 'BACKGROUND AND OBJECTIVE: Registrational trials for ciltacabtagene autoleucel [cilta-cel]) and idecabtagene vicleucel [ide-cel] chimeric antigen receptor T-cell (CAR-T) therapies were single-arm studies conducted with relapse refractory multiple myeloma (MM) patients who were triple-class-exposed (TCE) or triple-class-refractory (TCR).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33598857',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 276,\n",
       "     'text': 'BACKGROUND: Idecabtagene vicleucel (ide-cel, also called bb2121), a B-cell maturation antigen-directed chimeric antigen receptor (CAR) T-cell therapy, has shown clinical activity with expected CAR T-cell toxic effects in patients with relapsed and refractory multiple myeloma.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33626253',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 325,\n",
       "     'text': 'Idecabtagene vicleucel (ide-cel, bb2121), a chimeric antigen receptor (CAR) T cell therapy, has been investigated in patients with relapsed and refractory multiple myeloma (RRMM) who have received an immunomodulatory drug, proteasome inhibitor, and anti-CD38 antibody in the single-arm phase 2 KarMMa clinical trial. Two ther',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33896344',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 269,\n",
       "     'text': 'BACKGROUND: Idecabtagene vicleucel (ide-cel, also called bb2121), a B-cell maturation antigen-directed chimeric antigen receptor (CAR) T-cell therapy, has shown clinical activity with expected CAR T-cell toxic effects in patients with relapsed and refractory multiple m',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33626253',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 98,\n",
       "     'text': 'The role of idecabtagene vicleucel in patients with heavily pretreated refractory multiple myeloma',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34104374',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 99,\n",
       "     'text': 'The role of idecabtagene vicleucel in patients with heavily pretreated refractory multiple myeloma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34104374',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 316,\n",
       "     'text': 'Idecabtagene vicleucel (ide-cel, bb2121), a chimeric antigen receptor (CAR) T cell therapy, has been investigated in patients with relapsed and refractory multiple myeloma (RRMM) who have received an immunomodulatory drug, proteasome inhibitor, and anti-CD38 antibody in the single-arm phase 2 KarMMa clinical trial.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33896344',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 341,\n",
       "     'text': 'Idecabtagene vicleucel (ide-cel), a novel chimeric antigen receptor (CAR) T-cell therapy targeting B-cell maturation antigen (BCMA), has recently gained approval by the US FDA for relapsed and refractory multiple myeloma (RRMM) after multicenter trials have demonstrated unprecedented results in this difficult-to-treat subgroup of patients.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34854741',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 97,\n",
       "     'text': 'Idecabtagene vicleucel (ide-cel) CAR T-cell therapy for relapsed and refractory multiple myeloma.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34854741',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 392,\n",
       "     'text': 'BACKGROUND: Idecabtagene vicleucel (ide-cel, also called bb2121), a B-cell maturation antigen-directed chimeric antigen receptor (CAR) T-cell therapy, has shown clinical activity with expected CAR T-cell toxic effects in patients with relapsed and refractory multiple myeloma.METHODS: In this phase 2 study, we sought to confirm the efficacy and safety of ide-cel in patients with relapsed an',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33626253',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 499,\n",
       "     'offsetInEndSection': 886,\n",
       "     'text': 'The bulk of CAR T-cell constructs currently in clinical development target the B-cell maturation antigen (BCMA) and to date only idecabtagene vicleucel (ide-cel) is approved by the Food and Drug Administration (FDA) for commercial use in adult patients with relapsed or refractory MM based on the promising clinical responses and positive safety record shown in the pivotal KarMMa study.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34625232',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 233,\n",
       "     'offsetInEndSection': 405,\n",
       "     'text': 'Idecabtagene vicleucel (ide-cel, bb2121), a BCMA-directed CAR T-cell therapy, demonstrated efficacy in triple-class exposed RRMM patients in the KarMMa trial (NCT03361748).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34145225',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': \"Where in the body would  Schlemm's canal be found\",\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/28291197',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25061871',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/1426078',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/25061877',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/729457',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33488009',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33613251',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/10581564',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30242758',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32672594'],\n",
       "   'ideal_answer': [\"Schlemm's canal can be found in the anterior chamber of the eye.\",\n",
       "    \"Schlemm's canal is found in the eye.\",\n",
       "    \"Schlemm's canal  is in the eye\",\n",
       "    \"Schlemm's canal is found in the eye, specifically in the trabecular meshwork of the anterior chamber of the eye.\"],\n",
       "   'exact_answer': ['eye'],\n",
       "   'type': 'factoid',\n",
       "   'id': '640f8641201352f04a00002c',\n",
       "   'snippets': [{'offsetInBeginSection': 184,\n",
       "     'offsetInEndSection': 402,\n",
       "     'text': \" This retrospective case series included patients with PCG who underwent trabeculotomy using an illuminated microcatheter with the intent of catheterizing the full circumference of Schlemm's canal in a single procedure\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33488009',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 9,\n",
       "     'offsetInEndSection': 195,\n",
       "     'text': \"To study the effect of 3 Schlemm's canal (SC) microinvasive glaucoma surgery (MIGS) devices on outflow facility.DESIGN: Paired comparisons, randomized design, baseline-controlled study.P\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32672594',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 85,\n",
       "     'text': \"Outflow Facility Effects of 3 Schlemm's Canal Microinvasive Glaucoma Surgery Devices.\",\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32672594',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 146,\n",
       "     'offsetInEndSection': 282,\n",
       "     'text': \"Previously, blood reflux from Schlemm's canal (SC) after long-standing glaucoma surgery was reported in eyes treated with excisional LOT\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33613251',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 241,\n",
       "     'text': \"AIM: to comparatively evaluate the hypotensive effect of minimally invasive (type 1) and basic (type 2) subscleral removing of the external wall of Schlemm's canal with subsequent laser trabeculopuncture in patients with open-angle glaucoma \",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28291197',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 139,\n",
       "     'offsetInEndSection': 397,\n",
       "     'text': \"Schlemm's canal (SC) is a large, lymphatic-like vessel in the anterior chamber of the eye, which is essential for aqueous humor drainage required to maintain intraocular pressure and is sensitive to defects in blood and lymphatic vascular signaling pathways.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30242758',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 161,\n",
       "     'text': \"Schlemm's canal (SC) is a specialized vascular structure in the eye that functions to drain aqueous humor from the intraocular chamber into systemic circulation.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25061877',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 166,\n",
       "     'text': \"Schlemm's canal (SC) is a unique vascular structure that functions to maintain fluid homeostasis by draining aqueous humor from the eye into the systemic circulation.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/25061871',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 409,\n",
       "     'text': \"We have tried to answer the question of whether the endothelium of Schlemm's canal is derived from and retains properties of blood vessels by studying: (1) the development of Schlemm's canal in human fetal eyes; (2) the existence of Weibel-Palade bodies in human neonatal, adult human and adult monkey eyes; and (3) the presence of blood coagulation Factor VIII-related antigen in adult human and monkey eyes.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1426078',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1450,\n",
       "     'offsetInEndSection': 1628,\n",
       "     'text': \"Our results indicate that the endothelium of Schlemm's canal is derived from a vascular origin and that even in the adult eye it retains some of the properties of a blood vessel.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1426078',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1258,\n",
       "     'offsetInEndSection': 1449,\n",
       "     'text': \"(3) Blood coagulation Factor VIII-related antigen was detected in the endothelium of the collector channels and Schlemm's canal, as well as in the blood vessels of the other parts of the eye.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/1426078',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 454,\n",
       "     'offsetInEndSection': 579,\n",
       "     'text': \" Injecting fluorescein directly in Schlemm's canal proves that in cadaver eyes without glaucoma the lumen is open all around.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/729457',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 182,\n",
       "     'text': \"Differences in parameters of ciliary body and Schlemm's canal in different meridians of the eyeball has been described using histotopographic methods on 20 eyeballs of human corpses.\",\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/10581564',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What does mitapivat do?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/35576529',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/36260990',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35988546',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34889443',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34987744',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35964609',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/36124781',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31974203',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35417638',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31483964'],\n",
       "   'ideal_answer': ['Mitapivat is an oral drug that activates the pyruvate kinase enzyme, increasing ATP production and reducing levels of 2,3-diphosphoglycerate. It is used for the treatment of hereditary hemolytic anemias',\n",
       "    'Mitapivat (AG-348) is a novel, first-in-class oral small molecule allosteric activator of the pyruvate kinase enzyme. MitapivAT has been shown to significantly upregulate both wild-type and numerous mutant forms of erythrocyte Pyruvates (PKR), increasing adenosine triphosphate (ATP) production and reducing levels of 2,3-diphosphoglycerate.',\n",
       "    'Mitapivat is a novel pyruvate kinase activator, for the treatment of hereditary hemolytic anemia',\n",
       "    'Mitapivat (AG-348) is a novel, first-in-class oral small molecule allosteric activator of the pyruvate kinase enzyme.  Mitapvat has been shown to significantly upregulate both wild-type and numerous mutant forms of erythrocyte Pyruvates kinase (PKR), increasing adenosine triphosphate (ATP) production and reducing levels of 2,3-diphosphoglycerate.',\n",
       "    'Mitapivat (AG-348) is a novel, first-in-class oral small molecule allosteric activator of the pyruvate kinase enzyme. It has been shown to significantly upregulate both wild-type and numerous mutant forms of erythrocyte pyruve kinase (PKR), increasing adenosine triphosphate (ATP) production and reducing levels of 2,3-diphosphoglycerate.',\n",
       "    'Mitapivat is a novel, first-in-class oral small molecule allosteric activator of the pyruvate kinase enzyme. It has been shown to significantly upregulate both wild-type and numerous mutant forms of erythrocyte pyruvate kinase (PKR), increasing adenosine triphosphate (ATP) production and reducing levels of 2,3-diphosphoglycerate. It has been used for the treatment of hereditary hemolytic anemias, and has been shown to increase the hemoglobin level in patients with pyruvate kinase deficiency.',\n",
       "    'Mitapivat is an oral, small-molecule allosteric activator of pyruvate kinase in red cells.',\n",
       "    'Mitapivat is an oral, first-in-class medication that activates the pyruvate kinase enzyme in red blood cells. This activation increases ATP production and reduces 2,3-diphosphoglycerate levels, ultimately helping to treat hereditary hemolytic anemias such as pyruvate kinase deficiency by increasing hemoglobin levels',\n",
       "    'Mitapivat is an investigational drug that inhibits the activity of dihydroorotate dehydrogenase (DHODH), an enzyme involved in the de novo pyrimidine biosynthesis pathway. It is being studied as a potential treatment for pyruvate kinase deficiency, a rare inherited metabolic disorder.'],\n",
       "   'type': 'summary',\n",
       "   'id': '643def7057b1c7a31500003a',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 96,\n",
       "     'text': 'Mitapivat, a novel pyruvate kinase activator, for the treatment of hereditary hemolytic anemias.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34987744',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 348,\n",
       "     'text': 'Mitapivat (AG-348) is a novel, first-in-class oral small molecule allosteric activator of the pyruvate kinase enzyme. Mitapivat has been shown to significantly upregulate both wild-type and numerous mutant forms of erythrocyte pyruvate kinase (PKR), increasing adenosine triphosphate (ATP) production and reducing levels of 2,3-diphosphoglycerate. ',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34987744',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 112,\n",
       "     'offsetInEndSection': 202,\n",
       "     'text': 'Mitapivat is an oral, small-molecule allosteric activator of pyruvate kinase in red cells.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31483964',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 142,\n",
       "     'offsetInEndSection': 294,\n",
       "     'text': 'mitapivat, an oral, first-in-class activator of erythrocyte pyruvate kinase, increased the hemoglobin level in patients with pyruvate kinase deficiency.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35417638',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 118,\n",
       "     'offsetInEndSection': 347,\n",
       "     'text': 'Mitapivat has been shown to significantly upregulate both wild-type and numerous mutant forms of erythrocyte pyruvate kinase (PKR), increasing adenosine triphosphate (ATP) production and reducing levels of 2,3-diphosphoglycerate.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34987744',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 104,\n",
       "     'offsetInEndSection': 332,\n",
       "     'text': 'In activating red blood cell pyruvate kinase and glycolysis, mitapivat (AG-348) increases adenosine triphosphate (ATP) levels and decreases the 2,3-diphosphoglycerate (2,3-DPG) concentration, an upstream precursor in glycolysis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35576529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 117,\n",
       "     'text': 'Mitapivat (AG-348) is a novel, first-in-class oral small molecule allosteric activator of the pyruvate kinase enzyme.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34987744',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 236,\n",
       "     'text': 'BACKGROUND: Mitapivat, an oral activator of pyruvate kinase (PK) in red blood cells (RBCs), has shown significant improvements in haemoglobin and haemolysis among patients with pyruvate kinase deficiency who were not receiving regular t',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35988546',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 110,\n",
       "     'offsetInEndSection': 490,\n",
       "     'text': 'tic anemia. In a phase 2 study, mitapivat, an oral, first-in-class activator of erythrocyte pyruvate kinase, increased the hemoglobin level in patients with pyruvate kinase deficiency.METHODS: In this global, phase 3, randomized, placebo-controlled trial, we evaluated the efficacy and safety of mitapivat in adults with pyruvate kinase deficiency who were not receiving regular r',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35417638',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 175,\n",
       "     'text': 'Kuo and colleagues1 evaluated the safety and efficacy of mitapivat, an oral pyruvate kinase activator, in\\xa0adults with non-transfusion-dependent α-thalassemia or β-thalassemia.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36260990',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 186,\n",
       "     'text': 'Safety and efficacy of mitapivat, an oral pyruvate kinase activator, in adults with non-transfusion dependent α-thalassaemia or β-thalassaemia: an open-label, multicentre, phase 2 study.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35964609',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 180,\n",
       "     'text': 'AG-348 (Mitapivat), an allosteric activator of red blood cell pyruvate kinase, increases enzymatic activity, protein stability, and ATP levels over a broad range of PKLR genotypes.',\n",
       "     'beginSection': 'title',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31974203',\n",
       "     'endSection': 'title'},\n",
       "    {'offsetInBeginSection': 853,\n",
       "     'offsetInEndSection': 962,\n",
       "     'text': 'Mitapivat, a pyruvate kinase activator, represents a unique mechanism to mitigate ineffective erythropoiesis.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34889443',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 258,\n",
       "     'offsetInEndSection': 477,\n",
       "     'text': ' We aimed to investigate the safety and efficacy of mitapivat (Agios Pharmaceuticals, Cambridge, MA, USA), a pyruvate kinase activator, in adults with non-transfusion-dependent (NTD) α-thalassaemia or NTD β-thalassaemia',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35964609',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 754,\n",
       "     'offsetInEndSection': 899,\n",
       "     'text': 'RT OPINION: Mitapivat is a first-in-class allosteric activator of pyruvate kinase and the first FDA-approved disease directed therapy for PKD. It',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36124781',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 121,\n",
       "     'offsetInEndSection': 293,\n",
       "     'text': ' In a phase 2 study, mitapivat, an oral, first-in-class activator of erythrocyte pyruvate kinase, increased the hemoglobin level in patients with pyruvate kinase deficiency',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35417638',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 247,\n",
       "     'text': 'BACKGROUND: Mitapivat, an oral activator of pyruvate kinase (PK) in red blood cells (RBCs), has shown significant improvements in haemoglobin and haemolysis among patients with pyruvate kinase deficiency who were not receiving regular transfusions',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35988546',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What type of DMD can viltolarsen be used for?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/34938127',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32519222',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33285037',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/36401027',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32453377',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/36401022',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35634851',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32483212',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31720560',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32947786'],\n",
       "   'ideal_answer': ['Viltolarsen can be used for the treatment of Duchenne muscular dystrophy (DMD) by skipping exon 53 of the DMD gene.',\n",
       "    'Viltolarsen is a targeted antisense oligonucleotide indicated for the treatment of Duchenne muscular dystrophy (DMD) caused by a confirmed mutation amenable to exon 53 skipping.',\n",
       "    'Viltolarsen is a phosphorodiamidate morpholino antisense oligonucleotide (PMO) designed to skip exon 53 of the DMD gene for the treatment of Duchenne muscular dystrophy (DMD)',\n",
       "    'Viltolarsen can be used for patients with Duchenne Muscular Dystrophy who are amenable to exon 53 skipping therapy.',\n",
       "    'Viltolarsen can be used for patients with DMD amenable to exon 53 skipping.',\n",
       "    'Viltolarsen can be used for the treatment of Duchenne muscular dystrophy (DMD) by restoring the reading frame of the DMD gene by skipping exon 53 and producing a truncated but functional form of dystrophin.',\n",
       "    'Viltolarsen be used for DMD patients that have mutations amenable to exon 53 skipping.'],\n",
       "   'exact_answer': ['exon 53'],\n",
       "   'type': 'factoid',\n",
       "   'id': '6417900b690f196b5100002a',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 174,\n",
       "     'text': 'Viltolarsen is a phosphorodiamidate morpholino antisense oligonucleotide (PMO) designed to skip exon 53 of the DMD gene\\xa0for the treatment of Duchenne muscular dystrophy (DMD)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36401022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 694,\n",
       "     'offsetInEndSection': 829,\n",
       "     'text': 'Viltolarsen restores the reading frame of the DMD gene by skipping \\xa0exon 53\\xa0and produces a truncated but functional form of dystrophin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36401022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 501,\n",
       "     'offsetInEndSection': 705,\n",
       "     'text': 'In March 2020, intravenous viltolarsen received its first global approval in Japan for the treatment of DMD in patients with confirmed deletion of the dystrophin gene that is amenable to exon 53 skipping.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32519222',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 305,\n",
       "     'text': 'Viltolarsen is a phosphorodiamidate morpholino antisense oligonucleotide (PMO) designed to skip exon 53 of the DMD gene\\xa0for the treatment of Duchenne muscular dystrophy (DMD), one of the most common lethal genetic disorders characterized by progressive degeneration of skeletal muscles and cardiomyopathy.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36401022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 594,\n",
       "     'offsetInEndSection': 762,\n",
       "     'text': 'Viltolarsen, also known as NS-065/NCNP-01, is a PMO developed through comprehensive sequence optimization and is designed to skip exon 53 on the DMD primary transcript.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31720560',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 187,\n",
       "     'text': 'OBJECTIVE: The novel morpholino antisense oligonucleotide viltolarsen targets exon 53 of the dystrophin gene, and could be an effective treatment for patients with Duchenne muscular dystr',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33285037',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 264,\n",
       "     'offsetInEndSection': 500,\n",
       "     'text': 'Viltolarsen binds to exon 53 of the dystrophin mRNA precursor and restores the amino acid open-reading frame by skipping exon 53, resulting in the production of a shortened dystrophin protein that contains essential functional portions.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32519222',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 264,\n",
       "     'offsetInEndSection': 386,\n",
       "     'text': 'Viltolarsen is a drug of phosphorodiamidate morpholino oligomer (PMO) chemistry, designed to skip exon 53 of the DMD gene.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34938127',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 482,\n",
       "     'offsetInEndSection': 569,\n",
       "     'text': ' Viltolarsen, an AON for DMD exon 53 skipping, was approved in Japan earlier this year.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32483212',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 331,\n",
       "     'offsetInEndSection': 506,\n",
       "     'text': ' Two exon 53 skipping PMOs, golodirsen and viltolarsen, have received conditional approval for treating patients due to their ability to restore dystrophin protein expression.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36401027',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 420,\n",
       "     'offsetInEndSection': 570,\n",
       "     'text': 'en for DMD exon 51 skipping and golodirsen and viltolarsen for DMD exon 53 skipping, have been approved in the last 4 years. We are witnessing the sta',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32947786',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 546,\n",
       "     'text': 'Importance: An unmet need remains for safe and efficacious treatments for Duchenne muscular dystrophy (DMD). To date, there are limited agents available that address the underlying cause of the disease.Objective: To evaluate the safety, tolerability, and efficacy of viltolarsen, a novel antisense oligonucleotide, in participants with DMD amenable to exon 53 skipping.Design, Setting, and Participants: This phase 2 study was a 4-week randomized clinical trial for safety followed by a 20-week open-label treatment period of patients aged 4 to 9',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32453377',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 530,\n",
       "     'text': 'BACKGROUND: Duchenne muscular dystrophy (DMD) is a rare, genetic disease caused by mutations in the DMD gene resulting in an absence of functional dystrophin protein. Viltolarsen, an exon 53 skipping therapy, has been shown to increase endogenous dystrophin levels. Herein, long-term (>2 years) functional outcomes in viltolarsen treated patients were compared to a matched historical control group.OBJECTIVE: To evaluate long-term efficacy and safety of the anti-sense oligonucleotide viltolarsen in the treatment of patients wit',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35634851',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 705,\n",
       "     'text': 'Viltolarsen (Viltepso® in Japan) is a phosphorodiamidate morpholino antisense oligonucleotide being developed by Nippon Shinyaku, in collaboration with the National Center of Neurology and Psychiatry (NCNP), for the treatment of Duchenne muscular dystrophy (DMD). Viltolarsen binds to exon 53 of the dystrophin mRNA precursor and restores the amino acid open-reading frame by skipping exon 53, resulting in the production of a shortened dystrophin protein that contains essential functional portions. In March 2020, intravenous viltolarsen received its first global approval in Japan for the treatment of DMD in patients with confirmed deletion of the dystrophin gene that is amenable to exon 53 skipping.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32519222',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 175,\n",
       "     'text': 'Viltolarsen is a phosphorodiamidate morpholino antisense oligonucleotide (PMO) designed to skip exon 53 of the DMD gene\\xa0for the treatment of Duchenne muscular dystrophy (DMD),',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36401022',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Is there a way to distinguish COVID-19 clinically from other respiratory illnesses, such as influenza?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/32127123',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33135801',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/36119462',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32881022',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34484383',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34226206',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33995342',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/33615750',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/32903584',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35412847'],\n",
       "   'ideal_answer': ['No, the clinical features of COVID-19 overlap substantially with influenza and other respiratory viral illnesses. There is no way to distinguish among them without testing.',\n",
       "    \"No, there isn't a definitive way to distinguish COVID-19 clinically from other respiratory illnesses, such as influenza, as they share many similar symptoms. Proper testing, such as PCR or antibody tests, is required for accurate diagnosis.\"],\n",
       "   'exact_answer': 'no',\n",
       "   'type': 'yesno',\n",
       "   'id': '6428da98690f196b51000053',\n",
       "   'snippets': [{'offsetInBeginSection': 271,\n",
       "     'offsetInEndSection': 574,\n",
       "     'text': ' Unfortunately, COVID-19 patients have symptoms similar to other common illnesses. Here, we hypothesize the order of symptom occurrence could help patients and medical professionals more quickly distinguish COVID-19 from other respiratory diseases, yet such essential information is largely unavailable.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32903584',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1275,\n",
       "     'offsetInEndSection': 1382,\n",
       "     'text': 'No symptoms are characteristic of a single infectious agent, with flu-like disorders being the most common.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/36119462',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1227,\n",
       "     'offsetInEndSection': 1563,\n",
       "     'text': 'We show that although COVID-19 and influenza are different in many ways, there are numerous similarities; thus, in addition to using nucleic acid-based polymerase chain reaction (PCR) and antibody-based approaches, clinicians and epidemiologists should distinguish between the two using their respective characteristics in early stages.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33615750',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 12,\n",
       "     'offsetInEndSection': 210,\n",
       "     'text': 'An understanding of differences in clinical phenotypes and outcomes COVID-19 compared with other respiratory viral infections is important to optimise the management of patients and plan healthcare.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34226206',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1600,\n",
       "     'offsetInEndSection': 1859,\n",
       "     'text': 'SARS-CoV-2 is associated with more severe outcomes compared with other respiratory viruses, and although associated with specific patient and clinical characteristics at admission, a substantial overlap precludes discrimination based on these characteristics.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34226206',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 272,\n",
       "     'offsetInEndSection': 353,\n",
       "     'text': 'Unfortunately, COVID-19 patients have symptoms similar to other common illnesses.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32903584',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 168,\n",
       "     'offsetInEndSection': 284,\n",
       "     'text': 'Clinical presentation of COVID-19, however, can be difficult to distinguish from other respiratory viral infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32881022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 177,\n",
       "     'text': 'It is difficult to distinguish coronavirus disease-2019 (COVID-19) from other viral respiratory tract infections owing to the similarities in clinical and radiological findings.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33135801',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 224,\n",
       "     'offsetInEndSection': 370,\n",
       "     'text': 'Although COVID-19 has been extensively characterized clinically, the factors distinguishing SARS-CoV-2 from other respiratory viruses are unknown.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/33995342',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 297,\n",
       "     'offsetInEndSection': 546,\n",
       "     'text': ' Since the symptoms are similar to other respiratory infections, differential diagnosis in travellers arriving from countries with wide-spread COVID-19 must include other more common infections such as influenza and other respiratory tract diseases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32127123',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 167,\n",
       "     'offsetInEndSection': 284,\n",
       "     'text': ' Clinical presentation of COVID-19, however, can be difficult to distinguish from other respiratory viral infections.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/32881022',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1491,\n",
       "     'offsetInEndSection': 1576,\n",
       "     'text': 'COVID-19 and flu are two respiratory illnesses which share similar clinical symptoms.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35412847',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 700,\n",
       "     'offsetInEndSection': 881,\n",
       "     'text': 'During influenza season, differentiating other causes of respiratory illness from COVID-19 is difficult, because common clinical manifestations of COVID-19 mimic those of influenza.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34484383',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'What are Luminopsins?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/31670406',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/28862809',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30152529',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26733686',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35836081',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35126293',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/26399324',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/30957296',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34281270',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/29577367'],\n",
       "   'ideal_answer': ['Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin that provides control of neuronal activity.',\n",
       "    'luminopsins (lmos) are chimeric proteins consisting of a luciferase fused to an opsin that provide control of neuronal activity and can be used to increase neuronal excitability if exposed to a suitable substrate.',\n",
       "    'Luminopsins are genetically engineered luciferases fused with ion channels such as ChR2 and ArchT, which can be used for optical and chemical control of neural activities.',\n",
       "    'Luminopsins are chimeric proteins consisting of a luciferase fused to an opsin that allow for control of neuronal activity through optogenetics.',\n",
       "    'Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin. They provide control of neuronal activity by fusing a light-generating luciferase to a light-sensing ion channel',\n",
       "    'Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin that provide control of neuronal activity. They are fusion proteins of light-generating luciferase and light-sensing ion channels that can be used to increase neuronal excitability if exposed to a suitable substrate.',\n",
       "    'Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin that provide control of neuronal activity.',\n",
       "    'Luminopsins are chimeric proteins created by fusing a luciferase enzyme with an opsin ion channel. These fusion proteins allow for the control of neuronal activity by generating light in response to a specific substrate, which then activates the opsin ion channel',\n",
       "    'Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin.',\n",
       "    'Luminopsins (LMOs) are fusion proteins that consist of a luciferase and an opsin. LMOs can increase neuronal excitability when exposed to a suitable substrate, making them potentially useful in controlling neuronal activity',\n",
       "    'Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin that provide control of neuronal activity'],\n",
       "   'type': 'summary',\n",
       "   'id': '6419cd02690f196b5100003a',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 127,\n",
       "     'text': 'Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin that provide control of neuronal activity',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31670406',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 115,\n",
       "     'text': 'Previous work has demonstrated that fusion of a luciferase to an opsin, to create a luminescent opsin or luminopsin',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28862809',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 362,\n",
       "     'offsetInEndSection': 540,\n",
       "     'text': 'luminopsins, fusion proteins of light-generating luciferase and light-sensing ion channels that could be used to increase neuronal excitability if exposed to a suitable substrate',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34281270',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 499,\n",
       "     'offsetInEndSection': 583,\n",
       "     'text': 'BL-OG is realized through opsin-luciferase fusion proteins called luminopsins (LMOs)',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35836081',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 235,\n",
       "     'text': 'Luminopsins are fusion proteins of luciferase and opsin that allow interrogation of neuronal circuits at different temporal and spatial resolutions by choosing either extrinsic physical or intrinsic biological light for its activation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26733686',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 201,\n",
       "     'text': 'Luminopsins (LMOs) are chimeric proteins consisting of a luciferase fused to an opsin that provide control of neuronal activity, allowing for less cumbersome and less invasive optogenetic manipulation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31670406',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 168,\n",
       "     'offsetInEndSection': 309,\n",
       "     'text': 'This is commonly achieved by utilizing a luminopsin (LMO), a fusion protein of a light-emitting luciferase tethered to a light-sensing opsin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30152529',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 237,\n",
       "     'text': 'Previous work has demonstrated that fusion of a luciferase to an opsin, to create a luminescent opsin or luminopsin, provides a genetically encoded means of manipulating neuronal activity via both chemogenetic and optogenetic approaches.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/28862809',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 357,\n",
       "     'offsetInEndSection': 572,\n",
       "     'text': 'We have bypassed the challenges of external light delivery by directly coupling a bioluminescent light source (a genetically encoded luciferase) to an inhibitory opsin, which we term an inhibitory luminopsin (iLMO).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26399324',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 299,\n",
       "     'offsetInEndSection': 619,\n",
       "     'text': 'A non-invasive and highly specific approach to neuronal stimulation is bioluminescent-optogenetics (BL-OG), where genetically expressed light emitting luciferases are tethered to light sensitive channelrhodopsins (luminopsins, LMO); neurons are activated by the addition of the luciferase substrate coelenterazine (CTZ).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35126293',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 648,\n",
       "     'offsetInEndSection': 895,\n",
       "     'text': 'We generated a neuronal precursor cell line expressing luminopsin 3 (LMO3), a luciferase-channelrhodopsin fusion protein, which responds to the luciferase substrate coelenterazine (CTZ) with emission of blue light that in turn activates the opsin.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/29577367',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 1157,\n",
       "     'offsetInEndSection': 1457,\n",
       "     'text': 'Thus, although preserving the ability to be activated by external light sources, LMOs expand the use of optogenetics by making the same opsins accessible to noninvasive, chemogenetic control, thereby allowing the same probe to manipulate neuronal activity over a range of spatial and temporal scales.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/26733686',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 499,\n",
       "     'offsetInEndSection': 584,\n",
       "     'text': 'BL-OG is realized through opsin-luciferase fusion proteins called luminopsins (LMOs).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35836081',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 496,\n",
       "     'offsetInEndSection': 709,\n",
       "     'text': 'We have previously addressed these challenges by employing bioluminescence and have created a new line of opto-chemogenetic probes termed luminopsins by fusing light-sensing opsins with light-emitting luciferases.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/30957296',\n",
       "     'endSection': 'abstract'}]},\n",
       "  {'body': 'Can you summarize the function of CDK9?',\n",
       "   'documents': ['http://www.ncbi.nlm.nih.gov/pubmed/9766517',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/31603123',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/11532614',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/27833949',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34062779',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/21200140',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/12432243',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/35628286',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34605036',\n",
       "    'http://www.ncbi.nlm.nih.gov/pubmed/34146121'],\n",
       "   'ideal_answer': ['CDK9 (cyclin-dependent kinase 9) is a serine/threonine-specific protein kinase that is involved in the regulation of transcription. It is a component of the positive transcription elongation factor b (P-TEFb) complex, which is essential for the phosphorylation of the C-terminal domain of RNA polymerase II and the subsequent transcriptional elongation.',\n",
       "    'CDK9 (cyclin-dependent kinase 9) is a protein kinase that plays a role in the regulation of gene expression. It is involved in the phosphorylation of the C-terminal domain of RNA polymerase II, which is necessary for transcriptional elongation and the expression of genes. CDK9 also plays a role in the regulation of other transcription factors, such as NF-κB, and is involved in the regulation of cell cycle progression.',\n",
       "    'CDK9 is a member of the Cdc2-like family of kinases that forms a complex with cyclin T and cyclin K partners. This complex plays a crucial role in regulating transcription elongation and is involved in various physiological processes and cell differentiation',\n",
       "    'CDK9 is a member of the Cdc2-like family of kinases and is involved in regulating several physiological processes. It is able to phosphorylate the retinoblastoma protein and plays a role in the differentiation of certain cell types, as well as being involved in the cell cycle progression in the form of transcription elongation. CDK9 is also a clinically validated drug target for antitumoral activity.',\n",
       "    'Cyclin-dependent kinase 9 (CDK9) is critical for RNA Polymerase II (Pol II) transcription initiation, elongation, and termination in several key biological processes including development, differentiation, and cell fate responses.'],\n",
       "   'type': 'summary',\n",
       "   'id': '64089b21201352f04a000021',\n",
       "   'snippets': [{'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 238,\n",
       "     'text': 'Cdk9 is a member of the Cdc2-like family of kinases. Its cyclin partners are members of the family of cyclin T (T1, T2a and T2b) and cyclin K. The Cdk9/cyclin T complexes appear to be involved in regulating several physiological processes',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12432243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 120,\n",
       "     'offsetInEndSection': 236,\n",
       "     'text': 'We previously cloned a cdc2-related kinase, cdk9/PITALRE, that is able to phosphorylate the retinoblastoma protein i',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9766517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 924,\n",
       "     'offsetInEndSection': 1057,\n",
       "     'text': 'hese results suggest that, among other roles, cdk9/PITALRE plays a role not unlike cdk5 in the differentiation of certain cell types.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/9766517',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 10,\n",
       "     'offsetInEndSection': 143,\n",
       "     'text': ' CDK9/Cyclin T1 kinase is a protein kinase, indirectly involved in the cell cycle progression in the form of transcription elongation',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/31603123',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 152,\n",
       "     'offsetInEndSection': 302,\n",
       "     'text': 'Among various clinically validated kinases, the cyclin-dependent kinases (CDK) are one of the extensively studied drug targets for clinical developmen',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34605036',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 246,\n",
       "     'text': 'Cyclin-dependent kinases (CDKs) are a broad family of proteins involved in the cell cycle and transcriptional regulation. In this article, we explore the antitumoral activity of a novel proteolysis-targeting chimera (PROTAC) compound against CDK9',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/35628286',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 240,\n",
       "     'offsetInEndSection': 427,\n",
       "     'text': 'Cdk9/cyclin T1 belongs to the P-TEFb complex, and is responsible for the phosphorylation of the carboxyl-terminal domain (CTD) of the RNA Polymerase II, thus promoting general elongation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12432243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 93,\n",
       "     'offsetInEndSection': 403,\n",
       "     'text': 'Cdk9 function depends on its kinase activity and also on its regulatory units: the T-family cyclins and cyclin K. Recently, several studies confirmed the role of cdk9 in different cellular processes such as signal transduction, basal transcription, HIV-Tat- and MyoD-mediated transcription and differentiation.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/11532614',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 930,\n",
       "     'offsetInEndSection': 1146,\n",
       "     'text': 'This data suggests the involvement of Cdk9 in several physiological processes in the cell, the deregulation of which may be related to the genesis of transforming events, that may in turn lead to the onset of cancer.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/12432243',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 197,\n",
       "     'text': 'CDK9 is a prominent member of the transcriptional CDKs subfamily, a group of kinases whose function is to control the primary steps of mRNA synthesis and processing by eukaryotic RNA polymerase II.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27833949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 293,\n",
       "     'text': 'Cyclin-dependent kinase 9 (CDK9), with its cyclin T regulatory subunit, is a component of the positive transcription elongation factor b (P-TEFb) complex, which stimulates transcription elongation and also functions in co-transcriptional histone modification, mRNA processing, and mRNA export.',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/21200140',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 481,\n",
       "     'text': 'Cyclin Dependent Kinase 9 (CDK9) is one of the most important transcription regulatory members of the CDK family. In conjunction with its main cyclin partner-Cyclin T1, it forms the Positive Transcription Elongation Factor b (P-TEFb) whose primary function in eukaryotic cells is to mediate the positive transcription elongation of nascent mRNA strands, by phosphorylating the S2 residues of the YSPTSPS tandem repeats at the C-terminus domain (CTD) of RNA Polymerase II (RNAP II).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34062779',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 367,\n",
       "     'text': 'CDK9 is a prominent member of the transcriptional CDKs subfamily, a group of kinases whose function is to control the primary steps of mRNA synthesis and processing by eukaryotic RNA polymerase II. As a cyclin-dependent kinase, CDK9 activation in vivo depends upon its association with T-type cyclins to assemble the positive transcription elongation factor (P-TEFb).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/27833949',\n",
       "     'endSection': 'abstract'},\n",
       "    {'offsetInBeginSection': 0,\n",
       "     'offsetInEndSection': 201,\n",
       "     'text': 'Cyclin-dependent kinase 9 (CDK9), the kinase component of positive transcription elongation factor b (P-TEFb), is essential for transcription of most protein-coding genes by RNA polymerase II (RNAPII).',\n",
       "     'beginSection': 'abstract',\n",
       "     'document': 'http://www.ncbi.nlm.nih.gov/pubmed/34146121',\n",
       "     'endSection': 'abstract'}]}]}"
      ]
     },
     "execution_count": 130,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result for Lexical Auto-id Stopwords False"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 122,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.212\n",
      "Mean Average Precision = 0.387\n",
      "Mean Time needed to execute a query = 0.225\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('37552126', 1.82634), ('34811514', 1.0), ('32903585', 0.96468), ('15514168', 0.95311), ('18473913', 0.88323), ('11152495', 0.87631), ('12894230', 0.86355), ('21143121', 0.86165), ('34146121', 0.85881)]\n"
     ]
    }
   ],
   "source": [
    "evaluation(query_parser,data, query_type=\"lexical\", path = \"Eval_results_BioASQ_10_abstracts/lexical_results.json\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result for Lexical Pmid Stopwords False"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 123,
   "metadata": {},
   "outputs": [],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "query_parser_pmid = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=True, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.218\n",
      "Mean Average Precision = 0.404\n",
      "Mean Time needed to execute a query = 0.312\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 1.0), ('32903585', 0.967719668958904), ('15514168', 0.947230535488355), ('37552126', 0.914876509921199), ('11152495', 0.8772968157872466), ('18473913', 0.8726219844423809), ('12894230', 0.8592261693078889), ('27833949', 0.8573171214184112), ('34146121', 0.8571585020251682), ('21143121', 0.854840207293013)]\n"
     ]
    }
   ],
   "source": [
    "evaluation(query_parser_pmid,data, query_type=\"lexical\", path = \"Eval_results_BioASQ_10_abstracts/lexical_results_pmid.json\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result lexical pmid with stopwords"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 124,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'theirs', 'whom', 'against', 'out', 'we', 'after', 'some', 'couldn', 'her', 'too', 'only', 'his', 'me', \"didn't\", 'both', 'if', 'yourself', 'hadn', 'won', 'ma', 'isn', 'she', 'further', 'yours', 't', 'themselves', 'needn', 'where', 'shouldn', \"wouldn't\", 'this', 'but', 'above', 'itself', 'not', 'wasn', 'what', \"haven't\", 'here', 'again', 'don', 'didn', 'shan', 'they', \"shan't\", 'how', 'd', \"aren't\", 'once', 'have', 'those', 'any', 'the', 'own', \"needn't\", 've', 'more', 'him', 'was', 'at', 'about', 'when', \"hasn't\", 'weren', 'nor', 'through', 'just', 'on', 'it', 'm', 'an', 'were', 'other', 'into', 'he', \"you'd\", 'that', \"she's\", 'mightn', 're', 'hers', 'aren', 'himself', 'haven', 'for', 'or', 'over', 'with', \"don't\", 'until', 'my', 'as', 'having', 'which', 'hasn', 'ourselves', 'i', 'does', \"couldn't\", 'most', 'is', 'ours', 'doesn', 'herself', \"that'll\", 'of', 'then', 'them', 'your', 'in', \"you're\", 'am', 'ain', \"mightn't\", 'there', 'wouldn', 's', 'than', 'yourselves', 'should', 'because', 'myself', 'these', \"won't\", 'why', 'doing', 'no', 'y', \"mustn't\", \"you've\", 'a', 'each', 'same', \"you'll\", \"isn't\", \"hadn't\", \"shouldn't\", \"should've\", \"it's\", 'under', 'their', 'now', 'be', 'very', 'you', 'up', 'below', 'our', 'by', 'had', 'from', 'did', 'few', \"wasn't\", 'being', \"doesn't\", 'to', 'between', \"weren't\", 'such', 'while', 'has', 'can', 'down', 'are', 'before', 'so', 'mustn', 'all', 'll', 'who', 'during', 'o', 'been', 'do', 'its', 'off', 'will', 'and'}\n"
     ]
    }
   ],
   "source": [
    "english_stopwords = set(stopwords.words('english'))\n",
    "print(english_stopwords)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 125,
   "metadata": {},
   "outputs": [],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 71,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.226\n",
      "Mean Average Precision = 0.418\n",
      "Mean Time needed to execute a query = 0.171\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 1.0), ('15514168', 0.9471752537904271), ('37552126', 0.9150419477774727), ('32903585', 0.9036914396823411), ('11152495', 0.877323588313823), ('12894230', 0.8590731661838458), ('34146121', 0.8570739124739399), ('21143121', 0.8547033136881785), ('30579871', 0.8537225497965703), ('37349855', 0.8355345756465075)]\n"
     ]
    }
   ],
   "source": [
    "evaluation(query_parser_stopwords,data, query_type=\"lexical\", path = \"Eval_results_BioASQ_10_abstracts/lex_results_stopwords.json\",stopwords_preprocessing=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result lexical autoid stopword True"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 72,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.213\n",
      "Mean Average Precision = 0.406\n",
      "Mean Time needed to execute a query = 0.172\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('37552126', 1.8272), ('34811514', 1.0), ('15514168', 0.95297), ('32903585', 0.89945), ('11152495', 0.87637), ('12894230', 0.86315), ('21143121', 0.8613), ('34146121', 0.85859), ('30579871', 0.85806)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical\"\n",
    "lexical_pmid = False\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "evaluation(query_parser_stopwords,data, query_type=\"lexical\", path = \"Eval_results_BioASQ_10_abstracts/lex_results_stopwords_auto_id.json\",stopwords_preprocessing=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result for Semantic without rescore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.129\n",
      "Mean Average Precision = 0.322\n",
      "Mean Time needed to execute a query = 0.208\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('32043568', 1.0), ('11532614', 0.99887), ('15528994', 0.99645), ('17471235', 0.99491), ('26500059', 0.99216), ('11960010', 0.99027), ('31857848', 0.98869), ('32934219', 0.98808), ('31695056', 0.98789), ('34316683', 0.98703)]\n"
     ]
    }
   ],
   "source": [
    "evaluation(query_parser,data, query_type=\"semantic\", path = \"Eval_results_BioASQ_10_abstracts/semantic_results.json\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Result Semantic with rescore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 74,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.130\n",
      "Mean Average Precision = 0.327\n",
      "Mean Time needed to execute a query = 0.218\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('17471235', 1.0), ('11532614', 0.9998), ('26500059', 0.99703), ('15528994', 0.99474), ('32043568', 0.99391), ('11960010', 0.99228), ('19065266', 0.99096), ('26295956', 0.99073), ('31857848', 0.9901), ('31695056', 0.98878)]\n"
     ]
    }
   ],
   "source": [
    "query_parser.set_rescore(True)\n",
    "evaluation(query_parser,data, query_type=\"semantic\", path = \"Eval_results_BioASQ_10_abstracts/semantic_results.json\")\n",
    "query_parser.set_rescore(False) # re insert the rescore to False"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result for Hybrid with lexical autoid Stopwords False"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.163\n",
      "Mean Average Precision = 0.392\n",
      "Mean Time needed to execute a query = 0.468\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('37552126', 0.91317), ('34811514', 0.5), ('32043568', 0.5), ('11532614', 0.499435), ('15528994', 0.498225), ('17471235', 0.497455), ('26500059', 0.49608), ('11960010', 0.495135), ('31857848', 0.494345), ('32934219', 0.49404), ('31695056', 0.493945)]\n"
     ]
    }
   ],
   "source": [
    "evaluation(query_parser, data, query_type=\"hybrid\", path = \"Eval_results_BioASQ_10_abstracts/hybrid_results.json\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result for Hybrid with lexical pmid Stopwords False"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 76,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.163\n",
      "Mean Average Precision = 0.415\n",
      "Mean Time needed to execute a query = 0.460\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 0.5), ('32043568', 0.5), ('11532614', 0.499435), ('15528994', 0.498225), ('17471235', 0.497455), ('26500059', 0.49608), ('11960010', 0.495135), ('31857848', 0.494345), ('32934219', 0.49404), ('31695056', 0.493945), ('34316683', 0.493515)]\n"
     ]
    }
   ],
   "source": [
    "evaluation(query_parser_pmid, data, query_type=\"hybrid\", path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid.json\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid pmid with stopwords "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 126,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.162\n",
      "Mean Average Precision = 0.415\n",
      "Mean Time needed to execute a query = 0.417\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 0.5), ('32043568', 0.5), ('11532614', 0.499435), ('15528994', 0.498225), ('17471235', 0.497455), ('26500059', 0.49608), ('11960010', 0.495135), ('31857848', 0.494345), ('32934219', 0.49404), ('31695056', 0.493945), ('34316683', 0.493515)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.5, beta=0.5, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_stopwords.json\", stopwords_preprocessing=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid autoid with stopwords "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 78,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.164\n",
      "Mean Average Precision = 0.400\n",
      "Mean Time needed to execute a query = 0.426\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('37552126', 0.9136), ('34811514', 0.5), ('32043568', 0.5), ('11532614', 0.499435), ('15528994', 0.498225), ('17471235', 0.497455), ('26500059', 0.49608), ('11960010', 0.495135), ('31857848', 0.494345), ('32934219', 0.49404), ('31695056', 0.493945)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical\"\n",
    "lexical_pmid = False\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.5, beta=0.5, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_autoid_stopwords.json\", stopwords_preprocessing=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid pmid with rescore"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 127,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.164\n",
      "Mean Average Precision = 0.410\n",
      "Mean Time needed to execute a query = 0.414\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 0.5), ('17471235', 0.5), ('11532614', 0.4999), ('26500059', 0.498515), ('15528994', 0.49737), ('32043568', 0.496955), ('11960010', 0.49614), ('19065266', 0.49548), ('26295956', 0.495365), ('31857848', 0.49505), ('31695056', 0.49439)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.5, beta=0.5, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid 0.6 lexical and 0.4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.236\n",
      "Mean Average Precision = 0.423\n",
      "Mean Time needed to execute a query = 0.406\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 0.6), ('15514168', 0.5683051522742563), ('37552126', 0.5490251686664837), ('32903585', 0.5422148638094046), ('11152495', 0.5263941529882937), ('12894230', 0.5154438997103075), ('34146121', 0.514244347484364), ('21143121', 0.5128219882129071), ('30579871', 0.5122335298779421), ('37349855', 0.5013207453879045), ('17471235', 0.4)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.6, beta=0.4, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_06-04.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid with alpha 0.7 and Beta 0.3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 132,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.237\n",
      "Mean Average Precision = 0.427\n",
      "Mean Time needed to execute a query = 0.522\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 0.7), ('15514168', 0.6630226776532989), ('37552126', 0.6405293634442308), ('32903585', 0.6325840077776388), ('11152495', 0.614126511819676), ('12894230', 0.601351216328692), ('34146121', 0.5999517387317579), ('21143121', 0.5982923195817249), ('30579871', 0.5976057848575992), ('37349855', 0.5848742029525552), ('17471235', 0.3)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.7, beta=0.3, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_07-03.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid with 0.8 and 0.2"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.237\n",
      "Mean Average Precision = 0.427\n",
      "Mean Time needed to execute a query = 0.447\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 0.8), ('15514168', 0.7577402030323417), ('37552126', 0.7320335582219782), ('32903585', 0.7229531517458729), ('11152495', 0.7018588706510585), ('12894230', 0.6872585329470766), ('34146121', 0.685659129979152), ('21143121', 0.6837626509505429), ('30579871', 0.6829780398372562), ('37349855', 0.668427660517206), ('17471235', 0.2)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.8, beta=0.2, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_08-02.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid with 0.9 lexical and 0.1 semantic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.237\n",
      "Mean Average Precision = 0.427\n",
      "Mean Time needed to execute a query = 0.422\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('34811514', 0.8), ('15514168', 0.7577402030323417), ('37552126', 0.7320335582219782), ('32903585', 0.7229531517458729), ('11152495', 0.7018588706510585), ('12894230', 0.6872585329470766), ('34146121', 0.685659129979152), ('21143121', 0.6837626509505429), ('30579871', 0.6829780398372562), ('37349855', 0.668427660517206), ('17471235', 0.2)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.8, beta=0.2, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_09-01.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid with 0.4 lexical and 0.6 semantic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 84,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.160\n",
      "Mean Average Precision = 0.359\n",
      "Mean Time needed to execute a query = 0.425\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('17471235', 0.6), ('11532614', 0.59988), ('26500059', 0.5982179999999999), ('15528994', 0.5968439999999999), ('32043568', 0.5963459999999999), ('11960010', 0.595368), ('19065266', 0.594576), ('26295956', 0.594438), ('31857848', 0.5940599999999999), ('31695056', 0.593268), ('34811514', 0.4)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.4, beta=0.6, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_04-06.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid with 0.3 lexical and 0.7 semantic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.160\n",
      "Mean Average Precision = 0.355\n",
      "Mean Time needed to execute a query = 0.407\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('17471235', 0.7), ('11532614', 0.6998599999999999), ('26500059', 0.6979209999999999), ('15528994', 0.6963179999999999), ('32043568', 0.6957369999999999), ('11960010', 0.694596), ('19065266', 0.693672), ('26295956', 0.693511), ('31857848', 0.69307), ('31695056', 0.6921459999999999), ('34811514', 0.3)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.3, beta=0.7, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_03-07.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid with 0.2 lexical and 0.8 semantic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.160\n",
      "Mean Average Precision = 0.357\n",
      "Mean Time needed to execute a query = 0.420\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('17471235', 0.8), ('11532614', 0.7998400000000001), ('26500059', 0.797624), ('15528994', 0.795792), ('32043568', 0.7951280000000001), ('11960010', 0.7938240000000001), ('19065266', 0.792768), ('26295956', 0.7925840000000001), ('31857848', 0.79208), ('31695056', 0.7910240000000001), ('34811514', 0.2)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.2, beta=0.8, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_02-08.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Result Hybrid with 0.1 lexical and 0.9 semantic"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 87,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "FINAL RESULTS \n",
      "Mean precision = 0.160\n",
      "Mean Average Precision = 0.358\n",
      "Mean Time needed to execute a query = 0.412\n",
      "Relevantni ['9766517', '31603123', '11532614', '27833949', '34062779', '21200140', '12432243', '35628286', '34605036', '34146121']\n",
      "Vraćeni [('17471235', 0.9), ('11532614', 0.8998200000000001), ('26500059', 0.897327), ('15528994', 0.895266), ('32043568', 0.894519), ('11960010', 0.8930520000000001), ('19065266', 0.891864), ('26295956', 0.891657), ('31857848', 0.89109), ('31695056', 0.889902), ('34811514', 0.1)]\n"
     ]
    }
   ],
   "source": [
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "query_parser_stopwords = QueryProcessor(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic, stopwords=english_stopwords)\n",
    "query_parser_stopwords.set_rescore(True)\n",
    "evaluation(query_parser_stopwords, data, query_type=\"hybrid\",alpha=0.1, beta=0.9, path = \"Eval_results_BioASQ_10_abstracts/hybrid_results_pmid_recore_01-09.json\", stopwords_preprocessing=True)\n",
    "query_parser_stopwords.set_rescore(False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Evaluation based on PubMed website"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Id list =  ['11076767', '38284126', '15094110', '30388611', '38837578', '12666201', '30661986', '22242013', '36997062', '22937083']\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "# Always tell NCBI who you are (your email address)\n",
    "Entrez.email = \"adela.ljajic@ivi.ac.rs\"\n",
    "def search_pubmed(query, limit = 10, mesh=True):\n",
    "    if not mesh:\n",
    "        query += \"[Title/Abstract]\"\n",
    "    # Use Entrez.esearch to search for articles matching the query in PubMed\n",
    "    handle = Entrez.esearch(db=\"pubmed\", term=query, retmax=limit, sort=\"relevance\",)\n",
    "    \n",
    "    record = Entrez.read(handle)\n",
    "    handle.close()\n",
    "    # Get the list of Ids returned by the search\n",
    "    id_list = record[\"IdList\"]\n",
    "    return id_list\n",
    "\n",
    "def fetch_details(id_list):\n",
    "    # Use Entrez.efetch to get the article details from the list of Ids\n",
    "    ids = ','.join(id_list)\n",
    "    handle = Entrez.efetch(db=\"pubmed\", id=ids, retmode=\"xml\")\n",
    "    records = Entrez.read(handle)\n",
    "    handle.close()\n",
    "    return records\n",
    "# Example usage\n",
    "\n",
    "query = \"Is the protein Papilin secreted?\"\n",
    "id_list = search_pubmed(query)\n",
    "print(\"Id list = \",id_list)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 94,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Id list =  ['11076767', '38284126', '15094122', '21784067', '7515725']\n"
     ]
    }
   ],
   "source": [
    "query = \"Is the protein Papilin secreted?\"\n",
    "id_list = search_pubmed(query, mesh=False)\n",
    "print(\"Id list = \",id_list)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "metadata": {},
   "outputs": [],
   "source": [
    "def average_precision(retrived_doc, true_doc):\n",
    "    # Initialize variables\n",
    "    precision_sum = 0\n",
    "    num_retrieved_docs = 0\n",
    "    \n",
    "    # Calculate precision at each relevant document position\n",
    "    for i, retrived in enumerate(retrived_doc, start=1):\n",
    "        pmid = retrived  # CHAGEd from pmid, _ = retrived to pmid = retrived (CAUSED PROBLEM IN EVALUATION ON PUBMEDSEARCH)\n",
    "        if pmid in true_doc:  # Check if the document is relevant\n",
    "            num_retrieved_docs += 1\n",
    "            precision_sum += num_retrieved_docs / i  # Calculate precision at cutoff i\n",
    "\n",
    "    # Calculate average precision\n",
    "    if num_retrieved_docs == 0:\n",
    "        return 0\n",
    "    else:\n",
    "        return precision_sum / num_retrieved_docs\n",
    "\n",
    "def evaluation_pubmed(query_type, data,mesh=True,path = \"Eval_results_BioASQ_10_abstracts/query_result.json\"):\n",
    "    avg_precisions_sum = [] # sum all average precision and divide with number of queries \n",
    "    precisions_sum = []\n",
    "    queries_time = []\n",
    "    for i,question in enumerate(data['questions']):\n",
    "        dict_to_save = {}\n",
    "        query = question['body']\n",
    "        dict_to_save['query'] = query\n",
    "        dict_to_save['query_type'] = query_type\n",
    "        relevant_documents = clean_documents(question['documents'])\n",
    "        start_time = time.time()\n",
    "        \n",
    "        results = search_pubmed(query, limit = len(relevant_documents),mesh=mesh)\n",
    "        queries_time.append(time.time() - start_time)\n",
    "        \n",
    "        dict_to_save['true_documents'] = list(relevant_documents)\n",
    "        dict_to_save['retrieved_documents'] = results\n",
    "       \n",
    "        number_retrieved_documents = 0\n",
    "        print(results)\n",
    "        for pmid in results:\n",
    "            if pmid in relevant_documents:\n",
    "                number_retrieved_documents +=1\n",
    "\n",
    "        precision = number_retrieved_documents / len(relevant_documents)\n",
    "        recall = number_retrieved_documents / len(relevant_documents)\n",
    "        avg_precision = average_precision(results, relevant_documents)\n",
    "       \n",
    "        precisions_sum.append(precision)\n",
    "        #recalls.append(recall)\n",
    "        \n",
    "        avg_precisions_sum.append(avg_precision)\n",
    "        \n",
    "        dict_to_save['precision'] = precision\n",
    "        #dict_to_save['recall'] = recall\n",
    "        dict_to_save['avg_precision'] = avg_precision\n",
    "        with open(path, 'a') as output_file:\n",
    "            output_file.write(json.dumps(dict_to_save) + '\\n')\n",
    "        if (i+1) % 500 == 0:\n",
    "            print(f\"Analyzed {i+1} queries\")\n",
    "            print(\"Actual Results...\")\n",
    "            print(f\"Mean precision = {np.mean(precisions_sum):.3f}\")\n",
    "            #print(f\"Mean recall = {np.mean(recalls):.3f}\")\n",
    "            print(f\"Mean Average Precision = {np.mean(avg_precisions_sum):.3f}\")\n",
    "            print(f\"Mean Time needed to execute a query = {np.mean(queries_time):.3f}\")\n",
    "    print(\"FINAL RESULTS \")\n",
    "    print(f\"Mean precision = {np.mean(precisions_sum):.3f}\")\n",
    "    #print(f\"Mean recall = {np.mean(recalls):.3f}\")\n",
    "    print(f\"Mean Average Precision = {np.mean(avg_precisions_sum):.3f}\")\n",
    "    print(f\"Mean Time needed to execute a query = {np.mean(queries_time):.3f}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluation Pubmed with Mesh Term"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The mesh terms are applied automatically by PubMed website"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['11076767', '38284126', '15094110', '30388611', '38837578', '12666201', '30661986', '22242013', '36997062', '22937083']\n",
      "['30515780', '34024716', '36028118', '32092478', '29635231', '31545469', '33114380', '33893301', '29458123', '32193744']\n",
      "[]\n",
      "['22094949', '25618141', '36622962', '34843880', '23393190', '35015757', '23748105', '30607432', '25207863', '24344320']\n",
      "['21292530']\n",
      "['35582186', '21358698', '25230708', '33244632', '31109940', '36742404', '24529566', '28500396', '19114612', '22913895']\n",
      "['37057884', '29757017', '33460272', '30069763', '32673417', '37170083', '37998388', '26847859', '34696930', '25659402']\n",
      "['29899761', '16157025', '30031766', '10379685', '37686475', '10571517', '20442286', '26477352', '27351911', '30474100']\n",
      "['30247122', '38607052']\n",
      "['35302338', '779595', '39194734', '31663791', '33326012', '28082330', '21484537', '33339418', '23473408', '3779737']\n",
      "[]\n",
      "['26074382', '25201530', '22722839', '37020028', '32220301', '30527665', '29858674', '34408788', '34380014', '23986864']\n",
      "['22100391', '11566103', '23679081', '8910287', '10915787', '15888437', '9535927', '11564882', '12524447', '9030581']\n",
      "['34965415', '37315653', '38906019', '38144564', '37468929', '27096063', '23269521', '31031005', '37477784', '37215542']\n",
      "[]\n",
      "['36423648', '31131967', '38762324', '29273807', '36529768', '38583093', '37632469', '37698987', '37647046', '36452988']\n",
      "['30633901', '29362479', '38442890', '32105850', '36736970', '36059483', '28596378', '26528972', '33732142', '35143849']\n",
      "['34818112', '28678778', '34551229', '36856617', '30753825', '25891173', '30089911', '30675668', '30715153', '35867951']\n",
      "['27066913', '28904069', '35643636', '1524556', '32959501', '8752479']\n",
      "['35207380', '34879286', '31038764', '37147932', '20301377', '30017936', '36538025', '24011800', '14655925', '33985661']\n",
      "['28887948', '29447987', '33871055', '37209391', '32463723', '25268599', '31076057', '31917873', '25713988', '28197901']\n",
      "[]\n",
      "[]\n",
      "['36007534', '23418763', '34324048', '30185448', '34036532', '36662582', '30888540', '34643963', '24114955', '37539933']\n",
      "['36096847', '35892331', '35180868', '35696523', '33548389', '34331845', '31091522', '35538484', '36077348', '36768828']\n",
      "['30264496', '35840979', '36516271', '27905021', '23407075', '27759482', '5652709', '36419401', '12207579', '38779986']\n",
      "['37582765', '30417346', '38727722', '11390662', '25369423', '28938749', '20446924', '14576152', '12795601', '27059084']\n",
      "['32669442', '34844509', '33547286', '28663311']\n",
      "[]\n",
      "['30643265', '36288887', '24819662', '19651817', '37175809', '29495584', '22415709', '34919331', '31324602', '21460837']\n",
      "['30156005', '27540238', '24709446', '32293527', '34843669', '31032996', '38450797', '16869817', '35581390', '25136027']\n",
      "['32249969', '9216039']\n",
      "['31318171', '30061698', '37956047', '36443325', '37352775', '35199548', '37907514', '35249152', '37921070', '35605524']\n",
      "['37009826', '24469397', '30138317', '23938467', '15630085']\n",
      "[]\n",
      "['38512421', '20845110', '26521214', '35322536', '25436575', '21660047', '17460169', '1497922', '12370022', '28461498']\n",
      "['27292156', '35707985', '35313924', '12606185', '27081564', '37140364', '31715083', '37286896', '35409752', '32313074']\n",
      "[]\n",
      "['34002501']\n",
      "[]\n",
      "['37965210']\n",
      "['37614038']\n",
      "['26795636', '36589741', '34000626', '33494284', '32096922', '38097153', '27155222', '19448973', '25128087', '16631162']\n",
      "['30129968', '35279277', '38042725', '37115403', '36799783', '36640476', '38777646', '32812189', '26275039', '30940302']\n",
      "['26381604', '29942112', '23023959', '11477602', '35023948', '23516378', '38773883', '37730234', '33338304', '14564153']\n",
      "[]\n",
      "['36974474', '24830588', '34446277', '31061177', '39084816', '17701788', '29177641', '37057215', '22108283', '27130071']\n",
      "['19016324', '28069537', '20880963', '27115768', '27650974', '18279024', '18486124', '17204552', '29594048', '31931337']\n",
      "[]\n",
      "['37903791', '37883470', '29121339', '30259369', '15630479', '28204568', '30083922', '25289637', '19492354', '18279518']\n",
      "[]\n",
      "['27126040', '27611656', '22699394', '29066503', '35357570', '33426999', '21080056', '29018988', '22123459', '21046450']\n",
      "['30073149', '24163373', '22699452', '27077938', '19497860', '35003895', '28974232']\n",
      "['37066965']\n",
      "['34502127', '19407331', '37807920', '34423871', '34614543', '37807924', '31331009', '28045402', '28130064', '35795187']\n",
      "['28577343', '17168837', '9675642', '10516468', '7642575', '21559518', '25320291', '20140199', '15113910', '18610556']\n",
      "['36871404', '26559732', '36111884', '34200092', '28365490', '25620493', '29527493', '36058165', '28341824', '33132997']\n",
      "['35673561', '31036962', '33160389', '38181758', '37218012', '34522213', '33163006', '37196719', '36629270', '33152343']\n",
      "['27294413', '38457957', '23280760', '27824869', '38056243', '38383210', '17488586', '8850267', '24039139', '24223877']\n",
      "[]\n",
      "[]\n",
      "['10945981', '12468631', '22882569']\n",
      "['38034271', '36629534', '32115705', '37528649', '31448784', '15176428', '32339564', '32093079', '30615235', '27538377']\n",
      "['37711890', '28639965', '22009156', '26088475', '19942156', '34445238', '30124533', '21169670', '19034261', '15749805']\n",
      "[]\n",
      "[]\n",
      "['33838111', '30729798', '35968782', '30894700', '24492710', '30934924', '33984527', '33401659', '23669186', '31477236']\n",
      "['24349523']\n",
      "['32651541', '23400593', '24900694', '23084521', '36439412', '22753001', '26459177', '27327543', '36165233', '30022047']\n",
      "['37788591', '26206750', '18375577', '32946258']\n",
      "['35656794', '26530328', '31553938', '17624330', '28819924', '22575891', '24333015', '19762472', '22252321', '28561061']\n",
      "['20301390', '15685439', '19107440', '27118199', '30050733', '26093337', '31559554', '38758727', '33447892', '33099328']\n",
      "['31228046', '20301465', '25110572', '19209820', '11257471', '22358459', '30517146', '20012312', '19648653']\n",
      "['36518658', '35080077', '27466197', '29212949', '25300332', '26407011', '30594810', '23468085', '29783994', '38842106']\n",
      "['20378560', '24675553', '22045982', '19773262', '8630424', '15126435', '20454576', '25424902', '29408805', '29435930']\n",
      "[]\n",
      "['24508080', '19591228']\n",
      "['26626082', '28986702', '38505407', '28012079', '25772089', '23328875', '31440612', '33741509', '36248120', '34398337']\n",
      "[]\n",
      "['19277716', '18725402']\n",
      "['31705132', '21763610', '24105262', '29622626', '35333565', '35446649', '36408828', '24482116', '36918692', '28620025']\n",
      "['14529478', '32284336']\n",
      "['22084162']\n",
      "[]\n",
      "['28135145', '30643217', '37216690', '28644950', '10921325', '32412586', '22373003', '11530803', '1530123', '15611632']\n",
      "['24853414', '31900865', '37739573', '23393146', '38782015', '38972779', '18673445']\n",
      "['36777165', '37052588', '37165557', '25142900', '36179098', '38452153', '33423325', '33289449', '30557037', '37178386']\n",
      "['12663654', '15682436']\n",
      "['27030077', '27017286', '34544752', '37384539', '29703791', '31831640', '36183733', '33726508', '31174843', '28706010']\n",
      "[]\n",
      "[]\n",
      "['32338528', '25226019', '30968360', '22451171', '33773726', '27854218', '20569215', '23665194', '28331814', '28448740']\n",
      "['30696717', '6383477', '25122670', '26073855', '35020830', '27617035', '21825079', '34093640', '32338759', '24284169']\n",
      "['22422768']\n",
      "[]\n",
      "['31875553']\n",
      "['28985562', '26053496', '22780989', '24550742']\n",
      "['33927619', '35535861', '32520674', '32811367']\n",
      "['30458732', '34830207', '24504546', '21354436', '27017208', '20738246', '20139230', '27743518', '21700660', '36229870']\n",
      "['23103374', '19305745']\n",
      "['20951289', '23995804', '33112857', '32168960', '24892323', '33941164', '26412519', '36814464', '34310632', '31632667']\n",
      "[]\n",
      "['24703829', '32809652', '32757464', '11309216', '25726283', '28622122', '37590825', '29609724', '23467433', '20726288']\n",
      "['34817054', '38052811', '28778956', '35563127', '26598646', '33197229', '36434141', '20735237', '23924899', '21364325']\n",
      "['28076709', '35175439', '31893580', '34793745', '31562798', '34246328', '36735392', '33196153', '31572570', '36321557']\n",
      "['26649946', '32817534', '39060246']\n",
      "[]\n",
      "['35277681', '37344258', '26824195', '34656258', '31629553', '22290253', '38221545', '34778080', '37371674', '31536810']\n",
      "['30503519']\n",
      "['16973437', '22778132', '21738494', '25210768', '21878619', '24106327']\n",
      "['35685367', '34181707', '27083996', '37235058', '26711177', '32816606', '35182955', '33411704', '29199022', '21964334']\n",
      "['23814399', '34208124', '37051029', '27446290', '19949820', '22613401', '33293803', '34087844', '20608631', '34725607']\n",
      "['33317550', '35851375', '35584701', '35649485', '29980667', '31474572', '28344082', '31395983', '28344080', '30057200']\n",
      "['29261990', '38484868', '22537653', '34838485', '29361747', '37431665', '28153807', '37142240', '25482846', '29142973']\n",
      "['23060541', '22976135', '25163906', '25398844', '29502931', '28643756', '25762461', '23354807', '18616418', '24153102']\n",
      "[]\n",
      "['33824816', '35495849', '36576563', '31426529', '31969112', '35282610', '28090050', '29039237', '30240793', '35753675']\n",
      "['37493210', '33669431', '28878676', '36795905', '35844558', '28865357', '36274472', '28844952', '37589912', '34068797']\n",
      "[]\n",
      "['26804367', '27369318', '20622159', '28440774', '37715722', '17259493', '28476396', '38532027', '32102313', '37822082']\n",
      "['29319791', '35190066', '30888626', '3452124', '4764713', '12218915', '30075748', '19882789', '15569578', '14711147']\n",
      "[]\n",
      "['28657829', '28516949', '25237200', '34415298', '38278170', '29503741', '30193519', '35314883', '26205796', '30565236']\n",
      "['31643200', '37477153', '38186408', '34016626', '33209576', '26259424', '22937434', '28975065', '22564798', '33853431']\n",
      "['32310497', '35170334', '16307510', '32673595', '30246657', '22086450', '36762336', '18512714', '22095393', '15098162']\n",
      "['25959774', '37798116', '36824722', '19369356', '25113559', '25499216', '11507042']\n",
      "[]\n",
      "['37848984', '33037995', '9521088', '19396626', '24007659', '12869801', '31147910', '32751253', '24279997', '21933837']\n",
      "['27501293', '37947294', '30157388', '35743410', '37537928', '38286755', '37030640', '37460235', '33054533', '20200338']\n",
      "['37686035', '31544819', '33883551', '32488574', '35662162']\n",
      "['29385370', '34904798', '36097221', '31754328', '36138152', '30528964', '35690221', '34166502', '34921238', '36724512']\n",
      "['30699285', '18620092']\n",
      "['36877404', '34895496', '27409075', '12828802', '31524133', '22795212', '24533537', '35604633', '23896589', '34872018']\n",
      "['27294413', '27933523', '37543624', '37214980', '22080549', '32917924', '34211268', '30978202', '26656951', '27933521']\n",
      "['31536171', '33689456', '29692311']\n",
      "[]\n",
      "['30101644', '38639402', '38126311', '35366072', '35317768', '36529628', '38063364', '31432469', '36822009', '31977388']\n",
      "['30786043', '34367174', '28192188']\n",
      "['30446595', '35027572', '33974014', '31081399', '34128998', '37266987', '31132795', '35190383', '34633423', '28076702']\n",
      "[]\n",
      "[]\n",
      "['36403719', '38216005', '35231471', '32289295', '36223296', '35146185', '35686679', '32224803', '37228304', '38485975']\n",
      "['36283607', '32375958', '31076057', '32529893', '29318959', '38943971', '30193532']\n",
      "['29144467', '32660382', '29789364', '37236065', '36118023', '36967589', '34414776', '33641039', '36835135', '25505130']\n",
      "['33439265', '38218536', '24298022', '25381300', '27462398']\n",
      "['18341664']\n",
      "['32133006', '30932318', '33669066', '22660413', '29931674', '27411689', '26149933', '31998313', '19597846', '35385327']\n",
      "['33477371', '34424482', '29802543', '25219883', '28860152', '32234004', '36894562', '35347107', '35896610', '28887129']\n",
      "['30205942', '25221516', '29135293', '27314965', '30850267', '7202025', '6639422', '18392047', '804753']\n",
      "['36403719', '35041175', '32606014', '37513232', '33879198', '38886570', '37189659', '37121712', '34564697']\n",
      "[]\n",
      "['21171977', '27188817', '37629202', '31059641', '34918046', '28947124', '32657893', '33023443', '32745442', '33845728']\n",
      "['35568257', '35285276', '30837079', '24473371', '24280148', '22121323', '32341593', '19802555', '34509157', '30128422']\n",
      "['31629656', '38763154', '24223022', '33002438', '26936410', '39100827', '27568654', '37504343', '17644245', '30843003']\n",
      "['31905366', '31516082', '36471071', '34157074', '25313959', '35676475', '23043155', '35501690', '37975572', '37081319']\n",
      "['29233090', '19021773', '27391817']\n",
      "['36201396', '7254780']\n",
      "['30616405', '22751599', '25291944', '21439251', '33315881', '24647715', '28756532', '29020515', '28506134', '25807089']\n",
      "['38996202', '35638464', '33927725', '31381554', '8254024', '15778399', '8000227', '6998178']\n",
      "['32979540', '34166371', '15897469', '19515780']\n",
      "['32561705', '33513339', '34059811', '35531651', '37589246', '32521132', '37479139', '38204162', '32929000', '32200604']\n",
      "['32401547', '29194370', '26851604']\n",
      "['29930090', '19662496', '18050426', '36413017', '25936768', '23175478', '22118278', '33762200', '379595', '39020180']\n",
      "['32757523', '36350491', '34025110', '34897575', '35266417', '38174776', '33090003', '38032059', '29695594', '37033984']\n",
      "[]\n",
      "['36906275', '35154124', '32772418', '28925994', '34217595', '31219357', '35208467', '29808007', '33278357', '35044513']\n",
      "['31069056', '32469183', '36000243', '37335836', '36652173', '39159261', '34923058', '34408793', '38592603', '37863160']\n",
      "['24463181', '38613490', '11164044', '31433621', '34883451', '30798493', '38041485', '20461060', '2666845', '38469801']\n",
      "[]\n",
      "['16937360', '30992265', '34605776', '31339233', '32448380', '36228171', '27238576', '36349896', '21900192', '31732958']\n",
      "['34561985', '38109536', '38349514', '38215749', '28148298', '25344935', '38463392', '37350306', '32927603', '38413232']\n",
      "['29848770', '21035860']\n",
      "[]\n",
      "['35965424', '34096649', '37001971', '36349896', '31643285', '38429096', '30623346', '36180546', '32648854', '20301617']\n",
      "['35292111']\n",
      "['33626253', '36762851', '36627265', '34048683', '37958658', '38563345', '34893771', '36963592', '37731379', '37558706']\n",
      "['28717161', '24374259', '22022403', '403150', '1134721', '12692512', '6489767', '2751562']\n",
      "['36698833', '34889443', '36853806', '36935283', '35417638', '37934880', '37991635', '36355466', '36847623', '39041192']\n",
      "['34569969', '36995604', '38991781']\n",
      "[]\n",
      "['33398820', '33260589', '35836081', '38045286', '38559366', '28862809', '30957296', '35836082', '31670406', '36555724']\n",
      "[]\n",
      "FINAL RESULTS \n",
      "Mean precision = 0.077\n",
      "Mean Average Precision = 0.166\n",
      "Mean Time needed to execute a query = 1.124\n"
     ]
    }
   ],
   "source": [
    "evaluation_pubmed(data=data, query_type=\"PubMed website\", path=\"Eval_results_BioASQ_subset/Pubmed_mesh.json\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Evaluation PubMed without mesh"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['11076767', '38284126', '15094122', '21784067', '7515725']\n",
      "['31431733', '36028118', '27906049', '26925062', '27105133', '31486093', '22465238', '33308247', '31763801', '22360322']\n",
      "[]\n",
      "['22094949', '25618141', '34843880', '36622962', '35015757', '23748105', '30607432', '24344320', '23393190', '25207863']\n",
      "['21292530']\n",
      "[]\n",
      "['36880482']\n",
      "['29899761', '16157025', '30474100', '9496833', '21063743']\n",
      "['38607052']\n",
      "['39194734', '28082330', '33339418', '20875630', '20632954', '12643345', '21329899', '33191933', '35915685', '30732632']\n",
      "[]\n",
      "['30079888', '28332978', '30068710', '27170440']\n",
      "['22100391', '11566103', '23679081', '8910287', '15888437', '9030581', '23159405']\n",
      "['33925995']\n",
      "[]\n",
      "[]\n",
      "['30633901', '38442890', '28596378', '35143849', '20116986', '37298433', '21612942', '30642153', '37019879', '26738338']\n",
      "['27398650', '36856617', '30753825', '25891173', '30675668', '27433843', '29658430', '35867951', '30715153', '35367007']\n",
      "['27066913', '28904069', '35643636', '1524556']\n",
      "[]\n",
      "['28887948', '29447987', '33871055', '37209391', '32463723', '25268599', '31917873', '25713988', '28197901', '32598253']\n",
      "[]\n",
      "[]\n",
      "['33119240', '37539933', '28876750', '24114955', '23635878', '36458026', '33248373', '28779487', '17933616', '26604028']\n",
      "['34331845', '37575217', '30129068', '34384877', '37980468', '24724555', '34514028', '37542214', '33835295', '35720083']\n",
      "['30264496', '35840979', '36516271', '27905021', '27759482', '36419401', '23407075', '38779986', '19467033', '32951242']\n",
      "['30417346']\n",
      "['32669442', '34844509', '28663311']\n",
      "[]\n",
      "['24819662', '22415709', '17027967', '30219846']\n",
      "['30156005', '24709446', '27540238', '31032996', '32293527', '38450797', '35581390', '25136027']\n",
      "[]\n",
      "['37956047', '35605524', '36199867', '29750964', '33640616', '27083318', '21725702', '31318171', '26436483', '35181348']\n",
      "['37009826', '24469397', '30138317', '23938467']\n",
      "[]\n",
      "[]\n",
      "['27292156', '35707985', '35313924', '27081564', '37286896', '37140364', '35409752', '32313074', '24561283', '29411041']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['38097153', '16631162']\n",
      "['30129968', '36640476', '37115403', '32812189', '26275039', '34171759', '35813045', '37884407', '30940302', '26251008']\n",
      "['26381604', '29942112', '23516378', '23023959', '11477602', '38773883', '35023948', '33338304', '37730234', '38984779']\n",
      "[]\n",
      "['36974474', '24830588', '34446277', '31061177', '39084816', '17701788', '29177641', '22108283', '27130071', '30341577']\n",
      "['19016324', '28069537', '20880963', '27115768', '27650974', '18486124', '17204552', '29594048', '22073124', '24967815']\n",
      "[]\n",
      "['34906219', '28874668', '28859354', '31449516', '17760977', '26482678', '24365181', '24681619', '23193254', '30535005']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['34614543', '33886133', '32819372', '29369529', '26528002', '27885969', '38648172', '36210076', '34654781', '33523760']\n",
      "[]\n",
      "['28365490', '25620493', '32567335', '29572926']\n",
      "['34012073', '37218012', '31924176', '33463042', '36189207', '37500614', '38084420', '37742736', '37356744', '35415891']\n",
      "['7139628']\n",
      "[]\n",
      "[]\n",
      "['12468631']\n",
      "['36629534', '38034271', '32115705', '32339564', '31448784', '17578347', '23266818', '24025405', '28404607', '35341025']\n",
      "['22009156', '34445238', '34884891', '22994016']\n",
      "[]\n",
      "[]\n",
      "['33838111', '30729798', '35968782', '24492710', '30934924', '33984527', '33401659', '23669186', '31477236', '25587943']\n",
      "[]\n",
      "['32651541', '36439412', '27327543', '23400593', '23084521', '36165233', '24900694', '30022047', '22753001', '26459177']\n",
      "[]\n",
      "['35656794', '26530328', '31553938', '17624330', '28819924', '22575891', '24333015', '19762472', '22252321', '28561061']\n",
      "[]\n",
      "['31228046', '20301465', '25110572', '19209820', '11257471', '22358459', '30517146', '20012312', '19648653']\n",
      "['36518658', '29783994']\n",
      "['22045982', '24675553', '19773262', '20378560', '20454576', '15126435', '8630424', '29408805', '29435930', '28376382']\n",
      "[]\n",
      "['24508080']\n",
      "['25772089', '26536440', '25777743', '24502552']\n",
      "[]\n",
      "['19277716', '18725402']\n",
      "['31449799', '29622626', '35333565', '35446649', '36408828', '24105262', '24482116', '30467171', '37565750', '28620025']\n",
      "['32284336']\n",
      "[]\n",
      "[]\n",
      "['37216690', '10921325', '32412586', '28644950', '11530803', '15611632', '30715513', '9560531', '27165003', '14582604']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['27030077', '27017286', '36043521', '29125603', '37384539', '31831640', '27496135', '36183733', '34520734', '33726508']\n",
      "[]\n",
      "[]\n",
      "['32338528', '25226019', '30968360', '33773726', '27854218', '28331814', '28448740', '28448765', '23665194', '23932809']\n",
      "['6383477', '26073855', '30696717', '21825079', '34093640', '25122670', '32338759', '37600946', '27617035', '30018005']\n",
      "['22422768']\n",
      "[]\n",
      "[]\n",
      "['28985562', '26053496', '22780989', '24550742']\n",
      "[]\n",
      "[]\n",
      "['23103374']\n",
      "[]\n",
      "[]\n",
      "['24703829', '32809652', '32757464', '11309216', '25726283', '37590825', '29609724', '28622122', '23467433', '20726288']\n",
      "['34817054', '38052811', '28778956', '35563127', '33197229', '26598646', '36434141', '21364325', '27268234', '29160738']\n",
      "['38502720', '34212863', '33359193', '32592450', '36804737', '32181569', '21158712', '32478838', '31793882', '31763982']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['30503519']\n",
      "[]\n",
      "['35685367', '27083996', '34181707', '32816606', '37235058', '26711177', '35182955', '33411704', '12485467', '31831844']\n",
      "['27446290', '28913640', '29390489']\n",
      "['33317550', '35851375', '35584701', '35649485', '29980667', '31474572', '31395983', '28344080', '30057200', '33436560']\n",
      "['29361747', '29142973', '19633201', '19073821', '25331409', '19013120', '21979582']\n",
      "[]\n",
      "[]\n",
      "['33824816', '35495849', '36576563', '31426529', '31969112', '35282610', '28090050', '29039237', '30240793', '27629668']\n",
      "['32276534', '36008542', '37639472', '37493210', '28878676', '37602492', '28258697', '35844558', '34057191', '28844952']\n",
      "['37632708', '29024110', '37710073', '23709023', '24990201', '37710075', '35809768', '28877508', '35972838', '26910176']\n",
      "['26804367', '27369318', '28440774', '37715722', '20622159', '17259493', '28476396', '38532027', '32102313', '37822082']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['34016626', '33209576', '26259424', '22937434', '28975065', '22564798', '33853431', '7726366', '24977015', '32963956']\n",
      "[]\n",
      "['25959774', '37798116', '36824722', '25113559', '25499216', '19369356', '11507042']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['37686035', '31544819', '33883551', '35662162']\n",
      "['31754328', '30528964', '34921238', '34904798', '36097221', '36138152', '35770532', '34166502', '31187535', '34310745']\n",
      "['30699285', '18620092']\n",
      "['22795212', '24533537', '28723978', '29665120', '25358507', '6867245', '28302702', '37510511', '34121981', '26346049']\n",
      "['34211268']\n",
      "['37632708', '29024110', '37710073', '24990201', '37710075', '37931330', '26910176', '20392265', '27722918', '35408735']\n",
      "[]\n",
      "['30101644', '34449248', '38639402', '38126311', '36529628', '32630204', '38063364', '31432469', '33989272', '36822009']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['36403719', '38216005', '36223296', '35231471', '35146185', '35686679', '37228304', '38485975', '32289295', '32224803']\n",
      "['36283607', '32529893', '29318959', '30193532']\n",
      "['11544353', '16212498', '8288518', '20149602', '35334139', '36920212', '32192182', '29243493', '37814686', '33641039']\n",
      "['33439265', '38218536', '24298022', '25381300', '27462398']\n",
      "[]\n",
      "['26749537', '22660413', '30582882', '30932318', '32133006', '33669066', '29495527', '31998313', '33561383', '29931674']\n",
      "['31985384']\n",
      "[]\n",
      "['36403719', '35041175', '37513232', '33879198', '38886570', '37189659', '37121712', '34564697']\n",
      "[]\n",
      "['21171977', '27188817', '37629202', '31059641', '34918046', '31669734', '35746980', '34215255', '22027213', '28947124']\n",
      "['35285276', '30837079', '32000925', '25848493', '35797875', '32542130', '37362368', '37191879', '34613650', '34790475']\n",
      "['39100827', '15947588']\n",
      "['31905366', '31516082', '36471071', '34157074', '23043155', '37081319', '34611363', '25313959', '35676475', '35501690']\n",
      "[]\n",
      "[]\n",
      "['30616405', '22751599', '25291944', '24647715', '29020515', '26472414', '29185966', '21439251', '24418824', '33315881']\n",
      "['38996202', '35638464', '33927725', '31381554', '8254024', '15778399', '8000227', '6998178']\n",
      "['32979540', '34166371', '15897469', '19515780']\n",
      "['32561705', '33513339', '32521132', '38204162', '37479139', '36642627', '20301372', '33139979', '34375916', '37027823']\n",
      "[]\n",
      "['19662496', '34208174', '18050426', '27905614', '36413017', '25936768', '23175478', '22118278', '33762200', '10943411']\n",
      "[]\n",
      "[]\n",
      "['36906275', '28925994', '35154124', '32772418', '30580638', '38066881', '39117803', '29808007', '31219357', '33164775']\n",
      "['32469183', '36000243', '37335836', '36652173', '39159261', '34923058', '34408793', '38592603', '37863160', '34216282']\n",
      "['24463181', '38613490', '11164044', '31433621', '34883451', '30798493', '38041485', '20461060', '2666845', '38469801']\n",
      "[]\n",
      "['16937360', '30992265', '34605776', '31339233', '36228171', '27238576', '36349896', '31732958', '32448380', '21900192']\n",
      "['34561985', '38109536', '38349514', '28148298', '25344935', '38463392', '37350306', '32927603', '33144171', '38775138']\n",
      "[]\n",
      "[]\n",
      "['35965424', '34096649', '37001971', '36349896', '31643285', '30623346', '36180546', '38429096', '32648854', '20301617']\n",
      "['35292111']\n",
      "['33626253', '36762851', '36627265', '34048683', '37958658', '38563345', '37731379', '37558706', '37913909', '37307173']\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "[]\n",
      "['35836081', '38045286', '28862809', '35836082', '38559366', '30957296', '36555724', '35126293', '31670406', '26733686']\n",
      "[]\n",
      "FINAL RESULTS \n",
      "Mean precision = 0.055\n",
      "Mean Average Precision = 0.122\n",
      "Mean Time needed to execute a query = 1.060\n"
     ]
    }
   ],
   "source": [
    "evaluation_pubmed(data=data, query_type=\"PubMed website no mash\", mesh = False, path=\"Eval_results_BioASQ_subset/Pubmed_no_mesh.json\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Create json file for BioASQ questions with our retrieval results - for 182 question from BioASQ (contains exactly 10 relevant abstracts)\n",
    "#### Answer on BioASQ questions using our IR (0,7 0,3 combination) - look down after client initialization\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package punkt to\n",
      "[nltk_data]     C:\\Users\\Adela\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package punkt is already up-to-date!\n",
      "[nltk_data] Downloading package stopwords to\n",
      "[nltk_data]     C:\\Users\\Adela\\AppData\\Roaming\\nltk_data...\n",
      "[nltk_data]   Package stopwords is already up-to-date!\n"
     ]
    }
   ],
   "source": [
    "import nltk\n",
    "\n",
    "nltk.download('punkt')\n",
    "nltk.download('stopwords')\n",
    "\n",
    "from nltk.corpus import stopwords\n",
    "from nltk.tokenize import word_tokenize\n",
    "import pandas as pd\n",
    "\n",
    "\n",
    "\n",
    "# Create new query processor for extracting data for generative component\n",
    "class QueryProcessor_Generative:\n",
    "    def __init__(self, index_lexical:str = \"medline-faiss-hnsw-lexical\",lexical_pmid = False, index_name_semantic =\"medline-faiss-hnsw\", rescore = False, model=None, lexical_client=None, semantic_client=None, output_file_path=\"queries/queries.tsv\", stopwords=set([])):\n",
    "        self.index_lexical_name = index_lexical\n",
    "        self.index_name_semantic = index_name_semantic\n",
    "        # 2 index name (?)\n",
    "        self.model = model\n",
    "        #self.lexical_pmid = lexical_pmid\n",
    "        self.lexical_client = lexical_client\n",
    "        self.semantic_client = semantic_client\n",
    "        self.output_file_path = output_file_path\n",
    "        self.stop_words = stopwords\n",
    "        self.query_result = []\n",
    "        self.rescore = rescore\n",
    "        self.lexical_query = self.lexical_query_pmid if lexical_pmid else self.lexical_query\n",
    "    \n",
    "    def set_rescore(self, rescore):\n",
    "        self.rescore = rescore\n",
    "\n",
    "    def preprocess_query(self, query_str):\n",
    "        return ' '.join([word for word in word_tokenize(query_str) if word.lower() not in self.stop_words])\n",
    "\n",
    "    def save_results(self):\n",
    "        with open(self.output_file_path, \"w\") as file:\n",
    "            json.dump(self.query_result, file, indent=4)\n",
    "      \n",
    "    \n",
    "    def reorder_pmid(self, retrived_documents):\n",
    "        pmid_scores = {}\n",
    "        \n",
    "        # Iterate through the set data\n",
    "        for _, value in retrived_documents.items():\n",
    "            pmid = value['pmid']\n",
    "            score = value['score']\n",
    "            \n",
    "            # Check if pmid already exists in the dictionary\n",
    "            if pmid in pmid_scores:\n",
    "                pmid_scores[pmid] += score\n",
    "            else:\n",
    "                pmid_scores[pmid] = score\n",
    "           \n",
    "        return pmid_scores\n",
    "    \n",
    "    def lexical_query(self, query_str, limit=10):\n",
    "        if self.lexical_client == None:\n",
    "            raise ValueError(\"No Lexical client defined\")\n",
    "        \n",
    "        query = {\n",
    "                \"size\": limit,\n",
    "                \"query\": {\n",
    "                    \"multi_match\": {\n",
    "                        \"query\": query_str,\n",
    "                        \"fields\": [\"text\"]\n",
    "                    }\n",
    "                }\n",
    "            }\n",
    "       \n",
    "        results = self.lexical_client.search(index=self.index_lexical_name, body=query) \n",
    "        retrived_documents = {}\n",
    "        max_score = results['hits']['max_score']\n",
    "     \n",
    "        for hit in results[\"hits\"][\"hits\"]:\n",
    "            \n",
    "            pmid = hit[\"_source\"][\"pmid\"]\n",
    "            score = hit[\"_score\"]\n",
    "            auto_id = hit[\"_id\"]\n",
    "            \n",
    "            \n",
    "            \n",
    "            retrived_documents[auto_id] = {\n",
    "                \"score\": round(score/max_score, 5),\n",
    "                \"pmid\": pmid\n",
    "                }\n",
    "        \n",
    "        retrived_documents = self.reorder_pmid(retrived_documents)\n",
    "        return retrived_documents #adjust the return \n",
    "    \n",
    "    def lexical_query_pmid(self, query_str, limit=10):\n",
    "        #print(\"Lexical = \",query_str)\n",
    "        if self.lexical_client == None:\n",
    "            raise ValueError(\"No Lexical client defined\")\n",
    "        \n",
    "        query = {\n",
    "                \"size\": limit,\n",
    "                \"query\": {\n",
    "                    \"multi_match\": {\n",
    "                        \"query\": query_str,\n",
    "                        \"fields\": [\"full_text\"]\n",
    "                    }\n",
    "                }\n",
    "            }\n",
    "        \n",
    "        results = self.lexical_client.search(index=self.index_lexical_name, body=query) \n",
    "        \n",
    "        retrieved_documents = {}\n",
    "        max_score = results['hits']['max_score']\n",
    "        for hit in results[\"hits\"][\"hits\"]:\n",
    "            \n",
    "            pmid = hit[\"_source\"][\"pmid\"]\n",
    "            score = hit[\"_score\"] / max_score\n",
    "            \n",
    "            retrieved_documents[pmid] = score\n",
    "            \n",
    "        return retrieved_documents #adjust the return \n",
    "\n",
    "    def semantic_query(self, query, limit=10):\n",
    "        #print(\"semantic = \",query)\n",
    "        if self.semantic_client == None:\n",
    "            raise ValueError(\"No Semantic client defined\")\n",
    "        if self.model == None:\n",
    "            raise ValueError(\"No model defined\")\n",
    "        \n",
    "        query_vector = self.model.encode(query).tolist()\n",
    "    \n",
    "        search_params=models.SearchParams(\n",
    "            quantization=models.QuantizationSearchParams(rescore=self.rescore)\n",
    "            )\n",
    "        results = self.semantic_client.search(collection_name=self.index_name_semantic,query_vector=query_vector,search_params=search_params, limit=limit)\n",
    "    \n",
    "        #results = self.semantic_client.search(collection_name=self.index_name_semantic,query_vector=query_vector, limit=limit)\n",
    "        \n",
    "        retrived_documents = {}\n",
    "        max_score = None\n",
    "        for i,document in enumerate(results):\n",
    "            \n",
    "            pmid = document.payload['pmid']\n",
    "            score = document.score\n",
    "            if i == 0:\n",
    "                # first score is the max\n",
    "                max_score = score\n",
    "            retrived_documents[document.id] = { 'pmid': pmid, 'score': round(score / max_score, 5) } \n",
    "\n",
    "        retrived_documents = self.reorder_pmid(retrived_documents)\n",
    "        \n",
    "        return retrived_documents\n",
    "    \n",
    "\n",
    "    def hybrid_query(self, query_lexical, query_semantic, lex_parameter = 0.7, semantic_parameter = 0.3, limit=10):\n",
    "        if (lex_parameter + semantic_parameter) > 1:\n",
    "            raise ValueError(\"Uncorrect parameters for Hybrid Queries\")\n",
    "        #changed for hybrid (also need to be changed in general Query processor)\n",
    "        lexical_results = self.lexical_query(query_lexical, limit = limit) \n",
    "        semantic_results = self.semantic_query(query_semantic, limit)\n",
    "        max_score = 0\n",
    "        retrived_documents = {}\n",
    "        \n",
    "        for lex_pmid in lexical_results:\n",
    "            score = lexical_results[lex_pmid] * lex_parameter\n",
    "            if lex_pmid in semantic_results:\n",
    "                score += semantic_results[lex_pmid] * semantic_parameter\n",
    "\n",
    "            retrived_documents[lex_pmid] = score\n",
    "            max_score = max(max_score, score)\n",
    "            \n",
    "\n",
    "        for semantic_pmid in semantic_results:\n",
    "            if semantic_pmid not in lexical_results:\n",
    "                score = semantic_results[semantic_pmid] * semantic_parameter\n",
    "                retrived_documents[semantic_pmid] = score\n",
    "                max_score = max(max_score, score)\n",
    "                \n",
    "        return retrived_documents # just to have a starting point\n",
    "\n",
    " #changed line to retrieve exact number of documents (to return full_text), added return statement for the same reason\n",
    "    def execute_query(self, query_str, query_type='lexical', lex_parameter=0.7, semantic_parameter=0.3, limit=10, save=True, stopwords_preprocessing=True):\n",
    "        text_query = self.preprocess_query(query_str) if stopwords_preprocessing else query_str\n",
    "\n",
    "        if query_type == 'lexical':\n",
    "            results = self.lexical_query(text_query, limit=limit)\n",
    "        elif query_type == 'semantic':\n",
    "            results = self.semantic_query(query_str, limit=limit)\n",
    "        elif query_type == 'hybrid':\n",
    "            results = self.hybrid_query(text_query, query_str, lex_parameter, semantic_parameter, limit=limit)\n",
    "        else:\n",
    "            raise ValueError(\"Invalid query type specified. Choose 'lexical', 'semantic', or 'hybrid'.\")\n",
    "\n",
    "        document_retrieved = sorted(results.items(), key=lambda x: x[1], reverse=True)\n",
    "        document_retrieved = document_retrieved[:limit]  # Limit the number of documents\n",
    "\n",
    "        if save:\n",
    "            return self.process_results(document_retrieved, query_str, query_type)\n",
    "\n",
    "        # Optionally return the documents if not saving\n",
    "        return document_retrieved\n",
    "\n",
    "\n",
    " \n",
    "\n",
    "# changed to retrieve full_text (for creating custom dataset Bioasq question and our IR's retrieved abstracts)\n",
    "    def process_results(self, results, query_str, query_type):\n",
    "        retrieved_documents = []\n",
    "        for element in results:\n",
    "            pmid, _ = element\n",
    "            query = {\n",
    "                \"query\": {\n",
    "                    \"term\": {\n",
    "                        \"pmid\": int(pmid)\n",
    "                    }\n",
    "                }\n",
    "            }\n",
    "\n",
    "            # Fetching the document by pmid to get the full_text\n",
    "            results = self.lexical_client.search(index=self.index_lexical_name, body=query)\n",
    "            if results['hits']['hits']:\n",
    "                full_text = results['hits']['hits'][0][\"_source\"]['full_text']\n",
    "                full_text=' '.join(full_text.replace('\\n', ' ').split()) # remoce newlines\n",
    "                pmid = results['hits']['hits'][0][\"_source\"]['pmid']\n",
    "\n",
    "                retrieved_documents.append({\n",
    "                    \"pmid\": pmid,\n",
    "                    \"full_text\": full_text\n",
    "                })\n",
    "\n",
    "        dict_to_save = {'query': query_str, 'query_type': query_type, 'abstracts': retrieved_documents}\n",
    "        self.query_result.append(dict_to_save)\n",
    "        return retrieved_documents\n",
    "            \n",
    "\n",
    "        \n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Device cpu\n"
     ]
    }
   ],
   "source": [
    "import json\n",
    "\n",
    "\n",
    "index_name_lexical = \"medline-faiss-hnsw-lexical-pmid\"\n",
    "lexical_pmid = True\n",
    "\n",
    "model_card = 'sentence-transformers/msmarco-distilbert-base-tas-b'\n",
    "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
    "print(f\"Device {device}\")\n",
    "\n",
    "model=SentenceTransformer(model_card)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Comprehensive analysis of RET common and rare variants in a series of Spanish Hirschsprung patients confirms a synergistic effect of both kinds of events. BACKGROUND RET is the major gene associated to Hirschsprung disease (HSCR) with differential contributions of its rare and common, coding and noncoding mutations to the multifactorial nature of this pathology. In the present study, we have performed a comprehensive study of our HSCR series evaluating the involvement of both RET rare variants (RVs) and common variants (CVs) in the context of the disease. METHODS RET mutational screening was performed by dHPLC and direct sequencing for the identification of RVs. In addition Taqman technology was applied for the genotyping of 3 RET CVs previously associated to HSCR, including a variant lying in an enhancer domain within RET intron 1 (rs2435357). Statistical analyses were performed using the SPSS v.17.0 to analyze the distribution of the variants. RESULTS Our results confirm the strongest association to HSCR for the \"enhancer\" variant, and demonstrate a significantly higher impact of it in male versus female patients. Integration of the RET RVs and CVs analysis showed that in 91.66% of cases with both kinds of mutational events, the enhancer allele is in trans with the allele bearing the RET RV. CONCLUSIONS A gender effect exists on both the transmission and distribution of rare coding and common HSCR causing mutations. In addition, these RET CVs and RVs seem to act in a synergistic way leading to HSCR phenotype.\n"
     ]
    }
   ],
   "source": [
    "known_pmid = '15795476'\n",
    "known_pmid = '21995290'\n",
    "print(get_document_text(known_pmid, client_lexical))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "lexical_pmid = False\n",
    "\n",
    "if lexical_pmid:\n",
    "    index_name_lexical = 'medline-faiss-hnsw-lexical-pmid'\n",
    "else:\n",
    "    index_name_lexical ='medline-faiss-hnsw-lexical'\n",
    "\n",
    "coll_name_semantic = \"medline-faiss-hnsw\"\n",
    "\n",
    "query_parser_generative = QueryProcessor_Generative(index_lexical=index_name_lexical, lexical_pmid=lexical_pmid, index_name_semantic = coll_name_semantic, model= model, lexical_client=client_lexical, semantic_client=client_semantic)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'pmid': '10797929',\n",
       "  'full_text': 'Opercular epileptic syndrome: an unusual form of benign partial epilepsy in childhood. INTRODUCTION Functional opercular syndrome in childhood is an exceptional form of presentation of benign partial epilepsy with centro-temporal rolandic spikes (BECRS). CLINICAL CASES We studied the evolution of four patients, three of them followed for more than 15 years. Two were siblings, and their father suffered from BECRS with permanent language problems (verbal dyspraxia) and difficulty of protunding his tongue in adulthood. A third patient suffered benign familial neonatal convulsions (BFNC). In all four patients the actual illness begun as a BECRS with opercular troubles as an ictal phenomena. At about four years of age, the opercular disfunction became evident, with severe drooling, facial hypomobility and speech disturbance which waxed and vanished along weeks, months or years, apparently not ictal. Antiepileptic drugs not only were unable to control this situation but also, some of them, like carbamazepine, even worsened the opercular disfunction, increased the number of seizures and enhanced the neuropsychologic disfunction. Only clobazam could achieve the control on opercular disfunction. After 16 years, no further treatment was needed for all patients. There were some permanent sequelae, as speech and orolingual dyspraxia and different neuropsychologic problems. CONCLUSION Of noteworth the best performance was attained by the patient treated with clobazam on monotherapy.'},\n",
       " {'pmid': '9417187',\n",
       "  'full_text': \"The child with lip, maxillary, palatal cleft. Clefting of lip, alveolus and palate may occur in multiple variations. It causes aesthetic and functional detractions. Soft palate clefts may result in hearing-, speech- and swallowing-disorders. Therefore the otolaryngologist is a very important member in the interdisciplinary team directory. A cleft-palate child belongs to an interdisciplinary consulting hour in special hospitals, where different medical specialties are involved. Most important for a sufficient medical rehabilitation are maxillofacial surgery, otolaryngology, paediatrics, plastic surgery, speech therapy, psychology and human genetics. Also many other specialties may be involved. The cleft demands a complete follow up from the child's birth until it is grown up. Hearing disorders are caused by eustachian tube disfunction. There is a high prevalence of hearing loss and middle ear diseases in cleft palate patients. Hearing losses due to middle ear effusions in the very young child. Without therapy up to 50% of the cleft palate population will develop chronic middle ear diseases with and without cholesteatomas. Early and consequent therapy with myringotomy and insertion of a tympanostomy ventilation tube is necessary and helps to avoid chronic hearing problems. There is no general accepted system of speech disorders in cleft palate patients because of the difference in shaping of the cleft and rehabilitation development. Essential for speech rehabilitation are an intact velopharyngeal system and a keen sense of hearing. Both of it is disturbed in cleft palate children. Speech disorders are treated by speech therapists with prior consultation of the interdisciplinary team. The author presents a system of primary, secondary and tertiary speech disorders in cleft palate children. Primary speech disorders are caused by faulty velopharyngeal valving, offering in hypernasality, weak plosives, fricatives and affricates. Secondary speech disorders are substitute mechanisms for plosives, nasal and pharyngeal sounds. Tertiary speech disorders are hyper- and hypofunctional dysphonias following primary and secondary speech dysfunctions.\"},\n",
       " {'pmid': '21108403',\n",
       "  'full_text': 'Identification of a microdeletion at the 7q33-q35 disrupting the CNTNAP2 gene in a Brazilian stuttering case. Speech and language disorders are some of the most common referral reasons to child development centers accounting for approximately 40% of cases. Stuttering is a disorder in which involuntary repetition, prolongation, or cessation of the sound precludes the flow of speech. About 5% of individuals in the general population have a stuttering problem, and about 80% of the affected children recover naturally. The causal factors of stuttering remain uncertain in most cases; studies suggest that genetic factors are responsible for 70% of the variance in liability for stuttering, whereas the remaining 30% is due to environmental effects supporting a complex cause of the disorder. The use of high-resolution genome wide array comparative genomic hybridization has proven to be a powerful strategy to narrow down candidate regions for complex disorders. We report on a case with a complex set of speech and language difficulties including stuttering who presented with a 10 Mb deletion of chromosome region 7q33-35 causing the deletion of several genes and the disruption of CNTNAP2 by deleting the first three exons of the gene. CNTNAP2 is known to be involved in the cause of language and speech disorders and autism spectrum disorder and is in the same pathway as FOXP2, another important language gene, which makes it a candidate gene for causal studies speech and language disorders such as stuttering.'},\n",
       " {'pmid': '12060812',\n",
       "  'full_text': 'Language-impaired children: No sign of the FOXP2 mutation. A mutation in the FOXP2 gene has been found to be responsible for the autosomal dominant inheritance of a severe form of speech and language impairment in a family known as KE. We genotyped the FOXP2 mutation for 270 4-year-old children selected for low general language scores from a representative community sample of more than 18,000 children. No language-impaired child had the FOXP2 mutation. Although rare severe disorders such as those of the KE family are often caused by a single gene, common disorders such as language impairment are more likely to be the quantitative extreme of the same multiple genetic factors responsible for heritability throughout the distribution.'},\n",
       " {'pmid': '2149592',\n",
       "  'full_text': 'Language and speech disorders in preschool children and possibilities for rehabilitation--frequent ear-nose-throat findings that may be responsible for language and speech disorders. Speech disorders in preschool children generally have to be treated by speech therapy with continuing support of the parents. Depending on the severity and cause of the disorder supplementary therapeutic interventions in specialised institutions or individual psychotherapy may be necessary in addition to speech therapy. Careful differential diagnosis before starting any specific therapy is mandatory.'},\n",
       " {'pmid': '3343880',\n",
       "  'full_text': 'The role of adenoids in the development of normal speech following palate repair. The controversy surrounding the various approaches to palate repair continues unabated. Issues which have been studied relative to the development of normal speech following palate repair have included surgical technique and the timing of surgery. However, regardless of the various refinements in surgery, a percentage of children require secondary surgery to resolve velopharyngeal insufficiency. To date, other factors which might be responsible for the development of normal speech have been ignored. The purpose of this report is to discuss the importance of the relative size of the adenoids in relation to the success or failure of primary palatoplasty. Velopharyngeal valving in noncleft (normal) children was also observed. Eight hundred fifty children with nonsyndromic clefts and 138 children with syndromic clefts were analyzed for speech results postpalatoplasty relative to adenoid size. One hundred normal children were also studied. The data suggest a strongly positive correlation between the incidence of hypernasal resonance postpalatoplasty and relative adenoid size in the cleft children. Velar-adenoidal closure was consistently observed in both the cleft and normal children.'},\n",
       " {'pmid': '15841756',\n",
       "  'full_text': 'Differential motor speech outcomes in children treated for mid-line cerebellar tumour. PRIMARY OBJECTIVE To investigate the nature of the motor speech impairments and dysarthria that can arise subsequent to treatment for childhood mid-line cerebellar tumours (CMCT). RESEARCH DESIGN The motor speech ability of six cases of children with CMCT was analysed using perceptual and physiological measures and compared with that of a group of non-neurologically impaired children matched for age and sex. MAIN OUTCOME AND RESULTS Three of the children with CMCT were perceived to exhibit dysarthric speech, while the remaining three were judged to have normal speech. The speech disorder in three of the children with CMCT was marked by deviances in prosody, articulation and phonation. The underlying pathophysiology was linked to cerebellar damage and expressed as difficulty in co-ordinating the motor speech musculature as required for speech production. These deficits were not identified in the three non-dysarthric children with CMCT. CONCLUSION Differential motor speech outcomes occur for children treated for CMCT and these are discussed within the realm of possible mechanisms responsible for these differences. The need for further investigation of the risk factors for development of motor speech impairment in children treated for CMCT is also highlighted.'},\n",
       " {'pmid': '24052698',\n",
       "  'full_text': 'Detection of the GJB2 Mutation in Iranian Children with Hearing Loss Treated with Cochlear Implantation. The 35delG mutation in the gap junction protein, β2, 26kDa (GJB2) gene is the most common mutation that has been found in children with non syndromic hearing loss. Testing for the GJB2 gene mutation is simple and can directly answer the concerns of the parents about cause of the disorder and prognosis for their children. Cochlear implantation (CI) is one of the methods of hearing rehabilitation in patients with complete hearing loss. The present study was designed for genetic assessment of children who were referred for CI. Connexin 26 (Cx26) gene analyses were performed on 42 children with non syndromic hearing loss who were referred to the Baqiyatallah Hospital, Tehran, Iran for genetic consultation and CI. Clinical history was obtained and an examination conducted on each individual. Genomic DNA was extracted from peripheral blood and mutation identification of the Cx26 gene was performed by polymerase chain reaction (PCR) amplification and direct sequencing of the coding sequence of the gene. Cochlear implantation was performed for all patients and treatment response was assessed for all of them based on speech intelligibility rating (SIR) before and after CI. We found six patients (14.3%) with the 35delG mutation on the Cx26 gene, two homozygotes and four heterozygotes. No other mutation was detected. Treatment response in children with the homozygous 35delG mutation was better than in heterozygous patients, and treatment response in children with the mutation was better than in children with no mutation. Mutation screening for finding deafness causing mutations in the GJB2 gene is a useful predictor of post-implantation speech perception. We suggest microarray or other advanced mutation detection methods for assessment of other genes that might be responsible for non syndromic deafness.'},\n",
       " {'pmid': '22337755',\n",
       "  'full_text': 'Instant messages vs. speech: hormones and why we still need to hear each other. Human speech evidently conveys an adaptive advantage, given its apparently rapid dissemination through the ancient world and global use today. As such, speech must be capable of altering human biology in a positive way, possibly through those neuroendocrine mechanisms responsible for strengthening the social bonds between individuals. Indeed, speech between trusted individuals is capable of reducing levels of salivary cortisol, often considered a biomarker of stress, and increasing levels of urinary oxytocin, a hormone involved in the formation and maintenance of positive relationships. It is not clear, however, whether it is the uniquely human grammar, syntax, content and/or choice of words that causes these physiological changes, or whether the prosodic elements of speech, which are present in the vocal cues of many other species, are responsible. In order to tease apart these elements of human communication, we examined the hormonal responses of female children who instant messaged their mothers after undergoing a stressor. We discovered that unlike children interacting with their mothers in person or over the phone, girls who instant messaged did not release oxytocin; instead, these participants showed levels of salivary cortisol as high as control subjects who did not interact with their parents at all. We conclude that the comforting sound of a familiar voice is responsible for the hormonal differences observed and, hence, that similar differences may be seen in other species using vocal cues to communicate.'},\n",
       " {'pmid': '34238105',\n",
       "  'full_text': \"Conversational speech of school-age children after syllable-timed speech treatment for stuttering. Purpose: The purpose of this laboratory study was to investigate whether rhythmic speech was primarily responsible for stuttering reductions in four school-aged children after the instatement stage of the Westmead Program of syllable-timed speech (STS) intervention. The study was designed to inform further development of the program. Reduction in variability of vowel duration is a marker of STS, and it was predicted that this would be present in the children's conversational speech after Stage 1 of the program if they were using STS. To strengthen such a finding, it was also predicted that there would be no reduction in articulation rate, sentence complexity, and utterance length after treatment, as there is evidence that reductions in these can reduce stuttering. Perceptual judgments of speech quality after treatment were also made by independent listeners.Method: Participants were four children, ages 8-11 years, who completed Stage 1 of an STS program and whose stuttering had reduced significantly. Pre-treatment (PRE) and post-treatment (POST) within-clinic audio-visual recordings of conversational speech were analysed for percentage of syllables stuttered, variability of vowel duration, articulation rate, and length and complexity of utterance. Four blinded listeners made perceptual judgments of speech quality in the POST recordings.Result: Recordings of all children showed that variability of vowel duration clearly reduced from the PRE to POST speech samples. Importantly, articulation rate and language use were not compromised. Some possible indicators of rhythmicity were identified in one child in the perceptual study.Conclusion: The findings suggest that STS was primarily responsible for the clinically significant reductions in stuttering after Stage 1 of the program. There is an urgent need for more evidence-based interventions for stuttering in this age group and further development of STS interventions is warranted.\"}]"
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query_parser_generative.execute_query(query_str=\"Which gene is responsible for disfunction in speech for children?\", query_type='hybrid')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "english_stopwords = set(stopwords.words('english'))\n",
    "\n",
    "def query_ir_system(query_str, query_type='hybrid', lex_parameter=0.7, semantic_parameter=0.3, limit=10):\n",
    "    # You would need to initialize the QueryProcessor with actual parameters including the model and clients\n",
    "    qp = QueryProcessor_Generative(\n",
    "        model=model,  \n",
    "        lexical_client=client_lexical,  \n",
    "        semantic_client=client_semantic  \n",
    "    )\n",
    "    \n",
    "    # Execute the query using the hybrid method by default\n",
    "    results = qp.execute_query(query_str, query_type=query_type, lex_parameter=lex_parameter, semantic_parameter=semantic_parameter, limit=limit)\n",
    "    #print(results)\n",
    "    #print('/n')\n",
    "    # Convert results to the required format\n",
    "    #abstracts = [{'id': pmid, 'text': text} for pmid, text in results]\n",
    "    abstracts = [{'id': pmid, 'text': full_text, 'score': score} for pmid, full_text, score in results]\n",
    "\n",
    "    #abstracts = [{'id': result['pmid'], 'text': result['text']} for result in results]\n",
    "\n",
    "    return abstracts\n",
    "\n",
    "def process_questions(file_path):\n",
    "    with open(file_path, 'r') as file:\n",
    "        data = json.load(file)\n",
    "    \n",
    "    results = {'questions': []}\n",
    "    \n",
    "    for question in data['questions']:\n",
    "        question_text = question['body']\n",
    "        question_id = question['id']\n",
    "        question_type = question.get('type', 'summary')  # Assume 'summary' if not specified\n",
    "        \n",
    "        # Query the IR system to get abstracts\n",
    "        abstracts = query_parser_generative.execute_query(question_text,lex_parameter=0.7, semantic_parameter=0.3,limit=10, query_type='hybrid') \n",
    "        \n",
    "        results['questions'].append({\n",
    "            'id': question_id,\n",
    "            'question': question_text,\n",
    "            'type': question_type,\n",
    "            'abstracts': abstracts\n",
    "        })\n",
    "    \n",
    "    return results\n",
    "\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Output written to Generative/BioASQ_10_abstracts-AbbrsractFromOurIR.json\n"
     ]
    }
   ],
   "source": [
    "# Path to the file containing the questions\n",
    "\n",
    "file_path = 'BioASQ_10_abstracts.json'\n",
    "\n",
    "# Process the questions and retrieve abstracts\n",
    "output_data = process_questions(file_path)\n",
    "\n",
    "# Save the output data to a JSON file\n",
    "output_file_path = 'Generative/BioASQ_10_abstracts-AbbrsractFromOurIR.json'\n",
    "with open(output_file_path, 'w') as output_file:\n",
    "    json.dump(output_data, output_file, indent=4)\n",
    "\n",
    "print(f\"Output written to {output_file_path}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "[{'pmid': '31431276', 'full_text': 'Current trends in sport and exercise hip conditions: Intra-articular and extra-articular hip pain, with detailed focus on femoroacetabular impingement (FAI) syndrome. Musculoskeletal conditions, such as hip pain are leading causes of pain and disability. Hip pain is the second most common cause of lower limb musculoskeletal pain, and is commonly seen in active individuals. Hip and groin pain may have intra-articular and extra-articular causes. Femoroacetabular impingement (FAI) syndrome and the associated pathologies are common intra-articular causes of hip and groin pain in active individuals. There are also a number of extra-articular causes of pain, which include musculotendinous conditions, extra-articular impingements and the clinical entities of groin pain described in the Doha agreement. This chapter will describe these, with a detailed focus on FAI syndrome. Specifically, it addresses: 1. What is and what causes FAI syndrome; 2. How do I diagnose FAI syndrome; and 3. What is the evidence-based approach to managing FAI syndrome?'}, {'pmid': '1570146', 'full_text': 'Heterotopic bone formation after total hip arthroplasty. Formation of heterotopic bone can cause pain and limit motion, thus ruin what would otherwise have been an excellent result after total hip arthroplasty. The cause of bone formation remains controversial, and diphosphonates have been ineffective in its prevention. Radiotherapy and nonsteroidal anti-inflammatory drugs, however, are effective preventative agents. Once the bone is formed, however, surgical excision is the only effective treatment.'}, {'pmid': '26692249', 'full_text': 'Quadratus femoris tear as an unusual cause of hip pain: a case report. PURPOSE To show that quadratus femoris muscle partial or complete tear is an extremely rare if not underdiagnosed cause of hip pain. METHODS We present the case of a 75-year-old Caucasian woman with spontaneous non-traumatic quadratus femoris tear. RESULTS Computed Tomography showed quadratus femoris tear with occult underlying ischiofemoral impingement pathology. CONCLUSIONS Quadratus femoris muscle is a hip external rotator and adductor and is situated along the posterior aspect of the hip joint and its proximity of anatomy with other external rotators leads to diagnostic pitfalls for radiologists. Identification of the exact muscle tear will establish the diagnosis, guide the proper physical therapy and reliably inform the patient what to expect after treatment.'}, {'pmid': '30509429', 'full_text': 'Editorial Commentary: When Your Golf Game Is Handicapped by Your Hip. Golf is a common recreational and competitive sport that requires full hip rotation to allow for a smooth and effective swing. Therefore hip impingement and hip osteoarthritis, by limiting rotation, could cause pain in golfers and even encourage them to discontinue the sport. In my opinion, the lead hip in golfers is likely what generally drives the symptoms in the hip, back, or knee. After surgical correction, golfers are able to get back to golf and many experience improved performance.'}, {'pmid': '37751184', 'full_text': 'Hip Pain and Physical Therapy Nonarthritic Hip Pain: What Is It? What Should I Do About It? How Do I Get Back to My Regular Activities? Hip pain can have many causes, both arthritic and non-arthritic. Experts in managing hip pain developed a guideline to help rehabilitation clinicians help you understand more about hip pain. Here we explain what physical therapy might offer you if you have hip pain and arthritis has been ruled out. You will find out about some of the leg strength and balance tests your physical therapist might ask you to do, and exercises to help alleviate your hip pain and improve your strength, movement and balance.J Orthop Sports Phys Ther 2023;53(10):1-2. doi:10.2519/jospt.2023.12236.'}, {'pmid': '37777849', 'full_text': 'Hip Pain and Physical Therapy Nonarthritic Hip Pain: What Is It? What Should I Do About It? How Do I Get Back to My Regular Activities? Hip pain can have many causes, both arthritic and non-arthritic. Experts in managing hip pain developed a guideline to help rehabilitation clinicians help you understand more about hip pain. Here we explain what physical therapy might offer you if you have hip pain and arthritis has been ruled out. You will find out about some of the leg strength and balance tests your physical therapist might ask you to do, and exercises to help alleviate your hip pain and improve your strength, movement and balance.J Orthop Sports Phys Ther 2023;53(10):643-644. doi:10.2519/jospt.2023.0506.'}, {'pmid': '33132093', 'full_text': 'Greater trochanteric pain after primary total hip replacement, comparing the anterior and posterior approach: A secondary analysis of a randomized trial. BACKGROUND Greater trochanteric pain (GTP) after total hip replacement is a common cause of residual lateral hip pain, regardless of the approach used. The goal of our study was to evaluate GTP after a direct anterior approach (DAA) compared to a posterior approach (PA) as well as the clinical outcomes of both approaches and answer the following: 1) What is the incidence of trochanteric pain after primary THA with two different surgical approaches? 2) What is the functional outcome of patients with GTP? 3) What proportion of patients with GTP resort to peritrochanteric injections? HYPOTHESIS Our hypothesis is that GTP is present with both approaches but satisfaction is lower with the PA. PATIENTS AND METHODS A secondary analysis of a previously published clinical trial with 55 total hip arthroplasty patients randomized in one of two surgical approaches: 27 patients underwent the anterior modified Hueter approach, while the other group of 28 patients were operated using the posterior approach. Study outcomes were Modified Harris Hip Score (MHHS), satisfaction score, pain when lying on the affected side, and requiring an injection. Hip offset, femur lateralization and leg lengthening were measured before and after surgery. RESULTS Forty-five patients were available for complete follow-up at a mean of 62 months (range: 48-74). The incidence of GTP was higher in the posterior approach [PA: 6/21 (29%) vs DAA 4/24 (17%)) (p=0.3). Patients operated through a PA experienced more pain [5/21 (24%) of patients; VAS=mean 5.3] when lying on their operated side, compared to DAA patients [2/24 (8%) of patients; VAS=mean 2) (p=0.2)]. However, MHHS, patient satisfaction with surgery, radiological assessment for hip offset, femur lateralization or leg lengthening, and injections required were similar for both approaches. Overall, satisfaction and functional outcome with surgery was significantly lower in GTP patients, regardless of the approach. CONCLUSIONS GTP impacts patient satisfaction and functional outcome in total hip arthroplasty patients. PA patients reported more trochanteric pain than DAA patients, which affected their clinical outcome. LEVEL OF EVIDENCE III.'}, {'pmid': '31272626', 'full_text': 'Editorial Commentary: The Gain in Pain After Hip Arthroscopic Surgery: What Is Clinically Relevant, and Is Pain Related to Function in Patients With Femoroacetabular Impingement Syndrome? Hip arthroscopic surgery improves patient-reported outcomes in patients with femoroacetabular impingement syndrome and/or hip chondropathy. The minimal clinically important change on a pain visual analog scale has now been calculated in this patient group, and the pain level 1 year after surgery has been related to function. Next step: identifying what causes pain and decreased function 1 year after surgery.'}, {'pmid': '38090918', 'full_text': 'Hip Pain in Adolescent Patients. Hip pain is a common complaint in adolescents. There are several causes for hip pain in this population, with dysplasia and impingement being the most common; however, other conditions such as extra-articular impingement, torsional disorders, labral tears, and osteochondral lesions also require consideration. Many of these conditions are related to underlying anatomic abnormalities and increased activity in this age group. An understanding of the common pathologies of the adolescent hip is integral to the evaluation, diagnosis, and treatment of these patients.'}, {'pmid': '36621448', 'full_text': 'Clinical and Radiographic Criteria Define \"Acceptable\" Surgical Correction of Hip Femoroacetabular Impingement Syndrome as Well as Postoperative Complications: An International Modified Delphi Study. OBJECTIVES To develop recommendations for clinical and radiographic criteria to help define the \"acceptable\" surgical correction of femoroacetabular impingement syndrome (FAIS) and identify/define complications postoperatively. METHODS A 3-phase modified Delphi study was conducted involving a case-based survey; a Likert/multiple choice-based survey concerning radiographic and physical examination characteristics to help define FAIS correction, as well as the prevalence and definition of potential postoperative complications; and 2 consensus meetings. RESULTS Of the 75 experts invited, 54 completed the Phase I survey, 50 completed the Phase II survey (72% and 67% response rate), and 50 participated in the Phase III consensus meetings. For both typical and atypical (complex) cases, there was consensus that fluoroscopy with multiple views and dynamic hip assessment should be used intraoperatively (96% and 100%, respectively). For typical FAIS cases, the Expert Panel agreed that Dunn lateral and anteroposterior radiographs were the most important radiographs to evaluate the hip postoperatively (88%, consensus). When asked about evaluating the correction of cam impingement postoperatively, 87% voted that they use subjective evaluation of the \"sphericity\" of the femoral head. In the case of focal and global pincer-type FAIS, there was consensus that the reduction or elimination of the crossover sign (84%) and lateral center-edge angle (91%) were important to inform the extent of the FAIS correction. There was consensus for recommending further investigation at 6 months postoperatively if hip pain had increased/plateaued (92% agreed); that additional investigation and treatment should occur between 6 and 12 months (90% agreed); and that a reoperation may be recommended at 12 months or later following this investigation period (89% agreed). CONCLUSIONS This consensus project identified the importance of using fluoroscopy and dynamic hip assessment intraoperatively; Dunn lateral and anteroposterior view radiographs postoperatively; evaluating the \"sphericity\" of the femoral head for cam-type correction and the use of dynamic hip assessment; reducing/eliminating the crossover sign for focal pincer-type FAIS; evaluating the lateral center-edge angle for global pincer-type FAIS; and avoiding overcorrection of pincer-type FAIS. In cases in which postoperative hip pain increased/plateaued, further investigation and treatment is warranted between 6 and 12 months, and a reoperation may be recommended at a minimum of 12 months depending on the cause of the hip pain. CLINICAL RELEVANCE Hip arthroscopy surgeons have yet to reach a firm agreement on what constitutes an \"acceptable\" or \"good\" surgery radiographically and how they can achieve desired clinical outcomes. Although this was a comprehensive effort, more study is needed to determine therapeutic thresholds that can be universally applied.'}]\n"
     ]
    }
   ],
   "source": [
    "known_pmid = '15795476'\n",
    "qp = QueryProcessor_Generative(\n",
    "        model=model,  # You need to define or load your model\n",
    "        lexical_client=client_lexical,  # You need to define your ElasticSearch client or equivalent for lexical queries\n",
    "        semantic_client=client_semantic  # Define your semantic search client\n",
    "    )\n",
    "print(qp.execute_query(\"What is the cause of hip pain\",query_type='hybrid',lex_parameter=0.7, semantic_parameter=0.3,limit=10))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 42,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'31431276': 1.0, '1570146': 0.9046, '26692249': 0.90139, '30509429': 0.89442, '37751184': 0.88643, '37777849': 0.88643, '33132093': 0.86732, '31272626': 0.84404, '36621448': 0.84001, '1622374': 0.83702}\n"
     ]
    }
   ],
   "source": [
    "#print(qp.hybrid_query(client_lexical,client_semantic,lex_parameter=0.7,semantic_parameter=0.3,limit=10))\n",
    "print(qp.lexical_query(\"What is the cause of hip pain\",limit=10))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {},
   "outputs": [
    {
     "ename": "ValueError",
     "evalue": "not enough values to unpack (expected 3, got 2)",
     "output_type": "error",
     "traceback": [
      "\u001b[1;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[1;31mValueError\u001b[0m                                Traceback (most recent call last)",
      "Cell \u001b[1;32mIn[43], line 2\u001b[0m\n\u001b[0;32m      1\u001b[0m known_pmid \u001b[38;5;241m=\u001b[39m \u001b[38;5;124m'\u001b[39m\u001b[38;5;124m15795476\u001b[39m\u001b[38;5;124m'\u001b[39m\n\u001b[1;32m----> 2\u001b[0m \u001b[38;5;28mprint\u001b[39m(query_ir_system(\u001b[38;5;124m\"\u001b[39m\u001b[38;5;124mWhat is the cause of hip pain\u001b[39m\u001b[38;5;124m\"\u001b[39m, query_type\u001b[38;5;241m=\u001b[39m\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mhybrid\u001b[39m\u001b[38;5;124m'\u001b[39m, lex_parameter\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m0.7\u001b[39m, semantic_parameter\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m0.3\u001b[39m, limit\u001b[38;5;241m=\u001b[39m\u001b[38;5;241m10\u001b[39m))\n",
      "Cell \u001b[1;32mIn[39], line 17\u001b[0m, in \u001b[0;36mquery_ir_system\u001b[1;34m(query_str, query_type, lex_parameter, semantic_parameter, limit)\u001b[0m\n\u001b[0;32m     12\u001b[0m results \u001b[38;5;241m=\u001b[39m qp\u001b[38;5;241m.\u001b[39mexecute_query(query_str, query_type\u001b[38;5;241m=\u001b[39mquery_type, lex_parameter\u001b[38;5;241m=\u001b[39mlex_parameter, semantic_parameter\u001b[38;5;241m=\u001b[39msemantic_parameter, limit\u001b[38;5;241m=\u001b[39mlimit)\n\u001b[0;32m     13\u001b[0m \u001b[38;5;66;03m#print(results)\u001b[39;00m\n\u001b[0;32m     14\u001b[0m \u001b[38;5;66;03m#print('/n')\u001b[39;00m\n\u001b[0;32m     15\u001b[0m \u001b[38;5;66;03m# Convert results to the required format\u001b[39;00m\n\u001b[0;32m     16\u001b[0m \u001b[38;5;66;03m#abstracts = [{'id': pmid, 'text': text} for pmid, text in results]\u001b[39;00m\n\u001b[1;32m---> 17\u001b[0m abstracts \u001b[38;5;241m=\u001b[39m [{\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mid\u001b[39m\u001b[38;5;124m'\u001b[39m: pmid, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mtext\u001b[39m\u001b[38;5;124m'\u001b[39m: full_text, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mscore\u001b[39m\u001b[38;5;124m'\u001b[39m: score} \u001b[38;5;28;01mfor\u001b[39;00m pmid, full_text, score \u001b[38;5;129;01min\u001b[39;00m results]\n\u001b[0;32m     19\u001b[0m \u001b[38;5;66;03m#abstracts = [{'id': result['pmid'], 'text': result['text']} for result in results]\u001b[39;00m\n\u001b[0;32m     21\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m abstracts\n",
      "Cell \u001b[1;32mIn[39], line 17\u001b[0m, in \u001b[0;36m<listcomp>\u001b[1;34m(.0)\u001b[0m\n\u001b[0;32m     12\u001b[0m results \u001b[38;5;241m=\u001b[39m qp\u001b[38;5;241m.\u001b[39mexecute_query(query_str, query_type\u001b[38;5;241m=\u001b[39mquery_type, lex_parameter\u001b[38;5;241m=\u001b[39mlex_parameter, semantic_parameter\u001b[38;5;241m=\u001b[39msemantic_parameter, limit\u001b[38;5;241m=\u001b[39mlimit)\n\u001b[0;32m     13\u001b[0m \u001b[38;5;66;03m#print(results)\u001b[39;00m\n\u001b[0;32m     14\u001b[0m \u001b[38;5;66;03m#print('/n')\u001b[39;00m\n\u001b[0;32m     15\u001b[0m \u001b[38;5;66;03m# Convert results to the required format\u001b[39;00m\n\u001b[0;32m     16\u001b[0m \u001b[38;5;66;03m#abstracts = [{'id': pmid, 'text': text} for pmid, text in results]\u001b[39;00m\n\u001b[1;32m---> 17\u001b[0m abstracts \u001b[38;5;241m=\u001b[39m [{\u001b[38;5;124m'\u001b[39m\u001b[38;5;124mid\u001b[39m\u001b[38;5;124m'\u001b[39m: pmid, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mtext\u001b[39m\u001b[38;5;124m'\u001b[39m: full_text, \u001b[38;5;124m'\u001b[39m\u001b[38;5;124mscore\u001b[39m\u001b[38;5;124m'\u001b[39m: score} \u001b[38;5;28;01mfor\u001b[39;00m pmid, full_text, score \u001b[38;5;129;01min\u001b[39;00m results]\n\u001b[0;32m     19\u001b[0m \u001b[38;5;66;03m#abstracts = [{'id': result['pmid'], 'text': result['text']} for result in results]\u001b[39;00m\n\u001b[0;32m     21\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m abstracts\n",
      "\u001b[1;31mValueError\u001b[0m: not enough values to unpack (expected 3, got 2)"
     ]
    }
   ],
   "source": [
    "known_pmid = '15795476'\n",
    "print(query_ir_system(\"What is the cause of hip pain\", query_type='hybrid', lex_parameter=0.7, semantic_parameter=0.3, limit=10))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
